<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=140 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:20:56 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 5 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 3 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>February 19, 2016</h3>
<div class="news-item">
 <div class="date">01:46</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139591863116">Postdoc: UMassachusetts.ComparativeEndocrinology</a></div>
  <div class="description">POSTDOCTORAL Research Associate

A postdoctoral fellowship in comparative endocrinology is available at
the University of Massachusetts, Amherst beginning April 1, 2016.  The
fellowship is supported by an NSF grant with funding for 4 years and
will examine the hormonal control of osmoregulation in lamprey.

This is a benefited, full-time Postdoctoral Research Associate
position.  Initial appointment is for one year, reappointment beyond
the first year is contingent upon availability of funding and job
performance.  Primary responsibilities will include, but are not
limited to:  real time PCR measurement of ion transporter, hormone and
receptor mRNA levels; in situ hybridization of cells expressing ion
transporters, hormones and receptors; immunohistochemical and
microscopic analysis of pituitary and gill; maintenance and
experimental manipulation of juvenile lamprey; manipulation of hormone
levels in vivo and in vitro.  Qualified candidates are required to have
a Ph.D. in Biology or related field.  Two to three years experience
studying physiology, endocrinology and/or using molecular approaches in
fish biology research is preferred. Experience with radioimmunoassays,
in situ hybridization, and molecular genetic approaches is highly
desirable.

It is expected that the appointee will work extended hours as necessary
to complete individual experiments.

Postdoctoral Research Associates at the University of Massachusetts are
unionized and receive standard salary and benefits, depending on
experience.  Salary is subject to bargaining unit contract.
Candidates must apply online by submitting a cover letter, CV, summary of resear
ch interests, and the contact details of three references willing to provide let
ters of recommendation to:

<a href="http://bit.ly/1Q4Q5UP">http://bit.ly/1Q4Q5UP</a>

Questions can be addressed to Dr. Stephen McCormick at
mccormick@umext.umass.edu.

Review of applications will begin March 7, 2016 and continue until the
position is filled.  Applications received by March 7th will be given
priority consideration.

The University of Massachusetts Amherst is an Affirmative Action/Equal
Opportunity Employer of women, minorities, protected veterans, and
individuals with disabilities and encourages applications from these
and other protected group members.

Lisa Barry
Biology Department
Morrill Science Center South, Room 348
611 North Pleasant Street
University of Massachusetts
Amherst, MA  01003

Phone:  413-545-2602
Fax:  413-545-3243


Lisa Barry 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:13</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139590980556">Graduate position: IBDM_Marseille.EvolutionWingPigmentation</a></div>
  <div class="description">PhD position available in the Prud'homme lab (Marseille, France)

Evolution of the gene network controlling wing pigmentation in
Drosophila

We are looking for a PhD student to study the evolution of the gene
regulatory network controlling the formation of a wing pigmentation
pattern in Drosophila species. This wing spot has emerged from a
spot-less ancestor, around 15 millions years ago, and then diversified
in shape, color and intensity between species.

The goal of the project is to peer into the genomic changes responsible
for these different evolutionary transitions. The student will use
comparative functional genomics across species to identify candidate
genes and cis-regulatory sequences associated with these transitions.
These candidates will be further validated in vivo by functional
manipulations using genome editing approaches.

Ultimately, these results will help to better understand how a gene
regulatory network emerge during evolution and give rise to a novel
morphological trait, and how alterations of this network underlie
morphological diversification of a morphological trait.

Candidates (from any nationality, with no requirement to understand
French) are expected to have a background in developmental biology,
genetics, and a strong interest in evolution.

Please send a CV, a motivation letter, a description of research
experience and interests and e-mail contact for 2-3 references to
benjamin.prudhomme@univ-amu.fr

The position is funded for 3 years by an ERC grant and must start
before July 1st 2016.

Our lab is part of the Institute of Developmental Biology of Marseille
(IBDM), an interdisciplinary research center studying developmental
biology and neurobiology. More information about the lab and the
institute can be found here: <a href="http://www.prudhommelab.com/">www.prudhommelab.com</a>  &
<a href="http://www.ibdm.univ-mrs.fr/">www.ibdm.univ-mrs.fr</a>


Benjamin Prud'homme 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:31</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139589738296">Course: UBirmingham.EnvironmentalGenomics.Mar6-11</a></div>
  <div class="description"><p>NERC-MDIBL Environmental Genomics and Metabolomics Course, 2016

Online Registration Site: <a href="http://bit.ly/1A8xykv">http://bit.ly/1A8xykv</a>

Date: March 6-11, 2016

Location: The University of Birmingham, UK

Number of Places: up to 50

Participation: Open to everyone.

Priority is given to NERC-funded PhD students and early-career scientists.

Bursaries are available covering registration fee.  Please send email
to J.K.Colbourne@bham.ac.uk to inquire about eligibility.

Costs for Non-NERC Funded Researchers: Standard 2340.

Course Leaders: John Colbourne, Joseph Shaw, Xin Zhou and Ben Brown
(for genomics). Mark Viant and Warwick Dunn plus staff from the NERC
Biomolecular Analysis Facility and Birmingham Metabolomics Training
Facility (for metabolomics).

Research Organisations Involved: University of Birmingham (UK), BGI
China National GeneBank (Shenzhen, China), Mount Desert Island Biological
Laboratory (USA)

Online Registration Site: <a href="http://bit.ly/1A8xykv">http://bit.ly/1A8xykv</a>

DESCRIPTION
This training opportunity is the third European/UK installment of a
successful annual course originating from Mount Desert Island Biological
Laboratory. The curriculum is expanded to highlight a multi-omics (system
biology) approach to research in environmental sciences. It trains
PhD students and early career postdoctoral scientists to investigate
how gene function and metabolism are influenced by environmental
conditions while accounting for variation that exists within and among
natural populations. The course is built on the paradigm that this
multidisciplinary research field encompassing ecology, evolution,
toxicology, biostatistics and informatics will most effectively grow
by training early career environmental scientists to properly design
comprehensive, large-scale, Next Generation Sequencing and Metabolomics
experiments enabled by drastically increased sample-throughput and lower
costs. Most importantly, the challenges of manipulating and analysing
population-level omics (big) data must be addressed.

The course provides a significant introduction and much hands-on training
experience so that participants can initiate their own environmental
omics study and network with others in the field to launch Environmental
Scientist careers in academia and industry. Case studies using multi-omics
data sets collected at the University of Birmingham will be provided
so that you can gain practical experience of analysing and integrating
multi-omics data.

Course Details
The course will include ~1.5 days of generic environmental omics training
and ~3.5 days of specialised genomics or metabolomics training (parallel
tracks).

The combined sessions will include:
* Overviews of environmental genomics and metabolomics
* Practical experience of analysing and integrating multi-omics data
* Synthesis sessions to obtaining grant funding
* Question and answer sessions with a panel of experts

The genomics track includes:
* Library construction methods and QC
* Introduction to automation systems and sequence data workflow
* Software solutions for sequence workflows and their application in
  environmental research
* RNA-seq alignment
* Statistical considerations for analysing genome-scale data
* Practical experience in applying open-source analysis tools to
  visualise complex sequence data and explore genome sequence variation,
  analyse Tuxedo output and perform gene set enrichment analysis
* Synthesis sessions to explore using the most appropriate model
  organism and maximising outputs from your sequence data

The metabolomics track includes:
* Experimental design
* Quality assurance and quality control in metabolomics
* Hands-on sample preparation
* Analytical technologies including mass spectrometry and NMR
  spectroscopy
* Data processing, including LC-MS and SIMS stitching procedures
* Univariate and multivariate data analysis approaches
* Metabolite identification

Evening Lecture Series
Distinguished visiting academics will provide keynote lectures on each
day of the course to highlight environmental omic applications that draw
on the expertise of our guest faculty.

Daphnia is used for training because of its growing use as a model system
in the environmental sciences and for improving environmental health
protection, yet the skills learned during the course will be applicable
to all study systems with maturing genomics and metabolomics resources.

j.k.colbourne@bham.ac.uk


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>February 18, 2016</h3>
<div class="news-item">
 <div class="date">23:46</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139588148256">Graduate position: UEastAnglia.PopulationGenomics</a></div>
  <div class="description"><p>What: FULLY FUNDED PHD STUDENTSHIP - Population demography and thermal
adaptation in a model pest insect, supervised by Dr Lewis Spurgin and
Professor Matt Gage

Where: University of East Anglia, Norwich, UK


Details:

Whats the project and why is it important?

Climate change is causing substantial shifts in the range and abundance
of crop pests and pathogens, with potentially disastrous consequences
for food security. Studying adaptation to climate in an evolutionary
framework is a powerful way for identifying how, when and where pests
will respond to climate change. However, our understanding of how natural
selection enables individuals and populations to adapt to climate change,
and the specific traits or genes responsible, is limited.

This PhD studentship will use experimental evolution and novel
genomic approaches to understand how insect pests adapt to challenging
environments. Specifically, we will use ongoing and new experiments of
the pest beetle Tribolium castaneum to test how changes in demography
(e.g. population size and migration) affect adaptation to changing
temperatures. There will also be an opportunity to use genome sequencing
to identify the genetic basis of adaptation to climate. We expect the
project to significantly advance our understanding of how populations,
individuals and genomes adapt to novel climates, and identify the impacts
of demography and adaptation on natural and pest populations. The project
therefore addresses both a fundamental question in biology and a key
societal challenge.


What other training is involved?

The student will receive training in ecology, evolutionary genetics
and bioinformatics. UEA has an excellent training service for PhD
students, equipping them for a wide range of careers within or outside
of academia. Just round the corner, The Genome Analysis Centre has
an excellent training programme for genomics and bioinformatics. The
student will also be encouraged to attend external training courses,
and national and international conferences.


Why work with us?

1. Great supervision. The student will be supervised by Dr Lewis Spurgin
   and Prof Matt Gage. Together, we have broad expertise in evolutionary
   biology, population genetics, genomics, ecology, animal behaviour and
   molecular biology. We aim to provide the right balance of being
   supportive and inclusive, while also giving you the freedom to
   develop your own ideas.

2. Great Research Environment. Norwich is ranked 4th in the UK for the
   number of highly cited scientists, after Cambridge, Oxford and
   London. Within UEA, the School of Biological Sciences at UEA is a world-
   leading research department with a dynamic community of PhD students
   and researchers.

3. Great city. In a recent survey in the Guardian, Norwich was voted the
   &lsquo;happiest&rsquo; place to work in the UK. The stunning Norfolk Broads
   and North Norfolk Coast are also right on your doorstep.


About you

Applicants should:
- Be creative
- Be passionate about research, and about using research to address big
  societal challenges
- Have excellent analytical skills
- Be willing to learn how to code (e.g. in R or python)
- Be willing to actively contribute to the research group and wider
  research community

Apply

Interested applicants can get in touch with me (l.spurgin[at]uea.ac.uk)
or Matt (m.gage[at]uea.ac.uk). You can apply online, and view funding
details, via the UEA website:

<a href="http://bit.ly/1KYLdAc">http://bit.ly/1KYLdAc</a>

The application deadline is 15th April.

lewisspurgin@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">17:54</div>
 <div class="body">
  <div class="title"><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a></div>
  <div class="description">Ammonite phylogeny has mainly been established based on a stratigraphic approach, with cladistics underconsidered. The main arguments against the use of cladistics are the supposed large amount of homoplasy and the small number of characters. Resolving the phylogeny of the Hildoceratidae (Early Jurassic) is especially challenging because of its large diversity and disparity. Many forms that have not been determined as closely related in previous studies exhibit very similar shapes. Moreover, some groups are morphologically very different, adding difficulties to building a unique coding scheme at a low taxonomic resolution (i.e. species). Here we propose an integrated coding scheme of the peristome shape and the ornamentation, allowing an increased level of comparison. The shape of the peristome is used as a new reference to locate ornamental features and propose new homology hypotheses. In total, 105 taxa have been analysed for 47 characters. We code continuous characters by their means and ranges ± one standard deviation. We test two weighting schemes: equal weights standardized by unit range and implied weighting with several concavity constants. This work has led to redefinition of the phylogenetic inclusivenesses of all the hildoceratid subfamilies. The new coding scheme based on peristome shapes provides the fewest homoplastic characters. The schemes appear promising to improve phylogenetic analyses in ammonoids as well as molluscs as a whole by creating a general coding framework.</div>
  <div class="source">Source: <a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:56</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/conference-mceb-2016/614">Conference: MCEB 2016</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/stephaneguindon">@stephaneguindon</a> wrote:</p>
            
              <p>There's about a week left to register to MCEB 2016 (<a href="http://www.lirmm.fr/mceb2016/" rel="nofollow">http://www.lirmm.fr/mceb2016/</a>). The theme of this year's edition is "<strong>Forecasting evolution</strong>" and should be interest to many people on phylobabble.org. Below is the list of invited speakers:</p>

<ul>
<li>Jeffrey Jensen (Ecole Polytechnique Fédérale de Lausanne)</li>
<li>Laura S. Kubatko (Departments of Statistics and Evolution, Ecology, and Organismal Biology - The Ohio State University)</li>
<li>Amaury Lambert (Stochastics & Biology Group - Laboratoire de Probabilités & Modèles Aléatoires - UPMC - Université Paris 06 )</li>
<li>Richard Neher (Max Planck Institute for Developmental Biology)</li>
<li>Bruce Rannala (UC Davis - University of California)</li>
<li>Isabel Sanmartín Bastida (Real Jardín Botánico (RJB-CSIC), Madrid)</li>
<li>Yun S. Song (Departments of Mathematics and Biology - Universty of Pennsylvania)</li>
</ul>

<p>Hope to see you there,</p>

<p>-Stephane-</p>
            
            <p>Posts: 1</p>
            <p>Participants: 1</p>
            <p><a href="http://www.phylobabble.org/t/conference-mceb-2016/614">Read full topic</a></p></div>
  <div class="source">Source: <a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:24</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139534566846">Workshop: MalhamTarn_Yorkshire.UsingR.May2-6</a></div>
  <div class="description">Dear Brian,

Can you please post the following course to the workshop page of
evoldir, thanks in advance,

&ldquo;Advancing in Statistical Modelling using R&rdquo;  We have a limited number
of places left for this popular course.

<a href="http://bit.ly/1STQNa4">http://bit.ly/1STQNa4</a>

Delivered by Dr. Luc Bussiere and Dr. Tom Houslay

This course will run from 2^nd - 6^th May 2016 at Malhamtarn Field
Station, North Yorkshire, England

This is an introduction to model selection and simplification, mixed
effects models, generalised linear models and non-linear models.

The course is aimed at biologists with a basic to moderate knowledge in
R. The course content is designed to bridge the gap between basic R
coding and more advanced statistical modelling. This five day course
will consist of series of modules, each lasting roughly half a day and
comprised of lectures and practicals designed to either build required
skills for future modules or to perform a family of analyses that is
frequently encountered in the biological literature.

Course content is as follows

Day 1 Course introduction

Techniques for data manipulation, aggregation, and visualisation;
introduction to linear regression. Packages: {tidyr}, {dplyr},
{ggplot2}

Day 2 Linear models

Diagnostics, collinearity, scaling, plotting fitted values); fitting
and interpreting interaction terms; model selection and simplification;
general linear models and ANCOVA.

Packages: {stats}, {car}

Day 3 Generalized linear models

Logistic and Poisson regression; predicting using model objects and
visualizing model fits.

Packages: {broom}, {visreg}, {ggplot2}

Day 4 Mixed effects models

Theory and practice of mixed effect models; visualising fixed and
random effects.

Packages: {lme4}, {broom}, {ggplot2}, {sjPlot}

Day 5 Fitting nonlinear functions

Polynomial & Mechanistic models; brief introduction to more advanced
topics & combining methods (e.g., generalised linear mixed effects,
nonlinear mixed effects, and zero-inflated and zero-altered models)

Packages: {nlsTools}

Afternoon to discuss own data if time permits

Please email any inquiries to oliverhooker@prstatistics.com or visit
our website <a href="http://www.prstatistics.com/">www.prstatistics.com</a>

Please feel free to distribute this material anywhere you feel is
suitable

Upcoming courses - email for details oliverhooker@prstatistics.com

SPATIAL ANALYSIS OF ECOLOGIC AL DATA USING R (April)

TIMES SERIES DATA ANALYSIS FOR ECOLOGISTS AND CLIMATOLOGISTS (May)

INTRODUCTION TO PYTHON FOR BIOLOGISTS (May)

ADVANCES IN DNA TAXONOMY USING R (August)

GENETIC DATA ANALYSIS USING R (August)

INTRODUCTION TO BAYESIAN HIERARCHICAL MODELLING (August)

MODEL BASED MULTIVARIATE ANALYSIS OF ECOLOGICAL DATA USING R (October)

LANDSCAPE (POPULATION) GENETIC DATA ANALYSIS USING R (October)

APPLIED BAYESIAN MODELLING FOR ECOLOGISTS AND EPIDEMIOLOGISTS (October)

Dates still to be confirmed - email for details
oliverhooker@prstatistics.com

STABLE ISOTOPE MIXING MODELS USING SIAR, SIBER AND MIXSIAR USING R

INTRODUCTION TO R AND STATISTICS FOR BIOLOGISTS

PHYLOGENETIC DATA ANALYSIS USING R

BIOINFORMATICS FOR GENETICISTS AND BIOLOGISTS

Oliver Hooker

PR Statistics

&ldquo;oliverhooker@prstatistics.co.uk&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:51</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139533770861">Graduate position: LinkopingU_Sweden.EvolutionAvianCognition</a></div>
  <div class="description">PhD-position available on the evolution of individual variation in animal
cognition at Linkping University, Sweden.

Job description A fully funded 4-year PhD position is offered in the
subject of ethology, in the research group of Hanne Lvlie, Linkping
University (IFM Biology). The job description entails to conduct research
on the causes and consequences of variation in animal cognition, and we
are particularly interested in links with animal personality. Cognitive
processes are fundamental, but explanations to individual variation are
still poorly explored. Animal personality (i.e. consistent individual
behavioural responses), has recently experienced considerable research
attention. Despite so, fundamental questions remain unanswered, such
as why does variation in personality arise, and how is polymorphism
maintained? Personality and cognition are predicted to be linked, but the
causality and consequences of this relationship are still unknown. We
have demonstrated individual variation in various cognitive tasks and
observe links between learning speed and personality, in our model
species the red junglefowl (Gallus gallus).

The aim of the proposed project is to improve our understanding of
individual variation in animal cognition, by using the red junglefowl
as model. The fowl is a particularly suitable species for research on
this due to the possibility of combining our extensive understanding
of behaviour, physiology and genetics in chickens, to apply a strong
experimental design while examining behaviours under natural conditions
or in the lab. The advertised position can be combined with a teaching
assistantship of up to 20% within the biology undergraduate program,
which will extend the position accordingly.

Requirements The successful applicant will be a highly motivated and
talented candidate with a MSc-degree in biology (or equivalent), with a
specialization in ethology, behavioural ecology, evolutionary biology,
neurobiology or similar. Research experience in experimental animal
behaviour, sound statistical knowledge and analytical abilities are
merits, as is experience in scientific writing. Very good knowledge of
written and oral English is required. Both good cooperation skills and
the ability to work independently are emphasised. The successful candidate
should be enthusiastic about animal behaviour and handling birds.

What we offer Linkping University is a modern, innovative university
with over 27000 students and 3900 employees. Biology at Linkping
University offers excellent molecular facilities and exceptional avian
housing facilities. The successful candidate will be part of the AVIAN
behavioural genomics and physiology group (which currently has around
25 group members, including 10 PhD students), and part of the Centre of
Excellence for Animal Welfare Science (in collaboration with SLU Uppsala).

Linkping is a popular university city, and is the fifth largest city
in Sweden. It is located in central Sweden, just two hours by train from
Stockholm, and offers proximity to the beautiful nature of stergtland,
including the great lake Vttern, as well as both recent and pre-historic
history, and cultural attractions.

Equal opportunity Linkping University continues to develop as an
attractive and creative work place characterized by equal opportunities,
which works actively to promote diversity and gender equality.

Starting date The position is available immediately, and will be set
according to agreement.

Salary According to present rates at the university (starting at 25 000
SEK per month), including health insurance and other benefits, for 4 full
years. A short-term position may be offered to a very promising candidate
to ensure that the required experiences are acquired prior to start.

More information Contact Hanne Lvlie (hanne.lovlie@liu.se), tel +46
(0) 13-286681.

Application procedure The application should include a 1-2 page
cover letter describing your research experience and interests,
and your interest in this project, and a full CV (including name and
contact details of 2 references who are willing to provide letters of
support). Transcript of grades and copy of a relevant publication may
be required at a later stage. Application should be sent as one PDF to
hanne.lovlie@liu.se. The position is open until filled, and applications
will be reviewed continuously, but for full consideration please apply
by 31st of March 2016.

Hanne Lovlie 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:24</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139533064341">Job: ViriginiaTechU.PollinatorBiodiversity</a></div>
  <div class="description">Virginia Tech is looking for an Assistant Professor in Pollinator Biology &
Ecology
<a href="http://bit.ly/1KrDv1A">http://bit.ly/1KrDv1A</a>

Virginia Techs Department of Entomology invites applications for a
tenure-track position at the level of Assistant Professor in the area of
Pollinator Biology & Ecology starting in the Fall 2016 semester. This is a
nine-month, 70% research and 30% teaching appointment based on campus at
Virginia Tech, a land-grant university in the scenic New River Valley of
Virginia.

The successful candidate will be expected to develop an extramurally funded
research program focused on pollinating insects with emphasis on regional
and national concerns related to pollinator declines, species biodiversity
losses, ecosystem services, and/or conservation biology. Knowledge of
apicultural practices is desirable but not required. Preference will be
given to individuals with post-doctoral experience and a demonstrated
ability to secure extramural funding and publish in high quality,
peer-reviewed journals. The incumbent is expected to teach courses at the
undergraduate- and graduate-level, contribute to innovative curricula in
entomology and related disciplines, and mentor undergraduate and graduate
students. The incumbent will also support the mission of the land-grant
university by participating in outreach and engaging local, state, and
regional stakeholders. The successful candidate will have access to
existing honey bee colonies, apiculture facilities, and land for field
research, including diverse native habitats, as well as agronomic,
vegetable, and orchard field plots.

Applicants are required to hold a Ph.D. in entomology, zoology, or a
closely related field. Applicants with experience in pollinator biology
including, but not limited to, the ecology, physiology, molecular biology,
genetics, pathology, or behavior of honey bees and/or other pollinating
insects are strongly encouraged to apply.

Paul Marek 
via Gmail </div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:24</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139533064461">Postdoc: Bethesda_Maryland.HIV_Evolution</a></div>
  <div class="description"><p>The Henry M. Jackson Foundation for the Advancement of Military
Medicine Inc. (HJF) is seeking a post-doctoral fellow for the HIV
Research Program (MHRP) located in Bethesda, Maryland. HJF provides
scientific, technical and programmatic support services to MHRP. The
U.S. Military HIV Research Program (MHRP) (<a href="http://www.hivresearch.org/">www.hivresearch.org</a>) is at
the forefront of the battle against HIV to protect U.S. troops from
infection and to reduce the global impact of the disease. MHRP is one
of the leading clinical research programs developing a vaccine to
prevent HIV, improving treatments for HIV and co-infections and also
working towards a cure.

Job Summary

The post-doctoral fellow will be part of a team of biologists,
clinicians and statisticians doing highly collaborative,
multi-institutional and inter-disciplinary work on a range of questions
with a strong emphasis on vaccine research. The candidate will study
evolutionary processes both within and between hosts and comparative
genomics to investigate commonalities and differences between HIV and
other pathogens including Ebola, Dengue, MERS-CoV, Zika.

The incumbent may perform some or all of the following
responsibilities:

1. Use Bayesian phylogenetic, phylogeographic and comparative genomics
approaches.

2. Perform comprehensive molecular epidemiological analysis and
bioinformatic analysis of large datasets.

3. Develop mathematical and computational models of evolution to
explore various aspects of pathogen evolution, host adaptation and
transmission.

4. Maintain current knowledge by participating in scientific
conferences and reading specialty journals. Contribute to the science
through independent and collaborative research including by authoring
manuscripts.

Qualifications

The successful candidate will have a Ph.D. in population genetics,
evolutionary genetics, microbial genomics, comparative genomics,
computational biology, mathematics, statistics or a related area.
Background knowledge in immunology, virology or structural biology is a
plus.

The candidate should be a dedicated, social, detail-oriented person,
who is able to work independently within a team of researchers.
Excellent written and verbal communication skills are required.

To apply

Please contact Morgane Rolland

mrolland@hivresearch.org



via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:08</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139530769306">Workshop: SCENE_Glasgow.TimeSeriesDataModelling.May10-13</a></div>
  <div class="description">&ldquo;Time Series Models for Ecologists and Climatologists&rdquo;

 <a href="http://bit.ly/20FG2qM">http://bit.ly/20FG2qM</a>
limatologist/

There are still some places left on this new course suited to many
types of Ecology, Evolutionary and Climatology data

Delivered by Dr. Andrew Parnell and Dr. Doug McNeall

This course will run from 10th - 13th May 2016 at SCENE Field
Station, Loch Lomond national park, Scotland

This course will cover model-based time series analysis with a
particular focus on applications in ecology and climatology. All
methods will be illustrated using the free, open-source software
package R. Time Series data are ubiquitous in the physical sciences,
and models for their behaviour enable scientists to understand temporal
dynamics and predict future values.

Participants will be taught a wide range of suitable time series models
for both discrete and continuous time systems. The course takes a
foundational Bayesian approach, which will enable participants to have
a deeper understanding of the models being fitted, and to estimate all
unknown quantities with uncertainty. Participants are encouraged to
bring their own data sets for discussion with the course tutors.

Course content is as follows

Day 1 Basic concepts

Class 1: Introduction; some example time series datasets; prediction vs
explanation

Class 2: An introduction to Bayesian Statistics.

Class 3: The AR(1) model

Practical: revision on using R to load data, create plots and fit
statistical models

Round table discussion: understanding the output from a Bayesian model

Day 2 Arima modelling

Class 1: ARMA models for real data

Class 2: ARIMA and sARIMA modelling

Practical: An introduction to the Bayesian modelling language JAGS

Round table discussion: understanding and running a

JAGS model

Day 3 Continuous Time Series Modelling

Class 1: Brownian Motion and its application to real data sets

Class 2: An introduction to Stochastic Volatility Modelling

Practical: Fitting continuous time models in JAGS

Round table discussion: Issues of continuous vs discrete time

Day 4 Advanced Times Series Models

Class 1: Multivariate models

Class 2: Fractional differencing and models using differential
equations

Practical: Running advanced models in JAGS

Round table discussion: Bring your own data set

Please email any inquiries to oliverhooker@prstatistics.com or visit
our website <a href="http://www.prstatistics.com/">www.prstatistics.com</a>

Please feel free to distribute this material anywhere you feel is
suitable

Upcoming courses - email for details oliverhooker@prstatistics.com

SPATIAL ANALYSIS OF ECOLOGIC AL DATA USING R (April)

ADVANCING IN STATISTICAL MODELLING USING R (May)

INTRODUCTION TO PYTHON FOR BIOLOGISTS (May)

ADVANCES IN SPATIAL ANALYSIS OF MULTIVARIATE ECOLOGICAL DATA (July)

ADVANCES IN DNA TAXONOMY USING R (August)

GENETIC DATA ANALYSIS USING R (August)

INTRODUCTION TO BAYESIAN HIERARCHICAL MODELLING (August)

MODEL BASED MULTIVARIATE ANALYSIS OF ECOLOGICAL DATA USING R (October)

LANDSCAPE (POPULATION) GENETIC DATA ANALYSIS USING R (October)

APPLIED BAYESIAN MODELLING FOR ECOLOGISTS AND EPIDEMIOLOGISTS (October)

Dates still to be confirmed - email for details
oliverhooker@prstatistics.com

STABLE ISOTOPE MIXING MODELS USING SIAR, SIBER AND MIXSIAR USING R

INTRODUCTION TO R AND STATISTICS FOR BIOLOGISTS

PHYLOGENETIC DATA ANALYSIS USING R

BIOINFORMATICS FOR GENETICISTS AND BIOLOGISTS

Oliver Hooker

PR Statistics

Oliver Hooker

PR~Statistics

&ldquo;oliverhooker@prstatistics.co.uk&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>February 17, 2016</h3>
<div class="news-item">
 <div class="date">23:40</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139529739351">Postdoc: IowaStateU.PopGenomicsFig-FigWasp</a></div>
  <div class="description"><p>Postdoc: Phylo- and Population-Genomic Analyses of Fig-Fig Wasp
Co-Diversification and Introgression

Postdoctoral Research Associate
Department of Ecology, Evolution, and Organismal Biology
Iowa State University

The Nason and Heath labs are looking to recruit a Postdoctoral Research
Associate with expertise in phylo/population genomics. The successful
applicant will collaborate with us on a new, NSF supported project
using sequence capture and GBS data to investigate the coevolutionary
history of species interactions in Central American figs and their
pollinating (mutualistic) and non-pollinating (antagonistic) fig
wasps. Collaborators on the project include Drs. John Nason and Tracy
Heath (Iowa State University), Dr. E. Allen Herre (Smithsonian Tropical
Research Institute, Panama), Dr. Charlotte Jandr (Harvard University),
Dr. Carlos Machado (University of Maryland), and Dr. Robert Raguso
(Cornell University). Required qualifications, terms of employment,
application instructions, and a project summary follow below.

Required Qualifications

Education: A PhD degree in biological sciences or bioinformatics, or
acceptable equivalent combination of education and experience.

Experience/Skills: Experience working with genomic/transcriptomic/GBS
datasets; demonstrated experience working in a Linux/Unix shell
environment; competency with at least one scripting language (e.g.,
Perl, Python, R). Demonstrated experience in the phylogenetic and/or
population genetic analysis of NGS data, ideally obtained via sequence
capture or GBS/RAD-seq methods. Well-developed organizational and time
management skills, and leadership ability to direct (with the PIs)
a large and productive project.

Terms of Appointment

Starting salary is $45,000 plus benefits. Funds are available for one year
and are renewable for up to four years, pending satisfactory progress. The
optimal start date is June 1, 2016.

Application Instructions

For consideration, applicants must apply by April 1, 2016. Informal
inquiries are encouraged prior to formal application. For formal
application, please send 1) a cover letter, 2) a curriculum vitae,
3) a brief statement of research experiences/interests, and 4)
names and contact information for three references to Dr. John Nason
(jnason@iastate.edu).

Iowa State University is an Equal Opportunity/Affirmative Action
employer. All qualified applicants will receive consideration for
employment without regard to race, color, age, religion, sex, sexual
orientation, gender identity, genetic information, national origin,
marital status, disability, or protected veteran status, and will not
be discriminated against. Inquiries can be directed to the Director of
Equal Opportunity, 3350 Beardshear Hall, (515) 294-7612.

Project Summary

Figs and their fig wasp pollinators and parasites have co-evolved for
~90 million years to become both highly diverse (>750 species of figs)
and ecologically important keystone components of tropical forest
ecosystems. Figs and wasps have long been assumed to represent a case
of strict co-speciation, with highly specific pollinator and parasitic
(non-pollinator) wasps identifying appropriate hosts via distinctive
volatile chemical signals. More recent studies suggest a more complex
scenario, however, involving an evolutionary history punctuated by
host-shifts by individual wasp species. Although the wasp associations
with fig hosts have been widely studied, the genetic consequences for
the host figs of host-shifting pollinators and the mechanisms underlying
host recognition remain poorly understood.

This project will fill these gaps by producing robust, detailed,
many-gene phylogenies for 14 strangling fig (Ficus) species and their
associated pollinating (Pegoscapus) and non-pollinating (Idarnes)
fig wasps (~60 species) from the vicinity of Barro Colorado Island,
Panama. Using transcriptome sequences, we will target ~300 genes from
each of three species per lineage for capture and subsequent Illumina
sequencing. Phylogenies will be inferred using Bayesian methods and will
enable robust testing of phylogenetic congruence between figs and fig
wasps. Further, they will guide population-level genotype by sequencing to
test a priori predictions of potential cases of hybridization in the figs
and host shifting and race formation in both pollinator and non-pollinator
wasps. Combined with quantification of wasp-attracting fig volatiles and
fruit-surface chemicals, this work will detect and resolve the genomic
consequences of host introgression due to host-shifting pollinator wasps,
and link them to the chemical basis of host-recognition.

This research will significantly clarify both the patterns and processes
underlying the evolutionary ecology of fig and fig wasp interactions. Our
standardized, genomic approach is essential for: 1) obtaining robust fig
and fig wasp species trees, 2) delimiting fig species and discriminating
cases of introgressive gene flow from shared ancestral polymorphism,
and 3) linking introgression of figs and their chemical phenotypes to
cases of pollinator host shifting. Our community-level approach is also
essential to obtain the across-species replication necessary for robust
statistical inference of diversification pattern and process across
interacting fig and wasp taxa.


Please contact Drs. Nason or Heath if you are interested in or have any
questions about this postdoctoral position.

Dr. John Nason, Professor
Email: jnason@iastate.edu
Lab website: <a href="http://bit.ly/17Wyl4E">http://bit.ly/17Wyl4E</a>

Dr. Tracy Heath, Assistant Professor
Email: phylo@iastate.edu
Dept. page: <a href="http://bit.ly/1BjIriH">http://bit.ly/1BjIriH</a>



jnason@iastate.edu


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:51</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139478025576">Postdoc: CEES_UOslo.EvolutionDefense</a></div>
  <div class="description">&ldquo;A researcher position is available at the Centre
for Ecological and Evolutionary Synthesis, (CEES),
<a href="http://bit.ly/1ooeHxd,">http://bit.ly/1ooeHxd,</a> Department of Biosciences,
<a href="http://bit.ly/1Q106T0.">http://bit.ly/1Q106T0.</a>  The position is for 38 months, and the
estimated start date is May 1st 2016. The successful candidate will work
on the project "The evolution of defence systems: theory and experiment&rdquo;
(EVODEF) funded by the Research Council of Norway.

Project description
The EVODEF project focuses on the evolutionary ecology of organismal
defence traits. Organisms may use an impressive diversity of
defences against their natural enemies. Evidence from a range of
animal and plant systems suggests that the different defence traits
expressed by an organism cannot be understood in isolation from each
other, but must be studied as parts of complex suites of traits. The
project combines theoretical models (mainly developed by the PI) with
empirical experiments (conducted by the candidate) to understand the
properties of such defensive systems and how they evolve under different
selective conditions. Topics of interest include how defence lines are
phenotypically co-expressed and coevolve, how defences against different
enemies (e.g. predators or parasites) affect each other, and the role
of non-genetic inheritance (e.g. maternal and epigenetic effects)
in defence expression. In the theoretical part of the project we will
develop mathematical models of organismal defence relevant to several
empirical systems. In the experimental part of the project, we will use
the inducible defences of Daphnia to test novel predictions from theory;
the results will in turn stimulate further theory development.

The successful candidate will join our team and study the defence
traits of Daphnia in the lab. The project provides a unique opportunity
to design and run experiments that test novel theoretical ideas and
inform ongoing theoretical work. The ideal candidate will make a major
contribution to the planned research, but will also develop his/her
own complementary lines of research that reflect the candidate&rsquo;s own
expertise and interests and that broadly fit within the project&rsquo;s main
goals. There is growing interest in developing a research environment for
Daphnia at the department and it is highly desirable that the candidate
will take an active part in this, as well as contribute to developing
future collaborative projects that build upon achieved results.

Requirements
The Faculty of Mathematics and Natural Sciences has a strategic ambition
of being a leading research faculty. Candidates for these fellowships
will be selected in accordance with this, and expected to be in the
upper segment of their class with respect to academic credentials.
Applicants must hold a PhD-degree or other corresponding education
equivalent to a Norwegian doctoral degree in biology or a related
field. The successful candidate will be responsible for the establishment
and maintenance of Daphnia cultures, the daily running of experiments,
imaging, morphometrics, and data analysis. A proven, solid record of
running lab experiments with small aquatic organisms (preferably Daphnia)
and maintaining them in culture is required.

The ideal candidate has extensive lab experience with Daphnia, strong
statistical and writing skills, pays careful attention to detail
and theoretical context when designing experiments, and has some
experience with field sampling. A background from evolutionary ecology
or evolutionary biology is considered an advantage. Field sampling is
planned for the summer 2016, and it is desirable that applicants are
available to start by this time. A research stay abroad is planned during
one section of the project.

The successful candidate will work in close collaboration with the
project PI and other members of the team. We seek a highly motivated,
enthusiastic person with the ability to think creatively and critically,
and with the ambition to gain new insights and publish papers in leading,
international journals. Applicants must show good interpersonal skills
and be willing to work in close collaboration with others, as well as
have the ability to work independently. Applicants should have a good
publication record for the career stage.

A good command of English is required.

Salary
Position code 1109, Pay Grade: 57-63 (NOK 483 700 - 540 200 per year,
depending on qualifications).

The application must include:
.       Application letter including a statement of interest, briefly
        summarizing your scientific work and interests and describing
        how you fit the description of the person we seek
.       CV (summarizing education, research positions, and other
        qualifying activity)
.       Copies of educational certificates, transcript of records,
        letters of recommendation
.       A complete list of publications, and up to 5 peer-reviewed
        publications that the applicant wishes to be considered by the
        evaluation committee
.       Names and contact details of 2-3 references (name, relation to
        candidate, e-mail and telephone number)

Foreign applicants are advised to attach an explanation of their
University&rsquo;s grading system.

Please remember that all documents should be in English or a Scandinavian
language.

In accordance with the University of Oslo&rsquo;s equal opportunities policy,
we invite applications from all interested individuals regardless of
gender or ethnicity.  UiO has an agreement for all employees, aiming to
secure rights to research results a.o.

To apply for the position, please see
<a href="http://bit.ly/1ooeHNr">http://bit.ly/1ooeHNr</a>

Application deadline: March 25, 2016

Contact detalis:
 Dr. istein Holen - email: o.h.holen@bio.uio.no 
Telephone: +47 22856186 

Professor Dag Hessen - email : d.o.hessen@ibv.uio.no
Telephone: +47 22854553 

Questions regarding Easycruit, contact HR Officer Torunn Standal
Guttormsen - email : t.s.guttormsen@mn.uio.no

Telephone: +47 22 85 42 72


Best regards,

Torunn Standal Guttormsen
HR officer

Torunn Standal Guttormsen 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:51</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139478025846">Postdoc: ToulouseCNRS_France.HeritabilityAndSelectionInTheWild</a></div>
  <div class="description">Postdoc: ToulouseCNRS.HeritabilityAndSelectionInTheWild

A postdoctoral research associate position is available at CNRS in the 
laboratory Evolution & Biological Diversity in Toulouse, France.
The postdoctoral associate will participate to the ERC-funded ANGI 
project (PI: Benoit Pujol <a href="http://bit.ly/1svZZ6N">http://bit.ly/1svZZ6N</a>).
The aim is to study the heritability of traits (additive genetic 
variation, ecological niche transmission) and the heterogeneity of 
selection in wild populations of /Antirrhinum majus,/ the snapdragon.


To date, we have cumulated 6 years of data (4000plants, 6 generations) 
from the /in situ/ exhaustive survey of flowering snapdragon plants in 
fragmented populations growing on the Mediterranean coast in southern 
France. The postdoctoral associate will work with other team members on 
the creation and analysis of this large dataset combining phenotypes 
(multiple vegetative and reproductive traits), locations, pedigrees, and 
microsatellite marker data. The postdoctoral associate will participate 
to field work. A large part of the data is already available. Analyses 
and publication preparation will therefore not be limited by the field 
trip seasonality.

French is not mandatory. A Ph.D. in evolutionary ecology, quantitative 
genetics, or closely related field is required. We seek enthusiastic and 
motivated applicants with no experience, 3-5 years of postdoctoral 
research experience, or more. Proficiency in R andexpertise in 
quantitative genetics are welcome. Expertise in quantitative genetic 
and/or selection analyses conducted on wild population data is 
particularly welcome, even if such work was not carried out on plants. 
The initial appointment is for two years, with the potential for 
additional years of support conditional on performance. Start dates 
between early April and late May 2016, with preference for earlier dates.

To apply, please send (1) a cover letter explaining your interest in the 
position, (2) a CV, and (3) contact information (phone number and email) 
for three references 
tobenoit.pujol@univ-tlse3.fr

Closing date for application: March 1, 2016

Exemple of salary (depending on experience):

34,626 euros a year (monthly salary after tax ~2328 euros) for a 
researcher with 3-5 years experience
45,600 euros a year (monthly salary after tax ~2809 euros) for a 
researcher with 5-10 years experience

via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:02</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139476946476">Conference: UCambridge.EvolutionaryGenetics.Mar15.Deadline</a></div>
  <div class="description"><p>Evolutionary Genetics and Genomics Symposium (Abstract Submission
Deadline Approaching)

Please note that the deadline for the abstract submission for Evolutionary
Genetics and Genomics Symposium 2016 is approaching (February 22).

The annual Evolutionary Genetics and Genomics Symposium (EGGS) will take
place on Tuesday 15th March 2016 at the University of Cambridge.

This international one-day meeting is free to attend and no registration
is required. Talks will cover all areas of evolutionary genetics from
comparative genomics to evo-devo, and pathogen evolution to speciation.

Abstract submission is now open. If you would be interested in
presenting a talk please submit your abstract via this link:
<a href="http://bit.ly/1oIVerg">http://bit.ly/1oIVerg</a>

We will also have talks by four invited speakers:

- Mary J. OConnell, University of Leeds

- Magnus Nordborg, Gregor Mendel Institute of Molecular Plant
  Biology, Vienna

- Federica Di Palma, TGAC

- Eric Miska, Wellcome Trust Sanger Institute, University of Cambridge

For more information, please visit
<a href="http://bit.ly/1ajPZUg">http://bit.ly/1ajPZUg</a> or email
eggs.symp@gmail.com

With best wishes,

The organising committee

cc625@cam.ac.uk


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:02</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139476946331">Conference: Boise_Idaho.InvasivePlants.Oct17-20</a></div>
  <div class="description">Plan to join us October 17-20, 2016 at the Boise Centre in Boise, ID
for the 4th Conference - Invasive Plants in Natural Areas. This four
day conference will feature presentations on the latest developments in
invasive plant research and provide ample opportunity for managers and
researchers to engage and identify creative new approaches to managing
invasions. Join the discussion of possibilities and problems specific
to the management of invasive plants in natural areas, and perhaps more
importantly, contribute to solutions! More information about symposia
topics (hopefully one about evolution in invasions!) will be provided
soon!  See <a href="http://www.nripc.org/">www.nripc.org</a> for more information



&ldquo;Gaskin, John&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:47</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139476599296">Conference: Vairao_Portugal.ConservationGenetics.May3-6</a></div>
  <div class="description"><p>Congenomics2016 - Conference on Conservation Genomics
<a href="http://bit.ly/1PClvln">http://bit.ly/1PClvln</a>
Vairao, Portugal
3-6 May 2016

NEW ABSTRACT SUBMISSION DEADLINE! - 1 MARCH

TRAVEL GRANTS AVAILABLE:
ConGenOmics* will fund the participation of early career researchers 
with travel grants. These grants are available for graduate students 
(priority to PhD students) and postdocs (maximum 2 years after PhD) 
that will present their work. We encourage any kind of presentation, 
including early stages of scientific or academic projects. Priority will 
also be given to students and post-docs from ConGenOmics member countries.
Check conditions for application at <a href="http://bit.ly/1ocBg8n.">http://bit.ly/1ocBg8n.</a>

INVITED SPEAKERS:
Fred Allendorf (University of Montana)
Mike Bruford (Cardiff University)
Lisette Waits (University of Idaho)
Natasha de Vere (National Botanic Garden of Wales)
Emma Teeling (University College Dublin)
Kerry Naish (University of Washington)

IMPORTANT DATES:
1 March: Submit your abstract online.
18 March. Abstract acceptance notification. Decision on travel grant 
applications.
31 March: Early registration fees: 250 EUR (standard), 100 EUR (students).

ABOUT:
This conference aims to gather the best research currently being 
performed on Genomics and its applications to Conservation. The program 
includes six invited plenary lectures, submitted oral communications 
(regular and flash) and poster sessions, covering the state of art on 
conservation genomics. The following main subjects will be addressed, 
emphasizing their applied nature:
- Demographic inference and population history;
- Adaptive genetic variation (including quantitative traits and 
epigenetics);
- Population viability;
- Taxonomic identification (using whole genome, mitogenome, metabarcoding);
- Environmental DNA.

The conference is organized by ConGen (Conservation genetics and 
wildlife management) research group and hosted in CIBIO/InBIO Associate 
Laboratory (<a href="http://cibio.up.pt/">http://cibio.up.pt/</a>) and the University of Porto facilities 
in Vairao, Portugal, and the ConGenOmics Research Networking Programme*.

(*) ConGenOmics is an ESF Research Networking Programme that aims to 
bring together European and pan European specialists on conservation 
genomics, to develop and improve the knowledge transfer of the technical 
advances and challenges in conservation genomics, focusing specifically 
on the application of genomic techniques. ConGenOmics programme 
organizes workshops, summer schools, collaborative expert meetings and 
an exchange program at panEuropean level, to further develop the field 
of conservation genomics and to aid in the knowledge transfer of the 
technological advances and challenges among European research groups 
focusing specifically on applications of genomic technologies in 
conservation biology. More details in <a href="http://bit.ly/1Oz1eIl">http://bit.ly/1Oz1eIl</a>

ORGANIZING AND SCIENTIFIC COMMITTEE:
Paulo Celio Alves (University of Porto)
Ricardo Jorge Lopes (University of Porto)
Jose Melo-Ferreira (University of Porto)
Raquel Godinho (University of Porto)

Joop Ouborg (Radboud University)
Philippine Vergeer (Wageningen University)
Jacob Hoglund (Uppsala University)
Gernot Segelbacher (University of Freiburg)
Craig Primmer (University of Turku)


Questions? Contact us by email (info@congenomics2016.com) or on 
Facebook/twitter.


jmeloferreira@cibio.up.pt


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:24</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139476041431">Graduate position: CharlesU_Prague.SexChromosomeEvolution</a></div>
  <div class="description">CharlesU_Prague.SexChromosomeEvolution

PhD position in Evolution of sex chromosomes in squamate reptiles

A position for a PhD candidate is currently open in the laboratory
of Dr. Lukas Kratochvil, Department of Ecology, Charles University
in Prague, Czech Republic. The successful PhD candidate will join our
current projects on the study of the evolution of sex chromosomes in
squamate reptiles with molecular and cytogenetic methods.

Project description:
The aim of the project is to examine the evolution of sex chromosome and
their homology based on gene content in selected lineages of squamate
reptiles (eg. lizards and snakes). Squamate reptiles possess lineages
with either XX/XY, ZZ/ZW or temperature-dependent sex determination
systems. Despite the recent advantages in the field, our knowledge about
the sex determination systems and sex chromosome homology is limited
only to few groups of reptiles (eg. iguanas, advanced snakes), while the
majority of reptilian lineages remain unstudied. The successful candidate
will collaborate with the other team members and will study the evolution
of sex chromosomes in a multidirectional approach, involving cytogenetics
(eg. FISH, chromosome painting), molecular genetics (eg. qPCR) and
bioinformatics (eg. transcriptomics). More information about our current
research activity can be found in the following publications:

<a href="http://1.usa.gov/20DkTxA">http://1.usa.gov/20DkTxA</a>
<a href="http://1.usa.gov/20YFiT5">http://1.usa.gov/20YFiT5</a>
<a href="http://1.usa.gov/20DkWcy">http://1.usa.gov/20DkWcy</a>

Qualifications:
The applicant should possess Master degree (or equivalent title) and has
a proven expertise in a related field of biology: evolutionary biology,
cytogenetics, phylogenetics, genomics and/or bioinformatics. Previous
experience in herpetology is not necessary. The ideal candidate should
be scientifically motivated, able to work both independently and as a
part of the team and should have a publication record. High standard of
spoken and written English is required.

Application:
The applicant should send a letter explaining your background and
motivation, a full CV and contact information for two references to:
lukas.kratochvil@natur.cuni.cz; michail.rovatsos@natur.cuni.cz.

The deadline for submission of applications is March 31, 2016.



Lukas Kratochvil

Associate Professor
Department of Ecology
Charles University in Prague
Faculty of Science
Vinicn?? 7, 128 43 Praha 2
<a href="http://www.natur.cuni.cz/en">www.natur.cuni.cz/en</a>
Tel.: +420 221 95 1745
e-mail: lukas.kratochvil@natur.cuni.cz



&ldquo;Michail Rovatsos, Ph.D.&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:24</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139476041216">Conference: AsilomarCA.LocalAdaptation.Jul14-17.Registration</a></div>
  <div class="description">EARLY BIRD REGISTRATION NOW!

The American Genetic Association President$B!G(Bs Symposium, $B!H(BLocal
adaptation: from phenotype to genotype to fitness$B!I(B, will be held July
14-17, 2016, at beautiful Asilomar, CA (<a href="http://bit.ly/1mfUHEX">http://bit.ly/1mfUHEX</a>).

In the past, many studies dealt only with phenotypes/traits, but today the
connection to genotype is becoming feasible for more and more organisms.
Our speakers are a blend of established and new investigators, working
on a variety of organisms, and who are taking diverse approaches to
the question.

The AGA will provide several travel awards to cover registration
for graduate students who would like to attend and present posters.
Six abstracts will be selected for oral presentations, and the students
will receive $500 awards.  Details and registration are available at
the AGA website: <a href="http://bit.ly/15HcX0C">http://bit.ly/15HcX0C</a>

Invited speakers:
Key Distinguished Lecture by Victoria Sork (Dean of Life Sciences, UCLA)

Sally Aitken            University of British Columbia
Jill Anderson           University of Georgia
Zach Cheviron           University of Illinois
Emily Dittmar           Michigan State University
Suzanne Edmands         University of Southern California
David Field             Inst. Science and Tech, Austria
Lila Fishman            University of Montana
Jannice Freidman        Syracuse University
Steve Keller            University of Vermont
John Kelly              University of Kansas
Brian Langerhans        North Carolina State University
Curtis Lively           Indiana University
Robert Reed             Cornell University
Douglas Schemske        Michigan State University
Rena Schweizer          UCLA
Kerry Shaw              Cornell University
Peter Tiffin            University of Minnesota

 Lynda Delph, AGA President

AGAJOH 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:32</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139473030216">Graduate position: Barcelona.15.SingleCellGenomics</a></div>
  <div class="description"><p>15 PhD/Research positions on Promoting Single Cell Genomics to explore 
the ecology and evolution of hidden microeukaryotes (H2020-MSCA-ITN SINGEK)

Microbial eukaryotes cover the widest phylogenetic diversity in the 
eukaryotic tree of life and are essential components of all Earth&rsquo;s 
ecosystems. Paradoxically, they remain largely unknown, hiding a wealth 
of genes and genomes that are key to understand the ecology and 
evolution of microbial ecosystems. SINGEK is a H2020 
Marie-Skłodowska-Curie Innovative Training Network (MSCA-ITN) project 
aimed at providing a unique and structured training programme to 15 
Early Stage Researchers (ESRs) to study microeukaryotes at the 
single-cell level. ESRs will be trained in the use of an array of 
laboratory methods (cell sorting and molecular tools for nucleic acid 
amplification and high throughput sequencing), bioinformatic techniques 
(genome and transcriptome assembly and functional gene annotations), and 
will work on scientific questions in the fields of microbial ecology and 
evolution in order to investigate microeukaryotes in unparalleled 
detail. The training structure is a central part of SINGEK project and 
will include: local and network-wide activities, secondments to other 
network laboratories and workshops in transferable skills essential for 
successful career development.

All positions are for 3-years contracts starting in October 1st, 2016, 
most of them leading to a PhD degree. Specific conditions may apply to 
individual positions depending on local regulations.

Top-level graduates in biology, bioinformatics, microbial ecology, 
molecular biology, evolutionary biology, biodiversity, biotechnology, 
physics, nanoscience, engineering or computer science are encouraged to 
apply. No discrimination will be made on the basis of nationality, 
gender, race, religion or disability.

H2020 EU funding imposes strict eligibility criteria: At the time of 
recruitment the researcher must not have resided or carried out his/her 
main activity (work, studies, etc…) in the country of the host institute 
for more than 12 months in the 3 years immediately prior to his/her 
recruitment under the project. The researcher should also be in the 
first four years of their research careers at the time of recruitment by 
the host organisation and have not been awarded a doctoral degree.

Candidates may apply through the SINGEK applications system 
(<a href="http://bit.ly/1PEEYR4">http://bit.ly/1PEEYR4</a>), which will request the following 
information:
1. An application form, including a CV overview, ESRs&rsquo; positions of
   interest (up to 3), and a statement of their motivation for applying.
2. A full CV.
3. An academic record, with a scan of the degree qualifications.
4. Two recommendation letters.

Each project supervisor will revise the candidates’ documentation and, 
on the basis of the completeness and adequacy of the requested material 
and quick eligibility check, will score candidates based on: (1) 
academic profile; (2) personal motivation; (3) scientific skills and 
relevant experience; and (4) English proficiency. Shortlisted candidates 
will be invited to teleconference interviews with the relevant project 
supervisor(s).

Application deadline: 17:00 GMT on the 15th April 2016

More information: <a href="http://www.singek.eu/">www.singek.eu</a>

Candidates are invited to contact potential supervisors to get more details.

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="index3362.html?q=aggregator&amp;from=120">previous page</a></div><div>... <a href="index4543.html?q=aggregator&amp;from=60">4</a> <a href="index798e.html?q=aggregator&amp;from=80">5</a> <a href="index0e96.html?q=aggregator&amp;from=100">6</a> <a href="index3362.html?q=aggregator&amp;from=120">7</a> <strong>8</strong> <a href="indexf37c.html?q=aggregator&amp;from=160">9</a> <a href="index52c1.html?q=aggregator&amp;from=180">10</a> <a href="indexeef0.html?q=aggregator&amp;from=200">11</a> <a href="index933b.html?q=aggregator&amp;from=220">12</a> ...</div><div><a href="indexf37c.html?q=aggregator&amp;from=160">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=140 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:20:56 GMT -->
</html>
