<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/2&from=60 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:30:12 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexfc9c.html?q=node/431">About Women in Science (Evol2014)</a></li><li><a href="indexf09b.html?q=node/426">Researcher in Environmental Bioinformatics, University of Technology Sydney</a></li><li><a href="indexb172.html?q=node/425">Minutes of mid-year meeting 2014</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 5 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator/sources/2" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator">news aggregator</a>
<ul>
<li class="expanded"><a href="indexe5f5.html?q=aggregator/sources">sources</a>
<ul>
<li class="leaf"><a href="indexfd51.html?q=aggregator/sources/2" class="active">BMC Evolutionary Biology</a></li>
<li class="leaf"><a href="index4421.html?q=aggregator/sources/8">CiteULike Phylogeny</a></li>
<li class="leaf"><a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></li>
<li class="leaf"><a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></li>
<li class="leaf"><a href="index8aad.html?q=aggregator/sources/9">Evolutionary Bioinformatics</a></li>
<li class="leaf"><a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></li>
<li class="leaf"><a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></li>
<li class="leaf"><a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></li>
<li class="leaf"><a href="index9fb6.html?q=aggregator/sources/22">Phyloseminar</a></li>
<li class="leaf"><a href="index69a2.html?q=aggregator/sources/10">Systematics Association</a></li>
<li class="leaf"><a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></li>
<li class="leaf"><a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></li>
<li class="leaf"><a href="indexe0bd.html?q=aggregator/sources/15">dechronization</a></li>
<li class="leaf"><a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></li>
<li class="leaf"><a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></li>
<li class="leaf"><a href="index1f4a.html?q=aggregator/sources/7">postgenomic.com</a></li>

</ul>
</li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 4 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a> &raquo; <a href="index5640.html?q=aggregator">news aggregator</a> &raquo; <a href="indexe5f5.html?q=aggregator/sources">sources</a></div><h2>BMC Evolutionary Biology</h2>
<!-- begin content -->
<div id="aggregator"><div class="feed">The latest research articles published by BMC Evolutionary Biology<h3>URL</h3>
<div class="xml-icon"><a href="http://www.biomedcentral.com/bmcevolbiol/rss/"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div><a href="http://www.biomedcentral.com/bmcevolbiol/">http://www.biomedcentral.com/bmcevolbiol/</a>
<h3>Last update</h3>
11 weeks 8 hours ago</div><h3>October 7, 2015</h3>
<div class="news-item">
 <div class="date">18:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/206">Sand lizard (Lacerta agilis) phenology in a warming world</a></div>
  <div class="description">Background:
Present-day climate change has altered the phenology (the timing of periodic life cycle events) of many plant and animal populations worldwide. Some of these changes have been adaptive, leading to an increase in population fitness, whereas others have been associated with fitness decline. Representing short-term responses to an altered weather regime, hitherto observed changes are largely explained by phenotypic plasticity. However, to track climatically induced shifts in optimal phenotype as climate change proceeds, evolutionary capacity in key limiting climate- and fitness-related traits is likely to be crucial. In order to produce realistic predictions about the effects of climate change on species and populations, a main target for conservation biologists is thus to assess the potential of natural populations to respond by these two mechanisms. In this study we use a large 15-year dataset on an ectotherm model, the Swedish sand lizard (Lacerta agilis), to investigate how higher spring temperature is likely to affect oviposition timing in a high latitude population, a trait strongly linked to offspring fitness and survival.
Results:
With an interest in both the short- and potential long-term effect of rising temperatures, we applied a random regression model, which yields estimates of population-level plasticity and among-individual variation in the average, as well as the plastic, response to temperature. Population plasticity represents capacity for short-term adjustments whereas variation among individuals in a fitness-related trait indicates an opportunity for natural selection and hence for evolutionary adaptation. The analysis revealed both population-level plasticity and individual-level variation in average laying date. In contrast, we found no evidence for variation among females in their plastic responses to spring temperature, which could demonstrate a similarity in responses amongst females, but may also be due to a lack of statistical power to detect such an effect.
Conclusion:
Our findings indicate that climate warming may have positive fitness effects in this lizard population through an advancement of oviposition date. This prediction is consistent over shorter and potentially also longer time scales as the analysis revealed both population-level plasticity and individual-level variation in average laying date. However, the genetic basis for this variation would have to be examined in order to predict an evolutionary response.</div>
 </div>
</div>
<h3>October 5, 2015</h3>
<div class="news-item">
 <div class="date">18:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/220">Integration of molecular cytogenetics, dated molecular phylogeny, and model-based predictions to understand the extreme chromosome reorganization in the Neotropical genus Tonatia (Chiroptera: Phyllostomidae)</a></div>
  <div class="description">Background:
Defining factors that contributed to the fixation of a high number of underdominant chromosomal rearrangements is a complex task because not only molecular mechanisms must be considered, but also the uniqueness of natural history attributes of each taxon. Ideally, detailed investigation of the chromosome architecture of an organism and related groups, placed within a phylogenetic context, is required. We used multiple approaches to investigate the dynamics of chromosomal evolution in lineages of bats with considerable karyotypic variation, focusing on the different facets contributing to fixation of the exceptional chromosomal changes in Tonatia saurophila. Integration of empirical data with proposed models of chromosome evolution was performed to understand the probable conditions for Tonatia’s karyotypic evolution.
Results:
The trajectory of reorganization of chromosome blocks since the common ancestor of Glossophaginae and Phyllostominae subfamilies suggests that multiple tandem fusions, as well as disruption and fusions of conserved phyllostomid chromosomes were major drivers of karyotypic reshuffling in Tonatia. Considerable variation in the rates of chromosomal evolution between phyllostomid lineages was observed. Thirty–nine unique fusions and fission events reached fixation in Tonatia over a short period of time, followed by ~12 million years of chromosomal stasis. Physical mapping of repetitive DNA revealed an unusual accumulation of LINE-1 sequences on centromeric regions, probably associated with the chromosomal dynamics of this genus.
Conclusions:
Multiple rearrangements have reached fixation in a wave-like fashion in phyllostomid bats. Different biological features of Tonatia support distinct models of rearrangement fixation, and it is unlikely that the fixations were a result of solely stochastic processes in small ancient populations. Increased recombination rates were probably facilitated by expansion of repetitive DNA, reinforced by aspects of taxon reproduction and ecology.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/219">Weathering the storm: parental effort and experimental manipulation of stress hormones predict brood survival</a></div>
  <div class="description">Background:
Unpredictable and inclement weather is increasing in strength and frequency, challenging organisms to respond adaptively. One way in which animals respond to environmental challenges is through the secretion of glucocorticoid stress hormones. These hormones mobilize energy stores and suppress non-essential physiological and behavioral processes until the challenge passes. To investigate the effects of glucocorticoids on reproductive decisions, we experimentally increased corticosterone levels (the primary glucocorticoid in birds) in free-living female tree swallows, Tachycineta bicolor, during the chick-rearing stage. Due to an unprecedented cold and wet breeding season, 90 % of the nests in our study population failed, which created a unique opportunity to test how challenging environmental conditions interact with the physiological mechanisms underlying life-history trade-offs.
Results:
We found that exogenous corticosterone influenced the regulation of parental decisions in a context-dependent manner. Control and corticosterone-treated females had similar brood failure rates under unfavorable conditions (cold and rainy weather), but corticosterone treatment hastened brood mortality under more favorable conditions. Higher female nest provisioning rates prior to implantation were associated with increased probability of brood survival for treatment and control groups. However, higher pre-treatment male provisioning rates were associated with increased survival probability in the control group, but not the corticosterone-treated group.
Conclusions:
These findings reveal complex interactions between weather, female physiological state, and partner parental investment. Our results also demonstrate a causal relationship between corticosterone concentrations and individual reproductive behaviors, and point to a mechanism for why naturally disturbed populations, which experience multiple stressors, could be more susceptible and unable to respond adaptively to changing environmental conditions.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/218">High evolutionary turnover of satellite families in Caenorhabditis</a></div>
  <div class="description">Background:
The high density of tandem repeat sequences (satellites) in nematode genomes and the availability of genome sequences from several species in the group offer a unique opportunity to better understand the evolutionary dynamics and the functional role of these sequences. We take advantage of the previously developed SATFIND program to study the satellites in four Caenorhabditis species and investigate these questions.
Methods:
The identification and comparison of satellites is carried out in three steps. First we find all the satellites present in each species with the SATFIND program. Each satellite is defined by its length, number of repeats, and repeat sequence. Only satellites with at least ten repeats are considered. In the second step we build satellite families with a newly developed alignment program. Satellite families are defined by a consensus sequence and the number of satellites in the family. Finally we compare the consensus sequence of satellite families in different species.
Results:
We give a catalog of individual satellites in each species. We have also identified satellite families with a related sequence and compare them in different species. We analyze the turnover of satellites: they increased in size through duplications of fragments of 100-300 bases. It appears that in many cases they have undergone an explosive expansion. In C. elegans we have identified a subset of large satellites that have strong affinity for the centromere protein CENP-A. We have also compared our results with those obtained from other species, including one nematode and three mammals.
Conclusions:
Most satellite families found in Caenorhabditis are species-specific; in particular those with long repeats. A subset of these satellites may facilitate the formation of kinetochores in mitosis. Other satellite families in C. elegans are either related to Helitron transposons or to meiotic pairing centers.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/217">Deeply divergent sympatric mitochondrial lineages of the earthworm Lumbricus rubellus are not reproductively isolated</a></div>
  <div class="description">Background:
The accurate delimitation of species is essential to numerous areas of biological research. An unbiased assessment of the diversity, including the cryptic diversity, is of particular importance for the below ground fauna, a major component of global biodiversity. On the British Isles, the epigeic earthworm Lumbricus rubellus, which is a sentinel species in soil ecotoxicology, consists of two cryptic taxa that are differentiated in both the nuclear and the mitochondrial (mtDNA) genomes. Recently, several deeply divergent mtDNA lineages were detected in mainland Europe, but whether these earthworms also constitute cryptic species remains unclear. This information is important from an evolutionary perspective, but it is also essential for the interpretation and the design of ecotoxicological projects. In this study, we used genome-wide RADseq data to assess the reproductive isolation of the divergent mitochondrial lineages of L. rubellus that occur in sympatry in multiple localities in Central Europe.
Results:
We identified five divergent (up to 16 % net p-distance) mitochondrial lineages of L. rubellus in sympatry. Because the clustering of the RADseq data was according to the population of origin and not the mtDNA lineage, reproductive isolation among the mtDNA lineages was not likely. Although each population contained multiple mtDNA lineages, subdivisions within the populations were not observed for the nuclear genome. The lack of fixed differences and sharing of the overwhelming majority of nuclear polymorphisms between localities, indicated that the populations did not constitute allopatric species. The nucleotide diversity within the populations was high, 0.7–0.8 %.
Conclusions:
The deeply divergent mtDNA sympatric lineages of L. rubellus in Central Europe were not reproductively isolated groups. The earthworm L. rubellus, which is represented by several mtDNA lineages in continental Europe, apparently is a single highly polymorphic species rather than a complex of several cryptic species. This study demonstrated the critical importance of the use of multilocus nuclear data for the unbiased assessment of cryptic diversity and for the delimitation of species in soil invertebrates.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/216">The biogeographic origin of a radiation of trees in Madagascar: implications for the assembly of a tropical forest biome</a></div>
  <div class="description">Background:
Madagascar’s rain forests are characterized by extreme and uneven patterns of species richness and endemicity, the biogeographic and evolutionary origins of which are poorly understood.
Methods:
Here we use a time-calibrated phylogeny of a dominant group of trees in Madagascar’s eastern rain forests, Canarium, and related Burseraceae (Canarieae), to test biogeographic hypotheses regarding the origin and radiation of the flora of this unique biome.
Results:
Our findings strongly support the monophyly of Malagasy Canarium, suggesting that this clade represents a previously undocumented in situ radiation. Contrary to expectations of dispersal from Africa during the Oligocene, concurrent with the formation of Madagascar’s rain forest biome, our analyses support a late Miocene origin for Malagasy Canarium, probably by long distance dispersal from Southeast Asia.DiscussionOur study illustrates the importance of considering long distance dispersal as a viable explanation for clades with pantropical distributions diversifying subsequent to the Oligocene, and it highlights the formation of the Indo-Australian Archipelago and associated fast-moving equatorial surface currents, suggesting an under-appreciated evolutionary link among tropical centers of endemism.
Conclusions:
We postulate that the relatively recent establishment and radiation of Canarium in Madagascar may have been facilitated by the highly stochastic climates associated with these forest ecosystems.</div>
 </div>
</div>
<h3>September 30, 2015</h3>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/215">High genetic abundance of Rpi-blb2/Mi-1.2/Cami gene family in Solanaceae</a></div>
  <div class="description">Background:
Three NBS-LRR genes, Rpi-blb2, Mi-1.2, and Cami, constitute a very special plant resistance gene family. These genes confer resistance against 4 distantly related pathogen species in 3 different Solanaceae hosts. To characterize this noted resistance, we conducted a series of studies on this gene family.
Results:
First, homologs of this gene family were identified in the pepper, tomato and potato genomes. This revealed a large variation in copy number within this gene family among species and a great divergence was found both between and within species. To gain more information pertaining to gene resistance within this family, 121 LRR regions were cloned in 16 different wild/cultivated potato accessions. Again, frequent copy number variations and a high level of divergence between homolog were observed common among accessions. The divergence within species was so high that it reaches the level of divergence between species. Also, frequent frameshift mutations and abundant gene conversion events were identified in these LRR regions.
Conclusions:
Our findings suggest that this family harbors an unusually high level of genetic abundance, making it of particular interest. Together with other reported examples, our study also provides evidence that multi-resistance is a common trait in R gene families like this.</div>
 </div>
</div>
<h3>September 29, 2015</h3>
<div class="news-item">
 <div class="date">18:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/213">Intraspecific lineage divergence and its association with reproductive trait change during species range expansion in central Eurasian wild wheat Aegilops tauschii Coss. (Poaceae)</a></div>
  <div class="description">Background:
How species ranges form in landscapes is a matter of long-standing evolutionary interest. However, little is known about how natural phenotypic variations of ecologically important traits contribute to species range expansion. In this study, we examined the phylogeographic patterns of phenotypic changes in life history (seed production) and phenological (flowering time) traits during the range expansion of Aegilops tauschii Coss. from the Transcaucasus and Middle East to central Asia.
Results:
Our comparative analyses of the patterns of natural variations for those traits and their association with the intraspecific lineage structure showed that (1) the eastward expansion to Asia was driven by an intraspecific sublineage (named TauL1b), (2) high seed production ability likely had an important role at the initial dispersal stage of TauL1b’s expansion to Asia, and (3) the phenological change to early flowering phenotypes was one of the key adaptation events for TauL1b to further expand its range in Asia.
Conclusions:
This study provides for the first time a broad picture of the process of Ae. tauschii’s eastward range expansion in which life history and phenological traits may have had respective roles in its dispersal and adaptation in Asia. The clear association of seed production and flowering time patterns with the intraspecific lineage divergence found in this study invites further genetic research to bring the mechanistic understanding of the changes in these key functional traits during range expansion within reach.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/212">Host-parasite coevolution in populations of constant and variable size</a></div>
  <div class="description">Background:
The matching-allele and gene-for-gene models are widely used in mathematical approaches that study the dynamics of host-parasite interactions. Agrawal and Lively (Evolutionary Ecology Research 4:79–90, 2002) captured these two models in a single framework and numerically explored the associated time discrete dynamics of allele frequencies.
Results:
Here, we present a detailed analytical investigation of this unifying framework in continuous time and provide a generalization. We extend the model to take into account changing population sizes, which result from the antagonistic nature of the interaction and follow the Lotka-Volterra equations. Under this extension, the population dynamics become most complex as the model moves away from pure matching-allele and becomes more gene-for-gene-like. While the population densities oscillate with a single oscillation frequency in the pure matching-allele model, a second oscillation frequency arises under gene-for-gene-like conditions. These observations hold for general interaction parameters and allow to infer generic patterns of the dynamics.
Conclusion:
Our results suggest that experimentally inferred dynamical patterns of host-parasite coevolution should typically be much more complex than the popular illustrations of Red Queen dynamics. A single parasite that infects more than one host can substantially alter the cyclic dynamics.</div>
 </div>
</div>
<h3>September 28, 2015</h3>
<div class="news-item">
 <div class="date">18:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/211">In silico evolution of diauxic growth</a></div>
  <div class="description">Background:
The glucose effect is a well known phenomenon whereby cells, when presented with two different nutrients, show a diauxic growth pattern, i.e. an episode of exponential growth followed by a lag phase of reduced growth followed by a second phase of exponential growth. Diauxic growth is usually thought of as a an adaptation to maximise biomass production in an environment offering two or more carbon sources. While diauxic growth has been studied widely both experimentally and theoretically, the hypothesis that diauxic growth is a strategy to increase overall growth has remained an unconfirmed conjecture.
Methods:
Here, we present a minimal mathematical model of a bacterial nutrient uptake system and metabolism. We subject this model to artificial evolution to test under which conditions diauxic growth evolves.
Results:
As a result, we find that, indeed, sequential uptake of nutrients emerges if there is competition for nutrients and the metabolism/uptake system is capacity limited.DiscussionHowever, we also find that diauxic growth is a secondary effect of this system and that the speed-up of nutrient uptake is a much larger effect. Notably, this speed-up of nutrient uptake coincides with an overall reduction of efficiency.
Conclusions:
Our two main conclusions are: (i) Cells competing for the same nutrients evolve rapid but inefficient growth dynamics. (ii) In the deterministic models we use here no substantial lag-phase evolves. This suggests that the lag-phase is a consequence of stochastic gene expression.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">18:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/210">Alternative translation initiation codons for the plastid maturase MatK: unraveling the pseudogene misconception in the Orchidaceae</a></div>
  <div class="description">Background:
The plastid maturase MatK has been implicated as a possible model for the evolutionary “missing link” between prokaryotic and eukaryotic splicing machinery. This evolutionary implication has sparked investigations concerning the function of this unusual maturase. Intron targets of MatK activity suggest that this is an essential enzyme for plastid function. The matK gene, however, is described as a pseudogene in many photosynthetic orchid species due to presence of premature stop codons in translations, and its high rate of nucleotide and amino acid substitution.
Results:
Sequence analysis of the matK gene from orchids identified an out-of-frame alternative AUG initiation codon upstream from the consensus initiation codon used for translation in other angiosperms. We demonstrate translation from the alternative initiation codon generates a conserved MatK reading frame. We confirm that MatK protein is expressed and functions in sample orchids currently described as having a matK pseudogene using immunodetection and reverse-transcription methods. We demonstrate using phylogenetic analysis that this alternative initiation codon emerged de novo within the Orchidaceae, with several reversal events at the basal lineage and deep in orchid history.
Conclusion:
These findings suggest a novel evolutionary shift for expression of matK in the Orchidaceae and support the function of MatK as a group II intron maturase in the plastid genome of land plants including the orchids.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">18:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/209">Multilocus species trees and species delimitation in a temporal context: application to the water shrews of the genus Neomys</a></div>
  <div class="description">Background:
Multilocus data are becoming increasingly important in determining the phylogeny of closely related species and delimiting species. In species complexes where unequivocal fossil calibrations are not available, rigorous dating of the coalescence-based species trees requires accurate mutation rates of the loci under study but, generally, these rates are unknown. Here, we obtained lineage-specific mutation rates of these loci from a higher-level phylogeny with a reliable fossil record and investigated how different choices of mutation rates and species tree models affected the split time estimates. We implemented this strategy with a genus of water shrews, Neomys, whose taxonomy has been contentious over the last century.
Results:
We sequenced 13 introns and cytochrome b from specimens of the three species currently recognized in this genus including two subspecies of N. anomalus that were originally described as species. A Bayesian multilocus species delimitation method and estimation of gene flow supported that these subspecies are distinct evolutionary lineages that should be treated as distinct species: N. anomalus (sensu stricto), limited to part of the Iberian Peninsula, and N. milleri, with a larger Eurasian range. We then estimated mutation rates from a Bayesian relaxed clock analysis of the mammalian orthologues with several fossil calibrations. Next, using the estimated Neomys-specific rates for each locus in an isolation-with-migration model, the split time for these sister taxa was dated at 0.40 Myr ago (with a 95 % confidence interval of 0.26 – 0.86 Myr), likely coinciding with one of the major glaciations of the Middle Pleistocene. We also showed that the extrapolation of non-specific rates or the use of simpler models would lead to very different split time estimates.
Conclusions:
We showed that the estimation of rigorous lineage-specific mutation rates for each locus allows the inference of robust split times in a species tree framework. These times, in turn, afford a better understanding of the timeframe required to achieve isolation and, eventually, speciation in sister lineages. The application of species delimitation methods and an accurate dating strategy to the genus Neomys helped to clarify its controversial taxonomy.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">18:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/208">Life habits, hox genes, and affinities of a 311 million-year-old holometabolan larva</a></div>
  <div class="description">Background:
Holometabolous insects are the most diverse, speciose and ubiquitous group of multicellular organisms in terrestrial and freshwater ecosystems. The enormous evolutionary and ecological success of Holometabola has been attributed to their unique postembryonic life phases in which nonreproductive and wingless larvae differ significantly in morphology and life habits from their reproductive and mostly winged adults, separated by a resting stage, the pupa. Little is known of the evolutionary developmental mechanisms that produced the holometabolous larval condition and their Paleozoic origin based on fossils and phylogeny.
Results:
We provide a detailed anatomic description of a 311 million-year-old specimen, the oldest known holometabolous larva, from the Mazon Creek deposits of Illinois, U.S.A. The head is ovoidal, downwardly oriented, broadly attached to the anterior thorax, and bears possible simple eyes and antennae with insertions encircled by molting sutures; other sutures are present but often indistinct. Mouthparts are generalized, consisting of five recognizable segments: a clypeo-labral complex, mandibles, possible hypopharynx, a maxilla bearing indistinct palp-like appendages, and labium. Distinctive mandibles are robust, triangular, and dicondylic. The thorax is delineated into three, nonoverlapping regions of distinctive surface texture, each with legs of seven elements, the terminal-most bearing paired claws. The abdomen has ten segments deployed in register with overlapping tergites; the penultimate segment bears a paired, cercus-like structure. The anterior eight segments bear clawless leglets more diminutive than the thoracic legs in length and cross-sectional diameter, and inserted more ventrolaterally than ventrally on the abdominal sidewall.
Conclusions:
Srokalarva berthei occurred in an evolutionary developmental context likely responsible for the early macroevolutionary success of holometabolous insects. Srokalarva berthei bore head and prothoracic structures, leglet series on successive abdominal segments – in addition to comparable features on a second taxon eight million-years-younger – that indicates Hox-gene regulation of segmental and appendage patterning among earliest Holometabola. Srokalarva berthei body features suggest a caterpillar-like body plan and head structures indicating herbivory consistent with known, contemporaneous insect feeding damage on seed plants. Taxonomic resolution places Srokalarva berthei as an extinct lineage, apparently possessing features closer to neuropteroid than other holometabolous lineages.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">18:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/207">Divergence and adaptive evolution of the gibberellin oxidase genes in plants</a></div>
  <div class="description">Background:
The important phytohormone gibberellins (GAs) play key roles in various developmental processes. GA oxidases (GAoxs) are critical enzymes in GA synthesis pathway, but their classification, evolutionary history and the forces driving the evolution of plant GAox genes remain poorly understood.
Results:
This study provides the first large-scale evolutionary analysis of GAox genes in plants by using an extensive whole-genome dataset of 41 species, representing green algae, bryophytes, pteridophyte, and seed plants. We defined eight subfamilies under the GAox family, namely C
                      19
                   -GA2ox, C
                      20
                   -GA2ox, GA20ox,GA3ox, GAox-A, GAox-B, GAox-C and GAox-D. Of these, subfamilies GAox-A, GAox-B, GAox-C and GAox-D are described for the first time. On the basis of phylogenetic analyses and characteristic motifs of GAox genes, we demonstrated a rapid expansion and functional divergence of the GAox genes during the diversification of land plants. We also detected the subfamily-specific motifs and potential sites of some GAox genes, which might have evolved under positive selection.
Conclusions:
GAox genes originated very early—before the divergence of bryophytes and the vascular plants and the diversification of GAox genes is associated with the functional divergence and could be driven by positive selection. Our study not only provides information on the classification of GAox genes, but also facilitates the further functional characterization and analysis of GA oxidases.</div>
 </div>
</div>
<h3>September 25, 2015</h3>
<div class="news-item">
 <div class="date">18:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/205">Multidirectional chromosome painting substantiates the occurrence of extensive genomic reshuffling within Accipitriformes</a></div>
  <div class="description">Background:
Previous cross-species painting studies with probes from chicken (Gallus gallus) chromosomes 1–10 and a paint pool of nineteen microchromosomes have revealed that the drastic karyotypic reorganization in Accipitridae is due to extensive synteny disruptions and associations. However, the number of synteny association events and identities of microchromosomes involved in such synteny associations remain undefined, due to the lack of paint probes derived from individual chicken microchromosomes. Moreover, no genome-wide homology map between Accipitridae species and other avian species with atypical karyotype organization has been reported till now, and the karyotype evolution within Accipitriformes remains unclear.
Results:
To delineate the synteny-conserved segments in Accipitridae, a set of painting probes for the griffon vulture, Gyps fulvus (2n = 66) was generated from flow-sorted chromosomes. Together with previous generated probes from the stone curlew, Burhinus oedicnemus (2n = 42), a Charadriiformes species with atypical karyotype organization, we conducted multidirectional chromosome painting, including reciprocal chromosome painting between B. oedicnemus and G. fulvus and cross-species chromosome painting between B. oedicnemus and two accipitrid species (the Himalayan griffon, G. himalayensis 2n = 66, and the common buzzard, Buteo buteo, 2n = 68). In doing so, genome-wide homology maps between B. oedicnemus and three Accipitridae species were established. From there, a cladistic analysis using chromosomal characters and mapping of chromosomal changes on a consensus molecular phylogeny were conducted in order to search for cytogenetic signatures for different lineages within Accipitriformes.
Conclusion:
Our study confirmed that the genomes of the diurnal birds of prey, especially the genomes of species in Accipitriformes excluding Cathartidae, have been extensively reshuffled when compared to other bird lineages. The chromosomal rearrangements involved include both fusions and fissions. Our chromosome painting data indicated that the Palearctic common buzzard (BBU) shared several common chromosomal rearrangements with some Old World vultures, and was found to be more closely related to other Accipitridae than to Neotropical buteonine raptors from the karyotypic perspective. Using both a chromosome-based cladistic analysis as well as by mapping of chromosomal differences onto a molecular-based phylogenetic tree, we revealed a number of potential cytogenetic signatures that support the clade of Pandionidae (PHA) + Accipitridae. In addition, our cladistic analysis using chromosomal characters appears to support the placement of osprey (PHA) in Accipitridae.</div>
 </div>
</div>
<h3>September 24, 2015</h3>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/204">Divergence with gene flow across a speciation continuum of Heliconius butterflies</a></div>
  <div class="description">Background:
A key to understanding the origins of species is determining the evolutionary processes that drive the patterns of genomic divergence during speciation. New genomic technologies enable the study of high-resolution genomic patterns of divergence across natural speciation continua, where taxa pairs with different levels of reproductive isolation can be used as proxies for different stages of speciation. Empirical studies of these speciation continua can provide valuable insights into how genomes diverge during speciation.
Methods:
We examine variation across a handful of genomic regions in parapatric and allopatric populations of Heliconius butterflies with varying levels of reproductive isolation. Genome sequences were mapped to 2.2-Mb of the H. erato genome, including 1-Mb across the red color pattern locus and multiple regions unlinked to color pattern variation.
Results:
Phylogenetic analyses reveal a speciation continuum of pairs of hybridizing races and incipient species in the Heliconius erato clade. Comparisons of hybridizing pairs of divergently colored races and incipient species reveal that genomic divergence increases with ecological and reproductive isolation, not only across the locus responsible for adaptive variation in red wing coloration, but also at genomic regions unlinked to color pattern.DiscussionWe observe high levels of divergence between the incipient species H. erato and H. himera, suggesting that divergence may accumulate early in the speciation process. Comparisons of genomic divergence between the incipient species and allopatric races suggest that limited gene flow cannot account for the observed high levels of divergence between the incipient species.
Conclusions:
Our results provide a reconstruction of the speciation continuum across the H. erato clade and provide insights into the processes that drive genomic divergence during speciation, establishing the H. erato clade as a powerful framework for the study of speciation.</div>
 </div>
</div>
<h3>September 21, 2015</h3>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/203">The effects of natural selection across molecular pathways in Drosophila melanogaster</a></div>
  <div class="description">Background:
Whole-genome RNA interference post-transcriptional silencing (RNAi) is a widely used method for studying the phenotypic effects of knocking down individual genes. In this study, we use a population genomic approach to characterize the rate of evolution for proteins affecting 26 RNAi knockdown phenotypes in Drosophila melanogaster.
Results:
We find that only two of the 26 RNAi knockdown phenotypes are enriched for rapidly evolving proteins: innate immunity and regulation of Hedgehog signaling. Among all genes associated with an RNAi knockdown phenotype, we note examples in which the adaptively evolving proteins play a well-defined role in a given molecular pathway. However, most adaptively evolving proteins are found to perform more general cellular functions. When RNAi phenotypes are grouped into categories according to cellular function, we find that genes involved in the greatest number of phenotypic categories are also significantly more likely to have a history of rapid protein evolution.
Conclusions:
We show that genes that have been demonstrated to have a measurable effect on multiple molecular phenotypes show higher rates of protein evolution than genes having an effect on a single category of phenotype. Defining pleiotropy in this way yields very different results than previous studies that define pleiotropy by the number of physical interactions, which show highly connected proteins tend to evolve more slowly than lowly connected proteins. We suggest that a high degree of pleiotropy may increase the likelihood of compensatory substitution, consistent with modern theoretical work on adaptation.</div>
 </div>
</div>
<h3>September 18, 2015</h3>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/202">Wolbachia pseudogenes and low prevalence infections in tropical but not temperate Australian tephritid fruit flies: manifestations of lateral gene transfer and endosymbiont spillover?</a></div>
  <div class="description">Background:
Maternally inherited Wolbachia bacteria infect many insect species. They can also be transferred horizontally into uninfected host lineages. A Wolbachia spillover from an infected source population must occur prior to the establishment of heritable infections, but this spillover may be transient. In a previous study of tephritid fruit fly species of tropical Australia we detected a high incidence of identical Wolbachia strains in several species as well as Wolbachia pseudogenes. Here, we have investigated this further by analysing field specimens of 24 species collected along a 3,000 km climate gradient of eastern Australia.
Results:
Wolbachia sequences were detected in individuals of nine of the 24 (37 %) species. Seven (29 %) species displayed four distinct Wolbachia strains based on characterisation of full multi locus sequencing (MLST) profiles; the strains occurred as single and double infections in a small number of individuals (2–17 %). For the two remaining species all individuals had incomplete MLST profiles and Wolbachia pseudogenes that may be indicative of lateral gene transfer into host genomes. The detection of Wolbachia was restricted to northern Australia, including in five species that only occur in the tropics. Within the more widely distributed Bactrocera tryoni and Bactrocera neohumeralis, Wolbachia also only occurred in the north, and was not linked to any particular mitochondrial haplotypes.
Conclusions:
The presence of Wolbachia pseudogenes at high prevalence in two species in absence of complete MLST profiles may represent footprints of historic infections that have been lost. The detection of identical low prevalence strains in a small number of individuals of seven species may question their role as reproductive manipulator and their vertical inheritance. Instead, the findings may be indicative of transient infections that result from spillover events from a yet unknown source. These spillover events appear to be restricted to northern Australia, without proliferation in host lineages further south. Our study highlights that tropical fruit fly communities contain Wolbachia pseudogenes and may be exposed to frequent horizontal Wolbachia transfer. It also emphasises that global estimates of Wolbachia frequencies may need to consider lateral gene transfer and Wolbachia spillover that may be regionally restricted, transient and not inherited.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/201">From complex to simple: myogenesis in an aplacophoran mollusk reveals key traits in aculiferan evolution</a></div>
  <div class="description">Background:
Recent studies suggest a bifurcation at the base of Mollusca, resulting in the primarily single-shelled Conchifera (Bivalvia, Gastropoda, Scaphopoda, Monoplacophora, Cephalopoda) and the spicule-bearing Aculifera (Polyplacophora, Neomeniomorpha, Chaetodermomorpha). A recent study revealed a complex larval musculature exclusively shared by Neomeniomorpha and Polyplacophora, supporting a close relationship of both taxa. However, the ontogenetic transition from the complex larval to the simple adult neomeniomorph musculature, which mainly consists of a three-layered body-wall musculature and serially iterated dorsoventral muscles, remains unknown. To close this gap in knowledge, we studied remodeling of the larval musculature during metamorphosis in the neomeniomorph Wirenia argentea. A comparative analysis with a novel data set of a polyplacophoran, Leptochiton asellus, allows us to infer the morphology of the last common ancestor of Aculifera and the evolution of its subclades therefrom.
Results:
The complex larval musculature of Wirenia argentea persists through metamorphosis and becomes modified to form two of the three muscle layers of the adult body wall. The innermost longitudinal layer of the three-layered body wall musculature is generated by transformation and expansion of distinct larval longitudinal muscle bundles. The larval ventrolateral muscle strands are remodeled and eventually become the most ventral part of the adult longitudinal layer of the body wall musculature. The paired larval enrolling muscle forms the lateral parts and the former rectus muscle is destined to become the most dorsal part of the longitudinal layer of the body wall musculature. The transient ventromedian muscle is lost during postmetamorphic development.
Conclusions:
Postmetamorphic remodeling in W. argentea supports the hypothesis of a complex myoanatomy rather than a three-layered body wall musculature at the base of Aculifera, and thus argues against homology of the body wall musculature of adult Neomeniomorpha and other potential molluscan sister groups. Our data show that the neomeniomorph body wall musculature is a derived condition and not an aculiferan or molluscan plesiomorphy.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/200">Females sample more males at high nesting densities, but ultimately obtain less attractive mates</a></div>
  <div class="description">Background:
Sexual selection is largely driven by the availability of mates. Theory predicts that male competition and female choice should be density-dependent, with males competing more intensely at relatively high density, and females becoming increasingly discriminating when there are more males from whom to choose. Evidence for flexible mating decisions is growing, but we do not understand how environmental variation is incorporated into mate sampling strategies. We mimicked threespine stickleback (Gasterosteus aculeatus) breeding conditions in pools with high and low densities of nesting males and allowed females to search for mates to determine whether 1) mate search strategies change with the density of breeding males and 2) pre-copulatory components of mate choice (signalling, competition, search patterns, and mating decisions) are modified in parallel.
Results:
While females sampled more males at high male density, suggesting greater opportunity for sexual selection, the expanded search did not result in females choosing males with more attractive sexual signals. This is likely because red throat colouration was twice as great when half as many males competed. Instead, females chose similarly at high and low male density, using a relative strategy to compare male traits amongst potential suitors. Reduced throat colour could reflect a trade-off with costly male competition. However, we did not observe more intense competition at higher relative density. Density-dependent signalling appears largely responsible for females associating with males who have more attractive signals at low density. If we lacked knowledge of plasticity in signalling, we might have concluded that females are more discriminating at low male density.
Conclusions:
To understand interactions between mate choice and population dynamics, we should consider how components of mate choice that precede the mating decision interact.</div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="index79d7.html?q=aggregator/sources/2&amp;from=0">first page</a></div><div><a href="index092c.html?q=aggregator/sources/2&amp;from=40">previous page</a></div><div><a href="index79d7.html?q=aggregator/sources/2&amp;from=0">1</a> <a href="indexaab4.html?q=aggregator/sources/2&amp;from=20">2</a> <a href="index092c.html?q=aggregator/sources/2&amp;from=40">3</a> <strong>4</strong> <a href="index9d08.html?q=aggregator/sources/2&amp;from=80">5</a> </div><div><a href="index9d08.html?q=aggregator/sources/2&amp;from=80">next page</a></div><div><a href="index9d08.html?q=aggregator/sources/2&amp;from=80">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news." class="active">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/2&from=60 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:30:12 GMT -->
</html>
