<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=240 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 21:41:57 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index66e2.html?q=node/119">54(6) December 2005</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 3 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>January 22, 2016</h3>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/352?rss=1">Stepwise Functional Evolution in a Fungal Sugar Transporter Family</a></div>
  <div class="description"><p>Sugar transport is of the utmost importance for most cells and is central to a wide range of applied fields. However, despite the straightforward in silico assignment of many novel transporters, including sugar porters, to existing families, their exact biological role and evolutionary trajectory often remain unclear, mainly because biochemical characterization of membrane proteins is inherently challenging, but also owing to their uncommonly turbulent evolutionary histories. In addition, many important shifts in membrane carrier function are apparently ancient, which further limits our ability to reconstruct evolutionary trajectories in a reliable manner. Here, we circumvented some of these obstacles by examining the relatively recent emergence of a unique family of fungal sugar facilitators, related to drug antiporters. The former transporters, named Ffz, were previously shown to be required for fructophilic metabolism in yeasts. We first exploited the wealth of fungal genomic data available to define a comprehensive but well-delimited family of Ffz-like transporters, showing that they are only present in Dikarya. Subsequently, a combination of phylogenetic analyses and in vivo functional characterization was used to retrace important changes in function, while highlighting the evolutionary events that are most likely to have determined extant distribution of the gene, such as horizontal gene transfers (HGTs). One such HGT event is proposed to have set the stage for the onset of fructophilic metabolism in yeasts, a trait that according to our results may be the metabolic hallmark of close to 100 yeast species that thrive in sugar rich environments.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/367?rss=1">Ancient Male Recombination Shaped Genetic Diversity of Neo-Y Chromosome in Drosophila albomicans</a></div>
  <div class="description"><p>Researchers studying Y chromosome evolution have drawn attention to neo-Y chromosomes in <I>Drosophila</I> species due to their resembling the initial stage of Y chromosome evolution. In the studies of neo-Y chromosome of <I>Drosophila miranda</I>, the extremely low genetic diversity observed suggested various modes of natural selection acting on the nonrecombining genome. However, alternative possibility may come from its peculiar origin from a single chromosomal fusion event with male achiasmy, which potentially caused and maintained the low genetic diversity of the neo-Y chromosome. Here, we report a real case where a neo-Y chromosome is in transition from an autosome to a typical Y chromosome. The neo-Y chromosome of <I>Drosophila albomicans</I> harbored a rich genetic diversity comparable to its gametologous neo-X chromosome and an autosome in the same genome. Analyzing sequence variations in 53 genes and measuring recombination rates between pairs of loci by cross experiments, we elucidated the evolutionary scenario of the neo-Y chromosome of <I>D. albomicans</I> having high genetic diversity without assuming selective force, i.e., it originated from a single chromosomal fusion event, experienced meiotic recombination during the initial stage of evolution and diverged from neo-X chromosome by the suppression of recombination tens or a few hundreds of thousand years ago. Consequently, the observed high genetic diversity on the neo-Y chromosome suggested a strong effect of meiotic recombination to introduce genetic variations into the newly arisen sex chromosome.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/375?rss=1">A Distinct Mitochondrial Genome with DUI-Like Inheritance in the Ocean Quahog Arctica islandica</a></div>
  <div class="description"><p>Mitochondrial DNA (mtDNA) is strictly maternally inherited in metazoans. The major exception to this rule has been found in many bivalve species which allow the presence of different sex-linked mtDNA molecules. This mechanism, named doubly uniparental inheritance (DUI), is characterized by the presence of two mtDNAs: The female mtDNA is found in somatic tissue and female gonads, whereas the male mtDNA is usually found in male gonads and sperm. In this study we highlight the existence of two divergent mitochondrial haplotypes with a low genetic difference around 6&ndash;8% in <I>Arctica islandica</I>, a long-lived clam belonging to the Arcticidae, a sister group to the Veneridae in which DUI has been found. Phylogenetic analysis on <I>cytochrome b</I> and <I>16S</I> sequences from somatic and gonadic tissues of clams belonging to different populations reveals the presence of the "divergent" type in male gonads only and the "normal" type in somatic tissues and female gonads. This peculiar segregation of divergent mtDNA types speaks for the occurrence of the DUI mechanism in <I>A. islandica</I>. This example also highlights the difficulties to assess the presence of such particular mitochondrial inheritance system and underlines the possible misinterpretations in phylogeographic and phylogenetic studies of bivalve species linked to the presence of two poorly differentiated mitochondrial genomes.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/384?rss=1">Strong Constraint on Human Genes Escaping X-Inactivation Is Modulated by their Expression Level and Breadth in Both Sexes</a></div>
  <div class="description"><p>In eutherian mammals, X-linked gene expression is normalized between XX females and XY males through the process of X chromosome inactivation (XCI). XCI results in silencing of transcription from one ChrX homolog per female cell. However, approximately 25% of human ChrX genes escape XCI to some extent and exhibit biallelic expression in females. The evolutionary basis of this phenomenon is not entirely clear, but high sequence conservation of XCI escapers suggests that purifying selection may directly or indirectly drive XCI escape at these loci. One hypothesis is that this signal results from contributions to developmental and physiological sex differences, but presently there is limited evidence supporting this model in humans. Another potential driver of this signal is selection for high and/or broad gene expression in both sexes, which are strong predictors of reduced nucleotide substitution rates in mammalian genes. Here, we compared purifying selection and gene expression patterns of human XCI escapers with those of X-inactivated genes in both sexes. When we accounted for the functional status of each ChrX gene&rsquo;s Y-linked homolog (or "gametolog"), we observed that XCI escapers exhibit greater degrees of purifying selection in the human lineage than X-inactivated genes, as well as higher and broader gene expression than X-inactivated genes across tissues in both sexes. These results highlight a significant role for gene expression in both sexes in driving purifying selection on XCI escapers, and emphasize these genes&rsquo; potential importance in human disease.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/394?rss=1">Resolution of Brassicaceae Phylogeny Using Nuclear Genes Uncovers Nested Radiations and Supports Convergent Morphological Evolution</a></div>
  <div class="description"><p>Brassicaceae is one of the most diverse and economically valuable angiosperm families with widely cultivated vegetable crops and scientifically important model plants, such as <I>Arabidopsis thaliana</I>. The evolutionary history, ecological, morphological, and genetic diversity, and abundant resources and knowledge of Brassicaceae make it an excellent model family for evolutionary studies. Recent phylogenetic analyses of the family revealed three major lineages (I, II, and III), but relationships among and within these lineages remain largely unclear. Here, we present a highly supported phylogeny with six major clades using nuclear markers from newly sequenced transcriptomes of 32 Brassicaceae species and large data sets from additional taxa for a total of 55 species spanning 29 out of 51 tribes. Clade A consisting of Lineage I and <I>Macropodium nivale</I> is sister to combined Clade B (with Lineage II and others) and a new Clade C. The ABC clade is sister to Clade D with species previously weakly associated with Lineage II and Clade E (Lineage III) is sister to the ABCD clade. Clade F (the tribe Aethionemeae) is sister to the remainder of the entire family. Molecular clock estimation reveals an early radiation of major clades near or shortly after the Eocene&ndash;Oligocene boundary and subsequent nested divergences of several tribes of the previously polytomous Expanded Lineage II. Reconstruction of ancestral morphological states during the Brassicaceae evolution indicates prevalent parallel (convergent) evolution of several traits over deep times across the entire family. These results form a foundation for future evolutionary analyses of structures and functions across Brassicaceae.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/413?rss=1">Drosophila X-Linked Genes Have Lower Translation Rates than Autosomal Genes</a></div>
  <div class="description"><p>In <I>Drosophila</I>, X-linked and autosomal genes achieve comparable expression at the mRNA level. Whether comparable X-autosome gene expression is realized at the translational and, ultimately, the protein levels is, however, unknown. Previous studies suggest the possibility of higher translation rates for X-linked genes owing to stronger usage of preferred codons. In this study, we use public ribosome profiling data from <I>Drosophila melanogaster</I> to infer translation rates on the X chromosome versus the autosomes. We find that X-linked genes have consistently lower ribosome densities than autosomal genes in S2 cells, early embryos, eggs, and mature oocytes. Surprisingly, the lower ribosome densities of X-linked genes are not consistent with faster translation elongation but instead imply slower translation initiation. In particular, X-linked genes have sequence features known to slow translation initiation such as stronger mRNA structure near start codons and longer 5'-UTRs. Comparison to outgroup species suggests that stronger mRNA structure is an evolved feature of <I>Drosophila</I> X chromosomes. Finally, we find that the magnitude of the X-autosome difference in ribosome densities is smaller for genes encoding members of protein complexes, suggesting that stoichiometry constrains the evolution of translation rates. In sum, our analyses suggest that <I>Drosophila</I> X-linked genes have evolved lower translation rates than autosomal genes despite stronger usage of preferred codons.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/429?rss=1">SHV-129: A Gateway to Global Suppressors in the SHV {beta}-Lactamase Family?</a></div>
  <div class="description"><p>Enzymes are continually evolving in response to environmental pressures. In order to increase enzyme fitness, amino acid substitutions can occur leading to a changing function or an increased stability. These evolutionary drivers determine the activity of an enzyme and its success in future generations in response to changing conditions such as environmental stressors or to improve physiological function allowing continual persistence of the enzyme. With recent warning reports on antibiotic resistance and multidrug resistant bacterial infections, understanding the evolution of &beta;-lactamase enzymes, which are a large contributor to antibiotic resistance, is increasingly important. Here, we investigated a variant of the SHV &beta;-lactamase identified from a clinical isolate of <I>Escherichia coli</I> in 2011 (SHV-129, G238S-E240K-R275L-N276D) to identify the first instance of a global suppressor substitution in the SHV &beta;-lactamase family. We have used this enzyme to show that several evolutionary principles are conserved in different class A &beta;-lactamases, such as active site mutations reducing stability and requiring compensating suppressor substitutions in order to ensure evolutionary persistence of a given &beta;-lactamase. However, the pathway taken by a given &beta;-lactamase in order to reach its evolutionary peak under a given set of conditions is likely different. We also provide further evidence for a conserved stabilizing substitution among class A &beta;-lactamases, the back to consensus M182T substitution. In addition to expanding the spectrum of &beta;-lactamase activity to include the hydrolysis of cefepime, the amino acid substitutions found in SHV-129 provide the enzyme with an excess of stability, which expands the evolutionary landscape of this enzyme and may result in further evolution to potentially include resistance to carbapenems or &beta;-lactamase inhibitors.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/442?rss=1">Adaptive Evolution Is Substantially Impeded by Hill-Robertson Interference in Drosophila</a></div>
  <div class="description"><p>Hill&ndash;Robertson interference (HRi) is expected to reduce the efficiency of natural selection when two or more linked selected sites do not segregate freely, but no attempt has been done so far to quantify the overall impact of HRi on the rate of adaptive evolution for any given genome. In this work, we estimate how much HRi impedes the rate of adaptive evolution in the coding genome of <I>Drosophila melanogaster.</I> We compiled a data set of 6,141 autosomal protein-coding genes from <I>Drosophila</I>, from which polymorphism levels in <I>D. melanogaster</I> and divergence out to <I>D. yakuba</I> were estimated. The rate of adaptive evolution was calculated using a derivative of the McDonald&ndash;Kreitman test that controls for slightly deleterious mutations. We find that the rate of adaptive amino acid substitution at a given position of the genome is positively correlated to both the rate of recombination and the mutation rate, and negatively correlated to the gene density of the region. These correlations are robust to controlling for each other, for synonymous codon bias and for gene functions related to immune response and testes. We show that HRi diminishes the rate of adaptive evolution by approximately 27%. Interestingly, genes with low mutation rates embedded in gene poor regions lose approximately 17% of their adaptive substitutions whereas genes with high mutation rates embedded in gene rich regions lose approximately 60%. We conclude that HRi hampers the rate of adaptive evolution in <I>Drosophila</I> and that the variation in recombination, mutation, and gene density along the genome affects the HRi effect.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/456?rss=1">The Landscape of Realized Homologous Recombination in Pathogenic Bacteria</a></div>
  <div class="description"><p>Recombination enhances the adaptive potential of organisms by allowing genetic variants to be tested on multiple genomic backgrounds. Its distribution in the genome can provide insight into the evolutionary forces that underlie traits, such as the emergence of pathogenicity. Here, we examined landscapes of realized homologous recombination of 500 genomes from ten bacterial species and found all species have "hot" regions with elevated rates relative to the genome average. We examined the size, gene content, and chromosomal features associated with these regions and the correlations between closely related species. The recombination landscape is variable and evolves rapidly. For example in <I>Salmonella</I>, only short regions of around 1 kb in length are hot whereas in the closely related species <I>Escherichia coli</I>, some hot regions exceed 100 kb, spanning many genes. Only <I>Streptococcus pyogenes</I> shows evidence for the positive correlation between GC content and recombination that has been reported for several eukaryotes. Genes with function related to the cell surface/membrane are often found in recombination hot regions but <I>E. coli</I> is the only species where genes annotated as "virulence associated" are consistently hotter. There is also evidence that some genes with "housekeeping" functions tend to be overrepresented in cold regions. For example, ribosomal proteins showed low recombination in all of the species. Among specific genes, transferrin-binding proteins are recombination hot in all three of the species in which they were found, and are subject to interspecies recombination.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/472?rss=1">Gal3 Binds Gal80 Tighter than Gal1 Indicating Adaptive Protein Changes Following Duplication</a></div>
  <div class="description"><p>Derived from the yeast whole-genome duplication, <I>Saccharomyces cerevisiae GAL1</I> and <I>GAL3</I> encode the catabolic enzyme galactokinase (Gal1) and its transcriptional coinducer (Gal3), whereas the ancestral, preduplicated <I>GAL1</I> gene performed both functions. Previous studies indicated that divergence was primarily driven by changes in upstream promoter elements, and changes in <I>GAL3</I>&rsquo;s coding region are assumed to be the result of drift. We show that replacement of <I>GAL3</I>&rsquo;s open-reading-frame with <I>GAL1</I>&rsquo;s results in an extended lag phase upon switching to growth on galactose with up to 2.5-fold differences in the initial cell masses. Accordingly, the binding affinity of Gal3 to Gal80 was found to be greater than 10-folds higher than that of Gal1, with both a higher association rate (<I>ka</I>) and lower dissociation (<I>kd</I>) rate. Thus, while changes in the noncoding, regulatory regions were the initial driving force for <I>GAL3</I>&rsquo;s subfunctionalization as a coinducer, adaptive changes in the protein sequence seem to have followed.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/478?rss=1">Genomic Characterization of a South American Phytophthora Hybrid Mandates Reassessment of the Geographic Origins of Phytophthora infestans</a></div>
  <div class="description"><p>As the oomycete pathogen causing potato late blight disease, <I>Phytophthora infestans</I> triggered the famous 19th-century Irish potato famine and remains the leading cause of global commercial potato crop destruction. But the geographic origin of the genotype that caused this devastating initial outbreak remains disputed, as does the New World center of origin of the species itself. Both Mexico and South America have been proposed, generating considerable controversy. Here, we readdress the pathogen&rsquo;s origins using a genomic data set encompassing 71 globally sourced modern and historical samples of <I>P. infestans</I> and the hybrid species <I>P. andina</I>, a close relative known only from the Andean highlands. Previous studies have suggested that the nuclear DNA lineage behind the initial outbreaks in Europe in 1845 is now extinct. Analysis of <I>P. andina</I>&rsquo;s phased haplotypes recovered eight haploid genome sequences, four of which represent a previously unknown basal lineage of <I>P. infestans</I> closely related to the famine-era lineage. Our analyses further reveal that clonal lineages of both <I>P. andina</I> and historical <I>P. infestans</I> diverged earlier than modern Mexican lineages, casting doubt on recent claims of a Mexican center of origin. Finally, we use haplotype phasing to demonstrate that basal branches of the clade comprising Mexican samples are occupied by clonal isolates collected from wild <I>Solanum</I> hosts, suggesting that modern Mexican <I>P. infestans</I> diversified on <I>Solanum tuberosum</I> after a host jump from a wild species and that the origins of <I>P. infestans</I> are more complex than was previously thought.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/492?rss=1">A Hybrid Drug Limits Resistance by Evading the Action of the Multiple Antibiotic Resistance Pathway</a></div>
  <div class="description"><p>Hybrid drugs are a promising strategy to address the growing problem of drug resistance, but the mechanism by which they modulate the evolution of resistance is poorly understood. Integrating high-throughput resistance measurements and genomic sequencing, we compared <I>Escherichia coli</I> populations evolved in a hybrid antibiotic that links ciprofloxacin and neomycin B with populations evolved in combinations of the component drugs. We find that populations evolved in the hybrid gain less resistance than those evolved in an equimolar mixture of the hybrid&rsquo;s components, in part because the hybrid evades resistance mediated by the multiple antibiotic resistance (<I>mar</I>) operon. Furthermore, we find that the ciprofloxacin moiety of the hybrid inhibits bacterial growth whereas the neomycin B moiety diminishes the effectiveness of <I>mar</I> activation. More generally, comparing the phenotypic and genotypic paths to resistance across different drug treatments can pinpoint unique properties of new compounds that limit the emergence of resistance.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/501?rss=1">Shared Genetic Signals of Hypoxia Adaptation in Drosophila and in High-Altitude Human Populations</a></div>
  <div class="description"><p>The ability to withstand low oxygen (hypoxia tolerance) is a polygenic and mechanistically conserved trait that has important implications for both human health and evolution. However, little is known about the diversity of genetic mechanisms involved in hypoxia adaptation in evolving populations. We used experimental evolution and whole-genome sequencing in <I>Drosophila melanogaster</I> to investigate the role of natural variation in adaptation to hypoxia. Using a generalized linear mixed model we identified significant allele frequency differences between three independently evolved hypoxia-tolerant populations and normoxic control populations for approximately 3,800 single nucleotide polymorphisms. Around 50% of these variants are clustered in 66 distinct genomic regions. These regions contain genes that are differentially expressed between hypoxia-tolerant and normoxic populations and several of the differentially expressed genes are associated with metabolic processes. Additional genes associated with respiratory and open tracheal system development also show evidence of directional selection. RNAi-mediated knockdown of several candidate genes&rsquo; expression significantly enhanced survival in severe hypoxia. Using genomewide single nucleotide polymorphism data from four high-altitude human populations&mdash;Sherpas, Tibetans, Ethiopians, and Andeans, we found that several human orthologs of the genes under selection in flies are also likely under positive selection in all four high-altitude human populations. Thus, our results indicate that selection for hypoxia tolerance can act on standing genetic variation in similar genes and pathways present in organisms diverged by hundreds of millions of years.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/518?rss=1">Determinants of the Usage of Splice-Associated cis-Motifs Predict the Distribution of Human Pathogenic SNPs</a></div>
  <div class="description"><p>Where in genes do pathogenic mutations tend to occur and does this provide clues as to the possible underlying mechanisms by which single nucleotide polymorphisms (SNPs) cause disease? As splice-disrupting mutations tend to occur predominantly at exon ends, known also to be hot spots of <I>cis</I>-exonic splice control elements, we examine the relationship between the relative density of such exonic <I>cis</I>-motifs and pathogenic SNPs. In particular, we focus on the intragene distribution of exonic splicing enhancers (ESE) and the covariance between them and disease-associated SNPs. In addition to showing that disease-causing genes tend to be genes with a high intron density, consistent with missplicing, five factors established as trends in ESE usage, are considered: relative position in exons, relative position in genes, flanking intron size, splice sites usage, and phase. We find that more than 76% of pathogenic SNPs are within 3&ndash;69 bp of exon ends where ESEs generally reside, this being 13% more than expected. Overall from enrichment of pathogenic SNPs at exon ends, we estimate that approximately 20&ndash;45% of SNPs affect splicing. Importantly, we find that within genes pathogenic SNPs tend to occur in splicing-relevant regions with low ESE density: they are found to occur preferentially in the terminal half of genes, in exons flanked by short introns and at the ends of phase (0,0) exons with 3' non-"AGgt" splice site. We suggest the concept of the "fragile" exon, one home to pathogenic SNPs owing to its vulnerability to splice disruption owing to low ESE density.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/530?rss=1">Expansion of Noncanonical V-Arm-Containing tRNAs in Eukaryotes</a></div>
  <div class="description"><p>Transfer RNA (tRNA) is essential for the translation of genetic information into proteins, and understanding its molecular evolution is important if we are to understand the genetic code. In general, long variable-arm (V-arm) structures form in tRNALeu, tRNASer, and bacterial and organellar tRNATyr. However, as we have previously reported, noncanonical V-arms occur in nematode tRNAGly and tRNAIle, and potentially affect translational fidelity. Here, we comprehensively analyzed 69 eukaryotic genome sequences and examined the evolutionary divergence of the V-arm-containing tRNAs. In total, 253 V-arm-containing tRNAs, with neither leucine nor serine anticodons, were identified in organisms ranging from nematodes to fungi, plants, and vertebrates. We defined them as "noncanonical V-arm-containing tRNAs" (nov-tRNAs). Moreover, 2,415 nov-tRNA-like sequences lacking some of the conserved features of tRNAs were also identified, largely in vertebrate genomes. These nov-tRNA/nov-tRNA-like sequences can be categorized into three types, based on differences in their possible evolutionary origins. The type A nov-tRNAs in nematodes probably evolved not only from tRNALeu but also from tRNASer and other isotypes on several independent occasions. The type B nov-tRNAs are dispersed abundantly throughout vertebrate genomes, and seem to have originated from retrotransposable elements. The type C nov-tRNAs may have been acquired from plant chloroplasts or from bacteria through horizontal transfer. Our findings provide unexpected insight into the evolution of the tRNA molecule, which was diverse and occurred independently in nematodes, vertebrates, and plants.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/541?rss=1">Genetic Determinism and Evolutionary Reconstruction of a Host Jump in a Plant Virus</a></div>
  <div class="description"><p>In spite of their widespread occurrence, only few host jumps by plant viruses have been evidenced and the molecular bases of even fewer have been determined. A combination of three independent approaches, 1) experimental evolution followed by reverse genetics analysis, 2) positive selection analysis, and 3) locus-by-locus analysis of molecular variance (AMOVA) allowed reconstructing the <I>Potato virus Y</I> (PVY; genus <I>Potyvirus</I>, family <I>Potyviridae</I>) jump to pepper (<I>Capsicum annuum</I>), probably from other solanaceous plants. Synthetic chimeras between infectious cDNA clones of two PVY isolates with contrasted levels of adaptation to <I>C. annuum</I> showed that the P3 and, to a lower extent, the CI cistron played important roles in infectivity toward <I>C. annuum</I>. The three analytical approaches pinpointed a single nonsynonymous substitution in the P3 and P3N-PIPO cistrons that evolved several times independently and conferred adaptation to <I>C. annuum</I>. In addition to increasing our knowledge of host jumps in plant viruses, this study illustrates also the efficiency of locus-by-locus AMOVA and combined approaches to identify adaptive mutations in the genome of RNA viruses.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/554?rss=1">DNA Editing of LTR Retrotransposons Reveals the Impact of APOBECs on Vertebrate Genomes</a></div>
  <div class="description"><p>Long terminal repeat retrotransposons (LTR) are widespread in vertebrates and their dynamism facilitates genome evolution. However, these endogenous retroviruses (ERVs) must be restricted to maintain genomic stability. The APOBECs, a protein family that can edit C-to-U in DNA, do so by interfering with reverse transcription and hypermutating retrotransposon DNA. In some cases, a retrotransposon may integrate into the genome despite being hypermutated. Such an event introduces a unique sequence into the genome, increasing retrotransposon diversity and the probability of developing new function at the locus of insertion. The prevalence of this phenomenon and its effects on vertebrate genomes are still unclear. In this study, we screened ERV sequences in the genomes of 123 diverse species and identified hundreds of thousands of edited sites in multiple vertebrate lineages, including placental mammals, marsupials, and birds. Numerous edited ERVs carry high mutation loads, some with greater than 350 edited sites, profoundly damaging their open-reading frames. For many of the species studied, this is the first evidence that APOBECs are active players in their innate immune system. Unexpectedly, some birds and especially zebra finch and medium ground-finch (one of Darwin&rsquo;s finches) are exceptionally enriched in DNA editing. We demonstrate that edited retrotransposons may be preferentially retained in active genomic regions, as reflected from their enrichment in genes, exons, promoters, and transcription start sites, thereby raising the probability of their exaptation for novel function. In conclusion, DNA editing of retrotransposons by APOBECs has a substantial role in vertebrate innate immunity and may boost genome evolution.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/568?rss=1">Evolution of SUMO Function and Chain Formation in Insects</a></div>
  <div class="description"><p>SUMOylation, the covalent binding of Small Ubiquitin-like Modifier (SUMO) to target proteins, is a posttranslational modification that regulates critical cellular processes in eukaryotes. In insects, SUMOylation has been studied in holometabolous species, particularly in the dipteran <I>Drosophila melanogaster</I>, which contains a single <I>SUMO</I> gene (<I>smt3</I>). This has led to the assumption that insects contain a single <I>SUMO</I> gene. However, the analysis of insect genomes shows that basal insects contain two <I>SUMO</I> genes, orthologous to vertebrate <I>SUMO1</I> and <I>SUMO2/3.</I> Our phylogenetical analysis reveals that the <I>SUMO</I> gene has been duplicated giving rise to <I>SUMO1</I> and <I>SUMO2/3</I> families early in Metazoan evolution, and that later in insect evolution the <I>SUMO1</I> gene has been lost after the Hymenoptera divergence. To explore the consequences of this loss, we have examined the characteristics and different biological functions of the two <I>SUMO</I> genes (<I>SUMO1</I> and <I>SUMO3</I>) in the hemimetabolous cockroach <I>Blattella germanica</I> and compared them with those of <I>Drosophila Smt3.</I> Here, we show that the metamorphic role of the <I>SUMO</I> genes is evolutionary conserved in insects, although there has been a regulatory switch from <I>SUMO1</I> in basal insects to <I>SUMO3</I> in more derived ones. We also show that, unlike vertebrates, insect SUMO3 proteins cannot form polySUMO chains due to the loss of critical lysine residues within the N-terminal part of the protein. Furthermore, the formation of polySUMO chains by expression of ectopic human SUMO3 has a deleterious effect in <I>Drosophila</I>. These findings contribute to the understanding of the functional consequences of the evolution of <I>SUMO</I> genes.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/585?rss=1">Disentangling Sources of Selection on Exonic Transcriptional Enhancers</a></div>
  <div class="description"><p>In addition to coding for proteins, exons can also impact transcription by encoding regulatory elements such as enhancers. It has been debated whether such features confer heightened selective constraint, or evolve neutrally. We have addressed this question by developing a new approach to disentangle the sources of selection acting on exonic enhancers, in which we model the evolutionary rates of every possible substitution as a function of their effects on both protein sequence and enhancer activity. In three exonic enhancers, we found no significant association between evolutionary rates and effects on enhancer activity. This suggests that despite having biochemical activity, these exonic enhancers have no detectable selective constraint, and thus are unlikely to play a major role in protein evolution.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/591?rss=1">Computationally Efficient Composite Likelihood Statistics for Demographic Inference</a></div>
  <div class="description"><p>Many population genetics tools employ composite likelihoods, because fully modeling genomic linkage is challenging. But traditional approaches to estimating parameter uncertainties and performing model selection require full likelihoods, so these tools have relied on computationally expensive maximum-likelihood estimation (MLE) on bootstrapped data. Here, we demonstrate that statistical theory can be applied to adjust composite likelihoods and perform robust computationally efficient statistical inference in two demographic inference tools: <I></I>a<I></I>i and TRACTS. On both simulated and real data, the adjustments perform comparably to MLE bootstrapping while using orders of magnitude less computational time.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="index933b.html?q=aggregator&amp;from=220">previous page</a></div><div>... <a href="indexf37c.html?q=aggregator&amp;from=160">9</a> <a href="index52c1.html?q=aggregator&amp;from=180">10</a> <a href="indexeef0.html?q=aggregator&amp;from=200">11</a> <a href="index933b.html?q=aggregator&amp;from=220">12</a> <strong>13</strong> <a href="index3f45.html?q=aggregator&amp;from=260">14</a> <a href="index4603.html?q=aggregator&amp;from=280">15</a> <a href="index8c8d.html?q=aggregator&amp;from=300">16</a> <a href="index9efb.html?q=aggregator&amp;from=320">17</a> ...</div><div><a href="index3f45.html?q=aggregator&amp;from=260">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museums collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=240 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 21:41:57 GMT -->
</html>
