<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=360 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:36:28 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="indexacd7.html?q=node/208">56(4) August 2007</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 8 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>December 29, 2015</h3>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/i2?rss=1">Associate Editors</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/i3?rss=1">SMBE Editors and Council</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/4?rss=1">The Convergent Cancer Evolution toward a Single Cellular Destination</a></div>
  <div class="description"><p>The essence of Darwin&rsquo;s theory is that evolution is driven by purposeless mutations that are subsequently selected by natural environments, so there is often no predefined destination in organismal evolution. Using gene expressions of 107 cell types, we built a functional space of human cells to trace the evolutionary trajectory of 18 types of solid tumor cancers. We detected a dominant evolving trend toward the functional status of embryonic stem cells (ESC) for approximately 3,000 tumors growing in distinct tissue environments. This pattern remained the same after excluding known cancer/ESC signature genes (~3,000 genes) or excluding all oncogenic gene sets (~12,000 genes) annotated in MSigDB, suggesting a convergent evolution of the overall functional status in cancers. In support of this, the functional distance to ESC served as a common prognostic indicator for cancers of various types, with shorter distance corresponding to poor prognosis, which was true even when randomly selected gene sets were considered. Thus, regardless of the external environments, cancer evolution is a directional process toward a defined cellular destination, a finding reconciling development and evolution, the two seemingly incompatible philosophies both adopted by the cancer research community, and also raising new questions to evolutionary biology.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/i4?rss=1">Subscription Page</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/13?rss=1">Phenoscape: Identifying Candidate Genes for Evolutionary Phenotypes</a></div>
  <div class="description"><p>Phenotypes resulting from mutations in genetic model organisms can help reveal candidate genes for evolutionarily important phenotypic changes in related taxa. Although testing candidate gene hypotheses experimentally in nonmodel organisms is typically difficult, ontology-driven information systems can help generate testable hypotheses about developmental processes in experimentally tractable organisms. Here, we tested candidate gene hypotheses suggested by expert use of the Phenoscape Knowledgebase, specifically looking for genes that are candidates responsible for evolutionarily interesting phenotypes in the ostariophysan fishes that bear resemblance to mutant phenotypes in zebrafish. For this, we searched ZFIN for genetic perturbations that result in either loss of basihyal element or loss of scales phenotypes, because these are the ancestral phenotypes observed in catfishes (Siluriformes). We tested the identified candidate genes by examining their endogenous expression patterns in the channel catfish, <I>Ictalurus punctatus</I>. The experimental results were consistent with the hypotheses that these features evolved through disruption in developmental pathways at, or upstream of, <I>brpf1</I> and <I>eda</I>/<I>edar</I> for the ancestral losses of basihyal element and scales, respectively. These results demonstrate that ontological annotations of the phenotypic effects of genetic alterations in model organisms, when aggregated within a knowledgebase, can be used effectively to generate testable, and useful, hypotheses about evolutionary changes in morphology.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/40?rss=1">Using THz Spectroscopy, Evolutionary Network Analysis Methods, and MD Simulation to Map the Evolution of Allosteric Communication Pathways in c-Type Lysozymes</a></div>
  <div class="description"><p>It is now widely accepted that protein function is intimately tied with the navigation of energy landscapes. In this framework, a protein sequence is not described by a distinct structure but rather by an ensemble of conformations. And it is through this ensemble that evolution is able to modify a protein&rsquo;s function by altering its landscape. Hence, the evolution of protein functions involves selective pressures that adjust the sampling of the conformational states. In this work, we focus on elucidating the evolutionary pathway that shaped the function of individual proteins that make-up the mammalian c-type lysozyme subfamily. Using both experimental and computational methods, we map out specific intermolecular interactions that direct the sampling of conformational states and accordingly, also underlie shifts in the landscape that are directly connected with the formation of novel protein functions. By contrasting three representative proteins in the family we identify molecular mechanisms that are associated with the selectivity of enhanced antimicrobial properties and consequently, divergent protein function. Namely, we link the extent of localized fluctuations involving the loop separating helices A and B with shifts in the equilibrium of the ensemble of conformational states that mediate interdomain coupling and concurrently moderate substrate binding affinity. This work reveals unique insights into the molecular level mechanisms that promote the progression of interactions that connect the immune response to infection with the nutritional properties of lactation, while also providing a deeper understanding about how evolving energy landscapes may define present-day protein function.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/62?rss=1">Widespread and Adaptive Alterations in Genome-Wide Gene Expression Associated with Ecological Divergence of Two Oryza Species</a></div>
  <div class="description"><p>Ecological speciation is a common mechanism by which new species arise. Despite great efforts, the role of gene expression in ecological divergence and speciation is poorly understood. Here, we conducted a genome-wide gene expression investigation of two <I>Oryza</I> species that are evolutionarily young and distinct in ecology and morphology. Using digital gene expression technology and the paired-end RNA sequencing method, we obtained 21,415 expressed genes across three reproduction-related tissues. Of them, approximately 8% (1,717) differed significantly in expression levels between the two species and these differentially expressed genes are randomly distributed across the genome. Moreover, 62% (1,064) of the differentially expressed genes exhibited a signature of directional selection in at least one species. Importantly, the genes with differential expression between species evolved more rapidly at the 5' flanking sequences than the genes without differential expression relative to coding sequences, suggesting that <I>cis</I>-regulatory changes are likely adaptive and play an important role in the ecological divergence of the two species. Finally, we showed evidence of significant differentiation between species in phenotype traits and observed that genes with differential expression were overrepresented with functional terms involving phenotypic and ecological differentiation between the two species, including reproduction- and stress-related characteristics. Our findings demonstrate that ecological speciation is associated with widespread and adaptive alterations in genome-wide gene expression and provide new insights into the importance of regulatory evolution in ecological speciation in plants.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/79?rss=1">Transcriptome-Wide Differential Gene Expression in Bicyclus anynana Butterflies: Female Vision-Related Genes Are More Plastic</a></div>
  <div class="description"><p>Vision is energetically costly to maintain. Consequently, over time many cave-adapted species downregulate the expression of vision genes or even lose their eyes and associated eye genes entirely. Alternatively, organisms that live in fluctuating environments, with different requirements for vision at different times, may evolve phenotypic plasticity for expression of vision genes. Here, we use a global transcriptomic and candidate gene approach to compare gene expression in the heads of a polyphenic butterfly. <I>Bicyclus anynana</I> have two seasonal forms that display sexual dimorphism and plasticity in eye morphology, and female-specific plasticity in opsin gene expression. Nonchoosy dry season females downregulate opsin expression, consistent with the high physiological cost of vision. To identify other genes associated with sexually dimorphic and seasonally plastic differences in vision, we analyzed RNA-sequencing data from whole head tissues. We identified two eye development genes (<I>klarsicht</I> and <I>warts</I> homologs) and an eye pigment biosynthesis gene (<I>henna</I>) differentially expressed between seasonal forms. By comparing sex-specific expression across seasonal forms, we found that <I>klarsicht</I>, <I>warts</I>, <I>henna</I>, and another eye development gene (<I>domeless</I>) were plastic in a female-specific manner. In a male-only analysis, <I>white</I> (<I>w</I>) was differentially expressed between seasonal forms. Reverse transcription polymerase chain reaction confirmed that <I>warts</I> and <I>white</I> are expressed in eyes only, whereas <I>klarsicht, henna</I> and <I>domeless</I> are expressed in both eyes and brain. We find that differential expression of eye development and eye pigment genes is associated with divergent eye phenotypes in <I>B. anynana</I> seasonal forms, and that there is a larger effect of season on female vision-related genes.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/93?rss=1">Adaptive Evolution Favoring KLK4 Downregulation in East Asians</a></div>
  <div class="description"><p>The human <I>kallikrein</I> (<I>KLK</I>) cluster, located at chromosome 19q13.3&ndash;13.4, encodes 15 serine proteases, including neighboring genes (<I>KLK3</I>, <I>KLK2</I>, <I>KLK4</I>, and <I>KLK5</I>) with key roles in the cascades of semen liquefaction, tooth enamel maturation, and skin desquamation. <I>KLK2</I> and <I>KLK3</I> were previously identified as targets of adaptive evolution in primates through different mechanisms linked to reproductive biology and, in humans, genome-wide scans of positive selection captured, a yet unexplored, evidence for <I>KLK</I> neutrality departure in East Asians. We perform a detailed evaluation of <I>KLK3&ndash;KLK5</I> variability in the 1000 Genomes samples from East Asia, Europe, and Africa, which was sustained by our own sequencing. In East Asians, we singled out a 70-kb region surrounding <I>KLK4</I> that combined unusual low levels of diversity, high frequency variants with significant levels of population differentiation (<I>F</I>ST > 0.5) and fairly homogenous haplotypes given the large local recombination rates. Among these variants, rs1654556_G, rs198968_T, and rs17800874_A stand out for their location on putative regulatory regions and predicted functional effects, namely the introduction of several microRNA binding sites and a repressor motif. Our functional assays carried out in different cellular models showed that rs198968_T and rs17800874_A operate synergistically to reduce <I>KLK4</I> expression and could be further assisted by rs1654556_G. Considering the previous findings that <I>KLK4</I> inactivation causes enamel malformations in humans and mice, and that this gene is coexpressed in epidermal layers along with several substrates involved in either cell adhesion or keratinocyte differentiation, we propose <I>KLK4</I> as another target of selection in East Asians correlated to tooth and epidermal morphological traits.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/109?rss=1">Neofunctionalization of a Duplicate dachshund Gene Underlies the Evolution of a Novel Leg Segment in Arachnids</a></div>
  <div class="description"><p>The acquisition of a novel function, or neofunctionalization, protects duplicated genes from redundancy and subsequent loss, and is a major force that drives adaptive evolution. Neofunctionalization has been inferred for many duplicated genes based on differences in regulation between the parental gene and its duplicate. However, only few studies actually link the new function of a duplicated gene to a novel morphological or physiological character of the organism. Here we show that the duplication of <I>dachshund</I> (<I>dac</I>) in arachnids (spiders and allies) is linked with the evolution of a novel leg segment, the patella. We have studied <I>dac</I> genes in two distantly related spider species, the entelegyne spider <I>Parasteatoda tepidariorum</I> and the haplogyne spider <I>Pholcus phalangioides</I>. Both species possess two paralogous <I>dac</I> genes that duplicated before the split between entelegyne and haplogyne spiders. In contrast to the evolutionarily highly conserved <I>dac1</I>, its duplicate <I>dac2</I> is strongly expressed in the patella leg segment during embryogenesis in both species. Using parental RNA interference in <I>P. tepidariorum</I> we show that <I>dac2</I> is required for the development of the patella segment. If <I>dac2</I> function is impaired, then the patella is fused with the tibia into a single leg segment. Thus, removing the function of <I>dac2</I> experimentally reverts <I>P. tepidariorum</I> leg morphology into a stage before the duplication of <I>dac</I> and the evolution of the patella segment. Our results indicate that the origin of the patella is the result of the duplication and subsequent neofunctionalization of <I>dac</I> in the arachnid lineage.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/122?rss=1">Multiple Origins of Eukaryotic cox15 Suggest Horizontal Gene Transfer from Bacteria to Jakobid Mitochondrial DNA</a></div>
  <div class="description"><p>The most gene-rich and bacterial-like mitochondrial genomes known are those of Jakobida (Excavata). Of these, the most extreme example to date is the <I>Andalucia godoyi</I> mitochondrial DNA (mtDNA), including a <I>cox15</I> gene encoding the respiratory enzyme heme A synthase (HAS), which is nuclear-encoded in nearly all other mitochondriate eukaryotes. Thus <I>cox15</I> in eukaryotes appears to be a classic example of mitochondrion-to-nucleus (endosymbiotic) gene transfer, with <I>A. godoyi</I> uniquely retaining the ancestral state. However, our analyses reveal two highly distinct HAS types (encoded by <I>cox15-1</I> and <I>cox15-2</I> genes) and identify <I>A. godoyi</I> mitochondrial <I>cox15</I>-encoded HAS as type-1 and all other eukaryotic <I>cox15</I>-encoded HAS as type-2. Molecular phylogeny places the two HAS types in widely separated clades with eukaryotic type-2 HAS clustering with the bulk of &alpha;-proteobacteria (>670 sequences), whereas <I>A. godoyi</I> type-1 HAS clusters with an eclectic set of bacteria and archaea including two &alpha;-proteobacteria missing from the type-2 clade. This wide phylogenetic separation of the two HAS types is reinforced by unique features of their predicted protein structures. Meanwhile, RNA-sequencing and genomic analyses fail to detect either <I>cox15</I> type in the nuclear genome of any jakobid including <I>A. godoyi</I>. This suggests that not only is <I>cox15-1 </I>a relatively recent acquisition unique to the <I>Andalucia</I> lineage but also the jakobid last common ancestor probably lacked both <I>cox15</I> types. These results indicate that uptake of foreign genes by mtDNA is more taxonomically widespread than previously thought. They also caution against the assumption that all &alpha;-proteobacterial-like features of eukaryotes are ancient remnants of endosymbiosis.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/134?rss=1">Anarchy Is a Molecular Signature of Worker Sterility in the Honey Bee</a></div>
  <div class="description"><p>Worker sterility is a defining characteristic of eusociality. The existence of the sterile worker caste remains a fundamental question for evolutionary biology as it requires the existence of genes that reduce personal reproduction. Currently, little is known about the proximate mechanisms underpinning worker sterility. Studies into a mutant "anarchistic" strain (in which workers can activate their ovaries) of honey bee, <I>Apis mellifera</I>, identified a list of candidate genes that regulate ovary activation. We quantified the expression of the four most promising candidate genes (<I>Anarchy</I>, <I>Pdk1</I>, <I>S6k</I>, and <I>Ulk3</I>) in nonactivated and activated ovaries of wild-type workers. Ovarian expression of <I>Anarchy</I>, a peroxisomal membrane protein, predicts the ovary state of workers with 88.2% accuracy. Increased expression of <I>Anarchy</I> in the ovary is strongly associated with suppression of oogenesis and its expression is sensitive to the presence of the queen. Therefore, <I>Anarchy </I>satisfies key criteria for a "gene underlying altruism". When we knocked down expression of <I>Anarchy</I> in the ovary using RNA interference (RNAi) we altered the expression of <I>Buffy</I>, a gene that regulates programmed cell death. Whole-mount multiplex fluorescent in situ hybridization (mFISH) shows <I>Anarchy </I>transcripts localize to degenerating oocytes within the ovary. Our results suggest that <I>Anarchy </I>is involved in the regulation of oogenesis through programmed cell death. The evolution of facultative worker sterility most likely occurred when the conserved mechanism of programmed cell death was co-opted to regulate ovary activation. <I>Anarchy</I> may therefore be the first example of a gene that has evolved through kin selection to regulate worker sterility.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/143?rss=1">Prolonged Adaptive Evolution of a Defensive Gene in the Solanaceae</a></div>
  <div class="description"><p>Although plants and their natural enemies may coevolve for prolonged periods, little is known about how long individual plant defensive genes are involved in the coevolutionary process. We address this issue by examining patterns of selection on the defensive gene threonine deaminase (<I>TD</I>). Tomato (<I>Solanum lycopersicum</I>) has two copies of this gene. One performs the canonical housekeeping function in amino acid metabolism of catalyzing the first reaction in the conversion of threonine to isoleucine. The second copy functions as an antinutritive defense against lepidopteran herbivores by depleting threonine in the insect gut. Wild tobacco (<I>Nicotiana attenuata</I>) also contains a defensive copy. We show that a single copy of <I>TD</I> underwent two or three duplications near the base of the Solanaceae. One copy retains the housekeeping function, whereas a second copy evolved defensive functions. Positive selection occurred on the branch of the <I>TD2</I> gene tree subtending the common ancestor of the Nicotianoideae and Solanoideae. It also occurred within the Solanoideae clade but not within the Nicotianoideae clade. Finally, it occurred on most branches leading from the common ancestor to <I>S. lycopersicum</I>. Based on recent calibrations of the Solanaceae phylogeny, <I>TD2</I> experienced adaptive substitutions for a period of 30&ndash;50 My. We suggest that the most likely explanation for this result is fluctuating herbivore abundances: When herbivores are rare, relaxed selection increases the likelihood that slightly disadvantageous mutations will be fixed by drift; when herbivores are common, increased selection causes the evolution of compensatory adaptive mutations. Alternative explanations are also discussed.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/152?rss=1">The Roles of Compensatory Evolution and Constraint in Aminoacyl tRNA Synthetase Evolution</a></div>
  <div class="description"><p>Mitochondrial protein translation requires interactions between transfer RNAs encoded by the mitochondrial genome (mt-tRNAs) and mitochondrial aminoacyl tRNA synthetase proteins (mt-aaRS) encoded by the nuclear genome. It has been argued that animal mt-tRNAs have higher deleterious substitution rates relative to their nuclear-encoded counterparts, the cytoplasmic tRNAs (cyt-tRNAs). This dynamic predicts elevated rates of compensatory evolution of mt-aaRS that interact with mt-tRNAs, relative to aaRS that interact with cyt-tRNAs (cyt-aaRS). We find that mt-aaRS do evolve at significantly higher rates (exemplified by higher <I>d</I>N and <I>d</I>N/<I>d</I>S) relative to cyt-aaRS, across mammals, birds, and <I>Drosophila</I>. While this pattern supports a model of compensatory evolution, the level at which a gene is expressed is a more general predictor of protein evolutionary rate. We find that gene expression level explains 10&ndash;56% of the variance in aaRS <I>d</I>N/<I>d</I>S, and that cyt-aaRS are more highly expressed in addition to having lower <I>d</I>N/<I>d</I>S values relative to mt-aaRS, consistent with more highly expressed genes being more evolutionarily constrained. Furthermore, we find no evidence of positive selection acting on either class of aaRS protein, as would be expected under a model of compensatory evolution. Nevertheless, the signature of faster mt-aaRS evolution persists in mammalian, but not bird or <I>Drosophila</I>, lineages after controlling for gene expression, suggesting some additional effect of compensatory evolution for mammalian mt-aaRS. We conclude that gene expression is the strongest factor governing differential amino acid substitution rates in proteins interacting with mitochondrial versus cytoplasmic factors, with important differences in mt-aaRS molecular evolution among taxonomic groups.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/162?rss=1">Parallelism and Epistasis in Skeletal Evolution Identified through Use of Phylogenomic Mapping Strategies</a></div>
  <div class="description"><p>The identification of genetic mechanisms underlying evolutionary change is critical to our understanding of natural diversity, but is presently limited by the lack of genetic and genomic resources for most species. Here, we present a new comparative genomic approach that can be applied to a broad taxonomic sampling of nonmodel species to investigate the genetic basis of evolutionary change. Using our analysis pipeline, we show that duplication and divergence of <I>fgfr1a</I> is correlated with the reduction of scales within fishes of the genus <I>Phoxinellus</I>. As a parallel genetic mechanism is observed in scale-reduction within independent lineages of cypriniforms, our finding exposes significant developmental constraint guiding morphological evolution. In addition, we identified fixed variation in <I>fgf20a</I> within <I>Phoxinellus</I> and demonstrated that combinatorial loss-of-function of <I>fgfr1a</I> and <I>fgf20a</I> within zebrafish phenocopies the evolved scalation pattern. Together, these findings reveal epistatic interactions between <I>fgfr1a</I> and <I>fgf20a</I> as a developmental mechanism regulating skeletal variation among fishes.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/174?rss=1">Sexual Selection of Protamine 1 in Mammals</a></div>
  <div class="description"><p>Protamines have a crucial role in male fertility. They are involved in sperm chromatin packaging and influence the shape of the sperm head and, hence, are important for sperm performance. Protamine structure is basic with numerous arginine-rich DNA-binding domains. Postcopulatory sexual selection is thought to play an important role in protamine sequence evolution and expression. Here, we analyze patterns of evolution and sexual selection (in the form of sperm competition) acting on protamine 1 gene sequence in 237 mammalian species. We assessed common patterns as well as differences between the major mammalian subclasses (Eutheria, Metatheria) and clades. We found that a high arginine content in protamine 1 associates with a lower sperm head width, which may have an impact on sperm swimming velocity. Increase in arginine content in protamine 1 across mammals appears to take place in a way consistent with sexual selection. In metatherians, increase in sequence length correlates with sexual selection. Differences in selective pressures on sequences and codon sites were observed between mammalian clades. Our study revealed a complex evolutionary pattern of protamine 1, with different selective constraints, and effects of sexual selection, between mammalian groups. In contrast, the effect of arginine content on head shape, and the possible involvement of sperm competition, was identified across all mammals.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/185?rss=1">Evolution of DNA-Binding Sites of a Floral Master Regulatory Transcription Factor</a></div>
  <div class="description"><p>Flower development is controlled by the action of key regulatory transcription factors of the MADS-domain family. The function of these factors appears to be highly conserved among species based on mutant phenotypes. However, the conservation of their downstream processes is much less well understood, mostly because the evolutionary turnover and variation of their DNA-binding sites (BSs) among plant species have not yet been experimentally determined. Here, we performed comparative ChIP (chromatin immunoprecipitation)-seq experiments of the MADS-domain transcription factor SEPALLATA3 (SEP3) in two closely related <I>Arabidopsis</I> species: <I>Arabidopsis thaliana</I> and <I>A. lyrata</I> which have very similar floral organ morphology. We found that BS conservation is associated with DNA sequence conservation, the presence of the CArG-box BS motif and on the relative position of the BS to its potential target gene. Differences in genome size and structure can explain that SEP3 BSs in <I>A. lyrata</I> can be located more distantly to their potential target genes than their counterparts in <I>A. thaliana</I>. In <I>A. lyrata</I>, we identified transposition as a mechanism to generate novel SEP3 binding locations in the genome. Comparative gene expression analysis shows that the loss/gain of BSs is associated with a change in gene expression. In summary, this study investigates the evolutionary dynamics of DNA BSs of a floral key-regulatory transcription factor and explores factors affecting this phenomenon.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/201?rss=1">Multiple Genetic Mechanisms Contribute to Visual Sensitivity Variation in the Labridae</a></div>
  <div class="description"><p>Coral reefs are one of the most spectrally diverse environments, both in terms of habitat and animal color. Species identity, sex, and camouflage are drivers of the phenotypic diversity seen in coral reef fishes, but how the phenotypic diversity is reflected in the genotype remains to be answered. The labrids are a large, polyphyletic family of coral reef fishes that display a diverse range of colors, including developmental color morphs and extensive behavioral ecologies. Here, we assess the opsin sequence and expression diversity among labrids from the Great Barrier Reef, Australia. We found that labrids express a diverse palette of visual opsins, with gene duplications in both <I>RH2</I> and <I>LWS</I> genes. The majority of opsins expressed were within the mid-to-long wavelength sensitive classes (<I>RH2</I> and <I>LWS</I>). Three of the labrid species expressed <I>SWS1</I> (ultra-violet sensitive) opsins with the majority expressing the violet-sensitive <I>SWS2B</I> gene and none expressing <I>SWS2A</I>. We used knowledge about spectral tuning sites to calculate approximate spectral sensitivities (max) for individual species&rsquo; visual pigments, which corresponded well with previously published max values for closely related species (SWS1: 356&ndash;370 nm; SWS2B: 421&ndash;451 nm; RH2B: 452&ndash;492 nm; RH2A: 516&ndash;528 nm; LWS1: 554&ndash;555 nm; LWS2: 561&ndash;562 nm). In contrast to the phenotypic diversity displayed via color patterns and feeding ecology, there was little amino acid diversity within the known opsin sequence tuning sites. However, gene duplications and differential expression provide alternative mechanisms for tuning visual pigments, resulting in variable visual sensitivities among labrid species.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/216?rss=1">Recombination Rate Variation Modulates Gene Sequence Evolution Mainly via GC-Biased Gene Conversion, Not Hill-Robertson Interference, in an Avian System</a></div>
  <div class="description"><p>The ratio of nonsynonymous to synonymous substitution rates () is often used to measure the strength of natural selection. However,  may be influenced by linkage among different targets of selection, that is, Hill&ndash;Robertson interference (HRI), which reduces the efficacy of selection. Recombination modulates the extent of HRI but may also affect  by means of GC-biased gene conversion (gBGC), a process leading to a preferential fixation of G:C ("strong," S) over A:T ("weak," W) alleles. As HRI and gBGC can have opposing effects on , it is essential to understand their relative impact to make proper inferences of . We used a model that separately estimated S-to-S, S-to-W, W-to-S, and W-to-W substitution rates in 8,423 avian genes in the <I>Ficedula</I> flycatcher lineage. We found that the W-to-S substitution rate was positively, and the S-to-W rate negatively, correlated with recombination rate, in accordance with gBGC but not predicted by HRI. The W-to-S rate further showed the strongest impact on both <I>d</I>N and <I>d</I>S. However, since the effects were stronger at 4-fold than at 0-fold degenerated sites, likely because the GC content of these sites is farther away from its equilibrium,  slightly decreases with increasing recombination rate, which could falsely be interpreted as a consequence of HRI. We corroborated this hypothesis analytically and demonstrate that under particular conditions,  can decrease with increasing recombination rate. Analyses of the site-frequency spectrum showed that W-to-S mutations were skewed toward high, and S-to-W mutations toward low, frequencies, consistent with a prevalent gBGC-driven fixation bias.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/228?rss=1">Neofunctionalization of Androgen Receptor by Gain-of-Function Mutations in Teleost Fish Lineage</a></div>
  <div class="description"><p>Steroid hormone receptor family provides an example of evolution of diverse transcription factors through whole-genome duplication (WGD). However, little is known about how their functions have been evolved after the duplication. Teleosts present a good model to investigate an accurate evolutionary history of protein function after WGD, because a teleost-specific WGD (TSGD) resulted in a variety of duplicated genes in modern fishes. This study focused on the evolution of androgen receptor (AR) gene, as two distinct paralogs, AR&alpha; and AR&beta;, have evolved in teleost lineage after TSGD. AR&alpha; showed a unique intracellular localization with a higher transactivation response than that of AR&beta;. Using site-directed mutagenesis and computational prediction of protein&ndash;ligand interactions, we identified two key substitutions generating a new functionality of euteleost AR&alpha;. The substitution in the hinge region contributes to the unique intracellular localization of AR&alpha;. The substitution on helices 10/11 in the ligand-binding domain possibly modulates hydrogen bonds that stabilize the receptor&ndash;ligand complex leading to the higher transactivation response of AR&alpha;. These substitutions were conserved in Acanthomorpha (spiny-rayed fish) AR&alpha;s, but not in an earlier branching lineage among teleosts, Japanese eel. Insertion of these substitutions into ARs from Japanese eel recapitulates the evolutionary novelty of euteleost AR&alpha;. These findings together indicate that the substitutions generating a new functionality of teleost AR&alpha; were fixed in teleost genome after the divergence of the Elopomorpha lineage. Our findings provide a molecular explanation for an adaptation process leading to generation of the hyperactive AR subtype after TSGD.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="indexd191.html?q=aggregator&amp;from=340">previous page</a></div><div>... <a href="index4603.html?q=aggregator&amp;from=280">15</a> <a href="index8c8d.html?q=aggregator&amp;from=300">16</a> <a href="index9efb.html?q=aggregator&amp;from=320">17</a> <a href="indexd191.html?q=aggregator&amp;from=340">18</a> <strong>19</strong> <a href="index63c1.html?q=aggregator&amp;from=380">20</a> <a href="index940a.html?q=aggregator&amp;from=400">21</a> <a href="indexd6af.html?q=aggregator&amp;from=420">22</a> <a href="index764e.html?q=aggregator&amp;from=440">23</a> ...</div><div><a href="index63c1.html?q=aggregator&amp;from=380">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museums collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=360 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:36:28 GMT -->
</html>
