<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=480 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:38:04 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="indexacd7.html?q=node/208">56(4) August 2007</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 7 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>December 6, 2015</h3>
<div class="news-item">
 <div class="date">00:31</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134641190006">Graduate position:UWisconsin-Milwaukee.PopulationGenomics</a></div>
  <div class="description"><p>&ndash;_000_D285B96518ECFlatchuwmedu_
Content-Type: text/plain; charset=&ldquo;us-ascii&rdquo;
Content-Transfer-Encoding: quoted-printable

Position announcement: PhD position in genomics of captive populations in the lab of Dr. Emily Latch, Department of Biological Sciences, University of Wisconsin-Milwaukee.

The Latch Lab is seeking a highly motivated and enthusiastic graduate student to join the Ecology and Evolutionary Biology program in the Department of Biological Sciences at the University of Wisconsin-Milwaukee.  The student would participate in a project focused on integrating molecular data into the genetic management of wildlife breeding programs.  Specific aspects of the research will include generating and analyzing genomic (SNP) datasets for captive breeding programs, to be used to develop custom genomic and pedigree-based software tools that can be used by non-experts managing wildlife breeding programs.  This project has wide-reaching applications to in situ and ex situ conservation programs. The student will also assist with organizing and participating in training workshops for non-experts, and disseminating research through presentations and publications.   This project is a collaboration with the San Diego Zoo, and see also coordinating postdoc position at http:
 //life.mcmaster.ca/~brian/evoldir/PostDocs/SanDiegoZooGlobal.CaptivePopulationGenomics
Qualified candidates should have completed a B. S. in Biological Sciences or a related discipline, and be broadly interested in the conservation and management of vertebrate populations.  Qualified applicants will have a strong background in genetic/genomic analysis, bioinformatics, and/or computer modeling.  Proficiency in computer modeling skills and one or more scripting languages is preferred. The applicant should have the ability to collaborate well and communicate scientific materials to non-scientists. Funding in the form of assistantships, research support, and travel grants are available for qualified candidates.
Research in the Latch Lab employs molecular genetic tools and statistical genetic methods to address fundamental questions in vertebrate population genetics and evolutionary ecology.  Many projects have an applied focus, helping to inform conservation and management programs.  For more information about the Latch Lab, visit: <a href="http://bit.ly/1eBnXch.">http://bit.ly/1eBnXch.</a>
To learn more about graduate studies in the Department of Biological Sciences at UWM, visit: <a href="http://bit.ly/1P5JABk.">http://bit.ly/1P5JABk.</a>  The Department has an active research group in Behavioral and Molecular Ecology <a href="http://bit.ly/1Cmc0Cq">http://bit.ly/1Cmc0Cq</a>
If you are interested, please send me an email including a statement of research interests and a CV (including GPA and GRE scores).  I will start reviewing applications immediately, and will continue reviewing applications until the position is filled.  Qualified candidates will also have to apply to the UWM Graduate School (deadline Jan 1, 2016).  Anticipated start date is June or August 2016.




*********************************
Emily K. Latch
Associate Professor
Dept. of Biological Sciences
University of Wisconsin - Milwaukee
3209 N. Maryland Ave.
Milwaukee, WI 53211

Email: latch@uwm.edu
Tel: 414-229-4245
Web: <a href="http://bit.ly/1eBnXch">http://bit.ly/1eBnXch</a>
*********************************

&ndash;_000_D285B96518ECFlatchuwmedu_
Content-Type: text/html; charset=&ldquo;us-ascii&rdquo;
Content-ID: 
Content-Transfer-Encoding: quoted-printable





Position announcement: PhD position in genomics of captive populations in the lab of Dr. Emily Latch, Department of Biological Sciences, University of Wisconsin-Milwaukee.






<p class="MsoNormal">The Latch Lab is seeking a highly motivated and enthusiastic graduate student to join the Ecology and Evolutionary Biology program in the Department of Biological Sciences at the University
 of Wisconsin-Milwaukee.  The student would participate in a project focused on integrating molecular data into the genetic management of wildlife breeding programs.  Specific aspects of the research
 will include generating and analyzing genomic (SNP) datasets for captive breeding programs, to be used to develop custom genomic and pedigree-based software tools that can be used by non-experts managing
 wildlife breeding programs.  This project has wide-reaching applications to <i>in situ</i> and
<i>ex situ</i> conservation programs. The student will also assist with organizing and participating in training workshops for non-experts, and
disseminating research through presentations and publications.   This project is a collaboration with the San Diego Zoo, and see also coordinating postdoc position at <a href="http://bit.ly/1IxbEf0">http://bit.ly/1IxbEf0</a> </p>
<p class="MsoNormal">Qualified candidates should have completed a B. S. in Biological Sciences or a related discipline, and be broadly interested in the conservation and management
 of vertebrate populations.  Qualified applicants will have a strong background in genetic/genomic analysis, bioinformatics, and/or computer modeling.  Proficiency in computer modeling skills and one or more scripting languages is preferred.
The applicant should have the ability to collaborate well and communicate scientific materials to non-scientists.
Funding in the form of assistantships, research support, and travel grants are available for qualified candidates.
</p>
<p class="MsoNormal">Research in the Latch Lab employs molecular genetic tools and statistical genetic methods to address fundamental questions in vertebrate population genetics and
 evolutionary ecology.  Many projects have an applied focus, helping to inform conservation and management programs.  For more information about the Latch Lab, visit:
<a href="http://bit.ly/1eBnXch">http://bit.ly/1eBnXch</a>.</p>
<p class="MsoNormal">To learn more about graduate studies in the Department of Biological Sciences at UWM, visit:
<a href="http://bit.ly/1P5JABk">http://bit.ly/1P5JABk</a>.  The Department has an active research group in Behavioral and Molecular Ecology
<a href="http://bit.ly/1Cmc0Cq">http://bit.ly/1Cmc0Cq</a></p>
If you are interested, please send me an email including a statement of research interests and a CV (including GPA and GRE scores).  I will start reviewing applications immediately, and will continue reviewing applications
 until the position is filled.  Qualified candidates will also have to apply to the UWM Graduate School (deadline Jan 1, 2016).  Anticipated start date is June or August 2016.





*********************************
Emily K. Latch 
Associate Professor 
Dept. of Biological Sciences 
University of Wisconsin - Milwaukee 
3209 N. Maryland Ave. 
Milwaukee, WI 53211 
Email: <a href="mailto:latch@uwm.edu">latch@uwm.edu</a> 
Tel: 414-229-4245
Web: <a href="http://bit.ly/1eBnXch">http://bit.ly/1eBnXch</a>
********************************* 


</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:01</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134639980116">Graduate position: Heidelberg.Phyloinformatics</a></div>
  <div class="description"><p>Dear Community,

One PhD scholarship is available in my group at the
Heidelberg Institute for Theoretical Studies (HITS gGmbH) in Heidelberg, 
Germany

The Scientific Computing group (www.exelixis-lab.org), the home of 
RAxML, at the Heidelberg Institute for Theoretical Studies (HITS gGmbH) 
is soliciting applications for a PhD positions in the broader area of
Phyloinformatics.

HITS gGmbH is a private non-profit research institute carrying out 
multidisciplinary research in the computational sciences. It receives 
its base funding from the HITS Stiftung. PhD students will be inscribed 
at the computer science department of the Karlsruhe Institute of 
Technology (KIT), which is among the top 5 CS departments in Germany.

Applicants shall have a background and Masters degree in bioinformatics 
or computer science and excellent C/C++ programming skills. Expertise in 
the area of high performance computing will be a plus.

Our research mainly focuses on:

-Computational Molecular Phylogenetics
-Large scale evolutionary biology data analyses
-Supercomputing
-Quantifying Biodiversity
-Next and 3 rd Generation Sequence Data Analysis

Secondary research interests include, but are not limited to:

-Emerging parallel architectures (GPU, Xeon PHI)
-Discrete algorithms on trees and sequences
-Population Genetics

The starting date is flexible. To apply, please enter your application 
via the following link: 
(<a href="http://bit.ly/1TNlIlu">http://bit.ly/1TNlIlu</a>)

Applications must be submitted by December 31st, 2015.

Please note that applications not submitted via the online system will 
not be considered.

Please contact Alexandros Stamatakis (Alexandros dot Stamatakis at h-its 
dot org) if you have any further questions.

Also have a look at my supervision strategy:

<a href="http://bit.ly/1NOZCQF">http://bit.ly/1NOZCQF</a><a href="http://www.exelixis-lab.org/">www.exelixis-lab.org</a>), the home of 
RAxML, at the Heidelberg Institute for Theoretical Studies (HITS gGmbH) 
is soliciting applications for a PhD positions in the broader area of
Phyloinformatics.

HITS gGmbH is a private non-profit research institute carrying out 
multidisciplinary research in the computational sciences. It receives 
its base funding from the HITS Stiftung. PhD students will be inscribed 
at the computer science department of the Karlsruhe Institute of 
Technology (KIT), which is among the top 5 CS departments in Germany.

Applicants shall have a background and Masters degree in bioinformatics 
or computer science and excellent C/C++ programming skills. Expertise in 
the area of high performance computing will be a plus.

Our research mainly focuses on:

-Computational Molecular Phylogenetics
-Large scale evolutionary biology data analyses
-Supercomputing
-Quantifying Biodiversity
-Next and 3 rd Generation Sequence Data Analysis

Secondary research interests include, but are not limited to:

-Emerging parallel architectures (GPU, Xeon PHI)
-Discrete algorithms on trees and sequences
-Population Genetics

The starting date is flexible. To apply, please enter your application 
via the following link: 
(http://bit.ly/1TNlIlu)

Applications must be submitted by December 31st, 2015.

Please note that applications not submitted via the online system will 
not be considered.

Please contact Alexandros Stamatakis (Alexandros dot Stamatakis at h-its 
dot org) if you have any further questions.

Also have a look at my supervision strategy:

http://bit.ly/1NOZCQF


Alexis

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 5, 2015</h3>
<div class="news-item">
 <div class="date">23:16</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134637977606">Graduate position: ImperialCollege.EcolEvolution</a></div>
  <div class="description">PhD Studentship advertised at Imperial College London, Silwood Park
campus, UK

On: Trans-generational fitness consequences of chronic stress induced
by global change

Supervisors: Dr Julia Schroeder (Imperial College London) and Prof Terry
Burke (University of Sheffield)

Application deadline: 10 January 2016.

Project description: Human-induced global changes impose chronic
environmental stress on ever more natural populations. The physiological
symptoms of chronic stress are changes in hormone levels that can
lead to stunted growth and increased risk of infection. Furthermore,
the physiological mechanisms invoked by stressed are similar to those
in ageing individuals. Recent studies suggest that chronic stress can
lead to faster senescence (measured in telomere shortening rates), not
only in affected individuals but also in their offspring. What are the
trans-generational fitness consequences of a chronic stress response,
and how is it adaptive? Stress could $B!H(Bprime$B!I(B offspring to enable
them to endure stressful conditions in the future, by a $B!H(Blife-fast
die-young$B!I(B trade-off. This projects tests the hypothesis that chronic
stress primes the next generation. To test this, we make opportunistic
use of an exceptionally well suited long-term $B!H(Bnatural experiment$B!I(B in
wild passerines and supplement statistical analyses on existing long-term
data with focused experiments on captive birds.

The aim of this study is to quantify the fitness consequences of chronic
stress to gain a better understanding of the long-term fitness costs
and benefits of chronic stress in a wild population the consequences for
population viability. The student will utilise a unique long-term study
of a wild passerine population. The precise direction and emphasis of the
project will be determined by collaboration between the student and the
supervisors.  Next to the traditional academic skills, the student will
also learn and improve key skills such as state-of-the-art statistical
modelling, data management, experimental design and field work. The
student can expect to receive exceptional training in state-of-the-art
analytical methodology, as well as benefitting from a well established
and rewarding field project.

Imperial College London is a world leading university; it has been
Ranked 3rd in Europe and 8th in the world by the Time Higher Education
Rankings 2015-16. The student will be based at the Silwood Park campus
 which has a large
graduate student community, with internationally renowned researchers.

The funding for this post is covered by an SSCP DTP-NERC studentship,
and covers home fees and a stipend for 3.5 years. Applicants,
who must be residents of the EU OR have been $B!F(Bordinarily
resident$B!G(B in the UK for 3 years prior to the start of the
studentship, must have an outstanding honours degree, a masters
degree, or equivalent, in a relevant subject, should complete an
online application, upload a supporting statement explaining why they
want to pursue this PhD (maximum 1 page), their CV, and 2 references, at:
<a href="http://bit.ly/1TtfTbX">http://bit.ly/1TtfTbX</a>
Any enquiries about the project should be directed to:

Dr Julia Schroeder julia.schroeder@imperial.ac.uk


Julia Schroeder

Imperial College London, Silwood Park Campus
Buckhurst Road, SL5 7PY Ascot, Berks, UK
julia.schroeder@imperial.ac.uk
+44 (0) 20 7594 9086
<a href="http://bit.ly/1Q81ggy">http://bit.ly/1Q81ggy</a>

and:

Group Leader Evolutionary Biology
Max Planck Institute for Ornithology
Seewiesen, Germany
jschroeder@orn.mpg.de
+44 (0) 74 9254 9908


julia.schroeder@imperial.ac.uk


via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:40</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134580280781">Graduate position: PennsylvaniaState.EvolutionaryGenomics</a></div>
  <div class="description"><p>The Assis lab (<a href="http://bit.ly/1taDu27">http://bit.ly/1taDu27</a>) at Penn State
is recruiting a highly motivated Ph.D. student to work on a NSF-funded
project examining the role of natural selection in the evolution of novel
duplicate gene functions.

The project involves developing and applying computational and statistical
approaches for examining the types and strengths of natural selection
acting on duplicate gene functions. In particular, the project will
incorporate sequence-based analyses and mathematical modeling of gene
expression evolution. The major objective of this project will be to
elucidate the role of natural selection in the functional evolution of
duplicate genes, and to study differences in selective forces acting on
duplicates in populations of different sizes, over evolutionary time, and
with varying levels of functional divergence.

Note that our lab is solely computational. While prior knowledge of a
programming language is not necessary, candidates should have strong
quantitative skills and the drive to learn how to program.

If you are interested in joining the lab, please email Raquel Assis (
rassis@psu.edu) a current CV and description of your research interests.

In addition, candidates must submit a formal application to one of the
following three Ph.D. programs: Biology (
<a href="http://bit.ly/1taDr6v">http://bit.ly/1taDr6v</a>), Bioinformatics and
Genomics (
<a href="http://bit.ly/1AfN2CU">http://bit.ly/1AfN2CU</a>), or
Molecular, Cellular, and Integrative Biosciences (
<a href="http://bit.ly/1taDu2e">http://bit.ly/1taDu2e</a>).
Application deadlines for Fall 2016 admission to these programs are in
December and January.

CAMPUS SECURITY CRIME STATISTICS: For more about safety at Penn State, and
to review the Annual Security Report which contains information about crime
statistics and other safety and security matters, please go to
<a href="http://bit.ly/1O9db7y">http://bit.ly/1O9db7y</a> , which will also provide you with detail on
how to request a hard copy of the Annual Security Report.

Penn State is an equal opportunity, affirmative action employer, and is
committed to providing employment opportunities to minorities, women,
veterans, disabled individuals, and other protected groups.

rassis7@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:40</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134580280276">Conference: Seville.EvolEcol.Feb2017</a></div>
  <div class="description">To Evoldir members:

Below you will find the annoucement of a Meeting we are organizing at
Seville (Spain) in Feb. 2017 for those interested in the interface
between Ecology and Evolution

<a href="http://bit.ly/1O9daRe">http://bit.ly/1O9daRe</a>
ting_363_p.htm

Dear colleagues,

The International Society for Mediterranean Ecology (ISOMED) together
with The Spanish Society for Terrestrial Ecology (AEET), invites you to
attend the forthcoming MEDECOS XIV International Conference - XIII AEET
Meeting, that will be hold in Sevilla, Spain, from 31st January to
4th February 2017. This meeting is organized by the AEET andISOMED and
hosted by the University of Sevilla and the Doñana Biological Station.

More information about the Congress will be soon available
at <a href="http://www.aeet.org/">http://www.aeet.org</a> and <a href="http://bit.ly/1O9daRg">http://bit.ly/1O9daRg</a>
.html

For questions and suggestions please send an email to info@aeet.org

SAVE THE DATE. THE ORGANIZING COMMITEE LOOKS FORWARD TO YOUR ATTENDANCE
TO MEDECOS XIV -AEET XIII IN SEVILLA!!

Thank you very much for disseminate this announcement

Best Regards,

The Organizing Committee of MEDECOS XIV International Conference - XIII
AEET Meeting

Dr. Juan Arroyo
arroyo@us.es
Departamento de Biologia Vegetal y Ecologia. Dept. of Plant Biology and
Ecology
Universidad de Sevilla
Apartado 1095 -E41080 Sevilla, Spain
phone:
desk office: +34 954 557 058
lab: +34 955 420 845
fax: +34 954 557 059
Research group:
<a href="http://bit.ly/1NNnnZc">http://bit.ly/1NNnnZc</a>
<a href="http://bit.ly/1O9daRi">http://bit.ly/1O9daRi</a>

Juan Arroyo 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:10</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134579421446">Postdoc: SanDiegoZooGlobal.CaptivePopulationGenomics</a></div>
  <div class="description"><p>Post-doctoral Associate for San Diego Zoo Global

San Diego Zoo Global invites applications for a 3-year post-doctoral
research associate to participate in a project focused on integrating
molecular data into the genetic management of wildlife breeding
programs. Specific aspects of the research will include analyzing
genomic (SNP) datasets for captive breeding programs, modeling alternate
approaches for integrating genomic data into pedigree-based breeding
pair selection strategies, and developing custom genomic and
pedigree-based software tools that can be used by non-experts managing
wildlife breeding programs. This project has wide-reaching applications
to in situ and ex situ conservation programs. The research associate
will also be responsible for organizing and participating in training
workshops for non-experts, administering and analyzing evaluation
surveys at the beginning and end of the project period, and
disseminating research through presentations and publications. This
project is a collaboration with the Latch Lab at the University of
Wisconsin Milwaukee (<a href="http://bit.ly/1eBnXch">http://bit.ly/1eBnXch</a>). 

Qualified applicants should have completed a Ph.D. in Bioinformatics,
Computer Science, Statistics, Biology, Genetics, or a related field and
have a strong background in genomic (SNP) analysis and/or computer
modeling. Proficiency in computer modeling skills and one or more
scripting languages will be necessary for testing alternate approaches
for integrating genomic data into pedigree-based breeding pair selection
strategies and for developing custom software tools. Experience with a
C-based programming language and/or python are preferred. The applicant
should have the ability to collaborate well and communicate scientific
materials to non-scientists. 

Opportunities exist for the position to be based at either the San
Diego Zoo (San Diego, CA) or San Diego Zoo Safari Park (Escondido, CA).
The application deadline is December 23rd. For more information or to
apply go to: <a href="http://bit.ly/1PHtPkM">http://bit.ly/1PHtPkM</a> 



Jamie A. Ivy, Ph.D.
Population Biologist
San Diego Zoo Global
p. 619-557-3905 (Zoo)
p. 760-738-5056 (Safari Park)
p. 760-747-8702 x5709 (Institute)
P.O. Box 120551
San Diego, CA 92112-0551

JIvy@sandiegozoo.org


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:55</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134578982961">Course: FridayHarborLabs.Biodiversity.Jun13-Jul15</a></div>
  <div class="description"><p>Friday Harbor Laboratories Marine Invertebrate Zoology
June 13 - July 15, 2016

This course will be taught by Dr. Megan Schwartz (University of 
Washington, Tacoma) and Dr. Kevin Kocot (University of Alabama and 
Alabama Museum of Natural History). The course takes advantage of the 
rich marine biota of the Friday Harbor region to teach the principles of 
animal organization and biodiversity. It emphasizes comparative study of 
form and function, and of complexity and diversity in phylogenetic and 
environmental contexts. It focuses on the study of living animals in the 
laboratory and field in the diverse marine habitats surrounding San Juan 
Island.

The course reviews the diversity of animal life in an evolutionary and 
ecological context, focusing on the comparative study of form, function 
and life history. We will review all animal phyla, and also explore 
diversity within phyla based on available exemplars.

Biodiversity is one of the most topical subjects in biology, partly 
because of its accelerating erosion as a result of increasing human 
pressures and global change. Having a working knowledge of the diversity 
of life is fundamental to the study of any subject in biology. FHL is 
arguably the best location in the US for such a course, given the wealth 
of local diversity and accumulated knowledge of the local fauna built 
over a century of investigations.

Applications are welcome from advanced undergraduate students, 
post-baccalaureates, and graduate students. Prior coursework in 
invertebrate biology or animal diversity will be useful; if in doubt, 
please contact one of the instructors.

Enrollment is limited to 20 students. No textbook is required for this 
course.

For more details (including costs and application instructions) please 
see the course website:
<a href="http://bit.ly/21E5Jvg">http://bit.ly/21E5Jvg</a>

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:55</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134578982491">Other: RequestComments.EvoinEvoDevo?</a></div>
  <div class="description">.. might be of interest, particularly because aim of this paper
is to stimulate discussions on evo-devo and its possible future
directions, and we are thinking about expanding the subject in a
book, so any comments/thoughts are welcome (send them directly to me,
to avoid multiple messages circulating).. here&rsquo;s the paper and pdf:
<a href="http://bit.ly/21E5Jv8">http://bit.ly/21E5Jv8</a>


Thanks,
Rui Diogo 		 	   		   		 	   		  

Rui Diogo 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:55</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134578982741">Graduate position: UWesternAustralia.ClimateAdaptation</a></div>
  <div class="description">We have a fully funded PhD position available to investigate
the effects of sociality on physiological responses to high
temperatures. Please see following link for full details:
<a href="http://bit.ly/21E5Jva">http://bit.ly/21E5Jva</a>


Mandy Ridley 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:25</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134576052191">Postdoc: UC_SanFrancisco.MalariaPopGenetics.2</a></div>
  <div class="description">Postdoc Position available in Malaria Population Genetics at UC San
                               Francisco

A postdoctoral position is available in the laboratory of Bryan
Greenhouse, MD, MA in the Division of Infectious Diseases at U.C. San
Francisco. The current position will focus on analysis of parasite
genetics and epidemiologic data to characterize transmission and
evolution of malaria parasites. The successful candidate will have the
opportunity to work closely with a diverse team of scientists at UCSF
and international collaborators on a number of exciting projects. This
research includes work in malaria elimination regions, where densely
sampled genetic data will be used to construct transmission networks,
to studies in areas with higher malaria burden leveraging broader scale
spatial variation in genetic diversity. The fellow will be expected to
develop and extend computational methods for deriving epidemiologically
relevant information on malaria transmission from genetic, spatial, and
epidemiologic data, and will be encouraged to develop an independent
line of work under the mentorship of Bryan Greenhouse and co-mentorship
of Rasmus Nielsen (U.C. Berkeley). Our work is directly connected to
on-the-ground malaria control and elimination efforts, and the
scientific output from this position is expected to directly affect
interventions in the field in addition to leading to academic
publications. Competitive salary including full benefits will be
provided commensurate with experience and qualifications.

Essential Skills

- PhD in a relevant field (e.g. population genetics, computational
  biology, infectious disease dynamics)

- Strong background in population genetics

- Strong statistical and computational skills

- Demonstrated ability to produce independent, creative work

- Ability to work well as member of a team

- Strong written and oral communication skills

Helpful Skills

- Experience with geostatistical data analysis

- Experience in analysis/modelling of pathogen dynamics

- Experience with implementing Markov Chain Monte Carlo (MCMC)

- Familiarity with performance and development laboratory
  genetic/genomic assays and data

To Apply: Please send a CV including publications, brief statement of
research/career interests, and contact information for 3 references to
Bryan Greenhouse, MD, MA at bryan.greenhouse@ucsf.edu

&ldquo;Greenhouse, Bryan&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 4, 2015</h3>
<div class="news-item">
 <div class="date">23:25</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134573684806">postdoc: UniversityofTrento.ComparativeAvian_Cell_Recording</a></div>
  <div class="description">&ndash;Apple-Mail=_0139376C-BB9C-48BF-9C18-8C03DA386554
Content-Transfer-Encoding: quoted-printable
Content-Type: text/plain;
	charset=us-ascii

> Postdoc position on comparative single cell recording in the avian brain
> 
> A Postdoc position on single cell recording in the avian brain is available at the Center for Mind/Brain Sciences (University of Trento, Italy) from December 2015.
> The project focuses on the neural mechanisms underlying numerical capacities and predisposition to detect animacy using the developing domestic chick forebrain as a model. A PhD in neuroscience or related disciplines is required. Experience with single cell recording (not necessarily on the avian brain) is needed. 
> The laboratory is fully equipped and offers an exciting international and interdisciplinary work environment (<a href="http://bit.ly/1KKBj1q">http://bit.ly/1KKBj1q</a> ).
> If interested in this job, please send a motivation letter, CV, publications, and e-mail addresses of two academic referees to prof. Giorgio Vallortigara email: giorgio.vallortigara@unitn.it  
> For any question, do not hesitate to contact us: giorgio.vallortigara@unitn.it  
> 
> 
> 
>
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">11:10</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/troubleshooting-rpanda-code/595">Troubleshooting RPANDA code</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/anurag_mishra">@Anurag_Mishra</a> wrote:</p>
            
              <p>I'm trying to test maximum likelihood fit of environmental birth death model specified in Helen Morlone's paper. </p>

<p>Using the code given in the manual: </p>

data(a)
tot_time</div>
  <div class="source">Source: <a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/271">Phylogenomic methods outperform traditional multi-locus approaches in resolving deep evolutionary history: a case study of formicine ants</a></div>
  <div class="description">Background:
Ultraconserved elements (UCEs) have been successfully used in phylogenomics for a variety of taxa, but their power in phylogenetic inference has yet to be extensively compared with that of traditional Sanger sequencing data sets. Moreover, UCE data on invertebrates, including insects, are sparse. We compared the phylogenetic informativeness of 959 UCE loci with a multi-locus data set of ten nuclear markers obtained via Sanger sequencing, testing the ability of these two types of data to resolve and date the evolutionary history of the second most species-rich subfamily of ants in the world, the Formicinae.
Results:
Phylogenetic analyses show that UCEs are superior in resolving ancient and shallow relationships in formicine ants, demonstrated by increased node support and a more resolved phylogeny. Phylogenetic informativeness metrics indicate a twofold improvement relative to the 10-gene data matrix generated from the identical set of taxa. We were able to significantly improve formicine classification based on our comprehensive UCE phylogeny. Our divergence age estimations, using both UCE and Sanger data, indicate that crown-group Formicinae are older (104–117 Ma) than previously suggested. Biogeographic analyses infer that the diversification of the subfamily has occurred on all continents with no particular hub of cladogenesis.
Conclusions:
We found UCEs to be far superior to the multi-locus data set in estimating formicine relationships. The early history of the clade remains uncertain due to ancient rapid divergence events that are unresolvable even with our genomic-scale data, although this might be largely an effect of several problematic taxa subtended by long branches. Our comparison of divergence ages from both Sanger and UCE data demonstrates the effectiveness of UCEs for dating analyses. This comparative study highlights both the promise and limitations of UCEs for insect phylogenomics, and will prove useful to the growing number of evolutionary biologists considering the transition from Sanger to next-generation sequencing approaches.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>December 3, 2015</h3>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/262">A voyage to Terra Australis: human-mediated dispersal of cats</a></div>
  <div class="description">Background:
Cats have been transported as human commensals worldwide giving rise to many feral populations. In Australia, feral cats have caused decline and extinction of native mammals, but their time of introduction and origin is unclear. Here, we investigate hypotheses of cat arrival pre- or post-European settlement, and the potential for admixture between cats of different invasion events. We analyse the genetic structure and diversity of feral cats from six locations on mainland Australia, seven Australian islands and samples from Southeast Asia and Europe using microsatellite and mitochondrial DNA data.
Results:
Our results based on phylogeographic model selection are consistent with a European origin of cats in Australia. We find genetic distinctiveness of Australian mainland samples compared with Dirk Hartog Island, Flinders Island, Tasman Island and Cocos (Keeling) Island samples, and genetic similarities between some of the island populations. Historical records suggest that introduction of cats to these islands occurred at the time of European exploration and/or in connection with the pearling, whaling and sealing trades early in the 19th century. On-going influx of domestic cats into the feral cat population is apparently causing the Australian mainland populations to be genetically differentiated from those island populations, which likely are remnants of the historically introduced cat genotypes.
Conclusion:
A mainly European origin of feral cats in Australia, with possible secondary introductions from Asia following the initial establishment of cats in Australia is reasonable. The islands surrounding Australia may represent founding populations and are of particular interest. The results of the study provide an important timeframe for the impact of feral cats on native species in Australia.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/270">Population structure and genetic diversity in the nannandrous moss Homalothecium lutescens: does the dwarf male system facilitate gene flow?</a></div>
  <div class="description">Background:
Nannandry is a sexual system where males (”dwarf males”) are much smaller than the conspecific females. Dwarf males occur in a wide range of unrelated organisms but the evolutionary advantages of this condition are poorly understood. The dwarf male sexual system results in differences in the mode of dispersal and establishment as well as the life span between males and females. Such differences must have profound effects on the population dynamics and genetic structures. We have studied four populations of the nannandrous moss Homalothecium lutescens in southern Sweden. We genotyped dwarf males and female shoots with the aim of describing the genetic diversity and structure of the populations.
Results:
Dwarf males were most related to their host shoot, then their colony (within 0.5 m
                      2
                   ) and then the rest of the population, which suggests restricted spore dispersal. However, a few dwarf males in each population appeared to originate from other colonies and sometimes even other populations. Genetic diversity of dwarf males was generally high but showed no tendency to be consistently higher or lower than female genetic diversity within the four populations.
Conclusions:
Although most dwarf males have local origin, sporadic dispersal events occur. The ability of the dwarf males to establish in high numbers in mature colonies facilitates gene flow between populations as well as increases the potential to accumulate genetic diversity within populations.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/269">Erratum to: Molecular phylogeny of the subfamily Stevardiinae Gill, 1858 (Characiformes: Characidae): classification and the evolution of reproductive traits</a></div>
  <div class="description">No description available</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:02</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134452201866">Graduate Position: CUNY comparative population genetics</a></div>
  <div class="description"><p>&ndash;_000_144906756677728254ccnycunyedu_
Content-Type: text/plain; charset=&ldquo;iso-8859-1&rdquo;
Content-Transfer-Encoding: quoted-printable


The Hickerson lab at the City University of New York (CUNY) has an opening for a PhD student who is interested in developing models that link biogeographic history, community ecology and comparative population genomics. Our group is focusing on developing and implementing methods for understanding the evolutionary and demographic histories of species assemblages and ecosystems given environmental, spatial, and aggregate population genomic data. The statistical models we develop are broadly applicable to a variety of regions and members of our group have worked with a diverse set of taxa and ecosystems.

?The ideal candidate will have a strong interest and aptitude in quantitative biology, modeling, and programming as well as an interest in evolutionary genetics and biogeography. The lab welcomes qualified applicants with diverse backgrounds, including biology, anthropology, mathematics, physics, computer science, and related fields.

To offset teaching requirements, the PhD candidate will be supported by two five-year grants funded by NSF and NASA (1. DEB-1253710 - CAREER: Dynamic models of isolation and admixture for community-scale population genomic inference & 2. DEB-1343578 - Dimensions US-BIOTA-Sao Paulo: A multidisciplinary framework for biodiversity prediction in the Brazilian Atlantic forest hotspot -  <a href="http://bit.ly/1NrJZfK?">http://bit.ly/1NrJZfK?</a>).

Our lab is located in Manhattan and locally we have tight collaborations with the lab groups of Ana Carnaval (CUNY), Kyle McDonald (CUNY), Frank Burbrink (AMNH), and Brian Smith (AMNH), as well as international collaborations with Konrad Lohse (U. of Edinburgh, UK), Graham Stone (U. of Edinburgh, UK) and Brent Emerson (Canary Islands, Spain). CUNY has a large and thriving community of faculty, students, and post-docs studying ecology, evolution, and behavior and we benefit from the academic environment in New York City that allows us to have close ties to the AMNH, the New York Botanical Gardens as well as other local universities, including Columbia, Fordham, Rockefeller, NYU and Stony Brook.

The positions would start in the Fall of 2016. Contact mhickerson &lsquo;at&rsquo; ccny cuny.edu if there is interest. Note that applications for Fall 2016 to the CUNY EEB subprogram must be received before January 1rst of 2016.

For more information visit: <a href="http://bit.ly/1a2oJFK">http://bit.ly/1a2oJFK</a>



Michael J Hickerson
Associate Professor
City College of New York - Biology Department;
160 Convent Ave
New York, NY 10031
lab 212-650-3457

Chair, Ecology Evolution, and Behavior subprograms (EEB)
City University of New York and the Graduate Center
365 Fifth Avenue
New York, NY 10016-4309

Research Associate - Division of Invertebrate Zoology
American Museum of Natural History
<a href="http://bit.ly/1a2oJFK">http://bit.ly/1a2oJFK</a>


&ndash;_000_144906756677728254ccnycunyedu_
Content-Type: text/html; charset=&ldquo;iso-8859-1&rdquo;
Content-Transfer-Encoding: quoted-printable



<p></p>

The Hickerson lab at the City University of New York (CUNY) has an opening for a PhD student who is interested in developing models that link biogeographic history, community ecology and comparative population genomics. Our group is focusing on developing
 and implementing methods for understanding the evolutionary and demographic histories of species assemblages and ecosystems given environmental, spatial, and aggregate population genomic data. The statistical models we develop are broadly applicable to a variety
 of regions and members of our group have worked with a diverse set of taxa and ecosystems.           

​The ideal candidate will have a strong interest and aptitude in quantitative biology, modeling, and programming as well as an interest in evolutionary genetics
 and biogeography. The lab welcomes qualified applicants with diverse backgrounds, including biology,
anthropology, mathematics, physics, computer science, and related fields. 


To offset teaching requirements, the PhD candidate will be supported by two five-year grants funded by NSF and NASA (1. DEB-1253710 - CAREER: Dynamic models of isolation and admixture for community-scale population genomic inference & 2. DEB-1343578 -
<a href="http://bit.ly/1NrJZfK">Dimensions US-BIOTA-Sao Paulo: A multidisciplinary framework for biodiversity prediction in the Brazilian Atlantic forest hotspot - </a><a href="http://bit.ly/1NrJZfK">http://bit.ly/1NrJZfK</a>​). 

Our lab is located in Manhattan and locally we have tight collaborations with the lab groups of Ana Carnaval (CUNY), Kyle McDonald (CUNY), Frank Burbrink (AMNH), and Brian Smith (AMNH), as well as international collaborations with Konrad Lohse (U. of Edinburgh,
 UK), Graham Stone (U. of Edinburgh, UK) and Brent Emerson (Canary Islands, Spain). CUNY has a large and thriving community of faculty, students, and post-docs studying ecology, evolution, and behavior and we benefit from the academic environment in New York
 City that allows us to have close ties to the AMNH, the New York Botanical Gardens as well as other local universities, including Columbia, Fordham, Rockefeller, NYU and Stony Brook.

The positions would start in the Fall of 2016. Contact mhickerson 'at&rsquo; ccny cuny.edu if there is interest. Note that applications for Fall 2016 to the CUNY EEB subprogram must be received before January 1rst of 2016. 

For more information visit: <a href="http://bit.ly/1a2oJFK">http://bit.ly/1a2oJFK</a>
<p></p>
<p></p>


Michael J Hickerson
Associate Professor
City College of New York - Biology Department; 160 Convent Ave
New York, NY 10031
lab 212-650-3457

Chair, Ecology Evolution, and Behavior subprograms (EEB)
City University of New York and the Graduate Center
365 Fifth Avenue 
New York, NY 10016-4309
Research Associate - Division of Invertebrate Zoology 
American Museum of Natural History
<a href="http://bit.ly/1a2oJFK">http://bit.ly/1a2oJFK</a>






&ndash;_000_144906756677728254ccnycunyedu
via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:32</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134451207476">Other: AmericanGeneticAssoc.Awards</a></div>
  <div class="description"><p>Application Deadline 31 January 2016

Awards announced 31 March 2016

The American Genetic Association grants awards each year for support of
special events that advance the mission of AGA, particularly to support
students to attend the event.

Eligible events include specialized workshops and short courses in topical
areas of organismal genetics and genomics, but any event relevant to
AGAs purpose will be considered.  Events that could lead to Journal
of Heredity articles will receive special consideration.  Awards are
usually between ,000-5,000, with a total of 0,000 available for 2016.

Funding is competitive, and applications must follow the
guidelines. Details and instructions for preparing applications can be
found at <a href="http://bit.ly/15HcX0C">http://bit.ly/15HcX0C</a>

&ndash;
via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:00</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134450026266">Course: European Comparative Genomics - 11-22 January 2016 (Lyon,</a></div>
  <div class="description"><p>Dear colleagues,

Registrations for the 2015-2016 edition of the European course on 
&ldquo;Comparative Genomics&rdquo; are open.

The course is aimed at students from the ENS and is open to master and 
PhD students from European universities.

The course focuses on major discoveries, big challenges, innovative 
concepts and original approaches in the field of comparative genomics, 
their applications in biology, medicine and biotechnology, and their 
impact on society.

This year the course will be held on 11-22 January 2016 at the ENS 
(Lyon, France).

Registration is free. Lectures are in English.

The preliminary program and registration form are available at:

<a href="http://bit.ly/ICfmGK">http://bit.ly/ICfmGK</a>

Jean-Nicolas Volff (ENS, Lyon) and Céline Brochier (Univ. Lyon1)

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:00</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134450026726">Postdoc: Texas_BiomedicalResInst.MalariaEvolution</a></div>
  <div class="description"><p>Searching for an enthusiastic postdoctoral researcher to work on the
biology of the malaria parasite, Plasmodium falciparum. Plasmodium
parasites are of huge medical importance, causing ~700,000 deaths per
year and millions of infections. P. falciparum parasites are a highly
tractable organism which can be reared in the laboratory and placed under
strong selection. This project will use a combination of flow cytometry,
genome/transcriptome sequencing and experimental evolution to identify
the genetic underpinnings of disease related traits of the parasite,
including drug resistance, host-cell invasion and growth rate. A
large component of this project will be based at new flow sorting
facilities at Texas Biomed. The postdoctoral scientist will report to
Dr. Ian Cheeseman. The successful applicant will join a new lab in a
vigorous infectious disease program at Texas Biomed (including work on
malaria, schistosomiasis, ebola, HIV, and hepatitis C). The Genetics
department at Texas Biomed provides a stimulating environment with work
on statistical and quantitative genetics, metabolomics, and proteomics
and has outstanding computational, sequencing and wet lab facilities. San
Antonio is a vibrant, rapidly growing city with an interactive research
community.

RELEVANT PUBLICATIONS

Nair S, Nkhoma SC, Serre D, Zimmerman PA, Gorena K, Daniel BJ, Nosten F,
Anderson TJ, Cheeseman IH. Single-cell genomics for dissection of complex
malaria infections. Genome Res 2014; Jun; 24(6):1028-38. PMCID: PMC4032849

Nkhoma SC, Nair S, Cheeseman IH, Rohr-Allegrini C, Singlam S, Nosten F,
Anderson TJ: Close kinship within multiple-genotype malaria parasite
infections. Proc Biol Sci 2012; 279(1738):2589-2598. PMCID: PMC3350702.

Cheeseman IH, Miller BA, Nair S, Nkhoma S, Tan A, Tan JC, Al Saai S,
Phyo AP, Moo CL, Lwin KM, McGready R, Ashley E, Imwong M, Stepniewska K,
Yi P, Dondorp AM, Mayxay M, Newton PN, White NJ, Nosten F, Ferdig MT,
Anderson TJ. A major genome region underlying artemisinin resistance in
malaria. Science 2012; Apr 6; 336 (6077):79-82. PMCID: 22491853

Education
REQUIRED: Ph.D. in genetics, parasitology, molecular biology or a
related discipline.

Experience/Skills
Required: Substantial experience in flow cytometry and cell culture work,
need not be limited to work on malaria parasites.

Potential Hazards
This position may involve exposure to cryogenics and infectious materials
(BSL-2). Participation in a medical monitoring and surveillance program
is required. Safety training and protective clothing, equipment and
supplies will be provided.

Other
This is a full-time salaried (exempt) position. Texas Biomedical
Research Institute business hours are Monday through Friday - 8:00
a.m. to 5:00 p.m. Application packets are accepted electronically or
in hard copy. A completed application packet is a requirement for all
positions. Incomplete applications will not be accepted. EOE

Please apply online at
<a href="http://bit.ly/21x7wCe">http://bit.ly/21x7wCe</a>


job ID: 15-079 Postdoctoral Scientist Genetics


ianc@txbiomed.org


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="index3d2c.html?q=aggregator&amp;from=460">previous page</a></div><div>... <a href="index940a.html?q=aggregator&amp;from=400">21</a> <a href="indexd6af.html?q=aggregator&amp;from=420">22</a> <a href="index764e.html?q=aggregator&amp;from=440">23</a> <a href="index3d2c.html?q=aggregator&amp;from=460">24</a> <strong>25</strong> <a href="index8c9a.html?q=aggregator&amp;from=500">26</a> <a href="indexbe68.html?q=aggregator&amp;from=520">27</a> <a href="indexf601-2.html?q=aggregator&amp;from=540">28</a> <a href="index25b4.html?q=aggregator&amp;from=560">29</a> ...</div><div><a href="index8c9a.html?q=aggregator&amp;from=500">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=480 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:38:04 GMT -->
</html>
