<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=280 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:26:26 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="index66e2.html?q=node/119">54(6) December 2005</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 2 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>January 19, 2016</h3>
<div class="news-item">
 <div class="date">05:01</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137615809566">Graduate position: UEdinburgh.PlantBiodiversity</a></div>
  <div class="description">MSc/Dipl. Course in the Biodiversity and Taxonomy of Plants

A one year course organized by the Royal Botanic Garden Edinburgh and
the University of Edinburgh

Running since 1992, this highly acclaimed course benefits from a close
connection between the Royal Botanic Garden Edinburgh, one of the leading
institutions for the study and conservation of plants, and the University
of Edinburgh, one of the top universities in the world.


 *   Students are based at RBGE, with major collections of living and
     preserved plants, an extensive library, and modern research
     facilities;
 *   World experts deliver and design the course and students benefit
     from a supportive and inspiring environment;
 *   Edinburgh is one of four Scottish gardens run by RBGE, encompassing
     many habitats and an incredible plant collection;
 *   In addition, Edinburgh is a great city to be a student with a
     thrilling cultural and social life.

Taught part of the course
The course covers following modules:

 *   Evolution and biodiversity of angiosperms
 *   Evolution and biodiversity of cryptogamic plants and fungi
 *   Phylogenetics and Population genetics
 *   Conservation science, statistical analysis and fieldwork techniques
 *   Herbarium taxonomy, collections, and conservation
 *   A tropical field course in Central or South America

Research project
The course includes a three-month scientific research project enabling
the student to become part of a research team, benefiting from extensive
experience.  The course places great emphasis on fundamental knowledge
(identifying plants and their evolution), as well as acquiring practical
field- and herbarium techniques, giving a strong basis for botanists,
ecologists and other natural scientists embarking on a career in
biodiversity. There are excellent prospects for graduates in academia,
industry, and conservation.

The course aims to be more than just a degree V it unlocks a better
understanding of the world of plants and their importance to humanitys
future. This aspiration becomes increasingly important in our present
world, where demographic pressures push the natural world to its limits
despite huge areas remaining unexplored.  The course can be spread over
two years on request.  To find out more and download the course handbook
go to: <a href="http://bit.ly/1nhYfOW">http://bit.ly/1nhYfOW</a>

Contact
For more information contact: Vicky Mactaggart (Programme Administrator)
-Phone: +44 (0)131 651 7052 - Email: btpmsc@ed.ac.uk
or the Course Directors: Dr. Louis Ronse De Craene
(l.ronsedecraene@rbge.ac.uk) and Prof. Andrew Hudson
(andrew.hudson@ed.ac.uk)

To apply: go to
<a href="http://bit.ly/1Qb8oWM">http://bit.ly/1Qb8oWM</a> 
and search under Biological, Biomedical and Life Sciences, where you
will find a link to the program.

via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:01</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137612776661">Graduate position: UppsalaU.EvolutionSexualDimorphism</a></div>
  <div class="description">PhD position on the evolution of sexual dimorphism in life-histories,
at Uppsala University

A PhD position is available at Animal Ecology, the Department of Ecology
and Genetics, Evolutionary Biology Centre, Uppsala University, Sweden,
starting spring 2016.


Research focus
Males and females often have different optimal life histories (for example
regarding rate of reproduction and lifespan), but they largely share the
same genome. This leads to conflicts over trait expression. We expect
stronger conflict when individuals mate with multiple partners over
life and weaker conflict under monogamy, because then the evolutionary
interests of the sexes align. The conflict can be resolved by decoupling
the genetic basis of the trait, allowing evolution of sexual dimorphism,
but the evolutionary processes leading to such sexual dimorphism remain
poorly understood. This project will use quantitative genetic methods
combined with experimental evolution in the nematode worm, C. remanei,
to investigate how different mating systems lead to changes in the
genetic basis of sexual conflict over evolutionary time.

Qualifications
The ideal candidate will have a keen interest and documented knowledge
in evolutionary ecology. You are motivated, highly reliable, driven
and well-organised, with the ability to work both independently and
in a group. The position requires an MSc degree (or equivalent) in
evolution, ecology or a related field and fluency in spoken and written
English. Strong statistical skills are a plus.

Position
The candidate will receive her/his postgraduate training within
the postgraduate school at the Evolutionary Biology Centre
(<a href="http://bit.ly/1RQJa3G">http://bit.ly/1RQJa3G</a>). The postgraduate training
comprises four years of full time studies and entitles the holder to
salary according to local salary rules and full social benefits. The
position can be combined with up to 20% of teaching assistantship,
which will then prolong the position accordingly.

The environment
The Evolutionary Biology Centre (EBC, <a href="http://bit.ly/16Fa1Wt">http://bit.ly/16Fa1Wt</a>) is
one of the world’s leading research institutions in evolutionary
biology and part of Uppsala University ­ the oldest university in
Scandinavia. Uppsala University, ranked top among European Universities
in the subject of biology (CHE European ranking), attracts approximately
45.000 students from all over the world, creating an international and
stimulating research environment as well as providing a multitude of
opportunities for collaboration and socializing. The city of Uppsala
is a vibrant college town, less than an hour by train from Stockholm
(and even closer to Arlanda International Airport), with beautiful and
easily accessible surroundings.

How to apply
The application should include (1) a cover letter describing your
research interests and experience and outlining your motivation to
work on this project, (2) a detailed CV describing your education and
listing authored publications (if any), (3) an accredited copy of your
MSc degree or equivalent (if available at time of application), and
(4), names and contact information (address, email address, and phone
number) of two academic references. The application should be written
in English or Swedish and should be sent via e-mail as a single pdf
file to elisabeth.bolund@ebc.uu.se. Review of applications will start
on February 15th, but candidates will be considered until the position
is filled. The starting date can be discussed but ideally early this
year. If you have any questions, don’t hesitate to contact me:

Elisabeth Bolund, PhD
Associate Professor
Department of Animal Ecology
Evolutionary Biology Centre
Uppsala University
Norbyvägen 18D
Uppsala, SE-752 36
Sweden
Phone: +46(0)18-4712613
Email: elisabeth.bolund@ebc.uu.se
Homepage: <a href="http://bit.ly/1RQJa3M">http://bit.ly/1RQJa3M</a>

Elisabeth Bolund 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>January 16, 2016</h3>
<div class="news-item">
 <div class="date">23:51</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137464163361">Graduate position: CharlesU_Prague.BirdAdaptation</a></div>
  <div class="description"><p>PhD position available from October 2016 &ldquo;Causes and consequences of
ecological specialization in birds&rdquo;


Project background:
Populations of ecological specialists decline due to their sensitivity
to global environmental changes. Although various studies confirmed
this pattern, wider context of evolutionary causes and ecological
consequences of species’ specialization remains unclear. Proposed
project focuses on elucidating these issues using birds as a study
system. Birds are a taxon with detailed knowledge of ecological and
life history traits, population and range sizes, phylogenetic
relationships and extinction risk of all species. Within this project,
we will use this information to learn more about evolutionary
conservatism of ecological specialization (whereas avian species do
not usually expand their niches after introduction outside native
ranges, closely related species often show high variability in their
niche breadth) and to focus on possible changes in specialization
across species’ geographic ranges, e.g. in relation to their local
densities. The applicant will also collect own field data to examine
relationships among habitat specialization at the individual,
population and species level, respectively. Finally, knowledge from
the above mentioned relationships will be applied in the analysis
predicting global extinction risk of particular species by different
facets of their specialization.

Eligibility:
We expect a motivated student with good experience in ornithology,
able (after some training) to work independently in the field in
Eastern Europe. The ideal candidate for this PhD position is
interested in ecology and evolutionary biology, and is willing to
learn new methods. He/she must have finished the MSc. (or equivalent)
by September 2016 at the latest.

Time and place:
The student will work in a young team based at the Faculty of Science,
Charles University in Prague, Czech Republic. Fieldwork will be
conducted in the Czech Republic. The position is available for up to
four years, starting in October 2016.

Salary:
The PhD candidate’s net monthly income will start at 16.500 CZK (ca
610 EUR) in the first year, and may increase with experience and
achievements during the study. (Note that living expenses in the Czech
Republic are generally lower than in Western European countries.)

Research team:
Jiří Reif (Institute for Environmental Studies, Charles University in
Prague) – supervisor
David Hořák (Department of Ecology, Charles University in Prague) –
co-supervisor

How to apply:
If interested, send a motivation letter, CV, publication list, title
or abstract of Master (Diploma) thesis, and contact details of 2
senior scientists that can provide references on you in a single PDF
file to jirireif@natur.cuni.cz until February 10, 2016. Pre-selected
candidates will be encouraged to submit an official application to the
university.

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:36</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137463486986">Other: GeorgetownU.REU.summer2016</a></div>
  <div class="description"><p>REU Site: Environmental Science and Policy in the Nation’s Capital
Georgetown University
Summer 2016

We anticipate that Georgetown University’s Research Experience for Undergraduates (REU) Site program &ldquo; Environmental Science and Policy in the Nation’s Capital &rdquo; will be funded by a grant from the National Science Foundation (award is pending). This ten-week program will bring a cohort of students to Washington DC to conduct research in environmental science at Georgetown University and to take part in a mini-curriculum that explores the intersections of science and policy. Participating research mentors have expertise in various aspects of environmental science and policy.

The 2016 program will run from May 31 - August 5. Students will be housed on campus and will receive a $5250 stipend plus a $750 allowance for food. We are striving for a diverse group, so students from populations typically underrepresented in STEM fields are strongly encouraged to apply. Review of applications will begin on February 15, 2016.

For more information on the program and to apply, visit: <a href="http://bit.ly/1JRTPbs">http://bit.ly/1JRTPbs</a>

For other inquiries, contact Dr. Manus Patten (mmp64@georgetown.edu)


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">21:49</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/maximum-parsimony-wheather-the-best-tree-should-be-completely-resolved-under-this-criteria/601">Maximum parsimony. Wheather the best tree should be completely resolved under this criteria?</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/sergei">@Sergei</a> wrote:</p>
            
              <p>I've got some relults on the phylogeny of some living creatures based on 850 bp fragment of 28S RNA subunit.</p>

<p>Built a standard set of trees based on NJ, ML, MP and BI methods.</p>

<p>BI was constructed in MrBayes. And all the rest in R with "ape" and "phangorn" packages wherein MP topology obtained with "pratchet" function in phangorn.</p>

<p>MP has provided me with completely resolved phylogeny, whereas NJ, ML and BI shown polytomies in some terminal branches. I have never met with such results and realize this could be evidence of incorrect tree search strategy with other methods.</p>

<p>But still my questions are:</p>

<ol>
<li>Wheather MP method (especially parsimony ratchet) in some cases can reasonably produce more reliable results than others? (I remember about long-branch-attraction and homoplasy)</li>
<li>In addition to minimization of the number of "steps" wheather MP algorithms seek for fully resolved phylogenies and discard topologies with polytomy?</li>
</ol>

<p>Thank you!</p>
            
            <p>Posts: 2</p>
            <p>Participants: 2</p>
            <p><a href="http://www.phylobabble.org/t/maximum-parsimony-wheather-the-best-tree-should-be-completely-resolved-under-this-criteria/601">Read full topic</a></p></div>
  <div class="source">Source: <a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:40</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/maximum-likelihood-tree-reconstruction-pipeline/600">Maximum Likelihood tree reconstruction pipeline</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/arambaut">@arambaut</a> wrote:</p>
            
              <p>I recently had need to reconstruct a reasonably large Ebola virus tree and I wanted a maximum likelihood tree with good branch length estimates and bootstrap values for all the nodes in the ML tree.  We wrote this pipeline that bats back and forth between phyml (to get an initial NJ tree and optimise the branch lengths) and raxml (to find ML topologies and bootstrap). </p>

<p>I thought I would post it here for discussion and to see if anyone would do things differently. On a fixed topology phyml did a better job of optimising branch lengths but this may be because I couldn't find the appropriate settings in raxml. </p>

#phyml to find initial tree using BioNJ
phyml --quiet -i $1.phy -q -t e -a e -o lr -b 0
mv $1.phy_phyml_tree.txt initial_tree.newick

#raxml to find ml topology
raxml -f d -T 6 -j -s $1.fasta -n topology -m GTRGAMMA -t initial_tree.newick

#phyml to optimise branch lengths on ml topology
phyml --quiet -i $1.phy -q -t e -a e -o lr -u RAxML_result.topology -b 0
mv $1.phy_phyml_tree.txt $1.ml.tree

#use raxml to produce boostrap trees - the seed needs to be set here in $2
raxml -f a ­p $2 -s $1.fasta ­x 12345 -#200 -m GTRGAMMA -n bootstrap

#use raxml to project boostrap values on to ML tree
raxml -f b -t $1.ml.tree -z RAxML_bootstrap.bootstrap -m GTRGAMMA -n final
mv RAxML_result.final $1.ml_bootstrap.tree
            
            <p>Posts: 5</p>
            <p>Participants: 3</p>
            <p><a href="http://www.phylobabble.org/t/maximum-likelihood-tree-reconstruction-pipeline/600">Read full topic</a></p></div>
  <div class="source">Source: <a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></div>
 </div>
</div>
<h3>January 15, 2016</h3>
<div class="news-item">
 <div class="date">03:11</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137339696351">Job: HarvardU.ResAssist.MolEvol.DeadlineUpdate</a></div>
  <div class="description">A research fellow (research assistant) position is available within
the Department of Organismic and Evolutionary Biology, Edwards Lab at
Harvard University.

The researcher will contribute to the development of a new experimental
and synthetic approach that involves reconstructing ancient molecular
components and observing their biological function in modern bacterial
cells through experimental evolution. This will generate experimental
systems consisting of viable modern-ancient hybrid bacterial organisms
amenable to biological, biochemical and genetic characterization.

The position will be based at the main campus of Harvard University,
and will include access to resources available through the Organismic
and Evolutionary Department and the FAS Center for Systems Biology.

The application packet should include a cover letter, CV and list of
references. In the cover letter, please describe your prior knowledge
or experience with specific molecular biology and bacterial genetics
techniques including bacterial genomic engineering, experimental
evolution, molecular cloning, protein expression and purification,
as well as experience with whole genome sequencing library preparation
methods and data analysis. The planned starting date for the position
is on or about March 15th, 2016; the cover letter should also indicate
the applicant&rsquo;s intended starting date.

The position includes competitive salary and full benefits, and is for 1
year with potential extension to 2 or more years. The minimum academic
requirement for this position is a masters degree in molecular and
cell biology, genetics, biology, or a closely related field. Applicants
with at least 2 years of research experience or an equivalent combination
of education and experience will also be considered.

Direct all questions and application materials to the team leader
Betul Kacar, kacar@g.harvard.edu. For further information visit
<a href="http://kacarlab.org/">http://kacarlab.org</a>, for OEB visit <a href="http://bit.ly/1bL0By6.">http://bit.ly/1bL0By6.</a>

Application Deadline: February 1st 2016

Harvard University is an equal opportunity employer and all qualified
applicants will receive consideration for employment without regard
to race, color, religion, sex, sexual orientation, gender identity,
national origin, disability status, protected veteran status, or any
other characteristic protected by law.

&ldquo;Kacar, Betul&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>January 14, 2016</h3>
<div class="news-item">
 <div class="date">21:03</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/how-to-recognize-a-rearranged-tree/599">How to recognize a rearranged tree?</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/ematsen">@ematsen</a> wrote:</p>
            
              <p><a class="mention" href="users/cwhidden.html">@cwhidden</a> and I had an entertaining discussion about the following question, which apparently came up in the <a class="mention" href="users/alexis_raxml.html">@Alexis_RAxML</a> lab as well. By tree rearrangement I mean an operation on some class of trees, such as subtree-prune-regraft (SPR) on unrooted trees.</p>

<p>Given some collection <em>A</em> of trees and a member <em>t</em> of <em>A</em>, how can we decide if a rearranged version of <em>t</em> is a member of <em>A</em>?</p>

<p>It seems like we'd like some sort of means of serializing trees that is compatible with the class of tree rearrangements. <a class="mention" href="users/armanbilge.html">@armanbilge</a> suggested thinking of a tree as the collection of its clades, and encoding that collection of clades as a binary vector indexed by the power set on the taxa (i.e. the set of all taxon subsets), and then keep track of what clades appear and disappear upon rearrangement. This would work, though the power set gets pretty big. </p>

<p>Thoughts? cc <a class="mention" href="users/mathmomike.html">@mathmomike</a> <a class="mention" href="users/david_bryant.html">@david_bryant</a></p>
            
            <p>Posts: 12</p>
            <p>Participants: 7</p>
            <p><a href="http://www.phylobabble.org/t/how-to-recognize-a-rearranged-tree/599">Read full topic</a></p></div>
  <div class="source">Source: <a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></div>
 </div>
</div>
<h3>January 13, 2016</h3>
<div class="news-item">
 <div class="date">04:33</div>
 <div class="body">
  <div class="title"><a href="http://iphylo.blogspot.com/2016/01/surfacing-deep-data-of-taxonomy.html">Surfacing the deep data of taxonomy</a></div>
  <div class="description"><p>My paper "Surfacing the deep data of taxonomy" (based on a presentation I gave in 2011) has appeared in print as part to a special issue of <i>Zookeys</i>: Page, R. (2016, January 7). Surfacing the deep data of taxonomy. ZooKeys. Pensoft Publishers. <a href="http://doi.org/10.3897/zookeys.550.9293">http://doi.org/10.3897/zookeys.550.9293</a>The manuscript was written shortly after the talk, but as is the nature of edited volumes it's taken a while to appear.</p> <p>My tweet about the paper sparked some interesting comments from David Shorthouse. <p lang="en" dir="ltr"><a href="https://twitter.com/KlausRiede">@KlausRiede</a> <a href="https://twitter.com/rdmpage">@rdmpage</a> Revitalize? Yes. Through less focus on taxa and more on uniquely identified, shared data objects and altmetrics.</p>&mdash; David Shorthouse (@dpsSpiders) <a href="https://twitter.com/dpsSpiders/status/685587561234829312">January 8, 2016</a> This is an appealing vision, because it seems unlikely that having multiple, small communities clustered around taxa will ever have the impact that taxonomists might like to have. Perhaps if we switch to focussing on objects (sequences, specimens, papers), notions of identity (e.g., DOIs, ORCID), and alternative measures of impact we can increase the visibility and perceived importance of the field. In this context, the recent paper "Wikiometrics: A Wikipedia Based Ranking System" <a href="http://arxiv.org/abs/1601.01058">http://arxiv.org/abs/1601.01058</a> looks interesting. A big consideration will be how connected is the network connecting taxonomists, papers, sequences, specimens, and names. If it's anything like the <a href="https://synthese.wordpress.com/2010/11/02/mendeley-data-vs-netflix-data/">network of readers in Mendeley</a> then we may face some challenges in community building around such a network.</p></div>
  <div class="source">Source: <a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></div>
 </div>
</div>
<h3>January 11, 2016</h3>
<div class="news-item">
 <div class="date">21:05</div>
 <div class="body">
  <div class="title"><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12148">Editorial</a></div>
  <div class="source">Source: <a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">16:16</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2016/01/directional-processes-in-language-change.html">Directional processes in language change</a></div>
  <div class="description"><br />Given that we are still in the process of beginning the new year, it seems to be in order to talk about directions — not in general, but rather in specific, namely, about directions in language change. This is important in so far as many processes in language evolution are directional. This means that they follow a specific direction from a state <i>X</i> to a state <i>Y</i>, and this is frequently attested across a large number of the languages of the world, while the opposite process, that state <i>Y</i> changes to state <i>X</i>, is extremely rare or even unattested.<br /><br />In language evolution there are a lot of well-known and well-investigated processes with a strong directional tendency. In sound change, for example, a [p] can easily become an [f], whether it is in the Indo-European, the Austronesian, or the Sino-Tibetan languages. Yet the opposite process, that an [f] becomes a [p] is extremely rare. Similar tendencies hold for a [k] becoming a [ʧ], as in Italian [ˈtʃɛnto] <i>cento</i> "hundred", going back to Latin [kɛntum] <i>centum</i>"hundred", or a [g] becoming a [h], as in Czech [ɦora] <i>hora</i> "mountain", going back to Proto-Slavic *<i>gora</i> "mountain" (<a href="http://bibliography.lingpy.org/?key=Derksen2008">Derksen 2008</a>).<br /><br />In semantic change, unidirectional tendencies can also be observed, although it is often more difficult to identify them, let alone generalising them.  Nevertheless, I think it is a rather safe bet to claim that words which originally mean "head" have a certain tendency to shift their meaning to denote "(the) first, the boss" or "the upper part, the top", while the opposite shift (that words which mean "boss" or "top" will be used to denote "head") is very unlikely to happen. Finally, in grammatical change, or, to be more precise, in <i>grammaticalization</i> (the process by which languages acquire new grammatical categories) directionality is one of the most important constraints (<a href="http://bibliography.lingpy.org/?key=Haspelmath2004">Haspelmath 2004</a>).<br /><br />Linguists usually know these tendencies very well, and they use them in their daily work, be it when trying to reconstruct the original pronuncation of words in unattested ancestral languages, when deciphering historical documents, or when tracing the semantic development of words through history.  Directional changes are also important in evolutionary biology. Ratchet-like (that means: unidirectional) processes serve as a major explanans for constructive neutral evolution (<a href="http://bibliography.lingpy.org/?key=Gray2010b">Gray et al. 2010</a>), direction is at the core of lateral gene transfer, and — as David mentioned in <a href="http://phylonetworks.blogspot.fr/2012/12/networks-trees-and-little-red-riding.html">an earlier post</a>— the usage of directional (non-reversible) models in phylogenetic reconstruction even provides an elegant way to root a tree (see also <a href="http://bibliography.lingpy.org/?key=Huelsenbeck2002">Huelsenbeck et al. 2002</a>).<br /><br />Given the active transfer of ideas from the biological to the linguistic domain in the last two decades, and the important role that directional processes play in both domains, it is surprising to me that methodological transfer has so far been almost exclusively limited to time-reversible models.  The only approach known to me that explicitly makes use of linguistic knowledge of directions is that of Baxter (<a href="http://bibliography.lingpy.org/?key=Baxter2006a">2006</a>). In this paper, Baxter analysed <i>phonological mergers</i> in Chinese dialects within a framework of Camin-Sokal parsimony (<a href="http://bibliography.lingpy.org/?key=Camin1965">Camin and Sokal 1965</a>).<br /><br />Phonological merger is a specific systemic process in language evolution. When sounds change (and they always change in some way), it may happen that two formerly distinct sounds are pronounced in the same way. As a result, words that formerly sounded different may suddenly sound alike, such as English <i>write</i> and <i>right</i>, which remain different only in their spelling not pronunciation. Mergers are a prototypical irreversible process. Once a merger has happened, speakers cannot go back, unless they recorded the original distinction and artificially tuned their language. But even this may be less easy than it seems — it is always easy to reduce distinctions. For example, most English speakers wouldn't have many difficulties in artificially pronouncing all instances of <i>s</i> as <i>sh</i> during a conversation. But being asked to pronounce a randomly chosen set of words with <i>s</i> as <i>sh</i> will turn out to be much more difficult. For this reason, mergers are an ideal data type for directional models of language change. Their drawback is, however, that they are difficult to determine, which may also be the reason why Baxter's approach has never been tested on other language families since then.<br /><br />It may be justified to use time-reversible models for analyses that use lexical data, especially cognate sets, as in the approaches following Gray and Atkinson (<a href="http://bibliography.lingpy.org/?key=Gray2003">2003</a>), since it is difficult to determine the impact of directional processes on lexical replacement. Furthermore, due to the <a href="http://phylonetworks.blogspot.fr/2015/12/lexicostatistics-predecessor-of.html">specific way the data is sampled</a>, it is extremely difficult to determine directions. Yet in many other approaches that use different types of data, especially in those cases that model sound change processes (<a href="http://bibliography.lingpy.org/?key=Hruschka2015">Hruschka et al.  2015</a>, <a href="http://bibliography.lingpy.org/?key=Wheeler2015">Wheeler and Whiteley 2015</a>) or grammatical change (<a href="http://bibliography.lingpy.org/?key=Longobardi2013">Longobardi et al. 2013</a>), it might have a substantial impact on the results if directionality was explicitly modeled.<br /><br />What does this mean for the directions for the New Year? I keep being surprised by the similarities between evolutionary biology and historical linguistics, be it the organization of information in genomes and languages, the processes that drive evolution, the philosophical questions underlying our investigations, or the quarrels among scholars in their fields. Unfortunately, much of the transfer from the biological to the linguistic domain is still very simplistic, often ignoring the specific differences between the two domains. On the other hand, many fruitful analogies are still out there but have not yet been properly investigated. So, as a direction for those who work in interdisciplinary domains in this New Year, I think we should try to avoid reinventing the wheel, and we should also pay attention to not putting wheels on sledges.<br /><br /><b>References</b><br /><ul><li>Baxter, W. (2006): <b>Mandarin dialect phylogeny</b>. <i>Cah. Linguistique -- Asie Orientale</i> 35.1. 71-114.</li><li>Camin, J. and R. Sokal (1965): <b>A method for deducing branching sequences in phylogeny</b>. <i>Evolution</i> 19.3. 311-327.</li><li>Derksen, R. (2008): <b>Etymological dictionary of the Slavic inherited lexicon</b> . Brill: Leiden and Boston.</li><li>Gray, R. and Q. Atkinson (2003): <b>Language-tree divergence times support the Anatolian theory of Indo-European origin</b>. <i>Nature</i> 426.6965. 435-439.</li><li>Gray, M., J. Lukes, J. Archibald, P. Keeling, and W. Doolittle (2010): <b>Cell biology. Irremediable complexity?</b>. <i>Science</i> 330.6006. 920-921.</li><li>Haspelmath, M. (2004): <b>On directionality in language change with particular reference to grammaticalization</b>. In: Fischer, O., M. Norde, and H. Perridon (eds.): <i>Up and down the cline -- The nature of grammaticalization</i>. John Benjamins Publishing Company: 17-44.</li><li>Hruschka, D., S. Branford, E. Smith, J. Wilkins, A. Meade, M. Pagel, and T. Bhattacharya (2015): <b>Detecting regular sound changes in linguistics as events of concerted evolution</b>. <i>Curr. Biol.</i> 25.1. 1-9.</li><li>Huelsenbeck, J., J. Bollback, and A. Levine (2002): <b>Inferring the root of a phylogenetic tree</b>. <i>Systems Biology</i> 51.1. 32-43.</li><li>Longobardi, G., C. Guardiano, G. Silvestri, A. Boattini, and A. Ceolin (2013): <b>Toward a syntactic phylogeny of modern Indo-European languages</b>. <i>J. Hist. Linguist.</i> 3.1. 122-152.</li><li>Wheeler, W. and P. Whiteley (2014): <b>Historical linguistics as a sequence optimization problem: the evolution and biogeography of Uto-Aztecan languages</b>. <i>Cladistics</i> 30.1. 1-13.</li></ul></div>
  <div class="source">Source: <a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></div>
 </div>
</div>
<h3>January 8, 2016</h3>
<div class="news-item">
 <div class="date">03:43</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136873437141">Postdoc: TexasAM.PopulationGenomics</a></div>
  <div class="description">POSITION DESCRIPTION:

Postdoctoral fellowship in marine population genomics of Hawaiian
â€˜opihi.  Subthemes include but are not limited to connectivity,
community managed areas, Bayesian modeling of fisheries.

PROJECT:

Assessing the efficacy of community managed areas, i.e. fine scale gene
flow patterns, parentage analysis, .  2 yrs., funded by the NOAA
Saltonstall-Kennedy program.

QUALIFICATIONS:

Ph.D. in a related field  and experience in at least one of the
following: population genetics, genomics, bioinformatics, statistical
analysis, modelling, fishery ecology and management, unix/linux,
computer programming

People skills are desirable

COMPENSATION:

Commensurate with skills and experience.  Competitive.

OPPORTUNITIES:

Training in project management, mentoring graduate students, genomic
laboratory techniques, population genomic analysis and modelling,
community outreach, teaching bioinformatics for ecologists, manuscript
writing, and grant writing.

RESOURCES:

HPC, high throughput genotyping laboratory, staffed Genomics Core Lab.

OTHER:

You will be expected to live in Corpus Christi, TX.  There will be
field work in Hawaii.  I&rsquo;ll cover at least 1 conference per year.
Two  PhD and two Master&rsquo;s students and Dr. Rob Toonen (U Hawaii) are
also involved in the project.

CONTACT:

chris.bird@tamucc.edu

Assistant Professor of Molecular Ecology

Director, Genomics Core Lab

Texas A&M University - Corpus Christi

Affiliate Faculty

Hawaii Institute of Marine Biology

&ldquo;Bird, Chris&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:13</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136872716241">Postdoc: MaxPlanckEvolBiol.GenomeAnalysisBioinformatics</a></div>
  <div class="description">A position for a postdoc in the field of genome analysis and
bioinformatics is available in the group of Diethard Tautz (Max-Planck
Institute for Evolutionary Biology, Department of Evolutionary Genetics,
Ploen, Germany). The postdoc will join the sub-group working on de
novo evolution of genes, funded by an ERC advanced grant. The insights
into de novo evolution of genes shapes currently into a new paradigm of
genome evolution (see Neme and Tautz 2014: CurrBiol 24:R38).  A large
genomic and transcriptomic dataset has been created for for a range of
species representing the evolutionary lineage towards the house mouse (Mus
musculus). Further population-level genome and transcriptome sequencing is
ongoing. These data provide an excellent basis for studying in molecular
detail the emergence of new genes from non-coding sequences. The postdoc
will be part of a team of PhD students and postdocs who work on all
aspects of de novo evolution, including functional testing in cell lines
and mouse knockouts. The postdoc will also have the chance to initiate
own experiments.

Qualification: PhD in genome data analysis, including programming and
statistics; experience in protein structure analysis and/or population
genetics would be of advantage.

Salary will be according to the TvoD 13-14 scale (depending on
experience), including social benefits and pension scheme. The position
is initially for two years, but can be extended.

The Max-Planck Institut for Evolutionary Biology in Ploen has developed
in the past years into a major center for basic research on evolutionary
topics, ranging from evolutionary genetics, evolutionary ecology,
experimental evolution to evolutionary theory. It runs together with
the nearby University of Kiel an international graduate program
(IMPRS in Evolutionary Biology) and a Master program in Molecular
Biology and Evolution. Scientists come from all over the world and the
working language is english. Ploen is a small village in a beautiful
postglacially shaped landscape with all the amenities of a touristically
active area. Two major cities (Kiel and Luebeck) as well as the Baltic Sea
are only 30-40min away, Hamburg is about 90min away (all well connected
by train).

Applications should include a CV and a publication list (including
papers in press or submitted), a statement of motivation, as well as
two names for obtaining letters of reference. Applications should be
sent by email with documents attached as pdf files to Diethard Tautz
(tautz@evolbio.mpg.de).


Prof. Dr. Diethard Tautz
Max-Planck-Institut fuer Evolutionsbiologie
Abteilung Evolutionsgenetik
August-Thienemannstrasse 2
24306 Ploen (Germany)
Tel.: 04522 763 390
Fax: 04522 763 281
tautz@evolbio.mpg.de
<a href="http://bit.ly/1K1JGng">http://bit.ly/1K1JGng</a>




Diethard Tautz 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:58</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136872354051">Course: Leicester.NGSAssembly&VariantCalling.Feb1-3</a></div>
  <div class="description"><p>Deadline for applications is approaching for the BBASH training
course &ldquo;DNA-Seq Next Generation Sequencing&rdquo;. Applications are welcome 
until Friday 15/01/2016!

BBASH at University of Leicester is pleased to announce that 
applications for the upcoming Next Generation Sequencing training course 
are now open.

Deadline for applications: 15th January 2016
Notification of acceptance: 18th January 2016
Course dates: 1st-3rd February 2016

Venue: University of Leicester - College Court Conference Centre, 
Leicester, UK (<a href="http://bit.ly/1K1HPPp">http://bit.ly/1K1HPPp</a>)

Instructors:
Richard Badge (BBASH, University of Leicester, UK)
Chiara Batini (University of Leicester, UK)
Matthew Blades (BBASH, University of Leicester, UK)


Course description

This course will provide an introduction to next generation sequencing 
(NGS) platforms, data analysis and tools for data quality control, 
genome assembly and variant calling. The course will be delivered using 
a mixture of lectures and computer based hands on practical sessions 
using real data.

This course is aimed at wet-lab biologists who are involved in research 
projects that will require the handling and analysis of NGS data. A 
significant proportion of the course will be computer-based using 
command line tools in the Unix environment, therefore, in order to gain 
maximum benefit from the course all attendees will be required to have 
basic Unix skills.


For additional information: 
<a href="http://bit.ly/22P7s1g">http://bit.ly/22P7s1g</a>



The Organisers
Chiara Batini (University of Leicester, UK)
Matthew Blades (BBASH, University of Leicester, UK)


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:43</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136871967146">Course: SCENE_Glasgow.PythonForBiologists.May23-27</a></div>
  <div class="description">INTRODUCTION TO PYTHON FOR BIOLOGISTS
REPEAT COURSE!!

This course is being delivered by Dr Martin Jones, an expert in Python
and author of two text books,
Python for Biologists
[http://amzn.to/1NCtnR4
/dp/1492346136/]
Advanced Python for Biologists
[http://amzn.to/1O9c4Kw
44377/].

Due to high demand a repeat of this course will run from 23rd - 27th
May 2016 at SCENE (the Scottish Centre for Ecology and the Natural
Environment), Loch Lomond National Park, Glasgow.

Course overview: 
Python is a dynamic, readable language that is a popular platform for
all types of bioinformatics work, from simple one-off scripts to large,
complex software projects. This workshop is aimed at complete beginners
and assumes no prior programming experience. It gives an overview of the
language with an emphasis on practical problem-solving, using examples
and exercises drawn from various aspects of bioinformatics work. After
completing the workshop, students should be in a position to (1) apply
the skills they have learned to tackle problems in their own research and
(2) continue their Python education in a self-directed way.

Intended audience:
This workshop is aimed at all researchers and technical workers with a
background in biology who want to learn programming. The syllabus has
been planned with complete beginners in mind; people with previous
programming experience are welcome to attend as a refresher but may
find the pace a bit slow.

Teaching format:
The workshop is delivered over ten half-day sessions (see the detailed
curriculum below). Each session consists of roughly a one hour lecture
followed by two hours of practical exercises, with breaks at the
organizer&rsquo;s discretion. There will also be plenty of time for
students to discuss their own problems and data.

Assumed background:
Students should have enough biological background to appreciate the
examples and exercise problems (i.e. they should know about DNA and
protein sequences, what translation is, and what introns and exons
are). No previous programming experience or computer skills (beyond the
ability to use a text editor) are necessary, but you&rsquo;ll need to have a
laptop with Python installed.

Curriculum:
Day 1:
Module 1 - Introduction
Module 2 - Output and text manipulation
Day 2:
Module 3 - File IO and user interfaces
Module 4 - Flow control 1: loops
Day 3:
Module 5 - Flow control 2: conditionals
Module 6 - Organizing and structuring code
Day 4:
Module 7 - Regular expressions
Module 8 - Dictionaries
Day 5:
Module 9 - Interaction with the file system
Module 10 - Optional free afternoon to cover previous modules and
            discuss data

The cost is £500 including lunches and course materials. An
all-inclusive option is also available at £710; this includes
breakfast, lunch, dinner, refreshments, accommodation and course
materials. Participants will need a laptop with a recent version of
Python installed.

Please send inquiries to oliverhooker@prstatistics.co.uk or visit the
website <a href="http://www.prstatistics.com/">www.prstatistics.com</a>

Please fee free to distribute this information anywhere you think
suitable
Other upcoming courses - TIME SERIES MODELS FOR ECOLOGISTS AND
CLIMATOLOGISTS; GENETIC DATA ANALYSIS USING SIAR; ; APPLIED BAYESIAN
MODELLING FOR ECOLOGISTS AND EPIDEMIOLOGISTS; SPATIAL ANALYSIS OF
ECOLOGICAL DATA USING R; ADVANCING IN STATISTICAL MODELLING USING R;
INTRODUCTION TO STATISTICS AND R FOR BIOLOGISTS; DNA TAXONOMY; ;
BIOINFORMATICS FOR GENETICISTS AND BIOLOGISTS; MULTIVARIATE ANALYSIS OF
SPATIAL DATA; ADVANCES IN DNA TAXONOMY USING R;

Oliver Hooker
PR~Statistics

&ldquo;oliverhooker@prstatistics.co.uk&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:43</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136871967256">Postdoc: ColdSpringHarborLab.ComputationalGenomics</a></div>
  <div class="description">The Siepel Lab in the Simons Center for Quantitative Biology at Cold
Spring Harbor Laboratory is seeking full-time postdoctoral computational
researchers. The Siepel Group specializes in the development of
probabilistic models, algorithms for inference, prediction methods, and
application of these methods in large-scale genomic data analysis. Of
particular interest is research relevant to existing, NIH supported
projects in (1) HUMAN POPULATION GENOMICS, including demography inference
using Bayesian coalescent-based methods, inference of natural selection
on regulatory and other noncoding sequences, and prediction of fitness
consequences for noncoding mutations; and (2) TRANSCRIPTIONAL REGULATION
in mammals and Drosophila, including the estimation of rates and
patterns of transcriptional elongation from GRO-seq data, prediction of
transcription factor binding sites from DNase-seq data, and regulatory
network inference based on joint patterns of transcription and binding
in inducible systems. The research on transcriptional regulation will
continue to be carried out in close collaboration with Dr. John Lis at
Cornell University.

RELEVANT RECENT PAPERS:
Kuhlwilm M, Gronau I, Hubisz MJ, de Filippo C, Prado J, et al. Ancient
gene flow from modern humans into Siberian Neandertals. Nature (in
press), 2015.

Danko CG, Hyland SL, Core LJ, Martins AL, Waters CT, et al. Identification
of active transcriptional regulatory elements from GRO-seq
data. Nat. Methods. 2015. 12(5):433-438. PMCID: PMC4507281.

Rasmussen MD, Hubisz MJ, Gronau I, Siepel A. Genome-wide inference of
ancestral recombination graphs. PLOS Genet. 2014. 10(5): e1004342. PMCID:
PMC4022496.

Freedman AH, Gronau I, Schweizer RM, Ortega-Del Vecchyo D, Han E, et
al. (30 co-authors). Genome sequencing highlights the dynamic early
history of dogs. PLOS Genet. 2014. 10(1):e1004016. PMCID: PMC3894170.

Core LJ, Martins AL, Danko CG, Waters CT, Siepel A, et al. Analysis of
nascent RNA identifies a unified architecture of initiation regions at
mammalian promoters and enhancers. Nat. Genet. 2014. 46(12):1311-1320
PMCID: PMC4254663.

Capra JA, Hubisz MJ, Kostka D, Pollard KS, Siepel A. A model-based
analysis of GC-biased gene conversion in the human and chimpanzee
genomes. PLOS Genet. 2013. 9(8):e1003684. PMCID: PMC3744432.

Arbiza L, Gronau I, Aksoy BA, Hubisz MJ, Gulko B, Keinan A, Siepel
A. Genome-wide inference of natural selection on human transcription
factor binding sites. Nat. Genet. 2013. 45(7):723-729. PMCID: PMC3932982.

Gronau I, Arbiza L, Mohammed J, Siepel A. Inference of natural
selection from interspersed genomic elements based on polymorphism and
divergence. Mol. Biol. Evol. 2013. 30(5):1159-1171. PMCID: PMC3697874.

Gronau I, Hubisz MJ, Gulko B, Danko CG, Siepel A. Bayesian
inference of ancient human demography from individual genome
sequences. Nat. Genet. 2011. 43(10):1031-1034. PMCID: PMC3245873.

The ideal candidate will be enthusiastic, highly motivated and
independent; will have a strong background in comparative genomics,
population genomics, or transcriptional regulation, a serious
interest in molecular biology and genetics combined with good skills in
mathematics and computer science, and a strong record of accomplishment in
research. Postdoctoral associates in the group are expected to participate
in both methods development and data analysis, to be comfortable in
a fast-moving, interactive, and collaborative research environment,
and to actively publish and present results.

REQUIRED QUALIFICATIONS: Ph.D. in bioinformatics, computer science,
statistics, genetics, molecular biology, applied mathematics, or a
related field;  Research experience (with first-author publications)
in computational genomics or a closely related field;  Fluency in
probabilistic modeling and computational statistics;  Proficiency in
programming, ideally in C or C++, as well as in scripting languages
such as python, Perl, mat lab, or R; should be comfortable in a Linux
environment, with large data sets, computer clusters, and databases.

TO APPLY: Position Number
01229-R. <a href="http://bit.ly/1JtlFu9.">http://bit.ly/1JtlFu9.</a>  Informal inquiries
are also welcome.  Please submit a CV, a short description of research
interests and experience, and contact information for three references
by e-mail to asiepel@cshl.edu.

Adam Siepel, Ph.D.
Simons Center for Quantitative Biology
Cold Spring Harbor Laboratory
One Bungtown Road
Cold Spring Harbor, NY 11724

CSHL is an EO/AA Employer. All qualified applicants will receive
consideration for employment and will not be discriminated against on the
basis of race, color, religion, sex, sexual orientation, gender identity,
national origin, age, disability or protected veteran status. VEVRAA
Federal Contractor.


&ldquo;Gill, Irene&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:28</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136871565961">Graduate position: UAmsterdam.MothSexualSelection</a></div>
  <div class="description">PhD position available at the University of Amsterdam, Institute for
Biodiversity and Ecosystem Dynamics, Dept. Evolutionary Biology

Project: â€œUnderstanding sexual selection: which sex leads the dance
when both are signallers?â€�

Supervisor: Dr. Astrid T. Groot

<a href="http://bit.ly/1Jyq7I4">http://bit.ly/1Jyq7I4</a>

Sexual selection generally assumes that one sex is the signaller and
the other sex the responder. The forces of sexual selection are likely
more complex when both sexes are signallers and responders. In moths,
the species-specific female sex pheromones to which males are attracted
from a distance have been studied in detail, because these pheromones
can be used for pest management. However, at close-range males produce
and emit a biosynthetically related but different pheromone, which is
likely important for close-range female choice and/or male competition.
This project aims to determine the role of close-range courtship in
generating variation in the sexual communication channel to assess the
strength of mutual mate choice in a species where we have found a
within-population polymorphism in the female sex pheromone. This
project will combine behavioral and chemical analyses with genetic
(quantitative trait locus) analyses to answer the questions whether the
female and male sex pheromone genetically linked, what are the genes
underlying the male pheromone variation, what is the importance of male
sex pheromone variation in female choice and what is its role in
male-male competition.

Requirements

-MSc in Biology, with focus on evolution, and interest in chemical
ecology and entomology;

-Experience with molecular biological techniques, preferably also with
genetic analyses

-Interest to combine laboratory and field studies with genetic and
chemical analyses

-Skills in experimental design and statistical analysis

-Proficiency in scientific writing and communication

Applications should include a detailed CV, a motivation letter, and the
names and contact addresses of two references from which information
about the candidate can be obtained. Applications should be sent before
5 February to application-science@uva.nl. Please quote vacancy number
16-005 in the subject field. Interviews will be held in the week of 15
February.

Appointment
The position will start preferably March or April 2016. The full-time
appointment will be on a temporary basis for a maximum period of four
years. Initial appointment will be for a period of 18 months and after
satisfactory evaluation it will be extended for a total duration of
four years. The full-time gross monthly salary will range from â‚¬2,173
in the first year to â‚¬2,778 in the final year, according to the Dutch
salary scales for PhD candidates. The Collective Labour Agreement for
Dutch Universities is applicable. The annual salary will be increased
by 8 % holiday allowance and 8.3 % end-of-year bonus.

Astrid Groot 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:13</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136871160641">Graduate position: RoslinInstitute.UEdinburgh.CanineGenomics</a></div>
  <div class="description"><p>Ph.D. Studentship at The Roslin Institute (University of Edinburgh)
DEADLINE: 01 FEBRUARY 2016.

We are currently recruiting a 3-year Ph.D. student to investigate genetic diversity and evolutionary changes in dogs using genomic techniques. 

Domesticated animals provide an ideal study system for addressing a variety of questions in biology due to their substantial phenotypic variation, the rapid rate of phenotypic change and the broad diversification between breeds within a species. Dogs are particularly extreme in this regard, showing wide variation in both physical and behavioural characteristics, and thus are an excellent model species for questions in evolutionary biology, genetics and behavioural biology.
	
The student will apply population genetic and genomics techniques to investigate diversity within and between dog breeds, to identify genomic regions that show evidence of selection and to address whether signals of selection are enriched in regions of the genome associated with behavioural traits, as has been hypothesized more generally for domesticated animals. Bioinformatics analyses will be applied to investigate functional classes of genes that are enriched in genomic regions associated with these behavioural traits and in regions showing evidence of selection.
	
The studentship will provide training in transferable skills and techniques in quantitative and population genetics, statistics, bioinformatics and genomic analysis. The student will also be expected to regularly attend seminars, journal clubs and local meetings including the Edinburgh Alliance for Complex Traits. There will be opportunities for attendance at national and international conferences.

We are seeking an enthusiastic individual with an interest in population and evolutionary biology and animal genetics as well as aptitude with computer-based statistical analyses. The candidate should possess a good quality first degree in a relevant subject. Enthusiasm, dedication and aptitude are the most important characteristics; additional training in genetics, genome analysis and animal behaviour are available through MSc courses at the University of Edinburgh.

For further information about the project, contact Pam Wiener (pam.wiener@roslin.ed.ac.uk). For information about the application process and to apply, go to <a href="http://bit.ly/1kRFjoc.">http://bit.ly/1kRFjoc.</a> APPLICATIONS MUST BE SUBMITTED ONLINE.

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:28</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136869854211">Other: SouthAfrica.VolResAssist.SocialMoleRats</a></div>
  <div class="description">SouthAfrica.VolResearchAssistant.SocialMoleRats

Cooperative breeding in Damaraland mole-rats (Fukomys damarensis).

Voluntary field research assistant: Mid-February to Mid-May 2016 (3
months)

We are looking for one voluntary field research assistants to conduct
research on the completely subterranean, highly social Damaraland
mole-rat (Fukomys damarensis). Specifically, this project investigates
the group augmentation hypothesis and how benefits of living in large
groups can generate selection for altruism in animals. We are currently
conducting a large scale capture-mark and recapture experiment running
over one year in which the voluntary research assistant would play an
important role. Field work takes place in the southern Kalahari,
Northern Cape province of South Africa in a private game reserve.
Entire groups of mole-rats will be captured and individually marked.
Morphological measurements and tissue samples will be obtained before
the release of the animals. The work is physically demanding (heavy
digging to capture mole-rats) and weather conditions are very
challenging (heat during the day, cold during the night). Field work
will make trapping during the night necessary. The assistant will
mostly be working along one more experienced scientist but will need to
work independently at times.

Applicants should be enthusiastic, willing to work hard and keen to get
involved in an ecological research project in the African bush. The
research assistants are expected to be willing to work at night when
trapping schedules require it and need to be fit as capturing mole-rats
requires the capacity to do hard physical work (digging) in a
challenging environment. Applicants must be holders of a driving
license. A zoology related degree and/or previous field experience will
be considered an asset. The successful applicant will work in a small
team of 2 persons and hence need to be a team player and generally a
person that enjoys collaboration under sometimes challenging and
isolated circumstances.

What you will learn:

Conduct field research in a remote area in the African bush.

Capture-mark and recapture techniques of small rodents.

Biological sampling and processing of scientific samples (blood, tissue
and morphometrics).

Accommodation is provided, and research assistants are paid a weekly
allowance to cover their personal costs and food. Costs for travelling
from Johannesburg to the field site will be covered. We have no funding
to cove overseas travel to South Africa.

Applications received until the 15th of January 2016 are ensured full
consideration. Later applications can be considered.

To apply (CV + cover letter) or enquire further information:

Dr Markus Zöttl

Research associate

University of Cambridge

+441223336673

Mz338@cam.ac.uk

Markus Zoettl 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:58</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136868873451">Course: GreifswaldU.GeneticAdaptation.Mar15-18</a></div>
  <div class="description">07-01-2016

International Summer School on Genetic Adaptation

Date:         15^th -18^th March 2016

Venue:      Greifswald University, Zoology Lab Building, SoldmannstraÃŸe 14

The DFG funded Research Training Group RESPONSE (Biological Responses
to Novel and Changing Environments; <a href="http://bit.ly/1JyiO3k">http://bit.ly/1JyiO3k</a>)
organizes a summer school on genetic adaptation. We are broadly
interested in responses to environmental change, for which genetic
adaptation may play an important role. The summer school aims at
deepening the knowledge of the prospects and limitations of different
genetic approaches in understanding adaptation to environmental change.

We will cover the following topics: Adaptive genetic variation and its
detection, Omics technologies in ecological research, heritability and
quantitative genetics, dynamics of rapid adaptation and niche
evolution. The programme will include lectures given by renowned
experts in the field and a young investigators&rsquo; day providing the
opportunity to present the own research project. There will also be
computer courses including exercises on estimating heritabilities and
on eco-evolutionary population modelling (for details please visit
<a href="http://bit.ly/1JyiO3k">http://bit.ly/1JyiO3k</a>).

The summer school is open to a limited number of external graduate /
PhD students. If you would like to participate, please send an email
including a short CV and a letter of motivation (no more than half a
page) to Michael SchÃ¶ner (schoenerm@uni-greifswald.de) before 15^th
February 2016. Participation is free of charge, though we cannot cover
travel expenses and accommodation.



Prof. Dr. Klaus Fischer
Zoological Institute & Museum
Greifswald University
J.-S.-Bach-Str. 11/12
D-17489 Greifswald
Phone: +49-3834-864266
Fax: +49-3834-864252

&ldquo;kfischer@uni-greifswald.de&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="index3f45.html?q=aggregator&amp;from=260">previous page</a></div><div>... <a href="indexeef0.html?q=aggregator&amp;from=200">11</a> <a href="index933b.html?q=aggregator&amp;from=220">12</a> <a href="indexd118.html?q=aggregator&amp;from=240">13</a> <a href="index3f45.html?q=aggregator&amp;from=260">14</a> <strong>15</strong> <a href="index8c8d.html?q=aggregator&amp;from=300">16</a> <a href="index9efb.html?q=aggregator&amp;from=320">17</a> <a href="indexd191.html?q=aggregator&amp;from=340">18</a> <a href="index28a4.html?q=aggregator&amp;from=360">19</a> ...</div><div><a href="index8c8d.html?q=aggregator&amp;from=300">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=280 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:26:26 GMT -->
</html>
