<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=540 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:39:20 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="indexacd7.html?q=node/208">56(4) August 2007</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 7 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>November 2, 2015</h3>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/237">Phylogeographic data revealed shallow genetic structure in the kelp Saccharina japonica (Laminariales, Phaeophyta)</a></div>
  <div class="description">Background:
Population structure and genetic diversity of marine organisms in the Northwestern Pacific Ocean exhibited complex patterns. Saccharina japonica is a commercially and ecologically important kelp species widely distributed along the coast of Japan Sea. However, it is still poorly known about population genetics and phylogeographic patterns of wild S. japonica populations on a large geographic scale, which is an important contribution to breeding and conservation of this marine crop.
Results:
We collected 612 mitochondrial COI and trnW-trnL sequences. Diversity indices suggested that S. japonica populations along the coast of Hokkaido exhibited the highest genetic diversity. Bayesian Analysis of Population Structure (BAPS) revealed four clusters in the kelp species (cluster 1: Hokkaido and South Korea; cluster 2: northwestern Hokkaido; cluster 3: Far Eastern Russia; cluster 4: China). The network inferred from concatenated data exhibited two shallow genealogies corresponding to two BAPS groups (cluster 2 and cluster 3). We did not detect gene flow between the two shallow genealogies, but populations within genealogy have asymmetric gene exchange. Bayesian skyline plots and neutrality tests suggested that S. japonica experienced postglacial expansion around 10.45 ka.
Conclusions:
The coast of Hokkaido might be the origin and diversification center of S. japonica. Gene exchange among S. japonica populations could be caused by anthropogenic interference and oceanographic regimes. Postglacial expansions and gene exchange apparently led to more shared haplotypes and less differentiation that in turn led to the present shallow phylogeographical patterns in S. japonica.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>October 30, 2015</h3>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/235">Cryptic diversity in Hipposideros commersoni sensu stricto (Chiroptera: Hipposideridae) in the western portion of Madagascar</a></div>
  <div class="description">Background:
The Commerson’s leaf-nosed bat, Hipposideros commersoni sensu stricto, is endemic to Madagascar and is relatively common in the western portion of the island, where it is found in areas, including forested zones, from sea level to 1325 m. A previous study on morphological patterns of geographic variation within the species highlighted the presence of two distinct morphotypes; larger individuals in the north portion of the island and smaller individuals in the south. The main aim of this study was to use a combination of craniodental morphology and molecular data (mitochondrial and nuclear) to test previous hypotheses based on morphology and clarify the evolutionary history of the species group.
Methods:
We sequenced mitochondrial and nuclear genes from Hipposideros commersoni obtained from the western portion of Madagascar, and compared them with other African species as outgroups. We analyzed the sequence data using Maximum Likelihood and Bayesian phylogenetic inference. Divergence dates were estimated using Bayesian molecular clock approach. Variation in craniodental variables was also assessed from sequenced individuals.
Results:
The molecular analyses suggest that H. commersoni is not monophyletic, with strong support for the presence of several independently evolving lineages. Two individuals amongst those sequenced from Isalo (south central) and Itampolo (southwest) form a separate clade (Clade A), distinct from other H. commersoni, and sister to continental African H. vittatus and H. gigas. Within the H. commersoni clade, the molecular data support two geographically distributed clades; one from the south (Clade B) and the other from the north (Clade C), which diverged approximately 3.38 million years ago. Morphometric data were consistent with the molecular analyses, suggesting a north–south break within H. commersoni. However, at some localities, animals from both clades occurred in sympatry and these individuals could not be differentiated based on external and craniodental measurements.
Conclusions:
Using a combination of molecular and morphological characters, this study presents evidence of cryptic diversity in H. commersoni on Madagascar. Further fine-scale phylogeographic studies are needed to fully resolve the systematics of H. commersoni. This study highlights the utility of the combined approach in employing both morphological and molecular data to provide insights into the evolutionary history of Malagasy population currently assigned to H. commersoni.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>October 28, 2015</h3>
<div class="news-item">
 <div class="date">18:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/234">Initiation of recombination suppression and PAR formation during the early stages of neo-sex chromosome differentiation in the Okinawa spiny rat, Tokudaia muenninki</a></div>
  <div class="description">Background:
Sex chromosomes of extant eutherian species are too ancient to reveal the process that initiated sex-chromosome differentiation. By contrast, the neo-sex chromosomes generated by sex-autosome fusions of recent origin in Tokudaia muenninki are expected to be evolutionarily ‘young’, and therefore provide a good model in which to elucidate the early phases of eutherian sex chromosome evolution. Here we describe the genomic evolution of T. muenninki in neo-sex chromosome differentiation.
Results:
FISH mapping of a T. muenninki male, using 50 BAC clones as probes, revealed no chromosomal rearrangements between the neo-sex chromosomes. Substitution-direction analysis disclosed that sequence evolution toward GC-richness, which positively correlates with recombination activity, occurred in the peritelomeric regions, but not middle regions of the neo-sex chromosomes. In contrast, the sequence evolution toward AT-richness was observed in those pericentromeric regions. Furthermore, we showed genetic differentiation between the pericentromeric regions as well as an accelerated rate of evolution in the neo-Y region through the detection of male-specific substitutions by gene sequencing in multiple males and females, and each neo-sex–derived BAC sequencing.
Conclusions:
Our results suggest that recombination has been suppressed in the pericentromeric region of neo-sex chromosomes without chromosome rearrangement, whereas high levels of recombination activity is limited in the peritelomeric region of almost undifferentiated neo-sex chromosomes. We conclude that PAR might have been formed on the peritelomeric region of sex chromosomes as an independent event from spread of recombination suppression during the early stages of sex chromosome differentiation.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/232">Characterizing selective pressures on the pathway for de novo biosynthesis of pyrimidines in yeast</a></div>
  <div class="description">Background:
Selection on proteins is typically measured with the assumption that each protein acts independently. However, selection more likely acts at higher levels of biological organization, requiring an integrative view of protein function. Here, we built a kinetic model for de novo pyrimidine biosynthesis in the yeast Saccharomyces cerevisiae to relate pathway function to selective pressures on individual protein-encoding genes.
Results:
Gene families across yeast were constructed for each member of the pathway and the ratio of nonsynonymous to synonymous nucleotide substitution rates (dN/dS) was estimated for each enzyme from S. cerevisiae and closely related species. We found a positive relationship between the influence that each enzyme has on pathway function and its selective constraint.
Conclusions:
We expect this trend to be locally present for enzymes that have pathway control, but over longer evolutionary timescales we expect that mutation-selection balance may change the enzymes that have pathway control.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/231">Ancestral role of Pax2/5/8 in molluscan brain and multimodal sensory system development</a></div>
  <div class="description">Background:
Mollusks represent the largest lophotrochozoan phylum and exhibit highly diverse body plans. Previous studies have demonstrated that transcription factors such as Pax genes play important roles during their development. Accordingly, in ecdysozoan and vertebrate model organisms, orthologs of Pax2/5/8 are among others involved in the formation of the midbrain/hindbrain boundary, the auditory/geosensory organ systems, and the excretory system.
Methods:
Pax2/5/8 expression was investigated by in situ hybridization during the development of representatives of the two major molluscan subclades, Aculifera and Conchifera.
Results:
Compared to the investigated polyplacophoran and bivalve species that lack larval statocysts as geosensory organs and elaborate central nervous systems (CNS), cephalopods possess highly centralized brains and statocysts. Pax2/5/8 is expressed in regions where sensory cells develop subsequently during ontogenesis. Expression domains include esthetes and the ampullary system in polyplacophorans as well as the eyes of cephalopods. No Pax2/5/8 expression was observed in the less centralized CNS of bivalve, polyplacophoran, and gastropod embryos, thus arguing for a loss of Pax2/5/8 involvement in CNS development in these lineages. In contrast, Pax2/5/8 is expressed among others in brain lobes along the trajectory of the esophagus that divides the cephalopod brain.
Conclusions:
Our results, along with those on Otx- and Hox-gene expression, demonstrate that the cephalopod condition is similar to that in mouse and fruit fly, with Otx being expressed in the anterior-most brain region (except for the vertical lobe) and a Pax2/5/8 expression domain separating the Otx-domain from a Hox-gene expressing posterior brain region. Thus, Pax2/5/8 appears to have been recruited independently into regionalization of non-homologous complex brains of organisms as different as squid, fruit fly, and mouse. In addition, Pax2/5/8 is expressed in multimodal sensory systems in mollusks such as the esthetes and the ampullary system of polyplacophorans as well as the eyes of cephalopods. Pax2/5/8-expressing cells are present in regions where the future sensory cells such as the polyplacophoran esthetes are situated and hence Pax2/5/8 expression probably predates sensory cell development during ontogeny. In mollusks, Pax2/5/8 is only expressed in derivatives of the ectoderm and hence an ancestral role in molluscan ectoderm differentiation is inferred.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>October 26, 2015</h3>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/230">Evolution of the highly networked deubiquitinating enzymes USP4, USP15, and USP11</a></div>
  <div class="description">Background:
USP4, USP15 and USP11 are paralogous deubiquitinating enzymes as evidenced by structural organization and sequence similarity. Based on known interactions and substrates it would appear that they have partially redundant roles in pathways vital to cell proliferation, development and innate immunity, and elevated expression of all three has been reported in various human malignancies. The nature and order of duplication events that gave rise to these extant genes has not been determined, nor has their functional redundancy been established experimentally at the organismal level.
Methods:
We have employed phylogenetic and syntenic reconstruction methods to determine the chronology of the duplication events that generated the three paralogs and have performed genetic crosses to evaluate redundancy in mice.
Results:
Our analyses indicate that USP4 and USP15 arose from whole genome duplication prior to the emergence of jawed vertebrates. Despite having lower sequence identity USP11 was generated later in vertebrate evolution by small-scale duplication of the USP4-encoding region. While USP11 was subsequently lost in many vertebrate species, all available genomes retain a functional copy of either USP4 or USP15, and through genetic crosses of mice with inactivating mutations we have confirmed that viability is contingent on a functional copy of USP4 or USP15. Loss of ubiquitin-exchange regulation, constitutive skipping of the seventh exon and neural-specific expression patterns are derived states of USP11. Post-translational modification sites differ between USP4, USP15 and USP11 throughout evolution.
Conclusions:
In isolation sequence alignments can generate erroneous USP gene phylogenies. Through a combination of methodologies the gene duplication events that gave rise to USP4, USP15, and USP11 have been established. Although it operates in the same molecular pathways as the other USPs, the rapid divergence of the more recently generated USP11 enzyme precludes its functional interchangeability with USP4 and USP15. Given their multiplicity of substrates the emergence (and in some cases subsequent loss) of these USP paralogs would be expected to alter the dynamics of the networks in which they are embedded.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>October 20, 2015</h3>
<div class="news-item">
 <div class="date">18:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/229">Metamorphic remodeling of morphology and the body cavity in Phoronopsis harmeri (Lophotrochozoa, Phoronida): the evolution of the phoronid body plan and life cycle</a></div>
  <div class="description">Background:
Phoronids undergo a remarkable metamorphosis, in which some parts of the larval body are consumed by the juvenile and the body plan completely changes. According to the only previous hypothesis concerning the evolution of the phoronid body plan, a hypothetical ancestor of phoronids inhabited a U-shaped burrow in soft sediment, where it drew the anterior and posterior parts of the body together and eventually fused them. In the current study, we investigated the metamorphosis of Phoronopsis harmeri with light, electron, and laser confocal microscopy.
Results:
During metamorphosis, the larval hood is engulfed by the juvenile; the epidermis of the postroral ciliated band is squeezed from the tentacular epidermis and then engulfed; the larval telotroch undergoes cell death and disappears; and the juvenile body forms from the metasomal sack of the larva. The dorsal side of the larva becomes very short, whereas the ventral side becomes very long. The terminal portion of the juvenile body is the ampulla, which can repeatedly increase and decrease in diameter. This flexibility of the ampulla enables the juvenile to dig into the sediment. The large blastocoel of the larval collar gives rise to the lophophoral blood vessels of the juvenile. The dorsal blood vessel of the larva becomes the definitive median blood vessel. The juvenile inherits the larval protocoel, mesocoel, and metacoel. Late in metamorphosis, however, the protocoel loses its epithelial structure: the desmosomes between cells and the basal lamina under the cells disappear. This loss may reflect a reduction of the protocoel, which is a characteristic of some recent phoronids.
Conclusions:
Based on our investigation of P. harmeri metamorphosis, we hypothesize that the phoronid ancestor was worm-like animal that possessed preoral, tentacular, and trunk coeloms. It lived on the soft sediment and collected food with its tentacles. When threatened, this worm-like ancestor buried itself in the soft sediment by means of the ventral protrusion into which the loop of the intestine and the blood vessels were drawn. We propose that this behavior gave rise to the body plan of all recent phoronids. The evolution of phoronid life cycle seems having more in common with“intercalation” than “terminal addition” theories.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/228">Erratum to: Sail or sink: novel behavioural adaptations on water in aerially dispersing species</a></div>
  <div class="description">No description available</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>October 19, 2015</h3>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/227">Analysis of 5&#8217; gene regions reveals extraordinary conservation of novel non-coding sequences in a wide range of animals</a></div>
  <div class="description">Background:
Phylogenetic footprinting is a comparative method based on the principle that functional sequence elements will acquire fewer mutations over time than non-functional sequences. Successful comparisons of distantly related species will thus yield highly important sequence elements likely to serve fundamental biological roles. RNA regulatory elements are less well understood than those in DNA. In this study we use the emerging model organism Nasonia vitripennis, a parasitic wasp, in a comparative analysis against 12 insect genomes to identify deeply conserved non-coding elements (CNEs) conserved in large groups of insects, with a focus on 5’ UTRs and promoter sequences.
Results:
We report the identification of 322 CNEs conserved across a broad range of insect orders. The identified regions are associated with regulatory and developmental genes, and contain short footprints revealing aspects of their likely function in translational regulation. The most ancient regions identified in our analysis were all found to overlap transcribed regions of genes, reflecting stronger conservation of translational regulatory elements than transcriptional elements. Further expanding sequence analyses to non-insect species we also report the discovery of, to our knowledge, the two oldest and most ubiquitous CNE’s yet described in the animal kingdom (700 MYA). These ancient conserved non-coding elements are associated with the two ribosomal stalk genes, RPLP1 and RPLP2, and were very likely functional in some of the earliest animals.
Conclusions:
We report the identification of the most deeply conserved CNE’s found to date, and several other deeply conserved elements which are without exception, part of 5’ untranslated regions of transcripts, and occur in a number of key translational regulatory genes, highlighting translational regulation of translational regulators as a conserved feature of insect genomes.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/226">Evolutionary history of phosphatidylinositol- 3-kinases: ancestral origin in eukaryotes and complex duplication patterns</a></div>
  <div class="description">Background:
Phosphatidylinositol-3-kinases (PI3Ks) are a family of eukaryotic enzymes modifying phosphoinositides in phosphatidylinositols-3-phosphate. Located upstream of the AKT/mTOR signalling pathway, PI3Ks activate secondary messengers of extracellular signals. They are involved in many critical cellular processes such as cell survival, angiogenesis and autophagy. PI3K family is divided into three classes, including 14 human homologs. While class II enzymes are composed of a single catalytic subunit, class I and III also contain regulatory subunits. Here we present an in-depth phylogenetic analysis of all PI3K proteins.
Results:
We confirmed that PI3K catalytic subunits form a monophyletic group, whereas regulatory subunits form three distinct groups. The phylogeny of the catalytic subunits indicates that they underwent two major duplications during their evolutionary history: the most ancient arose in the Last Eukaryotic Common Ancestor (LECA) and led to the emergence of class III and class I/II, while the second – that led to the separation between class I and II – occurred later, in the ancestor of Unikonta (i.e., the clade grouping Amoebozoa, Fungi, and Metazoa). These two major events were followed by many lineage specific duplications in particular in vertebrates, but also in various protist lineages. Major loss events were also detected in Vidiriplantae and Fungi. For the regulatory subunits, we identified homologs of class III in all eukaryotic groups indicating that, for this class, both the catalytic and the regulatory subunits were presents in LECA. In contrast, homologs of the regulatory class I have a more recent origin.
Conclusions:
The phylogenetic analysis of the PI3K shed a new light on the evolutionary history of these enzymes. We found that LECA already contained a PI3K class III composed of a catalytic and a regulatory subunit. Absence of class II regulatory subunits and the recent origin of class I regulatory subunits is puzzling given that the class I/II catalytic subunit was present in LECA and has been conserved in most present-day eukaryotic lineages. We also found surprising major loss and duplication events in various eukaryotic lineages. Given the functional specificity of PI3K proteins, this suggests dynamic adaptation during the diversification of eukaryotes.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>October 16, 2015</h3>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/225">Color vision varies more among populations than among species of live-bearing fish from South America</a></div>
  <div class="description">Background:
Sensory Bias models for the evolution of mate preference place a great emphasis on the role of sensory system variation in mate preferences. However, the extent to which sensory systems vary across- versus within-species remains largely unknown. Here we assessed whether color vision varies in natural locations where guppies (Poecilia reticulata) and their two closest relatives, Poecilia parae and Poecilia picta, occur in extreme sympatry and school together. All three species base mate preferences on male coloration but differ in the colors preferred.
Results:
Measuring opsin gene expression, we found that within sympatric locations these species have similar color vision and that color vision differed more across populations of conspecifics. In addition, all three species differ across populations in the frequency of the same opsin coding polymorphism that influences visual tuning.
Conclusions:
Together, this shows sensory systems vary considerably across populations and supports the possibility that sensory system variation is involved in population divergence of mate preference.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>October 15, 2015</h3>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/224">Molecular phylogenetics reveals convergent evolution in lower Congo River spiny eels</a></div>
  <div class="description">Background:
The lower Congo River (LCR) is a region of exceptional species diversity and endemism in the Congo basin, including numerous species of spiny eels (genus Mastacembelus). Four of these exhibit distinctive phenotypes characterized by greatly reduced optic globes deeply embedded into the head (cryptophthalmia) and reduced (or absent) melanin pigmentation, among other characteristics. A strikingly similar cryptophthalmic phenotype is also found in members of a number of unrelated fish families, strongly suggesting the possibility of convergent evolution. However, little is known about the evolutionary processes that shaped diversification in LCR Mastacembelus, their biogeographic origins, or when colonization of the LCR occurred.
Methods:
We sequenced mitochondrial and nuclear genes from Mastacembelus species collected in the lower Congo River, and compared them with other African species and Asian representatives as outgroups. We analyzed the sequence data using Maximum Likelihood and Bayesian phylogenetic inference.
Results:
Bayesian and Maximum Likelihood phylogenetic analyses, and Bayesian coalescent methods for species tree reconstruction, reveal that endemic LCR spiny eels derive from two independent origins, clearly demonstrating convergent evolution of the cryptophthalmic phenotype. Mastacembelus crassus, M. aviceps, and M. simbi form a clade, allied to species found in southern, eastern and central Africa. Unexpectedly, M. brichardi and brachyrhinus fall within a clade otherwise endemic to Lake Tanganikya (LT) ca. 1500 km east of the LCR. Divergence dating suggests the ages of these two clades of LCR endemics differ markedly. The age of the crassus group is estimated at ~4 Myr while colonization of the LCR by the brichardi-brachyrhinus progenitor was considerably more recent, dated at ~0.5 Myr.
Conclusions:
The phylogenetic framework of spiny eels presented here, the first to include LCR species, demonstrates that cryptophthalmia and associated traits evolved at least twice in Mastacembelus: once in M. brichardi and at least once in the M. crassus clade. Timing of diversification is broadly consistent with the onset of modern high-energy flow conditions in the LCR and with previous studies of endemic cichlids. The close genetic relationship between M. brichardi and M. brachyrhinus is particularly notable given the extreme difference in phenotype between these species, and additional work is needed to better understand the evolutionary history of diversification in this clade. The findings presented here demonstrate strong, multi-trait convergence in LCR spiny eels, suggesting that extreme selective pressures have shaped numerous phenotypic attributes of the endemic species of this region.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>October 14, 2015</h3>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/223">The serotonin-lir nervous system of the Bryozoa (Lophotrochozoa): a general pattern in the Gymnolaemata and implications for lophophore evolution of the phylum</a></div>
  <div class="description">Background:
Serotonin represents an evolutionary ancient neurotransmitter that is ubiquitously found among animals including the lophotrochozoan phylum Bryozoa, a group of colonial filter-feeders. Comparatively little is known on their nervous system, and data on their serotonin-lir nervous system currently are mostly limited to the basal phylactolaemates. Previous investigations indicated a common ground-pattern of the serotonin-lir nervous system in these animals, but in order to assess this on a larger scale, 21 gymnolaemate species from 21 genera were comparatively analysed herein.
Methods:
Twenty-one species from 21 gymnolaemate genera were analysed by immunocytochemical stainings and confocal laser scanning microscopy.
Results:
In all species the serotonin-lir signal is concentrated in the cerebral ganglion from where a nerve tract emanates laterally and traverses orally to engulf the foregut. Serotonin-lir perikarya are situated at the base of the tentacles that almost always correspond to the number of tentacles minus two. The oral side in almost all species shows three serotonin-lir perikarya followed by a ‘serotonergic gap’ that to our knowledge is not reflected in the morphology of the nervous system. Some species show additional serotonin-lir signal in tentacle nerves, visceral innervation and pore complexes. Paludicella articulata is exceptional as it shows signal in the latero-visceral nerves with serotonin-lir perikarya in the esophagus, parts of the tentacle sheath nerves as well as the frontal body wall around the parietal muscle bundles.
Conclusions:
In general, the serotonin-lir nervous system in the Bryozoa shows a consistent pattern among its different clades with few deviations. Preliminary data on phylactolaemates suggest the presence of a ‘serotonergic gap’ similar to gymnolaemates. Both show a subset of oral tentacles and the remaining tentacles in gymnolaemates which correspond to the lateral tentacles of phylactolaemates. The lophophoral concavity lacks serotonin-lir perikarya indicating that due to their larger sizes and increased tentacle number, the horse-shoe shaped arrangement could represent an apomorphy of phylactolaemates.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>October 13, 2015</h3>
<div class="news-item">
 <div class="date">18:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/214">Distinguishing between determinate and indeterminate growth in a long-lived mammal</a></div>
  <div class="description">Background:
The growth strategy of a species influences many key aspects of its life-history. Animals can either grow indeterminately (throughout life), or grow determinately, ceasing at maturity. In mammals, continued weight gain after maturity is clearly distinguishable from continued skeletal growth (indeterminate growth). Elephants represent an interesting candidate for studying growth because of their large size, long life and sexual dimorphism. Objective measures of their weight, height and age, however, are rare.
Results:
We investigate evidence for indeterminate growth in the Asian elephant Elephas maximus using a longitudinal dataset from a semi-captive population. We fit growth curves to weight and height measurements, assess sex differences in growth, and test for indeterminate growth by comparing the asymptotes for height and weight curves. Our results show no evidence for indeterminate growth in the Asian elephant; neither sex increases in height throughout life, with the majority of height growth completed by the age of 15 years in females and 21 years in males. Females show a similar pattern with weight, whereas males continue to gain weight until over age 50. Neither sex shows any declines in weight with age.
Conclusions:
These results have implications for understanding mammalian life-history, which could include sex-specific differences in trade-offs between size and reproductive investment.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>October 12, 2015</h3>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/222">Tex19 and Sectm1 concordant molecular phylogenies support co-evolution of both eutherian-specific genes</a></div>
  <div class="description">Background:
Transposable elements (TE) have attracted much attention since they shape the genome and contribute to species evolution. Organisms have evolved mechanisms to control TE activity. Testis expressed 19 (Tex19) represses TE expression in mouse testis and placenta. In the human and mouse genomes, Tex19 and Secreted and transmembrane 1 (Sectm1) are neighbors but are not homologs. Sectm1 is involved in immunity and its molecular phylogeny is unknown.
Methods:
Using multiple alignments of complete protein sequences (MACS), we inferred Tex19 and Sectm1 molecular phylogenies. Protein conserved regions were identified and folds were predicted. Finally, expression patterns were studied across tissues and species using RNA-seq public data and RT-PCR.
Results:
We present 2 high quality alignments of 58 Tex19 and 58 Sectm1 protein sequences from 48 organisms. First, both genes are eutherian-specific, i.e., exclusively present in mammals except monotremes (platypus) and marsupials. Second, Tex19 and Sectm1 have both duplicated in Sciurognathi and Bovidae while they have remained as single copy genes in all further placental mammals. Phylogenetic concordance between both genes was significant (p-value </div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/221">Comparing species tree estimation with large anchored phylogenomic and small Sanger-sequenced molecular datasets: an empirical study on Malagasy pseudoxyrhophiine snakes</a></div>
  <div class="description">Background:
Using molecular data generated by high throughput next generation sequencing (NGS) platforms to infer phylogeny is becoming common as costs go down and the ability to capture loci from across the genome goes up. While there is a general consensus that greater numbers of independent loci should result in more robust phylogenetic estimates, few studies have compared phylogenies resulting from smaller datasets for commonly used genetic markers with the large datasets captured using NGS. Here, we determine how a 5-locus Sanger dataset compares with a 377-locus anchored genomics dataset for understanding the evolutionary history of the pseudoxyrhophiine snake radiation centered in Madagascar. The Pseudoxyrhophiinae comprise ~86 % of Madagascar’s serpent diversity, yet they are poorly known with respect to ecology, behavior, and systematics. Using the 377-locus NGS dataset and the summary statistics species-tree methods STAR and MP-EST, we estimated a well-supported species tree that provides new insights concerning intergeneric relationships for the pseudoxyrhophiines. We also compared how these and other methods performed with respect to estimating tree topology using datasets with varying numbers of loci.
Methods:
Using Sanger sequencing and an anchored phylogenomics approach, we sequenced datasets comprised of 5 and 377 loci, respectively, for 23 pseudoxyrhophiine taxa. For each dataset, we estimated phylogenies using both gene-tree (concatenation) and species-tree (STAR, MP-EST) approaches. We determined the similarity of resulting tree topologies from the different datasets using Robinson-Foulds distances. In addition, we examined how subsets of these data performed compared to the complete Sanger and anchored datasets for phylogenetic accuracy using the same tree inference methodologies, as well as the program *BEAST to determine if a full coalescent model for species tree estimation could generate robust results with fewer loci compared to the summary statistics species tree approaches. We also examined the individual gene trees in comparison to the 377-locus species tree using the program MetaTree.
Results:
Using the full anchored dataset under a variety of methods gave us the same, well-supported phylogeny for pseudoxyrhophiines. The African pseudoxyrhophiine Duberria is the sister taxon to the Malagasy pseudoxyrhophiines genera, providing evidence for a monophyletic radiation in Madagascar. In addition, within Madagascar, the two major clades inferred correspond largely to the aglyphous and opisthoglyphous genera, suggesting that feeding specializations associated with tooth venom delivery may have played a major role in the early diversification of this radiation. The comparison of tree topologies from the concatenated and species-tree methods using different datasets indicated the 5-locus dataset cannot beused to infer a correct phylogeny for the pseudoxyrhophiines under any method tested here and that summary statistics methods require 50 or more loci to consistently recover the species-tree inferred using the complete anchored dataset. However, as few as 15 loci may infer the correct topology when using the full coalescent species tree method *BEAST. MetaTree analyses of each gene tree from the Sanger and anchored datasets found that none of the individual gene trees matched the 377-locus species tree, and that no gene trees were identical with respect to topology.
Conclusions:
Our results suggest that ≥50 loci may be necessary to confidently infer phylogenies when using summaryspecies-tree methods, but that the coalescent-based method *BEAST consistently recovers the same topology using only 15 loci. These results reinforce that datasets with small numbers of markers may result in misleading topologies, and further, that the method of inference used to generate a phylogeny also has a major influence on the number of loci necessary to infer robust species trees.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>October 7, 2015</h3>
<div class="news-item">
 <div class="date">18:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/206">Sand lizard (Lacerta agilis) phenology in a warming world</a></div>
  <div class="description">Background:
Present-day climate change has altered the phenology (the timing of periodic life cycle events) of many plant and animal populations worldwide. Some of these changes have been adaptive, leading to an increase in population fitness, whereas others have been associated with fitness decline. Representing short-term responses to an altered weather regime, hitherto observed changes are largely explained by phenotypic plasticity. However, to track climatically induced shifts in optimal phenotype as climate change proceeds, evolutionary capacity in key limiting climate- and fitness-related traits is likely to be crucial. In order to produce realistic predictions about the effects of climate change on species and populations, a main target for conservation biologists is thus to assess the potential of natural populations to respond by these two mechanisms. In this study we use a large 15-year dataset on an ectotherm model, the Swedish sand lizard (Lacerta agilis), to investigate how higher spring temperature is likely to affect oviposition timing in a high latitude population, a trait strongly linked to offspring fitness and survival.
Results:
With an interest in both the short- and potential long-term effect of rising temperatures, we applied a random regression model, which yields estimates of population-level plasticity and among-individual variation in the average, as well as the plastic, response to temperature. Population plasticity represents capacity for short-term adjustments whereas variation among individuals in a fitness-related trait indicates an opportunity for natural selection and hence for evolutionary adaptation. The analysis revealed both population-level plasticity and individual-level variation in average laying date. In contrast, we found no evidence for variation among females in their plastic responses to spring temperature, which could demonstrate a similarity in responses amongst females, but may also be due to a lack of statistical power to detect such an effect.
Conclusion:
Our findings indicate that climate warming may have positive fitness effects in this lizard population through an advancement of oviposition date. This prediction is consistent over shorter and potentially also longer time scales as the analysis revealed both population-level plasticity and individual-level variation in average laying date. However, the genetic basis for this variation would have to be examined in order to predict an evolutionary response.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>October 5, 2015</h3>
<div class="news-item">
 <div class="date">18:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/220">Integration of molecular cytogenetics, dated molecular phylogeny, and model-based predictions to understand the extreme chromosome reorganization in the Neotropical genus Tonatia (Chiroptera: Phyllostomidae)</a></div>
  <div class="description">Background:
Defining factors that contributed to the fixation of a high number of underdominant chromosomal rearrangements is a complex task because not only molecular mechanisms must be considered, but also the uniqueness of natural history attributes of each taxon. Ideally, detailed investigation of the chromosome architecture of an organism and related groups, placed within a phylogenetic context, is required. We used multiple approaches to investigate the dynamics of chromosomal evolution in lineages of bats with considerable karyotypic variation, focusing on the different facets contributing to fixation of the exceptional chromosomal changes in Tonatia saurophila. Integration of empirical data with proposed models of chromosome evolution was performed to understand the probable conditions for Tonatia’s karyotypic evolution.
Results:
The trajectory of reorganization of chromosome blocks since the common ancestor of Glossophaginae and Phyllostominae subfamilies suggests that multiple tandem fusions, as well as disruption and fusions of conserved phyllostomid chromosomes were major drivers of karyotypic reshuffling in Tonatia. Considerable variation in the rates of chromosomal evolution between phyllostomid lineages was observed. Thirty–nine unique fusions and fission events reached fixation in Tonatia over a short period of time, followed by ~12 million years of chromosomal stasis. Physical mapping of repetitive DNA revealed an unusual accumulation of LINE-1 sequences on centromeric regions, probably associated with the chromosomal dynamics of this genus.
Conclusions:
Multiple rearrangements have reached fixation in a wave-like fashion in phyllostomid bats. Different biological features of Tonatia support distinct models of rearrangement fixation, and it is unlikely that the fixations were a result of solely stochastic processes in small ancient populations. Increased recombination rates were probably facilitated by expansion of repetitive DNA, reinforced by aspects of taxon reproduction and ecology.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/219">Weathering the storm: parental effort and experimental manipulation of stress hormones predict brood survival</a></div>
  <div class="description">Background:
Unpredictable and inclement weather is increasing in strength and frequency, challenging organisms to respond adaptively. One way in which animals respond to environmental challenges is through the secretion of glucocorticoid stress hormones. These hormones mobilize energy stores and suppress non-essential physiological and behavioral processes until the challenge passes. To investigate the effects of glucocorticoids on reproductive decisions, we experimentally increased corticosterone levels (the primary glucocorticoid in birds) in free-living female tree swallows, Tachycineta bicolor, during the chick-rearing stage. Due to an unprecedented cold and wet breeding season, 90 % of the nests in our study population failed, which created a unique opportunity to test how challenging environmental conditions interact with the physiological mechanisms underlying life-history trade-offs.
Results:
We found that exogenous corticosterone influenced the regulation of parental decisions in a context-dependent manner. Control and corticosterone-treated females had similar brood failure rates under unfavorable conditions (cold and rainy weather), but corticosterone treatment hastened brood mortality under more favorable conditions. Higher female nest provisioning rates prior to implantation were associated with increased probability of brood survival for treatment and control groups. However, higher pre-treatment male provisioning rates were associated with increased survival probability in the control group, but not the corticosterone-treated group.
Conclusions:
These findings reveal complex interactions between weather, female physiological state, and partner parental investment. Our results also demonstrate a causal relationship between corticosterone concentrations and individual reproductive behaviors, and point to a mechanism for why naturally disturbed populations, which experience multiple stressors, could be more susceptible and unable to respond adaptively to changing environmental conditions.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/218">High evolutionary turnover of satellite families in Caenorhabditis</a></div>
  <div class="description">Background:
The high density of tandem repeat sequences (satellites) in nematode genomes and the availability of genome sequences from several species in the group offer a unique opportunity to better understand the evolutionary dynamics and the functional role of these sequences. We take advantage of the previously developed SATFIND program to study the satellites in four Caenorhabditis species and investigate these questions.
Methods:
The identification and comparison of satellites is carried out in three steps. First we find all the satellites present in each species with the SATFIND program. Each satellite is defined by its length, number of repeats, and repeat sequence. Only satellites with at least ten repeats are considered. In the second step we build satellite families with a newly developed alignment program. Satellite families are defined by a consensus sequence and the number of satellites in the family. Finally we compare the consensus sequence of satellite families in different species.
Results:
We give a catalog of individual satellites in each species. We have also identified satellite families with a related sequence and compare them in different species. We analyze the turnover of satellites: they increased in size through duplications of fragments of 100-300 bases. It appears that in many cases they have undergone an explosive expansion. In C. elegans we have identified a subset of large satellites that have strong affinity for the centromere protein CENP-A. We have also compared our results with those obtained from other species, including one nematode and three mammals.
Conclusions:
Most satellite families found in Caenorhabditis are species-specific; in particular those with long repeats. A subset of these satellites may facilitate the formation of kinetochores in mitosis. Other satellite families in C. elegans are either related to Helitron transposons or to meiotic pairing centers.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="indexbe68.html?q=aggregator&amp;from=520">previous page</a></div><div>... <a href="index3d2c.html?q=aggregator&amp;from=460">24</a> <a href="indexb567.html?q=aggregator&amp;from=480">25</a> <a href="index8c9a.html?q=aggregator&amp;from=500">26</a> <a href="indexbe68.html?q=aggregator&amp;from=520">27</a> <strong>28</strong> <a href="index25b4.html?q=aggregator&amp;from=560">29</a> <a href="index3ad0.html?q=aggregator&amp;from=580">30</a> <a href="index79c1.html?q=aggregator&amp;from=600">31</a> <a href="indexdda5.html?q=aggregator&amp;from=620">32</a> ...</div><div><a href="index25b4.html?q=aggregator&amp;from=560">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=540 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:39:20 GMT -->
</html>
