<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=200 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 21:41:47 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index66e2.html?q=node/119">54(6) December 2005</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 2 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>February 9, 2016</h3>
<div class="news-item">
 <div class="date">23:26</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139037686836">Conference: Austin_Texas.SocStudyEvolution.Jun17-21</a></div>
  <div class="description">SOCIETY FOR THE STUDY OF EVOLUTION


Get Ready for EVOLUTION 2016!
Our annual meeting with SSB & ASN will be June 17-21, 2016 in Austin,
TX. Registration will open in late Feb. or early March.  In addition to
the standard sessions and symposia, this year will see the introduction
of â€œspotlight sessionsâ€�, organized by young leaders in the field.
SSE&rsquo;s spotlight session, â€œUnderstanding history and process in
rapid diversification with genomic data", is organized by Catherine
Wagner (UWyo).  The spotlight series consists of one 30 min, five 15
min, and twelve 5 min (â€œlightningâ€�) talks. If you are interested in
speaking in this series please submit an application to Catherine
Wagner (catherine.wagner@uwyo.edu).

The application must include: (1) Name, (2) Institution, (3) rank
(e.g., student, pdf, faculty), (4) names & institutions of
collaborators (5) a short abstract (max: 300 words), and (6) which talk
types you are willing to give (30, 15, or 5; the more flexible you are
the more likely you are to be selected). Applications must be received
by February 28, 2016. No financial support is provided to selected
speakers (i.e., speakers must pay their own travel, accommodation,
registration, etc.). You may still register to give a standard talk in
an â€œopen sessionâ€� at the Evolution Meetings even if you apply to
participate in a spotlight session. However, if you are selected to
speak in the spotlight series, you will have to choose which talk slot
to use as each attendee is allowed to give only one oral presentation.

LIFE DISCOVERY CONFERENCE
The dynamic and changing worlds of biology and education are
converging in the discovery, learning and practice of science. The Life
Discovery - Doing Science Conference seeks to create connections that
inspire Biology in Action. Now in its third year, the conference
(sponsored by SSE and other scientific societies) offers more than 30
hands-on workshops and presentations, networking sessions and
two keynotes by Andrew Martin from UC Boulder and Kevin Coyle from the
National Wildlife Federation that share ideas and strategies to engage
your students in discovering life on earth.  Register by March 4th.
Colleagues from all disciplines seeking to connect educational
approaches with the rich advances of Biology are encouraged to attend.

We&rsquo;re anticipating 2000+ attendees, over 1000 talks, hundreds of
posters, and a variety of workshops and other special events!  Please
visit the EVOLUTION 2016 site for further information about the
meeting, field trips, and more.

AWARD DEADLINES
The Theodosius Dobzhansky Prize is awarded annually by the Society for
the Study of Evolution to recognize the accomplishments and future
promise of an outstanding young evolutionary biologist. The Dobzhansky
Prize is accompanied by a check for U.S. $5000.  The candidate must
have a Ph.D. (or equivalent) awarded no earlier than May 2012 and no
later than 1st February 2016, and must be actively involved in research
in the field of evolutionary biology. Candidates may apply directly or
may be nominated.    The application/nomination materials must be sent
as PDF e-mail attachments is 15 February 2016.  All materials should be
sent to the SSE Secretary (Cynthia Weinig) at cweinig@uwyo.edu.

The W.D. Hamilton Award for Outstanding Student Presentation will be
given to a student who has presented an outstanding talk at the annual
meeting. Finalists will present their papers during a day-long
symposium of Hamilton Award candidate talks.  This year, finalists will
receive a $500 travel stipend from the SSE to attend the Evolution
meeting.  Graduate students must indicate their interest to compete by
checking a box for the Hamilton Award during meeting registration.
Submit a 400-word abstract and talk title as part of the meeting
registration process by 11:59 p.m. on APRIL 30. Finalists will be
notified by May 15.

Thank you for your continued work and support of our mission.
Best wishes,
Kimberly Hughes, SSE President
[3c37ac39-671f-40fb-aa19-f895fab2b20c.jpg]
SSE Business Office
PO Box 299  |  St. Louis Missouri, 63166
314-577-9566
<a href="http://bit.ly/1cwLiKE">http://bit.ly/1cwLiKE</a>

friend on Facebook | forward to a friend
Copyright Â© 2016 Society for the Study of Evolution, All rights
reserved.
You are receiving this communication as a member of the Society for the
Study of Evolution.
Our mailing address is:
Society for the Study of Evolution
PO Box 299
St Louis, MO 63166
Add us to your address book
 unsubscribe from this list | update subscription preferences


Society for the Study of Evolution 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:32</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138984041561">Graduate position: UKentucky_Lexington.Insects</a></div>
  <div class="description">PhD Assistantship in Insect Overwintering Biology
Department of Entomology 
University of Kentucky, Lexington, KY

Contact: 
Nicholas Teets
Assistant Professor
Email: n.teets@uky.edu
Phone: (859)-257-7459
Lab website: <a href="http://bit.ly/1SFBYrx">http://bit.ly/1SFBYrx</a>

Description: The Insect Stress Biology Lab at University of Kentucky is
seeking a highly motivated individual for a PhD in insect evolutionary
physiology and functional genomics. The successful applicant will
conduct research on the cell and molecular adaptations that drive the
evolution of cold tolerance insects. We are particularly interested in
the mechanism of rapid phenotypic plasticity at low temperature. Research
is conducted on both model insects like Drosophila melanogaster, and
non-model stress-adapted species like Antarctic arthropods. For more
information on the Insect Stress Biology Lab, visit the lab website
(<a href="http://bit.ly/1Q4XZcs">http://bit.ly/1Q4XZcs</a>).

Department: The Department of Entomology at University of Kentucky
offers excellent graduate training in diverse areas of entomology. The
Entomology graduate program is ranked in the top 10 nationally, and
it is consistently rated as one of the most productive programs at
the University of Kentucky, measured by the total number of student
publications and presentations. Students from our department go on to
have successful careers in a variety of sectors, including academia,
industry, government science, and extension, to name a few.

Qualifications: Qualified candidates should possess at least a
bachelors degree in entomology, biology, or a related field. Preference
will be given to students with previous research experience,
either through completion of a Masters degree or undergraduate
research. Desired qualities include the ability to work well in a
team, excellent written and oral communication skills, and attention to
detail. Information on graduate admissions at University of Kentucky can
be found at: <a href="http://bit.ly/1SFBYrz">http://bit.ly/1SFBYrz</a>

Start Date: The successful applicant will ideally start either summer
or fall, 2016.

Application Procedures: Interested applicants should submit 1)
a CV (including GPA and GRE scores), 2) a cover letter detailing
research experience, interests, and career goals, and 3) the
name and contact information for three professional references to
n.teets@uky.edu. Screening will begin immediately and continue until
the position is filled.


&ldquo;Teets, Nicholas M&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:30</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138982539606">Job: SanDiegoZoo.ConservationResearch</a></div>
  <div class="description">The Genetics Division of the San Diego Zoo Institute for Conservation
Research has two open positions for Research Associates to work in our
cell culture laboratory and assist with development and utilization of our
Frozen Zoo. This collection includes approximately 10,000 accessions
of early passage diploid fibroblast cultures that represent a crucial
resource for research and recovery of endangered species.

We are seeking applicants with excellent skills in sterile technique
and experience in cell culture methodologies, including establishing
and maintaining primary cell cultures. One position will also involve
assisting with production of induced pluripotent stem cells to support
programs in genetic rescue of endangered species.

A job description and application instructions can be found at:
<a href="http://bit.ly/1TPFz5f">http://bit.ly/1TPFz5f</a>

Applicants will be selected for interviews from the submissions through
this site.

Applicants should also send a letter of interest and a CV to
kbade@sandiegozoo.org attn: Genetics Division Positions

We are excited to be offering these challenging positions that will
continue our record of accomplishments in banking viable cell cultures
from over 1000 vertebrate taxa to date and take our conservation efforts
into new directions.

Thank you for sharing this information with your colleagues and any
individuals potentially interested in applying.

Oliver Ryder

&ldquo;Ryder, Oliver&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:50</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138981615971">Workshop: AuburnU.Bioinformatics.Jun6-10</a></div>
  <div class="description">Bioinformatics Bootcamp at Auburn University

>From June 6V10, 2016 the College of Science and Mathematics will
host the third annual Bioinformatics Bootcamp X a training workshop
in genomics and computational biology for researchers interested in
working with modern genomic data resources.  The Bootcamp is a week-long
immersion experience in the command line, scripting, data handling and
software skills required to process and analyze data from high-throughput
sequencing experiments. Previous experience in bioinformatics is not
required; interactive lectures and group data analysis activities will
introduce participants to best practices in sequencing experiment design
and analysis including widely used methods in genome and transcriptome
assembly, annotation, differential expression and variant analysis.

The 2016 Bootcamp will feature a keynote lecture by Dr. Monica
Munoz-Torres, Lead Biocurator and Bioinformatics Analyst of the Berkeley
Bioinformatics Open-source Projects (BBOP), Lawrence Berkeley National
Laboratory.

The Bootcamp registration fee is $500. To apply, send a brief
statement of interest and two-page CV (as a single pdf) to
bioinformatics@auburn.edu
by April 1, 2016. Additional information can be found at
<a href="http://bit.ly/1O0WsCY">http://bit.ly/1O0WsCY</a>

Kenneth Halanych 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:04</div>
 <div class="body">
  <div class="title"><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a></div>
  <div class="source">Source: <a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></div>
 </div>
</div>
<h3>February 8, 2016</h3>
<div class="news-item">
 <div class="date">06:24</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/tree-based-measure-of-species-longevity/610">Tree-based measure of species longevity?</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/rutgeraldo">@rutgeraldo</a> wrote:</p>
            
              <p>Hi all,</p>

<p>does anyone know of a reasonable way to come up with estimates of species longevity within clades? By which I mean, how long a species / lineage sticks around before it either goes extinct or speciates?</p>

<p>The general idea is that one might have a (near?) complete, dated phylogeny with a bunch of genera in it and that the method I'm after would get me an estimate for average species longevity for each genus.</p>

<p>I guess you could say, crudely, that average species longevity is more or less proportional to average interior branch length (if we pretend that a species is an internode) but when extinction rates are high we'd overestimate, especially for the older internodes.</p>

<p>Isn't there some clever R package for this?</p>

<p>Thanks,</p>

<p>Rutger</p>

<p>p.s. apologies for the ill thought-out question. Equally ill thought-out answers and opinions are very welcome.</p>
            
            <p>Posts: 6</p>
            <p>Participants: 3</p>
            <p><a href="http://www.phylobabble.org/t/tree-based-measure-of-species-longevity/610">Read full topic</a></p></div>
  <div class="source">Source: <a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></div>
 </div>
</div>
<h3>February 7, 2016</h3>
<div class="news-item">
 <div class="date">16:31</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2016/02/the-network-of-woodpeckers-etc.html">The network of woodpeckers, etc</a></div>
  <div class="description"><br />The world continues on its merry way, searching for fragments of the Tree of Life. That is, research papers continue to be published that give no credence to the possibility of reticulate evolutionary history, especially in zoology.<br /><br />A recent case in point is this one:<br />Matthew J. Dufort (2016) An augmented supermatrix phylogeny of the avian family Picidae reveals uncertainty deep in the family tree. <a href="http://www.sciencedirect.com/science/article/pii/S1055790315002560" target="_blank">Molecular Phylogenetics and Evolution</a> 94: 313–326.The author constructed a supermatrix for 26 loci for 78 taxa of the bird family containing woodpeckers, piculets, and wrynecks. The author used an array of phylogenetic techniques, including the construction of maximum-likelihood "gene" trees, several different maximum-likelihood species trees, plus time trees. All of these methods pre-suppose that the evolutionary history of the species was strictly tree-like.<br /><br />We can use an exploratory data analysis to evaluate how probable is this fundamental assumption. I constructed a SuperNetwork based on the 26 gene trees produced by the author, using the SplitsTree program, as shown here. The network is basically tree-like, with one major exception.<br /><br /><a href="https://4.bp.blogspot.com/-qcKKyxccSi8/VrcxpENPDvI/AAAAAAAAD44/qHA25XbfKn0/s1600/LocusTreesEdited.gif" imageanchor="1"></a><br />I have labeled only one taxon, which seems to be the culprit for the major non-tree-likeness. This species appears in 10 of the gene trees, being part of the <i>Leiopicus</i> group (as expected) in 7 of the rooted trees. It is unexpectedly related to the <i>Picus</i> group in two of the other rooted trees, and is close to <i>Dryocopus</i> in the remaining tree.<br /><br />This network EDA does not, of course, imply the existence of reticulate evolution. It does, however, highlight a pattern of incongruence that requires explanation, if the history of these birds is to fully elucidated. Reticulate evolution remains one of the possible explanations, pending further investigation.<br /><br />For an example of a network view of bird evolution, see:<br />Antonio Hernandez-Lopeza, Didier Raoult, Pierre Pontarotti (2013) Birds perching on bushes: networks to visualize conflicting phylogenetic signals during early avian radiation. <a href="http://www.sciencedirect.com/science/article/pii/S1631068313000419" target="_blank">Comptes Rendus Palevol</a> 12: 333-337. </div>
  <div class="source">Source: <a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></div>
 </div>
</div>
<h3>February 6, 2016</h3>
<div class="news-item">
 <div class="date">08:07</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/treebase-legacy-ids/609">TreeBase legacy IDs</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/max">@max</a> wrote:</p>
            
              <p>Dear babblers,</p>

<p>Does anyone know how to search <a href="https://treebase.org/treebase-web/home.html" rel="nofollow">treebase</a> for legacy matrix IDs? I'm trying to get some data sets from older [read 2012] papers but the records have changed and I can't search for the original (legacy) matrix ID.</p>

<p>Cheers,</p>

<p>Luiz</p>
            
            <p>Posts: 6</p>
            <p>Participants: 2</p>
            <p><a href="http://www.phylobabble.org/t/treebase-legacy-ids/609">Read full topic</a></p></div>
  <div class="source">Source: <a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:04</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138779784866">Postdoc: SpelmanC_Atlanta.MicrobialEvolution</a></div>
  <div class="description">Postdoc Position in: Microbial Evolution, Behavior and Genomics

Location: Atlanta, Georgia, USA

Funding Source: CURE Postdoctoral Fellowships [Spelman College is
piloting a postdoctoral fellowship program that combines a traditional
principal-investigator directed postdoctoral research experiences with
mentored teaching experiences.]


A postdoc position (2.5 years) is available to conduct experimental and
computational research on small eukaryotic amoeboid microbes relating
to their evolution and sexual life cycle behavior. Candidates should
have an inter-disciplinary training or familiarity with molecular
(e.g. next generation sequencing) and computational (bioinformatics,
phylogenetics) skills.

Potential research topics include (i) genomics and evolution of phenotypic
traits in cryptic species, (ii) evolution of sexual-like behavior and
development in amoeboids, (iii). origin, evolution and biodiversity
of amoeboids using large-scale analysis. (iv). evolution of amoeboid
movement and cytoskeleton architecture. Additional projects under
similar topics can be also considered based on individual experience
and interest. For additional information contact: ytekle@spelman.edu

This position offers competitive salary and benefit. Candidates should
complete Ph.D. training before starting this position. This position
is open to US citizens and lawful permanent residents only. Screening
of applicants will begin immediately and continue until the position
is filled.

Spelman College is a private four-year liberal arts college located in
Atlanta, GA, and ranked one of the top 100 liberal arts colleges by the
US News.

Please email a copy of curriculum vitae, a short statement of research
interest, names and contact information of at least three individuals
familiar with your research to Dr. Yonas Tekle through ytekle@spelman.edu



350 Spelman Ln SW, Box 1183,
Department of Biology, Spelman College, Atlanta, GA 30314-4399
LOCATIONS
Office: AFM Science Center 271;  Lab: AFM Science Center 247
Phone: 404-270-5779 (office);  Fax: 404-270-5725


Yonas Isaak Tekle 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>February 5, 2016</h3>
<div class="news-item">
 <div class="date">23:49</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138777169671">Course: Switzerland.TamingTheBeast.Jun26-Jul1</a></div>
  <div class="description">Dear colleagues,

We are happy to announce our summer school &ldquo;Taming the BEAST&rdquo; and would
kindly ask you to forward this email to interested students and
PostDocs. We apologize if you receive this email more than once.
Phylogenetics and phylodynamics are central topics in modern biology.
Phylogenetic inferences reconstruct the evolutionary relationships
between organisms, whereas phylodynamic inferences reveal the dynamics
that lead to the observed relationships. These two fields have many
practical applications in disciplines such as epidemiology,
developmental biology, paleontology, ecology and even linguistics.
However, phylogenetics and phylodynamics are complex and fast-evolving
fields. As such, inference tools are not easily accessible to
researchers who are not from a computational background.

Taming the BEAST is a summer school focusing on the BEAST2 software and
consisting of a mix of invited talks, lectures and hands-on tutorials
by leading and renowned experts in the field (including several of the
core developers of BEAST2). The aim of this summer school is to equip
participants with the skills necessary to confidently perform their own
phylogenetic and phylodynamic inferences in Bayesian settings, while
providing them with a firm grasp of the theory behind those
inferences. Participants are also highly encouraged to bring their own
datasets along and to engage with the organizers and speakers to
address any problems specific to their own datasets/analyses.
Registration is now open! We welcome applications from graduate
students and early-career scientists in the life sciences. Preference
will be given to applicants who are not from a computational background
and applicants who have already collected/assembled a dataset that they
need to analyze.

Invited speakers:

Tanja Stadler (ETH Zurich)

Alexei Drummond (University of Auckland)

Tracy Heath (Iowa State University)

Tim Vaughan (University of Auckland)

Oliver Pybus (University of Oxford)

Dates: June 26th to July 1st  (Registration deadline: March 15th).

Place: Engelberg, Switzerland

Registration Fee: 650 CHF for students and postdocs, 1000 CHF for
industry. (Registration fee includes accommodation and meals).

For more information regarding the program, registration,
venue, etc, please visit the summer school website:
<a href="http://bit.ly/1USzh3b">http://bit.ly/1USzh3b</a>

We hope to see you there,
the Taming the BEAST organizing team

Best regards,
Venelin Mitov
Department of Biosystems Science & Engineering
Mattenstrasse 26
4058 Basel
Switzerland
Email: venelin.mitov@bsse.ethz.ch
Phone: +41 61 38 73411
Office: 7.48

Mitov  Venelin 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:57</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138720407016">Postdoc: ToulouseCNRS.WildPlantQuantitativeGenetics</a></div>
  <div class="description"><p>This is a multi-part message in MIME format.
via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:30</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/summer-school-announcement-taming-the-beast-switzerland-june-2016/608">Summer school announcement: Taming the BEAST, Switzerland, June 2016</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/venelin_mitov">@Venelin_Mitov</a> wrote:</p>
            
              <p>Dear colleagues,</p>

<p>We are happy to announce our summer school "Taming the BEAST" and would kindly ask you to forward this email to interested students and PostDocs. We apologize if you receive this email more than once.</p>

<p>Phylogenetics and phylodynamics are central topics in modern biology. Phylogenetic inferences reconstruct the evolutionary relationships between organisms, whereas phylodynamic inferences reveal the dynamics that lead to the observed relationships. These two fields have many practical applications in disciplines such as epidemiology, developmental biology, paleontology, ecology and even linguistics. However, phylogenetics and phylodynamics are complex and fast-evolving fields. As such, inference tools are not easily accessible to researchers who are not from a computational background.</p>

<p><strong>Taming the BEAST</strong> is a summer school focusing on the BEAST2 software and consisting of a mix of invited talks, lectures and hands-on tutorials by leading and renowned experts in the field (including several of the core developers of BEAST2). The aim of this summer school is to equip participants with the skills necessary to confidently perform their own phylogenetic and phylodynamic inferences in Bayesian settings, while providing them with a firm grasp of the theory behind those inferences. Participants are also highly encouraged to bring their own datasets along and to engage with the organizers and speakers to address any problems specific to their own datasets/analyses.</p>

<p><strong>Registration is now open!</strong> We welcome applications from graduate students and early-career scientists in the life sciences. Preference will be given to applicants who are not from a computational background and applicants who have already collected/assembled a dataset that they need to analyze.</p>

<p><strong>Invited speakers:</strong>
Tanja Stadler (ETH Zurich)
Alexei Drummond (University of Auckland)
Tracy Heath (Iowa State University)
Tim Vaughan (University of Auckland)
Oliver Pybus (University of Oxford)</p>

<p><strong>Dates:</strong> June 26th to July 1st  (Registration deadline: March 15th).</p>

<p><strong>Place:</strong> Engelberg, Switzerland</p>

<p><strong>Registration Fee:</strong> 650 CHF for students and postdocs, 1000 CHF for industry. (Registration fee includes accommodation and meals).</p>

<p>For more information regarding the program, registration, venue, etc, please visit the summer school website: <a href="https://www.bsse.ethz.ch/cevo/taming-the-beast.html" rel="nofollow">https://www.bsse.ethz.ch/cevo/taming-the-beast.html</a></p>

<p>We hope to see you there,</p>

<p>the Taming the BEAST organizing team</p>
            
            <p>Posts: 1</p>
            <p>Participants: 1</p>
            <p><a href="http://www.phylobabble.org/t/summer-school-announcement-taming-the-beast-switzerland-june-2016/608">Read full topic</a></p></div>
  <div class="source">Source: <a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></div>
 </div>
</div>
<h3>February 4, 2016</h3>
<div class="news-item">
 <div class="date">23:27</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138715638391">Postdoc: CharlesU_Prague.NightingaleSpeciation</a></div>
  <div class="description"><p>Post doc position at Charles University in Prague

Genomics of speciation in nightingales

We are looking for a post-doc trained in bioinformatics and/or
population genetics. The main goal of the project is to elucidate the
genetic basis of reproductive isolation between two song bird species,
the Common Nightingale (Luscinia megarhynchos) and the Thrush
Nightingale (L. luscinia). The species hybridize in a secondary
contact zone comprising F1 as well as backcross hybrids. According to
Haldane’s rule F1 hybrid females are sterile, while F1 hybrid males
are fertile. Using ddRAD sequencing of natural hybrid population and
transcriptome sequencing of gonads and somatic tissues of both species
and hybrids we aim to identify candidate reproductive isolation loci
responsible for hybrid female sterility. The results should provide
the first insight into genetic basis of Haldane’s rule in
heterogametic female organisms.

The candidate will be mainly responsible for the bioinformatic part of
the project. Samples as well as most sequence data are already
available. The position is available from May 2016 to December 2017
with the possibility of extension. The starting date is flexible but
preferable before September 2016. The candidate will work at the
Department of Zoology at Charles University in Prague under the
supervision of Radka Reifova (<a href="http://bit.ly/1L2eCEl">http://bit.ly/1L2eCEl</a>). The
salary will be 35,000 CZK (ca 1,300 EUR) (Note that living expenses in
the Czech Republic are generally lower than in Western European
countries.)

If interested, please, send (1) a CV including a list of publications,
(2) motivation letter and (3) a letter of reference of your PhD
supervisor or research advisor to Radka Reifova
(radka.reifova@natur.cuni.cz) by 29th February 2016.


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">16:25</div>
 <div class="body">
  <div class="title"><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12143">Morphology disentangles the systematics of a ubiquitous but elusive meiofaunal group (Kinorhyncha: Pycnophyidae)</a></div>
  <div class="description">Kinorhyncha is a group of benthic, microscopic animals distributed worldwide in marine sediments. The phylum is divided into two classes, Cyclorhagida and Allomalorhagida, congruent with the two major clades recovered in recent phylogenetic analyses. Allomalorhagida accommodates more than one-third of the described species, most of them assigned to the family Pycnophyidae. All previous phylogenetic analyses of the phylum recovered the two genera within Pycnophyidae, Pycnophyes and Kinorhynchus, as paraphyletic and polyphyletic. A major problem in these studies was the lack of molecular data of most pycnophyids, due to the limited and highly localized distribution of most species, often in the Arctic and the deep-sea. We here overcame the problem by adding a morphological partition with data for 79 Pycnophyidae species, 15 of them also represented by molecular data. Model-based analyses yielded seven clades, which each was supported by several morphological apomorphies. Accordingly, Kinorhynchus is synonymized with Pycnophyes and six new genera are described for the remaining recovered clades: Leiocanthus gen. nov., Cristaphyes gen. nov., Higginsium gen. nov., Krakenella gen. nov., Setaphyes gen. nov. and Fujuriphyes gen. nov.</div>
  <div class="source">Source: <a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:12</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138658057506">Conference: Lyon.BehaviourEvolution.Jun27</a></div>
  <div class="description">We inform you that the twelfth edition of the &ldquo;ECOLOGY & BEHAVIOUR&rdquo;
meetings will occur this year in Lyon (France) between June 27th and July
1st 2016.

The &ldquo;Ecology & Behaviour&rdquo; conference is an international scientific
congress in the themes of ecology, evolutionary biology, and animal
behaviour that distinguishes itself by the will clearly displayed to favour
the participation of PhD and Post-doctoral students.

Unlike most congresses and symposium, NO CHARGE is asked to PhD students
and post-doctoral students (including accommodation and lunches) if they
participate through the presentation of a poster or an oral communication
(in English).



We are now accepting abstracts for graduate students and post-docs
presentations in talk and poster formats. Please visit our presentation
registration page for more information on submission requirements and
procedures:



eb2016.sciencesconf.org



*The deadline for abstract submissions is March 20th, 2016*.



The E&B organization committee
eb2016.sciencesconf.org

Christophe Plantamp 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>February 3, 2016</h3>
<div class="news-item">
 <div class="date">08:55</div>
 <div class="body">
  <div class="title"><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a></div>
  <div class="description"><p lang="en" dir="ltr"><a href="https://twitter.com/sckottie">@sckottie</a> <a href="https://twitter.com/rdmpage">@rdmpage</a> <a href="https://twitter.com/christgendreau">@christgendreau</a> Three small, early adopting projects should show expected network of synergies if JSON-LD actually useful</p>&mdash; David Shorthouse (@dpsSpiders) <a href="https://twitter.com/dpsSpiders/status/694243195585175554">February 1, 2016</a>  <p>In a recent Twitter conversation including David Shorthous and myself (and other poor souls who got dragged in) we discussed how to demonstrate that adopting JSON-LD as a simple linked-data friendly format might help bootstrap the long awaited "biodiversity knowledge graph" (see below for some suggestions for keeping JSON-LD simple). David suggests partnering with "Three small, early adopting projects". I disagree.</p> <p>I think we need to approach this problem, not from the perspective of who would like to try this approach, but what would it take to really be useful? I think if we look at the structure of the biodiversity knowledge graph and look at who has what identifiers, we can gain some insight into what the next steps are.</p> Sinks <p>A fundamental problem is that many data providers are essentially dead ends in terms of building a network. They have data, but no connections, and so they are "sinks". A data browser goes there and can't go any further, it has to retrace its steps and go somewhere else. For example, your typical museum might serve up its collection data like this: The museum has its own identifiers (e.g., a URL) and that's the only identifier in the data. Nomenclators are much the same: You get a nomenclator-specific identifier such as an LSID, but no other identifiers. Once again, if you are a data-crawler traversing the web of data, this is a dead end. (This is why I've become obsessed with linking nomenclators to the primary literature, I want to stop them being dead ends).</p> <p>Connected sourcesThen there are sources which have at least one external identifier, that is, an identifier that they themselves don't control. For example, CrossRef manages DOIs for articles, but if you get metadata for a DOI you also get an ISSN (identifying a journal): Now we have a connection to an external source of data, and some we can traverse the data graph further, which means we can start asking questions (such as how many articles in this journal have DOIs?).</p> <p>Another connected source is <a href="http://orcid.org/">ORCID</a>: This gives us identifiers for authors (the ORCID) linked to article identifiers such as DOIs and PubMed ids (PMID). Follow the DOI to CrossRef and we can link people to articles to journals. </p> <p>Another connected source is the <a href="http://www.ncbi.nlm.nih.gov/">NCBI</a>: NCBI has several internal identifiers (PMIDs, GenBank accession numbers, tax_ids) all of which lead to rich resources, and it's possible to get a lot of information by staying within NCBI's own silo, but there are external links such as DOIs (usually found attached to PubMed articles or to GenBank accessions) and links to external records such as DNA barcodes in <a href="http://www.boldsystems.org/">BOLD</a>.</p> <p>Lets join CrossRef, ORCID, and NCBI together: Now we have a <a href="https://en.wikipedia.org/wiki/Bipartite_graph">bipartite graph</a> linking sources with identifiers. We can imagine playing a game where we try and connect different entities by moving through the graph. For example, we could take an author identified by an ORCID, follow a DOI to a PMID, a PMID to a set of accessions numbers, and then be able to list all the sequences that an author has published. We could also step back and ask questions about which identifiers are the most useful in terms of making connections between different sources, and which sources provide the most cross links.</p> <p>Filling in the gapsNow, the graph above is obviously incomplete. I've restricted it to a few of the key services that I'm familiar with, and make use of external identifiers. One of the big obstacles to fleshing out the biodiversity knowledge graph is the frequent lack of reuse of identifiers. It's not enough to pump out data in linked-data form, you need to build the links. And not just "same as" style links connecting multiple identifiers for the same thing, you need to <b>connect identifiers for different things</b>. Until we tackle that, linked data approaches will not deliver much in the way of value. Hence we need sources that provide genuinely linked data by reusing exiting, external identifiers. If we have name in a nomenclator with a citation string, it needs to link that dumb literature string to a DOI. If we have a taxon concept, it needs to link the name of that taxon to a name in a nomenclator. If we have a specimen that has been sequenced, it needs to give the accession number of that sequence.</p> <p>From this perspective, choosing which kinds of data with which to explore JSON-LD and a linked data graph should be driven by how connected those sources are: the more connected the more interesting the questions that we can ask. Sadly the vast majority of biodiversity data providers don't provide the kind of connected data we need, which means we continually pay lip service to linked data without feeding it the kind of data it needs in order to grow.</p>  <p>* Notes on JSON-LDIt is relatively easy to write horrible JSON-LD, so I think it would be useful to strive to make it as simple and as human-readable as possible (ignoring the <a href="https://www.w3.org/TR/json-ld/#the-context">@context block</a> which is always going to be awful, this is the price we pay for simplicity elsewhere). To this end I think we should do at least the following:</p> <ol><li><b>No URLs as identifiers</b> URLs are ugly, and they are not reliable as identifiers. Providers can change URLs, even for persistent identifiers. The DOI prefix has changed from http://dx.doi.org/ to the preferred http://doi.org/, and the rise of HTTPS (prompted by concerns about security, among other issues, see <a href="https://medium.com/@davewiner/google-s-motives-part-2-498b33d94fc8#.q1g4o42hf">Google's motives, part 2</a>) is going to break a lot of older URLs (see <a href="http://www.w3.org/DesignIssues/Security-NotTheS.html">Web Security - "HTTPS Everywhere" harmful</a>). These changes keep happening, so lets try and shield ourselves from these by using standard prefixes for identifiers (such as 'DOI') for a DOI. Put all the alternative URL resolver prefixes in the @context block and use prefixes such as those standardised by <a href="http://identifiers.org/">http://identifiers.org</a> (which basically formalises what many people in bioinformatics have been doing for a while).</li><li><b>No prefixes for keys</b> In other words, no <a href="http://www.w3.org/TR/curie/">CURIEs</a>. Don't write "dwc:CatalogueNumber", just write "CatalogueNumber". I don't want to be told where the term comes from (that's what @context is for). If you have a namespace clash (i.e., two terms that are the same unless you include the namespace) then IMHO you're doing it wrong. Either you're using more than one vocabulary for the same thing (why?), or you've not modelling the data at the appropriate level of granularity. Either way, let's avoid clutter and keep things simple and readable.</li></ol></div>
  <div class="source">Source: <a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:30</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138594669601">Postdoc: Kiel_Germany.1PDF_3PhD.CelegansEvolution</a></div>
  <div class="description">Kiel University, Germany

    1 Postdoc + 3 PhD positions on the evolution and genetics of C.
                    elegans-microbiota interactions

The positions are funded within a larger research initiative on
host-microbiota interactions at Kiel University (Collaborative Research
Center CRC1182 â€œOrigin and function of metaorganismsâ€�).

The postdoc and one PhD position are supervised by Hinrich Schulenburg
(projects A1.1 and A4.3) and address the evolutionary dynamics and
genomics of C. elegans microbiota interactions, based on evolution
experiments and genome sequencing (Schulenburg group).

The second PhD project (project A1.2) is supervised by Katja Dierking
(Schulenburg lab) and looks at the transcriptomic and also genetic
basis of the interaction, using RNASeq and functional genetic analysis.

The third PhD project (project A1.3) is supervised by Matthias Leippe
and assesses the involvement of antimicrobial peptides and proteins in
shaping C. elegans-bacteria interactions using biochemical and
functional genetic approaches (Leippe group).

For further information, please check the webpage of the Collaborative
Research Center (<a href="http://bit.ly/1PfXkWc">http://bit.ly/1PfXkWc</a>
) or the Schulenburg group (which includes Katja Dierking;
<a href="http://bit.ly/1RZzweV">http://bit.ly/1RZzweV</a>) or the Leippe group
(<a href="http://bit.ly/1PfXkWe">http://bit.ly/1PfXkWe</a>).

Application procedures are described on the webpage of the Collaborative
Research Center (see above link; then under Jobs).  Deadline for
applications is 29 February 2016. If you have questions, then please
contact either Hinrich Schulenburg (hschulenburg@zoologie.uni-kiel.de),
Katja Dierking (kdierking@zoologie.uni-kiel.de), or Matthias Leippe
(mleippe@zoologie.uni-kiel.de).

Hinrich Schulenburg

Zoological Institute
Christian-Albrechts-Universitaet zu Kiel
Am Botanischen Garten 1-9
24118 Kiel
Germany
Tel: +49-431-880-4143/4141
Fax: +49-431-880-2403
Email: hschulenburg@zoologie.uni-kiel.de
Web: <a href="http://bit.ly/1RZzweV">http://bit.ly/1RZzweV</a>

Hinrich Schulenburg 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>February 2, 2016</h3>
<div class="news-item">
 <div class="date">02:25</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138535098621">Postdoc: NotreDame_BroadInst.MosquitoSpeciationGenomics</a></div>
  <div class="description">POSTDOC: Species boundaries and evolutionary history of a cryptic radiation
of malaria vectors

A Postdoctoral position funded by a 2-year NIH grant is available
April 2016 to work between two population genomics groups: that of Nora
Besansky at the University of Notre Dame (<a href="http://ntrda.me/1huzbgB">http://ntrda.me/1huzbgB</a>),
and that of Daniel Neafsey at the Broad Institute of MIT and Harvard
(<a href="http://bit.ly/20DuyG3">http://bit.ly/20DuyG3</a>).

We are seeking highly qualified and enthusiastic applicants with a
background in population genetics and evolution, to analyze whole genome
sequence data and infer speciation history in the *Anopheles funestus*
complex.

BACKGROUND: Major malaria vectors are a rarity in the mosquito genus
Anopheles, and they are not phylogenetically clustered.  On the contrary,
malaria vectors are almost invariably members of sibling species complexes
containing minor or non-vector species, indicating that the traits endowing
major vector status have evolved recently, rapidly and repeatedly in
different species complexes.  Powerful insights about the genetic basis of
vector traits can be gained by comparative genomics of vectors and
non-vectors across these replicate systems of vector evolution.  A crucial
first step toward that end was taken with the assembly of 16 *Anopheles*
genomes led by Besansky at Notre Dame and Neafsey at the Broad Institute,
with the *Anopheles gambiae* complex as one case study (Neafsey et al 2015
Science; Fontaine et al 2015 Science).  A next step is to consider other
lesser known but equally important sibling species complexes.  *Anopheles
funestus* is one of the three most important and widespread vectors of
human malaria in tropical Africa, but unlike *An. gambiae*, it is
understudied.  This major vector conceals a group of close relatives that
are morphologically similar or identical as adults.  The taxonomic
complexity of this group is at least as high as the *An. gambiae* complex;
within each named (and unnamed) species are genetically heterogeneous
clusters whose interrelationships have never been resolved, and additional
cryptic diversity is being reported by vector control programs.  Moreover,
echoing the situation in the *An. gambiae* complex, there is evidence for
historical or ongoing introgression between An. funestus and at least one
other taxon.

PROJECT: As part of this NIH project, complete de novo reference genome
assemblies will be generated at the Broad for available species in the *An.
funestus* complex, based on the DISCOVAR de novo algorithm.  Additional low
coverage HiSeq data will be generated from natural population samples.  The
successful postdoctoral candidate will use the new reference assemblies and
additional light whole genome sequencing (~12 samples/taxon) to construct
an initial portrait of population genomic patterns and species
relationships for the *An. funestus* complex. Although the position is
based at Notre Dame, the postdoc will have the opportunity to split time
between Notre Dame and Broad, benefitting from interactions with active
research groups in both stimulating locations.

REQUIREMENTS: (1) A PhD in genetics/genomics, evolutionary biology, or a
related discipline; (2) a strong background in statistics and population
genetics; (3) good communication skills and proficiency in English, both
speaking and writing.

PREFERENCES: The ideal candidate will be a highly motivated individual with
(i) proficiency in the Linux/Unix computing environment, familiarity with
R, and at least one other programming language (e.g. Python or Perl); (ii)
experience with the analysis of high throughput biological data sets; and
(iii) an excellent record of scientific publication.

HOW TO APPLY: Applicants should submit a single PDF file (
tofunestus.postdoc@gmail.com) containing (1) a cover letter briefly
summarizing past research accomplishments and future goals; (2) a CV; (3)
the names and contact information for three references. Screening of
applicants will begin immediately and continue until the position is
filled.

ADDITIONAL INFORMATION: The University of Notre Dame is located on the
northern limits of South Bend, Indiana [90 miles (150 km) east of Chicago].
Post-doctoral scholars in the Department of Biological Sciences
(biology.nd.edu) have access to outstanding faculty, cutting-edge research
infrastructure, and various avenues for career development. Research in
the department spans the wide spectrum of the life sciencesXfrom cells
and organs to whole organisms and ecosystems. This uncommon level of
wide-ranging yet integrative biological inquiry provides an extraordinary
environment for learning, innovation and collaboration. Centers and
Institutes enhance our intellectual mission, and include the Eck
Institute for Global Health, Center for Rare and Neglected Diseases,
and the Environmental Change Initiative.  Available resources include
the Freimann Animal Facility and state of the art core facilities:
Genomics, Proteomics, and Integrated Imaging. The Center for Research
Computing operates a High Performance Computing (HPC) facility providing
parallel supercomputers, clusters, grid networks and storage in systems
of various architectures and interconnects, and access to the TeraGrid
and Open Science Grid.

The University of Notre Dame is an Equal Opportunity / Affirmative Action
Employer.

Daniel Neafsey 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>January 31, 2016</h3>
<div class="news-item">
 <div class="date">16:30</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2016/02/tardigrades-and-phylogenetic-networks.html">Tardigrades and phylogenetic networks</a></div>
  <div class="description"><br />In this blog we have always championed the use of Exploratory Data Analysis prior to phylogenetic analyses. This approach explores the characteristics of the data before making formal inferences about possible evolutionary scenarios. One of the reasons for doing this is the possibility of data errors. That is, we need to distinguish between estimation errors deriving from our experimental procedures and real biological scenarios, because both of these will result in complex patterns in our data.<br /><br />One possible classification of the potential causes of complex data patterns in phylogenetics is this:<br /><br /><b>Estimation errors</b><br />(i) incorrect data<br />— inadequate data-collection protocol<br />—&nbsp;poor laboratory / museum / herbarium technique<br />—&nbsp;lack of quality control after data collection<br />—&nbsp;misadventure<br />(ii) inappropriate sampling<br />—&nbsp;distant outgroup<br />—&nbsp;rapid evolutionary rates<br />—&nbsp;short internal branches<br />(iii) model mis-specification<br />—&nbsp;wrong assessment of primary homology<br />—&nbsp;wrong substitution model<br />—&nbsp;different optimality criteria<br /><br /><b>Biological complexity</b><br />(iv) analogy<br />—&nbsp;parallelism<br />—&nbsp;convergence<br />—&nbsp;reversal<br />(v) homology<br />—&nbsp;deep coalescence<br />—&nbsp;duplication–loss<br />—&nbsp;hybridization<br />—&nbsp;introgression<br />—&nbsp;recombination<br />—&nbsp;horizontal gene transfer<br />—&nbsp;genome fusion<br /><br />The scientific literature has a number of prime examples where people have asserted a case of biological complexity that has subsequently been questioned, and attributed to estimation errors instead.<br /><br />For example, many of you will have noted the recent attention given to the release of various genome sequences from the Tardigrades, a group of microscopic animals often alleged to be the world's most resistant to environmental conditions. Two rival papers have appeared:<br />Thomas C. Boothby et al. (2015) Evidence for extensive horizontal gene transfer from the draft genome of a tardigrade. <a href="http://www.pnas.org/content/112/52/15976.abstract" target="_blank">Proceedings of the National Academy of Sciences of the USA</a> 112: 15976–15981.Georgios Koutsovoulos et al. (2015) The genome of the tardigrade <i>Hypsibius dujardini</i>. <a href="http://biorxiv.org/content/early/2015/12/13/033464" target="_blank">BioRxiv preprint 33464</a>.The former paper attributes their observed phylogenetic complexity to horizontal gene transfer (group v in the list above) while the latter attributes it to sequencing errors (group i). This situation is discussed in more detail elsewhere on the web, for example:<br /><a href="http://www.theatlantic.com/science/archive/2015/12/rival-scientists-kill-recent-discovery-about-invincible-animals/418755/" target="_blank">Rival scientists cast doubt upon recent discovery about invincible animals</a><a href="http://scopeweb.mit.edu/articles/how-did-these-indestructible-pond-critters-get-their-genes/" target="_blank">How did these indestructible pond critters get their genes?</a>This difference in possible cause (of complexity) matters particularly for the use of phylogenetic networks, because both estimation errors and biological complexity will appear as reticulation patterns in any network. This is particularly important for the assertion of evolutionary scenarios such as horizontal gene transfer, because usually the <b>only</b> evidence for any such gene flow is the complexity of the phylogenetic network — that is, there is no independent experimental evidence, and we are relying entirely on the phylogenetic pattern analysis. Estimation errors must thus be eliminated prior to the phylogenetic analysis, if we are to produce a high quality network.<br /><br />The current situation potentially has unfortunate consequences. For example, there are continual comments that horizontal gene flow is rare, particularly from zoologists, even though there is a large amount of evidence to the contrary. Situations like the current one can only add fuel to this argument, if strong claims of gene flow turn out to be erroneous. There is no quantitative basis for an assertion that gene flow is rare in zoology — those who have looked for reticulate evolution in animals have found it, and those who haven't haven't.<br /><br />In the end, data-display networks are useful for displaying incongruent data patterns, but the source of the incongruence needs to be identified before these networks are turned into evolutionary networks (either explicitly drawn or verbally implied).<br /><br /></div>
  <div class="source">Source: <a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></div>
 </div>
</div>
<h3>January 30, 2016</h3>
<div class="news-item">
 <div class="date">05:59</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/phylogenetic-placement-visualization/605">Phylogenetic Placement Visualization</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/alexis_raxml">@Alexis_RAxML</a> wrote:</p>
            
              <p>Ivica Letunic's itol3 tool can now visualize phylogenetic placements in .jplace format as generated by RAxML's EPA and pplacer  see <a href="http://itol3.embl.de/" rel="nofollow">itol3</a> You are free to send me feature requests which I will then collect and forward to Ivica,</p>

<p>Alexis</p>
            
            <p>Posts: 1</p>
            <p>Participants: 1</p>
            <p><a href="http://www.phylobabble.org/t/phylogenetic-placement-visualization/605">Read full topic</a></p></div>
  <div class="source">Source: <a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="index52c1.html?q=aggregator&amp;from=180">previous page</a></div><div>... <a href="index3362.html?q=aggregator&amp;from=120">7</a> <a href="index7b13.html?q=aggregator&amp;from=140">8</a> <a href="indexf37c.html?q=aggregator&amp;from=160">9</a> <a href="index52c1.html?q=aggregator&amp;from=180">10</a> <strong>11</strong> <a href="index933b.html?q=aggregator&amp;from=220">12</a> <a href="indexd118.html?q=aggregator&amp;from=240">13</a> <a href="index3f45.html?q=aggregator&amp;from=260">14</a> <a href="index4603.html?q=aggregator&amp;from=280">15</a> ...</div><div><a href="index933b.html?q=aggregator&amp;from=220">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=200 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 21:41:47 GMT -->
</html>
