<?xml version="1.0" encoding="utf-8"?>
<!DOCTYPE rss [<!ENTITY % HTMLlat1 PUBLIC "-//W3C//ENTITIES Latin 1 for XHTML//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml-lat1.ent">]>
<rss version="0.92" xml:base="http://systbio.org.whsites.net">
<channel>
 <title>Society of Systematic Biologists - Published elsewhere</title>
 <link>http://systbio.org.whsites.net/?q=taxonomy/term/4/0</link>
 <description></description>
 <language>en</language>
<item>
 <title>Contemporary and future studies in plant speciation, morphological/floral evolution and polyploidy</title>
 <link>http://systbio.org.whsites.net/?q=node/434</link>
 <description>&lt;p&gt;&lt;img src="http://systbio.org/files/roysoc.png" align="right" width="200"/&gt;The Royal Society has published a special issue celebrates the distinguished scientific career of Leslie Gottlieb, who investigated the evolutionary processes that generate the Earth‚Äôs life-sustaining plant diversity.&lt;br /&gt;
The articles in this issue discuss results, obtained largely from contemporary genetic-genomic approaches, within the context of Gottlieb‚Äôs views on several areas of plant evolution. Collectively, the articles document the importance of Leslie Gottlieb‚Äôs contributions to plant evolutionary biology because the same questions, and in some cases the same plants, that interested him are commanding the attention of contemporary workers employing cutting edge techniques today.&lt;br /&gt;
The issue is online at &lt;a href="http://bit.ly/PTB1648"&gt;bit.ly/PTB1648&lt;/a&gt;, you can also buy a print copy at the same link. Quote code &lt;b&gt;TB 1648&lt;/b&gt; for a discount (¬£35.00 versus usual price of ¬£59.50).&lt;/p&gt;
</description>
 <pubDate>Sun, 28 Sep 2014 04:35:10 -0600</pubDate>
</item>
<item>
 <title>Classical determination of monophyly </title>
 <link>http://systbio.org.whsites.net/?q=node/430</link>
 <description>
&lt;p&gt;Systematists with a yen for theory might be&lt;br /&gt;
interested in a three-part paper:&lt;/p&gt;
&lt;p&gt;Zander, R. H. 2014. Classical determination of monophyly,&lt;br /&gt;
exemplified with &lt;i&gt;Didymodon&lt;/i&gt; s. lat. (Bryophyta). Parts 1, 2 and 3. Phytoneuron issues 2014-78, 2014-79, and 2014-80.&lt;/p&gt;
&lt;p&gt;In which I attempt to formalize (find the statistical and&lt;br /&gt;
logical basis) of the heuristics classical taxonomists use to intuit monophyly.&lt;/p&gt;
&lt;p&gt;We are apparently good at it but can‚Äôt say why. Until now. If I‚Äôm right.&lt;/p&gt;
&lt;p&gt;These may be found on the Phytoneuron Web site:&lt;br /&gt;
&lt;a href="http://phytoneuron.net"&gt;http://phytoneuron.net&lt;/&lt;/p&gt;
</description>
 <pubDate>Sun, 10 Aug 2014 15:30:25 -0600</pubDate>
</item>
<item>
 <title>phyloseminar #33: Daniel Ksepka, NESCent</title>
 <link>http://systbio.org.whsites.net/?q=node/419</link>
 <description>&lt;p&gt;Daniel Ksepka, NESCent&lt;br /&gt;
"Including Fossil Taxa in Phylogenies: Advances and Issues"&lt;br /&gt;
December 3, 2013&lt;br /&gt;
&lt;iframe width="560" height="315" src="http://www.youtube.com/embed/cuEe1iArVVo?rel=0" frameborder="0" allowfullscreen&gt;&lt;/iframe&gt;&lt;/p&gt;
</description>
 <pubDate>Wed, 04 Dec 2013 10:00:54 -0700</pubDate>
</item>
<item>
 <title>Paleovirology: Insights from the genomic fossil record</title>
 <link>http://systbio.org.whsites.net/?q=node/414</link>
 <description>&lt;p&gt;&lt;img src="files/RSTB_368_1626_Thumbnails_web.jpg" align="right" /&gt;Royal Society Publishing has just published &lt;b&gt;Paleovirology: Insights from the genomic fossil record&lt;/b&gt;, compiled and edited by Aris Katzourakis.&lt;br /&gt;
This content can be accessed at &lt;a href="http://bit.ly/ZOA8Ib"&gt;http://bit.ly/ZOA8Ib&lt;/a&gt; A print version is also available at the special price of ¬£35.00. You can order online via the above web page (enter special code TB 1626 when prompted) or, alternatively, you can contact &lt;a href="mailto:debbie.vaughan@royalsociety.org"&gt;debbie.vaughan@royalsociety.org&lt;/a&gt;&lt;/p&gt;
</description>
 <pubDate>Tue, 08 Oct 2013 09:53:11 -0600</pubDate>
</item>
<item>
 <title>A Framework for Post-Phylogenetic Systematics</title>
 <link>http://systbio.org.whsites.net/?q=node/413</link>
 <description>&lt;p&gt;&lt;em&gt;A Framework for Post-Phylogenetic Systematics&lt;/em&gt;, by R. H. Zander, is now available on Amazon. From the back cover:&lt;/p&gt;
&lt;p&gt;The Framework reframes biological systematics to reconcile classical and cladistic schools. It combines scientific intuition and statistical inference in a new form of total evidence analysis developing a joint macroevolutionary process-based causal theory. Discrepancies between classical results and morphological and molecular cladograms are explained through heterophyletic inference of deep ancestral taxa, coarse priors leading to Bayesian Solution of total evidence, self-nesting ladders that can reverse branching order, and a superoptimization protocol that aids in distinguishing pseudoextinction from budding evolution. It determines direction of transformative evolution through Dollo evaluation at the taxon level. The genus as a basic, practical unit of evolution is postulated for taxa with dissilient evolution.&lt;/p&gt;
</description>
 <pubDate>Mon, 30 Sep 2013 07:59:33 -0600</pubDate>
</item>
<item>
 <title>A Philosophical Critique of Philogenetic Systematics</title>
 <link>http://systbio.org.whsites.net/?q=node/362</link>
 <description>&lt;p&gt;Zander, R. H. 2010 [2011]. Structuralism in Phylogenetic Systematics. Biological Theory 5: 383-394. [Abstract:] Systematics based solely on structuralist principles is nonscience because it is derived from first principles that are inconsistent in dealing with both synchronic and diachronic aspects of evolution, and its evolutionary models involve hidden causes, and unnamable and unobservable entities. Structuralist phylogenetics emulates axiomatic mathematics through emphasis on deduction, and "hypotheses" and "mapped trait changes" that are actually lemmas and theorems. Sister-group only evolutionary trees have no caulistic element of scientific realism. This results in a degenerate systematics based on patterns of fact or evidence being treated as so fundamental that all other data may be mapped to the cladogram, resulting in an apparently well-supported classification that is devoid of evolutionary theory. Structuralism in systematics is based on a non-ultrametric analysis of sister-group informative data that cannot detect or model a named taxon giving rise to a named taxon, resulting in classifications that do not reflect macroevolutionary changes unless they are sister lineages. Conservation efforts are negatively affected through epistemological extinction of scientific names. Evolutionary systematics is a viable alternative, involving both deduction and induction, hypothesis and theory, developing trees with both synchronic and diachronic dimensions often inferring nameable ancestral taxa, and resulting in classifications that advance evolutionary theory and explanations for particular groups.&lt;/p&gt;
</description>
 <pubDate>Tue, 01 May 2012 12:03:30 -0600</pubDate>
</item>
<item>
 <title>PLoS launches Currents:Tree of Life and Hubs:Biodiversity</title>
 <link>http://systbio.org.whsites.net/?q=node/347</link>
 <description>&lt;p&gt;&lt;img src="files/plos.gif" align="right" hspace="7" vspace="7"&gt;&lt;br /&gt;
PLoS have recently released &lt;a href="http://currents.plos.org/treeoflife"&gt;PLoS Currents: Tree of Life&lt;/a&gt; and &lt;a href="http://hubs.plos.org/web/biodiversity/"&gt;PLoS Hubs: Biodiversity&lt;/a&gt;, and members of the Society of Systematic Biologists are heavily involved in both initiatives.&lt;/p&gt;
&lt;p&gt;&lt;a href="http://currents.plos.org/treeoflife"&gt;PLoS Currents: Tree of Life&lt;/a&gt; is a new open-access publication that reduces the delay between the creation of data and publication to a matter of days. The content is peer-reviewed, citable, publicly archived and included in PubMed. Tree of Life  is edited by Keith Crandall (Brigham Young University, USA), Douglas Soltis (University of Florida, USA), and Pamela Soltis. For more details see the &lt;a href="http://blogs.plos.org/plos/2010/11/plos-currents-tol/"&gt;post on the official PLoS blog&lt;/a&gt;.&lt;/p&gt;
&lt;p&gt;PLoS have also launched &lt;a href="http://hubs.plos.org/web/biodiversity/"&gt;PLoS Hubs: Biodiversity&lt;/a&gt;, which aggregates open-access biodiversity articles from multiple journals (not just PLoS titles). You can find more details on the background behind the Hub in the guest blog post &lt;a href="http://blogs.plos.org/plos/2010/10/aggregating-tagging-and-connecting-biodiversity-studies/"&gt;Aggregating, tagging and connecting biodiversity studies&lt;/a&gt; by David Mindell and others.&lt;/p&gt;
</description>
 <pubDate>Wed, 26 Jan 2011 01:16:20 -0700</pubDate>
</item>
<item>
 <title>Old journal has new focus</title>
 <link>http://systbio.org.whsites.net/?q=node/272</link>
 <description>&lt;p&gt;&lt;img src="http://www.ctoz.nl/c/ctz/graphics/ctz-titel-groot.jpg" width="128" align="right" /&gt;The journal &lt;i&gt;Contributions to Zoology&lt;/i&gt; '.. since 1848 ..' solicits high-quality papers in all systematics-related branches of Zoology and Paleontology. Preference will be given to manuscripts dealing with conceptual issues and to integrative papers. Reviews and alpha-taxonomic contributions may be considered for publication, but acceptance will depend on their high quality and exceptional nature. The journal is published by the National Museum of Natural History Naturalis, Leiden and the Zoological Museum Amsterdam and is freely available online since 1997 at &lt;a href="http://www.ctoz.nl"&gt;www.ctoz.nl&lt;/a&gt;.&lt;/p&gt;
</description>
 <pubDate>Sun, 21 Dec 2008 07:17:07 -0700</pubDate>
</item>
<item>
 <title>Evolution of the animals - a Linnean tercentenary celebration</title>
 <link>http://systbio.org.whsites.net/?q=node/239</link>
 <description>&lt;p&gt;
&lt;img src="files/cube.jpg" align="right" width="200"&gt;&lt;br /&gt;
The &lt;i&gt;Philosophical Transactions of the Royal Society&lt;/i&gt; have published an issue on animal evolution, based on a meeting organised by Tim Littlewood and Max Telford. Papers cover the origins and relationships of major groups of animals, our attempts to reconstruct the common ancestor of all animals and the evolution of key characteristics including segments and wings.&lt;/p&gt;
&lt;p&gt;Subscribers to the journal can view the articles &lt;a href="http://journals.royalsociety.org/openurl.asp?genre=issue&amp;#038;volume=363&amp;#038;issue=1496&amp;#038;issn=0962-8436"&gt;online&lt;/a&gt;. Hard copy can be purchased at a specially discounted price of √?¬£47.50 (instead of the normal √?¬£59.50)  by contacting  &lt;a href="http://www.portlandpress.com/pcs/books/prod_det.cfm?product=9780854036813"&gt;Portland Press&lt;/a&gt; (quoting reference &lt;strong&gt;TB 1496&lt;/strong&gt;) or Debbie Vaughan at the Royal Society (&lt;a href="mailto:debbie.vaughan@royalsociety.org"&gt;debbie.vaughan@royalsociety.org&lt;/a&gt;).
&lt;/p&gt;
</description>
 <pubDate>Mon, 21 Apr 2008 09:11:06 -0600</pubDate>
</item>
<item>
 <title>Construction and annotation of large phylogenetic trees</title>
 <link>http://systbio.org.whsites.net/?q=node/206</link>
 <description>&lt;p&gt;Mike Sanderson's review "Construction and annotation of large phylogenetic trees" has just been published in &lt;i&gt;Australian Systematic Botany&lt;/i&gt; &lt;b&gt;20&lt;/b&gt;(4) 287√¢‚?¨"301 &lt;a href="http://dx.doi.org/10.1071/SB07006"&gt;doi:10.1071/SB07006&lt;/a&gt; &lt;a style="cursor:pointer" onclick="javascript:u='http://dx.doi.org/10.1071/SB07006';a=false;x=window;e=x.encodeURIComponent;d=document;w=open('http://www.connotea.org/addpopup?continue=confirm&amp;#038;uri='+e(u),'add','width=660,height=300,scrollbars,resizable');void(x.setTimeout('w.focus()',200));"&gt;&lt;img src="files/connotea.png" align="absmiddle"/&gt;&lt;/a&gt;.&lt;/p&gt;
&lt;p&gt;&lt;cite&gt;Broad availability of molecular sequence data allows construction of phylogenetic trees with 1000s or even 10 000s of taxa. This paper reviews methodological, technological and empirical issues raised in phylogenetic inference at this scale. Numerous algorithmic and computational challenges have been identified surrounding the core problem of reconstructing large trees accurately from sequence data, but many other obstacles, both upstream and downstream of this step, are less well understood. Before phylogenetic analysis, data must be generated de novo or extracted from existing databases, compiled into blocks of homologous data with controlled properties, aligned, examined for the presence of gene duplications or other kinds of complicating factors, and finally, combined with other evidence via supermatrix or supertree approaches. After phylogenetic analysis, confidence assessments are usually reported, along with other kinds of annotations, such as clade names, or annotations requiring additional inference procedures, such as trait evolution or divergence time estimates. Prospects for partial automation of large-tree construction are also discussed, as well as risks associated with √¢‚?¨À?outsourcing√¢‚?¨‚?¢ phylogenetic inference beyond the systematics community.&lt;/cite&gt;&lt;/p&gt;
</description>
 <pubDate>Wed, 12 Sep 2007 23:25:25 -0600</pubDate>
</item>
<item>
 <title>Joe Felsenstein interview (with advice to young scientists)</title>
 <link>http://systbio.org.whsites.net/?q=node/188</link>
 <description>&lt;p&gt;
&lt;img src="files/Felsenstein.jpeg" align="right" hspace="7" vspace="7" /&gt;&lt;br /&gt;
The Blind.Scientist blog carries an &lt;a href="http://blindscientist.genedrift.org/2007/04/19/sciview-scientific-interviews-part-1/"&gt;interview with Joe Felsenstein&lt;/a&gt;. Joe's comments on the history of the field won't be news to old hands, but his advice to young scientists may be of interest:&lt;/p&gt;
&lt;p&gt;&lt;i&gt;&lt;b&gt;Blind.Scientist&lt;/b&gt;: You have in Phylip√¢‚?¨‚?¢s grant webpage a √¢‚?¨≈?no thanks√¢‚?¨¬ù section listing everyone that refused supporting the program along the years. In my opinion this is a bold statement and not very common in the scientific environment (at least not online). What would be your advice for the young scientist that is searching for financial support for his/her research? Should s/he be vocal when a strong application is rejected?&lt;/i&gt;&lt;/p&gt;
&lt;p&gt;&lt;b&gt;JF:&lt;/b&gt; Generally, no. I could do this because I felt confident enough in my reputation. It was fun. These granting agencies can say all sorts of fatuous things in their evaluations and they are never called on this. A web page is editor-free publication so I had fun with that. I had a hope that it might cause grant reviewers to think twice about getting themselves on that list by coming up with arbitrary and ill-thought-out objections. (Based on our lab√¢‚?¨‚?¢s experience since then, alas, this isn√¢‚?¨‚?¢t working). But for a vulnerable young researcher I would say no, don√¢‚?¨‚?¢t succumb to that temptation. You may get a reputation as a sorehead. I know a few young scientists who are quite combative about any negative evaluation, and they do get a reputation as someone you don√¢‚?¨‚?¢t want to deal with. However in these days of increasing difficulty in finding funds, it is tough for newcomers as the agencies are looking for any reason to say no.&lt;/p&gt;
</description>
 <pubDate>Wed, 16 May 2007 02:26:51 -0600</pubDate>
</item>
<item>
 <title>New bumble bee phylogeny</title>
 <link>http://systbio.org.whsites.net/?q=node/187</link>
 <description>&lt;p&gt;
&lt;img src="http://www.nhm.ac.uk/research-curation/projects/bombus/trifasciatus.jpg" align="right" width="128" hspace="7" vspace="7"/&gt;Sydney Cameron and coauthors have published a phylogeny of the genus &lt;i&gt;Bombus&lt;/i&gt; (bumble bees). The paper is available here &lt;a href="http://dx.di.org/10.1111/j.1095-8312.2007.00784.x"&gt;doi:0.1111/j.1095-8312.2007.00784.x&lt;/a&gt;, and the data has been uploaded to the Systematic Biology web site as &lt;a href="files/combdivBayesiv19v3.nex"&gt;combdivBayesiv19v3.nex&lt;/a&gt;.&lt;/p&gt;
&lt;p&gt;Paul Williams (one of the coauthors) maintains a &lt;a href="http://www.nhm.ac.uk/research-curation/projects/bombus/"&gt;comprehensve web site on bumble bees&lt;/a&gt; at The Natural History Museum, from which the image of &lt;i&gt;Bombus trifasciatus&lt;/i&gt; at the right is taken.
&lt;/p&gt;
</description>
 <pubDate>Thu, 10 May 2007 02:03:01 -0600</pubDate>
</item>
<item>
 <title>Multiple sequence alignment for phylogenetic purposes</title>
 <link>http://systbio.org.whsites.net/?q=node/159</link>
 <description>&lt;p&gt;
&lt;img src="http://www.publish.csiro.au/covers/SBv19n6cover.jpg" align="right"/&gt;&lt;br /&gt;
A review of sequence alignment by David Morrison (our book review editor) has just appeared in &lt;i&gt;Australian Systematic Botany&lt;/i&gt;, as one of the L. A. S. Johnson review series. The paper can be accessed online here:&lt;br /&gt;
&lt;a href="http://dx.doi.org/10.1071/SB06020"&gt;doi:10.1071/SB06020&lt;/a&gt;. The PDF is freely available.&lt;br /&gt;
The abstract concludes:&lt;br /&gt;
&lt;cite&gt;Perhaps the most important suggestion is that alignment should be seen as a process where new sequences are added to a pre-existing alignment that has been manually curated by the biologist.&lt;/cite&gt;
&lt;/p&gt;
</description>
 <pubDate>Wed, 17 Jan 2007 11:51:42 -0700</pubDate>
</item>
<item>
 <title>PLoS Biology: Bushes in the Tree of Life</title>
 <link>http://systbio.org.whsites.net/?q=node/147</link>
 <description>&lt;p&gt;
&lt;img src="http://biology.plosjournals.org/images/icons/pbio_0040352_tn.gif" align="right"/&gt;&lt;br /&gt;
Antonis Rokas and Sean Carroll have an interesting opinion piece in PLoS Biology (&lt;a href="http://dx.doi.org/10.1371/journal.pbio.0040352"&gt;doi:10.1371/journal.pbio.0040352&lt;/a&gt;). They quote Richard Dawkins:&lt;br /&gt;
&lt;cite&gt;&lt;br /&gt;
√¢‚?¨≈?√¢‚?¨¬¶ there is, after all, one true tree of life, the unique pattern of evolutionary branchings that actually happened. It exists. It is in principle knowable. We don't know it all yet. By 2050 we should √¢‚?¨" or if we do not, we shall have been defeated only at the terminal twigs, by the sheer number of species.√¢‚?¨¬ù&lt;/cite&gt;&lt;/p&gt;
&lt;p&gt;and end their essay:&lt;br /&gt;
&lt;cite&gt;&lt;br /&gt;
We submit that if the current efforts to assemble the TOL have, by 2050 (if not much sooner), assembled an arborescent bush of life, Dawkins' prediction will have come to fruition.&lt;/cite&gt;&lt;/p&gt;
&lt;p&gt;And no, I don't know what they mean by that either...
&lt;/p&gt;
</description>
 <pubDate>Thu, 16 Nov 2006 15:18:03 -0700</pubDate>
</item>
<item>
 <title>Royal Society DNA Barcoding publication offer</title>
 <link>http://systbio.org.whsites.net/?q=node/113</link>
 <description>&lt;p&gt;
&lt;img src="files/ProcRoySocBarcoding.jpg" align="right" /&gt;&lt;/p&gt;
&lt;p&gt;The first international scientific conference on Barcoding of Life was  held at the Natural History Museum in London in February 2005 and the October 2005 issue of  &lt;em&gt;Philosophical Transactions of the Royal Society B: Biological Sciences&lt;/em&gt; reviews the scientific challenges discussed during  this conference and in previous publications.    Subscribers to &lt;em&gt;Philosophical Transactions&lt;/em&gt; can access the full content &lt;a href="http://www.journals.royalsoc.ac.uk/openurl.asp?genre=issue&amp;#038;issn=0962-8436&amp;#038;volume=360&amp;#038;issue=1462" target="_blank"&gt;online&lt;/a&gt;.&lt;/p&gt;
&lt;p&gt;Non-subscribers can purchase the print issue at a specially reduced price of √?¬£45/$US75 for a limited  amount of time (usual price: √?¬£115/US$195).  &lt;/p&gt;
&lt;p&gt;To place an order at the discounted price, please contact The Royal Society by any of  the methods below, quoting reference &lt;strong&gt;TB 1462&lt;/strong&gt;: &lt;/p&gt;
&lt;ul&gt;
&lt;li&gt;   telephone +44 (0)20 7451 2646   &lt;/li&gt;
&lt;li&gt;email &lt;a href="maito:sales@royalsoc.ac.uk"&gt;sales@royalsoc.ac.uk&lt;/a&gt; &lt;/li&gt;
&lt;li&gt;   post Publishing, The Royal Society, 6 Carlton House Terrace, London SW1Y 5AG, UK &lt;/li&gt;
&lt;/ul&gt;
&lt;/p&gt;
</description>
 <pubDate>Fri, 09 Dec 2005 05:55:39 -0700</pubDate>
</item>
</channel>
</rss>
