<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=500 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:39:10 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="indexacd7.html?q=node/208">56(4) August 2007</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 7 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>December 2, 2015</h3>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/268">Phylogenetic relationships of &#8224;Luisiella feruglioi (Bordas) and the recognition of a new clade of freshwater teleosts from the Jurassic of Gondwana</a></div>
  <div class="description">Background:
Teleosts constitute more than 99 % of living actinopterygian fishes and fossil teleosts have been studied for about two centuries. However, a general consensus on the definition of Teleostei and the relationships among the major teleostean clades has not been achieved. Our current ideas on the origin and early diversification of teleosts are mainly based on well-known Mesozoic marine taxa, whereas the taxonomy and phylogenetic relationships of many Jurassic continental teleosts are still poorly understood despite their importance to shed light on the early evolutionary history of this group. Here, we explore the phylogenetic relationships of the Late Jurassic (Oxfordian – Tithonian) freshwater †Luisiella feruglioi from Patagonia, in a comprehensive parsimony analysis after a thorough revision of characters from previous phylogenetic studies on Mesozoic teleosts.
Results:
We retrieved †Luisiella feruglioi as the sister taxon of the Late Jurassic †Cavenderichthys talbragarensis, both taxa in turn forming a monophyletic group with the Early Cretaceous †Leptolepis koonwarri. This new so far exclusively Gondwanan freshwater teleost clade, named †Luisiellidae fam. nov. herein, is placed outside crown Teleostei, as a member of the stem-group immediately above the level of †Leptolepis coryphaenoides. In addition, we did not retrieve the Late Jurassic †Varasichthyidae as a member of †Crossognathiformes. The position of †Crossognathiformes within Teleocephala is confirmed whereas †Varasichthyidae is placed on the stem.
Conclusions:
The general morphology of luisiellids is that of basal, stem Teleocephala; however, most of their synapomorphies have evolved independently in teleocephalans. Similarly, the resemblance between varasichthyids and crossognathiforms might be due to parallel evolution. In accordance to most teleostean phylogenies, our analysis shows that a major morphological change occurred along the stem line and are currently recorded at the level of †Leptolepis coryphaenoides. A stem-based total clade Teleostei has been accepted for this work.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/267">Erratum to: Positive selection in octopus haemocyanin indicates functional links to temperature adaptation</a></div>
  <div class="description">No description available</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:25</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134387449101">Postdoc: MissouriBotanicalGardens.ModelingGlobalChange</a></div>
  <div class="description">The Center for Conservation and Sustainable Development (CCSD) at the
Missouri Botanical Garden seeks to hire a full-time postdoctoral
scholar.  The postdoc will work closely with Dr. Adam Smith in the
CCSD&rsquo;s Global Change Program to develop methods for applying
next-generation species distribution models to data that traditionally
has been difficult to model.  In collaboration with Dr. Smith the
postdoc will 1) develop novel modeling algorithms; 2) test these
algorithms with simulations and high-quality specimen data obtained in
collaboration with taxonomists at the Missouri Botanical Garden; 3)
prepare and submit manuscripts for publication; 4) develop R packages
that encapsulate these novel methods; 5) craft project specifications
for a programmer who will create stand-alone software using these
methods; 6) actively consult with collaborators at UC Berkeley.  The
successful candidate will:

* Have a solid background in applied mathematics and statistics
(including Bayesian methods).

* Have expertise in R or related programming languages.

* Excellent communication skills, with experience presenting results
and publishing in peer-reviewed journals.

* Understand and appreciate the difficulties of working with occurrence
data that may have been collected in an opportunistic fashion, e.g.,
data from natural history museum specimens.

* Be enthusiastic about developing tools that enable conservation
practitioners to avert loss of biodiversity and be able to translate
complex ideas into straightforward procedures.

Candidates with a completed Ph.D. or that will soon graduate with a
Ph.D. in ecology, wildlife biology, environmental science, statistics,
or applied mathematics are encouraged to apply.  Ideal start time is
early 2016, but this is negotiable for applicants who may be graduating
in the spring.  The term for this grant-funded position will be 1 year,
with renewal for an additional two years or more contingent on
satisfactory performance.  Salary will be commensurate with experience,
and this position includes a comprehensive benefits package.

The position will be based in St. Louis, where a vibrant community of
ecologists, conservation practitioners, and evolutionary biologists
that interact through partnerships among MBG, Washington University,
the University of Missouri-St. Louis, Saint Louis University, plus
other area institutions. The position will be seated in the CCSD, which
explores and implements new, science-based approaches to the
conservation and sustainable use of plant diversity. CCSD&rsquo;s
strategies for conservation are based on a sound, scientific
understanding of the occurrence and distribution of plants. CCSD
applies the knowledge of plant diversity accumulated by Missouri
Botanical Garden researchers over many years, making that knowledge
usable for conservation planning and decision-making. Operating under
the auspices of the Garden and as part of its division of Science and
Conservation, CCSD builds upon the Garden&rsquo;s institutional expertise,
scientific programs, influence and resources.

To be considered, applicants should apply on-line by February 5, 2015
at:
<a href="http://bit.ly/1O2RMNs">http://bit.ly/1O2RMNs</a>

Please direct questions to Dr. Adam Smith (adam.smith@mobot.org).


Adam B. Smith, Ph.D.
Assistant Scientist in Global Change
Center for Conservation & Sustainable Development
Missouri Botanical Garden
PO Box 299
Saint Louis, MO 63166

314-577-9473 ext. 6314

<a href="http://bit.ly/1lWHlV8">http://bit.ly/1lWHlV8</a>

Adam Smith 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:54</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134386513091">Postdoc:NHM_London.AncientDNA</a></div>
  <div class="description"><p>Post-Doctoral Research Assistant in Ancient DNA of Pleistocene faunal remains.

Salary: 28,470 per annum plus benefits
Contract: 36 months fixed term contract
Location: Natural History Museum, London, UK.
Closing date: 9am on Monday 7 December 2015

We are currently seeking applications for a PDRA to contribute to the Palaeolithic colonisation and cultural diversification in unfamiliar landscapes (UP-NORTH) project. The post-holder will be responsible to Professor Ian Barnes, and will work closely with Dr Rhiannon Stevens at UCL Institute of Archaeology. The candidate will have demonstrable knowledge and experience of generating and analyzing ancient DNA using next generation sequencing technologies.

For further information go to PDRA-UpNorth here: <a href="http://bit.ly/1jxUORv">http://bit.ly/1jxUORv</a> or contact i.barnes@nhm.ac.uk


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:39</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134386012271">Postdoc:TuftsUniversity.AdaptationSpeciation</a></div>
  <div class="description"><p>The Dopman Lab in the Department of Biology at Tufts University is seeking a Postdoctoral Researcher with interests in Speciation or Adaptive Evolution. Current interests and possible postdoctoral projects include phenology and thermotolerance evolution; circadian and seasonal rhythms; adaptation to climate change; &ldquo;speciation genes&rdquo; and genomic divergence; sex-chromosome evolution; ecological and behavioral isolating barriers; and parallel evolution. A PhD in evolutionary biology, genetics & genomics, or in a related field is required. Interested individuals should contact Erik Dopman (erik.dopman@tufts.edu) and provide a CV and a brief statement of prior and future research interests. Informal inquiries are welcomed.  <a href="http://bit.ly/1rHlKQf">http://bit.ly/1rHlKQf</a>

FELLOWSHIP INFORMATION

The NIH-funded Postdoctoral Fellowship position is available in the Training in Education and Critical Research Skills Program (TEACRS) at Tufts University. This NIH-funded program is designed to prepare talented postdoctoral trainees with the career skills needed to succeed in academic biological research. Trainees spend on average 75% of their time conducting bench research and 25% of their time in career development activities that will help them manage the multiple demands of a faculty position. Teaching skills are developed through programs involving mentored classroom assignments at minority-serving institutions in the Boston area. The training is supplemented by workshops and seminars on teaching methods, career opportunities in academia, and essential skills such as grant writing, mentoring, lab management and scientific presentations. The training components are sequenced to allow trainees to balance the activities and to build on previous experiences. TEACRS fellows
  receive up to four years of salary support and a yearly travel and supply allowance. They will complete our program ready to manage a successful research program, teach, and balance the multi-faceted demands inherent in a career as an academic scientist. <a href="http://bit.ly/1luOEmk">http://bit.ly/1luOEmk</a>

APPLICATION INFORMATION

The application deadline for positions beginning in the fall of 2016 is March 1. Applicants must be U.S. citizens or permanent residents with no more than two years of postdoctoral training at the time of their appointment to the program. Commitment to the goals of the program, strong academic and research credentials and an interest in teaching and mentoring will be important criteria used in the selection process.

WHY BOSTON, MASSACHUSETTS?

Tufts is located in the academic and intellectual hub of Boston, home to more than 60 distinguished colleges and universities. A truly multi-cultural city, it boasts the largest concentration of international students in the world, providing an unparalleled environment for study and research opportunities. In addition to the numerous colleges and universities, many of the world&rsquo;s leading biomedical research institutions and biotech companies have headquarters or branches based in and around Boston. Recreational activities are abundant and range from music, museums, sports, theatre and restaurants. The beaches of Cape Cod and the hiking trails and ski slopes of Maine, New Hampshire, and Vermont are only 2 to 3 hours away. In Boston, you will discover one of America&rsquo;s most cultural, stimulating, and livable cities.


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:08</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134384910566">Graduate position:</a></div>
  <div class="description"><p>&ndash;_000_E54D8A8ECDEFC940A25C9847DD3E90E11314213FMBX111dethzch_
Content-Type: text/plain; charset=&ldquo;iso-8859-1&rdquo;
Content-Transfer-Encoding: quoted-printable


ETHZurich.BiodiversityModeling

The group of Landscape Ecology at ETH Zurich (<a href="http://www.ethz.ch/">www.ethz.ch</a>) and WSL Birmensdorf (<a href="http://www.wsl.ch/">www.wsl.ch</a>) under the supervision of Prof. Loc Pellissier is invites applications for a PhD position in biodiversity modelling.


The successful candidate will develop mechanistic models including speciation and extinction to explore how landscape dynamic history shaped extant biodiversity patterns. Ultimately the model will inform on ecosystem responses to global changes. The new approach will be applied to several ecosystems with an emphasis on tropical marine reef fishes. The PhD position will be embedded within the group&rsquo;s core research interests focused on three main themes: 1) Developing spatial models of biodiversity. 2) Understanding the effect of history on extant biodiversity patterns. 3) Monitoring and forecasting biodiversity and ecosystem responses to global changes.

Applicants must hold a Master degree in ecology, evolutionary biology, bioinformatics or another relevant discipline. Significant experience in programming (e.g. in R, Julia, C++) is expected. Experience in Geographic Information System (GIS) would be an advantage. Strong oral and written communication skills in English are required. The expected starting date is February 2016 with a funding for three years.

The chair of Landscape Ecology is shared between ETH Zrich and the Swiss Federal Institute for Forest, Snow and Landscape Research (WSL). This project is part of an international research collaboration with the teams of Prof. Fabien Leprieur in Montpellier, France and Dr. Laurent Vigliola (IRD, Nouma).

Please provide a maximum two-page cover letter that explains your motivation, research experience and interest, a CV including publications, and a list of three professional references. Application review will begin in January 1, and continue until the position is filled.


Please address your application online following this link:


<a href="http://bit.ly/21udC6h">http://bit.ly/21udC6h</a>


&ndash;_000_E54D8A8ECDEFC940A25C9847DD3E90E11314213FMBX111dethzch_
Content-Type: text/html; charset=&ldquo;iso-8859-1&rdquo;
Content-Transfer-Encoding: quoted-printable



P {margin-top:0;margin-bottom:0;}
ETHZurich.BiodiversityModeling  
The group of Landscape Ecology at ETH Zurich (<a href="http://www.ethz.ch/">www.ethz.ch</a>) and WSL Birmensdorf (<a href="http://www.wsl.ch/">www.wsl.ch</a>) under the supervision of Prof. Loc Pellissier is invites
 applications for a PhD position in biodiversity modelling.
 

<p>The successful candidate will develop mechanistic models including speciation and extinction to explore how landscape dynamic history shaped extant biodiversity patterns. Ultimately the model will inform on ecosystem responses to global changes. The new
 approach will be applied to several ecosystems with an emphasis on tropical marine reef fishes. The PhD position will be embedded within the group&rsquo;s core research interests focused on three main themes: 1) Developing spatial models of biodiversity. 2) Understanding
 the effect of history on extant biodiversity patterns. 3) Monitoring and forecasting biodiversity and ecosystem responses to global changes.</p>
<p>Applicants must hold a Master degree in ecology, evolutionary biology, bioinformatics or another relevant discipline. Significant experience in programming (e.g. in R, Julia, C++) is expected. Experience in Geographic Information System (GIS) would be an
 advantage. Strong oral and written communication skills in English are required. The expected starting date is February 2016 with a funding for three years.</p>
<p>The chair of Landscape Ecology is shared between ETH Zrich and the Swiss Federal Institute for Forest, Snow and Landscape Research (WSL). This project is part of an international research collaboration with the teams of Prof. Fabien Leprieur in Montpellier,
 France and Dr. Laurent Vigliola (IRD, Nouma).</p>
<p>Please provide a maximum two-page cover letter that explains your motivation, research experience and interest, a CV including publications, and a list of three professional references. Application review will begin in January 1, and continue until the position
 is filled. </p>
<p></p>
<p>Please address your application online following this link:</p>
<p></p>
<p><a href="http://bit.ly/21udC6h">http://bit.ly/21udC6h</a></p>






&ndash;_000_E54D8A8ECDEFC940A25C9847DD3E90E11314213FMBX111dethzch
via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:08</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134384910846">Postdoc: UCIrvine.PopulationGenomics</a></div>
  <div class="description"><p>A postdoctoral scholar position is available in the laboratory of Dr.
Kevin Thornton at University of California, Irvine.

Area of research: Population genetics of polygenic adaptation

Description of duties: The postdoc will be part of a project generally
focused on the integration of population- and quantitative- genetic
models of adaptation, and their implications for how we interpret
patterns of standing genetic variation, results of association
studies, etc. Within this project, there is room for considerable
freedom related to modeling, parameter inference, and data analysis.
The data analysis could have a focus on Drosophila and/or human
population-genomic data.

Salary: $50,000/year in the first year. Salary is adjusted in
subsequent years according to the NIH salary scale.

Required qualifications include a recent PhD in evolution, genetics,
bioinformatics or a related field. The ideal candidate would have a
background in population- and/or quantitative genetics, with a solid
understanding of theory. Additionally, the candidate should have some
programming experience. However, consideration will be given to any
motivated applicant interested in evolutionary genomics.

Qualified candidates should submit curriculum vitae, the names and
contact info of three references, and a cover letter explaining the
candidate&rsquo;s interest in the position and overall research aims and
accomplishments to the following on-line recruitment URL:

<a href="http://bit.ly/1N34pIs">http://bit.ly/1N34pIs</a>

For more information about this position contact Dr. Kevin Thornton
krthornt@uci.edu.

The University of California, Irvine is an Equal
Opportunity/Affirmative Action Employer advancing inclusive
excellence. All qualified applicants will receive consideration for
employment without regard to race, color, religion, sex, sexual
orientation, gender identity, national origin, disability, age,
protected veteran status, or other protected categories covered by the
UC nondiscrimination policy.


kevin.thornton.work@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:08</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134384910266">Graduate position: UMuenster.2.Evolution.2</a></div>
  <div class="description">2 PhD stipends within the interdisciplinary “Münster Graduate School
of Evolution”: PhD projects in Biology and Medicine

*Closing date for applications: 10 December 2015*

The “Münster Graduate School of Evolution” (MGSE) offers 2 PhD
positions funded by DAAD-stipends for international (non-German) students
within the stimulating environment of the University of Münster,
Germany. As an interdisciplinary graduate school, the MGSE uses the
unifying concept of evolution to bridge the faculties of Biology,
Medicine, Geosciences, Mathematics, and Philosophy. PhD students work on
their diverse disciplinary projects in one of the involved institutes
and benefit from interdisciplinary curricular activities as well as a
structured supervision and support throughout their PhD. The MGSE is
based in the stimulating city of Münster near Münster’s City Palace
and offers a family friendly and international atmosphere.

Location: Münster, Germany
Working language: English
Start of the PhD: October 2016
Duration: 3 years (4 years for students from developing and emerging
countries)

You can apply for one or several out of nine projects, each of them
involving high-quality research and state-of-the-art techniques. The
projects are supervised by excellent researchers within the MGSE:

- Prof. Dr. Erich Bornberg-Bauer (Institute for Evolution and
  Biodiversity, University of Münster)
- Prof. Dr. Jürgen Gadau (School of Life Science, Arizona State
  University)
- Prof. Dr. Ulrich Dobrindt (Institute of Hygiene, University
  Hospital Münster)
- Prof. Dr. Joachim Kurtz (Institute for Evolution and Biodiversity,
  University of Münster)
- Dr. Jörn Scharsack (Institute for Evolution and Biodiversity,
  University of Münster)
- Prof. Dr. Norbert Sachser (Institute of Neuro and Behavioural Biology,
  University of Münster)
- Prof. Dr. Stephan Ludwig (Center for Molecular Biology of
  Inflammation, University of Münster)
- Prof. Dr. Wojciech Makalowski (Institute of Bioinformatics, University
  of Münster)
- Prof. Dr. Monika Stoll (Institute of Human Genetics, University
  Hospital Münster)
- Dr. Shirin Glander (Institute of Human Genetics, University
  Hospital Münster)
- Dr. Frank Rühle (Institute of Human Genetics, University
  Hospital Münster)

Highly qualified and motivated candidates all over the world are invited
to submit their application.

For a list of projects and information on
requirements and the application procedure, please visit:
<a href="http://bit.ly/21udC6e.">http://bit.ly/21udC6e.</a>  In case of
questions please contact: mgse@uni-muenster.de.



Dr. Vanessa Kloke
Coordinator of the Münster Graduate School of Evolution
Westfälische Wilhelms-Universität Münster
Schlossplatz 6, D-48149 Münster, Germany
 
Phone: +49 251 83-21252
E-Mail: mgse@uni-muenster.de
Website: http://bit.ly/1LjwH65

&ldquo;Vanessa Kloke, MGSE&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 1, 2015</h3>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/266">Within-species divergence in the seminal fluid proteome and its effect on male and female reproduction in a beetle</a></div>
  <div class="description">Background:
Male seminal fluid proteins (SFPs), transferred to females during mating, are important reproductive proteins that have multifarious effects on female reproductive physiology and that often show remarkably rapid and divergent evolution. Inferences regarding natural selection on SFPs are based primarily on interspecific comparative studies, and our understanding of natural within-species variation in SFPs and whether this relates to reproductive phenotypes is very limited. Here, we introduce an empirical strategy to study intraspecific variation in and selection upon the seminal fluid proteome. We then apply this in a study of 15 distinct populations of the seed beetle Callosobruchus maculatus.
Results:
Phenotypic assays of these populations showed significant differences in reproductive phenotypes (male success in sperm competition and male ability to stimulate female fecundity). A quantitative proteomic study of replicated samples of male accessory glands revealed a large number of potential SFPs, of which ≥127 were found to be transferred to females at mating. Moreover, population divergence in relative SFP abundance across populations was large and remarkably multidimensional. Most importantly, variation in male SFP abundance across populations was associated with male sperm competition success and male ability to stimulate female egg production.
Conclusions:
Our study provides the first direct evidence for postmating sexual selection on standing intraspecific variation in SFP abundance and the pattern of divergence across populations in the seminal fluid proteome match the pattern predicted by the postmating sexual selection paradigm for SFP evolution. Our findings provide novel support for the hypothesis that sexual selection on SFPs is an important engine of incipient speciation.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">15:40</div>
 <div class="body">
  <div class="title"><a href="http://iphylo.blogspot.com/2015/12/frontiers-of-biodiversity-informatics.html">Frontiers of biodiversity informatics and modelling species distributions #GBIFfrontiers @AMNH videos</a></div>
  <div class="description"><p>For those of you who, like me, weren't at the "Frontiers Of Biodiversity Informatics and Modelling Species Distributions" held at the AMNH in New York, here are the videos of the talks and panel discussion, which the organisers have kindly put up on <a href="https://vimeo.com/album/3669289">Vimeo</a> with the following description: The Center for Biodiversity and Conservation (CBC) partnered with the Global Biodiversity Information Facility (GBIF) to host a special "Symposium and Panel Discussion: Frontiers Of Biodiversity Informatics and Modelling Species Distributions" at the American Museum of Natural History on November 4, 2015.</p> <p>The event kicked off a working meeting of the GBIF Task Group on Data Fitness for Use in Distribution Modelling at the City College of New York on November 5-6. GBIF convened the Task Group to assess the state of the art in the field, to connect with the worldwide scientific and modelling communities, and to share a vision of how GBIF support them in the coming decade.</p> <p>The event successfully convened a broad, global audience of students and scientists to exchange ideas and visions on emerging frontiers of biodiversity informatics. Using inputs from the symposium and from a web survey of experts, the Data Fitness task group will prepare a report, which will be open for consultation and feedback at GBIF.org and on the GBIF Community Site in December 2015. </p></div>
  <div class="source">Source: <a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/265">High major histocompatibility complex class I polymorphism despite bottlenecks in wild and domesticated populations of the zebra finch (Taeniopygia guttata)</a></div>
  <div class="description">Background:
Two subspecies of zebra finch, Taeniopygia guttata castanotis and T. g. guttata are native to Australia and the Lesser Sunda Islands, respectively. The Australian subspecies has been domesticated and is now an important model system for research. Both the Lesser Sundan subspecies and domesticated Australian zebra finches have undergone population bottlenecks in their history, and previous analyses using neutral markers have reported reduced neutral genetic diversity in these populations. Here we characterize patterns of variation in the third exon of the highly variable major histocompatibility complex (MHC) class I α chain. As a benchmark for neutral divergence, we also report the first mitochondrial NADH dehydrogenase 2 (ND2) sequences in this important model system.
Results:
Despite natural and human-mediated population bottlenecks, we find that high MHC class I polymorphism persists across all populations. As expected, we find higher levels of nucleotide diversity in the MHC locus relative to neutral loci, and strong evidence of positive selection acting on important residues forming the peptide-binding region (PBR). Clear population differentiation of MHC allele frequencies is also evident, and this may be due to adaptation to new habitats and associated pathogens and/or genetic drift. Whereas the MHC Class I locus shows broad haplotype sharing across populations, ND2 is the first locus surveyed to date to show reciprocal monophyly of the two subspecies.
Conclusions:
Despite genetic bottlenecks and genetic drift, all surveyed zebra finch populations have maintained high MHC Class I diversity. The diversity at the MHC Class I locus in the Lesser Sundan subspecies contrasts sharply with the lack of diversity in previously examined neutral loci, and may thus be a result of selection acting to maintain polymorphism. Given uncertainty in historical population demography, however, it is difficult to rule out neutral processes in maintaining the observed diversity. The surveyed populations also differ in MHC Class I allele frequencies, and future studies are needed to assess whether these changes result in functional immune differences.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>November 30, 2015</h3>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/264">Chloroplast phylogenomic analysis of chlorophyte green algae identifies a novel lineage sister to the Sphaeropleales (Chlorophyceae)</a></div>
  <div class="description">Background:
The class Chlorophyceae (Chlorophyta) includes morphologically and ecologically diverse green algae. Most of the documented species belong to the clade formed by the Chlamydomonadales (also called Volvocales) and Sphaeropleales. Although studies based on the nuclear 18S rRNA gene or a few combined genes have shed light on the diversity and phylogenetic structure of the Chlamydomonadales, the positions of many of the monophyletic groups identified remain uncertain. Here, we used a chloroplast phylogenomic approach to delineate the relationships among these lineages.
Results:
To generate the analyzed amino acid and nucleotide data sets, we sequenced the chloroplast DNAs (cpDNAs) of 24 chlorophycean taxa; these included representatives from 16 of the 21 primary clades previously recognized in the Chlamydomonadales, two taxa from a coccoid lineage (Jenufa) that was suspected to be sister to the Golenkiniaceae, and two sphaeroplealeans. Using Bayesian and/or maximum likelihood inference methods, we analyzed an amino acid data set that was assembled from 69 cpDNA-encoded proteins of 73 core chlorophyte (including 33 chlorophyceans), as well as two nucleotide data sets that were generated from the 69 genes coding for these proteins and 29 RNA-coding genes. The protein and gene phylogenies were congruent and robustly resolved the branching order of most of the investigated lineages. Within the Chlamydomonadales, 22 taxa formed an assemblage of five major clades/lineages. The earliest-diverging clade displayed Hafniomonas laevis and the Crucicarteria, and was followed by the Radicarteria and then by the Chloromonadinia. The latter lineage was sister to two superclades, one consisting of the Oogamochlamydinia and Reinhardtinia and the other of the Caudivolvoxa and Xenovolvoxa. To our surprise, the Jenufa species and the two spine-bearing green algae belonging to the Golenkinia and Treubaria genera were recovered in a highly supported monophyletic group that also included three taxa representing distinct families of the Sphaeropleales (Bracteacoccaceae, Mychonastaceae, and Scenedesmaceae).
Conclusions:
Our phylogenomic study advances our knowledge regarding the circumscription and internal structure of the Chlamydomonadales, suggesting that a previously unrecognized lineage is sister to the Sphaeropleales. In addition, it offers new insights into the flagellar structures of the founding members of both the Chlamydomonadales and Sphaeropleales.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/263">Variation in NAT2 acetylation phenotypes is associated with differences in food-producing subsistence modes and ecoregions in Africa</a></div>
  <div class="description">Background:
Dietary changes associated to shifts in subsistence strategies during human evolution may have induced new selective pressures on phenotypes, as currently held for lactase persistence. Similar hypotheses exist for arylamine N-acetyltransferase 2 (NAT2) mediated acetylation capacity, a well-known pharmacogenetic trait with wide inter-individual variation explained by polymorphisms in the NAT2 gene. The environmental causative factor (if any) driving its evolution is as yet unknown, but significant differences in prevalence of acetylation phenotypes are found between hunter-gatherer and food-producing populations, both in sub-Saharan Africa and worldwide, and between agriculturalists and pastoralists in Central Asia. These two subsistence strategies also prevail among sympatric populations of the African Sahel, but knowledge on NAT2 variation among African pastoral nomads was up to now very scarce. Here we addressed the hypothesis of different selective pressures associated to the agriculturalist or pastoralist lifestyles having acted on the evolution of NAT2 by sequencing the gene in 287 individuals from five pastoralist and one agriculturalist Sahelian populations.
Results:
We show that the significant NAT2 genetic structure of African populations is mainly due to frequency differences of three major haplotypes, two of which are categorized as decreased function alleles (NAT2*5B and NAT2*6A), particularly common in populations living in arid environments, and one fast allele (NAT2*12A), more frequently detected in populations living in tropical humid environments. This genetic structure does associate more strongly with a classification of populations according to ecoregions than to subsistence strategies, mainly because most Sahelian and East African populations display little to no genetic differentiation between them, although both regions hold nomadic or semi-nomadic pastoralist and sedentary agriculturalist communities. Furthermore, we found significantly higher predicted proportions of slow acetylators in pastoralists than in agriculturalists, but also among food-producing populations living in the Sahelian and dry savanna zones than in those living in humid environments, irrespective of their mode of subsistence.
Conclusion:
Our results suggest a possible independent influence of both the dietary habits associated with subsistence modes and the chemical environment associated with climatic zones and biomes on the evolution of NAT2 diversity in sub-Saharan African populations.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:30</div>
 <div class="body">
  <div class="title"><a href="https://www.researchgate.net/job/862233_Assistant_Professor-Physical_Oceanography?source=rss">Assistant Professor - Physical Oceanography</a></div>
  <div class="description">Character of Duties:&nbsp;We are seeking an exceptional candidate to support our new initiatives in coupling physical oceanography with population dynamics of coastal ecosystems.&nbsp; The successful candidate will have proven experience in studying the physical aspects of nearshore waters in coastal environments.&nbsp; Specific interests could include physical monitoring along the ridge-to-reef continuum, development and testing of oceanographic sensor arrays, and modeling climate change (sea level rise, ocean warming, nutrification, acidification, etc.) scenarios within coastal and nearshore environments. We are particularly interested in scientists who will increase the UOGML&rsquo;s ability to address large-scale questions through analysis and interpretation of physical oceanographic data across spatial and temporal scales, from in-situ sampling to remote sensing, by performing simulations of oceanographic phenomena using computer or mathematical models.

The successful individual is expected to develop a viable and energetic externally-funded research program, make collaborative contributions to University STEM-centric research programs, and may also contribute to other local and regional projects in a supporting role. The candidate will be required to develop and teach at least one graduate level course in physical oceanography and a second graduate course of choice in a relevant field, with a total of two semester-long courses being taught every four semesters. The candidate may also be requested to develop a seminar course(s) of their own design intermittently.

Salary Level:&nbsp;Salary will be commensurate with qualifications and experience relevant to the position. The following salary figures are for nine (9) months:





Assistant Professor



$38,699 - $66,899 Per Academic Year





Appointment/Relocation:&nbsp;The position is a nine (9) month faculty position, three-year tenure or non-tenure (1, 2, 3 year) contractual appointment.&nbsp; The University provides relocation assistance to new employees if hired off-island on a three-year initial contract.&nbsp; Provided are travel expenses for the applicant and the applicant&rsquo;s spouse (as defined by Guam Law) and children under the age of 18 by the least expensive and most direct air route from the point of hire and for moving household goods to Guam by the least expensive carrier up to 3,500 pounds for a family and 1,750 pounds for an applicant without dependents.&nbsp; Insurance of household goods, if desired, is at the employee&rsquo;s expense.

Benefits:&nbsp;Fringe benefits include a medical and dental plan, life insurance, cafeteria plan (flexible tax benefit plan), and the Government of Guam retirement plan.

&nbsp;

Police and Court Clearance: Pursuant to Public Law No. 28-24 and Executive Order 2005-34, applicants selected for a position are required to provide original police and court (Superior Court of Guam) clearances of no more than three (3) months old prior to commencement of employment.&nbsp; Off-island applicants must obtain clearances from their place of residence.&nbsp; Applicants are responsible for fees associated with obtaining the clearances.

&nbsp;</div>
  <div class="source">Source: <a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:30</div>
 <div class="body">
  <div class="title"><a href="http://www.researchgate.net/job/862233_Assistant_Professor-Physical_Oceanography?source=rss">Assistant Professor - Physical Oceanography</a></div>
  <div class="description">Character of Duties:&nbsp;We are seeking an exceptional candidate to support our new initiatives in coupling physical oceanography with population dynamics of coastal ecosystems.&nbsp; The successful candidate will have proven experience in studying the physical aspects of nearshore waters in coastal environments.&nbsp; Specific interests could include physical monitoring along the ridge-to-reef continuum, development and testing of oceanographic sensor arrays, and modeling climate change (sea level rise, ocean warming, nutrification, acidification, etc.) scenarios within coastal and nearshore environments. We are particularly interested in scientists who will increase the UOGML&rsquo;s ability to address large-scale questions through analysis and interpretation of physical oceanographic data across spatial and temporal scales, from in-situ sampling to remote sensing, by performing simulations of oceanographic phenomena using computer or mathematical models.

The successful individual is expected to develop a viable and energetic externally-funded research program, make collaborative contributions to University STEM-centric research programs, and may also contribute to other local and regional projects in a supporting role. The candidate will be required to develop and teach at least one graduate level course in physical oceanography and a second graduate course of choice in a relevant field, with a total of two semester-long courses being taught every four semesters. The candidate may also be requested to develop a seminar course(s) of their own design intermittently.

Salary Level:&nbsp;Salary will be commensurate with qualifications and experience relevant to the position. The following salary figures are for nine (9) months:





Assistant Professor



$38,699 - $66,899 Per Academic Year





Appointment/Relocation:&nbsp;The position is a nine (9) month faculty position, three-year tenure or non-tenure (1, 2, 3 year) contractual appointment.&nbsp; The University provides relocation assistance to new employees if hired off-island on a three-year initial contract.&nbsp; Provided are travel expenses for the applicant and the applicant&rsquo;s spouse (as defined by Guam Law) and children under the age of 18 by the least expensive and most direct air route from the point of hire and for moving household goods to Guam by the least expensive carrier up to 3,500 pounds for a family and 1,750 pounds for an applicant without dependents.&nbsp; Insurance of household goods, if desired, is at the employee&rsquo;s expense.

Benefits:&nbsp;Fringe benefits include a medical and dental plan, life insurance, cafeteria plan (flexible tax benefit plan), and the Government of Guam retirement plan.

&nbsp;

Police and Court Clearance: Pursuant to Public Law No. 28-24 and Executive Order 2005-34, applicants selected for a position are required to provide original police and court (Superior Court of Guam) clearances of no more than three (3) months old prior to commencement of employment.&nbsp; Off-island applicants must obtain clearances from their place of residence.&nbsp; Applicants are responsible for fees associated with obtaining the clearances.

&nbsp;</div>
  <div class="source">Source: <a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:30</div>
 <div class="body">
  <div class="title"><a href="http://www.researchgate.net/job/862233_Assistant_Professor_Physical_Oceanography?source=rss">Assistant Professor (Physical Oceanography)</a></div>
  <div class="description">Character of Duties:&nbsp;We are seeking an exceptional candidate to support our new initiatives in coupling physical oceanography with population dynamics of coastal ecosystems.&nbsp; The successful candidate will have proven experience in studying the physical aspects of nearshore waters in coastal environments.&nbsp; Specific interests could include physical monitoring along the ridge-to-reef continuum, development and testing of oceanographic sensor arrays, and modeling climate change (sea level rise, ocean warming, nutrification, acidification, etc.) scenarios within coastal and nearshore environments. We are particularly interested in scientists who will increase the UOGML&rsquo;s ability to address large-scale questions through analysis and interpretation of physical oceanographic data across spatial and temporal scales, from in-situ sampling to remote sensing, by performing simulations of oceanographic phenomena using computer or mathematical models.

The successful individual is expected to develop a viable and energetic externally-funded research program, make collaborative contributions to University STEM-centric research programs, and may also contribute to other local and regional projects in a supporting role. The candidate will be required to develop and teach at least one graduate level course in physical oceanography and a second graduate course of choice in a relevant field, with a total of two semester-long courses being taught every four semesters. The candidate may also be requested to develop a seminar course(s) of their own design intermittently.

Salary Level:&nbsp;Salary will be commensurate with qualifications and experience relevant to the position. The following salary figures are for nine (9) months:





Assistant Professor



$38,699 - $66,899 Per Academic Year





Appointment/Relocation:&nbsp;The position is a nine (9) month faculty position, three-year tenure or non-tenure (1, 2, 3 year) contractual appointment.&nbsp; The University provides relocation assistance to new employees if hired off-island on a three-year initial contract.&nbsp; Provided are travel expenses for the applicant and the applicant&rsquo;s spouse (as defined by Guam Law) and children under the age of 18 by the least expensive and most direct air route from the point of hire and for moving household goods to Guam by the least expensive carrier up to 3,500 pounds for a family and 1,750 pounds for an applicant without dependents.&nbsp; Insurance of household goods, if desired, is at the employee&rsquo;s expense.

Benefits:&nbsp;Fringe benefits include a medical and dental plan, life insurance, cafeteria plan (flexible tax benefit plan), and the Government of Guam retirement plan.

&nbsp;

Police and Court Clearance: Pursuant to Public Law No. 28-24 and Executive Order 2005-34, applicants selected for a position are required to provide original police and court (Superior Court of Guam) clearances of no more than three (3) months old prior to commencement of employment.&nbsp; Off-island applicants must obtain clearances from their place of residence.&nbsp; Applicants are responsible for fees associated with obtaining the clearances.

&nbsp;</div>
  <div class="source">Source: <a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></div>
 </div>
</div>
<h3>November 20, 2015</h3>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/261">Evolutionary comparison of prenylation pathway in kinetoplastid Leishmania and its sister Leptomonas</a></div>
  <div class="description">Background:
Leptomonas is monogenetic kinetoplastid parasite of insects and is primitive in comparison to Leishmania. Comparative studies of these two kinetoplastid may share light on the evolutionary transition to dixenous parasitism in Leishmania. In order to adapt and survive within two hosts, Leishmania species must have acquired virulence factors in addition to mechanisms that mediate susceptibility/resistance to infection in the pathology associated with disease. Rab proteins are key mediators of vesicle transport and contribute greatly to the evolution of complexity of membrane transport system. In this study we used our whole genome sequence data of these two divergent kinetoplastids to analyze the orthologues/paralogues of Rab proteins.
Results:
During change of lifestyle from monogenetic (Leptomonas) to digenetic (Leishmania), we found that the prenyl machinery remained unchanged. Geranylgeranyl transferase-I (GGTase-I) was absent in both Leishmania and its sister Leptomonas. Farnesyltransferase (FTase) and geranylgeranyl transferase-II (GGTase-II) were identified for protein prenylation. We predict that activity of the missing alpha-subunit (α-subunit) of GGTase-II in Leptomonas was probably contributed by the α-subunit of FTase, while beta-subunit (β-subunit) of GGTase-II was conserved and indicated functional conservation in the evolution of these two kinetoplastids. Therefore the β-subunit emerges as an excellent target for compounds inhibiting parasite activity in clinical cases of co-infections. We also confirmed that during the evolution to digenetic life style in Leishmania, the parasite acquired capabilities to evade drug action and maintain parasite virulence in the host with the incorporation of short-chain dehydrogenase/reductase (SDR/MDR) superfamily in Rab genes.
Conclusion:
Our study based on whole genome sequences is the first to build comparative evolutionary analysis and identification of prenylation proteins in Leishmania and its sister Leptomonas. The information presented in our present work has importance for drug design targeted to kill L. donovani in humans but not affect the human form of the prenylation enzymes.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/260">Elusive ditrysian phylogeny: an account of combining systematized morphology with molecular data (Lepidoptera)</a></div>
  <div class="description">Background:
Ditrysia comprise close to 99 % of all butterflies and moths. The evolutionary relationships among the ditrysian superfamilies have received considerable attention in phylogenetic studies based on DNA and transcriptomic data, but the deepest divergences remain for large parts unresolved or contradictory. To obtain complementary insight into the evolutionary history of the clade, and to test previous hypotheses on the subdivision of Ditrysia based on morphology, we examine the morphology of larvae, pupae and adult males and females of 318 taxa representing nearly all ditrysian superfamilies and families. We present the most comprehensive morphological dataset on Ditrysia to date, consisting of over 500 morphological characters. The data are analyzed alone and combined with sequence data (one mitochondrial and seven nuclear protein-coding gene regions, sequenced from 422 taxa). The full dataset consists of 473 exemplar species. Analyses are performed using maximum likelihood methods, and parsimony methods for the morphological dataset. We explore whether combining morphological data and DNA-data can stabilize taxa that are unstable in phylogenetic studies based on genetic data only.
Results:
Morphological characters are found phylogenetically informative in resolving apical nodes (superfamilies and families), but characters serving as evidence of relatedness of larger assemblages are few. Results include the recovery of a monophyletic Tineoidea, Sesioidea and Cossoidea, and a stable position for some unstable taxa (e.g. Epipyropidae, Cyclotornidae, Urodoidea + Schreckensteinioidea). Several such taxa, however, remain unstable even though morphological characters indicate a position in the tree (e.g. Immidae). Evidence supporting affinities between clades are suggested, e.g. a novel larval synapomorphy for Tineidae. We also propose the synonymy of Tineodidae with Alucitidae, syn. nov.
Conclusions:
The large morphological dataset provides information on the diversity and distribution of morphological traits in Ditrysia, and can be used in future research on the evolution of these traits, in identification keys and in identification of fossil Lepidoptera. The “backbone” of the phylogeny for Ditrysia remains largely unresolved. As previously proposed as an explanation for the scarcity of molecular signal in resolving the deeper nodes, this may be due to the rapid radiation of Ditrysia in the Cretaceous.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/259">A comparison of human and mouse gene co-expression networks reveals conservation and divergence at the tissue, pathway and disease levels</a></div>
  <div class="description">Background:
A deeper understanding of differences and similarities in transcriptional regulation between species can uncover important information about gene functions and the role of genes in disease. Deciphering such patterns between mice and humans is especially important since mice play an essential role in biomedical research.
Results:
Here, in order to characterize evolutionary changes between humans and mice, we compared gene co-expression maps to evaluate the conservation of co-expression. We show that the conservation of co-expression connectivity of homologous genes is negatively correlated with molecular evolution rates, as expected. Then we investigated evolutionary aspects of gene sets related to functions, tissues, pathways and diseases. Genes expressed in the testis, eye and skin, and those associated with regulation of transcription, olfaction, PI3K signalling, response to virus and bacteria were more divergent between mice and humans in terms of co-expression connectivity. Surprisingly, a deeper investigation of the PI3K signalling cascade revealed that its divergence is caused by the most crucial genes of this pathway, such as mTOR and AKT2. On the other hand, our analysis revealed that genes expressed in the brain and in the bone, and those associated with cell adhesion, cell cycle, DNA replication and DNA repair are most strongly conserved in terms of co-expression network connectivity as well as having a lower rate of duplication events. Genes involved in lipid metabolism and genes specific to blood showed a signature of increased co-expression connectivity in the mouse. In terms of diseases, co-expression connectivity of genes related to metabolic disorders is the most strongly conserved between mice and humans and tumor-related genes the most divergent.
Conclusions:
This work contributes to discerning evolutionary patterns between mice and humans in terms of gene interactions. Conservation of co-expression is a powerful approach to identify gene targets and processes with potential similarity and divergence between mice and humans, which has implications for drug testing and other studies employing the mouse as a model organism.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>November 19, 2015</h3>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/258">Maternal care in Acanthosomatinae (Insecta: Heteroptera: Acanthosomatidae)&#8212;correlated evolution with morphological change</a></div>
  <div class="description">Background:
Maternal care (egg-nymph guarding behavior) has been recorded in some genera of Acanthosomatidae. However, the origin of the maternal care in the family has remained unclear due to the lack of phylogenetic hypotheses. Another reproductive mode is found in non-caring species whose females smear their eggs before leaving them. They possess pairs of complex organs on the abdominal venter called Pendergrast’s organ (PO) and spread the secretion of this organ onto each egg with their hind legs, which is supposed to provide a protective function against enemies. Some authors claim that the absence of PO may be associated with the presence of maternal care. No study, however, has tested this hypothesis of a correlated evolution between the two traits.
Results:
We reconstructed the molecular phylogeny of the subfamily Acanthosomatinae using five genetic markers sequenced from 44 species and one subspecies with and without maternal care. Eight additional species from the other two acanthosomatid subfamilies were included as outgroups. Our results indicated that maternal care has evolved independently at least three times within Acanthosomatinae and once in the outgroup species. Statistical tests for correlated evolution showed that the presence of maternal care is significantly correlated with the secondary loss or reduction of PO. Ancestral state reconstruction for the node of Acanthosoma denticaudum (a non-caring species in which egg smearing with developed POs occurs) and A. firmatum (a caring species with reduced POs) suggested egg smearing was still present in their most recent common ancestor and that maternal care in A. firmatum has evolved relatively recently.
Conclusions:
We showed that maternal care is an apomorphic trait that has arisen multiple times from the presence of PO within the subfamily Acanthosomatinae. The acquisition of maternal care is correlated with the reduction or loss of PO, which suggests an evolutionary trade-off between the two traits resulting from physiological costs. This prediction also implies that presence of maternal care can be highly expected for those groups lacking behavioral data, which invariably also lack the organ. No secondary loss of maternal care was detected in the present tree. We suggest that the loss of maternal care may be suppressed due to the vulnerability of the PO-free condition, which thus maintains maternal care.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="indexb567.html?q=aggregator&amp;from=480">previous page</a></div><div>... <a href="indexd6af.html?q=aggregator&amp;from=420">22</a> <a href="index764e.html?q=aggregator&amp;from=440">23</a> <a href="index3d2c.html?q=aggregator&amp;from=460">24</a> <a href="indexb567.html?q=aggregator&amp;from=480">25</a> <strong>26</strong> <a href="indexbe68.html?q=aggregator&amp;from=520">27</a> <a href="indexf601-2.html?q=aggregator&amp;from=540">28</a> <a href="index25b4.html?q=aggregator&amp;from=560">29</a> <a href="index3ad0.html?q=aggregator&amp;from=580">30</a> ...</div><div><a href="indexbe68.html?q=aggregator&amp;from=520">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=500 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:39:10 GMT -->
</html>
