<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/14&from=240 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 21:45:00 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="index66e2.html?q=node/119">54(6) December 2005</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator/sources/14" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator">news aggregator</a>
<ul>
<li class="expanded"><a href="indexe5f5.html?q=aggregator/sources">sources</a>
<ul>
<li class="leaf"><a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></li>
<li class="leaf"><a href="index4421.html?q=aggregator/sources/8">CiteULike Phylogeny</a></li>
<li class="leaf"><a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></li>
<li class="leaf"><a href="index4dbd.html?q=aggregator/sources/14" class="active">EVOLDIR</a></li>
<li class="leaf"><a href="index8aad.html?q=aggregator/sources/9">Evolutionary Bioinformatics</a></li>
<li class="leaf"><a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></li>
<li class="leaf"><a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></li>
<li class="leaf"><a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></li>
<li class="leaf"><a href="index9fb6.html?q=aggregator/sources/22">Phyloseminar</a></li>
<li class="leaf"><a href="index69a2.html?q=aggregator/sources/10">Systematics Association</a></li>
<li class="leaf"><a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></li>
<li class="leaf"><a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></li>
<li class="leaf"><a href="indexe0bd.html?q=aggregator/sources/15">dechronization</a></li>
<li class="leaf"><a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></li>
<li class="leaf"><a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></li>
<li class="leaf"><a href="index1f4a.html?q=aggregator/sources/7">postgenomic.com</a></li>

</ul>
</li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 2 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a> &raquo; <a href="index5640.html?q=aggregator">news aggregator</a> &raquo; <a href="indexe5f5.html?q=aggregator/sources">sources</a></div><h2>EVOLDIR</h2>
<!-- begin content -->
<div id="aggregator"><div class="feed"><h3>URL</h3>
<div class="xml-icon"><a href="http://evoldir.tumblr.com/rss"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div><a href="http://evoldir.tumblr.com/">http://evoldir.tumblr.com/</a>
<h3>Last update</h3>
1 hour 40 min ago</div><h3>December 20, 2015</h3>
<div class="news-item">
 <div class="date">00:00</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135556207566">Graduate position: HumboldtU.ParasiteHostPopulationGenomics</a></div>
  <div class="description"><p>The Group &ldquo;Ecology and Evolution of Molecular Parasite-Host
Interactions&rdquo; of Juniorprofessor Emanuel Heitlinger at the Humboldt
University and the Leibniz Institute for Zoo and Wildlife research,
Berlin, Germany, invites applications for a:

PhD position in population genomics of parasite-host interaction

The position is offered for 3 years at the German 65% TV-L E13 pay
scale.

Research topic: We are establishing the house mouse hybrid zone (HMHZ)
close to Berlin as a system to study host-parasite interactions. In
the HMHZ two subspecies of the house mouse (Mus musculus domesticus
and Mus musculus musculus) meet and hybridize. We investigate parasite
specialization vs. generalism and recombination between strains of
unicellular parasites of the genus Eimeria. Eimeria spp. occur at high
prevalence (around 30%) and differ in specificity for the mouse
subspecies.

Requirements/Qualification of the PhD student:
•    Excellent degree (Master or equivalent) in (molecular) evolutionary
biology or computational biology/bioinformatics
•    Good communication skills in English
•    Some background in statistics and data analysis is required.
•    Experience in programming (R/Perl/Python/C) is an advantage, interest
in it is required.
•    Experience in genomics applications and next-generation sequencing
analysis is an advantage.
•    Background in evolutionary theory and population genetics is an
advantage.


The Heitlinger group works on ecology and evolution of parasite-host
interactions using tools from genomics and transcriptomics. The PhD
candidate will focus on genome evolution of Eimeria spp. in two
closely related subspecies of the house mouse. Phenotypic aspects
(immunology) of the same host individuals are investigated in an
ongoing project of another PhD student. A third PhD project in the
Heitlinger group is focused on gene expression of both host and
parsitized cells. The candidate is expected to closely collaborate
with both PhD students working on these ongoing projects and will be
directly supervised by Emanuel Heitlinger on genomics data analysis.

The Heitlinger group provides a multidisciplinary working
environment. Methods range from field studies, experiments &ldquo;at the
bench&rdquo; to computational analyses. The group is located at the
Department of Molecular Parasitology of the Humboldt University and
has access to laboratories and resources at the Leibniz Institute for
Zoo and wildlife research. This provides a diverse working environment
allowing interaction with researchers working on topics ranging from
conservation ecology to genetic manipulation of parasite biochemical
pathways.

The student has the opportunity to apply for an association with the
structured graduate training program GRK2046 &ldquo;Parasite Infections:
>From Experimental Models to Natural Systems&rdquo; integrating his project
in the Berlin community of parasitologists and providing additional
resources.

Please send your application as single pdf-file by January 31, 2016 to
emanuel.heitlinger@hu-berlin.de.


emanuelheitlinger@gmail.com


via Gmail</p></div>
 </div>
</div>
<h3>December 19, 2015</h3>
<div class="news-item">
 <div class="date">04:15</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135499542441">Conference:Biosphere2.PhenotypeRCN.February2016</a></div>
  <div class="description"><p>The Fifth Annual Summit of the Phenotype Ontology Research Coordination Network will be held at the University of Arizona&rsquo;s Biosphere2, about 40 miles north of Tucson, AZ, from February 26-28, 2016 (Friday through Sunday noon).  

The theme of this meeting will be &lsquo;Complex data integration with phenotypes&rsquo; with a focus on the integration of phenotype data with other data sets. We will summarize where our phenotype community is at with respect to integration with other data types, and we will highlight active projects. We will be looking to the future &ndash; what projects should be priorities for the future? Joining us this year will be folks from the newly funded 'FuturePhy&rsquo; (futurephy.org), who are interested in how to integrate multiple data types, including phenotype, with phylogenetic trees.  

We estimate that the costs for this meeting (transportation to meeting from airport, lodging, food) will be approximately $500, though we will be able to cover expenses for a small number of participants, particularly students and postdocs who have specific interests in using phenotypic data associated with environment in their research. Please contact one of us if you are interested in attending. It should be a great meeting!
 
Paula Mabee; pmabee@usd.edu
Eva Huala; huala@acoma.stanford.edu
Andy Deans; adeans@psu.edu
Suzanna Lewis; suzi@berkeleybop.org

The Phenotype Ontology RCN (<a href="http://bit.ly/1ZhgDVG">http://bit.ly/1ZhgDVG</a>) was funded by the U.S. NSF to establish a network of scientists who are interested in comparing phenotypes across species and in developing the tools and methods needed to enable comparisons.  In contrast to the many well-established efforts in the molecular community, the representation of phenotypic traits using ontologies is in its infancy. Phenotype ontologies, however, have the potential to integrate these data across all levels of the biological hierarchy and to the environment. This RCN is building a community that, because of its expertise, fosters communications across disciplines to enable co-development of interoperable community standards and best practices for phenotype.

via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135498760066">Conference: Pathogen Host Shifts, Cambridge UK, 12th April 2016</a></div>
  <div class="description"><p>This is a multi-part message in MIME format.
via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135498759911">Job: UToronto.GenomicsDataAnalyst</a></div>
  <div class="description">Institution/Company: University of Toronto

Location: Toronto, ON

Job Description:

Purpose

In support of research by PIs in the Department of Ecology and
Evolutionary Biology (EEB) the Genomics Data Analyst is expected to
conduct genome and transcriptome assembly and analysis of NGS
(next-generation sequencing) data, by using off-the-shelf software and
by writing and implementing scripts, to help solve important questions
about the evolution of plant and animal genomes.

Qualifications Required

Education:
Bachelor&rsquo;s degree in biological sciences, bioinformatics or computer
sciences, engineering, or statistics, or acceptable equivalent
combination of education and experience.

Experience:
Minimum 3 years experience working with genomic and/or transcriptomic
datasets; demonstrated experience working in a Linux/Unix shell
environment; competency with at least one scripting language (e.g.
Perl, Python, R). Demonstrated experience in analysis of NGS
(next-generation sequencing) with plant and polyploid genomes,
particularly whole genome assembly of large plant genomes, genome
annotation, and DNA and RNA structural modeling, is required.

Skills:
High proficiency with programming languages is required (e.g. C/C++,
Python, Java). Unix server administration and hardware specification. A
broad range of bioinformatics competencies demonstrated at a high
level. Basic understanding of statistics, probability, genetics, and
molecular biology. Good written and verbal communications skills,
organization skills, initiative, tact and analytical skills.
Reliability and good time management skills.

Other:
Ability to interface with diverse personnel and to work productively
both in collaborations with other team members and individually;
ability to balance workloads from multiple competing tasks; able to
effectively mentor and assist graduate students and postdoctoral
fellows in bioinformatics analyses.

If you are interested in applying for this position, please apply
online:

<a href="http://bit.ly/1Mn0uom">http://bit.ly/1Mn0uom</a>

Stephen Wright 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:31</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135498358816">Other: Tree with bootstraping from distance matrix</a></div>
  <div class="description"><p>Hi,

I&rsquo;m creating a tree from a distance matrix using phylip neighbor and 
phylib drawgram. I would like to have bootstrap values for the branches 
but all I have is this one distance matrix. Is this possible?

The distance matrix is calculated from a Genotyping by Sequencing (GBS) 
dataset with a few thousand SNPs. For each individual pair, the distance 
is basically the proportion of genotypes that differ for loci where both 
individuals are genotyped.

I could easily create different distance matrices by randomly sampling 
only some of the SNPs to calculate the distances, but that would not 
make much sense. For example, if I use a low proportion of the SNPs (ie: 
5%), the bootstrap values will end up being lower because the matrices 
will be more different from one another than if I use a high proportion 
of the SNPs (ie: 50+%). It feels like I can control the bootstrap values 
artificially, so that is not a good avenue.

Is there a way, given a single distance matrix, to create a tree with 
bootstrap values?

I&rsquo;ll be happy to summarize the answers and post them back on EvolDir.

Eric

via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:01</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135497557096">Conference: FloridaStateU.SoutheasternEcologyEvolution-SEEC.Mar11-13</a></div>
  <div class="description"><p>Dear colleagues,

We are excited to announce that registration is now open for the 2016
Southeastern Ecology and Evolution Conference (SEEC) at Florida State
University on March 11-13, 2016.

SEEC is a regional conference created by and for early-career scientists
in the fields of Evolution and Ecology. This is a great opportunity for
undergraduates to present for the first time, for graduate students to
practice for a larger meeting, or for anyone to present research in a
low-key setting. On a budget? It&rsquo;s just $35 for early bird registration
(until February 12). The first 40 student ASN members to register for
SEEC 2016 will receive a $30 discount on their registration costs,
making registration just $5!

Some of the great things in store this year:
- Plenary speaker Joseph Travis
- Alternative career panel lunch on Saturday

To register, visit our website at
<a href="http://bit.ly/1OezXkG.">http://bit.ly/1OezXkG.</a>

If you&rsquo;re not ready to send in your abstract yet, let us know you&rsquo;re
coming on Facebook.

Questions? Contact us at seec.host@gmail.com or on Facebook/twitter

via Gmail</p></div>
 </div>
</div>
<h3>December 17, 2015</h3>
<div class="news-item">
 <div class="date">23:57</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135430819351">Job: UVienna.SeniorScientistBioinformaticsCompBiol</a></div>
  <div class="description"><p>Job: UVienna.SeniorScientistBioinformaticsCompBiol

A permanent, full-time Senior Scientist position is available in the Department of Botany 
and Biodiversity Research, Faculty of Life Sciences, University of Vienna, Austria. The 
position is initially for 6 years with the possibility of subsequent unlimited employment. 
You will take the lead role in the implementation of research activities and approaches in 
bioinformatics and computational biology in our group. Our research focus is on the 
ecological & evolutionary genomics of adaptation to changing environments, speciation, 
and species diversification / radiation in plants and associated organisms. We address 
these topics using the latest DNA / RNA sequencing technologies in combination with 
spatial, ecological, and phenome data at different hierarchical levels of biological 
diversity.

University of Vienna offers an attractive and dynamic research location in a city with 
exceptional quality of life and in a country with excellent research infrastructure and 
funding provision.

Extent of Employment: 
40 hours/week. Salary grading in accordance with relevant work experience.

Job Description: 
Participation in cutting-edge research projects involving bioinformatics and 
computational biology, including the development / validation of new bioinformatic 
pipelines - Responsibility for the operation and maintenance of high performance 
computing resources - Liaison point for collaboration with external high performance 
computing centres - Participation in international publication and presentation activities - 
Supervision / mentoring of students engaged in bioinformatics and computational 
biology projects - Assistance with teaching in the department.
 
Profile: 
MSc or doctoral (or equivalent) degree in the natural or computer sciences - Experience 
with the analysis of so-called &lsquo;Next Generation Sequencing&rsquo; (NGS) data, i.e. second or 
third generation DNA / RNA sequence datasets - Extensive experience with 
computational work within a Unix / Linux based environment - Programming skills (bash 
scripting; R; basic skills in either Python or Perl) - Excellent written and oral 
communication skills in English - A keen interest to keep abreast with ongoing 
developments in bioinformatics and computational biology - Ability to work in teams - If 
from an educational path other than biology: basic understanding of biology including 
evolution.

To apply: 
Your application must include the following documents: 1-page motivation letter - 
Educational and scientific CV - List of publications - Contact details of three references. 
Please see full job description and submit your application via the website of the Job 
Center at the University of Vienna (<a href="http://bit.ly/1OAfxMY;">http://bit.ly/1OAfxMY;</a> email: 
jobcenter@univie.ac.at) no later than 01.02.2016, mentioning job reference no. 6254. 
For more information please contact: Christian Lexer, christian.lexer@univie.ac.at and 
christian.lexer@unifr.ch.


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:27</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135429601151">Graduate position: DurhamU.EvolutionaryEcology</a></div>
  <div class="description"><p>&ndash;_000_A3DEAA948B69DB4E8DDE546B54CA7FA72305A5BBCISAMRMBS01mdsa_
Content-Type: text/plain; charset=&ldquo;us-ascii&rdquo;
Content-Transfer-Encoding: quoted-printable

A competitive four year PhD studentship opportunity in evolutionary and ecological genetics is available at Durham University, UK.

Title:
The fitness consequences of inbreeding in wild flax.

Supervisory team:
Dr Adrian Brennan, Durham University, UK, with the co-supervision of Prof Rus Hoelzel, Durham University, UK, and Dr Kirsten Wolff, Newcastle University, UK.

Project description:
This project aims to improve our understanding of inbreeding and its relationship with fitness, a fundamental aspect of evolution. We will use wild flax as a model to study this important relationship, thereby raising the possibility of rapidly translating results into sustainably enhancing agricultural production via genetic improvement of crops. Our research in wild flax will allow for the comparison of natural populations exhibiting different levels of selfing, thereby revealing the extent to which natural selfing leads to purging and the development of a genome more resistant to inbreeding depression. We will use new genotyping by sequencing technologies to obtain an unprecedented genome-wide perspective on inbreeding and its evolution, allowing the identification of relevant genes, and therefore the consideration of molecular mechanisms.
Inbreeding is widely understood to reduce fitness through both the expression of recessive deleterious alleles in homozygotes, and through over-dominant allelic interactions. Even naturally highly inbreeding organisms cannot completely purge their genetic load by selection because of faster drift at reduced effective population size. For many crops, including flax, elite lines have been intensively inbred to optimize the stable expression of desirable traits. However, these same lines then suffer inbreeding depression.  Studies of flax have found up to 91 % heterotic increases in F1 seed yield following controlled crosses. However, the agronomic potential of heterosis has only been fully realized in hybrid varieties of maize so far. Only recently have genotyping technologies advanced to a stage where it is possible to test the long-held hypothesis that heterozygosity positively correlates with fitness using genome-wide data for many individuals of non-model organisms. Initial
  results are encouraging but more research is needed in organisms showing a wider range of mating systems and population structures to better understand the fitness consequences and genetic basis of inbreeding.  The student will be trained in next generation sequencing technologies, and integrate studies of wild and glass-house populations to test key hypotheses about inbreeding and fitness.

Eligibility and Funding:
For a UK-based applicant who is successful in competition with other PhD projects, the studentship will provide tuition fees, research funds and a maintenance allowance. A successful European economic area applicant will be provided tuition fees and research funds only.

Application details:
If interested in applying, please contact Dr Adrian Brennan, a.c.brennan@durham.ac.uk , with a CV and a covering letter. The deadline for applications is 29th January 2016.


&ndash;_000_A3DEAA948B69DB4E8DDE546B54CA7FA72305A5BBCISAMRMBS01mdsa_
Content-Type: text/html; charset=&ldquo;us-ascii&rdquo;
Content-Transfer-Encoding: quoted-printable


</p>
</div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:29</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135369745891">Conference: Marseilles.EvolBiol.Sep20-23.DeadlineJan31</a></div>
  <div class="description">Dear All

The early dead line for the,20th Evolutionary Biology Meeting at
Marseilles (September 20-23 2016), is January 31 2016



info: <a href="http://bit.ly/15fa2ML">http://bit.ly/15fa2ML</a> or
<a href="http://bit.ly/1mcBgPS">http://bit.ly/1mcBgPS</a>

best regards
Pierre 

PONTAROTTI Pierre 
via Gmail </div>
 </div>
</div>
<h3>December 16, 2015</h3>
<div class="news-item">
 <div class="date">01:09</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135307225791">Postdoc:UFlorida_WhitneyLab.CtenoEvoGeno</a></div>
  <div class="description"><p>POSTDOCTORAL POSITION
DEEPC: Diversity, Evolution, and EcoPhysiology of Ctenophores

An NSF-funded postdoctoral research position is currently available in the
Ryan group at the Whitney Laboratory for Marine Bioscience in Saint
Augustine, Florida (part of the University of Florida). The position is
part of the NSF-funded Diversity, Evolution, and EcoPhysiology of
Ctenophores (DEEPC) project. The goal of the project is to understand
evolution and diversification of Ctenophora using cutting-edge genomic
analyses and the deep-sea habitat as the generating force of novel
adaptations. We are collecting and generating transcriptomes and genomes
from many ctenophore species with habitat depths down to 4000 meters
below the surface.  The postdoctoral associate will use these molecular
data in a phylogenetic context to identify specific genetic events that
underlie physiological tolerances and adaptations to the unique
selective pressures of life in the deep sea.

Candidates should have or be close to obtaining a Ph.D. in bioinformatics,
computational biology, genetics, evolutionary biology,
ecology, or a closely related field. Skills in programming, phylogenetics,
statistical genetics, and/or the application of computational methods to
genomic data are desired. Familiarity with detecting episodic diversifying
selection using techniques such as branch-site models is highly desirable.
Candidates from groups that are traditionally underrepresented in the
sciences are strongly encouraged to apply.

Interested applicants should submit a curriculum vitae, a detailed letter
of interest, and the names of three potential referees to Joseph
Ryan at joseph.ryan@whitney.ufl.edu by January 1, 2016.

Ryan Group:  <a href="http://bit.ly/1Nwz8xJ">http://bit.ly/1Nwz8xJ</a>
Whitney Lab: <a href="http://bit.ly/1I8dvah">http://bit.ly/1I8dvah</a>








via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:09</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135305123456">Other: DryadLab.TeachingEvolution</a></div>
  <div class="description">&ldquo;DryadLab - Using real data to engage students in scientific inquiry&rdquo;: a
QUBES Faculty Mentoring Network

Spring 2016

Apply by January 1, 2016

DryadLab and Quantitative Undergraduate Biology Education Synthesis (QUBES)
are pleased to offer a unique networking and professional development
opportunity  from January V April 2016 for faculty interested in bringing
authentic research data into undergraduate classrooms.  DryadLab is an
educational extension of the Dryad Digital Repository, which contains
freely available research data from thousands of published scientific
papers. DryadLab provides open, high-quality, hands-on, data-driven
educational modules suitable for use in undergraduate classrooms, developed
by the authors of the original publications in collaboration with
experienced educators. Over the course of this network, faculty will
implement selected DryadLab  modules
in their courses with a focus on providing students with quantitative
skills.

QUBES (<a href="https://qubeshub.org/">https://qubeshub.org</a>) is an NSF-funded virtual center that
supports collaboration to promote quantitative biology education at the
undergraduate level.

Benefits of Participation

   -

   Access to ready-to-use teaching modules. Participants will incorporate
   at least 1 of the selected DryadLab modules in Spring 2016 classes
   -

   Access to peer mentors for effective teaching tips and strategies in
   small group virtual meetings every two weeks
   -

   Letter sent from QUBES to your department chair recognizing your
   contribution to teaching scholarship


Please visit <a href="http://bit.ly/1JbcjOf">http://bit.ly/1JbcjOf</a> for more details and
to apply.

Questions? Contact Sam Swauger (samswauger@datadryad.org) or Alison Hale (
anm116@pitt.edu).

via Gmail</div>
 </div>
</div>
<h3>December 15, 2015</h3>
<div class="news-item">
 <div class="date">23:53</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135304481586">Graduate position: SystemsPharmacol.UTromso</a></div>
  <div class="description"><p>PhD student in Systems Pharmacology (Mathematical Modelling)

———

A 4-year PhD student position is available immediately in the Systems Pharmacology group of Dr. Pia Abel zur Wiesch. The position is affiliated with the Centre for Molecular Medicine Norway (NCMM), Nordic EMBL Partnership Molecular Medicine and is located at the University of Tromso, Norway. While the primary location is in Tromso, the candidates have the option to visit Yale School of Public Health. The candidate is expected to spend substantial time at ETH Zurich, Switzerland.
——

About the group

The Systems-Pharmacology group (<a href="http://bit.ly/1NSEyH2">http://bit.ly/1NSEyH2</a>) is headed by Dr. Abel zur Wiesch, Tenure Track Associate Professor (seniorforsker) at UiT, NCMM Young Associate Investigator (<a href="http://bit.ly/1YjuSXt">http://bit.ly/1YjuSXt</a>) and Visiting Assistant Professor at Yale School of Public Health. The aim of our work is to improve therapy in infectious diseases and cancer by using mathematical models to predict how much drug a patient should receive and how long and how often the patient should be treated. Our mathematical models describe both the intracellular reaction kinetics of drug-target binding as well as growth and death of populations of bacterial or cancer cells (Abel zur Wiesch & al., Science Transl. Med. 2015). We collaborate closely with experimentalists and clinicians. Two examples of ongoing projects are:

- link single cell analyses (John McKinney, EPFL Lausanne, <a href="http://bit.ly/1NSEwyS">http://bit.ly/1NSEwyS</a>) to clinical studies (<a href="http://bit.ly/1YjuSXv">http://bit.ly/1YjuSXv</a> to improve TB therapy, funded by the Bill & Melinda Gates foundation 

- develop new extended release technologies together with gastroenterologists at Harvard (Giovanni Traverso, <a href="http://bit.ly/1NSEyH4">http://bit.ly/1NSEyH4</a>) and MIT (Andrew Bellinger <a href="http://bit.ly/1YjuSXw">http://bit.ly/1YjuSXw</a>). 

The most important collaborator for the PhD projects is Dr. Roland Regoes (ETH Zurich, <a href="http://bit.ly/1NSEwyU">http://bit.ly/1NSEwyU</a>). The projects will combine mathematical modeling with the analysis of clinical and experimental data. Both the Systems Pharmacology group and the Regoes group at ETH have documented expertise in all of these areas (Abel zur Wiesch & al., Science Transl. Medicine 2015, Abel zur Wiesch & al., Lancet Infectious Diseases 2011, Abel, Abel zur Wiesch & al., Nature Methods 2015, Diard & al., Nature 2013, Kaiser, Regoes & al., PLOS Biology, Regoes & al. AAC 2004). The project therefore offers a unique opportunity to get experience in a wide range of interesting research topics.
——

Your profile

We seek candidates that appreciate working in an international and interactive environment, are highly enthusiastic about basic research with medical applicability, motivated and willing to integrate in a young team of researchers. You should have solid programming skills and a strong quantitative background and be passionate about applying your skills to biological and medical questions. A strong interest in Public Health, Pharmacology, Infectious Diseases, Biochemistry and/or Mathematical Biology is desirable. Previous experience with mathematical modeling and/or analysis of large datasets is an advantage. Good command of English and scientific writing skills are required. Applicants must hold a Master&rsquo;s degree (or equivalent).
——

We offer

The successful candidate will work collaboratively with other faculty, scientists, and students in a dynamic and multi-disciplinary environment. We offer the unique opportunity of being embedded in several scientific environments at the UiT in Tromso, the Centre for Molecular Medicine Norway (NCMM) in Oslo and Yale School of Public Health (YSPH) in Boston. We strongly encourage career development beyond authorship of research manuscripts arising from this work. We will help the successful candidate in developing their own interests and also encourage attending international conferences and visiting collaborators. The salary will be commensurate with experience and in accordance with the Norwegian State salary scale code 1017 (starting approx. at EURO 46’500/ NOK 429’700 per year). UiT is a family friendly work environment and offers very attractive and internationally competitive benefits including pension arrangements, healthcare and other welfare benefits.
——

How to apply

The application deadline is January 12 2016 and must be submitted in English. The following documents are required:

* 1-2 page cover letter explaining your motivation and research interests
* CV containing an overview of education, supervised professional training and professional work
* A complete list of publications 
* Copies of diploma and transcript from completed degree(s) (translated to English or a Scandinavian language; certified copies must only be provided upon request)
* 2-3 reference letters (can be sent directly to Dr. Abel zur Wiesch or uploaded at jobbnorge.no)
* Documentation of English language proficiency may be required upon request
* A comparison between the candidate’s grades and the average grade of students in the same program may be requested later in the application process.

To apply please follow the instructions on the official University of Tromso recruiting homepage.

<a href="http://bit.ly/1YjuSXx">http://bit.ly/1YjuSXx</a>

For more information, please visit our webpage at 

<a href="http://bit.ly/1NSEyH7">http://bit.ly/1NSEyH7</a>

or

<a href="http://bit.ly/1YjuSXt">http://bit.ly/1YjuSXt</a>

Location:
Tromso, Norway (NO)
Zurich, Switzerland
New Haven, CT, USA

Salary band:
$50,000 - $74,999

Salary
starting at NOK 429’700 per year

Fields
Life Sciences, Biochemistry, Bioinformatics, Biomedical Sciences, Computational Biology, Evolutionary Biology, Microbiology, Pharmacokinetics / Pharmacodynamics, Pharmacology, Physical Sciences, Chemistry, Computer Sciences, Informatics, Mathematics, Health Sciences, Drug Development, Drug Discovery

Description
Full time position

PhD Fellowship
PhD Studentship

Date
Deadline January 12


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:07</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135241563976">Postdoc: Trondheim_Norway.EvolutionaryGenomics</a></div>
  <div class="description">A two-year postdoctoral position in evolutionary genomics funded by
a grant from the Norwegian Research Council to Assistant Professor
Arild Husby is available at the Centre for Biodiversity Dynamics (CBD),
Department of Biology,  NTNU, Norway.

Job description
The aim of the project is to examine ecological adaptation at the
molecular genetic level using already available SNP genotype data
from 4000 individuals genotyped on a 200k SNP array. The data has been
collected in a long-term study of a house sparrow model system consisting
of insular populations at the coast of Norway, with extensive pedigrees
and individual-level data on phenotypic trait values and fitness. The
two main goals of the post doc holder will be to use the SNP data to
test for signs of selection on the genomic level. This work will be
combined with ongoing GWAS work to link signs of selection in the genome
to the genetic variants underlying phenotypic traits, thereby providing
a link between population genomic approaches and quantitative genetic
work. The second aim is to use recently developed statistical methods
(Ovaskainen et al. 2011, Genetics 109), Berg & Coop 2014, PLoS Genet 10)
to test for polygenic adaption examining correlated changes in allele
frequencies in traits known to be under divergent selection on different
islands (Holand et al. 2011, Genet Res 93). Hence, the overall aim of the
project is to provide insights into the molecular genetic basis of trait
variation in natural populations. The successful candidate will have the
possibility to develop his/her own ideas to approach the above issues,
including opportunities for fieldwork and laboratory work.

The work will be carried out primarily in the research group of Assistant
Professor Arild Husby, but will be in close collaboration with Associate
Professor Henrik Jensen, and Professor Otso Ovaskainen. Ass. Prof. Husby
and Prof. Ovaskainen are jointly based at the Centre for Biodiversity
Dynamics (CBD), NTNU, and the Metapopulation Research Centre (MRC),
University of Helsinki, whereas Ass. Prof. Jensen is based at CBD. Both
CBD and MRC are Centres of excellence in Norway and Finland, respectively.

More information can be found here: www.ntnu.edu/cbd, and here:
<a href="http://bit.ly/1P4qzwV.">http://bit.ly/1P4qzwV.</a>

The candidate will be employed at CBD (Department of Biology, NTNU)
but will be expected to spend considerable time at MRC. The working
language at both institutions is English.

For further information about the postdoctoral position please contact
Assistant Professor Arild Husby (Arild.Husby@helsinki.fi, phone
no. +358294157691). Informal enquiries encouraged!

Required qualifications and personal qualities
Candidates must hold a PhD in biology and be able to document expertise
in statistical/quantitative genetics and population genomics. Experience
with handling data from next-generation sequencing and/or high-density SNP
genotyping techniques and statistical analyses of such data is strongly
desired and will be considered an advantage. The successful candidate
must be able to work efficiently and goal-oriented both when working
independently and when collaborating with others. We are looking for a
highly motivated and enthusiastic candidate.

What we offer
We can offer an exciting postdoctoral research project with focus on
fundamental questions in evolutionary biology using state-of-the-art
molecular genetic analyses. The postdoctoral researcher will carry
out scientifically challenging research with good opportunities
for development of her/his expertise. The successful candidate will
benefit from a good working environment and the expertise of members
in our research groups which span quantitative genetics, statistical
genetics/genomics and mathematical modelling.

Preferred starting date is in February 2016 but this is negotiable.The
appointment of the Postdoc will be made according to Norwegian guidelines
for universities and university colleges and to the general regulations
regarding university employees. NTNUs personnel policy objective is
that the staff must reflect the composition of the population to the
greatest possible extent.

The position as Postdoc is remunerated according to the Norwegian State
salary scale. There is a 2% deduction for superannuation contribution.

Application
The application with CV, pdfs of publications, reference letters and
relevant certificates must be submitted electronically through this
page. Applications submitted elsewhere will not be considered.  Please
indicate preferred starting date in your cover letter.  The reference
number of the position is: NT 84/15

Deadline for applying: January 15, 2016

Apply through this link:
<a href="http://bit.ly/1O6omnB">http://bit.ly/1O6omnB</a>


Arild Husby, Assistant Professor
Department of Biosciences (Biocenter 3, office 5415),
University of Helsinki
PO box 65,
FI-00014 Helsinki,
Finland

web: <a href="http://bit.ly/11XzFaY">http://bit.ly/11XzFaY</a><a href="http://www.ntnu.edu/cbd,">www.ntnu.edu/cbd,</a> and here:
http://bit.ly/1P4qzwV.

The candidate will be employed at CBD (Department of Biology, NTNU)
but will be expected to spend considerable time at MRC. The working
language at both institutions is English.

For further information about the postdoctoral position please contact
Assistant Professor Arild Husby (Arild.Husby@helsinki.fi, phone
no. +358294157691). Informal enquiries encouraged!

Required qualifications and personal qualities
Candidates must hold a PhD in biology and be able to document expertise
in statistical/quantitative genetics and population genomics. Experience
with handling data from next-generation sequencing and/or high-density SNP
genotyping techniques and statistical analyses of such data is strongly
desired and will be considered an advantage. The successful candidate
must be able to work efficiently and goal-oriented both when working
independently and when collaborating with others. We are looking for a
highly motivated and enthusiastic candidate.

What we offer
We can offer an exciting postdoctoral research project with focus on
fundamental questions in evolutionary biology using state-of-the-art
molecular genetic analyses. The postdoctoral researcher will carry
out scientifically challenging research with good opportunities
for development of her/his expertise. The successful candidate will
benefit from a good working environment and the expertise of members
in our research groups which span quantitative genetics, statistical
genetics/genomics and mathematical modelling.

Preferred starting date is in February 2016 but this is negotiable.The
appointment of the Postdoc will be made according to Norwegian guidelines
for universities and university colleges and to the general regulations
regarding university employees. NTNUs personnel policy objective is
that the staff must reflect the composition of the population to the
greatest possible extent.

The position as Postdoc is remunerated according to the Norwegian State
salary scale. There is a 2% deduction for superannuation contribution.

Application
The application with CV, pdfs of publications, reference letters and
relevant certificates must be submitted electronically through this
page. Applications submitted elsewhere will not be considered.  Please
indicate preferred starting date in your cover letter.  The reference
number of the position is: NT 84/15

Deadline for applying: January 15, 2016

Apply through this link:
http://bit.ly/1O6omnB


Arild Husby, Assistant Professor
Department of Biosciences (Biocenter 3, office 5415),
University of Helsinki
PO box 65,
FI-00014 Helsinki,
Finland

web: http://bit.ly/11XzFaY
mail: arild.husby@helsinki.fi
office phone: +358294157691


&ldquo;Husby, Arild&rdquo; 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:38</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135240570536">Graduate position: LMU_Munich.Paleobiology</a></div>
  <div class="description">Dear EvolDir readers,

we would like to announce that application to our Master Program
Geobiology and Paleobiology (MGAP), starting in the winter semester
2016/17 is now open for international students and kindly ask to forward
the attached message to interested students.

With kind regards, 
Oliver Voigt


*Applications invited for the Master&rsquo;s program &ldquo;Geobiology and
Paleobiology&rdquo; (MGAP) at the Ludwig-Maximilians-Universitt (LMU) Munich
(Germany)*

 The Master&rsquo;s program &ldquo;Geobiology and Paleobiology&rdquo; (MGAP) is a
 consecutive, research-focussed, two-year Master of Science program at
 the Faculty of Geosciences of the Ludwig-Maximilians-Universitt (LMU)
 Munich, in collaboration with researchers of the Bavarian Natural History
 Collections and the GeoBio-Center @ LMU. MGAP aims to provide students
 with a comprehensive introduction into the interdisciplinary research
 fields of geobiology and paleobiology to prepare them for careers in
 science and beyond.

 The MGAP program is based on interdisciplinary, research-oriented
 courses in geo- and biosciences that address patterns and processes
 of evolutionary and environmental geobiology and paleobiology. The
 module-based curriculum provides an integrative approach facilitated
 by experts in different areas of expertise. Students will learn and
 acquire routines with a wide range of scientific methods such as modern
 techniques in molecular biology, fieldwork, collection management,
 comparative morphology, phylogeny, bioinformatics, statistics and
 (paleo-) biodiversity assessments. Students will learn independent
 scientific work in individual and intensively supervised research
 projects at early stages of the curriculum.

*MGAP in brief*

        - International Master&rsquo;s program in Geobiology and Paleobiology
          (Master of Science, M.Sc.) at the LMU Munich

        - 2 years, 4 semesters, start in October (winter semester)

        - All courses taught in English

        - Course total: 120 credit points (ECTS)

 

More information is available on the program&rsquo;s website:


For the winter semester 2016-2017 (courses start mid-October), application
dates are as follows:

1) January 31, 2016 for non-EU international students (and EU students
   from outside of Germany. but application is also possible later),
   applications are possible now: 

2) February 15 to May 31, 2016 for EU students (including German
   students).

 All applications have to be submitted via an online submission portal:



Oliver Voigt 
via Gmail</div>
 </div>
</div>
<h3>December 14, 2015</h3>
<div class="news-item">
 <div class="date">23:37</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135238113986">Postdoc: McGillU_Montreal.ComputationalGenomics</a></div>
  <div class="description"><p>Postdoctoral position in computational genomics


A postdoctoral position is available immediately in the Computational
Genomics Laboratory (<a href="http://bit.ly/1UsnGYE">http://bit.ly/1UsnGYE</a>), Lady Davis Institute of
Medical Research and the Ludmer Centre for Neuroinformatics and Mental
Health. The center focuses on understanding the genetic, epigenetic and
environmental influences that shape brain development, including structure
and function, and the risk for mental disorders. This Centre spans the Lady
Davis Research Institute, the Douglas Mental Health University Institute
and the Montreal Neurological Institute, all affiliated with McGill
University, Montreal, Quebec, Canada.

Our lab uses genome-wide approaches to understand mechanisms regulating
gene expression and RNA processing in the developing brain, their interplay
with genetic and epigenetic factors, and how they cause disease. We work in
an interdisciplinary environment, interacting with statisticians,
physicists and molecular biologists. Successful candidates will conceive
and develop algorithms and analytical approaches to integrate large
genetic/epigenetic/expression datasets, generated both in-house and from
public sources, primarily from next-generation sequencing technologies.

Qualifications and Experience
We are looking for a very motivated individual capable of carrying out
independent research, and to interact in a collaborative setting


-> Doctoral degree in computer science, bioinformatics, statistics,
molecular biology, genetics or a related field

-> Strong programming skills are essential

-> Experience within one of the following areas is an asset, but not
required: genomics, neuroscience, statistical interpretation and analysis
of next-generation sequencing datasets.


For applying, please send a CV, a brief statement of scientific interest
and contact information for at least two references to
claudia.kleinman@mcgill.ca


cl.kleinman1@gmail.com


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:36</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135238113586">Postdoc: NewZealand.FishPhylogenetics</a></div>
  <div class="description">SUBJECT: Postdoc.NewZealand.Fish phylogenetics and community ecology

Postdoctoral Research Fellow
Institute of Natural & Mathematical Sciences, College of Sciences, Massey
University, Auckland, New Zealand

Apply here:
<a href="http://bit.ly/1QkodNY">http://bit.ly/1QkodNY</a>

We are seeking a Postdoctoral Research Fellow to contribute to the project
Synergising ecology and evolution: discovering patterns of functional and
phylogenetic diversity of New Zealand&rsquo;s marine fishes versus depth, funded
by the Marsden Fund, Royal Society of New Zealand. Applications close: 19
January (11:45pm, New Zealand Standard time).

Project Overview
Biologists know least about the largest habitat on earth - the deep sea.
Fundamental patterns of functional and phylogenetic biodiversity, from
coastal waters to the abyss, have never been quantified. Do biodiversity
trends in the ocean follow the patterns in alpha and beta diversity seen in
the terrestrial environment? New Zealands latitudinal range and nearshore
continental shelf offers a unique opportunity to address trends in
biodiversity and community structure and to disentangle the evolutionary
and ecological drivers of these patterns. This project will complement an
existing stratified and replicated depth by latitude dataset describing
fish community composition. Specifically, this project aims to combine
phylogenetic and functional information with knowledge of community
composition for the fishes of New Zealand and to understand the ecological
and evolutionary drivers of community composition from the sub-tropics to
the sub-antarctic, and from the shallows to the abyss. The results of these
analyses will have a major impact on the interpretation of the ecology and
evolutionary history of New Zealands fish communities and biodiversity
patterns in the ocean, and will contribute novel methods of analyses for
biodiversity research.

Position Overview
We are seeking a motivated and innovative researcher, keen to make their
mark in the field of quantitative biodiversity research. The ideal
candidate will have a PhD in biology and will have experience in
phylogenetics, diversification analyses, biogeographic analysis,
comparative methods, trait-based analyses, ecological modeling,
multivariate statistics, macroecology, and community ecology. Applicants
with only a subset of these skills are encouraged to apply. A track record
of publication in international journals and relevant statistical,
computational and programming skills are expected. The appointed Fellow
will conduct phylogenetic analyses of new and data-mined sequence data, and
will combine these data with existing community data, and functional trait
data to describe biodiversity patterns. The Fellow will contribute to the
development of statistical methods to help test explicit hypotheses that
will disentangle evolutionary and ecological drivers of diversity and
turnover. The Fellow is not expected to have any prior experience in marine
fieldwork, molecular genetic benchwork, or ichthyology. The appointee will
be encouraged to supplement the primary research goals of the project with
their own research objectives, in consultation with the wider research team
that includes Prof. Marti J. Anderson (co-Primary Investigator, Chair in
Statistics, Massey University), Dr. Clive Roberts (co-Primary Investigator,
Head Curator of Fishes, Museum of New Zealand Te Papa Tongarewa), Dr. Libby
Liggins (mentor/supervisory role, Lecturer in Marine Ecology, Massey
University) and a recruited PhD candidate (responsible for functional trait
analysis, Massey University). The Fellow will be expected and supported to
publish and present their findings to international peers. Proficiency in
written and spoken English is required.

Position specifics
The position is three years, starting March 2016 (a later start date may be
negotiated for an exceptional candidate). The Fellow will be based in the
Institute of Natural and Mathematical Sciences (INMS) at Massey
Universitys campus in Auckland, New Zealand. INMS is a recently formed
multi-disciplinary research and teaching Institute incorporating Biology,
Chemistry, Computer Science, Mathematics, Physics and Statistics. In
conjunction with world-class researchers in the closely-linked Institute
for Advanced Study (IAS), INMS has special strengths in quantitative
biology and ecology, evolution, genetics, mathematical biology,
computational science and modelling of complex systems. Massey University
is in the north of Auckland, with easy access to the city of Auckland (New
Zealands largest and most culturally diverse city) and recreational areas
such as the harbour, islands, beaches, and forest parks. The Fellow will be
joining a dynamic team of researchers working on the frontier of
statistics, ecology, and evolution. The Fellow will also work closely with
scientists of Te Papa Tongarewa, Museum of New Zealand, in Wellington, New
Zealands capital city. Te Papa holds New Zealands National Fish
Collection and supports an active ichthyology research team.

Enquiries regarding the funded project and specific role of the Fellow
should be directed to Dr. Libby Liggins (Email: l.liggins@massey.ac.nz) or
Prof. Marti J. Anderson (Email: m.j.anderson@massey.ac.nz).
Information about the Institute of Natural and Mathematical Sciences can be
found at <a href="http://bit.ly/1UsnGIp;">http://bit.ly/1UsnGIp;</a> information about Te Papa Tongarewa can
be found at <a href="http://bit.ly/1Qkoe4e.">http://bit.ly/1Qkoe4e.</a>

Libby Liggins 
via Gmail</div>
 </div>
</div>
<h3>December 13, 2015</h3>
<div class="news-item">
 <div class="date">23:22</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135170344851">Job: CaliforniaStateU_MontereyBay.AquaticEvolution</a></div>
  <div class="description">Our department at California State University, Monterey Bay is
currently looking for a freshwater aquatic biologist. Take a look at
the job description and if you are qualified and  interested, please
apply at <a href="http://bit.ly/1J6cKtf.">http://bit.ly/1J6cKtf.</a>
Regards,
Nate Jue

Position Title: Assistant Professor of Aquatic Ecology and Water
Quality
Degree/Program BS: Environmental Science & Technology, Environmental
Studies, Biology, Marine Science / MS: Applied Marine & Watershed,
Marine Science
Priority Screening Date: 01/18/2016
Recruitment Status: Open Until Filled
Position Description:
The Division of Science & Environmental Policy (SEP) at California
State University, Monterey Bay (CSUMB) seeks applicants with a
demonstrated commitment to teaching and research in freshwater aquatic
ecology and water quality within natural, agricultural, and urban
landscapes in California&rsquo;s Central Coast region.
Projected responsibilities would likely include the following:
 - Teach upper division course in aquatic ecology (and co-listed
graduate course)
 - Teach upper division course in environmental monitoring - using
the study of biological and chemical water quality as a case study in
the application of experimental design, contemporary statistics, and
quantitative reasoning
 - Contribute as needed to teaching upper division course in critical
thinking and communication in environmental science and/or lower
division course in statistics (using R software)
 - Employ innovative teaching and learning techniques that engage
students through the use of technology, hands on experience, and
problem solving
 - Develop and maintain applied research activities with relevance to
aquatic ecological and water quality issues  within perennial and
non-perennial streams, coastal lagoons, vernal pools, lakes, and/or
reservoirs in California&rsquo;s Central Coast region
 - Provide research opportunities for undergraduates, including
involving students in field-based sampling of aquatic organisms and
monitoring of aquatic biological and chemical variables
 - Pursue and obtain extramural funding
Minimum Qualifications
 - Ph.D. in a field relevant to the requirements of this position, by
date of appointment (August 17, 2016)
 - Ability to facilitate student learning effectively for diverse
learners in a wide range of courses
 - Demonstrated excellence in instruction and commitment to
undergraduate education
 - Applied expertise in aquatic ecology and water quality
 - Applied expertise in application of contemporary statistical
approaches using R software
 - Excellent written and oral communication skills
Preferred Qualifications
 - Demonstrated ability and desire to mentor and teach students from
diverse cultural, educational, and economic backgrounds, and a
commitment to work effectively in a culturally diverse campus community
 - Research experience and interests that are relevant to freshwater
and lagoon environments within 1-hour of the CSUMB campus
 - Research and teaching interests and expertise that complement and
strengthen those of existing faculty in the department
 - Experience and interest in mentoring undergraduate & graduate
student research and/or internships
 - Successful record in obtaining external funding
 - Success in obtaining relevant permits e.g. for sampling aquatic
organisms
 - Experience working with Institutional Animal Care and Use Committees
(IACUCs)

Nathaniel K. Jue, Ph.D.
Assistant Professor
Division of Science and Environmental Policy
California State University, Monterey Bay
Seaside, CA 93955

Nathaniel Jue 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:52</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135101182801">Postdoc:North Carolina State U Applied Evol Biology</a></div>
  <div class="description"><p>&ndash;Apple-Mail=_9353E08B-C0E7-4F50-9E14-E31C2AB12FBD
Content-Transfer-Encoding: quoted-printable
Content-Type: text/plain;
	charset=windows-1252

Postdoc: Applied Evolutionary Biology

PROJECT DESCRIPTION: 

	   Selfish genetic elements have been studied for over a century, and as far back as the 1960&rsquo;s researchers became interested in using the power of selfish genetic elements to drive genes into pest species to suppress their impacts (Gould et al. 2006). Until recently, the focus of applied work was on naturally occurring elements. In the past decade some progress was made on developing synthetic elements that mimicked natural meiotic drive and selective embryo-killing, but de-novo creation of a gene-drive system in a pest species was elusive. With the harnessing of the bacterial CRISPR-Cas9 system in the past few years there has been a revolution brewing in this field (Esvelt et al. 2014, Oye et al. 2014). In March 2015 a pivotal article by Gantz and Bier (2015) came out in Science demonstrating a CRISPR-Cas9 construct in Drosophila with strong gene drive. 

            In the past month, two empirical papers came out that demonstrate CRISPR-Cas9 gene drive in the mosquito that transmits malaria and the mosquiot that transmits dengue (Gantz et al. 2015, Hammond et al. 2015). Not everyone is comfortable with these developments and there has even been a call for a moratorium on certain experiments. There are also concerns about nefarious use of the technology. 

            The  bottom-line is that progress in molecular biology is ahead of the population genetic work needed to build systems that are less risky but accomplish changes in the public interest.

              We have been funded by the NIH and the W. M. Keck Foundation to conduct this kind of population genetic research. Our focus has been on mosquitoes that transmit dengue and malaria, but we are also interested in other biological systems (There is hope that these selfish genetic elements can save endangered species like Hawaiian honeycreepers and specific seabirds (Gould 2008, Esvelt et al. 2014).

            The postdoc in this position will build a set of simple to complex models to examine the expected dynamics of gene drive systems in mosquitoes and other taxa.

            The most detailed model that we have developed simulates the population dynamics and population genetics of Aedes aegypti, the vector of dengue, in a city on the Amazon river, Iquitos, for which there are rich data sets on both mosquito dynamics and dengue epidemiology (e.g. Magori et al. 2009, Okamoto et al 2014). An accompanying epidemiological model is currently under development. The goals of two other postdocs in our group are to expand the mosquito model and the human epidemiology model to encompass the entire city of about 400,000 people. The postdoc in this new position will collaborate with the other postdocs to use these detailed models to test gene drive systems, but will also develop more generic models (e.g. Huang et al. 2010).

            In addition to working on model development and analysis, the person in this position will have the opportunity to collaborate in an interdisciplinary research group composed of mosquito ecologists, disease epidemiologists, molecular biologists, biomathematicians, ethicists, and scientists from disease-endemic countries. We are dedicated to taking seriously the ethical and political issues surrounding this technology.



DESIREABLE SKILLS: A background in population genetics and the ability to program in C++ (or knowledge of a related programming language), and training in evaluation of mechanistic models.

TO APPLY: email a cover letter and CV to Fred_Gould@ncsu.edu

 

References:

 Esvelt,  K. M.,  A. L. Smidler, F. Catteruccia, G. M. Church. 2014. Concerning RNA-guided gene drives for the alteration of wild populations. eLife. 10.7554/eLife.03401.

 Gantz, V. M.  and Bier, E. 2015. The mutagenic chain reaction: A method for converting heterozygous to homozygous mutations. Science 24 April 2015- 442-444. 

Gantz, V.M., N. Jasinskiene, O.Tatarenkova, A. Fazekas, V. M. Macias, E. Bier, and A. A. James. 2015. Highly efficient Cas9-mediated gene drive for population modification of the malaria vector mosquito Anopheles stephensi. PNAS 2015 112 (49) E6736-E6743; published ahead of print November 23, 2015,doi:10.1073/pnas.1521077112

 Gould, F. 2008. Broadening the application of evolutionarily based genetic pest management. Evolution 62: 500-510. 

Hammond, A. et al. 2015. A CRISPR-Cas9 gene drive system targeting female reproduction in the malaria mosquito vector Anopheles gambiae. Nature Biotechnology   Published online 07 December 2015   doi:10.1038/nbt.3439

 Huang, Y., Lloyd, A.L., Legros, M., Gould, F. 2010. Gene-drive into insect populations with age and spatial structure: a theoretical assessment. Evol. Appl. ISSN 1752-4571.

 Magori, K., M. Legros, M. Puente, D. A. Focks, T. W. Scott, A. Lloyd, F. Gould. 2009. Skeeter Buster: a stochastic, spatially-explicit modeling tool for studying Aedes aegypti population replacement and population suppression strategies. PLoS Negl Trop Dis 3(9): e508. doi:10.1371/journal.pntd.0000508

 Okamoto, K. W., Robert M. A., Gould, F., Lloyd, A. L. 2014. Feasible Introgression of an Anti-pathogen Transgene into an Urban Mosquito Population without Using Gene-Drive. PLoS Negl Trop Dis 8(7): e2827. doi:10.1371/journal.pntd.0002827.



 
&ndash;Apple-Mail=_9353E08B-C0E7-4F50-9E14-E31C2AB12FBD
Content-Transfer-Encoding: quoted-printable
Content-Type: text/html;
	charset=windows-1252

</p><p class="MsoNormal">Postdoc: Applied Evolutionary Biology</p><p class="MsoNormal">PROJECT DESCRIPTION: </p><p class="MsoNormal">	   Selfish genetic elements have been studied for over a century, and as far back as the 1960&rsquo;s researchers became interested in using the power of selfish genetic elements to drive genes into pest species to suppress their impacts (Gould et al. 2006). Until recently, the focus of applied work was on naturally occurring elements. In the past decade some progress was made on developing synthetic elements that mimicked natural meiotic drive and selective embryo-killing, but de-novo creation of a gene-drive system in a pest species was elusive. With the harnessing of the bacterial CRISPR-Cas9 system in the past few years there has been a revolution brewin
 g in this field (Esvelt et al. 2014, Oye et al. 2014). In March 2015 a pivotal article by Gantz and Bier (2015) came out in Science demonstrating a CRISPR-Cas9 construct in Drosophila with strong gene drive. </p><p class="MsoNormal">            In the past month, two empirical papers came out that demonstrate CRISPR-Cas9 gene drive in the mosquito that transmits malaria and the mosquiot that transmits dengue (Gantz et al. 2015, Hammond et al. 2015). Not everyone is comfortable with these developments and there has even been a call for a moratorium on certain experiments. There are also concerns about nefarious use of the technology. </p><p class="MsoNormal">            The  bottom-line is that progress in molecular biology is ahead of the population genetic work needed to build systems that are less risky but accomplish changes in the p
 ublic interest.</p><p class="MsoNormal">              We have been funded by the NIH and the W. M. Keck Foundation to conduct this kind of population genetic research. Our focus has been on mosquitoes that transmit dengue and malaria, but we are also interested in other biological systems (There is hope that these selfish genetic elements can save endangered species like Hawaiian honeycreepers and specific seabirds (Gould 2008, Esvelt et al. 2014).</p><p class="MsoNormal">            The postdoc in this position will build a set of simple to complex models to examine the expected dynamics of gene drive systems in mosquitoes and other taxa.</p><p class="MsoNormal">            The most detailed model that we have developed simulates the population dynamics and population genetics of <i>Aedes aegypti</i>, the vecto
 r of dengue, in a city on the Amazon river, Iquitos, for which there are rich data sets on both mosquito dynamics and dengue epidemiology (e.g. Magori et al. 2009, Okamoto et al 2014). An accompanying epidemiological model is currently under development. The goals of two other postdocs in our group are to expand the mosquito model and the human epidemiology model to encompass the entire city of about 400,000 people. The postdoc in this new position will collaborate with the other postdocs to use these detailed models to test gene drive systems, but will also develop more generic models (e.g. Huang et al. 2010).</p><p class="MsoNormal">            In addition to working on model development and analysis, the person in this position will have the opportunity to collaborate in an interdisciplinary research group composed of mosquito ecologists, disease epidemiologists, molecular biologists, biomathematicia
 ns, ethicists, and scientists from disease-endemic countries. We are dedicated to taking seriously the ethical and political issues surrounding this technology.</p><p class="MsoNormal"></p><p class="MsoNormal">DESIREABLE SKILLS: A background in population genetics and the ability to program in C++ (or knowledge of a related programming language), and training in evaluation of mechanistic models.</p><p class="MsoNormal">TO APPLY: email a cover letter and CV to <a href="mailto:Fred_Gould@ncsu.edu">Fred_Gould@ncsu.edu</a></p><p> </p><p class="MsoNormal">References:</p><p> </p>Esvelt,  K. M.,  A. L. Smidler, F. Catteruccia, G. M. Church. 2014. Concerning RNA-guided gene drives for the alteration of wild populations. eLife. 10.7554/eLife.03401.<p> </p>Gantz, V. M.  and Bier, E. 2015. The mutagenic chain rea
 ction: A method for converting heterozygous to homozygous mutations. Science 24 April 2015- 442-444. <p></p>Gantz, V.M., N. Jasinskiene, O.Tatarenkova, A. Fazekas, V. M. Macias, E. Bier, and A. A. James. 2015. Highly efficient Cas9-mediated gene drive for population modification of the malaria vector mosquito Anopheles stephensi. PNAS 2015 112 (49) E6736-E6743; published ahead of print November 23, 2015,doi:10.1073/pnas.1521077112<p> </p>Gould, F. 2008. Broadening the application of evolutionarily based genetic pest management. Evolution 62: 500-510. <p class="MsoNormal">Hammond, A. et al. 2015. A CRISPR-Cas9 gene drive system targeting female reproduction in the malaria mosquito vector Anopheles gambiae. Nature Biotechnology   Published online 07 December 2015   doi:10.
 1038/nbt.3439</p><p> </p>Huang, Y., Lloyd, A.L., Legros, M., Gould, F. 2010. Gene-drive into insect populations with age and spatial structure: a theoretical assessment. Evol. Appl. ISSN 1752-4571.<p> </p>Magori, K., M. Legros, M. Puente, D. A. Focks, T. W. Scott, A. Lloyd, F. Gould. 2009. Skeeter Buster: a stochastic, spatially-explicit modeling tool for studying Aedes aegypti population replacement and population suppression strategies. PLoS Negl Trop Dis 3(9): e508. doi:10.1371/journal.pntd.0000508<p> </p>Okamoto, K. W., Robert M. A., Gould, F., Lloyd, A. L. 2014. Feasible Introgression of an Anti-pathogen Transgene into an Urban Mosquito Population without Using Gene-Drive. PLoS Negl Trop Dis 8(7): e2827. doi:10.1371/journal.pntd.0002827.<p class="MsoNormal"></p><p> </p>
&ndash;Apple-Mail=_9353E08B-C0E7-4F50-9E14-E31C2AB12FB
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:22</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135100029646">Postdoc: Switzerland.Basel.host-parasite coevolution/genetics</a></div>
  <div class="description"><p>University of Basel, Switzerland
Zoological Institute

Postdoc position in host-parasite
population genetics and genomics

A postdoc is available in the research group of Dieter Ebert, at the 
Zoological Institute at Basel University in Switzerland. I am looking 
for a highly motivated post-doc with interest in the (co-)evolutionary 
genetics of host-parasite interactions. This position is funded to work 
on the coevolutionary genetics/genomics of Daphnia and its 
microparasites. A background in evolutionary genetics and bioinformatics 
skills are welcome. Excellent written, verbal, and interpersonal skills, 
a superb work ethic, and the ability to think creatively and critically 
are desired. The starting date is flexible. The position is initially 
for 2 years, but can be extended to 3 years.

The post-doc will be part of Dieter Ebert&rsquo;s research group, working on 
the co-evolution of host-microbe interactions. Our experimental work 
uses Daphnia as the host system. Details about the group can be found 
under: <a href="http://bit.ly/18Irt0r">http://bit.ly/18Irt0r</a>

Please send application by E-mail to Dieter Ebert 
(dieter.ebert@unibas.ch). Applications (all in one single pdf file with 
your name as file-name, please) should include a CV, a list of 
publications and a 1-page description of your research interests and 
motivation. Please give names and email addresses of two persons who are 
willing to write a letter of recommendation. Deadline for applications 
is 31. January 2016.


Contact information: Prof. Dr. Dieter Ebert,
University of Basel,
Zoological Institute,
Vesalgasse 1, 4051 Basel, Switzerland,
Email: dieter.ebert@unibas.ch
Tel. +41-(0)61-267 03 60
<a href="http://bit.ly/1OPqIml">http://bit.ly/1OPqIml</a>


via Gmail</p></div>
 </div>
</div>
<h3>December 12, 2015</h3>
<div class="news-item">
 <div class="date">23:52</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135098759211">Postdoc: RBG_Kew.OrchidEvolution</a></div>
  <div class="description"><p>Early Career Research Fellowships (Sainsbury Orchid Fellowships) at
RBG Kew

The Royal Botanic Gardens, Kew seeks outstanding postdoctoral candidates
to take up two Early Career Research Fellowships focusing on orchid
diversity, taxonomy, systematics and evolution. The successful candidates
will establish their own independent research programmes closely aligned
with Kew&rsquo;s Science Strategy, developing funding streams, producing
excellent publications, and engaging fully with science life across
the institution. They will be future leaders in the science disciplines
pursued at Kew.

We seek applicants who are outstanding early career scientists with a PhD
in a relevant subject area (awarded within the past eight years) and,
ideally, some postdoctoral experience. Applicants will have a proven
aptitude for delivering excellent science publications and demonstrated
potential to raise science income. Applicants should also be outstanding
and enthusiastic communicators who are ready to engage with students,
peers and the general public.

Closing date for applications: 10 January 2016

For further details, follow this link:
<a href="http://bit.ly/1lXuWQp">http://bit.ly/1lXuWQp</a>


W.Baker@kew.org


via Gmail</p></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="index8713.html?q=aggregator/sources/14&amp;from=0">first page</a></div><div><a href="index9605.html?q=aggregator/sources/14&amp;from=220">previous page</a></div><div>... <a href="index7de1.html?q=aggregator/sources/14&amp;from=120">7</a> <a href="index2be7.html?q=aggregator/sources/14&amp;from=140">8</a> <a href="index5104.html?q=aggregator/sources/14&amp;from=160">9</a> <a href="index2b69.html?q=aggregator/sources/14&amp;from=180">10</a> <a href="index2379.html?q=aggregator/sources/14&amp;from=200">11</a> <a href="index9605.html?q=aggregator/sources/14&amp;from=220">12</a> <strong>13</strong> <a href="indexe9f7.html?q=aggregator/sources/14&amp;from=260">14</a> <a href="index9a9b.html?q=aggregator/sources/14&amp;from=280">15</a> </div><div><a href="indexe9f7.html?q=aggregator/sources/14&amp;from=260">next page</a></div><div><a href="index9a9b.html?q=aggregator/sources/14&amp;from=280">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news." class="active">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/14&from=240 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 21:45:00 GMT -->
</html>
