<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=40 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:20:30 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexfe3a.html?q=node/279">Contact</a></li><li><a href="index2875.html?q=node/72">MacClade and Adobe Illustrator</a></li><li><a href="indexf64b.html?q=node/63">LaTeX template and bibliography style file</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 5 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 2 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>February 28, 2016</h3>
<div class="news-item">
 <div class="date">00:28</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140135645486">Graduate position: PortlandStateU.REUinPlantGenomics</a></div>
  <div class="description">NSF Research Experience for Undergraduates (REU) in Plant Ecology and
Genomics

We are looking for two enthusiastic students interested in enhancing
their research skills and experience in plant ecology, genomics,
and bioinformatics in the Cruzan lab at Portland State University in
Portland, Oregon.

Successful applicants will become part of a research team working
on dispersal and gene flow in upland prairie plants in Oregon and
Washington under pressure from climate change. Dispersal is critical
for colonization, gene flow, and range expansion, and it is one of
the most important but least understood aspects of the ecology of
plants. Our research utilizes genomic techniques to detect the effects of
dispersal over different spatial scales. This is an integrative research
program that includes field work, wet lab preparation of genomic DNA for
next-generation sequencing, bioinformatics processing of sequence data,
and analysis of patterns of gene flow using GIS and landscape genetic
methods. Students will be exposed to a wide range of methods and are
expected to develop an independent research project within the scope of
the larger research program. More information on our research is posted
on our lab web site: <a href="http://bit.ly/1S9v1xG.">http://bit.ly/1S9v1xG.</a>

Students will be fully supported for 10 to 12 weeks during the summer of
2016. Funding is provided from an REU supplement to our National Science
Foundation Macrosystems Biology grant. Housing will be provided on campus
along with a daily allowance for food and a $500 weekly stipend.

The start of the REU is negotiable with optimal start dates between
May 1 and June 17.  To apply please email us an updated copy of your
CV (including GPA) and a short (one page) statement of your research
interests and experience, as well as goals for your education and
career. Students with experience in computer programming and/or
bioinformatics are encouraged to apply.

Application deadline is 29 February 2016. Feel free to email us with
questions.

Send complete applications to:
Pam Thompson, Postdoctoral Research Associate: thompson@pdx.edu
Mitch Cruzan, Principle Investigator: cruzan@pdx.edu



Mitch Cruzan
Professor of Biology
Portland State University
PO Box 751
Portland, OR 97207 USA
Web: <a href="http://bit.ly/1oGVeI6">http://bit.ly/1oGVeI6</a>



Pamela G. Thompson
Postdoctoral Researcher
Portland State University
Department of Biology, SRTC rm 246
Portland, OR 97201
503-725-9503

Pamela Thompson 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>February 27, 2016</h3>
<div class="news-item">
 <div class="date">23:58</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140134495966">Conference: Lyon.EvolutionaryBehaviour.Jun27-Jul1</a></div>
  <div class="description"><p>Conference : Ecology and Behaviour meeting_ Lyon(France) _ June
27thJuly 1st

>From the 27th of June to the 1st of July 2016, Lyon will host the twelfth
edition of Ecology and Behaviour meetings. Since 2005, this scientific
meeting about animal behavior, ecology and evolution is organized every
year in a different French city. This year, for the second time in the
history, this conference takes place in Lyon.

The Ecology and Behaviour meetings propose an uncommon format in the
landscape of scientific conferences. They offer to young researchers a
convivial atmosphere to present and exchange about their work in the
presence of renowned specialists. Plus, no registration fee is asked
for participants presenting either an oral communication or a poster.

Submissions are already possible and will run until March 20th on the
website of the meeting eb2016.sciencesconf.org, on the submit page.



The organization committee



eb2016@sciencesconf.org


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:28</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140133282011">Graduate position: UVienna.EcolEvolGenomics</a></div>
  <div class="description"><p>Graduate position: UVienna.EcolEvolGenomics

A PhD position is available in the Department of Botany and Biodiversity
Research, Faculty of Life Sciences, University of Vienna. The project
will investigate the mechanisms of diversification in bromeliads
(Bromeliaceae; the Ananas / pineapple family), a &lsquo;textbook&rsquo; adaptive
radiation in the tropical plant diversity hotspots of Meso- and South
America. Integrated within an international research consortium, you
will study the ecological & evolutionary genomics of speciation in
tillandsioid bromeliads (Tillandsia and related genera) using 'target
capture&rsquo; resequencing of candidate genes and pathways for key adaptive
traits and neutral reference genome regions. You will analyze these data
jointly with ecological and spatial information, using the latest tools
from population genomics, phylogenomics, and niche modelling. The results
will provide a strong basis for identifying the drivers of speciation in
this highly diverse and ecologically important group of flowering plants,
and potentially in species radiations more generally.

Work place: The Department of Botany and Biodiversity Research houses
several research groups active in ecological & evolutionary genomics,
plant systematics, niche modelling, conservation biology, structural &
functional botany, and plant-animal interactions. Molecular laboratories
and extensive computing resources are available locally and at the Vienna
Scientific Cluster, and second generation sequencing (NGS) facilities are
available in the nearby Vienna Biocenter Core Facilities. The project is
integrated within an international consortium involving three Universities
in Switzerland and several research groups in Mexico and other Meso-
and South American countries. The position is initially for 3 years, with
the possibility of a 4th year extension. University of Vienna offers an
attractive and dynamic research environment, embedded within a city with
exceptionally high quality of life and with numerous outdoor activities
within easy reach.

Start date: Anytime, but ideally before June 2016 for optimal match with
the consortium&rsquo;s field work plans.

Requirements: Experience in the use of molecular genetic markers in
ecology or evolutionary biology; prior experience with / exposure to
the analysis of second generation sequencing data would be beneficial;
good knowledge of English, written and oral; willingness to acquire
basic knowledge of Spanish to facilitate participation in a (moderate)
amount of field work.

To apply: Applications must include a motivation letter (
via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:13</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140077748566">Graduate position: ULincoln_UK.CanineGenetics</a></div>
  <div class="description">Fully funded PhD studentship in canine genetics: Linking phenotypes with
genotypes for canine chemosensory perception

School of Life Sciences, University of Lincoln, UK

We are seeking to recruit a Science graduate with MSc or a good BSc
degree (or equivalent) for a project on genetics of canine chemosensory
perception, starting in October 2016. Experience in molecular genetics,
bioinformatics and/or behavioural research methods will be seen as an
advantage. This is a fully-funded position with a stipend of 14,124
per annum for three years for both home and international students. The
project will be supervised by a team of researchers with complementary
expertise in evolutionary genetics (Malgorzata Pilot, primary supervisor),
veterinary behavioural medicine (Daniel Mills) and molecular neurobiology
(Humberto Gutierrez).

This project will address a fundamental question in chemosensory system
evolution concerning the trade-off between olfaction (sense of smell)
and vomeronasal sensing (detection of emotionally salient chemosignals),
and at the same time contribute to improving efficiency of selection of
dogs for detection work (e.g. explosives, drugs). Olfactory receptors
(OR) and vomeronasal receptors (V1R) provide systems to detect
molecules of odorants and chemicals that mediate pheromone perception,
respectively. There is growing evidence on functional and evolutionary
interactions between these two systems, but this has not been assessed yet
at an intra-specific level. The aim of this PhD project is to compare OR
and V1R repertoires and chemosensory perception abilities in different
groups of dogs, including breeds currently used for scent detection
and breeds considered as poor sniffers, and in dogs&rsquo; wild ancestors,
grey wolves. The project will involve the analysis of the complete
set of OR and V1R genes using NGS methods, behavioural experiments to
test odour and pheromone perception performance, and the analysis of
genotype-phenotype interactions.

The student will be supervised by an interdisciplinary team with
complementary expertise in evolutionary genetics and genomics, molecular
neurobiology, neuronal cell signalling, olfaction and pheromone detection,
veterinary behavioural medicine, and behaviour of domestic and wild
canids. Therefore, the student will have an opportunity to obtain a
broad range of research skills, and develop a unique interdisciplinary
research profile. The student will be also given opportunities to attend
external courses and workshops, and international conferences. The School
of Life Sciences provides a great research environment and has a large
international community of postgraduate students.


To apply, send a CV and cover letter to Malgorzata Pilot
(mpilot@lincoln.ac.uk) by 17 April 2016, 11 pm of UK time. You may also
contact Malgorzata Pilot for informal enquiries. Shortlisted applicants
will be invited for interview.

Malgorzata Pilot
School of Life Sciences
University of Lincoln
Joseph Banks Laboratories
Lincoln, LN6 7DL
United Kingdom
mpilot@lincoln.ac.uk


Malgorzata Pilot 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:13</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140077748426">Other: NSW_Australia.VolFieldAssist.Birds</a></div>
  <div class="description">**I&rsquo;m actually not sure this is most appropriate for the job section as
it is a volunteer job&hellip;**

Seeking Volunteer Field Assistant for Avian Project in NSW, Australia
Outback
Opportunity Type: Volunteer
Closing Date: 31 March, 2016

We are looking for volunteers to assist in an ongoing research project
studying the cooperatively breeding chestnut-crowned babbler in the
outback in Australia.  We have been studying this system since 2004 and
information on the system can be found here:
<a href="http://bit.ly/1VJTMzG">http://bit.ly/1VJTMzG</a>
=reseach or andrealliebl.com (check out the video on this page made by
a previous field assistant!)
We are hoping to find someone who can start in early July and continue
through the end of the breeding season (October/November).
We are not able to pay for travel expenses to Australia, but
accommodation at a research station
(<a href="http://bit.ly/1oOcZ95">http://bit.ly/1oOcZ95</a>) will be provided as well as a
contribution towards food.

Required:
Ability to work both individually and as a team
Willingness to work early mornings and long days
Relatively fit (there will be a lot of walking on uneven surfaces)
Interest in ecology (our interests in ecology span a relatively large
niche and we do work in physiology, metabolism, call variation, etc as
well)

Desired:
Bush experience
Technologically inclined
4WD/off road driving experience
Knowledge of Australian birds
Ringing experience
This is a great opportunity to gain fieldwork experience in an amazing
location.

If you&rsquo;re interested (or have any questions), please contact Andrea
Liebl (a.l.liebl@exeter.ac.uk).

Andrea L. Liebl, PhD
Postdoctoral Research Fellow
University of Exeter, Cornwall
<a href="http://bit.ly/1XQJTBp">http://bit.ly/1XQJTBp</a>

Andrea Liebl 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:58</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140077330686">Workshop: Faro_Portugal.StatisticalGenetics.BruceWeir</a></div>
  <div class="description"><p>ADVERTISEMENT WEBSITE: <a href="http://bit.ly/21p5KWg">http://bit.ly/21p5KWg</a>

WORKSHOP IN STATISTICAL GENETICS - taught by BRUCE WEIR - Professor of
Biostatistics, and Director of the Institute of Public Health Genetics at
the University of Washington. Bruce Weir is the author of the textbook
&ldquo;Genetic Data Analysis&rdquo; (3rd edition in preparation) and he has directed
the Summer Institute in Statistical Genetics since 1996.
<a href="http://bit.ly/1n3In1n,">http://bit.ly/1n3In1n,</a>
<a href="http://1.usa.gov/21p5NkP">http://1.usa.gov/21p5NkP</a>

DESCRIPTION: A unified treatment for the analysis of discrete genetic data,
starting with estimates and sample variances of allele frequencies to
illustrate genetic vs statistical sampling and Bayesian approaches. A
detailed look at Hardy-Weinberg and linkage disequilibrium, including the
use of exact tests with mid-p-values and a new look at X-chromosome
Hardy-Weinberg testing. A new characterization of population structure with
F-statistics, based on allelic matching within and between populations with
individual relationship estimation as a special case. Analyses illustrated
with applications to forensic science and association mapping, with
particular reference to rare variants.

VENUE: Centre of Marine Sciences, University of the Algarve, Faro,
Portugal. <a href="http://ccmar.ualg.pt/">http://ccmar.ualg.pt</a> - Google maps: <a href="http://bit.ly/1n3IlXp">http://bit.ly/1n3IlXp</a>

DATE: 8th-9th September 2016

FORMAT: 2-day workshop, lectures and hands-on practical sessions (attendees
will need to bring laptops). The course is equivalent to 3 European ECTS
credits.

INTENDED AUDIENCE: PhD students, post-doctoral and advanced researchers
(but no formal restriction)

REGISTRATION FEE: 240€

Registration does not include travel and lodging. Faro has an international
airport located 4 km from the Gambelas Campus of the University of Algarve
where the course will take place.

HOW TO APPLY: Applications in the form of a CV and motivational letter
should be sent toccmaratt@ccmar.ualg.pt by 31st March 2016. Applicants will
be selected on merit and successful applicants will be informed by 10th
April 2016.

SCHOLARSHIPS: At least 2 scholarships to cover registration fees will be
awarded to PhD students; applicants should indicate if they would like to
be considered and must send proof of student status.

ACCOMMODATION: Recommended: Hotel Aeromar at Faro beach (
<a href="http://bit.ly/21p5KWm">http://bit.ly/21p5KWm</a>) has temporarily reserved (until 31st May) and
discounted 8 double rooms, priced 62.5€ and 80€, double and single
occupancy, respectively (mention CCMAR reservation when booking).
Additional accommodation at the beach and in Faro centre:
<a href="http://goo.gl/mgG85g.">http://goo.gl/mgG85g.</a> A minibus service will run between the beach, town
centre, and campus.

CONTACTS:
Ester Serrão - Assistant Professor, UALG/CCMAR - eserrao@ualg.pt
Cymon J. Cox - FCT Coordinating Researcher, CCMAR - cymon@ualg.pt

Advanced Technologies and Training programme @ CCMAR 2016
ccmaratt@ccmar.ualg.pt, <a href="http://bit.ly/21p5KWg">http://bit.ly/21p5KWg</a>

The Centre of Marine Sciences (Centro de Ciências do Mar, CCMAR) is an
independent multidisciplinary, non-profit institute located on the Gambelas
campus of the University of the Algarve in Faro with ~250 members and a
mission to promote research and education in the marine sciences. Faro (
<a href="http://bit.ly/1n3IlXr">http://bit.ly/1n3IlXr</a>) is the capital
city of the Algarve region of southern Portugal and borders the Ria Formosa
National Park (<a href="http://bit.ly/21p5NkX">http://bit.ly/21p5NkX</a>). The Algarve (
<a href="http://bit.ly/1n3In1o">http://bit.ly/1n3In1o</a>) boasts > 300 days of sun per year,
stunning beaches and mountain vistas, opportunities for diving, golf, and
other sporting activities, and a unique traditional regional cuisine (esp.
seafood). A variety of optional social activities will be organised to
coincide with the course and special events can be organised upon request
for participants and/or accompanying persons (diving, surf, stand-up
paddle, windsurf, kayaking, horse-riding, tennis, bird-watching, nature
walks, boat guided tours, fishing, etc).


Bruce Weir - Recent Publications:

Zhu ZH, Bakshi1 A, Vinkhuyzen AAE, Hemani G, Lee SH, Nolte IM,  van
Vliet-Ostaptchouk JV,  Snieder H, The LifeLines Cohort Study,  Esko T,
Milani L, Mägi R,  Metspalu A,  Hill WG,  Weir BS, Goddard ME, Visscher PM,
Yang J. 2015. Dominance genetic variation contributes little to the missing
heritability for human complex traits. American Journal of Human Genetics
96:377-385.

Wasser SK, Brown L, Mailand C, Mondol S, Clark W, Laurie C, Weir BS. 2015.
Genetic assignment of large seizures of elephant ivory reveals Africa&rsquo;s
major poaching hotspots. Science 349:84-87.

Graffelman J, Nelson S, Gogarten SM, Weir BS. 2015. Exact Inference for
Hardy-Weinberg proportions with missing genotypes: Single and multiple
imputation. G3 (Genes, Genomes, Genetics) 5:2365-2373.

Weir BS, Zheng X. 2015. SNPs and SNVs in Forensic Science. Forensic Science
International: Genetics Supplement Series 5:e267-e268.

Zheng X, Weir BS. 2015. Eigenanalysis of SNP data with an interpretation of
identity by descent. Theoretical Population Biology 107:65-76.

Conomos MP, Reiner AP, Weir BS, Thornton TA. 2016. Model-free estimation of
recent genetic relatedness. American Journal of Human Genetics 98:127-148.

Conomos MP, Laurie CA, Stilp AM, Gogarten SM, McHugh CP, Nelson SC, Sofer
T, Fernandez-Rhodes L, Justice E, Graff M, Young KL, Seyerle AA, Avery CL,
Taylor KD, Rotter JT, Talavera GA, Daviglus ML Wassertheil-Smoller S,
Schneiderman N, Heiss G, Kaplan RC, Franceschini N, Reiner AP, Shaffer JR,
Barr RG,  Kerr KF, Browning SR, Browning BL, Weir BS,  Aviles-Santa ML,
Papanicolaou GJ, Lumley T, Szpiro AA, North KE,  Rice K, Thornton TA,
Laurie CC. 2016. Genetic diversity and association studies in U.S.
Hispanic/Latino populations: Applications in the Hispanic Community Health
Study/Study of Latinos. American Journal of Human Genetics 98:165-184.

Schick UM, Jan D, Hodonsky CJ, Morrison JV, Dvis JP, Brown L, Sofer T,
Conomos MP, Schumann C, McHugh CP, Nelson SC, Vadiamudi S, Stilp A,
Plantinga A, Baier L, Bien SA, Gogarten SM, Laurie CA, Taylor KD, Liu Y,
Auer PL, Franceschini N, Szpiro A, Rice K, Kerr KF, Roter JI, Hanson RL,
Papanicolaou, Rich SS, Loos RJF, Browing BL, Browning SR, Weir BS, Laurie
CC, Mohike KL, North KE, Thornton TA, Reiner AP. 2016. Genome-wide
association study of platelet count identifies ancestry-specific loci in
Hispanic/Latino Americans. American Journal of Human Genetics 98:229-242.

Curran JM, Weir BS. 2016. Modern methods of DNA interpretation. Chance (in
press)

Graffelman J, Weir BS. 2016. Testing for Hardy-Weinberg equilibrium at
bi-allelic genetic markers on the X chromosome. 2016. Heredity (in press)

Buckleton JS, Curran JM, Goudet J, Taylor D, Thiery A, Weir BS. 2016.
Population-specific Fst values: A worldwide survey. Forensic Science
International: Genetics (in press).

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:43</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140076908626">Postdoc: GeorgeWashingtonU.ComputationalBiology</a></div>
  <div class="description">GeorgeWashingtonU. PostdoctoralScientist
The Computational Biology Institute (CBI) brings together GW experts
and scholars from partner institutions to conduct cutting edge
research, enable broad educational activities in computational biology
and provides computational biology services to university researchers
and public and private partners. CBI is an interdisciplinary effort,
building on GW&rsquo;s strengths in life sciences and computational
science, in addition to the wealth of research centers in the
Washington, DC area. Additional information on the work of CBI can be
found at: <a href="http://cbi.gwu.edu/">http://cbi.gwu.edu/</a>

The CBI is searching for a Postdoctoral Scientist in the Goecks
Research Laboratory. This scientist will conduct computational biology
research in the following areas:

*innovative software development for the Galaxy platform
(<a href="http://bit.ly/1fg9tao">http://bit.ly/1fg9tao</a>) using Python and JavaScript, including new
methods for visualization, visual analytics, and parallelization;
*development of new computational methods and pipelines for analyzing
large biomedical â€œ-omicsâ€� datasets;
*using novel and existing analysis tools to process and extract
patterns/information/knowledge from â€œ-omicsâ€� datasets, with a
potential focus on cancer genomic datasets.
*leading and/or contributing to scientific manuscripts and grant
applications.

Please send questions to Dr. Jeremy Goecks at jgoecks@gwu.edu

Minimum Qualifications: Requires a PhD in the field or in a related
area.

Preferred Qualifications: Demonstrated programming experience, in the
form a Github profile or code samples, is ideal.
Campus Location: Ashburn, VA
The university is an Equal Employment Opportunity/Affirmative Action
employer that does not unlawfully discriminate in any of its programs
or activities on the basis of race, color, religion, sex, national
origin, age, disability, veteran status, sexual orientation, gender
identity or expression, or on any other basis prohibited by applicable
law.
Apply here: <a href="http://bit.ly/1QaednK">http://bit.ly/1QaednK</a>

&ldquo;Haight, Veronica&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:28</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140076478861">Conference: Edinburgh.QuantGenetics.Mar9</a></div>
  <div class="description"><p>The 10th meeting of the Edinburgh Alliance for Complex Trait Genetics,
sponsored by the Genetics Society, will take place on Wednesday 9th March
2016  from 13.00 to 18.30 at the Royal College of Physicians, 9 Queen
Street, Edinburgh, EH2 1JQ.

All welcome. 

If you want to attend, please sign up at <a href="http://bit.ly/1oFH0XY">http://bit.ly/1oFH0XY</a>  by
Friday 4th March.

Josephine Pemberton & Chris Haley

PROGRAMME

13.00        Arrival and registration

13.30 	   Session 1

13.30        Sophie Marion de Proce (Institute of Genetics and Molecular
Medicine) Locally elevated mutation rates in mammalian promoters.

14.00        Charley Xia (Institute of Genetics and Molecular Medicine)
Genetic analysis of obesity traits in Scotland.

14.30        Brian Charlesworth (Institute of Evolutionary Biology)
Associative overdominance.

15.00        Michael Morgan (Department of Physiology, Anatomy and Genetics,
Oxford)Teasing apart the genetic complexity of hair pigmentation in humans.

15.30        Tea

16.00        Session 2

16.00        Georgios Banos (SRUC Edinburgh)Genetic evaluations for bovine
tuberculosis resistance in dairy cattle.

16.30        Jacob Moorad (Institute of Evolutionary Biology) Why are humans
so longed-lived? Selection for genes that promote late-life human lifespan.

17.00        Peter Joshi (Usher Institute for Population Health Sciences and
Informatics) Genetic Architecture of Human Lifespan - UKBiobank GWAS
results.

17.30        Discussion and refreshments





via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:43</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140075077386">Job: RBGKew.PlantFungalTreeOfLife</a></div>
  <div class="description"><p>Three positions open
The Plant and Fungal Trees of Life project (PAFTOL)
Royal Botanic Gardens, Kew

1. Senior Researcher (PAFTOL Sampling & Project Co-ordination)
2. Senior Researcher (PAFTOL Phylogenomics)
3. Lead Developer-Bioinformatician (PAFTOL)

Kew is a global resource for plant and fungal knowledge. We are seeking
talented, creative individuals with outstanding track records to help
us build the Plant and Fungal Trees of Life, a major piece of global
science infrastructure that is central to RBG Kew&rsquo;s Science Strategy
2015-2020. This is a unique opportunity to be part of a team addressing
a critical challenge in the life sciences. Are you up to the task? If so,
read on to find out more.

The project:
There are few grander challenges in science than the tree of life
- uncovering the complete evolutionary history linking all life on
Earth. As part of RBG Kew&rsquo;s Science Strategy 2015-2020, we are initiating
a five-year multimillion-pound project, the Plant and Fungal Trees
of Life (or PAFTOL, for short), to complete the tree of life for all
genera of plants and fungi, drawing on our outstanding collections, our
broad collaborative networks and the latest high-throughput sequencing
technologies.  This ambitious project will create a unifying framework
for comparative plant and fungal research and takes us a step closer
to building the tree of life for all known species. We will generate
high impact findings publishable in top science journals, but we want
the results to reach beyond the science community. To achieve this we
will build innovative big-data and visualization tools accessible to
a scientist or a school child. PAFTOL has recently secured generous
funding from the Calleva and Sackler Foundations, and we are now ready
to get the work underway. This is a truly exciting time to join Kew and
become an integral part of a high-profile, cutting edge global endeavour.

The team:
At Kew, you&rsquo;ll be joining a truly multidisciplinary team comprising
experts from across a range of fields including systematics, evolution,
genomics, bioinformatics, spatial analysis, data architecture and
software development. At this time, we are recruiting into three
senior positions in phylogenomics, sampling/co-ordination and software
development/bioinformatics. We expect to hire three further support
positions later this year. You&rsquo;ll work alongside a highly supportive
team of Kew staff who have actively developed the PAFTOL concept.

The roles:
We are currently recruiting into the following three roles:

Senior Researcher (PAFTOL Sampling & Project Co-ordination)
<a href="http://bit.ly/1S7LyST">http://bit.ly/1S7LyST</a>
Senior Researcher (PAFTOL Phylogenomics)
<a href="http://bit.ly/1R9knSI">http://bit.ly/1R9knSI</a>
Lead Developer-Bioinformatician (PAFTOL)
<a href="http://bit.ly/1S7LwdO">http://bit.ly/1S7LwdO</a>

For further info:
<a href="http://bit.ly/1R9kqOt">http://bit.ly/1R9kqOt</a>
<a href="http://bit.ly/1S7LySV">http://bit.ly/1S7LySV</a>

Closing date: 15 March 2016
Interview dates: 21-23 March 2016



W.Baker@kew.org


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:43</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140075076991">Job: AMNH_NewYork.InvertZool</a></div>
  <div class="description"><p>The Division of Invertebrate Zoology at the American Museum of Natural
History has the following position open:  Scientific Assistant -
Full Time.

The American Museum of Natural History (AMNH) houses one of the world&rsquo;s
largest collections of biodiversity, including over 32 million biological
specimens and 4 million fossils. The Division of Invertebrate Zoology
is searching for a Scientific Assistant to assist the continued growth
and management of the collections and associated resources (archives,
databases, and tissues) of Arachnida and Myriapoda. The Division contains
the world&rsquo;s largest collection of spiders and second-largest collection
of scorpions, as well as the largest collection of minor arachnid orders
and one of the largest collections of myriapods in North America, with
worldwide representation of taxa, emphasizing material from Africa,
North America and elsewhere in the New World. The majority of specimens
are preserved in ethanol, although large slide-mounted and frozen tissue
collections are also represented.

Duties: The successful applicant will assist the curator in management
of the collections including specimen sorting, labeling, preparation,
accessioning and databasing new acquisitions; systematic arrangement,
storage and organization of the collection; conservation (ethanol levels,
pH, rehydration, label integrity, rehousing into better containers,
etc.); routine collections tasks (assistance to visitors, response to
requests for information, retrieving, packing and shipping loans to other
institutions, unpacking and curating loan returns and donations to AMNH,
databasing accessions, loan and visitor activity); museum and local
community service (collection tours and related activities); training
and supervision of students and volunteers; management of frozen tissues
and associated data.

Qualifications: BS in Biology, Zoology or Entomology. Candidates
with general knowledge of terrestrial arthropod (preferably
arachnid and/or myriapod) morphology and systematics, good
organizational skills and attention to details, and a good command
of MS Word and Excel are preferred, with more experienced databasing
ability desirable.  For more information and to apply please visit:
<a href="http://bit.ly/1R9knSG">http://bit.ly/1R9knSG</a> All applicants must
apply electronically through the AMNH Careers Website.



egaughan@amnh.org


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:28</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140074576626">Graduate position: UZurich.ComputationalEvolutionaryBiol</a></div>
  <div class="description"><p>&ndash;1__NBBF5F6DFBBB7198f9e8a93df938690918c4EBBF5F6DFBBB71
via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>February 26, 2016</h3>
<div class="news-item">
 <div class="date">23:28</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140072376271">Graduate position: USouthDakota.2.Bioinformatics</a></div>
  <div class="description">Graduate Student Opportunity in a New Lab at the University of South
Dakota

I will be starting a lab group in the Department of Biology at the
University of South Fall, 2016.  I am looking for graduate students (MS
or PhD) interested in molecular ecology.  Specifically, in
collaboration with Andrew Russell at the University of Exeter, we have
collected 10+ years of genetic data in the cooperatively breeding
species, the chestnut-crowned babbler. I am looking to recruit one or
two students interested in bioinformatics to work with me and Dr.
Erliang Zeng answering questions about heritability of traits (e.g.
helping), population structuring, and the like (this is relatively open
depending on the interests of the student(s)).  Although considerable
data is already available for use, some opportunity may exist to gain
lab experience if desired.

Some statistical background would be useful and attention to detail is
a requirement.

If interested, please contact Andrea Liebl (a.l.liebl@exeter.ac.uk).


Andrea L. Liebl, PhD
Postdoctoral Research Fellow
University of Exeter, Cornwall
<a href="http://bit.ly/1XQJTBp">http://bit.ly/1XQJTBp</a>

Andrea Liebl 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:42</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140023862831">Postdoc: TuftsU.Genomics Ecology Evolution of Food Microbiomes.</a></div>
  <div class="description">&ndash;089e013a089640c2a1052c9a6310
Content-Type: text/plain; charset=UTF-8

Post-doc position in genomics/ecology/evolution of food microbiomes in
the Wolfe
Lab @ Tufts University (Boston Area, USA).
<a href="http://bit.ly/1OzW4vi">http://bit.ly/1OzW4vi</a>

The Wolfe lab in the Department of Biology at Tufts University (
<a href="http://bit.ly/1Uo6TrL">http://bit.ly/1Uo6TrL</a>) is seeking a post-doc to join the lab in
June 2016 (start date flexible). Our lab studies the ecology and evolution
of microbial communities, using tractable microbial communities isolated
from food (cheese, kimchi, fermented tea) as model systems. Our work spans
taxonomic boundaries (we study both prokaryotes and eukaryotes) and
integrates a wide variety of techniques including experimental evolution,
metagenomics, comparative genomics/transcriptomics, genome engineering, and
in situ community reconstructions. Our research questions are strongly
anchored in basic biology, but our work will help address emerging issues
in human health and food quality/safety.

The ideal candidate will be a creative, independent, and integrative
scholar with interests in discovering the ecological and evolutionary
processes that generate microbiome diversity. Ideal candidates will have
the following:

   -

   a PhD in molecular biology, microbiology, systems biology, or genetics


   -

   extensive experience in the molecular genetics of bacteria or fungi
   -

   experience with generating and analyzing next-generation sequencing
   datasets
   -

   the ability to develop and implement bioinformatics tools
   -

   excellent time management, oral communication, and writing skills
   -

   a track record of supporting an engaging, thoughtful, and collaborative
   lab environment


This position also provides ample opportunities for the post-doc to develop
teaching/outreach skills. Our accessible and engaging model systems provide
powerful tools to promote microbial literacy in the college classroom as
well as in our local communities (<a href="http://bit.ly/1OzW5Q1">http://bit.ly/1OzW5Q1</a>).
Members of the lab communicate their science through social media, writing
for online publications, and by developing novel public outreach
experiences.

Current research in the lab is aimed at linking ecological and evolutionary
patterns of microbial diversity with the molecular mechanisms that generate
these patterns. What are the mechanisms that govern species distributions
or community-level traits? How do these mechanisms evolve within a
community and what are the consequences of trait evolution within microbial
communities? How can we manipulate these mechanisms to control microbiome
composition and function? We are also working on applied projects as part
of the Tufts University Science and Sensory Center (
<a href="http://bit.ly/1Uo6WDS">http://bit.ly/1Uo6WDS</a>) that
aim to link microbiome composition with the chemical and sensory properties
of microbial foods

The Wolfe lab is based in a newly renovated lab space associated with the
Tufts University Medford Campus. In addition to diverse research being
conducted in the Department of Biology that spans the entire breadth of
biology (<a href="http://bit.ly/1600Zpa">http://bit.ly/1600Zpa</a>), the Sackler School of Graduate
Biomedical Sciences at the Tufts Medical School has an outstanding group of
microbiologists (<a href="http://bit.ly/WDvgqA">http://bit.ly/WDvgqA</a>). There are also numerous potential
collaborators and resources at the Tufts Friedman School of Nutrition and
the Cummings School of Vet Medicine. The academic community in the Greater
Boston Area has an unprecedented collective wealth of resources in
microbiology, ecology, and evolution.

To apply, email Dr. Benjamin Wolfe (benjamin.wolfe@tufts.edu) with: 1) a
short description of why you are interested in and qualified for the
position, 2) a CV listing peer-reviewed publications, skills, and research
experiences, and 3) the names and contact information for three references.
Applications will be considered as they are received. For full
consideration, apply on or before April 15th, 2016.


*Benjamin E. Wolfe*
Assistant Professor
Tufts University
Department of Biology
200 Boston Ave., Suite 4700
Medford, MA 02155
*Email:* benjamin.wolfe@tufts.edu
*Office phone:* 617-627-0694
*Lab website:* <a href="http://bit.ly/1Uo6WDW">http://bit.ly/1Uo6WDW</a>

&ndash;089e013a089640c2a1052c9a6310
Content-Type: text/html; charset=UTF-8
Content-Transfer-Encoding: quoted-printable

Post-doc position in genomics/ecology/evolution of food microbiomes in the Wolfe Lab @ Tufts University (Boston Area, USA). <a href="http://bit.ly/1OzW4vi" target="_blank">http://bit.ly/1OzW4vi</a><p dir="ltr">The Wolfe lab in the Department of Biol
 ogy at Tufts University (<a href="http://bit.ly/1Uo6TrL" target="_blank"><a href="http://bit.ly/1Uo6TrL">http://bit.ly/1Uo6TrL</a></a>) is seeking a post-doc to join the lab in June 2016 (start date flexible). Our lab studies the ecology and evolution of microbial communities, using tractable microbial communities isolated from food (cheese, kimchi, fermented tea) as model systems. Our work spans taxonomic boundaries (we study both prokaryotes and eukaryotes) and integrates a wide variety of techniques including experimental evolution, metagenomics, comparative genomics/transcriptomics, genome engineering, and in situ community reconstructions. Our research questions are strongly anchored in basic biology, but our work will 
 help address emerging issues in human health and food quality/safety.</p><p dir="ltr">The ideal candidate will be a creative, independent, and integrative scholar with interests in discovering the ecological and evolutionary processes that generate microbiome diversity. Ideal candidates will have the following:</p><ul><li dir="ltr"><p dir="ltr">a PhD in molecular biology, microbiology, systems biology, or genetics</p></li></ul><ul><li dir="ltr"><p dir="ltr">extensive experience in the molecular genetics of bacteria or fungi</p></li><li dir="ltr"><p dir="ltr">experience with generating and analyzing next-generation sequencing datasets</p></li><li dir="ltr"><p dir="ltr">the ability to develop and implement bioinformatics tools</p></li><li dir="ltr"><p dir="ltr">excellent time management, oral communication, and writing skills</p></li><li dir="ltr"><p dir="ltr">a track record of supporting an engaging, thoughtful, and collaborative lab environment</p></li></ul><p dir="ltr">This position also provides ample opportunities for the post-doc to develop teaching/outreach skills. Our accessible and engaging model systems provide powerful tools to promote microbial literacy in the college classroom as well as in our local communities (<a href="http://bit.ly/1OzW4vk%0A olfelab/open-lab/" target="_blank"><a href="http://bit.ly/1OzW5Q1">http://bit.ly/1OzW5Q1</a></a>). Members of the lab communicate their science through social media, writing for online publications, and by developing novel public outreach experiences.</p><p dir="ltr">Current research in the lab is aimed at linking ecological and evolutionary patterns of microbial diversity with the molecular mechanisms that generate these patterns. What are the mechanisms that govern species distributions or community-level traits? How do these mechanisms evolve within a community and what are the consequences of trait evolution within microbial commun
 ities? How can we manipulate these mechanisms to control microbiome composition and function? We are also working on applied projects as part of the Tufts University Science and Sensory Center (<a href="http://bit.ly/1Uo6WDS" target="_blank"><a href="http://bit.ly/1Uo6WDS">http://bit.ly/1Uo6WDS</a></a>) that aim to link microbiome composition with the chemical and sensory properties of microbial foods</p><p dir="ltr">The Wolfe lab is based in a newly renovated lab space associated with the Tufts University Medford Campus. In addition to diverse research being c
 onducted in the Department of Biology that spans the entire breadth of biology (<a href="http://bit.ly/1600Zpa" target="_blank"><a href="http://bit.ly/1600Zpa">http://bit.ly/1600Zpa</a></a>), the Sackler School of Graduate Biomedical Sciences at the Tufts Medical School has an outstanding group of microbiologists (<a href="http://bit.ly/WDvgqA" target="_blank"><a href="http://bit.ly/WDvgqA">http://bit.ly/WDvgqA</a></a>). There are also numerous potential collaborators and resources at the Tufts Friedman School of Nutrition and the Cummings School of Vet Medicine. The academic community in th
 e Greater Boston Area has an unprecedented collective wealth of resources in microbiology, ecology, and evolution.</p><p dir="ltr">To
 apply, email Dr. Benjamin Wolfe (<a href="mailto:benjamin.wolfe@tufts.edu" target="_blank">benjamin.wolfe@tufts.edu</a>) with: 1) a short description of why you are interested in and qualified for the position, 2) a CV listing peer-reviewed publications, skills, and research experiences, and 3) the names and contact information for three references. Applications will be considered as they are received. For full consideration, apply on or before April 15th, 2016. </p><b><i></i></b><b><i></i></b><b><i>Benjamin E. Wolfe</i></b>Assistant Professor Tufts UniversityDepartment of Biology200 Boston Ave., Suite 4700Medford, MA 02155<b>Email:</b> <a href="mailto:benjamin.wolfe@tufts.edu" target="_blank">benjamin.wolfe@tufts.edu</a> <b>Office phone:</b> <a href="tel:617-627-0694" value="+16176270694" target="_blank">617-627-0694</a><b>Lab website:</b> <a href="http://bit.ly/1Uo6WDW" sty le="color:rgb(17,85,204)" target="_blank">http://bit.ly/1Uo6WDW</a>


&ndash;089e013a089640c2a1052c9a631
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:27</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140023486406">Job: Assistant Professor- Botanist, State University of New York Oswego</a></div>
  <div class="description"><p>&ndash;001a1139c644dd30e1052c9a94f5
Content-Type: text/plain; charset=UTF-8

ASSISTANT PROFESSOR-BOTANIST



The Department of Biological Sciences at the State University of New York
at Oswego invites applications for a full-time tenure track botanist at the
Assistant Professor level with preference given to candidates specializing
in plant structure or development and in advanced microscopy.


We hope to identify candidates who can use our microscopy resources on
campus (confocal and SEM) to investigate aspects of plant structure,
anatomy, and development to elucidate evolutionary relationships across
plant lineages.  A focus of this position would be teaching Plant Kingdom,
an evolutionary survey of plant lineages.  In addition, the candidate would
teach one or more sections of a senior level, capstone Evolution course.
This course is a required seminar that relies heavily on peer-reviewed
literature to discuss historical and current investigations into
evolutionary mechanisms and processes.



DATE POSTED: February 18, 2016



REVIEW DATE:  Review of applications will begin March 18, 2016 and will
continue until the position is filled.



The Department of Biological Sciences offers B.A. and B.S. degrees in
Biology and Zoology to approximately 550 majors. Faculty and students have
access to Rice Creek, a 400+ acre field station with a new building of labs
and classrooms; the Molecular Biology and Biochemistry Center; the Mass
Spectrometry and Proteomics Center; The Environmental Research Center; and
microscopy, greenhouse, and live animal facilities.  Classes in our new
building, the Richard S. Shineman Center for Science, Engineering, and
Innovation, started in Fall 2013.  Links to more information can be found
at: <a href="http://bit.ly/1XPvPZ9.">http://bit.ly/1XPvPZ9.</a>



SALARY: Commensurate with qualifications and experience.



DATE OF APPOINTMENT:  August 2016.



DESCRIPTION OF RESPONSIBILITIES: Candidates will be expected to teach Plant
Kingdom, Evolution, and to offer courses in area of specialty, and
contribute additionally to the core curriculum. Research suitable for
undergraduate student involvement and service to the department and
university are expected.



In addition, this position will be expected to display a demonstrated
potential for excellence in teaching and scholarship, commitment to
undergraduate and/or graduate education, and possess communication and
interpersonal skills sufficient to work effectively with an increasingly
diverse array of students and colleagues.



REQUIRED QUALIFICATIONS: Ph.D. by the time of appointment. Commitment to
excellence in teaching and providing meaningful learning experiences to
students from diverse backgrounds is required.



PREFERRED QUALIFICATIONS: Post-doctoral research experience, teaching
experience, and/or grantsmanship.



Preference will be given to candidates specializing in plant structure or
development and in advanced microscopy (i.e., CLSM and/or SEM).



QUESTIONS: Inquiries can be directed to the Chair of the Search Committee,
Dr. C. Eric Hellquist, eric.hellquist@oswego.edu



APPLICATIONS: Submit a cover letter, curriculum vitae, a statement of
teaching philosophy/interests, a statement of research interests,
unofficial copies of academic transcripts (official transcripts will be
required prior to appointment), and names and email addresses of three
references electronically at:


<a href="http://bit.ly/1oMjgSw">http://bit.ly/1oMjgSw</a>

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:57</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140022774811">Postdoc:KansasSt.QuantitativeGenomicsPhenomics</a></div>
  <div class="description">&ndash;_000_9B663AC178C742CA93BC96990C76E4B5ksuedu_
Content-Type: text/plain; charset=&ldquo;us-ascii&rdquo;
Content-Transfer-Encoding: quoted-printable

Postdoc in Quantitative Genomics & Phenomics

A postdoc position is available in the Morris Lab on quantitative genomics & phenomics of biomass accumulation and environmental adaptation in sorghum.

The Morris lab studies genomics of crop adaptation to support crop improvement (Morris et al. 2013 PNAS, Lasky et al. 2015 Science Advances, Hu et al. 2015 BMC Genomics). The successful candidate will lead quantitative genetics activities for a 4-year, $8M project funded by Department of Energy ARPA-E that will dissect the genomic basis of yield and adaptation traits in bioenergy sorghum (TERRA-REF, <a href="http://bit.ly/20Z4bc4">http://bit.ly/20Z4bc4</a>).

The postdoc will be primarily responsible for: (1) conducting genome-wide association and linkage mapping using multi-environment high-throughput phenotyping data and whole genome resequencing data and (2) publishing findings in high-quality peer-reviewed manuscripts.

Other activities include improving methods for unmanned aerial vehicle (UAV) phenotyping and population genomic analyses of diverse global germplasm to dissect adaptive traits.

Salary commensurate with experience and training, up to $50,000 + benefits.

For more details and instructions how to apply, go to the full job posting:
<a href="http://bit.ly/1KPL9mI">http://bit.ly/1KPL9mI</a>

Review of applications starts Mar 26, 2016.

via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:57</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140022774566">Job: UTexas_Tyler.InsectEvolution</a></div>
  <div class="description">POSITION ANNOUNCEMENT

ASSISTANT PROFESSOR, INSECT EVOLUTIONARY ECOLOGY
DEPARTMENT OF BIOLOGY
THE UNIVERSITY OF TEXAS AT TYLER

The Position: The Department of Biology at The University of Texas
in Tyler is seeking to fill a 9-month, tenure-track position in insect
evolutionary ecology. Any area of evolutionary ecology will be considered
but preference will be given to candidates using integrative approaches
and cutting-edge technologies, and addressing broad questions  in
evolution. The incumbent is required to establish a strong, extramurally
funded research program in his/her area of expertise. S/he is also
expected to have a strong commitment to undergraduate and graduate
education and mentoring. Participation in departmental, college, and
university committees and outreach programs is expected. Preference
will be given to candidates with a demonstrated record of collaborative
research.

Qualifications: Ph.D. in a relevant field (entomology, evolution, ecology,
population genetics, etc.) with a minimum of two years of  postdoctoral
experience. (ii) Demonstrated research creativity, productivity, and
grantsmanship. (iii) Ability to develop a strong, extramurally-funded
research program. (iv) Demonstrated record of collaborative research
(v) Evidence of effective teaching and other communication skills. (vi)
Teaching experience at undergraduate or graduate levels is highly
desirable.

Available resources: The Department of Biology offers a stimulating
intellectual environment and has an excellent reputation in research
and teaching. Twelve faculty members undertake research in diverse
sub-disciplines, including genomics, bioinformatics, population
genetics, evolution, ecology, neurobiology, microbiology, molecular
biology, and physiology. Research seminars by invited speakers are held
weekly. The multidisciplinary Center for Environment, Biodiversity,
and Conservation is housed in the Department of Biology. A new expansion
with state-of-the-art research and teaching labs has just been completed
along with renovation of all existing teaching labs. Three large and
sophisticated walk-in environmental chambers are available for research as
is a confocal microscope. A computer lab connects UT Tyler to the Texas
Advanced Computer Center via a 10 Gb internet connection. The School of
Pharmacy opened its doors in August 2015. Shared equipment and facilities,
including a brand new vivarium, are available at UT Tylers sister
institution, the University of Texas Health Northeast. For additional
information, please visit <a href="http://bit.ly/1fg6wtZ.">http://bit.ly/1fg6wtZ.</a>

Location: Tyler is located 90 miles east of Dallas in the scenic Piney
Woods area of East Texas. Tyler is the cultural center of East Texas,
a region with over one million people, and boasts many amenities such as
museums, a planetarium, a ballet, a symphony orchestra, and a renowned
performing arts center. One of the 14 campuses of the UT System, UT
Tyler offers excellence in teaching, research, artistic performance
and community service. More than 80 undergraduate and graduate degrees
are available at UT Tyler, which has an enrollment of more than 8,000
high-ability students at its campuses in Tyler, Longview, and Palestine.

Applications: Please include (as a single PDF file) a) cover letter;
b) detailed curriculum vitae; c) statement of research interests and
professional goals, d) statement of teaching interests and philosophy;
e) reprints of 3 relevant publications and email to. Dr. John Placyk,
Chair, Evolutionary Ecology Search Committee, Department of Biology,
The University of Texas at Tyler, 3900 University Blvd., Tyler, TX
75799. E-mail: jplacyk@uttyler.edu. Additionally, please arrange to have
3 reference letters sent to the same address.

Review of applications will begin immediately and continue until a
suitable candidate is found. Start date is negotiable.

The University of Texas is an Affirmative Action/Equal Opportunity
Employer. Women and minorities are strongly encouraged to apply. The
successful applicant must be able to demonstrate eligibility to legally
work in the United States. This position is security-sensitive and
subject to Texas Education Code Section 51.215 which authorizes employer
to obtain criminal history record information. The position is contingent
upon availability of funding.



John S. Placyk, Jr., Ph.D., Assistant Professor
Department of Biology
University of Texas at Tyler
3900 University Blvd.
Tyler, Texas 75799
Office:  903-566-7147
Fax: 903-566-7189

<a href="http://bit.ly/20Z4bc1">http://bit.ly/20Z4bc1</a>

&ldquo;A well-known scientist (some say it was Bertrand Russell) once gave a
public lecture on astronomy. He described how the earth orbits around
the sun and how the sun, in turn, orbits around the center of a vast
collection of stars called our galaxy. At the end of the lecture a little
old lady at the back of the room got up and said: &lsquo;What you have told
us is rubbish. The world is really a flat plate supported on the back of
a giant tortoise.&rsquo; The scientist gave a superior smile before replying,
'What is the turtle standing on?&rsquo; 'You&rsquo;re very clever, young man, very
clever,&rsquo; said the little old lady. 'But it&rsquo;s turtles all the way down.&rsquo;&rdquo;

~ Stephen William Hawking (b. 1942), British theoretical physicist 

John Placyk 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:27</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140022109561">Job: UMassachusetts_Lowell.Bioinformatics</a></div>
  <div class="description">Tenure-Track Faculty Position in Bioinformatics at the University of
Massachusetts Lowell

The University of Massachusetts Lowell Department of Biological
Sciences invites applications to fill a full-time tenure-track faculty
position, rank negotiable, to start September 2016. We seek an individual
experienced in bioinformatics and quantitative biology, who would apply
computational, mathematical and other quantitative tools to analyze
complex, high-throughput data. Examples would include the development of
new computational tools to analyze whole genomes or transcriptomes, model
intra- and extracellular signaling networks, and to model systems-level
processes within populations and ecosystems. This area would leverage
our campus strengths in computer science to develop and apply new tools
to understanding living processes. In addition to the broad range of
research interests within the Biological Sciences Department, UMass
Lowell has a diverse group of science and engineering faculty who use
multiple model systems to evaluate new technologies (e.g. nanomedicine,
biomaterials, robotics and tissue engineering), therefore candidates
interested in establishing collaborations with faculty in these fields are
strongly encouraged to apply. The successful candidate will be expected to
establish a vigorous, externally funded research program, and contribute
to the departments teaching mission. Teaching responsibilities may
include the development of an undergraduate Bioinformatics course,
in addition to upper-level courses in areas of interest to the candidate.

The University of Massachusetts Lowell is committed to increasing
diversity in its faculty, staff, and student populations, as well
as curriculum and support programs, while promoting an inclusive
environment. We seek candidates who can contribute to that goal and
encourage you to apply and to identify your strengths in this area.

To be considered for this position, please submit all required
application materials to the UMass Lowell Human Resources website:
<a href="http://bit.ly/1TCIspO">http://bit.ly/1TCIspO</a>

Review of applications will begin immediately and continue until the
position is filled. However, the position may close once an adequate
number of qualified applications are received.



&ldquo;Garb, Jessica&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:12</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140021785431">Workshop: LANGEBIO_Mexico.Drosophila.May25-28</a></div>
  <div class="description">2016 Drosophila Species Workshop

Applications are being accepted for the 2016 Drosophila Species Workshop 
held at LANGEBIO in Guanajuato, Mexico. Workshop dates are Wednesday, 
May 25th - Saturday, May 28th. Please email a one-page summary of 
research interests to Maxi Richmond (mrichmond@ucsd.edu) by March 20th, 
2016. Registration is $350 and will include workshop materials and 
several meals. The registration fee does not include lodging or airfare.

The main goal of the workshop is to teach participants how to 
incorporate non-melanogaster Drosophila species into their research 
program. Participants are introduced to Drosophila systematics, rearing 
and husbandry techniques, how to design and conduct mating experiments, 
how to collect flies from wild populations, and analysis of polytene 
chromosomes. In addition, workshop participants learn the key 
morphological traits necessary for identifying Drosophila species, and 
work in depth with the D. melanogaster, D. obscura, and D. repleta 
species groups along with instructors that specialize in each of these 
groups to facilitate the process of keying out specimens.

Workshop participants include domestic and international Drosophila 
researchers that use a variety of techniques pertaining to studies in 
genomics, proteomics, systems biology, microbial genetics, developmental 
biology, chromosome biology, and ecological genetics.

More information: <a href="http://bit.ly/152kgRS">http://bit.ly/152kgRS</a>
Please email Maxi Richmond (mrichmond@ucsd.edu) with any questions.


Maxi Polihronakis Richmond 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:57</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140021448656">Workshop: Debrecen_Hungary.SexRolesAdultSexRatios.Apr15-17</a></div>
  <div class="description"><p>Workshop: Sex roles and adult sex ratios

15-17 April, 2016, University of Debrecen, Debrecen, Hungary

Sexual selection, mating systems, parental care and sex roles attract
considerable interest in evolutionary biology, behavioural ecology and
beyond. One of core ideas is that the environment imposes specific
pressures that select for particular breeding system, mating behaviour and
patterns of parental care. Recent works are shifting the emphasis from the
abiotic environment to the social environment, of which an important
indicator is the adult sex ratio (ASR). ASR varies widely in nature, and
understanding the causes and consequences of this variation is an important
goal because ASR impacts behaviour and reproduction, and ultimately
population fitness. The causes and implications of ASR variation in wild
populations, however, remain obscure. The objectives of this workshop are
to overview recent studies in sex roles and sex ratios, and to evaluate the
use of phylogenetic comparative methods for the analyses of sex roles and
sex ratios. This workshop will be relevant to postgraduate students, young
scientists and established scholars with an interest in evolutionary
biology, behavioural ecology and population demography.

Confirmed invited speakers will include
Professor Robert Freckleton, University of Sheffield, UK
Professor Jan Komdeur, University of Groningen, The Netherlands
Dr Vladimir Remes, Palacky University, Czech Republic
Professor Zoltn Varga, University of Debrecen, Hungary
Professor Tams Szkely, University of Bath, UK
Dr Veronika Bkony, MTA Plant Protection Institute, Hungary
Dr Lszl Zsolt Garamszegi, Doana Biological Station, Spain

The Workshop will start on Friday (15 April), and continue all day Saturday
(16 April). For Sunday (17 April) we plan a full-day excursion to Hortobgy
National Park, a UNESCO World Heritage site.

For further details on University of Debrecen, please see
<a href="http://bit.ly/1S5sBQJ">http://bit.ly/1S5sBQJ</a>
Information on the city of Debrecen can be found at <a href="http://bit.ly/1Q72S7P">http://bit.ly/1Q72S7P</a>
Information on Hortobgy National Park can be found at <a href="http://bit.ly/1S5sEfi">http://bit.ly/1S5sEfi</a>

Workshop organizers:
Dr Balzs Vgi, University of Debrecen
Dr Zsolt Vgvri, University of Debrecen and Hortobgy National Park
Professor Andrs Liker, University of Pannonia

In addition to the presentations by invited speakers, we will accept a
limited number of contributed talks.

The conference will be held at Life Science Building of University of
Debrecen. There is no registration fee. Attendees are expected to book
their own accommodation (for accommodation, see
<a href="http://bit.ly/1Q72Qgq">http://bit.ly/1Q72Qgq</a> (on campus);
<a href="http://www.campushotel.hu/">www.campushotel.hu</a>; or visit <a href="http://www.booking.com/">www.booking.com</a>). The prices vary between 10
EUR and 50 EUR per night per person. Meals will be at the university
Canteen on campus (approx.10 EUR per meal).

Application deadline: Friday 11 March 2016
Applications (max 5 pages that may include a CV, motivation letter and the
title and abstract of the proposed contribution) should be emailed as a
single PDF to Dr Balzs Vgi (bi.vagi@gmail.com)



Balzs Vgi, PhD
research fellow
Department of Evolutionary Zoology, University of Debrecen,
Debrecen, Egyetem ter 1., 4032, Hungary
Phone: +36 52 316 666 ext. 62356 
   | Fax: +36 52 512 941


bi.vagi@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:42</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140021103856">Workshop: UParis_Diderot.LociOfEvolution.Sep5-7</a></div>
  <div class="description">The Loci of Evolution Meta-Analysis Workshop 
2016 September 5, 9am - September 7, 5pm
Universit?? Paris Diderot - Paris 7, France

&ldquo;To collect and codify the fact of variation is the first duty of the
naturalist [???] ; we need a compact catalogue of the known facts, a list
which shall contain as far as possible all cases of variation observed
&rdquo; - William Bateson (1894)

Gephebase is an online database scheduled for public release in Fall
2016 that will integrate our current knowledge of genotype-phenotype
relationships in a time of accelerated discovery (GWAS, QTL mapping,
reverse genetics). The first version will include no less than 1500
entries, forming an extensive catalogue of the empirical studies that
have associated genetic changes to specific phenotypic variations in
Eukaryotes.

We are inviting 20 researchers working on different aspects of genetic
evolution to join us in Paris next September, and kick-start a community
effort on the meta-analysis of the so-called Loci of Evolution. The
first day will consist in a presentation of Gephebase contents, and
its general potential for meta-analysis. On the second and third day,
we will have small discussion groups with the overarching goal of
cross-pollinating opinions and initiate the writing of collaborative
reviews. Participants are encouraged to come to the Workshop with a
research question in mind. Here are a few possibilities:

-	Trait-centered themes: Domestication; Reproductive Isolation;
Innate Immunity, Morphological Patterns ; Xenobiotic Resistance; Behavior;
Adaptation to Extreme Environments???

-	Concept-centered themes: Micro- vs. Macro-evolution;
Convergence: mutations in the same vs. different genes; Lateral
transfers and Ancestral polymorphisms; Epistasis and complex genetic
architectures; Mutational Biases; Optimal Pleiotropy and Mutational
Target Size; Structural, regulatory and coding evolution; Balancing
selection Vs. Mutation-selection balance; Effects of population size;
Loss vs. Gain of function in evolution???

We are looking forward to meeting you and synergizing our efforts to
synthetize genetic knowledge. This is an exciting time for evolutionary
genetics!

No registration fee, meals and refreshments offered, lodging and travel
in charge of the attendants (participants who need assistance to cover
these costs are welcome to contact us). Graduate students, post-doctoral
researchers and permanent researchers are encouraged to apply before
April 15th 2016.

Please contact Dr. Arnaud Martin (arnaud@gwu.edu) for applying or more
information.





Dr. Virginie Orgogozo

virginie.orgogozo@ijm.fr

Directrice de Recherche CNRS ???

Institut Jacques Monod (Paris, France)


Dr. Arnaud Martin
arnaud@gwu.edu
Assistant Professor, Department of Biological Sciences ??? George
Washington University (Washington, DC)


Arnaud Martin, Ph.D.
Assistant Professor
The George Washington University
Department of Biological Sciences
Mail: 800 22nd St. NW, Suite 6000
Washington, D.C.  20052  USA
Office: SEH 6520 ; Lab: SEH 6480a
Phone: 202-994-2384
Cell: 949-302-3296
Google Scholar
@evolvwing


Arnaud Martin 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="index8503.html?q=aggregator&amp;from=20">previous page</a></div><div><a href="indexbe48.html?q=aggregator&amp;from=0">1</a> <a href="index8503.html?q=aggregator&amp;from=20">2</a> <strong>3</strong> <a href="index4543.html?q=aggregator&amp;from=60">4</a> <a href="index798e.html?q=aggregator&amp;from=80">5</a> <a href="index0e96.html?q=aggregator&amp;from=100">6</a> <a href="index3362.html?q=aggregator&amp;from=120">7</a> <a href="index7b13.html?q=aggregator&amp;from=140">8</a> <a href="indexf37c.html?q=aggregator&amp;from=160">9</a> ...</div><div><a href="index4543.html?q=aggregator&amp;from=60">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=40 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:20:30 GMT -->
</html>
