<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/4&from=20 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:30:23 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexfc9c.html?q=node/431">About Women in Science (Evol2014)</a></li><li><a href="indexf09b.html?q=node/426">Researcher in Environmental Bioinformatics, University of Technology Sydney</a></li><li><a href="indexb172.html?q=node/425">Minutes of mid-year meeting 2014</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 5 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator/sources/4" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator">news aggregator</a>
<ul>
<li class="expanded"><a href="indexe5f5.html?q=aggregator/sources">sources</a>
<ul>
<li class="leaf"><a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></li>
<li class="leaf"><a href="index4421.html?q=aggregator/sources/8">CiteULike Phylogeny</a></li>
<li class="leaf"><a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></li>
<li class="leaf"><a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></li>
<li class="leaf"><a href="index8aad.html?q=aggregator/sources/9">Evolutionary Bioinformatics</a></li>
<li class="leaf"><a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></li>
<li class="leaf"><a href="index9bc3.html?q=aggregator/sources/4" class="active">Molecular Biology and Evolution</a></li>
<li class="leaf"><a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></li>
<li class="leaf"><a href="index9fb6.html?q=aggregator/sources/22">Phyloseminar</a></li>
<li class="leaf"><a href="index69a2.html?q=aggregator/sources/10">Systematics Association</a></li>
<li class="leaf"><a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></li>
<li class="leaf"><a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></li>
<li class="leaf"><a href="indexe0bd.html?q=aggregator/sources/15">dechronization</a></li>
<li class="leaf"><a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></li>
<li class="leaf"><a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></li>
<li class="leaf"><a href="index1f4a.html?q=aggregator/sources/7">postgenomic.com</a></li>

</ul>
</li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 4 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a> &raquo; <a href="index5640.html?q=aggregator">news aggregator</a> &raquo; <a href="indexe5f5.html?q=aggregator/sources">sources</a></div><h2>Molecular Biology and Evolution</h2>
<!-- begin content -->
<div id="aggregator"><div class="feed">Molecular Biology and Evolution - RSS feed of current issue<h3>URL</h3>
<div class="xml-icon"><a href="http://mbe.oxfordjournals.org/rss/current.xml"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div><a href="http://mbe.oxfordjournals.org/">http://mbe.oxfordjournals.org</a>
<h3>Last update</h3>
11 min 11 sec ago</div><h3>March 3, 2016</h3>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1053?rss=1">Cross-Study Comparison Reveals Common Genomic, Network, and Functional Signatures of Desiccation Resistance in Drosophila melanogaster</a></div>
  <div class="description"><p>Repeated attempts to map the genomic basis of complex traits often yield different outcomes because of the influence of genetic background, gene-by-environment interactions, and/or statistical limitations. However, where repeatability is low at the level of individual genes, overlap often occurs in gene ontology categories, genetic pathways, and interaction networks. Here we report on the genomic overlap for natural desiccation resistance from a Pool-genome-wide association study experiment and a selection experiment in flies collected from the same region in southeastern Australia in different years. We identified over 600 single nucleotide polymorphisms associated with desiccation resistance in flies derived from almost 1,000 wild-caught genotypes, a similar number of loci to that observed in our previous genomic study of selected lines, demonstrating the genetic complexity of this ecologically important trait. By harnessing the power of cross-study comparison, we narrowed the candidates from almost 400 genes in each study to a core set of 45 genes, enriched for stimulus, stress, and defense responses. In addition to gene-level overlap, there was higher order congruence at the network and functional levels, suggesting genetic redundancy in key stress sensing, stress response, immunity, signaling, and gene expression pathways. We also identified variants linked to different molecular aspects of desiccation physiology previously verified from functional experiments. Our approach provides insight into the genomic basis of a complex and ecologically important trait and predicts candidate genetic pathways to explore in multiple genetic backgrounds and related species within a functional framework.</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1068?rss=1">Convergent Substitutions in a Sodium Channel Suggest Multiple Origins of Toxin Resistance in Poison Frogs</a></div>
  <div class="description"><p>Complex phenotypes typically have a correspondingly multifaceted genetic component. However, the genotype&ndash;phenotype association between chemical defense and resistance is often simple: genetic changes in the binding site of a toxin alter how it affects its target. Some toxic organisms, such as poison frogs (Anura: Dendrobatidae), have defensive alkaloids that disrupt the function of ion channels, proteins that are crucial for nerve and muscle activity. Using protein-docking models, we predict that three major classes of poison frog alkaloids (histrionicotoxins, pumiliotoxins, and batrachotoxins) bind to similar sites in the highly conserved inner pore of the muscle voltage-gated sodium channel, Nav1.4. We predict that poison frogs are somewhat resistant to these compounds because they have six types of amino acid replacements in the Nav1.4 inner pore that are absent in all other frogs except for a distantly related alkaloid-defended frog from Madagascar, <I>Mantella aurantiaca</I>. Protein-docking models and comparative phylogenetics support the role of these replacements in alkaloid resistance. Taking into account the four independent origins of chemical defense in Dendrobatidae, phylogenetic patterns of the amino acid replacements suggest that 1) alkaloid resistance in Nav1.4 evolved independently at least seven times in these frogs, 2) variation in resistance-conferring replacements is likely a result of differences in alkaloid exposure across species, and 3) functional constraint shapes the evolution of the Nav1.4 inner pore. Our study is the first to demonstrate the genetic basis of autoresistance in frogs with alkaloid defenses.</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1082?rss=1">Detecting Genomic Signatures of Natural Selection with Principal Component Analysis: Application to the 1000 Genomes Data</a></div>
  <div class="description"><p>To characterize natural selection, various analytical methods for detecting candidate genomic regions have been developed. We propose to perform genome-wide scans of natural selection using principal component analysis (PCA). We show that the common <I>F</I>ST index of genetic differentiation between populations can be viewed as the proportion of variance explained by the principal components. Considering the correlations between genetic variants and each principal component provides a conceptual framework to detect genetic variants involved in local adaptation without any prior definition of populations. To validate the PCA-based approach, we consider the 1000 Genomes data (phase 1) considering 850 individuals coming from Africa, Asia, and Europe. The number of genetic variants is of the order of 36 millions obtained with a low-coverage sequencing depth (3x). The correlations between genetic variation and each principal component provide well-known targets for positive selection (EDAR, SLC24A5, SLC45A2, DARC), and also new candidate genes (APPBPP2, TP1A1, RTTN, KCNMA, MYO5C) and noncoding RNAs. In addition to identifying genes involved in biological adaptation, we identify two biological pathways involved in polygenic adaptation that are related to the innate immune system (beta defensins) and to lipid metabolism (fatty acid omega oxidation). An additional analysis of European data shows that a genome scan based on PCA retrieves classical examples of local adaptation even when there are no well-defined populations. PCA-based statistics, implemented in the <I>PCAdapt</I> R package and the <I>PCAdapt fast</I> open-source software, retrieve well-known signals of human adaptation, which is encouraging for future whole-genome sequencing project, especially when defining populations is difficult.</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1094?rss=1">VHICA, a New Method to Discriminate between Vertical and Horizontal Transposon Transfer: Application to the Mariner Family within Drosophila</a></div>
  <div class="description"><p>Transposable elements (TEs) are genomic repeated sequences that display complex evolutionary patterns. They are usually inherited vertically, but can occasionally be transmitted between sexually independent species, through so-called horizontal transposon transfers (HTTs). Recurrent HTTs are supposed to be essential in life cycle of TEs, which are otherwise destined for eventual decay. HTTs also impact the host genome evolution. However, the extent of HTTs in eukaryotes is largely unknown, due to the lack of efficient, statistically supported methods that can be applied to multiple species sequence data sets. Here, we developed a new automated method available as a R package "vhica" that discriminates whether a given TE family was vertically or horizontally transferred, and potentially infers donor and receptor species. The method is well suited for TE sequences extracted from complete genomes, and applicable to multiple TEs and species at the same time. We first validated our method using <I>Drosophila</I> TE families with well-known evolutionary histories, displaying both HTTs and vertical transmission. We then tested 26 different lineages of <I>mariner</I> elements recently characterized in 20 <I>Drosophila</I> genomes, and found HTTs in 24 of them. Furthermore, several independent HTT events could often be detected within the same <I>mariner</I> lineage. The VHICA (Vertical and Horizontal Inheritance Consistence Analysis) method thus appears as a valuable tool to analyze the evolutionary history of TEs across a large range of species.</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1110?rss=1">Avoiding Missing Data Biases in Phylogenomic Inference: An Empirical Study in the Landfowl (Aves: Galliformes)</a></div>
  <div class="description"><p>Production of massive DNA sequence data sets is transforming phylogenetic inference, but best practices for analyzing such data sets are not well established. One uncertainty is robustness to missing data, particularly in coalescent frameworks. To understand the effects of increasing matrix size and loci at the cost of increasing missing data, we produced a 90 taxon, 2.2 megabase, 4,800 locus sequence matrix of landfowl using target capture of ultraconserved elements. We then compared phylogenies estimated with concatenated maximum likelihood, quartet-based methods executed on concatenated matrices and gene tree reconciliation methods, across five thresholds of missing data. Results of maximum likelihood and quartet analyses were similar, well resolved, and demonstrated increasing support with increasing matrix size and sparseness. Conversely, gene tree reconciliation produced unexpected relationships when we included all informative loci, with certain taxa placed toward the root compared with other approaches. Inspection of these taxa identified a prevalence of short average contigs, which potentially biased gene tree inference and caused erroneous results in gene tree reconciliation. This suggests that the more problematic missing data in gene tree&ndash;based analyses are partial sequences rather than entire missing sequences from locus alignments. Limiting gene tree reconciliation to the most informative loci solved this problem, producing well-supported topologies congruent with concatenation and quartet methods. Collectively, our analyses provide a well-resolved phylogeny of landfowl, including strong support for previously problematic relationships such as those among junglefowl (<I>Gallus</I>), and clarify the position of two enigmatic galliform genera (<I>Lerwa</I>, <I>Melanoperdix</I>) not sampled in previous molecular phylogenetic studies.</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1126?rss=1">Wasabi: An Integrated Platform for Evolutionary Sequence Analysis and Data Visualization</a></div>
  <div class="description"><p>Wasabi is an open source, web-based environment for evolutionary sequence analysis. Wasabi visualizes sequence data together with a phylogenetic tree within a modern, user-friendly interface: The interface hides extraneous options, supports context sensitive menus, drag-and-drop editing, and displays additional information, such as ancestral sequences, associated with specific tree nodes. The Wasabi environment supports reproducibility by automatically storing intermediate analysis steps and includes built-in functions to share data between users and publish analysis results. For computational analysis, Wasabi supports PRANK and PAGAN for phylogeny-aware alignment and alignment extension, and it can be easily extended with other tools. Along with drag-and-drop import of local files, Wasabi can access remote data through URL and import sequence data, GeneTrees and EPO alignments directly from Ensembl. To demonstrate a typical workflow using Wasabi, we reproduce key findings from recent comparative genomics studies, including a reanalysis of the <I>EGLN1</I> gene from the tiger genome study: These case studies can be browsed within Wasabi at <A HREF="http://wasabiapp.org:8000/?id=usecases">http://wasabiapp.org:8000?id=usecases. Wasabi runs inside a web browser and does not require any installation. One can start using it at http://wasabiapp.org</A>. All source code is licensed under the AGPLv3.</p></div>
 </div>
</div>
<h3>February 19, 2016</h3>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/i1?rss=1">Cover</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/i2?rss=1">Associate Editors</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/i3?rss=1">SMBE Editors and Council</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/i4?rss=1">Subscription Page</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/595?rss=1">On the Causes of Evolutionary Transition:Transversion Bias</a></div>
  <div class="description"><p>A pattern in which nucleotide transitions are favored several fold over transversions is common in molecular evolution. When this pattern occurs among amino acid replacements, explanations often invoke an effect of selection, on the grounds that transitions are more conservative in their effects on proteins. However, the underlying hypothesis of conservative transitions has never been tested directly. Here we assess support for this hypothesis using direct evidence: the fitness effects of mutations in actual proteins measured via individual or paired growth experiments. We assembled data from 8 published studies, ranging in size from 24 to 757 single-nucleotide mutations that change an amino acid. Every study has the statistical power to reveal significant effects of amino acid exchangeability, and most studies have the power to discern a binary conservative-vs-radical distinction. However, only one study suggests that transitions are significantly more conservative than transversions. In the combined set of 1,239 replacements (544 transitions, 695 transversions), the chance that a transition is more conservative than a transversion is 53 % (95 % confidence interval 50 to 56) compared with the null expectation of 50 %. We show that this effect is not large compared with that of most biochemical factors, and is not large enough to explain the several-fold bias observed in evolution. In short, the available data have the power to verify the "conservative transitions" hypothesis if true, but suggest instead that selection on proteins plays at best a minor role in the observed bias.</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/603?rss=1">Population Structure Shapes Copy Number Variation in Malaria Parasites</a></div>
  <div class="description"><p>If copy number variants (CNVs) are predominantly deleterious, we would expect them to be more efficiently purged from populations with a large effective population size (<I>N</I>e) than from populations with a small <I>N</I>e. Malaria parasites (<I>Plasmodium falciparum</I>) provide an excellent organism to examine this prediction, because this protozoan shows a broad spectrum of population structures within a single species, with large, stable, outbred populations in Africa, small unstable inbred populations in South America and with intermediate population characteristics in South East Asia. We characterized 122 single-clone parasites, without prior laboratory culture, from malaria-infected patients in seven countries in Africa, South East Asia and South America using a high-density single-nucleotide polymorphism/CNV microarray. We scored 134 high-confidence CNVs across the parasite exome, including 33 deletions and 102 amplifications, which ranged in size from </div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/621?rss=1">Shotgun Mitogenomics Provides a Reference Phylogenetic Framework and Timescale for Living Xenarthrans</a></div>
  <div class="description"><p>Xenarthra (armadillos, sloths, and anteaters) constitutes one of the four major clades of placental mammals. Despite their phylogenetic distinctiveness in mammals, a reference phylogeny is still lacking for the 31 described species. Here we used Illumina shotgun sequencing to assemble 33 new complete mitochondrial genomes, establishing Xenarthra as the first major placental clade to be fully sequenced at the species level for mitogenomes. The resulting data set allowed the reconstruction of a robust phylogenetic framework and timescale that are consistent with previous studies conducted at the genus level using nuclear genes. Incorporating the full species diversity of extant xenarthrans points to a number of inconsistencies in xenarthran systematics and species definition. We propose to split armadillos into two distinct families Dasypodidae (dasypodines) and Chlamyphoridae (euphractines, chlamyphorines, and tolypeutines) to better reflect their ancient divergence, estimated around 42 Ma. Species delimitation within long-nosed armadillos (genus <I>Dasypus</I>) appeared more complex than anticipated, with the discovery of a divergent lineage in French Guiana. Diversification analyses showed Xenarthra to be an ancient clade with a constant diversification rate through time with a species turnover driven by high but constant extinction. We also detected a significant negative correlation between speciation rate and past temperature fluctuations with an increase in speciation rate corresponding to the general cooling observed during the last 15 My. Biogeographic reconstructions identified the tropical rainforest biome of Amazonia and the Guiana Shield as the cradle of xenarthran evolutionary history with subsequent dispersions into more open and dry habitats.</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/643?rss=1">Iron-Sulfur Cluster Biogenesis Chaperones: Evidence for Emergence of Mutational Robustness of a Highly Specific Protein-Protein Interaction</a></div>
  <div class="description"><p>Biogenesis of iron&ndash;sulfur clusters (FeS) is a highly conserved process involving Hsp70 and J-protein chaperones. However, Hsp70 specialization differs among species. In most eukaryotes, including <I>Schizosaccharomyces pombe</I>, FeS biogenesis involves interaction between the J-protein Jac1 and the multifunctional Hsp70 Ssc1. But, in <I>Saccharomyces cerevisiae</I> and closely related species, Jac1 interacts with the specialized Hsp70 Ssq1, which emerged through duplication of <I>SSC1</I>. As little is known about how gene duplicates affect the robustness of their protein interaction partners, we analyzed the functional and evolutionary consequences of Ssq1 specialization on the ubiquitous J-protein cochaperone Jac1, by comparing <I>S. cerevisiae</I> and <I>S. pombe</I>. Although deletion of <I>JAC1</I> is lethal in both species, alanine substitutions within the conserved His&ndash;Pro&ndash;Asp (HPD) motif, which is critical for Jac1:Hsp70 interaction, have species-specific effects. They are lethal in <I>S. pombe,</I> but not in <I>S. cerevisiae</I>. These in vivo differences correlated with in vitro biochemical measurements. Charged residues present in the J-domain of <I>S. cerevisiae</I> Jac1, but absent in <I>S. pombe</I> Jac1, are important for tolerance of <I>S. cerevisiae</I> Jac1 to HPD alterations. Moreover, Jac1 orthologs from species that encode Ssq1 have a higher sequence divergence. The simplest interpretation of our results is that Ssq1&rsquo;s coevolution with Jac1 resulted in expansion of their binding interface, thus increasing the efficiency of their interaction. Such an expansion could in turn compensate for negative effects of HPD substitutions. Thus, our results support the idea that the robustness of Jac1 emerged as consequence of its highly efficient and specific interaction with Ssq1.</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/657?rss=1">Estimating the Ages of Selection Signals from Different Epochs in Human History</a></div>
  <div class="description"><p>Genetic variation harbors signatures of natural selection driven by selective pressures that are often unknown. Estimating the ages of selection signals may allow reconstructing the history of environmental changes that shaped human phenotypes and diseases. We have developed an approximate Bayesian computation (ABC) approach to estimate allele ages under a model of selection on new mutations and under demographic models appropriate for human populations. We have applied it to two resequencing data sets: An ultra-high depth data set from a relatively small sample of unrelated individuals and a lower depth data set in a larger sample with transmission information. In addition to evaluating the accuracy of our method based on simulations, for each SNP, we assessed the consistency between the posterior probabilities estimated by the ABC approach and the ancient DNA record, finding good agreement between the two types of data and methods. Applying this ABC approach to data for eight single nucleotide polymorphisms (SNPs), we were able to rule out an onset of selection prior to the dispersal out-of-Africa for three of them and more recent than the spread of agriculture for an additional three SNPs.</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/670?rss=1">Phylogenomics Controlling for Base Compositional Bias Reveals a Single Origin of Eusociality in Corbiculate Bees</a></div>
  <div class="description"><p>As increasingly large molecular data sets are collected for phylogenomics, the conflicting phylogenetic signal among gene trees poses challenges to resolve some difficult nodes of the Tree of Life. Among these nodes, the phylogenetic position of the honey bees (Apini) within the corbiculate bee group remains controversial, despite its considerable importance for understanding the emergence and maintenance of eusociality. Here, we show that this controversy stems in part from pervasive phylogenetic conflicts among GC-rich gene trees. GC-rich genes typically have a high nucleotidic heterogeneity among species, which can induce topological conflicts among gene trees. When retaining only the most GC-homogeneous genes or using a nonhomogeneous model of sequence evolution, our analyses reveal a monophyletic group of the three lineages with a eusocial lifestyle (honey bees, bumble bees, and stingless bees). These phylogenetic relationships strongly suggest a single origin of eusociality in the corbiculate bees, with no reversal to solitary living in this group. To accurately reconstruct other important evolutionary steps across the Tree of Life, we suggest removing GC-rich and GC-heterogeneous genes from large phylogenomic data sets. Interpreted as a consequence of genome-wide variations in recombination rates, this GC effect can affect all taxa featuring GC-biased gene conversion, which is common in eukaryotes.</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/679?rss=1">Evolution of Prdm Genes in Animals: Insights from Comparative Genomics</a></div>
  <div class="description"><p><I>Prdm</I> genes encode transcription factors with a subtype of SET domain known as the PRDF1-RIZ (PR) homology domain and a variable number of zinc finger motifs. These genes are involved in a wide variety of functions during animal development. As most <I>Prdm</I> genes have been studied in vertebrates, especially in mice, little is known about the evolution of this gene family. We searched for <I>Prdm</I> genes in the fully sequenced genomes of 93 different species representative of all the main metazoan lineages. A total of 976 <I>Prdm</I> genes were identified in these species. The number of <I>Prdm</I> genes per species ranges from 2 to 19. To better understand how the <I>Prdm</I> gene family has evolved in metazoans, we performed phylogenetic analyses using this large set of identified <I>Prdm</I> genes. These analyses allowed us to define 14 different subfamilies of <I>Prdm</I> genes and to establish, through ancestral state reconstruction, that 11 of them are ancestral to bilaterian animals. Three additional subfamilies were acquired during early vertebrate evolution (Prdm5, Prdm11, and Prdm17). Several gene duplication and gene loss events were identified and mapped onto the metazoan phylogenetic tree. By studying a large number of nonmetazoan genomes, we confirmed that <I>Prdm</I> genes likely constitute a metazoan-specific gene family. Our data also suggest that <I>Prdm</I> genes originated before the diversification of animals through the association of a single ancestral SET domain encoding gene with one or several zinc finger encoding genes.</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/697?rss=1">No Accumulation of Transposable Elements in Asexual Arthropods</a></div>
  <div class="description"><p>Transposable elements (TEs) and other repetitive DNA can accumulate in the absence of recombination, a process contributing to the degeneration of Y-chromosomes and other nonrecombining genome portions. A similar accumulation of repetitive DNA is expected for asexually reproducing species, given their entire genome is effectively nonrecombining. We tested this expectation by comparing the whole-genome TE loads of five asexual arthropod lineages and their sexual relatives, including asexual and sexual lineages of crustaceans (<I>Daphnia</I> water fleas), insects (<I>Leptopilina</I> wasps), and mites (<I>Oribatida</I>). Surprisingly, there was no evidence for increased TE load in genomes of asexual as compared to sexual lineages, neither for all classes of repetitive elements combined nor for specific TE families. Our study therefore suggests that nonrecombining genomes do not accumulate TEs like nonrecombining genomic regions of sexual lineages. Even if a slight but undetected increase of TEs were caused by asexual reproduction, it appears to be negligible compared to variance between species caused by processes unrelated to reproductive mode. It remains to be determined if molecular mechanisms underlying genome regulation in asexuals hamper TE activity. Alternatively, the differences in TE dynamics between nonrecombining genomes in asexual lineages versus nonrecombining genome portions in sexual species might stem from selection for benign TEs in asexual lineages because of the lack of genetic conflict between TEs and their hosts and/or because asexual lineages may only arise from sexual ancestors with particularly low TE loads.</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/707?rss=1">Global Transcriptional Profiling of Diapause and Climatic Adaptation in Drosophila melanogaster</a></div>
  <div class="description"><p>Wild populations of the model organism <I>Drosophila melanogaster</I> experience highly heterogeneous environments over broad geographical ranges as well as over seasonal and annual timescales. Diapause is a primary adaptation to environmental heterogeneity, and in <I>D. melanogaster</I> the propensity to enter diapause varies predictably with latitude and season. Here we performed global transcriptomic profiling of naturally occurring variation in diapause expression elicited by short day photoperiod and moderately low temperature in two tissue types associated with neuroendocrine and endocrine signaling, heads, and ovaries. We show that diapause in <I>D. melanogaster</I> is an actively regulated phenotype at the transcriptional level, suggesting that diapause is not a simple physiological or reproductive quiescence. Differentially expressed genes and pathways are highly distinct in heads and ovaries, demonstrating that the diapause response is not uniform throughout the soma and suggesting that it may be comprised of functional modules associated with specific tissues. Genes downregulated in heads of diapausing flies are significantly enriched for clinally varying single nucleotide polymorphism (SNPs) and seasonally oscillating SNPs, consistent with the hypothesis that diapause is a driving phenotype of climatic adaptation. We also show that chromosome location-based coregulation of gene expression is present in the transcriptional regulation of diapause. Taken together, these results demonstrate that diapause is a complex phenotype actively regulated in multiple tissues, and support the hypothesis that natural variation in diapause propensity underlies adaptation to spatially and temporally varying selective pressures.</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/721?rss=1">Restriction-Modification Systems as Mobile Genetic Elements in the Evolution of an Intracellular Symbiont</a></div>
  <div class="description"><p>Long-term vertical transmission of intracellular bacteria causes massive genomic erosion and results in extremely small genomes, particularly in ancient symbionts. Genome reduction is typically preceded by the accumulation of pseudogenes and proliferation of mobile genetic elements, which are responsible for chromosome rearrangements during the initial stage of endosymbiosis. We compared the genomes of an endosymbiont of termite gut flagellates, "<I>Candidatus</I> Endomicrobium trichonymphae," and its free-living relative <I>Endomicrobium proavitum</I> and discovered many remnants of restriction-modification (R-M) systems that are consistently associated with genome rearrangements in the endosymbiont genome. The rearrangements include apparent insertions, transpositions, and the duplication of a genomic region; there was no evidence of transposon structures or other mobile elements. Our study reveals a so far unrecognized mechanism for genome rearrangements in intracellular symbionts and sheds new light on the general role of R-M systems in genome evolution.</p></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="index8134.html?q=aggregator/sources/4&amp;from=0">first page</a></div><div><a href="index8134.html?q=aggregator/sources/4&amp;from=0">previous page</a></div><div><a href="index8134.html?q=aggregator/sources/4&amp;from=0">1</a> <strong>2</strong> <a href="index2f09.html?q=aggregator/sources/4&amp;from=40">3</a> <a href="index7cbe.html?q=aggregator/sources/4&amp;from=60">4</a> <a href="index9243.html?q=aggregator/sources/4&amp;from=80">5</a> <a href="indexe4fa.html?q=aggregator/sources/4&amp;from=100">6</a> </div><div><a href="index2f09.html?q=aggregator/sources/4&amp;from=40">next page</a></div><div><a href="indexe4fa.html?q=aggregator/sources/4&amp;from=100">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news." class="active">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/4&from=20 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:30:23 GMT -->
</html>
