<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/14&from=160 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:29:17 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexfc9c.html?q=node/431">About Women in Science (Evol2014)</a></li><li><a href="indexf09b.html?q=node/426">Researcher in Environmental Bioinformatics, University of Technology Sydney</a></li><li><a href="indexb172.html?q=node/425">Minutes of mid-year meeting 2014</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 5 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator/sources/14" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator">news aggregator</a>
<ul>
<li class="expanded"><a href="indexe5f5.html?q=aggregator/sources">sources</a>
<ul>
<li class="leaf"><a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></li>
<li class="leaf"><a href="index4421.html?q=aggregator/sources/8">CiteULike Phylogeny</a></li>
<li class="leaf"><a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></li>
<li class="leaf"><a href="index4dbd.html?q=aggregator/sources/14" class="active">EVOLDIR</a></li>
<li class="leaf"><a href="index8aad.html?q=aggregator/sources/9">Evolutionary Bioinformatics</a></li>
<li class="leaf"><a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></li>
<li class="leaf"><a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></li>
<li class="leaf"><a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></li>
<li class="leaf"><a href="index9fb6.html?q=aggregator/sources/22">Phyloseminar</a></li>
<li class="leaf"><a href="index69a2.html?q=aggregator/sources/10">Systematics Association</a></li>
<li class="leaf"><a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></li>
<li class="leaf"><a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></li>
<li class="leaf"><a href="indexe0bd.html?q=aggregator/sources/15">dechronization</a></li>
<li class="leaf"><a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></li>
<li class="leaf"><a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></li>
<li class="leaf"><a href="index1f4a.html?q=aggregator/sources/7">postgenomic.com</a></li>

</ul>
</li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 4 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a> &raquo; <a href="index5640.html?q=aggregator">news aggregator</a> &raquo; <a href="indexe5f5.html?q=aggregator/sources">sources</a></div><h2>EVOLDIR</h2>
<!-- begin content -->
<div id="aggregator"><div class="feed"><h3>URL</h3>
<div class="xml-icon"><a href="http://evoldir.tumblr.com/rss"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div><a href="http://evoldir.tumblr.com/">http://evoldir.tumblr.com/</a>
<h3>Last update</h3>
9 min 37 sec ago</div><h3>January 8, 2016</h3>
<div class="news-item">
 <div class="date">03:43</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136873437141">Postdoc: TexasAM.PopulationGenomics</a></div>
  <div class="description">POSITION DESCRIPTION:

Postdoctoral fellowship in marine population genomics of Hawaiian
â€˜opihi.  Subthemes include but are not limited to connectivity,
community managed areas, Bayesian modeling of fisheries.

PROJECT:

Assessing the efficacy of community managed areas, i.e. fine scale gene
flow patterns, parentage analysis, .  2 yrs., funded by the NOAA
Saltonstall-Kennedy program.

QUALIFICATIONS:

Ph.D. in a related field  and experience in at least one of the
following: population genetics, genomics, bioinformatics, statistical
analysis, modelling, fishery ecology and management, unix/linux,
computer programming

People skills are desirable

COMPENSATION:

Commensurate with skills and experience.  Competitive.

OPPORTUNITIES:

Training in project management, mentoring graduate students, genomic
laboratory techniques, population genomic analysis and modelling,
community outreach, teaching bioinformatics for ecologists, manuscript
writing, and grant writing.

RESOURCES:

HPC, high throughput genotyping laboratory, staffed Genomics Core Lab.

OTHER:

You will be expected to live in Corpus Christi, TX.  There will be
field work in Hawaii.  I&rsquo;ll cover at least 1 conference per year.
Two  PhD and two Master&rsquo;s students and Dr. Rob Toonen (U Hawaii) are
also involved in the project.

CONTACT:

chris.bird@tamucc.edu

Assistant Professor of Molecular Ecology

Director, Genomics Core Lab

Texas A&M University - Corpus Christi

Affiliate Faculty

Hawaii Institute of Marine Biology

&ldquo;Bird, Chris&rdquo; 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:13</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136872716241">Postdoc: MaxPlanckEvolBiol.GenomeAnalysisBioinformatics</a></div>
  <div class="description">A position for a postdoc in the field of genome analysis and
bioinformatics is available in the group of Diethard Tautz (Max-Planck
Institute for Evolutionary Biology, Department of Evolutionary Genetics,
Ploen, Germany). The postdoc will join the sub-group working on de
novo evolution of genes, funded by an ERC advanced grant. The insights
into de novo evolution of genes shapes currently into a new paradigm of
genome evolution (see Neme and Tautz 2014: CurrBiol 24:R38).  A large
genomic and transcriptomic dataset has been created for for a range of
species representing the evolutionary lineage towards the house mouse (Mus
musculus). Further population-level genome and transcriptome sequencing is
ongoing. These data provide an excellent basis for studying in molecular
detail the emergence of new genes from non-coding sequences. The postdoc
will be part of a team of PhD students and postdocs who work on all
aspects of de novo evolution, including functional testing in cell lines
and mouse knockouts. The postdoc will also have the chance to initiate
own experiments.

Qualification: PhD in genome data analysis, including programming and
statistics; experience in protein structure analysis and/or population
genetics would be of advantage.

Salary will be according to the TvoD 13-14 scale (depending on
experience), including social benefits and pension scheme. The position
is initially for two years, but can be extended.

The Max-Planck Institut for Evolutionary Biology in Ploen has developed
in the past years into a major center for basic research on evolutionary
topics, ranging from evolutionary genetics, evolutionary ecology,
experimental evolution to evolutionary theory. It runs together with
the nearby University of Kiel an international graduate program
(IMPRS in Evolutionary Biology) and a Master program in Molecular
Biology and Evolution. Scientists come from all over the world and the
working language is english. Ploen is a small village in a beautiful
postglacially shaped landscape with all the amenities of a touristically
active area. Two major cities (Kiel and Luebeck) as well as the Baltic Sea
are only 30-40min away, Hamburg is about 90min away (all well connected
by train).

Applications should include a CV and a publication list (including
papers in press or submitted), a statement of motivation, as well as
two names for obtaining letters of reference. Applications should be
sent by email with documents attached as pdf files to Diethard Tautz
(tautz@evolbio.mpg.de).


Prof. Dr. Diethard Tautz
Max-Planck-Institut fuer Evolutionsbiologie
Abteilung Evolutionsgenetik
August-Thienemannstrasse 2
24306 Ploen (Germany)
Tel.: 04522 763 390
Fax: 04522 763 281
tautz@evolbio.mpg.de
<a href="http://bit.ly/1K1JGng">http://bit.ly/1K1JGng</a>




Diethard Tautz 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:58</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136872354051">Course: Leicester.NGSAssembly&VariantCalling.Feb1-3</a></div>
  <div class="description"><p>Deadline for applications is approaching for the BBASH training
course &ldquo;DNA-Seq Next Generation Sequencing&rdquo;. Applications are welcome 
until Friday 15/01/2016!

BBASH at University of Leicester is pleased to announce that 
applications for the upcoming Next Generation Sequencing training course 
are now open.

Deadline for applications: 15th January 2016
Notification of acceptance: 18th January 2016
Course dates: 1st-3rd February 2016

Venue: University of Leicester - College Court Conference Centre, 
Leicester, UK (<a href="http://bit.ly/1K1HPPp">http://bit.ly/1K1HPPp</a>)

Instructors:
Richard Badge (BBASH, University of Leicester, UK)
Chiara Batini (University of Leicester, UK)
Matthew Blades (BBASH, University of Leicester, UK)


Course description

This course will provide an introduction to next generation sequencing 
(NGS) platforms, data analysis and tools for data quality control, 
genome assembly and variant calling. The course will be delivered using 
a mixture of lectures and computer based hands on practical sessions 
using real data.

This course is aimed at wet-lab biologists who are involved in research 
projects that will require the handling and analysis of NGS data. A 
significant proportion of the course will be computer-based using 
command line tools in the Unix environment, therefore, in order to gain 
maximum benefit from the course all attendees will be required to have 
basic Unix skills.


For additional information: 
<a href="http://bit.ly/22P7s1g">http://bit.ly/22P7s1g</a>



The Organisers
Chiara Batini (University of Leicester, UK)
Matthew Blades (BBASH, University of Leicester, UK)


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:43</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136871967146">Course: SCENE_Glasgow.PythonForBiologists.May23-27</a></div>
  <div class="description">INTRODUCTION TO PYTHON FOR BIOLOGISTS
REPEAT COURSE!!

This course is being delivered by Dr Martin Jones, an expert in Python
and author of two text books,
Python for Biologists
[http://amzn.to/1NCtnR4
/dp/1492346136/]
Advanced Python for Biologists
[http://amzn.to/1O9c4Kw
44377/].

Due to high demand a repeat of this course will run from 23rd - 27th
May 2016 at SCENE (the Scottish Centre for Ecology and the Natural
Environment), Loch Lomond National Park, Glasgow.

Course overview: 
Python is a dynamic, readable language that is a popular platform for
all types of bioinformatics work, from simple one-off scripts to large,
complex software projects. This workshop is aimed at complete beginners
and assumes no prior programming experience. It gives an overview of the
language with an emphasis on practical problem-solving, using examples
and exercises drawn from various aspects of bioinformatics work. After
completing the workshop, students should be in a position to (1) apply
the skills they have learned to tackle problems in their own research and
(2) continue their Python education in a self-directed way.

Intended audience:
This workshop is aimed at all researchers and technical workers with a
background in biology who want to learn programming. The syllabus has
been planned with complete beginners in mind; people with previous
programming experience are welcome to attend as a refresher but may
find the pace a bit slow.

Teaching format:
The workshop is delivered over ten half-day sessions (see the detailed
curriculum below). Each session consists of roughly a one hour lecture
followed by two hours of practical exercises, with breaks at the
organizer&rsquo;s discretion. There will also be plenty of time for
students to discuss their own problems and data.

Assumed background:
Students should have enough biological background to appreciate the
examples and exercise problems (i.e. they should know about DNA and
protein sequences, what translation is, and what introns and exons
are). No previous programming experience or computer skills (beyond the
ability to use a text editor) are necessary, but you&rsquo;ll need to have a
laptop with Python installed.

Curriculum:
Day 1:
Module 1 - Introduction
Module 2 - Output and text manipulation
Day 2:
Module 3 - File IO and user interfaces
Module 4 - Flow control 1: loops
Day 3:
Module 5 - Flow control 2: conditionals
Module 6 - Organizing and structuring code
Day 4:
Module 7 - Regular expressions
Module 8 - Dictionaries
Day 5:
Module 9 - Interaction with the file system
Module 10 - Optional free afternoon to cover previous modules and
            discuss data

The cost is £500 including lunches and course materials. An
all-inclusive option is also available at £710; this includes
breakfast, lunch, dinner, refreshments, accommodation and course
materials. Participants will need a laptop with a recent version of
Python installed.

Please send inquiries to oliverhooker@prstatistics.co.uk or visit the
website <a href="http://www.prstatistics.com/">www.prstatistics.com</a>

Please fee free to distribute this information anywhere you think
suitable
Other upcoming courses - TIME SERIES MODELS FOR ECOLOGISTS AND
CLIMATOLOGISTS; GENETIC DATA ANALYSIS USING SIAR; ; APPLIED BAYESIAN
MODELLING FOR ECOLOGISTS AND EPIDEMIOLOGISTS; SPATIAL ANALYSIS OF
ECOLOGICAL DATA USING R; ADVANCING IN STATISTICAL MODELLING USING R;
INTRODUCTION TO STATISTICS AND R FOR BIOLOGISTS; DNA TAXONOMY; ;
BIOINFORMATICS FOR GENETICISTS AND BIOLOGISTS; MULTIVARIATE ANALYSIS OF
SPATIAL DATA; ADVANCES IN DNA TAXONOMY USING R;

Oliver Hooker
PR~Statistics

&ldquo;oliverhooker@prstatistics.co.uk&rdquo; 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:43</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136871967256">Postdoc: ColdSpringHarborLab.ComputationalGenomics</a></div>
  <div class="description">The Siepel Lab in the Simons Center for Quantitative Biology at Cold
Spring Harbor Laboratory is seeking full-time postdoctoral computational
researchers. The Siepel Group specializes in the development of
probabilistic models, algorithms for inference, prediction methods, and
application of these methods in large-scale genomic data analysis. Of
particular interest is research relevant to existing, NIH supported
projects in (1) HUMAN POPULATION GENOMICS, including demography inference
using Bayesian coalescent-based methods, inference of natural selection
on regulatory and other noncoding sequences, and prediction of fitness
consequences for noncoding mutations; and (2) TRANSCRIPTIONAL REGULATION
in mammals and Drosophila, including the estimation of rates and
patterns of transcriptional elongation from GRO-seq data, prediction of
transcription factor binding sites from DNase-seq data, and regulatory
network inference based on joint patterns of transcription and binding
in inducible systems. The research on transcriptional regulation will
continue to be carried out in close collaboration with Dr. John Lis at
Cornell University.

RELEVANT RECENT PAPERS:
Kuhlwilm M, Gronau I, Hubisz MJ, de Filippo C, Prado J, et al. Ancient
gene flow from modern humans into Siberian Neandertals. Nature (in
press), 2015.

Danko CG, Hyland SL, Core LJ, Martins AL, Waters CT, et al. Identification
of active transcriptional regulatory elements from GRO-seq
data. Nat. Methods. 2015. 12(5):433-438. PMCID: PMC4507281.

Rasmussen MD, Hubisz MJ, Gronau I, Siepel A. Genome-wide inference of
ancestral recombination graphs. PLOS Genet. 2014. 10(5): e1004342. PMCID:
PMC4022496.

Freedman AH, Gronau I, Schweizer RM, Ortega-Del Vecchyo D, Han E, et
al. (30 co-authors). Genome sequencing highlights the dynamic early
history of dogs. PLOS Genet. 2014. 10(1):e1004016. PMCID: PMC3894170.

Core LJ, Martins AL, Danko CG, Waters CT, Siepel A, et al. Analysis of
nascent RNA identifies a unified architecture of initiation regions at
mammalian promoters and enhancers. Nat. Genet. 2014. 46(12):1311-1320
PMCID: PMC4254663.

Capra JA, Hubisz MJ, Kostka D, Pollard KS, Siepel A. A model-based
analysis of GC-biased gene conversion in the human and chimpanzee
genomes. PLOS Genet. 2013. 9(8):e1003684. PMCID: PMC3744432.

Arbiza L, Gronau I, Aksoy BA, Hubisz MJ, Gulko B, Keinan A, Siepel
A. Genome-wide inference of natural selection on human transcription
factor binding sites. Nat. Genet. 2013. 45(7):723-729. PMCID: PMC3932982.

Gronau I, Arbiza L, Mohammed J, Siepel A. Inference of natural
selection from interspersed genomic elements based on polymorphism and
divergence. Mol. Biol. Evol. 2013. 30(5):1159-1171. PMCID: PMC3697874.

Gronau I, Hubisz MJ, Gulko B, Danko CG, Siepel A. Bayesian
inference of ancient human demography from individual genome
sequences. Nat. Genet. 2011. 43(10):1031-1034. PMCID: PMC3245873.

The ideal candidate will be enthusiastic, highly motivated and
independent; will have a strong background in comparative genomics,
population genomics, or transcriptional regulation, a serious
interest in molecular biology and genetics combined with good skills in
mathematics and computer science, and a strong record of accomplishment in
research. Postdoctoral associates in the group are expected to participate
in both methods development and data analysis, to be comfortable in
a fast-moving, interactive, and collaborative research environment,
and to actively publish and present results.

REQUIRED QUALIFICATIONS: Ph.D. in bioinformatics, computer science,
statistics, genetics, molecular biology, applied mathematics, or a
related field;  Research experience (with first-author publications)
in computational genomics or a closely related field;  Fluency in
probabilistic modeling and computational statistics;  Proficiency in
programming, ideally in C or C++, as well as in scripting languages
such as python, Perl, mat lab, or R; should be comfortable in a Linux
environment, with large data sets, computer clusters, and databases.

TO APPLY: Position Number
01229-R. <a href="http://bit.ly/1JtlFu9.">http://bit.ly/1JtlFu9.</a>  Informal inquiries
are also welcome.  Please submit a CV, a short description of research
interests and experience, and contact information for three references
by e-mail to asiepel@cshl.edu.

Adam Siepel, Ph.D.
Simons Center for Quantitative Biology
Cold Spring Harbor Laboratory
One Bungtown Road
Cold Spring Harbor, NY 11724

CSHL is an EO/AA Employer. All qualified applicants will receive
consideration for employment and will not be discriminated against on the
basis of race, color, religion, sex, sexual orientation, gender identity,
national origin, age, disability or protected veteran status. VEVRAA
Federal Contractor.


&ldquo;Gill, Irene&rdquo; 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:28</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136871565961">Graduate position: UAmsterdam.MothSexualSelection</a></div>
  <div class="description">PhD position available at the University of Amsterdam, Institute for
Biodiversity and Ecosystem Dynamics, Dept. Evolutionary Biology

Project: â€œUnderstanding sexual selection: which sex leads the dance
when both are signallers?â€�

Supervisor: Dr. Astrid T. Groot

<a href="http://bit.ly/1Jyq7I4">http://bit.ly/1Jyq7I4</a>

Sexual selection generally assumes that one sex is the signaller and
the other sex the responder. The forces of sexual selection are likely
more complex when both sexes are signallers and responders. In moths,
the species-specific female sex pheromones to which males are attracted
from a distance have been studied in detail, because these pheromones
can be used for pest management. However, at close-range males produce
and emit a biosynthetically related but different pheromone, which is
likely important for close-range female choice and/or male competition.
This project aims to determine the role of close-range courtship in
generating variation in the sexual communication channel to assess the
strength of mutual mate choice in a species where we have found a
within-population polymorphism in the female sex pheromone. This
project will combine behavioral and chemical analyses with genetic
(quantitative trait locus) analyses to answer the questions whether the
female and male sex pheromone genetically linked, what are the genes
underlying the male pheromone variation, what is the importance of male
sex pheromone variation in female choice and what is its role in
male-male competition.

Requirements

-MSc in Biology, with focus on evolution, and interest in chemical
ecology and entomology;

-Experience with molecular biological techniques, preferably also with
genetic analyses

-Interest to combine laboratory and field studies with genetic and
chemical analyses

-Skills in experimental design and statistical analysis

-Proficiency in scientific writing and communication

Applications should include a detailed CV, a motivation letter, and the
names and contact addresses of two references from which information
about the candidate can be obtained. Applications should be sent before
5 February to application-science@uva.nl. Please quote vacancy number
16-005 in the subject field. Interviews will be held in the week of 15
February.

Appointment
The position will start preferably March or April 2016. The full-time
appointment will be on a temporary basis for a maximum period of four
years. Initial appointment will be for a period of 18 months and after
satisfactory evaluation it will be extended for a total duration of
four years. The full-time gross monthly salary will range from â‚¬2,173
in the first year to â‚¬2,778 in the final year, according to the Dutch
salary scales for PhD candidates. The Collective Labour Agreement for
Dutch Universities is applicable. The annual salary will be increased
by 8 % holiday allowance and 8.3 % end-of-year bonus.

Astrid Groot 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:13</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136871160641">Graduate position: RoslinInstitute.UEdinburgh.CanineGenomics</a></div>
  <div class="description"><p>Ph.D. Studentship at The Roslin Institute (University of Edinburgh)
DEADLINE: 01 FEBRUARY 2016.

We are currently recruiting a 3-year Ph.D. student to investigate genetic diversity and evolutionary changes in dogs using genomic techniques. 

Domesticated animals provide an ideal study system for addressing a variety of questions in biology due to their substantial phenotypic variation, the rapid rate of phenotypic change and the broad diversification between breeds within a species. Dogs are particularly extreme in this regard, showing wide variation in both physical and behavioural characteristics, and thus are an excellent model species for questions in evolutionary biology, genetics and behavioural biology.
	
The student will apply population genetic and genomics techniques to investigate diversity within and between dog breeds, to identify genomic regions that show evidence of selection and to address whether signals of selection are enriched in regions of the genome associated with behavioural traits, as has been hypothesized more generally for domesticated animals. Bioinformatics analyses will be applied to investigate functional classes of genes that are enriched in genomic regions associated with these behavioural traits and in regions showing evidence of selection.
	
The studentship will provide training in transferable skills and techniques in quantitative and population genetics, statistics, bioinformatics and genomic analysis. The student will also be expected to regularly attend seminars, journal clubs and local meetings including the Edinburgh Alliance for Complex Traits. There will be opportunities for attendance at national and international conferences.

We are seeking an enthusiastic individual with an interest in population and evolutionary biology and animal genetics as well as aptitude with computer-based statistical analyses. The candidate should possess a good quality first degree in a relevant subject. Enthusiasm, dedication and aptitude are the most important characteristics; additional training in genetics, genome analysis and animal behaviour are available through MSc courses at the University of Edinburgh.

For further information about the project, contact Pam Wiener (pam.wiener@roslin.ed.ac.uk). For information about the application process and to apply, go to <a href="http://bit.ly/1kRFjoc.">http://bit.ly/1kRFjoc.</a> APPLICATIONS MUST BE SUBMITTED ONLINE.

via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:28</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136869854211">Other: SouthAfrica.VolResAssist.SocialMoleRats</a></div>
  <div class="description">SouthAfrica.VolResearchAssistant.SocialMoleRats

Cooperative breeding in Damaraland mole-rats (Fukomys damarensis).

Voluntary field research assistant: Mid-February to Mid-May 2016 (3
months)

We are looking for one voluntary field research assistants to conduct
research on the completely subterranean, highly social Damaraland
mole-rat (Fukomys damarensis). Specifically, this project investigates
the group augmentation hypothesis and how benefits of living in large
groups can generate selection for altruism in animals. We are currently
conducting a large scale capture-mark and recapture experiment running
over one year in which the voluntary research assistant would play an
important role. Field work takes place in the southern Kalahari,
Northern Cape province of South Africa in a private game reserve.
Entire groups of mole-rats will be captured and individually marked.
Morphological measurements and tissue samples will be obtained before
the release of the animals. The work is physically demanding (heavy
digging to capture mole-rats) and weather conditions are very
challenging (heat during the day, cold during the night). Field work
will make trapping during the night necessary. The assistant will
mostly be working along one more experienced scientist but will need to
work independently at times.

Applicants should be enthusiastic, willing to work hard and keen to get
involved in an ecological research project in the African bush. The
research assistants are expected to be willing to work at night when
trapping schedules require it and need to be fit as capturing mole-rats
requires the capacity to do hard physical work (digging) in a
challenging environment. Applicants must be holders of a driving
license. A zoology related degree and/or previous field experience will
be considered an asset. The successful applicant will work in a small
team of 2 persons and hence need to be a team player and generally a
person that enjoys collaboration under sometimes challenging and
isolated circumstances.

What you will learn:

Conduct field research in a remote area in the African bush.

Capture-mark and recapture techniques of small rodents.

Biological sampling and processing of scientific samples (blood, tissue
and morphometrics).

Accommodation is provided, and research assistants are paid a weekly
allowance to cover their personal costs and food. Costs for travelling
from Johannesburg to the field site will be covered. We have no funding
to cove overseas travel to South Africa.

Applications received until the 15th of January 2016 are ensured full
consideration. Later applications can be considered.

To apply (CV + cover letter) or enquire further information:

Dr Markus Zöttl

Research associate

University of Cambridge

+441223336673

Mz338@cam.ac.uk

Markus Zoettl 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:58</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136868873451">Course: GreifswaldU.GeneticAdaptation.Mar15-18</a></div>
  <div class="description">07-01-2016

International Summer School on Genetic Adaptation

Date:         15^th -18^th March 2016

Venue:      Greifswald University, Zoology Lab Building, SoldmannstraÃŸe 14

The DFG funded Research Training Group RESPONSE (Biological Responses
to Novel and Changing Environments; <a href="http://bit.ly/1JyiO3k">http://bit.ly/1JyiO3k</a>)
organizes a summer school on genetic adaptation. We are broadly
interested in responses to environmental change, for which genetic
adaptation may play an important role. The summer school aims at
deepening the knowledge of the prospects and limitations of different
genetic approaches in understanding adaptation to environmental change.

We will cover the following topics: Adaptive genetic variation and its
detection, Omics technologies in ecological research, heritability and
quantitative genetics, dynamics of rapid adaptation and niche
evolution. The programme will include lectures given by renowned
experts in the field and a young investigators&rsquo; day providing the
opportunity to present the own research project. There will also be
computer courses including exercises on estimating heritabilities and
on eco-evolutionary population modelling (for details please visit
<a href="http://bit.ly/1JyiO3k">http://bit.ly/1JyiO3k</a>).

The summer school is open to a limited number of external graduate /
PhD students. If you would like to participate, please send an email
including a short CV and a letter of motivation (no more than half a
page) to Michael SchÃ¶ner (schoenerm@uni-greifswald.de) before 15^th
February 2016. Participation is free of charge, though we cannot cover
travel expenses and accommodation.



Prof. Dr. Klaus Fischer
Zoological Institute & Museum
Greifswald University
J.-S.-Bach-Str. 11/12
D-17489 Greifswald
Phone: +49-3834-864266
Fax: +49-3834-864252

&ldquo;kfischer@uni-greifswald.de&rdquo; 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:28</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136867792381">Workshop: UWashington.SummerInstituteGenetics.RegistratonsOpen</a></div>
  <div class="description">Registrations for the 2016 Summer Institute in Statistical Genetics are 
now being accepted. Scholarship applications are processed as they are 
received.

Information is available at:



Bruce Weir
Department of Biostatistics
University of Washington
Box 359461
Seattle, WA 98195-9461
Phone: (206) 221-7947

Bruce Weir 
via Gmail</div>
 </div>
</div>
<h3>January 7, 2016</h3>
<div class="news-item">
 <div class="date">23:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136866503111">Other: Evolution 2016 and art</a></div>
  <div class="description"><p>Attention artists! The deadline for submission of evolution-themed works
(or proposed-works) for the &lsquo;Evolution 2016 Art Exhibit&rsquo; is quickly
approaching (Jan. 18).

The 2016 Evolution conference (the annual meeting of theSociety for
the Study of Evolution, theSociety of Systematic Biologists, and
theAmerican Society of Naturalists) will be held from June 17-21 in
beautiful Austin, TX. In conjunction with the meeting, the societies
and a local art gallery (Art.Science.Gallery) are co-sponsoring an
evolution-themed art exhibit. The exhibit will provide an opportunity for
meeting attendees and others to showcase their creative talents, increase
the visibility of the meeting and the societies to the local community,
and provide opportunities for public education about evolutionary
biology through the exciting lens of visual arts. The exhibit will run
at the Art.Science.Gallery leading up to, and during, the conference,
and a selection of the works will be on display at the Austin Convention
Center as a pop-up exhibit during the evening of the opening reception
of the meeting.

Art.Science.Gallery has issued an open call seeking submissions for this
exhibition. Works may explore (but are not limited to) the sub-disciplines
within evolutionary biology, notable evolutionary biologists, current
research topics, important discoveries and concepts, and the history of
evolutionary thought. You need not be a member of any of the societies
to submit something for consideration.

Pass the word along and, if you&rsquo;re at all artistically inclined, consider
submitting something. Additional details can be found on our permanent
meeting website (<a href="http://bit.ly/1PNgrcn">http://bit.ly/1PNgrcn</a>) under the 'News&rsquo; heading, or
on Art.Science.Gallery&rsquo;s website (<a href="http://bit.ly/1SDywfn">http://bit.ly/1SDywfn</a>).

The open call closes on Jan. 18.

Sincerely,
Your Evolution2016 organizers

howard.rundle@uottawa.ca


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:41</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136806587701">Other: Passing of Ted J. Case</a></div>
  <div class="description">It is with great sadness we inform his friends and colleagues that Prof.
Ted J. Case passed away on Thursday, December 31, 2015, at the age of 68.
Ted was a renowned island biogeographer, community ecologist, and
herpetologist.  He retired in 2004 after more than 25 years of service in
the Biology Department at UC San Diego.  David Quammen captured the essence
of Ted exceptionally well in his book $B!H(BThe Song of the Dodo$B!I(B.   A further
obituary will follow.


Robert Fisher
U.S. Geological Survey
San Diego Field Station
4165 Spruance Road, Suite 200
San Diego, CA 92101-0812

Phone: 619-225-6422
Phone (Admin):  619-225-6451
Fax: 619-225-6436

rfisher@usgs.gov


&ldquo;Fisher, Robert&rdquo; 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136805477651">Conference: UMichigan.CommunityEvolution.Mar12</a></div>
  <div class="description"><p>CALL FOR NOMINATIONS: 12th ANNUAL EARLY CAREER SCIENTISTS SYMPOSIUM

The Department of Ecology and Evolutionary Biology at the University of
Michigan invites nominations of outstanding scientists early in their
careers to participate in an exciting international symposium entitled
&ldquo;Frontiers in Community Assembly.&rdquo; The symposium events will take place on
12 March 2016, in Ann Arbor, Michigan. Dr. Tadashi Fukami and Dr. Rosemary
Gillespie will be the keynote speakers.

Potential topics include phylogenetic and ecological models for inferring
community structure, speciation and the evolution of secondary sympatry,
genomic processes underlying colonization and local adaptation, the
structuring of communities across environmental gradients, inferences from
paleo-communities.

We welcome nominations of early career scientists who are studying
ecological and evolutionary approaches to community assembly. Early career
scientists are considered senior graduate students (who stand to receive
their Ph.D. within one year) or postdoctoral researchers.  A colleague or
advisor may provide the nomination, however self nominations will be
accepted. The selected nominees will be invited to present their work and
to participate in panel discussions at the symposium.

The nomination consists of the nominees curriculum vitae, a brief abstract
of the proposed presentation (


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:41</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136805090251">Graduate position: University of Cologne, Adaptation in Arabidopsis</a></div>
  <div class="description"><p>1 PhD position
on the genomic basis of Adaptive plasticity in Arabidopsis lyrata
Lab. J. de Meaux

With a joint analysis of transcriptomic and phenotypic data, you will 
investigate patterns of parallel evolution in A. lyrata and its close 
relative A. thaliana. You will quantify regional differences in various 
plastic responses including response to temperature modifications or 
changing light quality in A. lyrata and compare this response to the 
related species A. thaliana. You will use a crossing scheme derived from 
classical quantitative genetics methods to investigate the distribution 
of various sources of genetic variation controlling gene regulation. 
Your project will shed light on the molecular basis of adaptive plastic 
responses in A. lyrata, an outcrossing relative of the model inbred 
species A. thaliana. With this project, you will develop skills ranging 
from genomics to molecular genetics and ecology.

The applicant must hold a Master degree in Evolutionary Biology, 
Genetics, Genomics or Bioinformatics and prove interest in plant 
molecular, population or ecological genetics. Experience in statistical 
analysis of quantitative data is required. This position is open to 
applicants of all nationalities but the language in the lab is English. 
Applications or questions regarding the position should be sent by mail 
to jdemeaux@uni-koeln.de, with the following subject line – PhD 
application adaptation lyrata – de Meaux lab. A letter of motivation, a 
CV and the contact to at least 2 independent referees should be 
provided, all in a single pdf file.  Revision of applications will begin 
in Jan. 20th, 2016 and continue until the position is filled. Funding is 
for 3-4 years with flexible starting date. For more information on our 
lab and research visit our website 
<a href="http://bit.ly/1Azj0bW">http://bit.ly/1Azj0bW</a> Interested students currently 
completing their Master thesis are encouraged to informally contact the 
PI if they have questions concerning the position.


Cologne is Germany’s vibrant Metropolis on the Rhine. The city is well 
known for its wild carnival, its famous Kölsch beer, its Cathedral and 
its vivid contemporary art and musical scene. Cologne is the fourth 
biggest city in Germany with over a million inhabitants from all over 
the world and an interesting mix of restored historic buildings and 
modern post-war architecture. Most importantly, Cologne University is 
one of the oldest and largest Universities in the Country. Our research 
group is hosted at the Biological Center of the University of Cologne 
and associated to the Collaborative Research network SFB680 
(<a href="http://bit.ly/1ezeCwf">http://bit.ly/1ezeCwf</a>) and to the Excellence Research Cluster 
CEPLAS (<a href="http://ceplas.eu/de/">http://ceplas.eu/de/</a>), which fosters active interactions between 
plant scientists of the Universities of Cologne, Düsseldorf and the Max 
Planck Institute of Plant Breeding Research. In this context, our PhD 
students are assured to start their scientific career in a world-class 
scientific environment.



via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:26</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136804703456">Postdoc: OregonStateU.SalmonEvolution</a></div>
  <div class="description">Post-Doctoral Research Associate

Oregon State University/Northwest Fisheries Science Center (NOAA
Fisheries)

Responsibilities: We are seeking a post-doctoral associate to
participate in our studies of the mechanisms and evolution of dispersal
(i.e. olfactory imprinting, homing, and straying) in Pacific salmon.
The post-doctoral associate will join a collaborative team of University,
Federal, and State researchers working to develop tools for improving
homing fidelity to salmon hatcheries. Specifically, the successful
candidate will develop and conduct studies to (1) identify olfactory cues
that may improve homing fidelity and, (2) identify critical periods for
olfactory learning of homing cues by salmon using an interdisciplinary
approach including behavioral experiments, electrophysiology, and
cellular/molecular biology.

Qualifications:

Ph.D. in Zoology, Biology, Neuroscience, Fisheries Science, or a closely
related field. Demonstrated research accomplishments and publications in
the primary research literature. Knowledge of fish physiology, behavior,
and ecology; a background in neurobiology and experience with molecular
and/or electrophysiology techniques would be helpful. An ability to work
independently and as part of multi-faceted research team is critical.

Salary: Competitive salary + Benefits

Start date: ASAP

Contact/Email:

Dr. David L. G. Noakes

Fisheries & Wildlife Department

Oregon Hatchery Research Center

104 Nash Hall, Oregon State University

Corvallis, Oregon

97331-3803 USA

david.noakes@oregonstate.edu

telephone: (01) (541) 737-1953  (OSU office)

                   (01) (541) 487-5513  (OHRC office)

Dr. Andrew Dittman

Environmental Physiology Program

Northwest Fisheries Science Center, NOAA Fisheries

2725 Montlake Boulevard East

Seattle, WA 98112 USA

andy.dittman@noaa.gov

telephone: (01) (206) 860-3392

&ldquo;Johnson, Marc&rdquo; 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:11</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136804287916">Postdoc: GeorgiaTech.MarineMicrobiology</a></div>
  <div class="description"><p>Postdoc in Marine Microbiology and Biogeochemistry 

Georgia Tech is looking for a postdoctoral fellow to study the diversity and physiology of marine microbes coupling anaerobic methane oxidation to denitrification. The postdoc will be based in the lab of Dr. Frank Stewart in the School of Biology, but will work jointly between the Stewart lab and the lab of Dr. Jen Glass (Earth and Atmospheric Sciences) on a collaborative NSF-funded project.  The work will involve oceanographic cruises (at least one per year) to oxygen minimum zone (OMZ) sites in the Pacific and Gulf of Mexico, shipboard and laboratory experiments, and a blend of biochemical, molecular, and bioinformatic analyses. Research tasks will involve both culture-dependent and independent methods, with the former focused on enrichments of target microbes from OMZs and the latter on comparative analysis of genomic and meta-omic datasets.  The postdoc will be encouraged to develop independent lines of research within the broader goals of the project, and will work colla
 boratively with Dr. Stewart, Dr. Glass, international collaborators, and graduate students to perform research and synthesize results for publication.  

The ideal candidate will be enthusiastic, motivated by experimental and analytical challenges, and proficient in biogeochemical and molecular microbiology techniques, ideally with experience in genomic or meta-omic analyses.  Candidates should have a Ph.D. focusing on environmental microbiology or bioinformatics (or a related topic).  Direct experience with anaerobic methane or nitrogen cycling or marine microbiology is highly desirable.  

The School of Biology at Georgia Tech is a dynamic research environment with a strong core of researchers interested in marine systems, microbiology, and genomics.  The Institute offers exceptional resources for biomolecular analysis, bioinformatics and high-performance computing, and exciting opportunities for cross-departmental collaboration with earth and atmospheric and computational scientists.  Georgia Tech was recently voted one of the best places to work, and Atlanta is consistently ranked among the top ten places to live for young professionals. 

This position begins in spring/summer 2016.  Funding is available for at least one year, with continuation contingent upon satisfactory progress in year one; applicants should express their ability to commit to the project for at least two years.  Application materials should be emailed to Frank Stewart at frank.stewart@biology.gatech.edu and should include a cover letter (describing your interest in the position, work experience, and availability), CV, and contact information (name, email, phone number) for at least three references.  Please include the word &ldquo;Postdoc&rdquo; in the subject line.  Salary will be competitive and commensurate with experience and will include fringe benefits.  Review of applications will begin on February 1 and continue until the position is filled.

Informal inquiries about the position can be sent to Frank Stewart at frank.stewart@biology.gatech.edu.  Additional details about the Stewart lab can be found at <a href="http://b.gatech.edu/1NG3uxB">http://b.gatech.edu/1NG3uxB</a> and the Glass lab at <a href="http://bit.ly/1SBJPER">http://bit.ly/1SBJPER</a>

Georgia Tech is a unit of the University System of Georgia and an Affirmative Action/Equal Opportunity Employer and requires compliance with the Immigration Control Reform Act of 1986.





via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:26</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136802991976">Graduate position:EdinburghNapierU.ModellingSocialEvolution</a></div>
  <div class="description"><p>PhD Studentship in modelling the evolutionary origin of social groups

School of Computing, Edinburgh Napier University, Edinburgh, UK

Dr Simon T. Powers

Deadline: 14th January 2016

The School of Computing at Edinburgh Napier University is advertising fully-funded PhD studentships, open to both UK and international students, to start in the first or second quarter of 2016. One of the proposed studentships is to work with Dr Simon T. Powers on modelling the evolutionary origins of social groups. Specific projects include the evolution of protocells at the origin of life, the evolution of multicellularity, and the evolution of human social institutions during the Neolithic transition from hunter-gatherer groups to agricultural societies. Our main approach is to investigate the selective drivers towards social group formation by modelling the co-evolution of traits that affect group formation, such as dispersal or group size preferences, with social behaviours such as cooperation and punishment. For human evolution, we are interested in how cultural evolution can lead to the creation of social institutions that select for cooperation in large groups of unre
 lated individuals. Our main methodology is analytical and simulation modelling. However, we are also interested in projects proposing the use of evolutionary robotics as a modelling tool, and projects that propose applications to cooperation in agent-based computing.

Related reading can be found in Dr Powers&rsquo; publications: <a href="http://bit.ly/1TGql0p">http://bit.ly/1TGql0p</a>

<a href="http://bit.ly/1mEczRr">http://bit.ly/1mEczRr</a>

Candidates should have strong quantitative skills, demonstrated experience of computer programming, and an interest in the evolution of social behaviour.

The formal advert for the studentships can be found at: <a href="http://bit.ly/1TGqn8o">http://bit.ly/1TGqn8o</a>

Students are encouraged to propose their own projects that broadly fit within the areas described above. Potential applicants are encouraged to informally discuss the position by email with Dr Powers: S.Powers@napier.ac.uk.

via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136802000746">Postdoc: UKentucky.EvolutionaryGenomics</a></div>
  <div class="description"><p>The Linnen Lab in the Biology Department at the University of Kentucky is
seeking a full-time postdoctoral researcher to participate in a USDA-funded
project that uses comparative genomics to investigate the genetic basis of
ecological specialization in plant-feeding insects. The ultimate goal of
the project is to test the hypothesis that changes in host use involve
predictable genetic changes. To test this hypothesis, we are combining
chemosensory gene annotations and host-use data from an experimentally
tractable and well-studied group of pine-feeding insect pests in the genus
*Neodiprion* (Hymenoptera: Diprionidae). This project dovetails with
ongoing work in the lab that seeks to understand both the causes (molecular
mechanisms) and consequences (population divergence and speciation) of
phenotypic variation. For additional details about current research in the
lab, please visit our website: <a href="http://bit.ly/1OcrtoF.">http://bit.ly/1OcrtoF.</a>

The ideal candidate will be independent, organized, and motivated and have
a keen interest in using genomic/transcriptomic data to address fundamental
questions in evolutionary biology. Candidates should have a Ph.D. in
evolutionary biology, genetics/genomics, or a related field.
Interest/experience in developing functional approaches for non-model
organisms is also desirable. Importantly, we are looking for someone who is
personable and enthusiastic about working in a collaborative environment.

The preferred start date is Spring or Summer 2016 (negotiable) and funding
is available for 2-3 years. Salary is commensurate with experience and
includes benefits.

Interested candidates should submit a cover letter describing research
interests/experience and a full CV with contact information for three
references. Application materials and all queries should be sent to:
catherine.linnen@uky.edu. Review of applications will begin immediately and
will continue until the position is filled.

The University of Kentucky is an equal opportunity/affirmative action
employer. Women, persons with disabilities, and members of other
underrepresented groups are encouraged to apply.

via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:41</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136801472801">Course: AveiroU_Portugal.AmphibianConservation.May16-20</a></div>
  <div class="description">I am contacting asking for the divulgation of the Amphibian Husbandry
and Conservation Course at the Biology Department of Aveiro University,
Portugal. We invited, Amphibian Ark to give us the pleasure of
their partnership.

The 1st Course on Amphibian  Husbandry and Conservation (AHC) is
an interesting and intensive course to researchers, technical staff
working with amphibians or anyone looking for more complete basis on
Amphibian Husbandry and Conservation. The course will be held from 16th
to 20th May 2016.

Registrations for this edition already available

<a href="http://bit.ly/1Uykcn8">http://bit.ly/1Uykcn8</a>

The course explores the principles of amphibians&rsquo;
husbandry, nutrition and dietary needs, captive
reproduction, population management, veterinary aspects (diseases,
pathology, and necropsy), biosecurity and quarantine, conservation,
threats and global action. Hands on demonstrations, practical and group
exercises are also included (enclosure demonstrations: tank drilling,
false bottoms and plumbing, filters).

Speakers:

  * Luis Carrillo, University of Mexico City, AArk
  * Michael Bungard, Wjitley Wildlife Conservation Trust (Paignton Zoo)
  * Arturo Munoz Saraiva, Coordinator of the Bolivian amphibian
    initiative and PhD in Ghent University
  * Isabel Lopes, University of Aveiro, CESAM

     Important note: the course will not start if a minimum of 10
   participants will not be reached; similarly the maximum number of
                          participants is 15.

Inscription fees include: attending at the course and the use of all
the materials provided by University of Aveiro and the AArk.

Fees:

AHC Course Early registration (29th February)

General: 280 euros

Student: 180 euros´

AHC Course

General: 350 euros

Student: 250 euros

Applications should be sent to: amphibianhc@sapo.pt till the 31st of
March 2016.

Instructions for payment will be notified to participants no later than
48h after. The payment proof must be sent to us for the registration to
be valid.

I grateful for your help.

The organizing committe

&ldquo;amphibianhc@sapo.pt&rdquo; 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:26</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136800917806">Graduate position: UNorthDakota.BisonGeneticsEvolution</a></div>
  <div class="description">U. North Dakota. AncientModernBisonGenetics.
The Laboratory of Human and Forensic Genetics at the University of North
Dakota (www.und.edu) is inviting applications from potential graduate
students who pursue the MS degree. PhD candidates will be also considered.

A student will be engaged to a project on the historic, current and future
status of bison herds from biological, archaeological, and cultural
perspectives. It is a collaborative effort with Theodore Roosevelt
National Park (THRO). This cross-disciplinary project represents an
opportunity to get intensive training in the methods of ancient and
modern DNA analyses including DNA isolation from a variety of tissues,
next-generation and traditional sequencing, quality control of DNA
templates, and bioinformatics. Genetic, phylogenetic, and demographic
studies of extirpated and extant bison specimens will be used to generate
important insights for management of bison at THRO and other locations
throughout the Midwest.

Candidates should demonstrate motivation for hard laboratory work
and strong interest in molecular genetics and evolution. Preference
will be given to students with a proven record of molecular biology
skills. Additional experience in bioinformatics is a plus.  If you are
interested you need to apply to the University of North Dakota Biology
Graduate Program using the regular procedure that can be found in the
UND Graduate School website:

<a href="http://bit.ly/1KW1wvu">http://bit.ly/1KW1wvu</a>

The additional information can be also found in the Biology Department
website:

<a href="http://bit.ly/1G8L7yc">http://bit.ly/1G8L7yc</a><a href="http://www.und.edu/">www.und.edu</a>) is inviting applications from potential graduate
students who pursue the MS degree. PhD candidates will be also considered.

A student will be engaged to a project on the historic, current and future
status of bison herds from biological, archaeological, and cultural
perspectives. It is a collaborative effort with Theodore Roosevelt
National Park (THRO). This cross-disciplinary project represents an
opportunity to get intensive training in the methods of ancient and
modern DNA analyses including DNA isolation from a variety of tissues,
next-generation and traditional sequencing, quality control of DNA
templates, and bioinformatics. Genetic, phylogenetic, and demographic
studies of extirpated and extant bison specimens will be used to generate
important insights for management of bison at THRO and other locations
throughout the Midwest.

Candidates should demonstrate motivation for hard laboratory work
and strong interest in molecular genetics and evolution. Preference
will be given to students with a proven record of molecular biology
skills. Additional experience in bioinformatics is a plus.  If you are
interested you need to apply to the University of North Dakota Biology
Graduate Program using the regular procedure that can be found in the
UND Graduate School website:

http://bit.ly/1KW1wvu

The additional information can be also found in the Biology Department
website:

http://bit.ly/1G8L7yc


The position starts in August /September 2016. To receive full
consideration, applications and required materials should be received
by the Biology Graduate Program not later than February 15, 2016.

Potential graduate students are strongly encouraged to make contact with
Dr. Igor Ovchinnikov. Contact information:

Dr. Igor Ovchinnikov
Associate Professor
Lab. Of Human and Forensic Genetics
Department of Biology
University of North Dakota
Email: igor.ovtchinnikov@email.und.edu


&ldquo;Ovtchinnikov, Igor&rdquo; 
via Gmail</div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="index8713.html?q=aggregator/sources/14&amp;from=0">first page</a></div><div><a href="index2be7.html?q=aggregator/sources/14&amp;from=140">previous page</a></div><div>... <a href="indexac35.html?q=aggregator/sources/14&amp;from=80">5</a> <a href="index2946.html?q=aggregator/sources/14&amp;from=100">6</a> <a href="index7de1.html?q=aggregator/sources/14&amp;from=120">7</a> <a href="index2be7.html?q=aggregator/sources/14&amp;from=140">8</a> <strong>9</strong> <a href="index2b69.html?q=aggregator/sources/14&amp;from=180">10</a> <a href="index2379.html?q=aggregator/sources/14&amp;from=200">11</a> <a href="index9605.html?q=aggregator/sources/14&amp;from=220">12</a> <a href="index024a.html?q=aggregator/sources/14&amp;from=240">13</a> ...</div><div><a href="index2b69.html?q=aggregator/sources/14&amp;from=180">next page</a></div><div><a href="index9a9b.html?q=aggregator/sources/14&amp;from=280">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news." class="active">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/14&from=160 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:29:17 GMT -->
</html>
