<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/12 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 17:29:16 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexfe3a.html?q=node/279">Contact</a></li><li><a href="index5ffe.html?q=node/6">Join</a></li><li><a href="index94ca.html?q=node/449">Policy on Discrimination and Harassment at Society Meetings</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 5 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator/sources/12" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator">news aggregator</a>
<ul>
<li class="expanded"><a href="indexe5f5.html?q=aggregator/sources">sources</a>
<ul>
<li class="leaf"><a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></li>
<li class="leaf"><a href="index4421.html?q=aggregator/sources/8">CiteULike Phylogeny</a></li>
<li class="leaf"><a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></li>
<li class="leaf"><a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></li>
<li class="leaf"><a href="index8aad.html?q=aggregator/sources/9">Evolutionary Bioinformatics</a></li>
<li class="leaf"><a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></li>
<li class="leaf"><a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></li>
<li class="leaf"><a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></li>
<li class="leaf"><a href="index9fb6.html?q=aggregator/sources/22">Phyloseminar</a></li>
<li class="leaf"><a href="index69a2.html?q=aggregator/sources/10">Systematics Association</a></li>
<li class="leaf"><a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></li>
<li class="leaf"><a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></li>
<li class="leaf"><a href="indexe0bd.html?q=aggregator/sources/15">dechronization</a></li>
<li class="leaf"><a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></li>
<li class="leaf"><a href="index2d4e.html?q=aggregator/sources/12" class="active">phylobabble.org</a></li>
<li class="leaf"><a href="index1f4a.html?q=aggregator/sources/7">postgenomic.com</a></li>

</ul>
</li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 4 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a> &raquo; <a href="index5640.html?q=aggregator">news aggregator</a> &raquo; <a href="indexe5f5.html?q=aggregator/sources">sources</a></div><h2>phylobabble.org</h2>
<!-- begin content -->
<div id="aggregator"><div class="feed">Latest topics<h3>URL</h3>
<div class="xml-icon"><a href="http://phylobabble.org/latest.rss"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div><a href="http://www.phylobabble.org/latest">http://www.phylobabble.org/latest</a>
<h3>Last update</h3>
1 hour 23 min ago</div><h3>March 2, 2016</h3>
<div class="news-item">
 <div class="date">10:34</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/holzdan">@holzdan</a> wrote:</p>
            
              <p>Hey peeps !</p>

<p>im very new to phylogenetics and currently trying to read myself a bit into the thematics of tree construction and getting started with all the different tools/software.</p>

<p>i would like to reconstruct a tree of 138 cyanobacterial strains defined by 16S , 16S-ITS , rpoc1 , RnaseP , rbclx , PC-IGS , PSA-IGS. The authors originally constructed the tree using maximum likelihood and the F84 model</p>

<p>I would like to implement another 4 strains where i got all loci sequences except the 16S-ITS.
Now im not sure how to deal with missing data  , should i construct a tree with all loci where i got missing sequences for only 4 strains , or should i use just the loci where i got total coverage ? is there a way to find the "best" method and will the choice of algorithm or substition model be influenced by this ?</p>

<p>cheers</p>
            
            <p>Posts: 1</p>
            <p>Participants: 1</p>
            <p><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Read full topic</a></p></div>
 </div>
</div>
<h3>February 22, 2016</h3>
<div class="news-item">
 <div class="date">09:51</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/ematsen">@ematsen</a> wrote:</p>
            
              <p>What would you like as a topic for the next series of talks on phyloseminar? Right now we are in a series of talks on biased sampling, and then the following one is going to be about phylogenetics and (protein) structure.</p>

<p>You can see the previous topics here: <a href="http://phyloseminar.org/recorded.html">http://phyloseminar.org/recorded.html</a></p>

<p>Feel free to nominate a previous topic if you would like to hear about new developments!</p>
            
            <p>Posts: 4</p>
            <p>Participants: 3</p>
            <p><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">Read full topic</a></p></div>
 </div>
</div>
<h3>February 20, 2016</h3>
<div class="news-item">
 <div class="date">17:11</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/ematsen">@ematsen</a> wrote:</p>
            
              <p>Hello everyone-- </p>

<p>I've decided to try automatically posting every new topic from phylobabble to the @phylobabble Twitter feed. If you don't like this, please comment. </p>

<p>Thanks!</p>
            
            <p>Posts: 1</p>
            <p>Participants: 1</p>
            <p><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Read full topic</a></p></div>
 </div>
</div>
<h3>February 18, 2016</h3>
<div class="news-item">
 <div class="date">02:56</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/conference-mceb-2016/614">Conference: MCEB 2016</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/stephaneguindon">@stephaneguindon</a> wrote:</p>
            
              <p>There's about a week left to register to MCEB 2016 (<a href="http://www.lirmm.fr/mceb2016/" rel="nofollow">http://www.lirmm.fr/mceb2016/</a>). The theme of this year's edition is "<strong>Forecasting evolution</strong>" and should be interest to many people on phylobabble.org. Below is the list of invited speakers:</p>

<ul>
<li>Jeffrey Jensen (Ecole Polytechnique Fédérale de Lausanne)</li>
<li>Laura S. Kubatko (Departments of Statistics and Evolution, Ecology, and Organismal Biology - The Ohio State University)</li>
<li>Amaury Lambert (Stochastics & Biology Group - Laboratoire de Probabilités & Modèles Aléatoires - UPMC - Université Paris 06 )</li>
<li>Richard Neher (Max Planck Institute for Developmental Biology)</li>
<li>Bruce Rannala (UC Davis - University of California)</li>
<li>Isabel Sanmartín Bastida (Real Jardín Botánico (RJB-CSIC), Madrid)</li>
<li>Yun S. Song (Departments of Mathematics and Biology - Universty of Pennsylvania)</li>
</ul>

<p>Hope to see you there,</p>

<p>-Stephane-</p>
            
            <p>Posts: 1</p>
            <p>Participants: 1</p>
            <p><a href="http://www.phylobabble.org/t/conference-mceb-2016/614">Read full topic</a></p></div>
 </div>
</div>
<h3>February 8, 2016</h3>
<div class="news-item">
 <div class="date">06:24</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/tree-based-measure-of-species-longevity/610">Tree-based measure of species longevity?</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/rutgeraldo">@rutgeraldo</a> wrote:</p>
            
              <p>Hi all,</p>

<p>does anyone know of a reasonable way to come up with estimates of species longevity within clades? By which I mean, how long a species / lineage sticks around before it either goes extinct or speciates?</p>

<p>The general idea is that one might have a (near?) complete, dated phylogeny with a bunch of genera in it and that the method I'm after would get me an estimate for average species longevity for each genus.</p>

<p>I guess you could say, crudely, that average species longevity is more or less proportional to average interior branch length (if we pretend that a species is an internode) but when extinction rates are high we'd overestimate, especially for the older internodes.</p>

<p>Isn't there some clever R package for this?</p>

<p>Thanks,</p>

<p>Rutger</p>

<p>p.s. apologies for the ill thought-out question. Equally ill thought-out answers and opinions are very welcome.</p>
            
            <p>Posts: 6</p>
            <p>Participants: 3</p>
            <p><a href="http://www.phylobabble.org/t/tree-based-measure-of-species-longevity/610">Read full topic</a></p></div>
 </div>
</div>
<h3>February 6, 2016</h3>
<div class="news-item">
 <div class="date">08:07</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/treebase-legacy-ids/609">TreeBase legacy IDs</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/max">@max</a> wrote:</p>
            
              <p>Dear babblers,</p>

<p>Does anyone know how to search <a href="https://treebase.org/treebase-web/home.html" rel="nofollow">treebase</a> for legacy matrix IDs? I'm trying to get some data sets from older [read 2012] papers but the records have changed and I can't search for the original (legacy) matrix ID.</p>

<p>Cheers,</p>

<p>Luiz</p>
            
            <p>Posts: 6</p>
            <p>Participants: 2</p>
            <p><a href="http://www.phylobabble.org/t/treebase-legacy-ids/609">Read full topic</a></p></div>
 </div>
</div>
<h3>February 5, 2016</h3>
<div class="news-item">
 <div class="date">01:30</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/summer-school-announcement-taming-the-beast-switzerland-june-2016/608">Summer school announcement: Taming the BEAST, Switzerland, June 2016</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/venelin_mitov">@Venelin_Mitov</a> wrote:</p>
            
              <p>Dear colleagues,</p>

<p>We are happy to announce our summer school "Taming the BEAST" and would kindly ask you to forward this email to interested students and PostDocs. We apologize if you receive this email more than once.</p>

<p>Phylogenetics and phylodynamics are central topics in modern biology. Phylogenetic inferences reconstruct the evolutionary relationships between organisms, whereas phylodynamic inferences reveal the dynamics that lead to the observed relationships. These two fields have many practical applications in disciplines such as epidemiology, developmental biology, paleontology, ecology and even linguistics. However, phylogenetics and phylodynamics are complex and fast-evolving fields. As such, inference tools are not easily accessible to researchers who are not from a computational background.</p>

<p><strong>Taming the BEAST</strong> is a summer school focusing on the BEAST2 software and consisting of a mix of invited talks, lectures and hands-on tutorials by leading and renowned experts in the field (including several of the core developers of BEAST2). The aim of this summer school is to equip participants with the skills necessary to confidently perform their own phylogenetic and phylodynamic inferences in Bayesian settings, while providing them with a firm grasp of the theory behind those inferences. Participants are also highly encouraged to bring their own datasets along and to engage with the organizers and speakers to address any problems specific to their own datasets/analyses.</p>

<p><strong>Registration is now open!</strong> We welcome applications from graduate students and early-career scientists in the life sciences. Preference will be given to applicants who are not from a computational background and applicants who have already collected/assembled a dataset that they need to analyze.</p>

<p><strong>Invited speakers:</strong>
Tanja Stadler (ETH Zurich)
Alexei Drummond (University of Auckland)
Tracy Heath (Iowa State University)
Tim Vaughan (University of Auckland)
Oliver Pybus (University of Oxford)</p>

<p><strong>Dates:</strong> June 26th to July 1st  (Registration deadline: March 15th).</p>

<p><strong>Place:</strong> Engelberg, Switzerland</p>

<p><strong>Registration Fee:</strong> 650 CHF for students and postdocs, 1000 CHF for industry. (Registration fee includes accommodation and meals).</p>

<p>For more information regarding the program, registration, venue, etc, please visit the summer school website: <a href="https://www.bsse.ethz.ch/cevo/taming-the-beast.html" rel="nofollow">https://www.bsse.ethz.ch/cevo/taming-the-beast.html</a></p>

<p>We hope to see you there,</p>

<p>the Taming the BEAST organizing team</p>
            
            <p>Posts: 1</p>
            <p>Participants: 1</p>
            <p><a href="http://www.phylobabble.org/t/summer-school-announcement-taming-the-beast-switzerland-june-2016/608">Read full topic</a></p></div>
 </div>
</div>
<h3>January 30, 2016</h3>
<div class="news-item">
 <div class="date">05:59</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/phylogenetic-placement-visualization/605">Phylogenetic Placement Visualization</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/alexis_raxml">@Alexis_RAxML</a> wrote:</p>
            
              <p>Ivica Letunic's itol3 tool can now visualize phylogenetic placements in .jplace format as generated by RAxML's EPA and pplacer  see <a href="http://itol3.embl.de/" rel="nofollow">itol3</a> You are free to send me feature requests which I will then collect and forward to Ivica,</p>

<p>Alexis</p>
            
            <p>Posts: 1</p>
            <p>Participants: 1</p>
            <p><a href="http://www.phylobabble.org/t/phylogenetic-placement-visualization/605">Read full topic</a></p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:59</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/phylogenetic-placement-vislualization/605">Phylogenetic Placement Vislualization</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/alexis_raxml">@Alexis_RAxML</a> wrote:</p>
            
              <p>Ivica Letunic's itol3 tool can now visualize phylogenetic placements in .jplace format as generated by RAxML's EPA and pplacer  see <a href="http://itol3.embl.de/" rel="nofollow">itol3</a> You are free to send me feature requests which I will then collect and forward to Ivica,</p>

<p>Alexis</p>
            
            <p>Posts: 1</p>
            <p>Participants: 1</p>
            <p><a href="http://www.phylobabble.org/t/phylogenetic-placement-vislualization/605">Read full topic</a></p></div>
 </div>
</div>
<h3>January 26, 2016</h3>
<div class="news-item">
 <div class="date">10:59</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/evolution-2016-spotlight-session-on-next-generation-phylogenetic-inference/604">Evolution 2016 spotlight session on next-generation phylogenetic inference</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/ematsen">@ematsen</a> wrote:</p>
            
              <p>Along with <a class="mention" href="users/trayc7.html">@trayc7</a> (Iowa State) and Felipe Zapata (San Francisco State), I'm organizing a SSB spotlight session at Evolution 2016.This is a call for speakers. ‘Spotlight sessions’ are a new type of session that differ from the traditional symposia at these meetings. For more information see links below.</p>

<p>Topic: 
Statistical phylogenetic inference methods, and in particular Bayesian methods, are reaching a hard limit on the number of taxa that can be used in an analysis. Although many interesting and worthwhile efforts have been made to increase the computational efficiency of these methods, it appears that existing inference strategies will simply not scale to orders of magnitude more taxa. In this session we will explore alternate strategies and algorithms for phylogenetic inference. </p>

<p>The spotlight session consists of one 28 min, four 13 min, and twelve 5 min (“lightning”) talks.  If you are interested in speaking in this session please submit an application to Erick Matsen at matsen@fredhutch.org.  The application must include your:  (1) Name; (2) Institution; (3) rank (e.g., student, postdoc, faculty); (4) names & institutions of co-authors on your talk; (5) a short abstract (max: 300 words); and (6) which talk type(s) you are willing to give (28, 13, or 5; the more flexible you are the more likely you are to be selected). Applications must be received by March 19th, 2016. No financial support is provided to selected speakers (i.e., speakers must pay their own travel, accommodation, registration, etc.).</p>

<p>You may still register to give a standard 13 min. talk in a regular contributed session at the meeting even if you apply to participate in a Spotlight session. However, if you are selected to speak in the spotlight session, you will have to choose which talk slot to use as each conference attendee is allowed to give only one oral presentation. Applicants will be informed about the outcome of their application by late March.</p>

<p>Organized sessions are committed to representing diversity.  All interested individuals are encouraged to apply.</p>

<ul>
<li><a href="http://www.evolutionmeetings.org/evolution-2016---austin-texas.html">Evolution 2016 conference website</a></li>
<li><a href="http://www.evolutionmeetings.org/instructions-for-presenters.html">Information for presenters, including Spotlight sessions</a></li>
</ul>
            
            <p>Posts: 3</p>
            <p>Participants: 1</p>
            <p><a href="http://www.phylobabble.org/t/evolution-2016-spotlight-session-on-next-generation-phylogenetic-inference/604">Read full topic</a></p></div>
 </div>
</div>
<h3>January 16, 2016</h3>
<div class="news-item">
 <div class="date">21:49</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/maximum-parsimony-wheather-the-best-tree-should-be-completely-resolved-under-this-criteria/601">Maximum parsimony. Wheather the best tree should be completely resolved under this criteria?</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/sergei">@Sergei</a> wrote:</p>
            
              <p>I've got some relults on the phylogeny of some living creatures based on 850 bp fragment of 28S RNA subunit.</p>

<p>Built a standard set of trees based on NJ, ML, MP and BI methods.</p>

<p>BI was constructed in MrBayes. And all the rest in R with "ape" and "phangorn" packages wherein MP topology obtained with "pratchet" function in phangorn.</p>

<p>MP has provided me with completely resolved phylogeny, whereas NJ, ML and BI shown polytomies in some terminal branches. I have never met with such results and realize this could be evidence of incorrect tree search strategy with other methods.</p>

<p>But still my questions are:</p>

<ol>
<li>Wheather MP method (especially parsimony ratchet) in some cases can reasonably produce more reliable results than others? (I remember about long-branch-attraction and homoplasy)</li>
<li>In addition to minimization of the number of "steps" wheather MP algorithms seek for fully resolved phylogenies and discard topologies with polytomy?</li>
</ol>

<p>Thank you!</p>
            
            <p>Posts: 2</p>
            <p>Participants: 2</p>
            <p><a href="http://www.phylobabble.org/t/maximum-parsimony-wheather-the-best-tree-should-be-completely-resolved-under-this-criteria/601">Read full topic</a></p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:40</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/maximum-likelihood-tree-reconstruction-pipeline/600">Maximum Likelihood tree reconstruction pipeline</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/arambaut">@arambaut</a> wrote:</p>
            
              <p>I recently had need to reconstruct a reasonably large Ebola virus tree and I wanted a maximum likelihood tree with good branch length estimates and bootstrap values for all the nodes in the ML tree.  We wrote this pipeline that bats back and forth between phyml (to get an initial NJ tree and optimise the branch lengths) and raxml (to find ML topologies and bootstrap). </p>

<p>I thought I would post it here for discussion and to see if anyone would do things differently. On a fixed topology phyml did a better job of optimising branch lengths but this may be because I couldn't find the appropriate settings in raxml. </p>

#phyml to find initial tree using BioNJ
phyml --quiet -i $1.phy -q -t e -a e -o lr -b 0
mv $1.phy_phyml_tree.txt initial_tree.newick

#raxml to find ml topology
raxml -f d -T 6 -j -s $1.fasta -n topology -m GTRGAMMA -t initial_tree.newick

#phyml to optimise branch lengths on ml topology
phyml --quiet -i $1.phy -q -t e -a e -o lr -u RAxML_result.topology -b 0
mv $1.phy_phyml_tree.txt $1.ml.tree

#use raxml to produce boostrap trees - the seed needs to be set here in $2
raxml -f a ­p $2 -s $1.fasta ­x 12345 -#200 -m GTRGAMMA -n bootstrap

#use raxml to project boostrap values on to ML tree
raxml -f b -t $1.ml.tree -z RAxML_bootstrap.bootstrap -m GTRGAMMA -n final
mv RAxML_result.final $1.ml_bootstrap.tree
            
            <p>Posts: 5</p>
            <p>Participants: 3</p>
            <p><a href="http://www.phylobabble.org/t/maximum-likelihood-tree-reconstruction-pipeline/600">Read full topic</a></p></div>
 </div>
</div>
<h3>January 14, 2016</h3>
<div class="news-item">
 <div class="date">21:03</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/how-to-recognize-a-rearranged-tree/599">How to recognize a rearranged tree?</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/ematsen">@ematsen</a> wrote:</p>
            
              <p><a class="mention" href="users/cwhidden.html">@cwhidden</a> and I had an entertaining discussion about the following question, which apparently came up in the <a class="mention" href="users/alexis_raxml.html">@Alexis_RAxML</a> lab as well. By tree rearrangement I mean an operation on some class of trees, such as subtree-prune-regraft (SPR) on unrooted trees.</p>

<p>Given some collection <em>A</em> of trees and a member <em>t</em> of <em>A</em>, how can we decide if a rearranged version of <em>t</em> is a member of <em>A</em>?</p>

<p>It seems like we'd like some sort of means of serializing trees that is compatible with the class of tree rearrangements. <a class="mention" href="users/armanbilge.html">@armanbilge</a> suggested thinking of a tree as the collection of its clades, and encoding that collection of clades as a binary vector indexed by the power set on the taxa (i.e. the set of all taxon subsets), and then keep track of what clades appear and disappear upon rearrangement. This would work, though the power set gets pretty big. </p>

<p>Thoughts? cc <a class="mention" href="users/mathmomike.html">@mathmomike</a> <a class="mention" href="users/david_bryant.html">@david_bryant</a></p>
            
            <p>Posts: 12</p>
            <p>Participants: 7</p>
            <p><a href="http://www.phylobabble.org/t/how-to-recognize-a-rearranged-tree/599">Read full topic</a></p></div>
 </div>
</div>
<h3>December 4, 2015</h3>
<div class="news-item">
 <div class="date">11:10</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/troubleshooting-rpanda-code/595">Troubleshooting RPANDA code</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/anurag_mishra">@Anurag_Mishra</a> wrote:</p>
            
              <p>I'm trying to test maximum likelihood fit of environmental birth death model specified in Helen Morlone's paper. </p>

<p>Using the code given in the manual: </p>

data(a)
tot_time</div>
 </div>
</div>
</div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news." class="active">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/12 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 17:29:47 GMT -->
</html>
