<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=220 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 21:41:52 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index66e2.html?q=node/119">54(6) December 2005</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 2 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>January 30, 2016</h3>
<div class="news-item">
 <div class="date">05:59</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/phylogenetic-placement-vislualization/605">Phylogenetic Placement Vislualization</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/alexis_raxml">@Alexis_RAxML</a> wrote:</p>
            
              <p>Ivica Letunic's itol3 tool can now visualize phylogenetic placements in .jplace format as generated by RAxML's EPA and pplacer  see <a href="http://itol3.embl.de/" rel="nofollow">itol3</a> You are free to send me feature requests which I will then collect and forward to Ivica,</p>

<p>Alexis</p>
            
            <p>Posts: 1</p>
            <p>Participants: 1</p>
            <p><a href="http://www.phylobabble.org/t/phylogenetic-placement-vislualization/605">Read full topic</a></p></div>
  <div class="source">Source: <a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></div>
 </div>
</div>
<h3>January 29, 2016</h3>
<div class="news-item">
 <div class="date">03:04</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138271747141">Graduate position: Halle_Germany.HoneybeeResistance.2</a></div>
  <div class="description">Halle_Germany.HoneybeeResistance

PhD project in the Molecular Ecology group at Martin-Luther-University,
Halle-Wittenberg, Germany.

The aim of the project is to find genetic basis of the honey bee Apis
mellifera resistance to Varroa destructor. This will be done by using
the latest methods in genomics, transcriptomics and proteomics. Thus
representing a unique opportunity to gain experience in an area which
is of high demand in the research field. The group has excellent
laboratory facilities with all state of the art equipments including a
next generation sequencing platform.

The PhD position is funded for three years in which time the PhD
student is expected to finish her/his degree.

The chosen PhD student is expected to work independently within the
project framework. The skills should reflect the project description.
The candidate will be familiar with standard molecular methods in the
lab as well as bioinformatics and running scripts. Willingness to learn
new methods is a must as well as ability to adapt to ever-changing
circumstances. Some travelling to France or India might be included.
Excellent command of English in both writing and speaking is essential.
Knowledge of German is beneficial but not must.

Information of the research group can be obtained from Dr. Jarkko
Routtu, tel: 0049 (0) 345 55 26382, email:
jarkko.routtu(at)zoologie.uni-halle.de

<a href="http://bit.ly/1PXCY3G">http://bit.ly/1PXCY3G</a>

<a href="http://bit.ly/1SNxEG2">http://bit.ly/1SNxEG2</a>

Extended application deadline before 8th of February 2016, subject line
referring to &ldquo;PhD resistance&rdquo;, together with your letter of
motivation, CV, publication list, names of two referees in a single pdf
file.

Jarkko Routtu 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:04</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138271746921">Graduate position: MaxPlanckInst_Berlin.EvolutionHumanBehaviour</a></div>
  <div class="description">PhD position on Human Collective Behaviour. Max Planck Institute for
Human Development, Berlin.

Starting data: The position is available from May 2016 onwards, but
later start dates are possible.

Description: The PhD project will investigate the role of individual
differences in human and animal collective behaviours, including group
formation, group coordination and conflict resolution. It will involve
experiments with human crowds using GPS tracking devices & video
tracking, virtual interactive platforms and simulations. Limited work
on animal groups (e.g., fish) is also possible. We seek a PhD student
with a strong empirical background and excellent skills in (spatial)
data analysis. Programming skills are a bonus.

The Center for Adaptive Rationality at the Max Planck Institute for
Human Development (<a href="http://bit.ly/1PXCY3C">http://bit.ly/1PXCY3C</a>) is a highly
interdisciplinary and international research group where English is the
working language. We offer an excellent infrastructure including
support staff and equipment for conducting experiments (e.g.,
behavioural laboratory, GPS tracking and supercomputers).

The predoctoral contract is for three years. Applications (consisting
of a cover letter describing your research interests, a CV, up to two
publications, and two letters of recommendation) should be sent as a
single PDF file, with your name as the file name, to Monika Oppong
(oppong@mpib-berlin.mpg.de; Max Planck Institute for Human Development,
Lentzeallee 94, 14195 Berlin).

Applications will be considered until the position is filled.
Application review will begin on February 1, 2016 but applications
after this date will be considered equally. For further inquiries about
the position, please contact Ralf Kurvers (kurvers@mpib-berlin.mpg.de).

More information:

<a href="http://bit.ly/1SNxChy">http://bit.ly/1SNxChy</a>
nanzeige_predoctoral_contract-arc_39-2015_10.12.2015_intern.pdf

&ldquo;Kurvers, Ralf&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:33</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138271013966">Postdoc: UExeter.EvolutionInsectAntibiotic</a></div>
  <div class="description">New post-doctoral position available with Ben Raymond and Angus Buckling
at the University of Exeter Penryn campus.  This is an 18 month project
looking at how best to use phage to combat the evolution of plasmid borne
antibiotic resistance in insect/gut bacteria model systems.   More details
available at <a href="http://bit.ly/1KeGUjX">http://bit.ly/1KeGUjX</a>  or
contact b.raymond@exeter.ac.uk or A.J.Buckling@exeter.ac.uk


&ldquo;Raymond, Ben&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>January 28, 2016</h3>
<div class="news-item">
 <div class="date">02:26</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138207036661">Job: DenverBotanicGardens.AssistCuratorMycology</a></div>
  <div class="description">*Job Title:* Assistant Curator of Mycology

*Job Summary:* Coordinates the growth and improvement of Denver Botanic
Gardens mycology program, advocate biodiversity science and natural
history collections, develop an active externally funded research program
that incorporates postdoctoral fellows, graduate students and
undergraduates, and oversees volunteers.

*Career Type*: Full-time

*Location*: York Street

*Qualifications*: Ph. D. in mycology, evolution, or equivalent with
interest in developing field and collection orientated research projects
related to the study and conservation of natural ecosystems within our
region and show leadership in science communication and engagement; or
minimum 6 years related experience and/or training; or equivalent
combination of education and experience. Must demonstrate a strong record
of research accomplishments and acquisition of funding. Experience with
fungal identification, specimen digitization and curation techniques and
interest in formal and informal science education, preferred. Must possess
strong communication skills, both written and spoken, needs excellent time
management skills, and good interpersonal skills.

*Benefits*: Major benefits include choice of medical plans, dental, vision
and life insurance, medical and daycare Flexible Spending Account (FSA) and
Healthcare Spending Account (HSA) ; a generous retirement plan plus paid
holidays, vacation, personal and sick time and access to personal lines of
insurance.   Additional benefits include short-term and long-term
disability coverage, an Employee Assistance Program (EAP), an ECO pass and
a complimentary Denver Botanic Gardens membership, access to Gardens
education classes and summer concert tickets.

*Additional Information: *

Review of applications will begin April 2016 and continue until the
position is filled with an anticipated start date of September 1, 2016.
Applications should include a curriculum vitae and statement of research
interests (3 pages maximum).

Send resume and/or application to *Human Resources, Denver Botanic Gardens,
1007 York St., Denver CO 80206*, or *e-mail your resume to
hr@botanicgardens.org*. We are a nonprofit, EOE.

Rebecca Hufft 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>January 26, 2016</h3>
<div class="news-item">
 <div class="date">10:59</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/evolution-2016-spotlight-session-on-next-generation-phylogenetic-inference/604">Evolution 2016 spotlight session on next-generation phylogenetic inference</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/ematsen">@ematsen</a> wrote:</p>
            
              <p>Along with <a class="mention" href="users/trayc7.html">@trayc7</a> (Iowa State) and Felipe Zapata (San Francisco State), I'm organizing a SSB spotlight session at Evolution 2016.This is a call for speakers. ‘Spotlight sessions’ are a new type of session that differ from the traditional symposia at these meetings. For more information see links below.</p>

<p>Topic: 
Statistical phylogenetic inference methods, and in particular Bayesian methods, are reaching a hard limit on the number of taxa that can be used in an analysis. Although many interesting and worthwhile efforts have been made to increase the computational efficiency of these methods, it appears that existing inference strategies will simply not scale to orders of magnitude more taxa. In this session we will explore alternate strategies and algorithms for phylogenetic inference. </p>

<p>The spotlight session consists of one 28 min, four 13 min, and twelve 5 min (“lightning”) talks.  If you are interested in speaking in this session please submit an application to Erick Matsen at matsen@fredhutch.org.  The application must include your:  (1) Name; (2) Institution; (3) rank (e.g., student, postdoc, faculty); (4) names & institutions of co-authors on your talk; (5) a short abstract (max: 300 words); and (6) which talk type(s) you are willing to give (28, 13, or 5; the more flexible you are the more likely you are to be selected). Applications must be received by March 19th, 2016. No financial support is provided to selected speakers (i.e., speakers must pay their own travel, accommodation, registration, etc.).</p>

<p>You may still register to give a standard 13 min. talk in a regular contributed session at the meeting even if you apply to participate in a Spotlight session. However, if you are selected to speak in the spotlight session, you will have to choose which talk slot to use as each conference attendee is allowed to give only one oral presentation. Applicants will be informed about the outcome of their application by late March.</p>

<p>Organized sessions are committed to representing diversity.  All interested individuals are encouraged to apply.</p>

<ul>
<li><a href="http://www.evolutionmeetings.org/evolution-2016---austin-texas.html">Evolution 2016 conference website</a></li>
<li><a href="http://www.evolutionmeetings.org/instructions-for-presenters.html">Information for presenters, including Spotlight sessions</a></li>
</ul>
            
            <p>Posts: 3</p>
            <p>Participants: 1</p>
            <p><a href="http://www.phylobabble.org/t/evolution-2016-spotlight-session-on-next-generation-phylogenetic-inference/604">Read full topic</a></p></div>
  <div class="source">Source: <a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:25</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138077598351">Workshop:</a></div>
  <div class="description">UCalifornia_LosAngeles.ConservationGenomics.Mar20-24
Reply-To: Mario Colon 
x-mailer: Apple Mail (2.3112)

UCLA/La Kretz Workshop in Conservation Genomics, 20-24 March, 2016

Conservation biology, genetics, and evolutionary biology have had a long and intimate 
relationship, and conservation constitutes one of the key applications of evolutionary 
analysis to real-world biological problems. The impacts of population and landscape 
genetics and gene expression studies have been particularly striking, and are helping to 
solve some of the most pressing problems in biological conservation.

As the field of conservation genetics continues to grow and mature, the availability of 
genome-scale data stand to make profound new contributions to our ability to identify and 
protect at-risk populations and recover those that are most endangered. However, 
genomic analyses also carry a heavy burdenXdata sets are enormous, often requiring 
diverse computational approaches for assembly, quality control and analysis.

The La Kretz annual workshop provides a comfortable, rigorous, but informal training 
environment for a small group of motivated graduate students to explore how conservation 
problems can best be addressed with genomic-level data. Our goal is to provide hands-on 
experience in the efficient collection, troubleshooting, and analysis of large data sets for 
conservation-relevant problems. One of the highlights of our workshop is active 
participation from members of several governmental agencies who are at the forefront of 
endangered species protection and management, providing a forum for exploring the most 
relevant aspects of conservation genomics to managers.

The UCLA/La Kretz workshop is held at the La Kretz Field Station 
(<a href="http://bit.ly/1aXROSZ">http://bit.ly/1aXROSZ</a>) and Stunt Ranch Reserve 
(<a href="http://bit.ly/1BdyJvO">http://bit.ly/1BdyJvO</a>) in the heart of the Santa Monica Mountains. Only 30 miles 
from UCLA and the LAX airport, but nestled in the relatively undeveloped 160,000 acre 
Santa Monica Mountains National Recreation Area, the Field Station/Stunt Reserve 
provides an ideal location to explore new developments in genomic science and pressing 
needs in conservation and management together in a single setting.

Our current instructor list, drawn from UCLA faculty and several other partner institutions, 
includes:

Ben Fitzpatrick (U. Tennessee Knoxville)

Kirk Lohmueller

Evan McCartney-Melstad

Gideon Bradburd (UC Berkeley)

Brad Shaffer

Victoria Sork

Bob Wayne

Ying Zhen

Plus a soon-to-be-disclosed RNA-seq teamK


Agency partners represented include:
USGS

BLM

USFWS

CDFW


Topics covered include:

Overview of traditional conservation genetics

Next generation platforms: the best tool for the job

Data management pipelines:

Quality Control

Data storage

Data organization

Data types and analyses:

SNPs

Sequences

Exploring very large data sets

Functional genomic data

RNA-seq

Genomic data and GIS

Visualizing geographic structure and demographic history

Admixture, clines, and hybridization

Detecting adaptive variation

RNA-seq and the analysis of genes that matter


Prerequisites

Available housing limits course enrollment to ~20 students. Preference is given to doctoral 
candidates who are in the early to middle stages of their thesis research, and who have 
some familiarity with using a command line interface or programming languages (i.e. Perl, 
python etc.). Postdocs, faculty, and government researchers may also apply, but 
preference will be given to graduate students. We encourage applications from women, 
minorities, and individuals from under-represented demographics in the sciences.

Admission and Fees

Applicants will be admitted based on academic qualifications and appropriateness of 
research interests. The course fee is $425. This includes food and lodging at the La Kretz 
Field Station, as well as any incidental fees, for the duration of the course (arriving Sunday 
March 20, departing Friday March 25). In addition, course participants who would like to 
extend their stay at the field station for the remainder of the weekend may do so for no 
extra charge. For those opting to stay the weekend, departure time will be by 5:00 pm on 
Sunday March 27.

This year, the workshop is co-sponsored by a UC Catalyst program grant that will provide 
some support for faculty and student support and staffing needs. University of California 
students may apply for partial support (up to $225) from the newly awarded Conservation 
Genomics Consortium ( <a href="http://bit.ly/1S8bT44">http://bit.ly/1S8bT44</a> ) funded 
under the UC Catalyst program.

UCLA students are encouraged to take the La Kretz Workshop for graduate credit. Other 
UC students may also be able to take the course for credit.


Application Forms and Information

Visit the UCLA/La Kretz Center for California Conservation Science website for additional 
information and to download the application 
(<a href="http://bit.ly/1KBzQZo">http://bit.ly/1KBzQZo</a>)


Application Deadline:

Applications are due by February 15, 2016. Please send a completed application form and 
a short letter of recommendation from your major advisor. Students will be notified via e-
mail by February 19, 2016 of acceptance.

Applications should be emailed as PDFs to:

Mario Colon, (mario.colon@ucla.edu). Please note if you are a University of California 
student or postdoc, and if so, whether you are requesting financial assistance.

Mario Colon 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:25</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138077598106">Graduate position:ETHZurich.EcologyEvolutionModeling</a></div>
  <div class="description"><p>PhD position: Modeling ecology/evolution of a plant pathogen, ETH Zurich


***The project***

will explore population dynamics of Zymoseptoria tritici, an important
fungal pathogen of wheat, using eco-evolutionary modeling and field/lab
experimentation.

Plant diseases often cause serious yield losses in agriculture. Current
chemical and genetic technologies used to manage plant diseases are
highly vulnerable to pathogen evolution and are not sustainable.
Pathogen evolution is facilitated by the genetic uniformity underlying
modern agroecosystems. Thus, there is an urgent need to develop new,
efficient and sustainable ways to control plant diseases.

This PhD project intends to make an important contribution towards
achieving this goal.  The first phase aims at comprehensive
characterization of both epidemic and evolutionary factors of the
pathogen by combining eco-evolutionary modeling, field/lab experiments
and genome sequence analyses. In the second phase the models will be
used to predict the rate of pathogen adaptation to disease control and
propose optimal spatio-temporal patterns for applications of fungicides
and resistant host varieties that can disrupt pathogen adaptation.


***Working conditions***

Plant Pathology group at the Institute of Integrative Biology, ETH
Zurich offers a creative, collaborative research environment and
excellent infrastructure (<a href="http://www.path.ethz.ch/">www.path.ethz.ch</a>). Salary is approximately
48000 Swiss francs per year. Zurich is a vibrant, international and
green city.

The PhD project is funded for three years by the Ambizione grant from
the Swiss National Foundation with a possibility of extension for the
fourth year.

Unique training in mathematical modeling, bioinformatics and field/lab
experiments increases chances to find a job in academia, industry or
governmental agencies.


***Requirements***

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>January 25, 2016</h3>
<div class="news-item">
 <div class="date">10:50</div>
 <div class="body">
  <div class="title"><a href="http://iphylo.blogspot.com/2016/01/ispecies-is-back-mashing-up-species-data.html">iSpecies is back: mashing up species data</a></div>
  <div class="description"><p>A decade ago (OMG, that can't be right, an actual decade ago) I created "iSpecies", a simple little tool to mashup a variety of data from GBIF, NCBI, Yahoo, Wikipedia, and Google Scholar to create a search engine for species. It was written in PHP, relied on some degree of *cough* web scraping to get its data, and was a bit of a toy (although that didn't stop me complaining that it could do more than EOL at the time). Eventually I got sick of dealing with Google Scholar constantly changing it's HTML and blocking IP addresses to stop people harvesting data (I once managed to get my entire campus blocked), or services disappearing such as Yahoo's image search, and I eventually pulled the plug on it.</p> <p>A short <a href="http://iphylo.org/~rpage/phyloinformatics">course I run on "phyloinformatics"</a> starts this week and one of the examples I show is a crude Javascript-based mashup. It struck me that I could tweak that and recreate a simple version of iSpecies, and that's exactly what I've done: <a href="http://ispecies.org/">http://ispecies.org</a>.</p>  <p>It's nothing fancy, just takes a species name and searches GBIF, EOL, CrossRef, and Open Tree of Life, grabs some data and puts it together on a web page. There are lots of limitations (e.g., only fetches the first 300 localities in GBIF, requires scientific names, tree viewer is pretty awful) but it was pretty simple to put together. It's entirely client-side based, the code is all in the HTML file (and a few Javascript libraries) (the code is on GitHub: <a href="https://github.com/rdmpage/ispecies">https://github.com/rdmpage/ispecies</a>).</p> <p>Fun as this was, there's a bigger problem with iSpecies and that's that it is a "mashup". I'm simply grabbing data from different sources and redisplaying it. What I really want is what has been described as a "mashup" (awful term, don't use it), that is, I want to combine the data so that it is more than the sum of its parts. For example, some of the data could be cross linked (especially if add a few more sources and we drill down a bit). Some of the papers discovered by CrossRef may include original descriptions, or may be the source of some of the points plotted on the GBIF map. Some may include the phylogenies used to build the Open Tree of Life tree. In order to build a data mashup instead of a web mashup we need to operate at the level of data rather than just human-readable web pages. That is the next thing I'd like to work on, and in many ways it shouldn't be a big leap. The new iSpecies was fairly easy to create because we now have a bunch of web services that all speak JSON. It's a small step from JSON to JSON-LD (especially if the JSON-LD is constructed with reuse in mind). So while it's nice to see iSpecies back, there's a much more interesting next step to think about.</p></div>
  <div class="source">Source: <a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></div>
 </div>
</div>
<h3>January 24, 2016</h3>
<div class="news-item">
 <div class="date">16:30</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2016/01/what-is-in-traditional-chinese.html">What is in Traditional Chinese Medicines — a network analysis</a></div>
  <div class="description"><br />I used to work for the New South Wales Institute of Technology, which in the late 1980s mutated into the <a href="https://en.wikipedia.org/wiki/University_of_Technology_Sydney" target="_blank">University of Technology Sydney</a> (UTS). During this process it acquired an organization called the College of Traditional Chinese Medicine. This group was placed in the Faculty of Science, for lack of anywhere else to put it.<br /><br />These people had little contact with the rest of the faculty, and I don't recall ever meeting any of them. Indeed, their work was not really based on Western science. These days, the UTS College of Traditional Chinese Medicine offers a <a href="http://handbook.uts.edu.au/courses/c10186.html" target="_blank">Bachelor of Health Science in Traditional Chinese Medicine</a>, although they are most obvious in the <a href="http://www.uts.edu.au/about/faculty-science/traditional-chinese-medicine-clinic/about-us/uts-chinese-herbal-medicine" target="_blank">UTS Chinese Herbal Medicine Clinic</a>, which is also nominally still part of the Faculty of Science.<br /><br />The presence of Traditional Chinese Medicine (TCM) in an Australian university setting is relevant to today's blog post, because Australia seems to be one of the few places to have shown any interest in connecting TCM and Western science. Indeed, there is also a <a href="https://www.uws.edu.au/ssh/school_of_science_and_health/uniclinic/traditional_chinese_medicine" target="_blank">Uniclinic of Traditional Chinese Medicine</a> within the School of Science and Health at Western Sydney University. Most of the interest in studying TCMs has otherwise been confined to Asia (see Dennis Normile. 2003. The new face of Traditional Chinese Medicine. <a href="http://science.sciencemag.org/content/299/5604/188" target="_blank">Science</a> 299: 188-190).<br /><br /><a href="http://1.bp.blogspot.com/-nGEmbBD8gGQ/VqSjrfpZwpI/AAAAAAAAD4o/XQXHqZ__Gtg/s1600/TCM.gif" imageanchor="1"></a><br />Recently, a group of Australian researchers decided to have a look at the content of some of the TCMs available in their country:<br />Megan L. Coghlan, Garth Maker, Elly Crighton, James Haile, Dáithí C. Murray, Nicole E. White, Roger W. Byard, Matthew I. Bellgard, Ian Mullaney, Robert Trengove, Richard J.N. Allcock, Christine Nash, Claire Hoban, Kevin Jarrett, Ross Edwards, Ian F. Musgrave & Michael Bunce (2015) Combined DNA, toxicological and heavy metal analyses provides an auditing toolkit to improve pharmacovigilance of traditional Chinese medicine (TCM). <a href="http://www.nature.com/articles/srep17475" target="_blank">Nature Scientific Reports</a> 5: 17475.Some of these TCMs (12 out of 26) are registered for use with the Therapeutic Goods Administration, which regulates their use within Australia, while the other TCMs are not (which technically means that they should not have been commercially available). However, there is little in the way of pharmacovigilance of herbal medicines anywhere in the world.<br /><br />All of the products were comprehensively audited for their biological (via next generation DNA sequencing), toxicological (LC-MS analysis) and heavy metal (arsenic, cadmium and lead, via SF-ICP-MS analysis) contents. For the latter two analyses the amount of material detected was also quantified.<br /><br />As usual, we can use a phylogenetic network to visualize these data, which I have done using a neighbor-net network on the presence-absence data. The result is shown in the figure. TCMs that are closely connected in the network are similar to each other based on their detected contents, and those that are further apart are progressively more different from each other. The registered products are highlighted in red.<br /><br /><a href="http://2.bp.blogspot.com/-nBuzV5QzkJM/VqSSiA5rbGI/AAAAAAAAD4Y/55B8XJIpD9I/s1600/TCMcontent.gif" imageanchor="1"></a><br />There is wide variation among the products. The seven most divergent TCMs in the network are all unregistered, with the remaining seven being more similar to the registered TCMs. Only two TCMs (TCM10 and TCM17) have no discrepancies between the detected contents and what was declared (either to the regulatory agency, or to the consumer in the form of an ingredients list).<br /><br />The authors summarize this situation:<br />Genetic analysis revealed that 50% of samples contained DNA of undeclared plant or animal taxa, including an endangered species of <i>Panthera</i> (snow leopard). In 50% of the TCMs, an undeclared pharmaceutical agent was detected including warfarin, dexamethasone, diclofenac, cyproheptadine and paracetamol. Mass spectrometry revealed heavy metals including arsenic, lead and cadmium, one with a level of arsenic >10 times the acceptable limit.This study presents genetic, toxicological, and heavy metal data that should be of serious concern to regulatory agencies, medical professionals and the public who choose to adopt TCM as a treatment option. Of the 26 TCMs investigated, all but two can be classified as non-compliant on the grounds of DNA, toxicology and heavy metals, or a combination thereof. In total, 92% were deemed non-compliant with some medicines posing a serious health risk.Such findings are not only of concern to the consumer, but also flag the need for detailed auditing of herbal preparations prior to evaluation in clinical trials.<br /></div>
  <div class="source">Source: <a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:05</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137936249766">Postdoc: UCalifornia_Berkeley.3.EvolutionAntSociality</a></div>
  <div class="description">*JPF00959*

The Department of Environmental Science, Policy, and Management at
UC-Berkeley is currently seeking one Postdoctoral Scholar in *Chemical
Ecology *in the laboratory of Dr. Neil Tsutsui. We are seeking a
Postdoctoral Scholar to study the chemical basis of ant social behaviors.
Previous work has focused on the genetics, behavior, and chemical ecology
of the invasive Argentine ant (*Linepithema humile*). The central focus of
the Postdoctoral Scholar&rsquo;s research will be the chemical communication in
social insects, including the production and perception of pheromones that
Argentine ants use to regulate fundamental behaviors, including foraging,
queen production, and colonymate recognition.


BASIC QUALIFICATIONS

Candidates must have completed all degree requirements except the
dissertation or be enrolled in an accredited PhD or equivalent degree
program in biology, chemistry, or related field at the time of application.

ADDITIONAL QUALIFICATIONS
Candidates must have a PhD, MD, or equivalent degree in biology, chemistry,
or related field by appointment start date.

PREFERRED QUALIFICATIONS
Demonstrated expertise in chemical analysis using gas chromatography-mass
spectrometry (GC-MS). Candidates fluent in chemical, genetic, and/or
biochemical analysis of insect pheromones or experienced studying the
mechanisms of chemoreception are desirable. Preferred candidates will
demonstrate excellent communication skills and the ability to work both
independently and as a member of a small team. Applicants must have fewer
than five years of prior post-doctoral experience.

APPOINTMENT
The targeted start date for this position is March 1, 2016. The initial
appointment is for one year, with renewal based on performance and funding.
This is a full-time appointment.

SALARY AND BENEFITS
The salary range is between $42,840 - $50,112 commensurate with
qualifications and experience and based on UC Berkeley salary scales.
Generous benefits are included (<a href="http://bit.ly/1gm3hSu">http://bit.ly/1gm3hSu</a>)

TO APPLY
Visit:  <a href="http://bit.ly/1ZKkM2M">http://bit.ly/1ZKkM2M</a>
Interested individuals should include a 1-2 page cover letter describing
their research experience, 1-3 relevant publications, a current CV, and the
names and contact information of three references. Letters of reference may
be requested for finalists. It is optional to include a statement
addressing past and/or potential contributions to diversity through
research, teaching, and/or service.

This position will remain open until filled.

Questions regarding this recruitment can be directed to Professor Neil
Tsutsui (ntsutsui@berkeley.edu).

All letters will be treated as confidential per University of California
policy and California state law. Please refer potential referees, including
when letters are provided via a third party (i.e. dossier service or career
center) to the UC Berkeley Statement of Confidentiality (
<a href="http://bit.ly/1wTEkon">http://bit.ly/1wTEkon</a>) prior to submitting their letters.

The University of California is an Equal Opportunity/Affirmative Action
Employer. All qualified applicants will receive consideration for
employment without regard to race, color, religion, sex, sexual
orientation, gender identity, national origin, disability, age or protected
veteran status. For the complete University of California nondiscrimination
and affirmative action policy see:
<a href="http://bit.ly/1tbbcKE">http://bit.ly/1tbbcKE</a>

The Department is interested in candidates who will contribute to diversity
and equal opportunity in higher education through their research or
teaching.

The University of California, Berkeley has an excellent benefits package as
well as a number of policies and programs in place to support employees as
they balance work and family.


JPF00958 

The Department of Environmental Science, Policy, and Management at UC
Berkeley is currently seeking two Postdoctoral Scholars in *Genomics and
Functional Genetics* in the laboratory of Dr. Neil Tsutsui. We are seeking
two Postdoctoral Scholars to study the genetic basis of ant social
behaviors. Previous work has focused on the genetics, behavior, and
chemical ecology of the invasive Argentine ant (Linepithema humile). The
central focus of the Postdoctoral Scholars research will be the genetic
basis of chemical communication, including the production and perception of
pheromones that Argentine ants use to regulate fundamental behaviors,
including foraging, queen production, and colonymate recognition.

BASIC QUALIFICATIONS
Candidates must have completed all degree requirements except the
dissertation or be enrolled in an accredited PhD or equivalent degree
program in biology, genetics, or related field at the time of application.

ADDITIONAL QUALIFICATIONS
Candidates must have a PhD, MD, or equivalent degree in biology, genetics,
or related field by appointment start date.

PREFERRED QUALIFICATIONS
Demonstrated expertise in one or more of the following areas: production
and analysis of whole- genome datasets, RNAseq, RNAi-mediated gene
silencing, and bioinformatic analysis of large genomic datasets. Candidates
fluent in chemical ecology, insect biology, population genetics, and/or
behavioral ecology are desirable. Preferred candidates will demonstrate
excellent communication skills and the ability to work both independently
and as a member of a small team. Applicants must have fewer than five years
of prior post-doctoral experience.

APPOINTMENT
The targeted start date for this position is March 1, 2016. The initial
appointment is for one year, with renewal based on performance and funding.
This s a full-time appointment.

SALARY AND BENEFITS
The salary range is between $42,840 - $50,112 commensurate with
qualifications and experience and based on UC Berkeley salary scales.
Generous benefits are included (<a href="http://bit.ly/1gm3hSu">http://bit.ly/1gm3hSu</a>)

TO APPLY
Visit: <a href="http://bit.ly/1PruLud">http://bit.ly/1PruLud</a>
Interested individuals should include a 1-2 page cover letter describing
their research experience, 1-3 relevant publications, a current CV, and the
names and contact information of three references. Letters of reference may
be requested for finalists. It is optional to include a statement
addressing past and/or potential contributions to diversity through
research, teaching, and/or service.

This position will remain open until filled.

Questions regarding this recruitment can be directed to Professor Neil
Tsutsui (ntsutsui@berkeley.edu).

All letters will be treated as confidential per University of California
policy and California state law. Please refer potential referees, including
when letters are provided via a third party (i.e. dossier service or career
center) to the UC Berkeley Statement of Confidentiality (
<a href="http://bit.ly/1wTEkon">http://bit.ly/1wTEkon</a>) prior to submitting their letters.

The University of California is an Equal Opportunity/Affirmative Action
Employer. All qualified applicants will receive consideration for
employment without regard to race, color, religion, sex, sexual
orientation, gender identity, national origin, disability, age or protected
veteran status. For the complete University of California nondiscrimination
and affirmative action policy see:
<a href="http://bit.ly/1tbbcKE">http://bit.ly/1tbbcKE</a>

The Department is interested in candidates who will contribute to diversity
and equal opportunity in higher education through their research or
teaching.

The University of California, Berkeley has an excellent benefits package as
well as a number of policies and programs in place to support employees as
they balance work and family.

via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>January 22, 2016</h3>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/i1?rss=1">Cover</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/i2?rss=1">Associate Editors</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/i3?rss=1">SMBE Editors and Council</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/i4?rss=1">Subscription Page</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/305?rss=1">Gene Acquisitions from Bacteria at the Origins of Major Archaeal Clades Are Vastly Overestimated</a></div>
  <div class="description"><p>In a recent article, Nelson-Sathi et al. (NS) report that the origins of major archaeal lineages (MAL) correspond to massive group-specific gene acquisitions via HGT from bacteria (Nelson-Sathi et al. 2015. Origins of major archaeal clades correspond to gene acquisitions from bacteria. Nature 517(7532):77-80.). If correct, this would have fundamental implications for the process of diversification in microbes. However, a reexamination of these data and results shows that the methodology used by NS systematically inflates the number of genes acquired at the root of each MAL, and incorrectly assumes bacterial origins for these genes. A reanalysis of their data with appropriate phylogenetic models accounting for the dynamics of gene gain and loss between lineages supports the continuous acquisition of genes over long periods in the evolution of Archaea.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/311?rss=1">Cyclostomes Lack Clustered Protocadherins</a></div>
  <div class="description"><p>The brain, comprising billions of neurons and intricate neural networks, is arguably the most complex organ in vertebrates. The diversity of individual neurons is fundamental to the neuronal network complexity and the overall function of the vertebrate brain. In jawed vertebrates, clustered protocadherins provide the molecular basis for this neuronal diversity, through stochastic and combinatorial expression of their various isoforms in individual neurons. Based on analyses of transcriptomes from the Japanese lamprey brain and sea lamprey embryos, genome assemblies of the two lampreys, and brain expressed sequence tags of the inshore hagfish, we show that extant jawless vertebrates (cyclostomes) lack the clustered protocadherins. Our findings indicate that the clustered protocadherins originated from a nonclustered protocadherin in the jawed vertebrate ancestor, after the two rounds of whole-genome duplication. In the absence of clustered protocadherins, cyclostomes might have evolved novel molecules or mechanisms for generating neuronal diversity which remains to be discovered.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/316?rss=1">Functional Analyses of Transcription Factor Binding Sites that Differ between Present-Day and Archaic Humans</a></div>
  <div class="description"><p>We analyze 25 previously identified transcription factor binding sites that carry DNA sequence changes that are present in all or nearly all present-day humans, yet occur in the ancestral state in Neandertals and Denisovans, the closest evolutionary relatives of humans. When the ancestral and derived forms of the transcription factor binding sites are tested using reporter constructs in 3 neuronal cell lines, the activity of 12 of the derived versions of transcription factor binding sites differ from the respective ancestral variants. This suggests that the majority of this class of evolutionary differences between modern humans and Neandertals may affect gene expression in at least some tissue or cell type.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/323?rss=1">Spectral Tuning of Killer Whale (Orcinus orca) Rhodopsin: Evidence for Positive Selection and Functional Adaptation in a Cetacean Visual Pigment</a></div>
  <div class="description"><p>Cetaceans have undergone a remarkable evolutionary transition that was accompanied by many sensory adaptations, including modification of the visual system for underwater environments. Recent sequencing of cetacean genomes has made it possible to begin exploring the molecular basis of these adaptations. In this study we use in vitro expression methods to experimentally characterize the first step of the visual transduction cascade, the light activation of rhodopsin, for the killer whale. To investigate the spectral effects of amino acid substitutions thought to correspond with absorbance shifts relative to terrestrial mammals, we used the orca gene as a background for the first site-directed mutagenesis experiments in a cetacean rhodopsin. The S292A mutation had the largest effect, and was responsible for the majority of the spectral difference between killer whale and bovine (terrestrial) rhodopsin. Using codon-based likelihood models, we also found significant evidence for positive selection in cetacean rhodopsin sequences, including on spectral tuning sites we experimentally mutated. We then investigated patterns of ecological divergence that may be correlated with rhodopsin functional variation by using a series of clade models that partitioned the data set according to phylogeny, habitat, and foraging depth zone. Only the model partitioning according to depth was significant. This suggests that foraging dives might be a selective regime influencing cetacean rhodopsin divergence, and our experimental results indicate that spectral tuning may be playing an adaptive role in this process. Our study demonstrates that combining computational and experimental methods is crucial for gaining insight into the selection pressures underlying molecular evolution.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:13</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/2/337?rss=1">Peeping at TOMs--Diverse Entry Gates to Mitochondria Provide Insights into the Evolution of Eukaryotes</a></div>
  <div class="description"><p>Mitochondria are essential for eukaryotic life and more than 95% of their proteins are imported as precursors from the cytosol. The targeting signals for this posttranslational import are conserved in all eukaryotes. However, this conservation does not hold true for the protein translocase of the mitochondrial outer membrane that serves as entry gate for essentially all precursor proteins. Only two of its subunits, Tom40 and Tom22, are conserved and thus likely were present in the last eukaryotic common ancestor. Tom7 is found in representatives of all supergroups except the Excavates. This suggests that it was added to the core of the translocase after the Excavates segregated from all other eukaryotes. A comparative analysis of the biochemically and functionally characterized outer membrane translocases of yeast, plants, and trypanosomes, which represent three eukaryotic supergroups, shows that the receptors that recognize the conserved import signals differ strongly between the different systems. They present a remarkable example of convergent evolution at the molecular level. The structural diversity of the functionally conserved import receptors therefore provides insight into the early evolutionary history of mitochondria.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="indexeef0.html?q=aggregator&amp;from=200">previous page</a></div><div>... <a href="index7b13.html?q=aggregator&amp;from=140">8</a> <a href="indexf37c.html?q=aggregator&amp;from=160">9</a> <a href="index52c1.html?q=aggregator&amp;from=180">10</a> <a href="indexeef0.html?q=aggregator&amp;from=200">11</a> <strong>12</strong> <a href="indexd118.html?q=aggregator&amp;from=240">13</a> <a href="index3f45.html?q=aggregator&amp;from=260">14</a> <a href="index4603.html?q=aggregator&amp;from=280">15</a> <a href="index8c8d.html?q=aggregator&amp;from=300">16</a> ...</div><div><a href="indexd118.html?q=aggregator&amp;from=240">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=220 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 21:41:52 GMT -->
</html>
