<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/14&from=200 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 21:44:50 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="index66e2.html?q=node/119">54(6) December 2005</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator/sources/14" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator">news aggregator</a>
<ul>
<li class="expanded"><a href="indexe5f5.html?q=aggregator/sources">sources</a>
<ul>
<li class="leaf"><a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></li>
<li class="leaf"><a href="index4421.html?q=aggregator/sources/8">CiteULike Phylogeny</a></li>
<li class="leaf"><a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></li>
<li class="leaf"><a href="index4dbd.html?q=aggregator/sources/14" class="active">EVOLDIR</a></li>
<li class="leaf"><a href="index8aad.html?q=aggregator/sources/9">Evolutionary Bioinformatics</a></li>
<li class="leaf"><a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></li>
<li class="leaf"><a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></li>
<li class="leaf"><a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></li>
<li class="leaf"><a href="index9fb6.html?q=aggregator/sources/22">Phyloseminar</a></li>
<li class="leaf"><a href="index69a2.html?q=aggregator/sources/10">Systematics Association</a></li>
<li class="leaf"><a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></li>
<li class="leaf"><a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></li>
<li class="leaf"><a href="indexe0bd.html?q=aggregator/sources/15">dechronization</a></li>
<li class="leaf"><a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></li>
<li class="leaf"><a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></li>
<li class="leaf"><a href="index1f4a.html?q=aggregator/sources/7">postgenomic.com</a></li>

</ul>
</li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 2 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a> &raquo; <a href="index5640.html?q=aggregator">news aggregator</a> &raquo; <a href="indexe5f5.html?q=aggregator/sources">sources</a></div><h2>EVOLDIR</h2>
<!-- begin content -->
<div id="aggregator"><div class="feed"><h3>URL</h3>
<div class="xml-icon"><a href="http://evoldir.tumblr.com/rss"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div><a href="http://evoldir.tumblr.com/">http://evoldir.tumblr.com/</a>
<h3>Last update</h3>
1 hour 40 min ago</div><h3>January 5, 2016</h3>
<div class="news-item">
 <div class="date">03:35</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136670954146">Postdoc: Yeast evolutionary genomics in University of Michigan</a></div>
  <div class="description"><p>&ndash;94eb2c06bef0c79f2b05288b036f
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: quoted-printable

A postdoctoral position is available immediately in the laboratory of
Jianzhi “George” Zhang at University of Michigan, Ann Arbor, Michigan. The
ideal candidate will use the budding yeast Saccharomyces cerevisiae and its
relatives as model organisms to study evolutionary processes. Potential topics
include but are not limited to (1) the fitness effects of various mutations
including gene duplication, (2) genic/genomic basis of reproductive
isolation, (3) evolution of dominance, (4) genetic mechanisms of heterosis,
(5) evolution of gene essentiality, and (6) evolution of gene expression
level and noise. The position requires a motivated individual with an
interest in evolutionary genetics and wet lab experience in molecular
genetics or genomics. Prior training in yeast genetics is a plus.

Applicants should email a short statement of research interests, CV,
and contact
information of three references to jianzhi@umich.edu. For further information
about the Zhang lab, see <a href="http://bit.ly/1MQtrt6.">http://bit.ly/1MQtrt6.</a>

&ndash;94eb2c06bef0c79f2b05288b036f
Content-Type: text/html; charset=UTF-8
Content-Transfer-Encoding: quoted-printable

</p>A postdoctoral position is available immediately in the laboratory of Jianzhi “George” Zhang at University of Michigan, Ann Arbor, Michigan. The ideal candidate will use the budding yeast Saccharomyces cerevisiae and its relatives as model organisms to study evolutionary processes. Potential topics include but are not limited to (1) the fitness effects of various mutations including gene duplication, (2) genic/genomic basis of reproductive isolation, (3) evolution of dominance, (4) genetic mechanisms of heterosis, (5) evolution of gene essentiality, and (6) evolution of gene expression level and noise. The position requires a motivated in
 dividual with an interest in evolutionary genetics and wet lab experience in molecular genetics or genomics. Prior training in yeast genetics is a plus.<br><br>Applicants should email a short statement of research interests, CV, and contact information of three references to <a href="mailto:jianzhi@umich.edu">jianzhi@umich.edu</a>. For further information about the Zhang lab, see <a href="http://bit.ly/1MQtrt6" rel="noreferrer" target="_blank">http://bit.ly/1MQtrt6</a>.<br>


&ndash;94eb2c06bef0c79f2b05288b036
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:20</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136670546046">Job: AuburnU.2.ViralPlantOmics</a></div>
  <div class="description"><p>Beloware two positions At Auburn University as part of an -Omics and
Informatics Cluster hire

Oneis virology/viromics in the Department of Biological Sciences and
the other is molecular ecology in a the department of Plant Pathology.

Deadlinesare soon.

1)Virology/Viromics
TheDepartment of Biological Sciences at Auburn University invites
applications for a tenure-track faculty position (9-month) at the
level of Assistant or Associate Professor beginning Fall of 2016. This
position is part of an Auburn University cluster hire in &ldquo;omics&rdquo; and
informatics and will strengthen our departmental research core area of
host-microbial interactions. The successful candidate is expected to
establish an extramurally funded, internationally recognized research
program. Instructional responsibilities include development of graduate
and/or undergraduate courses in virology including informatics and
related topics.

TheOmics and Informatics Cluster represents an interdisciplinary
research initiative at Auburn University which has made a significant
financial commitment to hiring multiple new faculty. The successful
candidate for this position will be expected to participate actively in
Auburn University&rsquo;s multidisciplinary Cluster Hires Initiative in the
Omics and Informatics cluster. New faculty with expertise in viromics
will complement the University&rsquo;s considerable existing expertise in
these areas to form the core of this innovative multidisciplinary
initiative. Participation in the Omics and Informatics cluster will be
an important component in faculty annual reviews. For more information
regarding the Cluster Hires please click on the following link:
<a href="http://bit.ly/1Re5pAj">http://bit.ly/1Re5pAj</a>

Applicantsshould submit a 1 page cover letter emphasizing specific
qualifications, a curriculum vitae, a description of research interests,
a statement of teaching philosophy/experience, and names and contact
information of at least 3 references.

Moreinformation can be found at: <a href="http://bit.ly/1waOwJI">http://bit.ly/1waOwJI</a>

MinimumQualifications
Applicantsmust have a Ph.D. in an appropriate discipline and experience
in virology and informatics. Excellent written and interpersonal
communications skills are required. Qualifications include postdoctoral
or professional experience, a strong record of publication and potential
for funding. The candidate selected for this position must be able to
meet eligibility requirements for work in the United States at the time
the appointment is scheduled to begin and continue working legally for
the proposed term of employment.

DesiredQualifications
Applicantsat the level of Associate Professor are expected to have current
funding. Applicants studying host-viral interactions using informatics
are encouraged to apply.

SpecialInstructions to Applicants
Reviewof applications will begin January 8th, 2016 and will continue
until the position is filled.

2)Plant Pathology - Molecular ecology

TheDepartment of Entomology and Plant Pathology at Auburn University and
Auburn University&rsquo;s Cluster Hiring Initiative (CHI)  Omics and Informatics
Cluster  is seeking applicants for the position of Assistant Professor
in MOLECULARECOLOGY of PLANT and SOIL SYSTEMS. The expected start date
is August 16, 2016.

Responsibilities:This is a 9 month tenure-track position with 75% research
and 25% instruction responsibilities. The successful candidate for this
position will be expected to participate actively in Auburn University
multidisciplinary Cluster Hires Initiative in the cluster of Omics
and Informatics. New faculty with expertise in Molecular Ecology of
Plant and Soil Systems will complement the University&rsquo;s considerable
existing expertise in these areas to form the core of this innovative
multidisciplinary initiative. Participation in the Omics and Informatics
will be an important component in faculty annual reviews. For more
information regarding the Cluster Hires please click on the following
link: <a href="http://bit.ly/1Re5pAj">http://bit.ly/1Re5pAj</a>

Thesuccessful candidate will also be expected to build and lead an
extramurally-funded research program focused on fundamental aspects of
Molecular Ecology of Plant and Soil Systems. The successful candidate
(Research/Teaching) selected for this position will lead a research
program based on understanding population dynamics of microbes
associated with crops of agricultural importance for Alabama and the
U.S. The candidate will use diverse bioinformatic approaches such as
genomics, metagenomics, population genomics and/or phylogenomics to
understand microbiome/plant interactions. Understanding the microbiome
(including pathogenic, beneficial and saprophytic microorganisms) is
important for epidemiological studies and to develop disease management
strategies to eliminate or reduce the impact of plant pathogens. The
incumbent will be expected to secure extramural funding to support
programs and summer salary, publish technical and refereed articles,
and collaborate effectively with departmental faculty and colleagues in
other disciplines. Instructional responsibilities will include teaching
undergraduate and graduate classes in the department curriculum. In
addition, the candidate will be expected to direct and advise graduate
students. Service to the department, college and university are expected.

Qualifications:An earned doctoral degree in Plant Pathology or a
related scientific discipline at the time of hire; evidence of research
productivity (publications, grants, etc.) commensurate with career level;
ability to develop an original research program on fundamental aspects
of plant-pathogen interactions using genomics and informatic approaches;
and evidence of effective teamwork and capability to teach undergraduate
and graduate level courses. In addition, candidate must possess excellent
organizational, written, and interpersonal communication skills. Candidate
must demonstrate subject matter knowledge and the ability to communicate
effectively. The successful candidate must meet eligibility requirements
for work in the United States at the time the appointment is scheduled
to begin and continue working legally for the term of employment.

Desiredqualifications include postdoctoral research experience.

Application:Applicants must apply for the position electronically
through: <a href="http://bit.ly/1Re5ns2.">http://bit.ly/1Re5ns2.</a> Complete
applications must include: 1) a letter of application that addresses the
candidate&rsquo;s research and teaching experiences, 2) a statement of future
research plans or interests, 3) teaching philosophy, 4) current vita,
and 5) copies of all transcripts. When prompted during the on-line
application process, please provide names, phone numbers and email
addresses of three professional references. Only complete application
materials will be considered. Active review of applications will begin
February 1, 2016, but the search will continue until the position is
filled. Questions about the position should be directed to:

Dr.Leonardo De La Fuente, Search Committee Chair
Departmentof Entomology and Plant Pathology
AuburnUniversity
Email:lzd0005@auburn.edu 

TheUniversity: Auburn University is a land-grant institution with
a student enrollment of more than 25,000. The University provides
instruction in approximately 70 academic departments and maintains
graduate research and public service programs of a broad scope. The
University is located in the city of Auburn in east-central Alabama near
the metropolitan areas of Birmingham, Alabama, and Atlanta, Georgia.

TheDepartment: There are 18 faculty and 12 support positions
in the department. Additional information can be found at
<a href="http://bit.ly/1O2u3j8">http://bit.ly/1O2u3j8</a>




KennethM. Halanych
SchnellerChair
AlumniProfessor
BiologicalSciences Department
LifeSciences Bld. 101
AuburnUniversity
Auburn,AL 36849

<a href="http://bit.ly/1Re5ns4">http://bit.ly/1Re5ns4</a>

Phone:(334)-844-3222
Ken'sFax:   (334)-844-2333
BiologyFax (334)-844-1645
e-mail:ken@auburn.edu



ken@auburn.edu


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:35</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136669321336">Conference: DurhamNC.EvolutionaryMedicineSociety.Jun24-27.Deadline</a></div>
  <div class="description">Jan 30 is the closing date for abstract submission and early registration
for the 2016 meeting of the International Society for Evolution, Medicine
and Public Health. <a href="http://evmed.org/">http://evmed.org</a>


All interested scientists, scholars, students and clinicians are welcome.



The 2016 annual meeting of the International Society for Evolution,
Medicine and Public Health (ISEMPH) will take place from June 22-25, 2016
in Durham, North Carolina, thanks to host Charles Nunn and the Triangle
Center for Evolutionary Medicine.



The Program Committee welcomes your abstracts for oral or poster
presentations on all topics in the field of evolution, medicine and public
health.



Plenary speakers include:

Andrea Graham (Princeton University),

Carl Zimmer (New York Times),

Helen Ball (Durham University, UK),

Joshua Schiffman (University of Utah),

Marion Koopmans (Erasmus University) and

Martin Blaser (New York University).



Full information about registration and abstract submission is at

http://:evolutionarymedicine.org/isemph2016.



We look forward to seeing you for a memorable and exciting meeting!



Randolph Nesse, President, ISEMPH  (nesse@asu.edu)

Sir Peter Gluckman, Vice-President, ISEMPH

Cynthia Beall, Treasurer, ISEMPH, Chair, Program Committee

Charles Nunn, Chair, Meeting Planning Committee



PROGRAM COMMITTEE


Chair: Cynthia Beall, Case Western Reserve University

Joe Alcock, University of New Mexico

Gillian Bentley, Durham University, UK

Michelle Blyth, Louisiana State University

Jon Laman, Rijksuniversiteit Groningen, Netherlands
Randolph Nesse, Arizona State University
Charles Nunn, Duke University
Frank Rhli, University of Zurich

Joshua Schiffman, University of Utah

Randolph Nesse 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:20</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136668891686">Other: Online educational module on human population genomics</a></div>
  <div class="description"><p>We have written an online educational module that examines human,
Neanderthal and chimpanzee genome sequences for chromosome 6p
with an eye to discovering:

(a) what the overall variability of the genome says about human
prehistory,

(b) what anomalous regions of the genome say about specific forces
(population admixture, natural selection) acting on genes in those
regions.

The module is intended for college undergraduates in biology and 
related fields (a reviewer thought it was also relevant to anthropology
students) and is free to use.  We&rsquo;d appreciate feedback from anyone
who uses it in a class.

It can be found at:

<a href="http://bit.ly/1INW2nT">http://bit.ly/1INW2nT</a>

Best wishes,
Mary Kuhner mkkuhner@uw.edu


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:35</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136667521096">Postdoc: UCapeTown.AvianCoevolution</a></div>
  <div class="description">Post-doctoral research opportunity at the Percy FitzPatrick Institute
of African Ornithology, Department of Biological Sciences, University
of Cape Town

Project: Sociable Weaver nests as a resource to animal communities and
the life history of the obligate associate African Pygmy Falcon

We invite applications for the above full-time research opportunity at the
FitzPatrick Institute, a world-renowned, national Centre of Excellence
(CoE) in ornithological research with a strong emphasis on postgraduate
studies. The tenure of the Fellowship is for 2 years, with possible
extension to a 3rd year considering satisfactory academic progress and
availability of funds.

The successful applicant will focus on the interactions in animal
communities associated with Sociable Weaver (Philetairus socius) nests,
with particular focus on the role of the closely associated African Pygmy
Falcon (Polihierax semitorquatus). The candidate will conduct research
with Dr Robert Thomson. Applicants must have strong analytical skills,
including a demonstrated expertise in programming with R. Supervision
experience of post-graduates and a good publication record for the career
stage are also desirable.

This project examines the importance of Sociable Weaver nests to
a desert animal community and the role of African Pygmy Falcons in
these communities. The exact research questions will be tailored to,
and developed with, the successful candidate. The research could focus
on the diet, behaviour, demography and physiology of the African Pygmy
Falcon, investigating how the evolution of the species life history
strategy is linked to its obligate association with Sociable Weaver nests
(weaver nests provide the only nesting substrate used by the falcons in
southern Africa). The research could also use the falcon-weaver system
as a co-evolutionary model to understand the interplay between climate
and biotic interactions in shaping the South African distribution of
these two species. Areas of mismatch (where weavers are present but not
falcons) could be identified and additional fieldwork would enable to
collect environmental data aimed to explain the absence of Pygmy falcons
in these areas. Ultimately, by integrating multi-species interactions
into an ecological niche modelling framework, this project will offer a
unique opportunity to predict the future range of this Kalahari falcon
under different scenarios of climate change.

The researcher could also help supervise a recently initiated and
closely linked PhD project investigating the diversity of species
utilizing weaver nests and their interactions in an aridity gradient
across the weaver range. Conditions of the award of a Postdoctoral
Fellowship at UCT are: 
(i) postdoctoral fellowships are only available to individuals who
    have achieved a doctoral degree in a relevant subject within the
    past 5 years.
(ii) applicants may not previously have held full-time professional or
     academic employment.
(iii) the successful incumbent may, as part of their professional
      development, be required to participate in departmental
      activities including limited student supervision and duties
      incidental thereto.
(iv) the successful incumbent will be required to comply with
     UCTs approved policies, procedures and practises for the
     postdoctoral sector.

It is expected that the candidate will submit papers to high quality
peer-reviewed journals within the fellowship term.

The value of the Fellowship is R180 000 per annum. The Fellowship
is compliant with the SARS guidelines and rules and is exempt from
taxation. Adequate project running costs are available.

The successful incumbent should be available to commence with the
Fellowship in the first half of 2016, although flexibility might be
required if a non-South African resident is successful.

To apply, please send a CV (including your academic record,
publication list & the names and contact details of two academic
referees) plus a short motivation letter to Hilary Buchanan at
hilary.buchanan@uct.ac.za. Informal enquires can be directed to Robert
Thomson: robert.thomson@uct.ac.za.

For more information on the FitzPatrick Institute visit
<a href="http://bit.ly/1MQmV5z">http://bit.ly/1MQmV5z</a>

Closing date: 15 January 2016 (possible interviews to be held in January)
UCT is committed to the pursuit of excellence, diversity and
redress. UCT reserves the right to disqualify ineligible, incomplete
and/or inappropriate applications, and reserves the right to change the
conditions of award or to make no awards at all.

UNIVERSITY OF CAPE TOWN


Robert Thomson 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:20</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136667006441">Postdoc: IowaStateU.EvolBioinformaticsDosageCompensation</a></div>
  <div class="description">Bioinformatics PostDoctoral Position in Turtle Dosage Compensation

A postdoctoral position is available to work in the laboratory of
Dr. Nicole Valenzuela at Iowa State University on an NSF-funded project
to study the evolution of dosage compensation using turtles with
independently evolved XX/XY and ZZ/ZW sex chromosome systems.

The project combines bioinformatics, transcriptomics, molecular
cytogenetics, and phylogenetic analyses.

*Project description:
Species with sex chromosomes face the challenge posed by differential
gene dosage between XX and XY individuals (or ZZ and ZW) that may lead
to disease, suboptimal phenotypes or death. Dosage compensation is a
mechanism to equalize the activity of X- or Z-linked genes between sex
chromosomes and autosomes, and between males and females. As part of this
NSF-funded project we are leveraging the multiple independently-evolved
sex chromosome systems found in turtles to study the evolution of dosage
compensation.

The successful candidate will help uncover the full diversity of dosage
compensation mechanisms and their molecular underpinnings by contrasting
turtles with alternative sex-determining mechanisms, leveraging and
augmenting turtle genomic resources.

*Position Requirements:
The ideal candidate will have a PhD degree and strong background in
bioinformatics, including genome assembly and transcriptome analyses.

Other qualifications such as evolutionary biology background or experience
with molecular biology and molecular cytogenetics are a plus but not
required.

*Position details:
Funding is available for 2-3 years with annual renewal contingent upon
performance. The position is available at any time starting February 2016.

*Application deadline:
1 February 2016, or until position is filled.

*How to apply:
For inquiries or to apply please email the following to Dr. Nicole
Valenzuela at nvalenzu@iastate.edu:
1.           Cover letter describing research interests and experience
2.           Current CV
3.           Copies of up to two relevant publications if available
4.           Applicants should arrange to have three letters of
             recommendation be sent directly to nvalenzu@iastate.edu.

Iowa State University does not discriminate on the basis of race, color,
age, religion, national origin, sexual orientation, gender identity,
genetic information, sex, marital status, disability, or status as a
U.S. veteran.

Dr. Nicole Valenzuela
Associate Professor
nvalenzu@iastate.edu
Dept. of Ecology, Evolution, and Organismal Biology
Iowa State University
Office: 239 Bessey Hall
251 Bessey Hall, Ames IA 50011
URL: <a href="http://bit.ly/17Wyl4I">http://bit.ly/17Wyl4I</a>

&ldquo;Valenzuela, M. N [EEOBS]&rdquo; 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:05</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136666489571">Conference: Roscoff_France.50yrsMolecularHistory.May9-13.Reminder</a></div>
  <div class="description">There is only one month left to apply to the Jacques Monod conference on
molecular evolution in May 2016. We particularly encourage the application
of PhD students. Note that the inscription fees listed below include board
and lodging.  Selected  applicants will be expected to present a poster.


  Molecules as documents of evolutionary history: 50 years after - Roscoff,
  Brittany, France.

  May 9-13 2016



It is with the popularization of molecular tools and the disclosure
of genome sequences that evolutionary models have become decisive in
biology, in part because this information is simply so abundant and
so complex that only a comparative method could reveal its meaning.
In 2016, it will be 50 years that Emile Zuckerkandl and Linus Pauling
have laid the foundation of molecular phylogeny, and molecular evolution
in general, in their seminal article entitled &ldquo;Molecules as documents
of evolutionary history&rdquo;. The fundamental realization of this article
was that DNA does not only encode the elementary functional elements of
an organisms, it also harbours abundant information on the evolutionary
history and life history traits of its ancestors.

The modelling of the evolutionary processes that generated extent
genomes should reveal together the patterns of the diversification
of life, the processes by which evolution proceeds at the molecular
level, and how both are affected by external conditions. Our challenge
today, as evolutionary biologists, is to bridge the gap between species
ecology, life history traits, population dynamics, species phylogeny,
gene phylogeny, genome structure, gene network and molecular mechanisms.

The recent years have seen tremendous developments in this direction,
which we will attempt to review in this conference.


Invited speakers 
(provisional titles)

 

ABBY Sophie (Paris, France)
Evolution playing a molecular building set: origins and distribution of
bacterial secretion systems

ACHTMAN Mark (Warwick, United Kingdom)
Ages of bacterial pathogenes

ALM Eric (Cambridge, USA)
Driving forces of microbiome evolution

DURET Laurent (Lyon, France)
Biased gene conversion as a 4rth evolutionary force

FARIA Nuno (Oxford, United Kingdom)
Viral evolution

GALTIER Nicolas (Montpellier, France)
Population genomics of non model animals: genetic diversity, adaptive
rate and effective population size

GILBERT Thomas (Copenhaguen, Denmark)
Ancient DNA, ancient genomes, and the hype of de-extinction

GOGARTEN Peter (Storrs, USA)
Horizontal gene transfer: the pan-genome as shared genetic resource of
a lineage

GRAUR Dan (Houston, USA)
The vocabulary of molecular evolution and the transgressions of functional
genomics: a rose by any other name would be misleading

GRIBALDO Simonetta (Paris, France)
Phylogenomics and the tree of life

HEATH Tracy (Ames, USA)
Molecular, morphological, and macroevolutionary models for dating species
divergence times

KATZ Laura (Northampton, USA)
The eukaryotic phylogeny

KOSIOL Carolin (Vienna, Austria)
Polymorphism-aware phylogenetic models for species trees

LAGERGREN Jens (Stockholm, Sweden)
Generative models of gene family evolution

LỐPEZ-GARCIA Purificación (Orsay, France)
Diversity of microbial life

McLYSAGHT  Aoife (Dublin, Ireland)
Whole genome duplication

MORAN Nancy (Austin, USA)
Symbiosis in evolution

PAGEL Mark (Reading, United Kingdom)
Beyond molecular data: information and evolution in the cultural realm

PHILIPPE Hervé (Moulis, France)
Models of sequence evolution

SANMARTIN Isabel (Madrid, Spain)
Spatiotemporal evolution of lineages and biotas using Bayesian approaches

SEMON Marie (Lyon, France)
Insights into making different types of the same organ using developmental
transcriptomes as molecular microscopes

STADLER Tanja (Zürich, Switzerland)
Modelling and inference of species diversification

SZÖLLÖSI Gergely (Budapest, Hungary)
Genome scale reconstruction of phylogenies

TELFORD Maximilian J. (London, United Kingdom)
The new animal phylogeny

WARNOW Tandy (Urbana, USA)
New coalescent-based species tree estimation methods

 
Deadline for application: February 1rst, 2016

 
Registration fee (including board and lodging)

430 € for PhD students 
625 € for other participants

 
Application for registration 
The total number of participants is limited to 115 and all participants
are expected to attend for the whole duration of the conference. Selection
is made on the basis of the affinity of potential participants with the
topics of the conference. Scientists and PhD Students interested in the
meeting should send:

- their curriculum vitae 
- the list of their main publications for the 3 last years 
- the abstract of their presentation

to the Chairperson of the conference (vincent.daubin@univ-lyon1.fr
) before the deadline. After it,
the organizers will select the participants. Except in some particular
cases approved by the Chairperson, it is recommended that all selected
participants present their work during the conference, either in
poster form or by a brief in- session talk. The organizers choose the
form in which the presentations are made. No payment will be sent with
application. Information on how and when to pay will be mailed in due
time to those selected.

 

  -
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:05</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136666489326">Conference: Volterra_Italy.SystemsBiol.Jul8-14</a></div>
  <div class="description"><p>* Our sincere apologies if you receive multiple copies of this announcement *

The 3rd International Synthetic & Systems Biology Summer School is a  
great opportunity to exchange ideas and information with colleagues  
and peers from around the world and discover the latest trends and new  
exciting results in Synthetic and Systems Biology.

International Synthetic and Systems Biology Summer School - SSBSS 2016,
8-14 July 2016, Volterra (Pisa) - Tuscany, Italy

*  Application:March 31, 2016
*  Oral Talk/Poster Submission:March 31, 2016
<a href="http://bit.ly/1OJeB8P">http://bit.ly/1OJeB8P</a>


ssbss.school@gmail.com
<a href="http://bit.ly/22ImA0y">http://bit.ly/22ImA0y</a>

Previous Editions:
SSBSS 2015 <a href="http://bit.ly/ZWPdub">http://bit.ly/ZWPdub</a>
SSBSS 2014 <a href="http://bit.ly/LmZmtw">http://bit.ly/LmZmtw</a>



Other Events in Tuscany:
2nd International Workshop on Machine learning, Optimization & big  
Data - MOD 2016
August 26-29, 2016 - Volterra (Pisa), Tuscany, Italy
<a href="http://bit.ly/1OJezOp">http://bit.ly/1OJezOp</a>

Call for Papers:February28, 2016
<a href="http://bit.ly/22ImBBu">http://bit.ly/22ImBBu</a>
<a href="http://bit.ly/1OJezOt">http://bit.ly/1OJezOt</a>


cfp.ssbss@dmi.unict.it


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:05</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136666488761">Other: evolution honors seminar text</a></div>
  <div class="description">Hello list!

I was hoping that some of you might have a suggestion for me. I am
fortunate enough to be teaching a one credit honors seminar this coming
semester tied to my intro level evolution, diversity, and physiology
course. During the second half of the semester we will be reading Neil
Shubin&rsquo;s Your Inner Fish to connect the physiology we are covering in
lecture to evolutionary principles.

During the first half of the semester I would like to dive a little
into evolutionary theory. I&rsquo;m looking for a &ldquo;popular&rdquo; text that is
approachable to a group of eighteen and nineteen year olds. Suggestions
are greatly appreciated and can be sent me to off list.

I thought of reading Darwin but I want to avoid the issues of
&ldquo;translating English into English&rdquo; that I can already foresee. I like
Dawkins, but given the population I want to avoid anyone feeling
patronized and turned off.

Thanks for any help.

Sean

&ldquo;Beckmann, Sean&rdquo; 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:19</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136664623136">Graduate Position: IndianaStateU.SparrowGenomics</a></div>
  <div class="description"><p>PhD opportunities in Ecological Genomics

We are looking for motivated students interested in behavioral,
evolutionary, and ecological genomics to join our laboratory at Indiana
State University (ISU).  These graduate positions are part of an exciting
new interdisciplinary initiative at ISU, The Center for Genomic Advocacy
(TCGA), which is focused on the application of genomic technology to
the betterment of society.  TCGA is developing a state-of-the-art next
generation sequencing facility, which will be used to provide hands-on
experiences for students as well as strong infrastructure for modern
genomic studies.

Graduate research will be expected to combine traditional behavioral
ecology studies with next-generation sequencing technology to examine the
evolution of polymorphism in the white-throated sparrow.  Students will
conduct extensive laboratory work as well as participate in field based
data collection during the breeding season.  This species exhibits a clear
link between phenotype and genotype, making it an ideal system in which
to pinpoint the determinants of complex sexual and parental behavior.
We have amassed 27 years of detailed data on this species making it
possible to identify the genetic, epigenetic, and environmental bases
of behavior.  Morphs of the white-throated sparrow provide a unique
opportunity to study intraspecific genomic differences, which have
resulted from two separate, yet linked evolutionary trajectories -
such results can transform our understanding of the evolution of genomes.

To apply, please send a letter of intent, curriculum vitae, and the
names and email addresses of 3 references.  Applications will be accepted
until January 30, 2015.

For more information about the positions, please feel free to contact
Dr. Elaina M. Tuttle (Elaina.Tuttle@indstate.edu) or Dr. Rusty A. Gonser
(Rusty.Gonser@indstate.edu), at The Center for Genomic Advocacy (TCGA),
Indiana State University, Terre Haute, IN 47809.

For more information about the lab, TCGA, the department, and the
university, see:

<a href="http://bit.ly/1O29Dqw">http://bit.ly/1O29Dqw</a>
<a href="http://bit.ly/1OwrWWl">http://bit.ly/1OwrWWl</a>
<a href="http://bit.ly/1O29Dqy">http://bit.ly/1O29Dqy</a>
<a href="http://bit.ly/1OwrUxS">http://bit.ly/1OwrUxS</a>





Dr. Elaina M. Tuttle
Interim Associate Dean of Graduate Programs, College of Graduate &
Professional Studies (CGPS)
Chairperson, Institutional Animal Care & Use Committee (IACUC)
Professor, Department of Biology & The Center for Genomic Advocacy (TCGA)
Indiana State University
Terre Haute, IN 47809

Phone:  812-237-4363 (CGPS)
812-237-2838 (Biology)
Fax:  812-237-8060 (CGPS)
812-237-3378 (Biology)
Email:  Elaina.Tuttle@indstate.edu
Elaina.Tuttle@gmail.com
Web: <a href="http://bit.ly/1O29Dqw">http://bit.ly/1O29Dqw</a> 
<a href="http://bit.ly/1O29Dqz">http://bit.ly/1O29Dqz</a> 
<a href="http://bit.ly/1OwrUxT">http://bit.ly/1OwrUxT</a> 
<a href="http://bit.ly/1O29G5R">http://bit.ly/1O29G5R</a>

&ldquo;Nothing in biology makes sense except in the light of evolution.&rdquo;
Theodosius Dobzhansky



Elaina.Tuttle@indstate.edu


via Gmail</p></div>
 </div>
</div>
<h3>January 3, 2016</h3>
<div class="news-item">
 <div class="date">23:57</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136592424956">Graduate Position:UConnecticut.PlantComputationalGenomics</a></div>
  <div class="description"><p>&ndash;56895dad_1078e674_2d8
Content-Type: text/plain; charset=&ldquo;utf-8&rdquo;
Content-Transfer-Encoding: quoted-printable
Content-Disposition: inline

The Plant Computational Genomics lab in the Department of Ecology and Evolutionary Biology at the University of Connecticut seeks motivated PhD and MS students to join the lab in the Summer/Fall of 2016. Our research focuses on the computational analysis of genomic and transcriptomic data generated by next-generation sequencing platforms from non-model forest tree species. We implement this through analysis related to gene finding, gene expression, transcriptome assembly, and conserved element identification, through machine learning and computational statistics. We use these methods to address questions related to genome biology and population genomics. In addition, we develop web-based applications that integrate BIG data across domains to facilitate the forest geneticist or ecologist’s ability to analyze, share, and visualize their data (<a href="http://bit.ly/1mudxQ8">http://bit.ly/1mudxQ8</a>). Such integration requires the implementation of semantic technologies and ontologies to connect genotype, phen
 otype, and environmental resources.

Research Topics:
Potential research topics in our group include 1) development of visualization tools to support genome-wide association studies in forest trees; 2) application of genomic and transcriptomic techniques to evaluate the impact of climate change on tree populations; 3) development of software solutions to improve assembly and characterization of non-model plant transcriptomes; 4) interrogation of natural genetic variation across populations in large, complex conifer genomes; 5) and your ideas here!

To Apply:
Financial support for M.S. and Ph.D. students is available through research assistantships, teaching assistantships, and university fellowships. To learn more about our research, please visit: <a href="http://bit.ly/1PFAut2.">http://bit.ly/1PFAut2.</a>  Excellent written and oral communication, as well as strong quantitative skills, are required. Backgrounds in genetics/genomics, evolutionary biology, bioinformatics, and computer science are desired.  Interested candidates should send an email describing their research interests along with a CV, unofficial undergraduate/graduate transcripts, and GRE scores to Jill Wegrzyn (jill.wegrzyn@uconn.edu). Qualified candidates will be contacted directly for Skype interviews following review.

About UConn:
The University of Connecticut (UConn) has been one of the nation’s leading public institutions since its founding in 1881. Located in Storrs, UConn’s main campus is situated in the picturesque rolling forests and fields quintessential of New England, yet is only 30 minutes from Hartford, and has close connections to Providence, Boston and New York. The Department of Ecology and Evolutionary Biology consists of over 30 faculty and 60 graduate students with research spanning nearly all major groups of organisms. The Department maintains close ties with the Departments of Physiology and Neurobiology, Molecular and Cell Biology, Marine Sciences, and Natural Resources Management and Engineering, as well as the Center for Environmental Sciences and Engineering and the Institute for Systems Genomics, which together comprise one of the largest groups of biologists in the Northeast.



&ndash;56895dad_1078e674_2d8
Content-Type: text/html; charset=&ldquo;utf-8&rdquo;
Content-Transfer-Encoding: quoted-printable
Content-Disposition: inline

</p>body{font-family:Helvetica,Arial;font-size:13px}The Plant Computational Genomics lab in the Department of Ecology and Evolutionary Biology at the University of Connecticut seeks motivated PhD and MS students to join the lab in the Summer/Fall of 2016. Our research focuses on the computational analysis of genomic and transcriptomic data generated by next-generation sequencing platforms from non-model forest tree species. We implement this through analysis related to gene finding, gene expression, transcriptome assembly, and conserved element identification, through machine learning and computational statistics. We use these methods to address questions related to genome biology and population genomics. In addition, we develop web-based applications that integrate BIG data across domains to facilitate the forest geneticist or ecologist’s
  ability to analyze, share, and visualize their data (<a href="http://bit.ly/1mudxQ8">http://bit.ly/1mudxQ8</a>). Such integration requires the implementation of semantic technologies and ontologies to connect genotype, phenotype, and environmental resources.Research Topics:Potential research topics in our group include 1) development of visualization tools to support genome-wide association studies in forest trees; 2) application of genomic and transcriptomic techniques to evaluate the impact of climate change on tree populations; 3) development of software solutions to improve assembly and characterization of non-model plant transcriptomes; 4) interrogation of natural genetic variation across populations in large, complex conifer genomes; 5) and your ideas here!To Apply:Financial support for M.S. and Ph.D. students is available through research assistantships, teaching assistantships, and university fellowships. To learn more about ou
 r research, please visit: <a href="http://bit.ly/1PFAut2.">http://bit.ly/1PFAut2.</a>  Excellent written and oral communication, as well as strong quantitative skills, are required. Backgrounds in genetics/genomics, evolutionary biology, bioinformatics, and computer science are desired.  Interested candidates should send an email describing their research interests along with a CV, unofficial undergraduate/graduate transcripts, and GRE scores to Jill Wegrzyn (jill.wegrzyn@uconn.edu). Qualified candidates will be contacted directly for Skype interviews following review.About UConn:The University of Connecticut (UConn) has been one of the nation’s leading public institutions since its founding in 1881. Located in Storrs, UConn’s main campus is situated in the picturesque rolling forests and fields quintessential of New England, yet is only 30 minutes from Hartford, and has close connections to Providence, Boston and New York. The Department of Ecolog
 y and Evolutionary Biology consists of over 30 faculty and 60 graduate students with research spanning nearly all major groups of organisms. The Department maintains close ties with the Departments of Physiology and Neurobiology, Molecular and Cell Biology, Marine Sciences, and Natural Resources Management and Engineering, as well as the Center for Environmental Sciences and Engineering and the Institute for Systems Genomics, which together comprise one of the largest groups of biologists in the Northeast.
&ndash;56895dad_1078e674_2d
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:57</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136592424551">Graduate Position: UMississippi.AquaticEvolution</a></div>
  <div class="description">Graduate Student Research Assistantships in Aquatic Ecology/Evolution

The Resetarits Lab (<a href="http://bit.ly/1PFAusX">http://bit.ly/1PFAusX</a>) at
The University of Mississippi currently has openings for highly
qualified PhD. students as Doherty Research Assistants in Freshwater
Biology.  These are newly established, competitive, 12 month Research
Assistantships in the Department of Biology, providing 5 years of
support.  Current stipend is 25k/year, with health insurance, and full
tuition remission.  Recipients are expected to design and implement
independent dissertation projects (empirical and/or theoretical) at the
interface between community, behavioral, and evolutionary ecology in
freshwater systems, or at the freshwater/terrestrial interface, as well
as participate in ongoing projects.  Current studies in the Lab address
a variety of questions, including the role of habitat selection in the
assembly of communities and the dynamics of metacommunities, the role
of diversity and species interactions in community assembly/ecosystem
function, life history evolution in amphibians and insects, and
biochemical, behavioral, evolution and community dynamics of chemical
camouflage.   Study organisms include amphibians, aquatic insects,
other aquatic invertebrates, and fish, while focal habitats range from
small ephemeral, coastal plain ponds to headwater mountain streams.
Funding for past and ongoing research has come primarily from the
National Science Foundation, along with EPA/NASA.

Recent/representative papers:

Resetarits, W. J. Jr. & A. Silberbush.  2015.  Local contagion and
regional compression: habitat selection drives spatially explicit,
multi-scale dynamics of colonization in experimental metacommunities.
Ecology Letters  in press  doi: 10.1111/ele.12553

Resetarits, W. J. Jr. and C. A. Binckley.  2013. Patch quality and
context, but not patch number, drive multi-scale colonization dynamics
in experimental aquatic landscapes.  Oecologia  173:933-946. pdf

*Resetarits, W. J. Jr. and C. A. Binckley.  2013.  Is the Pirate really
a Ghost?  Evidence for generalized chemical camouflage in an aquatic
predator, Pirate Perch (Aphredoderus sayanus).  American Naturalist
181:690-699.  *Featured in a variety of media, including: New
Scientist, Nature Research Briefs, Inside Science, Scientific American,
and Wikinews. pdf

Resetarits, W. J., Jr. and C. A. Binckley. 2009. Spatial contagion of
predation risk affects colonization rate and community structure in
experimental landscapes. Ecology 90:869-876. pdf

Resetarits, W. J., Jr. and D. R. Chalcraft. 2007. Functional diversity
within a morphologically conservative genus of predators: implications
for functional equivalence and redundancy in ecological communities.
Functional Ecology 21:793-804.*Selected for 100 Influential Papers
Published in 100 Years of British Ecological Society Journals. pdf

Resetarits, W. J., Jr. 2005. Habitat selection links local and regional
scales in aquatic systems. Ecology Letters 8:480-486. pdf

The Resetarits Lab is based in the Department of Biology and housed
primarily at the University of Mississippi Field Station (UMFS),
approximately 11 miles from the main campus in Oxford.  The Lab has
outstanding space and facilities and access to over 200 experimental
ponds and wetlands at the UMFS (check us out on Google maps

<a href="http://bit.ly/1mudwMh">http://bit.ly/1mudwMh</a>), hundreds of
mesocosms of various sizes for experimental work, and dedicated field
vehicles.  The Department of Biology at The University of Mississippi
has an organismal focus, including a dynamic and growing group of
ecologists and evolutionary biologists (<a href="http://bit.ly/1PFAusZ">http://bit.ly/1PFAusZ</a>).
The University of Mississippi is dedicated to fostering diversity at
all levels within the University community (<a href="http://bit.ly/1mudwMj">http://bit.ly/1mudwMj</a>
).  Oxford is a small, dynamic, progressive community with excellent
cultural amenities, great food, a fun atmosphere, and a reasonable cost
of living.  The University of Mississippi campus is one of the most
beautiful in the country.

For more information contact me at the address below.  To begin the
application procedure, please attach a letter of interest, unofficial
transcripts and GRE scores, and resume (including contact information
for 3 references) to Dr. William Resetarits (wresetar@olemiss.edu).

William J. Resetarits, Jr.

Professor of Biology and
Henry L. and Grace Doherty Chair in Freshwater Research
Department of Biology
The University of Mississippi
P.O. Box 1848
University, MS 38677-1848
Phone: (662) 915-5804
Fax: (662) 915-6554
<a href="http://bit.ly/1PFAut0">http://bit.ly/1PFAut0</a>
Experiments are only experience carefully planned in advance.   R. A.
Fisher
You can&rsquo;t step twice in the same river.   Heraclitus

wresetar 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:11</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136516925886">Graduate position: ImperialCollege_London.StressFitnessConsequences</a></div>
  <div class="description">PhDStudentship advertised at Imperial College London, Silwood Park
campus, UK

On:Trans-generational fitness consequences of chronic stress induced by
global change

Supervisors:Dr Julia Schroeder (Imperial College London) and Prof Terry
Burke (University of Sheffield)

Applicationdeadline: 10 January 2016
.

Projectdescription:
Human-inducedglobal changes impose chronic environmental stress on ever
more natural populations. The physiological symptoms of chronic stress are
changes in hormone levels that can lead to stunted growth and increased
risk of infection. Furthermore, the physiological mechanisms invoked
by stressed are similar to those in ageing individuals. Recent studies
suggest that chronic stress can lead to faster senescence (measured in
telomere shortening rates), not only in affected individuals but also in
their offspring. What are the trans-generational fitness consequences of a
chronic stress response, and how is it adaptive? Stress could &ldquo;prime&rdquo;
offspring to enable them to endure stressful conditions in the future, by
a &ldquo;life-fast die-young&rdquo; trade-off. This projects tests the hypothesis
that chronic stress primes the next generation. To test this, we make
opportunistic use of an exceptionally well suited long-term &ldquo;natural
experiment&rdquo; in wild passerines and supplement statistical analyses on
existing long-term data with focused experiments on captive birds.

Theaim of this study is to quantify the fitness consequences of chronic
stress to gain a better understanding of the long-term fitness costs
and benefits of chronic stress in a wild population the consequences for
population viability. The student will utilise a unique long-term study
of a wild passerine population. The precise direction and emphasis of the
project will be determined by collaboration between the student and the
supervisors.  Nextto the traditional academic skills, the student will
also learn and improve key skills such as state-of-the-art statistical
modelling, data management, experimental design and field work. The
student can expect to receive exceptional training in state-of-the-art
analytical methodology, as well as benefitting from a well established
and rewarding field project.

ImperialCollege London is a world leading university; it has been
Ranked 3rd in Europe and 8th in the world by the Time Higher Education
Rankings 2015-16. The student will be based at the Silwood Park campus
 which has a large
graduate student community, with internationally renowned researchers.

Thefunding for this post is covered by an SSCP DTP-NERC studentship,
and covers home fees and a stipend for 3.5 years. Applicants,
who must be residents of the EU OR have been ordinarily
resident in the UK for 3 years prior to the start of the
studentship, must have an outstanding honours degree, a masters
degree, or equivalent, in a relevant subject, should complete an
online application, upload a supporting statement explaining why they
want to pursue this PhD (maximum 1 page), their CV, and 2 references, at:
<a href="http://bit.ly/1TtfTbX">http://bit.ly/1TtfTbX</a>
Anyenquiries about the project should be directed to:

DrJulia Schroeder julia.schroeder@imperial.ac.uk


julia.schroeder@imperial.ac.uk


via Gmail</div>
 </div>
</div>
<h3>January 2, 2016</h3>
<div class="news-item">
 <div class="date">23:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136516190581">Job:UniversityofCaliforniaMerced.ComputationalSciences</a></div>
  <div class="description"><p>University of California Merced is recruiting for four Assistant Professor Positions as part of a cluster hire in the area of Computational Sciences and Data Analytics. Topics of interest include, but are not limited to: statistical inference from comparative or population analysis of high-throughput omics data to address fundamental problems in molecular and cellular biology. Full details are available at <a href="http://bit.ly/1Tx5xs4">http://bit.ly/1Tx5xs4</a>

Emilia Huerta-Sanchez (ehuerta-sanchez@ucmerced.edu) 
Assistant Professor, School of Natural Sciences

via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:11</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136444119246">Graduate Position: PhD at U of Sheffield; evolutionary genomics of</a></div>
  <div class="description"><p>&ndash;001a1140389635f0d005284621a2
Content-Type: text/plain; charset=UTF-8

A NERC funded PhD (3.5 years) is available with Prof Jon Slate (University
of Sheffield, <a href="http://bit.ly/1VvPZG9">http://bit.ly/1VvPZG9</a>) and Dr Kanchon
Dasmahaptara (University of York, <a href="http://goo.gl/RFxuOI">http://goo.gl/RFxuOI</a>). The project is on
the evolutionary genomics of great tits (Parus major) but the candidate
will have considerable flexibility to develop the specific questions.
Details of the project can be found here: <a href="http://bit.ly/1OqoEE8.">http://bit.ly/1OqoEE8.</a>

We have typed approximately 2000 birds from >20 populations at a 500K SNP
chip. Approximately half of the samples come from an intensively studied
population at Wytham Woods near Oxford. The great tit genome has recently
been assembled and annotated (to a high quality), and we collaborate with
groups in Oxford (Ben Sheldon), Wageningen (Martien Groenen) and the
Netherlands Institute of Ecology (Marcel Visser).

The lab is interested in a broad set of evolutionary quantitative genetic
questions, and the project could focus on one or more of the following
topics: understanding the genetic architecture of quantitative traits;
identifying genes driving adaptive evolution; exploring the role of copy
number and structural variation in evolutionary processes; using genomics
tools to understand inbreeding and inbreeding depression; prediction of
phenotype from genomic data. The project does not involve lab or field
work, and is ideally suited to an evolutionary biologist that enjoys data
analysis or other scientist with strong quantitative skills.

The project is part of the ACCE (Adapting to the Challenges of a Changing
Environment) doctoral training program run by the Universities of
Sheffield, Liverpool and York (see
<a href="http://bit.ly/1VvPXOy">http://bit.ly/1VvPXOy</a>). Note that the
projects which are funded will be decided by open competition between the
best candidates put forward by each supervisor.

Unfortunately NERC studentships have quite restrictive conditions.
Applicants can be from the UK or EU, but non-UK applicants do not receive
any living allowance (effectively ruling them out). Applications MUST be
made through the University of Sheffield online system outlined on the ACCE
webpage (URL above). However, Jon Slate welcomes informal enquiries (
j.slate@sheffield.ac.uk). The closing date is Monday 11th January.

&ndash;001a1140389635f0d005284621a2
Content-Type: text/html; charset=UTF-8
Content-Transfer-Encoding: quoted-printable

</p>A NERC funded PhD (3.5 years) is available with Prof Jon Slate (University of Sheffield, <a href="http://bit.ly/1VvPZG9">http://bit.ly/1VvPZG9</a>) and Dr Kanchon Dasmahaptara (University of York, <a href="http://goo.gl/RFxuOI">http://goo.gl/RFxuOI</a>). The project is on the evolutionary genomics of great tits (Parus major) but the candidate will have considerable flexibility to develop the specific questions. Details of the project can be found here: <a href="http://bit.ly/1OqoEE8">http://bit.ly/1OqoEE8</a>. We have typed approximately 2000 birds from >20 populations at a 500K SNP chip. Approximately half of the samples come from an intensively studied population at Wytham Woods near Oxford. The great tit genome has recently been assembled and annotated (to a high quality), and we collaborate with groups in Oxford (Ben Sheldon), Wageningen (Martien Groenen) and the Netherlands Institute of Ecology (Marcel Visser). The lab is interested in a broad set of evolutionary quantitative genetic questions, and the project could focus on one or more of the following topics: understanding the genetic architecture of quantitative traits; identifying genes driving adaptive evolution; exploring the role of copy number and structural variation in evolutionary processes; using genomics tools to understand inbreeding and inbreeding depression; prediction of phenotype from genomic data. The project does not involve lab or field work, and is ideally suited to an evolutionary biologist that enjoys data analysis or other scientist with strong quantitative skills.The project is part of the ACCE (Adapting to the Challenges of a Changing Environment) doctoral training program run by the Universities of Sheffield, Liverpool and York (see <a href="http://bit.ly/1VvPXOy">http://bit.ly/1VvPXOy</a>). Note that the projects which are funded wil
 l be decided by open competition between the best candidates put forward by each supervisor.Unfortunately NERC studentships have quite restrictive conditions. Applicants can be from the UK or EU, but non-UK applicants do not receive any living allowance (effectively ruling them out). Applications MUST be made through the University of Sheffield online system outlined on the ACCE webpage (URL above). However, Jon Slate welcomes informal enquiries (<a href="mailto:j.slate@sheffield.ac.uk">j.slate@sheffield.ac.uk</a>). The closing date is Monday 11th January.

&ndash;001a1140389635f0d005284621a
via Gmail</div>
 </div>
</div>
<h3>January 1, 2016</h3>
<div class="news-item">
 <div class="date">23:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136443359051">Conference: Vairao_Portugal.Congenomics2016.May3-6</a></div>
  <div class="description"><p>Congenomics2016 - Conference on Conservation Genomics
<a href="http://bit.ly/1PClvln">http://bit.ly/1PClvln</a>
Vairao, Portugal
3-6 May 2016

Early registration fees (deadline 31 March): 250 EUR (standard), 100 EUR
(students)
Submit your abstract online (deadline 21 February)

We are pleased to announce the Congenomics2016 - Conference
on Conservation Genomics, the final meeting of the ConGenOmics ESF
Research Networking Programme (*). This conference aims to gather the
best research currently being performed on Genomics and its applications
to Conservation.

The conference is organized by ConGen (Conservation genetics and wildlife
management) research group and hosted in CIBIO/InBIO Associate Laboratory
and the University of Porto facilities in Vairao, Portugal.

We aim to provide an inspiring atmosphere for discussing the aplication
of genomics to conservation, namely for infering demography, population
history, population viability, adaptative viability, taxonomy, as well
as the use of metagenomics and environmental DNA.

The program includes six invited plenary lectures, submitted oral
communications (regular and flash) and poster sessions, covering the state
of art on conservation genomics. Travel grants are available for students
and early career researchers (please see conference website for details).


Confirmed Invited Speakers:

Fred Allendorf (University of Montana)
Mike Bruford (Cardiff University)
Lisette Waits (University of Idaho)
Natasha de Vere (National Botanic Garden of Wales)


Organizing & Scientific Committee:

Paulo Clio Alves (University of Porto)
Ricardo Jorge Lopes (University of Porto)
Jos Melo-Ferreira (University of Porto)
Raquel Godinho (University of Porto)

Joop Ouborg (Radboud University)
Philippine Vergeer (Wageningen University)
Jacob Hglund (Uppsala University)
Gernot Segelbacher (University of Freiburg)
Craig Primmer (University of Turku)


(*) ConGenOmics is a ESF Research Networking Programme that aims to bring
together European and pan European specialists on conservation genomics,
to develop and improve the knowledge transfer of the technical advances
and challenges in conservation genomics, focusing specifically on the
application of genomic techniques. ConGenOmics programme organizes
workshops, summer schools, collaborative expert meetings and an
exchange program at panEuropean level, to further develop the field
of conservation genomics and to aid in the knowledge transfer of the
technological advances and challenges among European research groups
focusing specifically on applications of genomic technologies in
conservation biology. More details in <a href="http://bit.ly/1Oz1eIl">http://bit.ly/1Oz1eIl</a>


Questions? Contact us by email (info@congenomics2016.com) or on
Facebook/twitter

riclopes@me.com


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:11</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136372692311">Graduate position: ImperialCollege_London.PlantFungiInteractions</a></div>
  <div class="description">ImperialCollege_London.PlantFungiInteractions

Plant-fungal interactions in alpine ecosystems

A Ph.D. studentship is available to investigate plant-fungal interactions
in alpine ecosystems focusing on environmental drivers and the influence
of plant host genetic variation on mycorrhizas across elevation and
latitudinal gradients.  Alpine plant and fungal communities are models for
understanding pioneer communities, the primary components of resilient
ecosystems.  Climate change is one of the biggest threats to organismal
and functional diversity and this project aims to understand its impacts
on widespread symbiotic interactions.  A strong interest in fungal and/or
plant molecular ecology and evolution is essential.  A first class degree
in biological sciences is required and a Masters degree in a relevant
area is desirable.   The student will be affiliated with the Department
of Life Sciences at Imperial College London (<a href="http://bit.ly/1OAevGM">http://bit.ly/1OAevGM</a>)
and based at the Royal Botanic Gardens (<a href="http://www.kew.org/">http://www.kew.org</a>).  A stipend,
tuition fees and research funds are provided.  The student can join
the Grand Challenges in Ecosystems and Environment Doctoral Training
Programme.  The starting date is in the Spring of 2016.

Informal enquiries should be made to Dr Martin Bidartondo
(m.bidartondo@imperial.ac.uk).  Applications should contain a full CV
and the names and contact details of two academic referees and should
be sent to Dr Bidartondo.  Only UK or EU applicants are eligible.
Deadline for receipt of applications is 15 February 2016.

&ldquo;Martin I. Bidartondo&rdquo; 
via Gmail</div>
 </div>
</div>
<h3>December 31, 2015</h3>
<div class="news-item">
 <div class="date">23:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136372092711">EvolDir Mnth Rev</a></div>
  <div class="description"><p>The monthly review of the EvolDir is available from the EvolDir
website - <a href="http://bit.ly/17VdShI.">http://bit.ly/17VdShI.</a>

For more information or for help send mail to Golding@McMaster.CA.
Please see the above website for instructions for the EvolDir.  This
includes ways to limit/reduce the messages you receive from the
EvolDir.

A reminder:  The userid for the web site user list is `evoldir&rsquo; and
the password is `2EvolBulletins&rsquo;.  [ This was done to limit the access 
of spammers (e.g. SpeakOut.Com and others). ]

Brian Golding


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:04</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136312319111">Job: Mercyhurst University- Physiology</a></div>
  <div class="description">&ndash;_000_D2AA01D21C0A2sturnermercyhurstedu_
Content-Type: text/plain; charset=&ldquo;us-ascii&rdquo;
Content-Transfer-Encoding: quoted-printable

The Department of Biology at Mercyhurst University seeks applications for a tenure-track faculty position at the Assistant Professor level to begin August 2016.  Candidates must have a Ph.D. in the biological sciences or a related field, and a strong commitment to teaching and maintaining an active research program involving undergraduates. Teaching responsibilities will include Human Physiology and contribution to freshman-level, introductory biology courses, and/or anatomy-based courses for non-majors.  The candidate will also have the opportunity to develop specialty courses in her/his area of expertise.  Available research equipment for the candidate includes an Illumina MiSeq, Applied Biosystems Real-time PCR, LI-COR Odyssey, LI-COR LI6400 Portable Photosynthesis System, epifluorescent microscope, and tissue culture facility. The area of research is open, but particular consideration will be given to candidates whose interests complement those of the existing faculty. Pr
 eference will be given to candidates with previous teaching experience and success in securing extramural funding to support research. Contractual responsibilities also include academic advising, and service to the university and department.  To apply, please send a single pdf containing a cover letter, CV, statement of teaching philosophy, and research statement, and arrange for three letters of reference to be sent to Dr. Chris Dolanc at cdolanc@mercyhurst.edu.  The initial review of complete applications will begin January 8, 2016 and will continue until the position is filled.

Mercyhurst University is a fully accredited, four-year, liberal arts institution in Erie, Pennsylvania founded in 1926 by the Sisters of Mercy. Today, the university is home to more than 3,000 undergraduate and graduate students and >160 faculty in more than 50 undergraduate majors and eight graduate degree programs.  The Department of Biology is a vibrant, growing department of 10 full-time faculty and >170 majors.

Mercyhurst University is an Equal Employment Opportunity Employer that encourages members of diverse groups to apply.

&ndash;_000_D2AA01D21C0A2sturnermercyhurstedu_
Content-Type: text/html; charset=&ldquo;us-ascii&rdquo;
Content-ID: 
Content-Transfer-Encoding: quoted-printable



The Department of Biology at Mercyhurst University seeks applications for a tenure-track faculty position at the Assistant Professor
 level to begin August 2016.  Candidates must have a Ph.D. in the biological sciences or a related field, and a strong commitment to teaching and maintaining an active research program involving undergraduates. Teaching responsibilities will include Human Physiology
 and contribution to freshman-level, introductory biology courses, and/or anatomy-based courses for non-majors.  The candidate will also have the opportunity to develop specialty courses in her/his area of expertise.  Available research equipment for the candidate
 includes an Illumina MiSeq, Applied Biosystems Real-time PCR, LI-COR Odyssey, LI-COR LI6400 Portable Photosynthesis System, epifluorescent microscope, and tissue culture facility. The area of research is open, but particular consideration will be given to
 candidates whose interests complement those of the existing faculty. Preference will be given to candidates with previous teaching experience and success in securing extramural funding to support research. Contractual responsibilities also include academic
 advising, and service to the university and department.  To apply, please send a single pdf containing a cover letter, CV, statement of teaching philosophy, and research statement, and arrange for three letters of reference to be sent to Dr. Chris Dolanc at <a href="mailto:cdolanc@mercyhurst.edu" rel="nofollow">cdolanc@mercyhurst.edu</a>. 
 The initial review of complete applications will begin January 8, 2016 and will continue until the position is filled.

Mercyhurst University is a fully accredited, four-year, liberal arts institution in Erie, Pennsylvania founded in 1926 by the
 Sisters of Mercy. Today, the university is home to more than 3,000 undergraduate and graduate students and >160 faculty in more than 50 undergraduate majors and eight graduate degree programs.  The Department of Biology is a vibrant, growing department of
 10 full-time faculty and >170 majors.

Mercyhurst University is an Equal Employment Opportunity Employer that encourages members of diverse groups to apply.



&ndash;_000_D2AA01D21C0A2sturnermercyhurstedu
via Gmail</div>
 </div>
</div>
<h3>December 30, 2015</h3>
<div class="news-item">
 <div class="date">23:49</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136311569896">Postdoc:UIowa.InsectEvolution</a></div>
  <div class="description"><p>The Forbes Lab in the Department of Biology at the University of Iowa is seeking a highly motivated evolutionary ecologist with an interest in tropical insect diversity and expertise in population genomics for a 2-3 year postdoctoral position funded by NSF&rsquo;s Dimensions of Biodiversity Program. 

Job Summary:
The aim of the project is to examine patterns of present day and historical gene flow among species in the tephritid fruit fly genus Blepharoneura and the same for the Bellopius parasitoid wasps that attack Blepharoneura flies. These flies and wasps are highly host-specific, incredibly diverse, and many species often overlap in their apparent niche space. A major goal is to understand why these insects are so diverse. The postdoc will develop and score double-digest RAD markers for several existing fly and wasp collections from across Central and South America, use those data to model gene flow and migration within each fly and wasp species, and collaborate with other project participants to test hypotheses regarding species interactions and the origin of reproductive isolation in these hyper-diverse insect systems. More information about this system can be found in the following two publications:
<a href="http://bit.ly/1Vrajsf">http://bit.ly/1Vrajsf</a> 
<a href="http://bit.ly/1YQakeN">http://bit.ly/1YQakeN</a> 

Education required:
* PhD in evolutionary biology or closely related field

Desirable Qualifications:
* Experience developing and working with RAD-based marker system (or similar marker system) in non-model organisms
* Strong background in population genetics / genomics
* Interest in diversification, insect evolutionary ecology, insect population genomics, or related field
* Interest in working as part of a collaborative research team

Preferred Starting date: September 2016 (negotiable)

Application Deadline: April 15th, 2016, or until position is filled

How to apply: send C.V., a letter of inquiry, two representative publications (if available), and contact information for three potential referees to: andrew-forbes@uiowa.edu 


via Gmail</p></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="index8713.html?q=aggregator/sources/14&amp;from=0">first page</a></div><div><a href="index2b69.html?q=aggregator/sources/14&amp;from=180">previous page</a></div><div>... <a href="index7de1.html?q=aggregator/sources/14&amp;from=120">7</a> <a href="index2be7.html?q=aggregator/sources/14&amp;from=140">8</a> <a href="index5104.html?q=aggregator/sources/14&amp;from=160">9</a> <a href="index2b69.html?q=aggregator/sources/14&amp;from=180">10</a> <strong>11</strong> <a href="index9605.html?q=aggregator/sources/14&amp;from=220">12</a> <a href="index024a.html?q=aggregator/sources/14&amp;from=240">13</a> <a href="indexe9f7.html?q=aggregator/sources/14&amp;from=260">14</a> <a href="index9a9b.html?q=aggregator/sources/14&amp;from=280">15</a> </div><div><a href="index9605.html?q=aggregator/sources/14&amp;from=220">next page</a></div><div><a href="index9a9b.html?q=aggregator/sources/14&amp;from=280">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news." class="active">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/14&from=200 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 21:44:50 GMT -->
</html>
