<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=400 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:36:38 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="indexacd7.html?q=node/208">56(4) August 2007</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 9 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>December 24, 2015</h3>
<div class="news-item">
 <div class="date">03:16</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135838809466">Other: DAMBE update</a></div>
  <div class="description">&ndash;_000_D4C76255EEDA2E4F83A13A4447297DD2EA904BA0CMSP03uottawaou_
Content-Type: text/plain; charset=&ldquo;us-ascii&rdquo;
Content-Transfer-Encoding: quoted-printable

Dear All,

I just noticed that I have packed an outdated NucSubMat.BIN file (dated 2003) with DAMBE. This file contains a collection of nucleotide score matrices for sequence alignment in binary format. The new function for phylogenetics with pairwise alignment uses a matrix that is not in that file, resulting in wrong output for nucleotide sequences (amino acid and codon sequence alignment will be fine). I have uploaded a new version with the correction.

Merry Christmas!

Best
Xuhua
<a href="http://bit.ly/1bVebhK">http://bit.ly/1bVebhK</a>
<a href="http://bit.ly/1OMMS7N">http://bit.ly/1OMMS7N</a>


&ndash;_000_D4C76255EEDA2E4F83A13A4447297DD2EA904BA0CMSP03uottawaou_
Content-Type: text/html; charset=&ldquo;us-ascii&rdquo;
Content-Transfer-Encoding: quoted-printable



</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 22, 2015</h3>
<div class="news-item">
 <div class="date">00:26</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135696500746">Course: Hungary.ConGen.May30-Jun10</a></div>
  <div class="description"><p>Course: ConGen2016 - Recent Advances in Conservation Genetics (May 30
-  June 10, 2016)

We are pleased to announce the course &ldquo;ConGen 2016 - Recent Advances in
Conservation Genetics.&rdquo;

The 2016 course will be held at the Lake Balaton Limnological Institute
on the shores of this charming freshwater lake in the Transdanubian
region of Hungary. The course will be directed by Dr. Stephen J.
O'Brien, and taught by renowned scientists in methods,
interpretation, and applications of molecular genetic and genomic
analyses for conservation of endangered species, who will also share a
variety of their personal experiences in this important field.

Information about the course, participating faculty, venue, and
registration can be found at the following link:

<a href="http://bit.ly/1V0rYHf">http://bit.ly/1V0rYHf</a>

Thank you for your help.

Best regards,

Klaus Koepfli

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:11</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135695868341">Postdoc: LakeBaikal.MolluscanRadiation</a></div>
  <div class="description">Short term post-doctoral position - Lake Baikal, Russia

9-12 month position from February 2016 working on molecular biology of
an endemic molluscan radiation from Lake Baikal, Siberia. This project
is a collaboration between molluscan taxonomist Tatiana Sitnikova and the
molecular biology lab of Dmitri Sherbakov at the Limnological Institute,
Russian Academy of Sciences, Irkutsk, Siberia, studying the endemic
gastropods of the worlds most ancient and diverse freshwater lake. More
specifically, the project aims at the elucidation of the discrepancies
between the diversity patterns obtained from mitochondrial DNA and
nuclear DNA data in order to obtain a better evolutionary history of
the group and to fit it to the well-studied history of the Baikalian
eco-system transformations.

The project is funded for Sanger sequencing and some NGS (obtaining
depleted libraries or 2-3 transcriptoms). It also includes participation
in 10-15 days expedition around Lake Baikal.

The successful applicant should have skills in molecular laboratory
and analytical work (general lab routines, PCR, purification of
amplicons, dealing with DNA sequences and phylogenetic analysis of
various kinds). The applicant will also be involved into analysis and
publication of the data already obtained.

The project will deliver publications with the successful applicant as
co-author or lead author as appropriate.

The position includes salary at local level. Knowledge of Russian is
not required, though willingness to learn is an advantage.

This is a great opportunity for a young researcher with a sense of
adventure and good academic research skills. Siberia is a region of
can-do optimism and Lake Baikal is one of the most interesting systems
for an evolutionary biologist to study.

For further information on outputs from our lab see the
Limnological institutes webpage <a href="http://lin.irk.ru/">http://lin.irk.ru</a> and
<a href="http://bit.ly/1NIKxeb.">http://bit.ly/1NIKxeb.</a>

Contact  Professor Dmitry Sherbakov
dysh007@gmail.com for specifics on the
application process.

Ellinor Michel 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 21, 2015</h3>
<div class="news-item">
 <div class="date">23:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135695211816">Postdoc: UBristol.MarineEnvDNA</a></div>
  <div class="description">SeaDNA - Assessing marine biodiversity and structure using environmental
DNA: from groundtruthing to food web structure and stability.

We are looking for a postdoctoral researcher with interests in evolutionary
biology and ecology to take a key role in a new 1M collaborative project
studying environmental DNA (eDNA) in marine ecosystems. The overall
objective of the project is to test the potential for eDNA as a tool for
marine biodiversity assessment. The key aims of the project are to i)
establish how long eDNA from fish and invertebrates persists in the marine
environment before it is broken down beyond the point of detectability, 2)
test the ability of eDNA to be used as a tool in surveying and monitoring
marine species, through direct comparisons of eDNA data to those from
traditional surveys, 3) use eDNA data for reconstructing food webs, test
for stability in food webs over space and time. The project will combine
experimental work on eDNA degradation in the laboratory with analyses of
eDNA samples collected from European shelf seas and the Southern Ocean. The
project is a partnership between the University of Bristol (Martin Genner),
Salford University (Stefano Mariani), British Antarctic Survey (Geraint
Tarling, Sophie Fielding and Will Goodall-Copestake), the Marine Biological
Association (David Sims) and Imperial College (Eoin O'Gorman).

Key roles of will be undertaking experimental work on eDNA degradation in
the laboratory, conducting analyses of eDNA samples collected from European
shelf seas and the Southern Ocean, liaising closely with partners
organisations to ensure project progression, disseminating research through
talks and peer reviewed manuscripts, and contributing to project impact
through engagement with research end-users. You should have a doctoral
degree and should also have an aptitude for experimental design, and
enthusiasm for aquarium-based experiments and fieldwork. You should also
have experience of molecular laboratory work and knowledge of
bioinformatics tools.. There is funding to support this post for 39 months,
is Bristol based, and will start on 1 March 2016.

Closing Date 10 January 2014, with interviews in mid-late January.
Salary 31656 to 35609 per annum.
For the job description and information on how to apply, visit
<a href="http://bit.ly/1ZmztKU">http://bit.ly/1ZmztKU</a> using the job reference code: ACAD101769

For informal enquiries please contact Dr Martin Genner
(m.genner@bristol.ac.uk)

Dr Martin Genner
School of Biological Sciences
University of Bristol
Bristol Life Sciences Building
24 Tyndall Avenue
Bristol
BS8 1TQ

Tel: +44 (0) 117 39 41182

M Genner 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:41</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135694549361">Workshop: Switzerland.Evolutionary_Biology_in_the_Alps.June21-27</a></div>
  <div class="description">Dear Colleagues

The 2016 edition of the Evolutionary Biology Workshop in the Alps will 
take place on 21-27June 2016 in Arolla, Switzerland.

Target participants are PhD students in early stages of PhD and advanced 
Master students - please pass the info to potential participants.

The main goals of this annual workshop, based on a concept developed by 
Stephen Stearns and John Maynard Smith, are to develop the following skills:
•    developing your scientific ideas through discussions in groups;
•    thinking critically and expressing oneself clearly;
•    turning a general idea into a research project;
•    writing a research proposal and defending it.

Faculty:
Bill Rice (University of California Santa Barbara)
Dolph Schluter (University of British Columbia)
Tanja Schwander (University of Lausanne)
John Pannell (University of Lausanne)
Tad Kawecki (University of Lausanne)

It is you, the students, who will be in charge in this course. You will 
be divided in groups of 4-5 students. In those groups, you will work on 
your ideas. You, as a group, will decide what the important open 
questions in broadly defined evolutionary biology are, you will choose 
one, and attempt to develop a proposal for a research project that will 
address it. The faculty will visit the groups during the discussions to 
answer your questions, provide coaching and give you feedback on your 
projects, but they will generally take the back seat. Additionally, the 
faculty will give talks about their research and be available for 
informal discussion with individual students. At the end you will 
present your projects to other participants, and we will party.

The workshop will take place in the 100-year old hotel Kurhaus
<a href="http://bit.ly/1hkynKm,">http://bit.ly/1hkynKm,</a>
located in a small alpine village at an altitude of 2000 m, which will 
help you to concentrate on the course while giving you also the chance 
to enjoy the views, small hikes, marmots and the alpine flora.

Fee: CHF 450.- (this covers part of the cost for room and board, the 
rest being subsidized by the Herbette Foundation).

Participants will receive a course certificate for 3 ETCS credits.

To apply, send a single file (pdf or rtf) containing a short motivation 
letter including a brief summary of your research interest, a cv, and 
the name of your scientific advisor to Caroline Betto-Colliard 
, with Cc to tadeusz.kawecki@unil.ch .  
Please put &ldquo;Evolutionary workshop&rdquo; in the subject. Application Deadline: 
February 14, 2016.

via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135628361741">Other: IUCN.subspeciesListing.answers</a></div>
  <div class="description">*Subspecies diversification and its importance in conservation*

Dear all,

This is in reference to the issue I raised last week whether the subspecies
should be considered as a conservation unit for the assessment under IUCN
Vred list. I emphasise this particularly for those species whose mulitple
forms (subspecies) exists and they have not accorded separately instead
lumped to a species complex under IUCN.

The responses I received are overwhelming and some of you were also
experiencing the same problem that I am facing here in India. I have
seriously considered each of your response/reply to compile my view point
for the species of my concern.

thanking you so much for writing, sharing your experiences, views and
suggesting me the possible way to look forward and convince the
authorities. I greatly acknowledge those who shared articles in support of
their views and had discussion with me. I am herewith sharing you the
compilation report of responses.

Thanking you so much.

Regards

Mukesh Thakur

1 # Ron Okimoto replied:

Get some population geneticists involved.  We are just starting to get a
glimpse of what a genetic load is for a population.   My take is that all
endangered species should be classified and managed by sub species.  The
evolutionary facts of life are that nearly all the sub species are going to
go extinct for one reason or another.  We are just getting data on what the
genetic load is for a population in terms mutations in the genome.  It may
be that some populations will survive their genetic load upon population
reduction and most will not.  The more populations the better the chance
that one will survive.  If you start mixing the populations artificially to
increase current reproductive success you may be dooming the species to
extinction by just making all the populations the same in terms of their
chance at ultimate survival.

Trying to maintain the largest effective population may be the simplest to
manage, but it may not be the best solution in terms of ultimate survival
of that lineage.  We only see the survivors in the current biosphere and
the survivors have been few and the failures have been many.

Ron Okimoto

2 # Dr. Acacia Alcivar-Warren replied:

Dear Mukesh,

Thank you very much for raising this very important IUCN issue.

On a separate but related concern related to marine shrimp populations in
their natural range in the Pacific coast of Latin America, I have not
succeeded in encouraging IUCN leaders to take action to review the penaeus
taxon.

The natural populations of the Pacific whiteleg shrimp, *Litopenaeus
(Penaeus) vannamei,* from northern Peru to northern Mexico, may be at risk.
The unregulated international trade threatens food safety, food security
and national security. There are no international authorities, or concerned
veterinarians, in charge of addressing the lack of supervision for the
massive translocation of a subspecies from one continent to another
-without quarantine!. Nobody seems to take responsibility for this.

The severe *Vibrio* spp-causing disease (AHPND/EMS) of fast growing SPF *L.
vannamei *from US, which first appeared in China in 2009 and quickly spread
to other countries in Asia and then to Mexico in April 2013, is of concern
to conserve healthy biodiversity in light of climate change.

My requests to international agencies and governments to protect the
natural populations of *L. vannamei* have received no replies, not from
IUCN, FAO, World Bank, US authorities, or the authorities of the leading
shrimp producing-countries (Ecuador, India and Indonesia), that claim to be
EMS-free (unconfirmed - we are checking into these claims).

My requests to AVMA leaders and US government&rsquo;s health officials in charge
of protecting our food supply have gone nowhere. Nobody seems to care, in
spite of my repeated calls for action at meetings of the World Aquaculture
Society (<a href="http://bit.ly/1mvDaQg">http://bit.ly/1mvDaQg</a>).
A group in France, supported by IUCN, is actually working on an
industry-sponsored project to &lsquo;address the safety and nutrition of current
feeds used in shrimp aquaculture&rsquo;.

Can I do something to help you *bring together the basic researchers on
population genetics and the leaders of  IUCN?*, to address the issues of
concern to both of us?

Would you be interested in *organizing a Biodiversity Conservation
meeting, *to invite the IUCN leaders to address our issues of concern?.
Friends of the FUCOBI Foundation of Ecuador, of which I was the Founder and
first President, would be very happy to support an international meeting
like this, *if *you include collaborators from our 'ONE HEALTH Epigenomics
Educational Initiative (OHEEI)&rsquo; in India and Nigeria. Indeed, we just sent
a message out to a 'ONE HEALTH&rsquo; list, offering travel awards to pay for
expenses of speakers and students to attend conservation meetings
addressing the holistic concept of 'ONE HEALTH Epigenomics&rsquo;.

Let me know if interested in this offer at: fucobi@gmail.com. The release
is copied below.

Dr. Acacia Alcivar-Warren

ONE HEALTH Practitioner

3 # Gary Thorgaard, Washington State University replied:

Dear Mukesh,

The situation you are interested in sounds similar to that for Pacific
salmon types in the Pacific Northwesr and California, USA.  The U.S.
Government, through its organization NOAA (National Oceanic and Atmospheric
Administration), has listed a number of subgroups within the various salmon
species under the U.S. Endangered Species Act.  The terminology that has
been used for recognition is &ldquo;evolutionarily significant unit&rdquo; (ESU) and
much of the justification for this was developed by Dr. Robin Waples of
NOAA.  I would encourage you to look up his papers and potentially to
correspond with him.
I am not sure about the IUCN&rsquo;s policy in regard to such listings.

Gary

4 # K.A.Bindu replied

Dear Mukesh,

Houbara bustards (chlamydotis undulata )was recently got separated into two
separate species,viz Chlamydotis undulata undulata (African houbara) and
Chlamydotis undulata macqueenii (Asian houbara). The IUCN has listed them
as
vulnerable. But the African bustards comprise of two sub species,
Chlamydotis
undulata undulata and Chlamydotis undulata fuertaventurae (Canary bustards)
and
is there any clear mention about the status of Canary bustards in the IUCN
list?
Regards
K.A.Bindu

5 # *Jonathan Mee replied:*

Hello Mukesh.

Perhaps you would be interested in this article (Ive provided the
open-access link, below). Its about recognizing diversity below the
species level in a widely-distributed fish species.

<a href="http://bit.ly/1OFBmei">http://bit.ly/1OFBmei</a>
Regards,
jon

6 # *Nagarjun V replied:*

Hi Mukesh,

You are doing interesting work with wide reaching implications.

Species de-limitation is a major issue, especially with regards to
conservation. While a few years ago barcoding was used to de-limit species,
now it is being realized that in some species, even use of whole genome
sequencing data is not sufficient to identify species boundaries. One the
other hand one could use ever increasing amount of data to delimit species
based on population structure alone. The crux of the whole issue is species
definition. Should species be defined based on morphology, reproductive
isolation (biological species concept) or just genetic structure? Hence,
conservation genomics is at a cross roads of sorts.

Based on the discussions during a conference on conservation genomics, i
was involved in writing a opinion piece about the subject (see attached).
Hope it will provide you a new perspective at looking

Regards,

Nagarjun

7 # Dr. Axel Hochkirch replied

hi Mukesh,

The official IUCN policy is that a red list assessment has first to be done
on the species level, before subspecies assessments will be published. I
attach the current IUCN guidelines for using the Red List Categories and
Criteria (see page 4). Other intraspecific ranks (formas, morphs,
subvarieties, cultivars etc.) are not included on the IUCN Red List.
So, if you are dealing with a widespread species that has been assessed at
the species level, you are allowed to submit an assessment at the
subspecies unit.
Best regards,

Axel

8 # Keith A. Crandall, PhD replied:

Dear Mukesh,

Great issue!  Thanks for your email.  Ive been involved in red listing of
the freshwater crayfish and the subspecies issue has been problematic.
Basically, IUCN doesnt care about subspecies.  The US Endangered Species
Act, likewise, does not care about subspecies for invertebrates, but it
does for vertebrates!  The ignoring of subspecies in the crayfish leads to
issues of counting because most other lists count subspecies (formally
recognized subspecies).  They often become formal species after further
investigation.  Given that reality, my feeling is that IUCN should be
treating formal subspecies as distinct OTUs deserving of their own
conservation assessment.  Well, thats my two cents anyway.

Take care,

Keith


mukesh thakur 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:16</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135627138761">Graduate position: StellenboschU.LizardAdaptation</a></div>
  <div class="description">A PhD studentship is available in the Botany and Zoology department at
Stellenbosch University in the Western Cape of South Africa. The project is
funded by the South African National Research Foundation and focuses on the
evolutionary responses of lizards to climate change. The student will be
supervised by Susana Clusella-Trullas (Stellenbosch University), Michael
Logan (Stellenbosch University), and Jeffrey Streicher (Natural History
Museum, London).

Global climate is changing rapidly and the degree to which natural
populations respond to temperature variation is essential for predicting
the impacts of climate change on biodiversity. Individuals can adjust to
temperature variation through behaviour, acclimation responses and/or
evolutionary mechanisms. While much work has focused on behavioural
compensation and plastic responses, less emphasis has been placed on
evolutionary adaptation. Furthermore, the links between behaviour and
evolutionary adaptation have rarely been addressed, although theory
predicts that behaviour could either inhibit or promote evolutionary
processes. This project will combine physiological ecology and genetics to
examine the evolutionary paths by which populations adapt to climate
variability. This project should generate a powerful predictive framework
for how climate change drives evolutionary adaptation, and how this is
mediated by behaviour in terrestrial ectotherms.

In addition to collecting measurements of locomotor performance and
metabolic rates in the lab, the student will conduct an ethological field
study. Further, the student will travel to the Natural History Museum in
London to receive training in high-throughput DNA sequencing techniques.
Ideal applicants will have an exemplary academic record and prior research
experience. All project running costs have been secured and successful
candidates will receive a bursary for 3 years starting in 2016. For more
information, or to apply, please send a CV and academic transcripts to
Susana Clusella-Trullas: sct333@sun.ac.za. Review of applications will
begin from the 20th of January 2016 onwards and short-listed candidates
will be contacted to set up interviews by phone or skype.

Michael Logan 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 20, 2015</h3>
<div class="news-item">
 <div class="date">23:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135625712571">Postdoc: ArizonaStateU.EvolutionMedicine</a></div>
  <div class="description"><p>Postdoctoral Fellow in Evolution & Medicine: Education Resources
Development and Online Learning

Salary: $40,000 &ldquo; 48,000, Salary is negotiable based on the candidate&rsquo;s
experience and ASU salary scales.

Reference # 11450

Full-time

The Center for Evolution & Medicine (CEM) at Arizona State University (ASU),
and Founding Director Dr. Randolph Nesse, invites applications for a
Postdoctoral Fellow in Education Resources Development and Online Learning.
Anticipated start date is August 1, 2016. This is a full-time (1.0) benefits
-eligible, fiscal year (July 1 - June 30) appointment. The successful
candidate will be co-mentored by Dr. Nesse, an expert in evolutionary
medicine and Dr. Sara Brownell, an expert in undergraduate biology
education research. As a physician, Dr. Nesse&rsquo;s research on the
neuro-endocrinology of anxiety led to a series of publications that
established the field of evolutionary medicine. Dr. Brownell is a
neuroscientist whose research is focused on biology education.

Renewal is possible on an annual basis contingent on satisfactory
performance, availability of resources, and the needs of the program. For
additional information and policies regarding postdoctoral scholars at ASU,
please see <a href="http://bit.ly/1AE42hI.">http://bit.ly/1AE42hI.</a> 

The successful candidate will help curate existing online resources in
evolutionary medicine, participate in the development and delivery of new
innovative evolutionary medicine courses, both in-person and online, and
will evaluate and study the impact and effectiveness of these resources and
courses. The Fellow will help develop a project plan to build courses that
will initially be offered to ASU and ASU online students and later to
medical personnel all over the world. This is an exciting opportunity to
work with an interdisciplinary faculty in a growing field of study to
develop, implement, and research innovative curriculum and other resources
related to evolution and medicine.

While specific technology expertise is not required, an ideal candidate
would demonstrate a strong interest in the applications of technology in
higher education, and would have an aptitude for developing and assessing
new approaches to learning and how they are best supported by new
technologies. This, together with experience in curriculum development and
teaching, will allow the Education Resources Fellow to provide disciplinary
and pedagogical expertise in creating innovative and rigorously assessed
learning experiences. The Fellow may have supervisory responsibilities that
include the training and mentoring of staff and/or student interns.

This Postdoctoral Fellows Program is a service and training program designed
for early career scientist-teachers who are interested in an academic
pathway involving curriculum development, teaching, and educational
programming in the interdisciplinary field of Evolutionary Medicine. Fellows
will receive mentorship to support their development as educators throughout
the duration of the fellowship and have an opportunity to participate in the
ASU Science Education Group, the Evidenced-based Teaching Seminar series, as
well as the Biology Education Research Lab.

The Center for Evolution & Medicine is a university-wide Presidential
Initiative whose mission is to establish evolutionary biology as an
essential basic science for medicine, worldwide. As an interdisciplinary
unit, and a member of ASU&rsquo;s BioDesign Institute, the postdoctoral fellow
will have multiple opportunities to work in collaboration with faculty from
a wide-range of disciplines. The Center brings leading scientists to ASU to
join existing faculty in research that demonstrates the power of
evolutionary biology to address problems in medicine and public health.

The School of Life Sciences (SOLS) is a creative hub for research and
teaching excellence at Arizona State University. Fast becoming a global
leader in life science education and discovery faculty, graduate students,
postdoctoral fellows and staff, work together to support an environment of
interdisciplinary research to inspire and transform life science students
by providing an innovative learning experience that prepares them to thrive
in a dynamic and demanding world. SOLS aims to improve life by stimulating
scientific discovery and solving critical problems at the intersection of
the life sciences and society.

By the start date, candidates must have completed a Ph.D. in anthropology,
biology or other natural science or science education field that provides an
extensive background in evolutionary biology. Cross-training and experience
in education is desirable. Candidates with experience in the following are
preferred: developing and assessing education resources; conducting
educational research; and using developing digital resources for online and
classroom use. The successful candidates will have a commitment to science
education, must have outstanding writing and organizational skills,
demonstrated capacity for independence and innovation, and the ability to
work as part of a team.

To apply, please email a single pdf document containing a letter of
application that states your interest in and qualifications for the
position, a curriculum vitae, names of three references, and an item of
choice that represents your approach to teaching in undergraduate science
classroom to evmedsearch@asu.edu. The initial closing date for receipt of
complete applications is January 15, 2016; applications will be reviewed
weekly thereafter until the search is closed. A background check is required
for employment.

Arizona State University is a new model for American higher education, an
unprecedented combination of academic excellence, entre preneurial energy
and broad access. This New American University is a single, unified
institution comprising four differentiated campuses positively impacting
the economic, social, cultural and environmental health of the communities
it serves. Its research is inspired by real world application blurring the
boundaries that traditionally separate academic disciplines. ASU serves
more than 80,000 students in metropolitan Phoenix, Arizona, the nation&rsquo;s
fifth largest city. ASU champions intellectual and cultural diversity, and
welcomes students from all fifty states and more than one hundred nations
across the globe.

Arizona State University is an equal opportunity/affirmative action
employer committed to excellence through diversity. All qualified
applicants will be considered without regard to race, color, sex,
religion, national origin, disability, protected veteran status, or
any other basis protected by law.  Women and minorities are encouraged
to apply (<a href="http://bit.ly/1OZMIzD">http://bit.ly/1OZMIzD</a>).  For more information on ASU&rsquo;s
non-discrimination policy, visit ACD 401. For additional information on
the Center for Evolution & Medicine, visit *http://bit.ly/1YqEbdN. For
additional information on the position, please contact Kathleen Holladay,
Assistant Director, at kholladay@asu.edu.


rmnesse@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135625712146">Job: UCalifornia_Merced.LabFieldTech.PopGenetics</a></div>
  <div class="description"><p>Applications are invited for a lab & field tech position in marine
molecular ecology (comparative population genetics / genomics).

The position entails (1) surveys and collection of marine intertidal
invertebrates in California, (2) all stages of wetlab analyses for
comparative phylogeographic analyses, and (2) preliminary summary analyses
of generated data.  The technician also is responsible for maintaining
a clean, orderly, functioning laboratory.

Full details of the position and application form are available at
<a href="http://bit.ly/1JnuY9F">http://bit.ly/1JnuY9F</a>

More information about the lab is available @ mnd.ucmerced.edu/

Michael N Dawson
Associate Professor
University of California, Merced


dawson.mn@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:00</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135556208001">Other: Namibia.VolFieldAssist.Baboons</a></div>
  <div class="description">Tsaobis Baboon Project 2016 - Volunteer Field Assistants

The Tsaobis Baboon Project is a long-term study of desert baboons in
Namibia.  Based at the Institute of Zoology (the research arm of the
Zoological Society of London), and affiliated with the Gobabeb Research
and Training Centre (Namibia), our aim is to carry out fundamental
research in behavioural ecology and population ecology, and to inform
conservation policy and practice for social species.

We are seeking to appoint four volunteer field assistants for periods
ranging from 3-5 months between 1 May and 30 September 2016. The
fieldwork will primarily involve daily follows of baboon troops on
foot, collecting data on the behaviour and ecology of individually
recognisable animals. Our research focus this year is on both baboon
cognition and collective behaviour.

Further information about the Tsaobis Baboon Project, the volunteer
field assistant positions, and the application procedure can be found
on the Project&rsquo;s webpages:

<a href="http://bit.ly/1jTKMml">http://bit.ly/1jTKMml</a>

The deadline for applications is 6pm Sunday 10 January 2016

Claudia Martina 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:00</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135556207566">Graduate position: HumboldtU.ParasiteHostPopulationGenomics</a></div>
  <div class="description"><p>The Group &ldquo;Ecology and Evolution of Molecular Parasite-Host
Interactions&rdquo; of Juniorprofessor Emanuel Heitlinger at the Humboldt
University and the Leibniz Institute for Zoo and Wildlife research,
Berlin, Germany, invites applications for a:

PhD position in population genomics of parasite-host interaction

The position is offered for 3 years at the German 65% TV-L E13 pay
scale.

Research topic: We are establishing the house mouse hybrid zone (HMHZ)
close to Berlin as a system to study host-parasite interactions. In
the HMHZ two subspecies of the house mouse (Mus musculus domesticus
and Mus musculus musculus) meet and hybridize. We investigate parasite
specialization vs. generalism and recombination between strains of
unicellular parasites of the genus Eimeria. Eimeria spp. occur at high
prevalence (around 30%) and differ in specificity for the mouse
subspecies.

Requirements/Qualification of the PhD student:
•    Excellent degree (Master or equivalent) in (molecular) evolutionary
biology or computational biology/bioinformatics
•    Good communication skills in English
•    Some background in statistics and data analysis is required.
•    Experience in programming (R/Perl/Python/C) is an advantage, interest
in it is required.
•    Experience in genomics applications and next-generation sequencing
analysis is an advantage.
•    Background in evolutionary theory and population genetics is an
advantage.


The Heitlinger group works on ecology and evolution of parasite-host
interactions using tools from genomics and transcriptomics. The PhD
candidate will focus on genome evolution of Eimeria spp. in two
closely related subspecies of the house mouse. Phenotypic aspects
(immunology) of the same host individuals are investigated in an
ongoing project of another PhD student. A third PhD project in the
Heitlinger group is focused on gene expression of both host and
parsitized cells. The candidate is expected to closely collaborate
with both PhD students working on these ongoing projects and will be
directly supervised by Emanuel Heitlinger on genomics data analysis.

The Heitlinger group provides a multidisciplinary working
environment. Methods range from field studies, experiments &ldquo;at the
bench&rdquo; to computational analyses. The group is located at the
Department of Molecular Parasitology of the Humboldt University and
has access to laboratories and resources at the Leibniz Institute for
Zoo and wildlife research. This provides a diverse working environment
allowing interaction with researchers working on topics ranging from
conservation ecology to genetic manipulation of parasite biochemical
pathways.

The student has the opportunity to apply for an association with the
structured graduate training program GRK2046 &ldquo;Parasite Infections:
>From Experimental Models to Natural Systems&rdquo; integrating his project
in the Berlin community of parasitologists and providing additional
resources.

Please send your application as single pdf-file by January 31, 2016 to
emanuel.heitlinger@hu-berlin.de.


emanuelheitlinger@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 19, 2015</h3>
<div class="news-item">
 <div class="date">04:15</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135499542441">Conference:Biosphere2.PhenotypeRCN.February2016</a></div>
  <div class="description"><p>The Fifth Annual Summit of the Phenotype Ontology Research Coordination Network will be held at the University of Arizona&rsquo;s Biosphere2, about 40 miles north of Tucson, AZ, from February 26-28, 2016 (Friday through Sunday noon).  

The theme of this meeting will be &lsquo;Complex data integration with phenotypes&rsquo; with a focus on the integration of phenotype data with other data sets. We will summarize where our phenotype community is at with respect to integration with other data types, and we will highlight active projects. We will be looking to the future &ndash; what projects should be priorities for the future? Joining us this year will be folks from the newly funded 'FuturePhy&rsquo; (futurephy.org), who are interested in how to integrate multiple data types, including phenotype, with phylogenetic trees.  

We estimate that the costs for this meeting (transportation to meeting from airport, lodging, food) will be approximately $500, though we will be able to cover expenses for a small number of participants, particularly students and postdocs who have specific interests in using phenotypic data associated with environment in their research. Please contact one of us if you are interested in attending. It should be a great meeting!
 
Paula Mabee; pmabee@usd.edu
Eva Huala; huala@acoma.stanford.edu
Andy Deans; adeans@psu.edu
Suzanna Lewis; suzi@berkeleybop.org

The Phenotype Ontology RCN (<a href="http://bit.ly/1ZhgDVG">http://bit.ly/1ZhgDVG</a>) was funded by the U.S. NSF to establish a network of scientists who are interested in comparing phenotypes across species and in developing the tools and methods needed to enable comparisons.  In contrast to the many well-established efforts in the molecular community, the representation of phenotypic traits using ontologies is in its infancy. Phenotype ontologies, however, have the potential to integrate these data across all levels of the biological hierarchy and to the environment. This RCN is building a community that, because of its expertise, fosters communications across disciplines to enable co-development of interoperable community standards and best practices for phenotype.

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135498760066">Conference: Pathogen Host Shifts, Cambridge UK, 12th April 2016</a></div>
  <div class="description"><p>This is a multi-part message in MIME format.
via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135498759911">Job: UToronto.GenomicsDataAnalyst</a></div>
  <div class="description">Institution/Company: University of Toronto

Location: Toronto, ON

Job Description:

Purpose

In support of research by PIs in the Department of Ecology and
Evolutionary Biology (EEB) the Genomics Data Analyst is expected to
conduct genome and transcriptome assembly and analysis of NGS
(next-generation sequencing) data, by using off-the-shelf software and
by writing and implementing scripts, to help solve important questions
about the evolution of plant and animal genomes.

Qualifications Required

Education:
Bachelor&rsquo;s degree in biological sciences, bioinformatics or computer
sciences, engineering, or statistics, or acceptable equivalent
combination of education and experience.

Experience:
Minimum 3 years experience working with genomic and/or transcriptomic
datasets; demonstrated experience working in a Linux/Unix shell
environment; competency with at least one scripting language (e.g.
Perl, Python, R). Demonstrated experience in analysis of NGS
(next-generation sequencing) with plant and polyploid genomes,
particularly whole genome assembly of large plant genomes, genome
annotation, and DNA and RNA structural modeling, is required.

Skills:
High proficiency with programming languages is required (e.g. C/C++,
Python, Java). Unix server administration and hardware specification. A
broad range of bioinformatics competencies demonstrated at a high
level. Basic understanding of statistics, probability, genetics, and
molecular biology. Good written and verbal communications skills,
organization skills, initiative, tact and analytical skills.
Reliability and good time management skills.

Other:
Ability to interface with diverse personnel and to work productively
both in collaborations with other team members and individually;
ability to balance workloads from multiple competing tasks; able to
effectively mentor and assist graduate students and postdoctoral
fellows in bioinformatics analyses.

If you are interested in applying for this position, please apply
online:

<a href="http://bit.ly/1Mn0uom">http://bit.ly/1Mn0uom</a>

Stephen Wright 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:31</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135498358816">Other: Tree with bootstraping from distance matrix</a></div>
  <div class="description"><p>Hi,

I&rsquo;m creating a tree from a distance matrix using phylip neighbor and 
phylib drawgram. I would like to have bootstrap values for the branches 
but all I have is this one distance matrix. Is this possible?

The distance matrix is calculated from a Genotyping by Sequencing (GBS) 
dataset with a few thousand SNPs. For each individual pair, the distance 
is basically the proportion of genotypes that differ for loci where both 
individuals are genotyped.

I could easily create different distance matrices by randomly sampling 
only some of the SNPs to calculate the distances, but that would not 
make much sense. For example, if I use a low proportion of the SNPs (ie: 
5%), the bootstrap values will end up being lower because the matrices 
will be more different from one another than if I use a high proportion 
of the SNPs (ie: 50+%). It feels like I can control the bootstrap values 
artificially, so that is not a good avenue.

Is there a way, given a single distance matrix, to create a tree with 
bootstrap values?

I&rsquo;ll be happy to summarize the answers and post them back on EvolDir.

Eric

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:01</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135497557096">Conference: FloridaStateU.SoutheasternEcologyEvolution-SEEC.Mar11-13</a></div>
  <div class="description"><p>Dear colleagues,

We are excited to announce that registration is now open for the 2016
Southeastern Ecology and Evolution Conference (SEEC) at Florida State
University on March 11-13, 2016.

SEEC is a regional conference created by and for early-career scientists
in the fields of Evolution and Ecology. This is a great opportunity for
undergraduates to present for the first time, for graduate students to
practice for a larger meeting, or for anyone to present research in a
low-key setting. On a budget? It&rsquo;s just $35 for early bird registration
(until February 12). The first 40 student ASN members to register for
SEEC 2016 will receive a $30 discount on their registration costs,
making registration just $5!

Some of the great things in store this year:
- Plenary speaker Joseph Travis
- Alternative career panel lunch on Saturday

To register, visit our website at
<a href="http://bit.ly/1OezXkG.">http://bit.ly/1OezXkG.</a>

If you&rsquo;re not ready to send in your abstract yet, let us know you&rsquo;re
coming on Facebook.

Questions? Contact us at seec.host@gmail.com or on Facebook/twitter

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 18, 2015</h3>
<div class="news-item">
 <div class="date">19:07</div>
 <div class="body">
  <div class="title"><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12147">On the use of median-joining networks in evolutionary biology</a></div>
  <div class="description">Median-joining (MJ) was proposed as a method for phylogeographical analysis and is enjoying increasing popularity. Herein, we evaluate the efficacy of the approach as originally intended. We show that median-joining networks (MJNs) are theoretically untenable for evolutionary inference, and that confusion has afflicted their use for over 15 years. The approach has two obvious shortcomings: its reliance on distance-based phenetics (overall similarity instead of character transformations) and the lack of rooting (no direction or history). Given that evolution involves both change and time, and the absence of rooting removes time (ancestor–descendant relationships) from the equation, the approach cannot yield defensible evolutionary interpretations. We also examine the impact of MJ analyses on evolutionary biology via an analysis of citations and conclude that the spread of MJNs through the literature is difficult to explain, especially given the availability of character-based analyses.</div>
  <div class="source">Source: <a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></div>
 </div>
</div>
<h3>December 17, 2015</h3>
<div class="news-item">
 <div class="date">23:57</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135430819351">Job: UVienna.SeniorScientistBioinformaticsCompBiol</a></div>
  <div class="description"><p>Job: UVienna.SeniorScientistBioinformaticsCompBiol

A permanent, full-time Senior Scientist position is available in the Department of Botany 
and Biodiversity Research, Faculty of Life Sciences, University of Vienna, Austria. The 
position is initially for 6 years with the possibility of subsequent unlimited employment. 
You will take the lead role in the implementation of research activities and approaches in 
bioinformatics and computational biology in our group. Our research focus is on the 
ecological & evolutionary genomics of adaptation to changing environments, speciation, 
and species diversification / radiation in plants and associated organisms. We address 
these topics using the latest DNA / RNA sequencing technologies in combination with 
spatial, ecological, and phenome data at different hierarchical levels of biological 
diversity.

University of Vienna offers an attractive and dynamic research location in a city with 
exceptional quality of life and in a country with excellent research infrastructure and 
funding provision.

Extent of Employment: 
40 hours/week. Salary grading in accordance with relevant work experience.

Job Description: 
Participation in cutting-edge research projects involving bioinformatics and 
computational biology, including the development / validation of new bioinformatic 
pipelines - Responsibility for the operation and maintenance of high performance 
computing resources - Liaison point for collaboration with external high performance 
computing centres - Participation in international publication and presentation activities - 
Supervision / mentoring of students engaged in bioinformatics and computational 
biology projects - Assistance with teaching in the department.
 
Profile: 
MSc or doctoral (or equivalent) degree in the natural or computer sciences - Experience 
with the analysis of so-called &lsquo;Next Generation Sequencing&rsquo; (NGS) data, i.e. second or 
third generation DNA / RNA sequence datasets - Extensive experience with 
computational work within a Unix / Linux based environment - Programming skills (bash 
scripting; R; basic skills in either Python or Perl) - Excellent written and oral 
communication skills in English - A keen interest to keep abreast with ongoing 
developments in bioinformatics and computational biology - Ability to work in teams - If 
from an educational path other than biology: basic understanding of biology including 
evolution.

To apply: 
Your application must include the following documents: 1-page motivation letter - 
Educational and scientific CV - List of publications - Contact details of three references. 
Please see full job description and submit your application via the website of the Job 
Center at the University of Vienna (<a href="http://bit.ly/1OAfxMY;">http://bit.ly/1OAfxMY;</a> email: 
jobcenter@univie.ac.at) no later than 01.02.2016, mentioning job reference no. 6254. 
For more information please contact: Christian Lexer, christian.lexer@univie.ac.at and 
christian.lexer@unifr.ch.


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:27</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135429601151">Graduate position: DurhamU.EvolutionaryEcology</a></div>
  <div class="description"><p>&ndash;_000_A3DEAA948B69DB4E8DDE546B54CA7FA72305A5BBCISAMRMBS01mdsa_
Content-Type: text/plain; charset=&ldquo;us-ascii&rdquo;
Content-Transfer-Encoding: quoted-printable

A competitive four year PhD studentship opportunity in evolutionary and ecological genetics is available at Durham University, UK.

Title:
The fitness consequences of inbreeding in wild flax.

Supervisory team:
Dr Adrian Brennan, Durham University, UK, with the co-supervision of Prof Rus Hoelzel, Durham University, UK, and Dr Kirsten Wolff, Newcastle University, UK.

Project description:
This project aims to improve our understanding of inbreeding and its relationship with fitness, a fundamental aspect of evolution. We will use wild flax as a model to study this important relationship, thereby raising the possibility of rapidly translating results into sustainably enhancing agricultural production via genetic improvement of crops. Our research in wild flax will allow for the comparison of natural populations exhibiting different levels of selfing, thereby revealing the extent to which natural selfing leads to purging and the development of a genome more resistant to inbreeding depression. We will use new genotyping by sequencing technologies to obtain an unprecedented genome-wide perspective on inbreeding and its evolution, allowing the identification of relevant genes, and therefore the consideration of molecular mechanisms.
Inbreeding is widely understood to reduce fitness through both the expression of recessive deleterious alleles in homozygotes, and through over-dominant allelic interactions. Even naturally highly inbreeding organisms cannot completely purge their genetic load by selection because of faster drift at reduced effective population size. For many crops, including flax, elite lines have been intensively inbred to optimize the stable expression of desirable traits. However, these same lines then suffer inbreeding depression.  Studies of flax have found up to 91 % heterotic increases in F1 seed yield following controlled crosses. However, the agronomic potential of heterosis has only been fully realized in hybrid varieties of maize so far. Only recently have genotyping technologies advanced to a stage where it is possible to test the long-held hypothesis that heterozygosity positively correlates with fitness using genome-wide data for many individuals of non-model organisms. Initial
  results are encouraging but more research is needed in organisms showing a wider range of mating systems and population structures to better understand the fitness consequences and genetic basis of inbreeding.  The student will be trained in next generation sequencing technologies, and integrate studies of wild and glass-house populations to test key hypotheses about inbreeding and fitness.

Eligibility and Funding:
For a UK-based applicant who is successful in competition with other PhD projects, the studentship will provide tuition fees, research funds and a maintenance allowance. A successful European economic area applicant will be provided tuition fees and research funds only.

Application details:
If interested in applying, please contact Dr Adrian Brennan, a.c.brennan@durham.ac.uk , with a CV and a covering letter. The deadline for applications is 29th January 2016.


&ndash;_000_A3DEAA948B69DB4E8DDE546B54CA7FA72305A5BBCISAMRMBS01mdsa_
Content-Type: text/html; charset=&ldquo;us-ascii&rdquo;
Content-Transfer-Encoding: quoted-printable


</p>
</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">08:36</div>
 <div class="body">
  <div class="title"><a href="http://iphylo.blogspot.com/2015/12/will-json-nosql-and-graph-databases.html">Will JSON, NoSQL, and graph databases save the Semantic Web?</a></div>
  <div class="description"><p>OK, so the title is pure click bait, but here's the thing. It seems to me that the Semantic Web as classically conceived (RDF/XML, SPARQL, triple stores) has had relatively little impact outside academia, whereas other technologies such as JSON, NoSQL (e.g., MongoDB, CouchDB) and graph databases (e.g., Neo4J) have got a lot of developer mindshare.</p> <p>In biodiversity informatics the Semantic Web has been a round for a while. We've been pumping out millions of RDF documents (mostly served by LSIDs) since 2005 and, to a first approximation, nothing has happened. I've repeatedly blogged about why I think this is (see <a href="http://iphylo.blogspot.co.uk/2015/07/steve-baskauf-on-rdf-and-page-challenge.html">this post for a summary</a>).</p> <p>I was an early fan of RDF and the Semantic Web, but soon decided that it was far more hassle than it was worth. The obsession with ontologies, the problems of globally unique identifiers based on HTTP (http-14 range, anyone?), the need to get a lot of ducks in a row all mad it a colossal pain. Then I discovered the NoSQL document database <a href="http://couchdb.apache.org/">CouchDB</a>, which is a JSON store that features map-reduce views rather than on the fly queries. To somebody with a relational database background this is a bit of a headfuck:</p> <a href="http://insidebigdata.com/wp-content/uploads/2014/05/humor_mapreduce.png"></a> <p>But CouchDB has a great interface, can be replicated to the cloud, and is FUN (how many times can you say that about a database?). So I starting playing with CouchDB for small projects, then used it to build <a href="http://bionames.org/">BioNames</a> and more recently moved <a href="http://biostor.org/">BioStor</a> to CouchDB hosted both locally and in the <a href="http://cloudant.com/">cloud</a>.</p> <p>Then there are graph databases such as <a href="http://neo4j.com/">Neo4J</a>, which has some really cool things such as <a href="http://graphgist.neo4j.com/#!/gists">GraphGists</a> which is a playground where you can create interactive graphs and query them (<a href="http://iphylo.blogspot.co.uk/2015/08/testing-gbif-taxonomy-with-graph.html">here's an example I created</a>). Once again, this is FUN.</p> <p>Another big trend over the last decade is the flight from XML and its hideous complexities (albeit coupled with great power) to the simplicity of <a href="http://json.org/">JSON</a> (part of the <a href="https://github.com/blog/2047-language-trends-on-github">rise of JavaScript</a>). JSON makes it very easy to pass around data in simple key-value documents (with more complexity such as lists if you need them). Pretty much any modern API will serve you data in JSON.</p> <p>So, what happened to RDF? Well, along with a plethora of formats (XML, triples, quads, etc., etc.) it adopted JSON in the form of JSON-LD (see <a href="http://manu.sporny.org/2014/json-ld-origins-2/">JSON-LD and Why I Hate the Semantic Web</a> for background). JSON-LD lets you have data in JSON (which both people and machines find easy to understand) and all the complexity/clarity of having the data clearly labelled using controlled vocabularies such as Dublin Core and schema.org. This complexity is shunted off into a "@context" variable where it can in many cases be safely ignored.</p> <p>But what I find really interesting is that instead of JSON-LD being a way to get developers interested in the rest of the Semantic Web stack (e.g. HTTP URIs as identifiers, SPARQL, and triple stores), it seems that what it is really going to do is enable well-described structured to get access to all the cool things being developed around JSON. For example, we have document databases such as CouchDB which speaks HTTP and JSON, and search servers such as <a href="https://www.elastic.co/">ElasticSearch</a> which make it easy to work with large datasets. There are also some cool things happening with graph databases and Javascript, such as <a href="https://github.com/crubier/Hexastore">Hexastore</a> (see also Weiss, C., Karras, P., & Bernstein, A. (2008, August 1). Hexastore. Proc. VLDB Endow. VLDB Endowment. <a href="http://doi.org/10.14778/1453856.1453965">http://doi.org/10.14778/1453856.1453965</a>, PDF <a href="http://karras.rutgers.edu/hexastore.pdf">here</a>) where we create the six possible indexes of the classic RDF [subject,predicate,object] triple (this is the sort of thing can also be done in CouchDB). Hence we can have graph databases implemented in a web browser!</p> <p>So, when we see large-scale "Semantic Web" applications that actually exist and solve real problems, we may well be more likely to see technologies other than the classic Semantic Web stack. As an example, see the following paper:</p> Szekely, P., Knoblock, C. A., Slepicka, J., Philpot, A., Singh, A., Yin, C., … Ferreira, L. (2015). Building and Using a Knowledge Graph to Combat Human Trafficking. The Semantic Web - ISWC 2015. Springer Science + Business Media. <a href="http://doi.org/10.1007/978-3-319-25010-6_12">http://doi.org/10.1007/978-3-319-25010-6_12</a> <p>There's a <a href="http://usc-isi-i2.github.io/papers/szekely15-iswc.pdf">free PDF here</a>, and a <a href="http://videolectures.net/iswc2015_szekely_human_trafficking/">talk online</a>. The consortium behind this project researchers did extensive text mining, data cleaning and linking, creating a massive collection of JSON-LD documents. Rather than use a triple store and SPARQL, they indexed the JSON-LD using ElasticSearch (notice that they generated graphs for each of the entities they care about, in a sense denormalising the data).</p> <p>I think this is likely to be the direction many large-scale projects are going to be going. Use the Semantic Web ideas of explicit vocabularies with HTTP URIs for definitions, encode the data in JSON-LD so it's readable by developers (no developers, no projects), then use some of the powerful (and fun) technologies that have been developed around semi-structured data. And if you have JSON-LD, then you get SEO for free by embedding that JSON-LD in your web pages.</p> <p>In summary, if biodiversity informatics wants to play with the Semantic Web/linked data then it seems obvious that some combination of JSON-LD with NoSQL, graph databases, and search tools like ElasticSearch are the way to go.</p></div>
  <div class="source">Source: <a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="index63c1.html?q=aggregator&amp;from=380">previous page</a></div><div>... <a href="index9efb.html?q=aggregator&amp;from=320">17</a> <a href="indexd191.html?q=aggregator&amp;from=340">18</a> <a href="index28a4.html?q=aggregator&amp;from=360">19</a> <a href="index63c1.html?q=aggregator&amp;from=380">20</a> <strong>21</strong> <a href="indexd6af.html?q=aggregator&amp;from=420">22</a> <a href="index764e.html?q=aggregator&amp;from=440">23</a> <a href="index3d2c.html?q=aggregator&amp;from=460">24</a> <a href="indexb567.html?q=aggregator&amp;from=480">25</a> ...</div><div><a href="indexd6af.html?q=aggregator&amp;from=420">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=400 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:36:38 GMT -->
</html>
