<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/21 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 17:30:17 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexfe3a.html?q=node/279">Contact</a></li><li><a href="index5ffe.html?q=node/6">Join</a></li><li><a href="index94ca.html?q=node/449">Policy on Discrimination and Harassment at Society Meetings</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 5 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator/sources/21" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator">news aggregator</a>
<ul>
<li class="expanded"><a href="indexe5f5.html?q=aggregator/sources">sources</a>
<ul>
<li class="leaf"><a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></li>
<li class="leaf"><a href="index4421.html?q=aggregator/sources/8">CiteULike Phylogeny</a></li>
<li class="leaf"><a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></li>
<li class="leaf"><a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></li>
<li class="leaf"><a href="index8aad.html?q=aggregator/sources/9">Evolutionary Bioinformatics</a></li>
<li class="leaf"><a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></li>
<li class="leaf"><a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></li>
<li class="leaf"><a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></li>
<li class="leaf"><a href="index9fb6.html?q=aggregator/sources/22">Phyloseminar</a></li>
<li class="leaf"><a href="index69a2.html?q=aggregator/sources/10">Systematics Association</a></li>
<li class="leaf"><a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></li>
<li class="leaf"><a href="indexf777.html?q=aggregator/sources/21" class="active">The Genealogical World of Phylogenetic Networks</a></li>
<li class="leaf"><a href="indexe0bd.html?q=aggregator/sources/15">dechronization</a></li>
<li class="leaf"><a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></li>
<li class="leaf"><a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></li>
<li class="leaf"><a href="index1f4a.html?q=aggregator/sources/7">postgenomic.com</a></li>

</ul>
</li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 4 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a> &raquo; <a href="index5640.html?q=aggregator">news aggregator</a> &raquo; <a href="indexe5f5.html?q=aggregator/sources">sources</a></div><h2>The Genealogical World of Phylogenetic Networks</h2>
<!-- begin content -->
<div id="aggregator"><div class="feed">Biology, anthropology, computational science, and networks in phylogenetic analysis<h3>URL</h3>
<div class="xml-icon"><a href="http://phylonetworks.blogspot.com/feeds/posts/default?alt=rss"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div><a href="http://phylonetworks.blogspot.com/">http://phylonetworks.blogspot.com/</a>
<h3>Last update</h3>
1 hour 23 min ago</div><h3>March 1, 2016</h3>
<div class="news-item">
 <div class="date">00:44</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a></div>
  <div class="description"><br />There have been a number of posts in this blog about the use of phylogenetics in anthropology, broadly defined (eg. <a href="http://phylonetworks.blogspot.com/2013/01/false-analogies-between-anthropology.html" target="_blank">False analogies between anthropology and biology</a>; <a href="http://phylonetworks.blogspot.com/2015/07/productive-and-unproductive-analogies.html" target="_blank">Productive and unproductive analogies between biology and linguistics</a>). One of the essential points under consideration is the extent to which the phylogenetic model as used in biology can be realistically applied to the evolutionary history of human cultural artifacts (languages, tools, books, tales, concepts, etc).<br /><br />In spite of a number of vocal (and influential) proponents, especially in linguistics and stemmatology, one of the principal criticisms of phylogenetic practice is the use of a tree model. There has simply been too much cross-fertilization in human history for a divergent model of evolution to be viable. A network seems intuitively to be a better model than a tree.<br /><br />Nevertheless, the extent to which tree models are imposed on representations of anthropological histories varies considerably. Indeed, several people have argued for the use of trees even in the face of reticulate histories.<br /><br />It seems to me that one interesting example of this variation concerns the history of religions. Below I show several examples of such phylogenetic histories gathered from around the web. These start with the most tree-like ones and proceed to more realistic representations. However, even the most reticulate one is still basically a "tree covered in vines".<br /><br /><a href="https://4.bp.blogspot.com/-0nO4OenLs5k/VtVFdgl04mI/AAAAAAAAD5g/2dhMKcNkm24/s1600/XRbrS.png" imageanchor="1"></a><br /><br /><a href="https://2.bp.blogspot.com/-pG1f1go6cvw/VtVFdJUDMRI/AAAAAAAAD5Y/nLqN8yIS8sE/s1600/800px-christianity-branches-2013update.png" imageanchor="1"></a><br /><br /><a href="https://4.bp.blogspot.com/-r3OZZ3CSGG8/VtVFc2oMk5I/AAAAAAAAD5c/ToTKKXKbEt0/s1600/2000px-Religions_timeline.svg.png" imageanchor="1"></a><br />The fully scalable version of this final one is available in <a href="https://commons.wikimedia.org/wiki/File:Religions_timeline.svg" target="_blank">Wikimedia</a>.<br /><br /></div>
 </div>
</div>
<h3>February 24, 2016</h3>
<div class="news-item">
 <div class="date">01:18</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a></div>
  <div class="description"><br />I am publishing this post because both the author and publisher involved have stopped responding to my emails.<br /><br />Some of you may have noticed the recent publication of the following book:<br />Dan Graur (2016) <i>Molecular and Genome Evolution</i>. Sinauer Associates.Chapter 6 is of interest to the readers of this blog, being entitled "Reticulate evolution and phylogenetic networks". Unfortunately, as originally published, not all of the figures in that chapter acknowledge the source of the illustration. Of personal interest to me, Figure 6.4 [which can be <a href="http://www.rjr-productions.org/Graur.html" target="_blank">viewed here</a>] is a direct copy of the first part of my <a href="http://acacia.atspace.eu/Tutorial/Tutorial.html" target="_blank">Primer of Phylogenetic Networks</a>. Needless to say, Graur's figure prominently claims to be the copyright of the publisher rather than myself.<br /><br />Neither the author not the publisher has provided a satisfactory explanation, and have made it clear that nothing will be done to rectify this except in some possible future edition of the book.<br /><br />So, in the meantime, could you all please note that the idea is mine, not either Graur's or Sinauer's. This annoys me, not just because of the laziness that lead to this situation, but because the figures are an original conception of mine to explain Median Networks to non-experts, and I was very happy to have developed them. Having someone else trying to take the credit takes the shine off that.<br /><br /></div>
 </div>
</div>
<h3>February 16, 2016</h3>
<div class="news-item">
 <div class="date">04:13</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a></div>
  <div class="description"><br />In an earlier <a href="http://phylonetworks.blogspot.fr/2015/12/lexicostatistics-predecessor-of.html">blogpost</a>I mentioned the now largely abandoned discipline of <i>lexicostatistics</i> that was in vogue in the 1950s, originally initiated by Morris Swadesh (1909-1967; <a href="http://bibliography.lingpy.org/?key=Swadesh1950">Swadesh 1950</a>, <a href="http://bibliography.lingpy.org/?key=Swadesh1952">1952</a>, <a href="http://bibliography.lingpy.org/?key=Swadesh1955">1955</a>), but abandoned in the 1960s and henceforth often labeled as some kind of a <i>failed theory</i> that was explicitly proven to be wrong. <br /><br />The crucial idea of Swadesh was to investigate lexical change from the perspective of the <i>meaning</i> of words. This perspective is contrasted with the perspective which takes similar (cognate) word forms in different languages as a starting point and compares to which degree they differ in their meanings.  Swadesh's perspective, instead, starts from a set of meanings and investigates by which word forms they are expressed, and is also called an <i>onomasiological perspective</i> (which "names" are assigned to concepts?), while the other perspective is called a <i>semasiological perspective</i>(which "meanings" can words have?).<br /><br />From a semasiological perspective, we would start from a set of related words and investigate their meanings. In this way, we could compare English <i>head</i>with German <i>Hauptstadt</i> "capital city" or English <i>cup</i> with German <i>Kopf</i> "head". Through such an analysis, we would learn that there was a semantic shift from the German word <i>Haupt</i>, which originally meant "head", to a more abstract meaning that is now probably best translated as "capital" or "main", and only occurs in compounds, such as <i>Hauptstadt</i> "capital city", <i>Hauptursache</i> "main reason", etc.<br /><br />From an onomasialogical perspective, we would start from a set of  meanings and investigate which words are use in order to express  them in different languages:<br /><br />No.ItemsGermanEnglishDutchRussian1handHandhandhandruka2armArmarmarmruka3mainlyhauptsächlichmainlyhoofdzakelijkglavny4headKopf, (Haupt)headhoofd, kopgolova5cupTassecupkopstakan..................<br />When looking at specific meanings in this way, one can find interesting patterns within one and the same language whenever a language uses the same or similar words to express what are different concepts in other languages. Russian thus uses the same word for "hand" and "arm", Dutch shows the same word for "head" and "cup", and Russian, Dutsch, and German have similar forms for "mainly" and "head". These patterns can be historically interpreted by reconstructing patterns of semantic shift. In the case of English <i>cup</i>, German <i>Kopf</i>, and Dutch <i>kop</i>, for example, the original meaning of the words was "vessel" or "cup". Later on, the word changed its meaning and came to denote "head" in German. The transition is still reflected in Dutch, where the word can denote both meanings.<br /><br />We can model this situation by assuming that every word in a language has a certain <i>reference potential</i> (<a href="http://bibliography.lingpy.org/?key=Schwarz1996">Schwarz 1996: 175</a>; <a href="http://bibliography.lingpy.org/?key=Allwood2003">Allwood 2003</a>; <a href="http://bibliography.lingpy.org/?key=List2014d">List 2014: 21f, 36</a>). This means that every word has the potential to denote different things in the world, due to the concept it denotes primarily. In List (2014: 21), I have tried to depict this as follows:<br /><br /><a href="https://2.bp.blogspot.com/-EMVzmfZX5wk/VsL17WiwovI/AAAAAAAAAKE/jOnN3HJS1hI/s1600/referencep.png" imageanchor="1"></a>Reference Potential of the Linguistic Sign<br />In this visualization, a word form refers to a meaning, and the meaning itself has the potential to denote various things in the world, but with different probabilities. A word that primarily means "head", for example, may likewise be used to denote the "first person", as in the "head of a group", and a word that primarily means "melon" may also be used to denote a "head", due to the similarity in form. We can investigate the reference potential of words by simply looking at different translations in dictionaries. As an example (from List 2014: 36), when looking at our three words English <i>cup</i>, Dutch <i>kop</i>, and German <i>Kopf</i>, we find the following rough arrangement with respect to the reference potential of the word (the thickness of the arrows indicating differences in denotation probability):<br /><br /><a href="https://2.bp.blogspot.com/-gXYDhZFb8L0/VsL17RBUr9I/AAAAAAAAAKM/wDuD8c1SgOw/s1600/referencec.png" imageanchor="1"></a>Reference Potential of Words Across Languages<br />Why do I mention all of this? First, I wanted to show that lexical change, no matter which perspective we take, is a very complex phenomenon. In a simplifying model, we could think of a lexicon as a <i>bipartite network </i>consisting of nodes that represent <i>word forms</i> in a language and nodes that represent <i>meanings</i>, and weighted links between word forms and meanings denoting the<i> frequency</i> by which a word is used to denote a given meaning. In such a network representation, lexical change could be modelled as the re-arrangement of the edges between word forms and meanings. If a word form looses all its edges, this word is lost from the language, but we could also think of new words entering the language, be it that they are borrowed, or created from the language itself.  Such a model would be very simplistic, ignoring aspects like word compounding, by which new words are created from existing ones. But it would be much more realistic than the idea that lexical change is just about the gain and loss of words, as assumed in the quasi-standard model of lexical change in phylogenetic reconstruction.<br /><br />This brings us to my second point. When Swadesh introduced  lexicostatistics, and his very specific onomasiological perspective on lexical change, he  established a model of lexical change that would deliberately ignore all interesting processes underlying the phenomenon.  Since then, we have been looking through a glass darkly. This is like a crime  inspector having no other means but watching potential suspects through  the windows of their apartments, noticing changes, like the differently coloured words in state A and state B in the Figure below, but never knowing  what was really going on inside those flats (state C).<br /><br /><a href="https://4.bp.blogspot.com/-pan3kjCmBhg/VsL17d85C_I/AAAAAAAAAKI/k2OBolxc22Y/s1600/replacement.png" imageanchor="1"></a>Trough a Glass Darkly: The lexicostatistic perspective on lexical change (A, B), and what is really going on (C).<br />Yet, when being honest with oneself, the problem of looking through a glass darkly does not pertain to the lexicostatistic perspective alone, but effectively applies to all of our research on language change. It is just the size and the number of windows that we survey, and the cleanliness of the glasses, that may make a little difference.<br /><br /><b>References</b><br /><ul><li>Allwood, J. (2003) Meaning potentials and context: Some consequences for the analysis of variation in meaning. In: Cuyckens, H., R. Dirven, and J. Taylor (eds.): <i>Cognitive approaches to lexical semantics</i>. Mouton de Gruyter: Berlin and New York. 29-65.</li><li>List, J.-M. (2014) Sequence comparison in historical linguistics. Düsseldorf University Press: Düsseldorf.</li><li>Schwarz, M. (1996) Einführung in die kognitive Linguistik. Francke: Basel and Tübingen.</li><li>Swadesh, M. (1950) Salish internal relationships. <i>Int. J. Am. Linguist.</i> 16.4. 157-167.</li><li>Swadesh, M. (1952) Lexico-statistic dating of prehistoric ethnic contacts. With special reference to North American Indians and Eskimos. <i>Proc. Am. Philol. Soc.</i> 96.4. 452-463.</li><li>Swadesh, M. (1955) Towards greater accuracy in lexicostatistic dating. <i>Int. J. Am. Linguist.</i> 21.2. 121-137.</li></ul></div>
 </div>
</div>
<h3>February 7, 2016</h3>
<div class="news-item">
 <div class="date">16:31</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2016/02/the-network-of-woodpeckers-etc.html">The network of woodpeckers, etc</a></div>
  <div class="description"><br />The world continues on its merry way, searching for fragments of the Tree of Life. That is, research papers continue to be published that give no credence to the possibility of reticulate evolutionary history, especially in zoology.<br /><br />A recent case in point is this one:<br />Matthew J. Dufort (2016) An augmented supermatrix phylogeny of the avian family Picidae reveals uncertainty deep in the family tree. <a href="http://www.sciencedirect.com/science/article/pii/S1055790315002560" target="_blank">Molecular Phylogenetics and Evolution</a> 94: 313–326.The author constructed a supermatrix for 26 loci for 78 taxa of the bird family containing woodpeckers, piculets, and wrynecks. The author used an array of phylogenetic techniques, including the construction of maximum-likelihood "gene" trees, several different maximum-likelihood species trees, plus time trees. All of these methods pre-suppose that the evolutionary history of the species was strictly tree-like.<br /><br />We can use an exploratory data analysis to evaluate how probable is this fundamental assumption. I constructed a SuperNetwork based on the 26 gene trees produced by the author, using the SplitsTree program, as shown here. The network is basically tree-like, with one major exception.<br /><br /><a href="https://4.bp.blogspot.com/-qcKKyxccSi8/VrcxpENPDvI/AAAAAAAAD44/qHA25XbfKn0/s1600/LocusTreesEdited.gif" imageanchor="1"></a><br />I have labeled only one taxon, which seems to be the culprit for the major non-tree-likeness. This species appears in 10 of the gene trees, being part of the <i>Leiopicus</i> group (as expected) in 7 of the rooted trees. It is unexpectedly related to the <i>Picus</i> group in two of the other rooted trees, and is close to <i>Dryocopus</i> in the remaining tree.<br /><br />This network EDA does not, of course, imply the existence of reticulate evolution. It does, however, highlight a pattern of incongruence that requires explanation, if the history of these birds is to fully elucidated. Reticulate evolution remains one of the possible explanations, pending further investigation.<br /><br />For an example of a network view of bird evolution, see:<br />Antonio Hernandez-Lopeza, Didier Raoult, Pierre Pontarotti (2013) Birds perching on bushes: networks to visualize conflicting phylogenetic signals during early avian radiation. <a href="http://www.sciencedirect.com/science/article/pii/S1631068313000419" target="_blank">Comptes Rendus Palevol</a> 12: 333-337. </div>
 </div>
</div>
<h3>January 31, 2016</h3>
<div class="news-item">
 <div class="date">16:30</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2016/02/tardigrades-and-phylogenetic-networks.html">Tardigrades and phylogenetic networks</a></div>
  <div class="description"><br />In this blog we have always championed the use of Exploratory Data Analysis prior to phylogenetic analyses. This approach explores the characteristics of the data before making formal inferences about possible evolutionary scenarios. One of the reasons for doing this is the possibility of data errors. That is, we need to distinguish between estimation errors deriving from our experimental procedures and real biological scenarios, because both of these will result in complex patterns in our data.<br /><br />One possible classification of the potential causes of complex data patterns in phylogenetics is this:<br /><br /><b>Estimation errors</b><br />(i) incorrect data<br />— inadequate data-collection protocol<br />—&nbsp;poor laboratory / museum / herbarium technique<br />—&nbsp;lack of quality control after data collection<br />—&nbsp;misadventure<br />(ii) inappropriate sampling<br />—&nbsp;distant outgroup<br />—&nbsp;rapid evolutionary rates<br />—&nbsp;short internal branches<br />(iii) model mis-specification<br />—&nbsp;wrong assessment of primary homology<br />—&nbsp;wrong substitution model<br />—&nbsp;different optimality criteria<br /><br /><b>Biological complexity</b><br />(iv) analogy<br />—&nbsp;parallelism<br />—&nbsp;convergence<br />—&nbsp;reversal<br />(v) homology<br />—&nbsp;deep coalescence<br />—&nbsp;duplication–loss<br />—&nbsp;hybridization<br />—&nbsp;introgression<br />—&nbsp;recombination<br />—&nbsp;horizontal gene transfer<br />—&nbsp;genome fusion<br /><br />The scientific literature has a number of prime examples where people have asserted a case of biological complexity that has subsequently been questioned, and attributed to estimation errors instead.<br /><br />For example, many of you will have noted the recent attention given to the release of various genome sequences from the Tardigrades, a group of microscopic animals often alleged to be the world's most resistant to environmental conditions. Two rival papers have appeared:<br />Thomas C. Boothby et al. (2015) Evidence for extensive horizontal gene transfer from the draft genome of a tardigrade. <a href="http://www.pnas.org/content/112/52/15976.abstract" target="_blank">Proceedings of the National Academy of Sciences of the USA</a> 112: 15976–15981.Georgios Koutsovoulos et al. (2015) The genome of the tardigrade <i>Hypsibius dujardini</i>. <a href="http://biorxiv.org/content/early/2015/12/13/033464" target="_blank">BioRxiv preprint 33464</a>.The former paper attributes their observed phylogenetic complexity to horizontal gene transfer (group v in the list above) while the latter attributes it to sequencing errors (group i). This situation is discussed in more detail elsewhere on the web, for example:<br /><a href="http://www.theatlantic.com/science/archive/2015/12/rival-scientists-kill-recent-discovery-about-invincible-animals/418755/" target="_blank">Rival scientists cast doubt upon recent discovery about invincible animals</a><a href="http://scopeweb.mit.edu/articles/how-did-these-indestructible-pond-critters-get-their-genes/" target="_blank">How did these indestructible pond critters get their genes?</a>This difference in possible cause (of complexity) matters particularly for the use of phylogenetic networks, because both estimation errors and biological complexity will appear as reticulation patterns in any network. This is particularly important for the assertion of evolutionary scenarios such as horizontal gene transfer, because usually the <b>only</b> evidence for any such gene flow is the complexity of the phylogenetic network — that is, there is no independent experimental evidence, and we are relying entirely on the phylogenetic pattern analysis. Estimation errors must thus be eliminated prior to the phylogenetic analysis, if we are to produce a high quality network.<br /><br />The current situation potentially has unfortunate consequences. For example, there are continual comments that horizontal gene flow is rare, particularly from zoologists, even though there is a large amount of evidence to the contrary. Situations like the current one can only add fuel to this argument, if strong claims of gene flow turn out to be erroneous. There is no quantitative basis for an assertion that gene flow is rare in zoology — those who have looked for reticulate evolution in animals have found it, and those who haven't haven't.<br /><br />In the end, data-display networks are useful for displaying incongruent data patterns, but the source of the incongruence needs to be identified before these networks are turned into evolutionary networks (either explicitly drawn or verbally implied).<br /><br /></div>
 </div>
</div>
<h3>January 24, 2016</h3>
<div class="news-item">
 <div class="date">16:30</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2016/01/what-is-in-traditional-chinese.html">What is in Traditional Chinese Medicines — a network analysis</a></div>
  <div class="description"><br />I used to work for the New South Wales Institute of Technology, which in the late 1980s mutated into the <a href="https://en.wikipedia.org/wiki/University_of_Technology_Sydney" target="_blank">University of Technology Sydney</a> (UTS). During this process it acquired an organization called the College of Traditional Chinese Medicine. This group was placed in the Faculty of Science, for lack of anywhere else to put it.<br /><br />These people had little contact with the rest of the faculty, and I don't recall ever meeting any of them. Indeed, their work was not really based on Western science. These days, the UTS College of Traditional Chinese Medicine offers a <a href="http://handbook.uts.edu.au/courses/c10186.html" target="_blank">Bachelor of Health Science in Traditional Chinese Medicine</a>, although they are most obvious in the <a href="http://www.uts.edu.au/about/faculty-science/traditional-chinese-medicine-clinic/about-us/uts-chinese-herbal-medicine" target="_blank">UTS Chinese Herbal Medicine Clinic</a>, which is also nominally still part of the Faculty of Science.<br /><br />The presence of Traditional Chinese Medicine (TCM) in an Australian university setting is relevant to today's blog post, because Australia seems to be one of the few places to have shown any interest in connecting TCM and Western science. Indeed, there is also a <a href="https://www.uws.edu.au/ssh/school_of_science_and_health/uniclinic/traditional_chinese_medicine" target="_blank">Uniclinic of Traditional Chinese Medicine</a> within the School of Science and Health at Western Sydney University. Most of the interest in studying TCMs has otherwise been confined to Asia (see Dennis Normile. 2003. The new face of Traditional Chinese Medicine. <a href="http://science.sciencemag.org/content/299/5604/188" target="_blank">Science</a> 299: 188-190).<br /><br /><a href="http://1.bp.blogspot.com/-nGEmbBD8gGQ/VqSjrfpZwpI/AAAAAAAAD4o/XQXHqZ__Gtg/s1600/TCM.gif" imageanchor="1"></a><br />Recently, a group of Australian researchers decided to have a look at the content of some of the TCMs available in their country:<br />Megan L. Coghlan, Garth Maker, Elly Crighton, James Haile, Dáithí C. Murray, Nicole E. White, Roger W. Byard, Matthew I. Bellgard, Ian Mullaney, Robert Trengove, Richard J.N. Allcock, Christine Nash, Claire Hoban, Kevin Jarrett, Ross Edwards, Ian F. Musgrave & Michael Bunce (2015) Combined DNA, toxicological and heavy metal analyses provides an auditing toolkit to improve pharmacovigilance of traditional Chinese medicine (TCM). <a href="http://www.nature.com/articles/srep17475" target="_blank">Nature Scientific Reports</a> 5: 17475.Some of these TCMs (12 out of 26) are registered for use with the Therapeutic Goods Administration, which regulates their use within Australia, while the other TCMs are not (which technically means that they should not have been commercially available). However, there is little in the way of pharmacovigilance of herbal medicines anywhere in the world.<br /><br />All of the products were comprehensively audited for their biological (via next generation DNA sequencing), toxicological (LC-MS analysis) and heavy metal (arsenic, cadmium and lead, via SF-ICP-MS analysis) contents. For the latter two analyses the amount of material detected was also quantified.<br /><br />As usual, we can use a phylogenetic network to visualize these data, which I have done using a neighbor-net network on the presence-absence data. The result is shown in the figure. TCMs that are closely connected in the network are similar to each other based on their detected contents, and those that are further apart are progressively more different from each other. The registered products are highlighted in red.<br /><br /><a href="http://2.bp.blogspot.com/-nBuzV5QzkJM/VqSSiA5rbGI/AAAAAAAAD4Y/55B8XJIpD9I/s1600/TCMcontent.gif" imageanchor="1"></a><br />There is wide variation among the products. The seven most divergent TCMs in the network are all unregistered, with the remaining seven being more similar to the registered TCMs. Only two TCMs (TCM10 and TCM17) have no discrepancies between the detected contents and what was declared (either to the regulatory agency, or to the consumer in the form of an ingredients list).<br /><br />The authors summarize this situation:<br />Genetic analysis revealed that 50% of samples contained DNA of undeclared plant or animal taxa, including an endangered species of <i>Panthera</i> (snow leopard). In 50% of the TCMs, an undeclared pharmaceutical agent was detected including warfarin, dexamethasone, diclofenac, cyproheptadine and paracetamol. Mass spectrometry revealed heavy metals including arsenic, lead and cadmium, one with a level of arsenic >10 times the acceptable limit.This study presents genetic, toxicological, and heavy metal data that should be of serious concern to regulatory agencies, medical professionals and the public who choose to adopt TCM as a treatment option. Of the 26 TCMs investigated, all but two can be classified as non-compliant on the grounds of DNA, toxicology and heavy metals, or a combination thereof. In total, 92% were deemed non-compliant with some medicines posing a serious health risk.Such findings are not only of concern to the consumer, but also flag the need for detailed auditing of herbal preparations prior to evaluation in clinical trials.<br /></div>
 </div>
</div>
<h3>January 11, 2016</h3>
<div class="news-item">
 <div class="date">16:16</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2016/01/directional-processes-in-language-change.html">Directional processes in language change</a></div>
  <div class="description"><br />Given that we are still in the process of beginning the new year, it seems to be in order to talk about directions — not in general, but rather in specific, namely, about directions in language change. This is important in so far as many processes in language evolution are directional. This means that they follow a specific direction from a state <i>X</i> to a state <i>Y</i>, and this is frequently attested across a large number of the languages of the world, while the opposite process, that state <i>Y</i> changes to state <i>X</i>, is extremely rare or even unattested.<br /><br />In language evolution there are a lot of well-known and well-investigated processes with a strong directional tendency. In sound change, for example, a [p] can easily become an [f], whether it is in the Indo-European, the Austronesian, or the Sino-Tibetan languages. Yet the opposite process, that an [f] becomes a [p] is extremely rare. Similar tendencies hold for a [k] becoming a [ʧ], as in Italian [ˈtʃɛnto] <i>cento</i> "hundred", going back to Latin [kɛntum] <i>centum</i>"hundred", or a [g] becoming a [h], as in Czech [ɦora] <i>hora</i> "mountain", going back to Proto-Slavic *<i>gora</i> "mountain" (<a href="http://bibliography.lingpy.org/?key=Derksen2008">Derksen 2008</a>).<br /><br />In semantic change, unidirectional tendencies can also be observed, although it is often more difficult to identify them, let alone generalising them.  Nevertheless, I think it is a rather safe bet to claim that words which originally mean "head" have a certain tendency to shift their meaning to denote "(the) first, the boss" or "the upper part, the top", while the opposite shift (that words which mean "boss" or "top" will be used to denote "head") is very unlikely to happen. Finally, in grammatical change, or, to be more precise, in <i>grammaticalization</i> (the process by which languages acquire new grammatical categories) directionality is one of the most important constraints (<a href="http://bibliography.lingpy.org/?key=Haspelmath2004">Haspelmath 2004</a>).<br /><br />Linguists usually know these tendencies very well, and they use them in their daily work, be it when trying to reconstruct the original pronuncation of words in unattested ancestral languages, when deciphering historical documents, or when tracing the semantic development of words through history.  Directional changes are also important in evolutionary biology. Ratchet-like (that means: unidirectional) processes serve as a major explanans for constructive neutral evolution (<a href="http://bibliography.lingpy.org/?key=Gray2010b">Gray et al. 2010</a>), direction is at the core of lateral gene transfer, and — as David mentioned in <a href="http://phylonetworks.blogspot.fr/2012/12/networks-trees-and-little-red-riding.html">an earlier post</a>— the usage of directional (non-reversible) models in phylogenetic reconstruction even provides an elegant way to root a tree (see also <a href="http://bibliography.lingpy.org/?key=Huelsenbeck2002">Huelsenbeck et al. 2002</a>).<br /><br />Given the active transfer of ideas from the biological to the linguistic domain in the last two decades, and the important role that directional processes play in both domains, it is surprising to me that methodological transfer has so far been almost exclusively limited to time-reversible models.  The only approach known to me that explicitly makes use of linguistic knowledge of directions is that of Baxter (<a href="http://bibliography.lingpy.org/?key=Baxter2006a">2006</a>). In this paper, Baxter analysed <i>phonological mergers</i> in Chinese dialects within a framework of Camin-Sokal parsimony (<a href="http://bibliography.lingpy.org/?key=Camin1965">Camin and Sokal 1965</a>).<br /><br />Phonological merger is a specific systemic process in language evolution. When sounds change (and they always change in some way), it may happen that two formerly distinct sounds are pronounced in the same way. As a result, words that formerly sounded different may suddenly sound alike, such as English <i>write</i> and <i>right</i>, which remain different only in their spelling not pronunciation. Mergers are a prototypical irreversible process. Once a merger has happened, speakers cannot go back, unless they recorded the original distinction and artificially tuned their language. But even this may be less easy than it seems — it is always easy to reduce distinctions. For example, most English speakers wouldn't have many difficulties in artificially pronouncing all instances of <i>s</i> as <i>sh</i> during a conversation. But being asked to pronounce a randomly chosen set of words with <i>s</i> as <i>sh</i> will turn out to be much more difficult. For this reason, mergers are an ideal data type for directional models of language change. Their drawback is, however, that they are difficult to determine, which may also be the reason why Baxter's approach has never been tested on other language families since then.<br /><br />It may be justified to use time-reversible models for analyses that use lexical data, especially cognate sets, as in the approaches following Gray and Atkinson (<a href="http://bibliography.lingpy.org/?key=Gray2003">2003</a>), since it is difficult to determine the impact of directional processes on lexical replacement. Furthermore, due to the <a href="http://phylonetworks.blogspot.fr/2015/12/lexicostatistics-predecessor-of.html">specific way the data is sampled</a>, it is extremely difficult to determine directions. Yet in many other approaches that use different types of data, especially in those cases that model sound change processes (<a href="http://bibliography.lingpy.org/?key=Hruschka2015">Hruschka et al.  2015</a>, <a href="http://bibliography.lingpy.org/?key=Wheeler2015">Wheeler and Whiteley 2015</a>) or grammatical change (<a href="http://bibliography.lingpy.org/?key=Longobardi2013">Longobardi et al. 2013</a>), it might have a substantial impact on the results if directionality was explicitly modeled.<br /><br />What does this mean for the directions for the New Year? I keep being surprised by the similarities between evolutionary biology and historical linguistics, be it the organization of information in genomes and languages, the processes that drive evolution, the philosophical questions underlying our investigations, or the quarrels among scholars in their fields. Unfortunately, much of the transfer from the biological to the linguistic domain is still very simplistic, often ignoring the specific differences between the two domains. On the other hand, many fruitful analogies are still out there but have not yet been properly investigated. So, as a direction for those who work in interdisciplinary domains in this New Year, I think we should try to avoid reinventing the wheel, and we should also pay attention to not putting wheels on sledges.<br /><br /><b>References</b><br /><ul><li>Baxter, W. (2006): <b>Mandarin dialect phylogeny</b>. <i>Cah. Linguistique -- Asie Orientale</i> 35.1. 71-114.</li><li>Camin, J. and R. Sokal (1965): <b>A method for deducing branching sequences in phylogeny</b>. <i>Evolution</i> 19.3. 311-327.</li><li>Derksen, R. (2008): <b>Etymological dictionary of the Slavic inherited lexicon</b> . Brill: Leiden and Boston.</li><li>Gray, R. and Q. Atkinson (2003): <b>Language-tree divergence times support the Anatolian theory of Indo-European origin</b>. <i>Nature</i> 426.6965. 435-439.</li><li>Gray, M., J. Lukes, J. Archibald, P. Keeling, and W. Doolittle (2010): <b>Cell biology. Irremediable complexity?</b>. <i>Science</i> 330.6006. 920-921.</li><li>Haspelmath, M. (2004): <b>On directionality in language change with particular reference to grammaticalization</b>. In: Fischer, O., M. Norde, and H. Perridon (eds.): <i>Up and down the cline -- The nature of grammaticalization</i>. John Benjamins Publishing Company: 17-44.</li><li>Hruschka, D., S. Branford, E. Smith, J. Wilkins, A. Meade, M. Pagel, and T. Bhattacharya (2015): <b>Detecting regular sound changes in linguistics as events of concerted evolution</b>. <i>Curr. Biol.</i> 25.1. 1-9.</li><li>Huelsenbeck, J., J. Bollback, and A. Levine (2002): <b>Inferring the root of a phylogenetic tree</b>. <i>Systems Biology</i> 51.1. 32-43.</li><li>Longobardi, G., C. Guardiano, G. Silvestri, A. Boattini, and A. Ceolin (2013): <b>Toward a syntactic phylogeny of modern Indo-European languages</b>. <i>J. Hist. Linguist.</i> 3.1. 122-152.</li><li>Wheeler, W. and P. Whiteley (2014): <b>Historical linguistics as a sequence optimization problem: the evolution and biogeography of Uto-Aztecan languages</b>. <i>Cladistics</i> 30.1. 1-13.</li></ul></div>
 </div>
</div>
<h3>December 24, 2015</h3>
<div class="news-item">
 <div class="date">16:30</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2015/12/fast-food-studies.html">Fast food studies</a></div>
  <div class="description"><br />Season's greetings.<br /><br />For your Christmas reading, this blog usually provides a seasonally appropriate post on fast food, including to date posts about: nutrition (<a href="http://phylonetworks.blogspot.com/2012/12/network-analysis-of-mcdonalds-fast-food.html" target="_blank">McDonald's fast-food</a>), geography (<a href="http://phylonetworks.blogspot.com/2013/12/fast-food-maps-network-analysis.html" target="_blank">Fast-food maps</a>) and diet (<a href="http://phylonetworks.blogspot.com/2014/12/fast-food-and-diet.html" target="_blank">Fast food and diet</a>). This year, we will update some of the geographical and diet information about the effects of fast food on people worldwide.<br /><br />First, there seems to be a general perception that access to fast food is continuing to increase in the modern world, even though that increase started more than half a century ago. Such a perception is easy to verify in the USA, as shown in a previous post  (<a href="http://phylonetworks.blogspot.com/2013/12/fast-food-maps-network-analysis.html" target="_blank">Fast-food maps</a>). However, this trend also appears globally.<br /><br />For example, in 2013 the <i>Guardian</i> newspaper produced a dataset (called the <a href="https://docs.google.com/spreadsheets/d/1L7-l_GSVY7UwKKS6meAxVeSNEZETaR_FCG6VkZfkHk8/edit?pref=2&amp;pli=1#gid=0" target="_blank">McMap of the World</a>) illustrating the recent growth in the number of McDonald's restaurants worldwide (<a href="http://www.theguardian.com/news/datablog/2013/jul/17/mcdonalds-restaurants-where-are-they" target="_blank">McDonald's 34,492 restaurants: where are they?</a>). This first graph shows the relative number of restaurants in 2007 and 2012, with each dot representing a single country. Almost all of the 116 countries showed an increase during the five years (ie. their dots are above the pink line). The only country with a major decrease in McDonald's restaurants was Greece (to the right of the pink line), due, no doubt, to its ongoing financial problems. The country with the largest number of restaurants is, of course, the USA, with Japan a clear second.<br /><br /><a href="http://2.bp.blogspot.com/-XDjOM5L6zOw/Vnu2cz2uC3I/AAAAAAAAD3Q/XbKtgY2f-AI/s1600/McDonalds.gif" imageanchor="1"></a><br />There is also a perception that fast-food restaurants compete for customers against other restaurants, so that suburbs can have one or the other but not both. This can be checked using the data in the <a href="http://ers.usda.gov/data-products/food-environment-atlas/data-access-and-documentation-downloads.aspx" target="_blank">Food Environment Atlas 2014</a> (produced by the USDA Economic Research Service), which show the number of both Fast-food and Full-service restaurants / 1,000 popele in 2012 for each of the more than 3,100 counties in the USA. This is illustrated in the next graph, where each dot represents a single county.<br /><br /><a href="http://4.bp.blogspot.com/-3cw8I8KRCBs/Vnu4e5C2SGI/AAAAAAAAD3o/3mOdmQZmz3w/s1600/FullServe.gif" imageanchor="1"></a><br />For most counties, full-service restaurants actually out-number fast-food restaurants, per capita. There are even a few counties that have no fast-food places at all, but also a few with no full-service restaurants. There are even a few with neither restaurant type, notably in AK (2 counties), KY and ND. Interestingly, 3 out of the 4 counties with the largest density of full-service restaurants are in CO (including one not shown because it is off the top of the graph).<br /><br />Nevertheless, you can't go far in the USA without encountering a fast-food place. As shown in a previous post  (<a href="http://phylonetworks.blogspot.com/2013/12/fast-food-maps-network-analysis.html" target="_blank">Fast-food maps</a>), Subway has the largest number of establishments, not McDonald's. The Flowing Data blog has recently compiled a couple of maps showing the dominance of Subway in the sandwich business (<a href="http://flowingdata.com/2015/05/13/where-subway-dominates-its-sandwich-place-competition-basically-everywhere/" target="_blank">Where Subway dominates its sandwich place competition, basically everywhere</a>). This map shows the subway dominance — each dot is an area with a 10-mile radius, colored by the brand of the nearest sandwich chain.<br /><br /><a href="http://4.bp.blogspot.com/-vpvrcb2Nycc/Vnu-wl6hXiI/AAAAAAAAD34/oCRt9CZE2Bs/s1600/sandwich-modal-subway-21.png" imageanchor="1"></a>Unfortunately, studying the effects of geography on access to fast food is not as simple as it might seem. Large-scale patterns such as those shown above are only part of the picture, because access to fast food is usually assumed to be determined at a very local scale — how far is the nearest fast-food place and how easy is it to get there?<br /><br />There have been many studies over the years, based on different methods and with different study criteria. These have been summarized (from different perspectives) by SE Fleischhacker, KR Evenson, DA Rodriguez, AS Ammerman (2010. A systematic review of fast food access studies. <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1467-789X.2010.00715.x/abstract" target="_blank">Obesity Reviews</a> 12: e460) and LK Fraser, KL Edwards, J Cade, GP Clarke (2010. The geography of fast food outlets: a review. <a href="http://www.mdpi.com/1660-4601/7/5/2290" target="_blank">International Journal of Environmental Research and Public Health</a> 7: 2290-2308).<br /><br />Their conclusions from their worldwide literature reviews include:<br /><ul><li>most studies indicated fast-food restaurants were more prevalent in low-income areas compared with middle- to higher-income areas (ie. there is a positive association between availability of fast-food outlets and increasing socio-economic deprivation);</li><li>most studies found that fast food restaurants were more prevalent in areas with higher concentrations of ethnic minority groups;</li><li>those studies that included overweight or obesity data (usually measured as the body mass index) showed conflicting results between obesity / overweight and fast-food outlet availability — most studies found that higher body mass index was associated with living in areas with increased exposure to fast food, but the remaining studies did not find any such association;</li><li>there is some evidence that fast food availability is associated with lower fruit and vegetable intake.</li></ul>In a previous post (<a href="http://phylonetworks.blogspot.com/2014/12/fast-food-and-diet.html" target="_blank">Fast food and diet</a>) I illustrated the association between fast food and obesity in the USA. Here, I use the data from the <i>Guardian</i> article mentioned above (<a href="https://docs.google.com/spreadsheets/d/1L7-l_GSVY7UwKKS6meAxVeSNEZETaR_FCG6VkZfkHk8/edit?pref=2&amp;pli=1#gid=0" target="_blank">McMap of the World</a>) to do the same thing at a global scale. This next graph shows the relationship between the per capita density of McDonald's restaurants and overweight / obesity for those countries for which there are data available (each dot represents a single country).<br /><br /><a href="http://3.bp.blogspot.com/-yCoxgymVZeE/Vnu2cuoiAXI/AAAAAAAAD3U/KsMkMp15_s4/s1600/Obesity.gif" imageanchor="1"></a><br />These patterns have continued over the five years since the reviews appeared, with published studies both pro (eg. J Currie, S DellaVigna, E Moretti, V Pathania. 2010. The effect of fast food restaurants on obesity and weight gain. <a href="http://www.aeaweb.org/articles.php?doi=%2010.1257/pol.2.3.32" target="_blank">American Economic Journal: Economic Policy</a> 2: 32-63) and con (AS Richardson, J Boone-Heinonen, BM Popkin, P Gordon-Larsen. 2011. Neighborhood fast food restaurants and fast food consumption: a national study. <a href="http://www.biomedcentral.com/1471-2458/11/543" target="_blank">BMC Public Health</a> 11: 543) the association between fast food availability and health. To them has been added the issue of Type II diabetes and fast-food consumption (see DH Bodicoat et al. 2014. Is the number of fast-food outlets in the neighbourhood related to screen-detected type 2 diabetes mellitus and associated risk factors? <a href="http://dx.doi.org/10.1017/S1368980014002316" target="_blank">Public Health Nutrition</a> 18 : 1698-1705).<br /><br />Moving on, people have also considered how the role of restaurants might define the identity of American cities. For example, Zachary Paul Neal has considered whether US cities can be classified on the basis of the local prevalence of specific types of restaurants (2006. Culinary deserts, gastronomic oases: a classification of US cities. <a href="http://creativeclass.typepad.com/thecreativityexchange/files/zachary_neal_restaurants_paper.pdf" target="_blank">Urban Studies</a> 43: 1-21). He counted the numbers of several different types of restaurants in 243 of the most populous cities in the USA, and ended up classifying them into four distinct city types: Urbane oases (where one finds an abundance of restaurants of all sorts), McCulture oases (which have larger than normal concentrations of "highly standardised eating places designed for mass consumption"), Urbane deserts and McCulture deserts (both of which have fewer restaurants than their respective oasis counterpart).<br /><br />Unfortunately, this sort of classification approach is self-fulfilling, because any mathematical grouping algorithm will form groups, by definition, even if there are no groups in the data. I have shown this a number of times in this blog (eg. <a href="http://phylonetworks.blogspot.com/2012/05/network-analysis-of-scotch-whiskies.html" target="_blank">Network analysis of scotch whiskies</a>; <a href="http://phylonetworks.blogspot.com/2014/02/single-malt-scotch-whiskies-network.html" target="_blank">Single-malt scotch whiskies — a network</a>). These culinary data are thus crying out for a network analysis, and I would normally present one at this point in the blog post. However, I do not have a copy of Neal's dataset.<br /><br />So, instead, I will finish by analyzing some data on the salt content of fast food (E Dunford et al. 2012. The variability of reported salt levels in fast foods across six countries: opportunities for salt reduction. <a href="http://www.cmaj.ca/content/184/9/1023.long" target="_blank">Canadian Medical Association Journal</a> 184: 1023-1028).<br /><br />The authors collected data on the salt content of products served by six fast food chains that operate in Australia, Canada, France, New Zealand, the United Kingdom and the United States — Burger King, Domino’s Pizza, Kentucky Fried Chicken, McDonald’s, Pizza Hut and Subway. The product categories included: savoury breakfast items, burgers, chicken products, french fries, pizza, salads, and sandwiches. Data were collated for all of the products provided by all of the companies that fitted into these categories (137-523 products per country). Mean salt contents and their ranges were calculated, and compared within and between countries and companies.<br /><br />We can use a phylogenetic network to visualize these data. As usual, I have used the manhattan distance and a neighbor-net network. The result is shown in the next figure. Countries that are closely connected in the network are similar to each other based on their fast-food salt content, and those that are further apart are progressively more different from each other.<br /><br /><a href="http://3.bp.blogspot.com/-7XqBQCqLseM/VnvqwgyILqI/AAAAAAAAD4I/SWNxxKKVG5Y/s1600/Salt.gif" imageanchor="1"></a><br />You will note that the North American countries are on one side of the network, with the highest salt content, while the European countries are on the other, with the lowest salt content (on average 85% of the American salt content). This difference was reflected even between the same products in different countries — for example, McDonald's Chicken McNuggets contained 0.6 g of salt per 100 g in the UK but 1.6 g of salt per 100 g in the USA). As the authors note: "the marked differences in salt content of very similar products suggest that technical reasons are <i>not</i> a primary explanation."<br /><br /></div>
 </div>
</div>
<h3>December 16, 2015</h3>
<div class="news-item">
 <div class="date">16:30</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2015/12/is-ring-of-life-network.html">Is the Ring of Life a network?</a></div>
  <div class="description"><br />Ten years ago, Rivera and Lake decided to emphasize the series if genome fusions that seem to have been involved in the origin of the major phylogenetic groups by calling it the ring of Life rather than the Tree of Life:<br />Maria C. Rivera and James A. Lake. 2004. The Ring of Life provides evidence for a genome fusion origin of eukaryotes. <a href="http://www.nature.com/nature/journal/v431/n7005/abs/nature02848.html" target="_blank">Nature</a> 431: 182-185).<a href="http://4.bp.blogspot.com/-TWxuupgcDOk/VnBvwbOVfmI/AAAAAAAAD2g/ab8BTzVc9HQ/s1600/Ring.gif" imageanchor="1"></a><br />This terminology has been repeated in a number of subsequent papers, including:<br />James McInerney, Davide Pisani and Mary J. O'Connell (2015) The Ring of Life hypothesis for eukaryote origins is supported by multiple kinds of data. <a href="http://rstb.royalsocietypublishing.org/content/370/1678/20140323" target="_blank">Philosophical Transactions of the Royal Society of London B</a> 370: 20140323.However, life is not that simple, and it has more recently become accepted that a set of inter-connected rings is involved in the metaphor, rather than the simple ring originally presented. Thus we now have the plural Rings of Life, instead.<br /><br /><a href="http://2.bp.blogspot.com/-v9PB6ibGmV0/VnBvwS3P66I/AAAAAAAAD2s/5Ycr9w1PiuA/s1600/Ring2.gif" imageanchor="1"></a>James A. Lake and Janet S. Sinsheimer (2013) The deep roots of the Rings of Life. <a href="http://gbe.oxfordjournals.org/content/5/12/2440.abstract" target="_blank">Genome Biology and Evolution</a> 5: 2440-2448.James A. Lake, Joseph Larsen, Brooke Sarna, Rafael R. de la Haba, Yiyi Pu, HyunMin Koo, Jun Zhao and Janet S. Sinsheimer (2016) Rings reconcile genotypic and phenotypic evolution within the Proteobacteria. <a href="http://gbe.oxfordjournals.org/content/early/2015/12/09/gbe.evv221.abstract" target="_blank">Genome Biology and Evolution</a> (in press).I think that the rest of us would still call each of these diagrams a network. Indeed, most of the metaphors that have been used over the years can also be called a network (see <a href="http://phylonetworks.blogspot.com/2012/09/metaphors-for-evolutionary-relationships.html" target="_blank">Metaphors for evolutionary relationships</a>).<br /><br /></div>
 </div>
</div>
<h3>December 8, 2015</h3>
<div class="news-item">
 <div class="date">22:30</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2015/12/lexicostatistics-predecessor-of.html">Lexicostatistics: the predecessor of phylogenetic analyses in historical linguistics</a></div>
  <div class="description"><br />Phylogenetic approaches in historical linguistics are extremely common nowadays.  Especially, probabilistic models that model lexical change as a birth-death process of cognate sets evolving along a phylogenetic tree (<a href="http://bibliography.lingpy.org/?key=Pagel2009">Pagel 2009</a>) are very popular (<a href="http://bibliography.lingpy.org/?key=Lee2011">Lee and Hasegawa 2011</a>, <a href="http://bibliography.lingpy.org/?key=Kitchen2009">Kitchen et al. 2009</a>, <a href="http://bibliography.lingpy.org/?key=Bowern2012">Bowern and Atkinson 2012</a>), but also splits networks are frequently used (<a href="https://www.blogger.com/:ref:Hamed2005">Ben Hamed 2005</a>, <a href="https://www.blogger.com/:ref:Heggarty2010">Heggarty et al. 2010</a>).<br /><br />However, the standard procedure to produce a family tree or network with phylogenetic software in linguistics goes back to the method of <b>lexicostatistics</b>, which was developed in the 1950s by <a href="http://en.wikipedia.org/wiki/:Morris_Swadesh">Morris Swadesh</a> (1909-1967) in a series of papers (<a href="http://bibliography.lingpy.org/?key=Swadesh1950">Swadesh 1950</a>, <a href="http://bibliography.lingpy.org/?key=Swadesh1952">1952</a>, <a href="http://bibliography.lingpy.org/?key=Swadesh1955">1955</a>). Lexicostatistics was discarded by the linguistic community not long after it was proposed (<a href="http://bibliography.lingpy.org/?key=Hoijer1956">Hoijer 1956</a>, <a href="http://bibliography.lingpy.org/?key=Bergsland1962">Bergsland and Vogt 1962</a>). Since then, lexicostatistics is considered a <i>methodus non gratus</i> in classical circles of historical linguistics, and using it openly may drastically downgrade one's perceived credibility in certain parts of the community.<br /><br />To avoid the conflicts, most linguists practicing modern phylogenetic approaches emphasize the fundamental differences between early lexicostatistics and modern phylogenetics. These differences, however, apply only to the way the data is analysed. The basic assumptions underlying the selection and preparation of data have not changed since the 1950s, and it is important to keep this in mind, especially when searching for appropriate phylogenetic models to analyse the data.<br /><br /><b>The Theory of Basic Vocabulary</b><br /><br />Swadesh's basic idea was that in the lexicon of every human language there are words that are <i>culturally neutral</i> and <i>functionally universal</i>; and he used the term "basic vocabulary" to refer to these words.  Culturally neutral hereby means that the meanings expressed by the words are independently used across different cultures. Functional universality means that the meanings are expressed by all human languages independent of the time and place where they are spoken.  The idea is that these meanings are so important for the functioning of a language as a tool of communication, that every language needs to express them.<br /><br />Cultural neutrality and functional universality guarantee two important aspects of basic words: their <i>stability</i> and their <i>resistance to borrowing</i>. Stability means that words expressing a basic concept are less likely to change their meaning or to be replaced by another word. An argument for this claim is the functional importance of the words — if the words are important for the functioning of a language, it would not make much sense to change them too quickly.  Humans are good at changing the meanings of words, as we can see from daily conversations in the media, where new words tend to pop up seemingly on a daily basis, and old words often drastically change their meanings.  But changing words that express basic meanings like "head", "stone", "foot", or "mountain" too often might give rise to confusion in communication. As a result, one can assume that words change at a different pace, depending on the meaning they express, and this is one of the core claims of lexicostatistics.<br /><br />Resistance to borrowing follows also from stability, since the replacement of words expressing basic meanings may again have an impact on our daily communication, and we may thus assume that speakers avoid borrowing these words too quickly. Cultural neutrality of concepts is another important point to guarantee resistance to borrowing. Words expressing concepts which play an important cultural role may easily be transferred from one language to another along with the culture. Thus, although it seems likely that every language has a word for "god" or "spirit" and the like (so the concept is to a certain degree functionally universal), the lack of cultural independency makes words expressing religious terms very likely candidates for borrowing, and it is probably no coincidence that words expressing religion and belief rank first in the scale of borrowability (<a href="http://bibliography.lingpy.org/?key=Tadmor2009">Tadmor 2009: 232</a>).<br /><br /><b>Lexical Replacement, Data Preparation, and Divergence Time Estimation</b><br /><br />Swadesh had further ideas regarding the importance of basic vocabulary. He assumed that the process of <i>lexical replacement</i>&nbsp;follows universal rates as far as the basic vocabulary is concerned, and that this would allow us to date the divergence of languages, provided we are able to identify the shared cognates.  In lexical replacement, a word <i>w₁</i> expressing a given meaning <i>x</i> in a language is replaced by a word <i>w₂</i> which then expresses the meaning <i>x</i>, while <i>w₁</i> either shifts to express another meaning, or completely disappears from the language. For example, older <i>thou</i> did in English was replaced by the plural form <i>you</i>, which now also expresses the singular. In order to search for cognates and determine the time when two languages diverged, Swadesh proposed a straightforward procedure, consisting of very concrete steps (compare <a href="http://bibliography.lingpy.org/?key=Dyen1992">Dyen et al. 1992</a>): <br /><ul><li>Compile a list of basic concepts (concepts that you think are culturally neutral and functionally universal; see <a href="http://concepticon.clld.org/">here</a> for a comparative collection of different lists that have been proposed and used in the past)</li><li>translate these concepts into the different languages you want to analyse</li><li>search for cognates between the languages in each meaning slot; if words in two languages are not cognate for a given meaning, then this points to former processes of lexical replacement in at least one of the languages since their divergence</li><li>count the number of shared cognates, and use some mathematics to calculate the divergence time (which has been independently calibrated using some test cases of known divergence times).</li></ul>As an example for such a wordlist with cognate judgments, compare the table in the first figure, where I have entered just a few basic concepts from Swadesh's standard concept list and translated them into four languages. Cognacy is assigned with help of IDs in the column at the right of each language column, but also further highlighted with different colors. <br /><br /><a href="http://4.bp.blogspot.com/-2RoFbOVUXu8/VmahRzx0III/AAAAAAAAAJY/TMqU_Evp6Pg/s1600/cognates-1.png" imageanchor="1"></a>Classical cognate coding in lexicostatistics<br /><b>Phylogenetic Approaches in Historical Linguistics</b><br /><br />Modern phylogenetic approaches in historical linguistics basically follow the same workflow that Swadesh propagated for lexicostatistics, the only difference being the last step of the working procedure. Instead of Swadesh's formula, which compared lexical replacement with radioactive decay and was based on aggregated distances in its core, character-based methods are used to infer phylogenetic trees.  Characters are retrieved from the data by extracting each cognate from a lexicostatistical wordlist and annotating the presence or absence of each cognate set in each language.<br /><br />Thus, while Swadesh's lexicostatistical data model would state that the words for "hand" in German and English were cognate, and also in Italian and French, but not in Germanic and Romance, the binary presence-absence coding states that the cognate set formed by words like English <i>hand</i> and German <i>Hand</i> is not present in Romance languages, and that the cognate set formed by words like Italian <i>mano</i> and French <i>main</i> is absent in Germanic languages. This is illustrated in the table below, where the same IDs and colors are used to mark the cognate sets as in the table shown above.<br /><br /><a href="http://4.bp.blogspot.com/-LfzpRrY_BBQ/VmahTTBXX8I/AAAAAAAAAJk/ZNWjZq08W0Y/s1600/cognates-2.png" imageanchor="1"></a>Presence-absence cognate coding for modern phylogenetic analyses<br />The new way of cognate coding along with the use of phylogenetic software methods has brought, without doubt, many improvements compared to Swadesh's idea of dating divergence times by counting percentages of shared cognates. A couple of problems, however, remain, and one should not forget them when applying computational methods to originally lexicostatistic datasets.<br /><br />First, we could ask whether the main assumptions of functional universality and cultural neutrality really hold. It seems to be true that words can be remarkably stable throughout the history of a language family. It is, however, also true that the most stable words are not necessarily the same across all language families. Ever since Swadesh established the idea of basic vocabulary, scholars have tried to improve the list of basic vocabulary items. Swadesh himself started from a list of 215 concepts (<a href="http://bibliography.lingpy.org/?key=Swadesh1950">Swadesh 1950</a>), which he then reduced to 200 concepts (<a href="http://bibliography.lingpy.org/?key=Swadesh1952">1952</a>) and then later to 100 concepts (<a href="http://bibliography.lingpy.org/?key=Swadesh1955">1952</a>). Other scholars went further, like Dolgopolsky (<a href="http://bibliography.lingpy.org/?key=Dolgopolsky1964">1964 [1986]</a>) who reduced the list to 16 concepts. The <a href="http://concepticon.clld.org/">Concepticon</a> is a resource that links many of the concept lists that have been proposed in the past.&nbsp;When comparing these lists, which all represent what some scholars would label "basic vocabulary items", it becomes obvious that the number of items that all scholars agree upon sinks drastically, while the number of concepts that have been claimed to be basic increases.<br /><br />An even greater problem than the question of universality and neutrality of basic vocabulary, however, is the underlying model of cognacy in combination with the proposed process of change.  Swadesh's model of cognacy controls for meaning. While this model of cognacy is consistent with Swadesh's idea of lexical replacement as a basic process of lexical change, it is by no means consistent with birth-death models of cognate gain and cognate loss if they are created from lexicostatistical data. In biology, birth-death models are usually used to model the evolution of homologous gene families distributed across whole genomes. If we use the traditional view according to which words can be cognate regardless of meaning, the analogy holds, and birth-death processes seem to be adequate in order to analyze datasets that are based on these <i>root cognates</i> (<a href="http://bibliography.lingpy.org/?key=Starostin1989">Starostin 1989</a>) or <i>etymological cognates</i> (<a href="http://bibliography.lingpy.org/?key=Starostin2013">Starostin 2013</a>). But if we control for meaning in the cognate judgments, we do not necessarily capture processes of gain and loss in our data. Instead, we capture processes in which links between word forms and concepts are shifted, and we investigate these shifts through the very narrow "windows" of pre-defined slots of basic concepts, as I have tried to depict in the following graphic.<br /><br /><a href="http://2.bp.blogspot.com/-ILItz0MUjQw/Vmak55Hh6YI/AAAAAAAAAJ0/yueeOxGBMng/s1600/rect8062-5-5-4-9-7.jpg" imageanchor="1"></a>Looking at kexical replacement through the small windows of basic vocabulary<br /><b>Conclusion</b><br /><br />As <a href="http://phylonetworks.blogspot.fr/2015/11/are-realistic-mathematical-models.html">David has mentioned before</a>: We do not necessarily need realistic models in phylogenetic research to infer meaningful processes. The same can probably be said about the discrepancy between our lexicostatistical datasets (Swadesh's heritage, which we keep using for practical reasons) and the birth-death models we now use to analyse the data. Nevertheless, I cannot avoid an uncomfortable feeling when thinking that an algorithm is modeling gain and loss of characters in a dataset that was not produced for this purpose. In order to model the traditional lexicostatistical data consistently, we would either (i) need explicit multistate-models in which concepts are a character and the forms represent the states (<a href="http://bibliography.lingpy.org/?key=Ringe2002">Ringe et al. 2002</a>, <a href="http://bibliography.lingpy.org/?key=Hamed2006">Ben Hamed and Wang 2006</a>), or (ii) we should directly turn to "root-cognate" methods. These methods have been discussed for some time now (<a href="http://bibliography.lingpy.org/?key=Starostin1989">Starostin 1989</a>, <a href="http://bibliography.lingpy.org/?key=Holm2000">Holm 2000</a>), but there is only one <a href="http://escholarship.org/uc/item/1331v899">recent approach by Michael et al. (forthcoming)</a> in which this is consistently tested. <br /><br /><b>References</b><br /><ul><li>Bergsland, K. and H. Vogt (1962): <b>On the validity of glottochronology</b>. <i>Curr. Anthropol.</i> 3.2. 115-153.</li><li>Bowern, C. and Q. Atkinson (2012): <b>Computational phylogenetics of the internal structure of Pama-Nguyan</b>. <i>Language</i> 88. 817-845.</li><li>Dolgopolsky, A. (1964): <b>Gipoteza drevnejšego rodstva jazykovych semej Severnoj Evrazii s verojatnostej točky zrenija</b> [A probabilistic hypothesis concering the oldest relationships among the language families of Northern Eurasia]. <i>Voprosy Jazykoznanija</i> 2. 53-63.</li><li>Dyen, I., J. Kruskal, and P. Black (1992): <b>An Indoeuropean classification</b><b>. A lexicostatistical experiment</b>. <i>T. Am. Philos. Soc.</i> 82.5. iii-132.</li><li>Ben Hamed, M. and F. Wang (2006): <b>Stuck in the forest: Trees, networks and Chinese dialects</b>. <i>Diachronica</i> 23. 29-60.</li><li>Hoijer, H. (1956): <b>Lexicostatistics</b><b>. A critique</b>. <i>Language</i> 32.1. 49-60.</li><li>Holm, H. (2000): <b>Genealogy of the main Indo-European branches applying the separation base method</b>. <i>J. Quant. Linguist.</i> 7.2. 73-95.</li><li>Kitchen, A., C. Ehret, S. Assefa, and C. Mulligan (2009): <b>Bayesian phylogenetic analysis of Semitic languages identifies an Early Bronze Age origin of Semitic in the Near East</b>. <i>Proc. R. Soc. London, Ser. B</i> 276.1668. 2703-2710.</li><li>Lee, S. and T. Hasegawa (2011): <b>Bayesian phylogenetic analysis supports an agricultural origin of Japonic languages</b>. <i>Proc. R. Soc. London, Ser. B</i> 278.1725. 3662-3669.</li><li>Pagel, M. (2009): <b>Human language as a culturally transmitted replicator</b>. <i>Nature Reviews. Genetics</i> 10. 405-415.</li><li>Ringe, D., T. Warnow, and A. Taylor (2002): <b>Indo-European and computational cladistics</b>. <i>T. Philol. Soc.</i> 100.1. 59-129.</li><li>Starostin, S. (1989): <b>Sravnitel'no-istoričeskoe jazykoznanie i leksikostatistika</b> [Comparative-historical linguistics and lexicostatistics]. In: Kullanda, S., J. Longinov, A. Militarev, E. Nosenko, and V. Shnirel'man (eds.): <i>Materialy k diskussijam na konferencii</i>[Materials for the discussion on the conference].1. Institut Vostokovedenija: Moscow. 3-39.</li><li>Starostin, G. (2013): <b>Lexicostatistics as a basis for language classification</b>. In: Fangerau, H., H. Geisler, T. Halling, and W. Martin (eds.): <i>Classification and evolution in biology, linguistics and the history of science. Concepts – methods – visualization.</i>. Franz Steiner Verlag: Stuttgart. 125-146.</li><li>Swadesh, M. (1950): <b>Salish internal relationships</b>. <i>Int. J. Am. Linguist.</i> 16.4. 157-167.</li><li>Swadesh, M. (1952): <b>Lexico-statistic dating of prehistoric ethnic contacts</b><b>. With special reference to North American Indians and Eskimos</b>. <i>Proc. Am. Philol. Soc.</i> 96.4. 452-463.</li><li>Swadesh, M. (1955): <b>Towards greater accuracy in lexicostatistic dating</b>. <i>Int. J. Am. Linguist.</i> 21.2. 121-137.</li><li>Tadmor, U. (2009): <b>Loanwords in the world’s languages</b><b>. Findings and results</b>. In: Haspelmath, M. and U. Tadmor (eds.): <i>Loanwords in the world's languages</i>. de Gruyter: Berlin and New York. 55-75.</li></ul><br /></div>
 </div>
</div>
<h3>December 7, 2015</h3>
<div class="news-item">
 <div class="date">06:48</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2015/12/recent-book-reviews.html">Recent book reviews</a></div>
  <div class="description"><br />In one of the earliest blog posts (<a href="http://phylonetworks.blogspot.com/2012/02/reviews-of-recent-books.html" target="_blank">Reviews of recent books</a>) I provided links to some book reviews. Recently, a few have appeared for Dan Gusfield's book: <i>ReCombinatorics: the algorithmics of ancestral recombination graphs and explicit phylogenetic networks</i> (2014. The MIT Press, Cambridge, MA).<br /><br />In addition to the three endorsements that appear as part of the publisher's blurb, a number of independent book reviews have appeared since its publication:<br /><br />(2014) <a href="http://www.computingreviews.com/review/Review_review.cfm?review_id=143064" target="_blank">Computing Reviews</a> Review#143064.<br /><br />Michael Sanderson (2015) <a href="http://www.jstor.org/stable/10.1086/682642" target="_blank">Quarterly Review of Biology</a> 90: 344-345.<br /><br />Luay Nakhleh (2015) <a href="http://epubs.siam.org/doi/abs/10.1137/15N97399X" target="_blank">SIAM Reviews</a> 57: 638-642.<br /><br />From the mathematical point of view, the reviews make it clear that this book is necessary because networks are very much part of the fringe of the computational sciences. Indeed, the challenge is to convince mathematicians that interesting mathematical problems exist with the the study of networks. In this sense, the main limitation of the book is its focus on the parsimony criterion for optimization, rather than statistical approaches to inference, which play such a large part in phylogenetic analyses.<br /><br />From the biological point of view, the principal issue seems to be the reliance of the book on the infinite sites model, which does not currently have wide applicability in phylogenetics (eg. mostly in population studies such as haplotype inference and association mapping).<br /><br />The ultimate goal for both computational end biological scientists is working out how to include recombination in the framework of other types of phylogenetic networks. A basic assumption of many phylogenetic analyses is that there has been no recombination. This is because recombination can destroy much of the evidence left by historically preceding processes, so that neither genotype nor phenotype data can reveal patterns and processes that pre-date the recombination events. In this sense, recombination becomes the reticulation process, rather than processes like hybridization or introgression.<br /><br /></div>
 </div>
</div>
</div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news." class="active">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/21 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 17:30:17 GMT -->
</html>
