<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=260 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:26:20 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="index66e2.html?q=node/119">54(6) December 2005</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 2 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>January 22, 2016</h3>
<div class="news-item">
 <div class="date">04:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137808293191">Graduate position: Liverpool_JMU.MolecularEvolutionEcology</a></div>
  <div class="description">PhD Studentship in Molecular Evolution and Ecology

Liverpool John Moores University - School of Natural Sciences and
Psychology

Duration of Studentship:  36 months.

There is an opportunity to apply for a fully funded PhD studentship at the
interface between Molecular Evolution and Ecology. Please note that due to
funding regulations the position is available to UK or EU nationals only.

Project title: Using biogeographic and phylogenetic patterns of Conophytum
to inform the conservation of this endemic genus of the Succulent Karoo

Project description:

The aim of this project is to understand the key drivers of the current
distribution and evolutionary divergence of one of the most diverse
plant genera of the Succulent Karoo (SK). In doing so we will be better
able to project the potential responses of the Conophytum to global
environmental change, towards the development of conservation measures
that reduce the extinction risk of these vulnerable taxa.

Lying within south-western corner of Africa, the SK biome is recognised
as one of the World&rsquo;s most important regions of floral biodiversity. Here
members of the Aizoaceae are especially well adapted to the arid desert
environment, occupying niche habitats. This is most evident in the
speciose dwarf succulent genus Conophytum, a SK specialist that shows a
remarkable degree of range-restricted endemism and 20%+ of taxa can be
considered as such [1]. The SK has also been identified as being at high
risk from the effects of climate change, with preliminary data suggesting
that dwarf succulents may be especially vulnerable [2-4]. However, neither
the underlying mechanisms that influence the distribution and associated
speciation of these succulents nor how they might respond to specific
environmental conditions is understood. We now have an opportunity to
address this by bringing together a range of complimentary techniques
using the genus Conophytum as a model.

The project will take a unique, multidisciplinary approach, combining
the expertise of the three supervisors (spatial modelling, molecular
phylogeny and evolution, biochemistry and succulent plant biology). In
addition, this study is only now possible because of the availability of
two unique resources at LJMU: (1) the student will have full access to
the most comprehensive database of ~3000 locality records covering all
taxa in the genus and (2) a genetic resource arising from a complete in
situ collection of all Conophytum taxa in the genus.

A three step approach to the work will be employed: (1) The existing
spatial database and an ensemble of Species Distribution Modelling
(SDMs) techniques (regression and machine learning models) available in
the BIOMOD2 library [5,6], run and calibrated within the R statistical
programming environment [7], will be used to quantify the role of climate
and other environmental variables on the current distribution of the 166
taxa in the Conophytum genus. Of particular interest will be a comparison
of the climatic niches of (a) 16 defined closely-related groups of taxa
of varying vulnerability to extinction within the genus [8,1] and (b)
point endemics (small vulnerable populations) versus those taxa with a
ubiquitous distribution. (Months 1-12)   (2) The Conophytum phylogeny
will be reconstructed using suitable genetic markers developed based on
Next Generation Sequencing from isolated DNA available from the in situ
collection at LJMU. This combined with the biogeographic and spatial
data will allow an assessment of the phylogeny in the context of the
climatic and environmental drivers. In addition, a methodology typically
applied to island endemic plants (see previous study by Kapralov et al.,
2013 in Hawaii [9]), will be applied to the island-like populations of
Conophytum on isolated inselbergs to understand the role of adaptation
during radiation. The Conophytum phylogeny will therefore act as the
foundation for further studies and will be used to map on it species
ecology, morphology, distribution, pigment composition, physiology,
and anatomy. This would allow us to better understand mechanisms
behind the evolution of the genus and to compare roles of adaptation
versus stochastic isolation for its prolific speciation. (Months 10-30)
(3) Future threats from climate change will be assessed using the SDMs
outlined in Step 1 incorporating a range of climate change scenarios to
project the impacts on the future distribution of the Conophytum taxa
[10-12]. Two time slices (2021 to 2050 and 2050 to 2080) will be used
with a range of Regional Climate Models (RCM) (CCSM3, HadCM3, ECHAM5)
and Special Report on Emissions Scenarios (SRES) (B1, A2, A1b) to provide
12 different climate change datasets to account for the RCM and SRES
uncertainty in the modelling. Gap analysis will be used to assess the
effectiveness of the current protected area network under future climates
[10]. (Months 24-36 - thesis submission)

The student will collaborate with a range of international researchers,
reinforcing research networks of the supervisory team: a word leading
biodiversity and climate change modelling research group at Universit
Joseph Fourier, Grenoble, France and 3 recognised global centres of
excellence in South Africa, Stellenbosch University, the University
of Cape Town (succulent phylogeny) and the South African National
Biodiversity Institute (SANBI).

Results from this study will better inform both the National Red List and
conservation planning in South Africa via existing links with SANBI (work
currently being developed as part of an Impact Case Study for REF2020).

The student will acquire a range of state-of-the-art techniques
including spatial modelling, molecular phylogenetics, and strong
statistical/computational skills.  References:

1.    Hammer, S.A. & Young, A.J. (2015). Conophytum: Ruschioideae,
      In: Hartman, HE.K. (Ed.), Illustrated Handbook of Succulent
      Plants, Aizoaceae A-E. Springer-Verlag. In Press

2.    Midgley, G.F. & Thuiller, W. (2007) Potential vulnerability of
      Namaqualand plant diversity to anthropogenic climate change.
      Journal of Arid Environments, 70, 615-628.

3.    Midgley, G.F., Hannah, L., Millar, D., Rutherford, M.C. &
      Powrie, L.W. (2002). Assessing the vulnerability of species
      richness to anthropogenic climate change in a biodiversity
      hotspot. Global Ecology and Biogeography, 11, 445-451.

4.    Young, A.J., Guo, D., Desmet, P.G., & Midgley, G.F. (in
      review). Biodiversity and climate change: risks to dwarf
      succulents in Southern Africa.

5.    Thuiller, W., Lafourcade, B., Engler, R., Arajo, M.B.,
      (2009). BIOMOD - a platform for ensemble forecasting of species
      distributions. Ecography, 32, 369-373.

6.    <a href="http://bit.ly/1OBH1oi">http://bit.ly/1OBH1oi</a>

7.    R Development Core Team, (2015). R, A language and environment
      for statistical computing. R Foundation for Statistical
      Computing, Vienna, Austria. URL <a href="http://bit.ly/1ZRgSdA.">http://bit.ly/1ZRgSdA.</a>

8.    Hammer, S.A. et al. (2002). Dumpling and his wife; new views of
      the genus Conophytum. EAE Creative Colour Limited.

9.    Kapralov, M.V., Votintseva, A.A., & Filatov, D.A. (2013).
      Molecular adaptation during a rapid adaptive radiation.
      Molecular Biology and Evolution, 30(5), 1051-59.

10.   Hodd, R.L., Bourke, D., & Sheehy Skeffington, M. (2014). Projected
      Range Contractions of European Protected Oceanic Montane Plant
      Communities: Focus on Climate Change Impacts Is Essential for
      Their Future Conservation. PLoS ONE 9(4): e95147.
      doi:10.1371/journal.pone.0095147

11.   Coll, J., Bourke, D., Sheehy Skeffington, M., Gormally, M., and
      Sweeney, J. (2014). Projected loss of active blanket bogs in
      Ireland. Climate Research, 59,103-115.

12.   Sharkey, N., Jones, M. and Bourke, D. (2013). Climate change
      impacts on woodland species: implications for the conservation of
      woodland habitats in Ireland, Biology and Environment: Proceedings
      of the Royal Irish Academy, 113B (3), 1-31. DOI:
      10.3318/BIOE.2013.20

Any specific eligibility requirements:

Please note that due to funding regulations the position is available
to UK or EU nationals only.

Candidates must hold a 1st Class Honours Degree and/or an appropriate
Master&rsquo;s degree.

See more details at <a href="http://bit.ly/1OBH4Ay">http://bit.ly/1OBH4Ay</a>

Deadline: 18th March 2016

In their application cover letter and CV, candidates are asked to include
and highlight their skills, experience and/or knowledge relevant to this
project; in particular:

Plant Ecology & Evolution

Species Distribution Modelling/Ecological Statistics

Geographic Information Systems

Phylogenetics/Molecular Ecology

Name/email/telephone number of someone applicants can contact with
enquiries:

In the first instance make an informal enquiry by e-mail to project
supervisors by sending a CV with contact details of two academic
referees. Your e-mail should have as its subject &ldquo;Application for
PhD&rdquo;. It is essential that you give the grades of all degrees and state
your nationality.

Dr Maxim Kapralov email: m.v.kapralov@ljmu.ac.uk

Dr David Bourke email: d.o.bourke@ljmu.ac.uk

Professor Andy Young email: a.j.young@ljmu.ac.uk

Dr Maxim Kapralov

Senior Lecturer || School of Natural Sciences and Psychology ||
Liverpool John Moores University || Room 2.07 ||  Life Sciences
Building || Byrom Street || Liverpool || L3 3AF || United Kingdom
|| t: +44 (0)151 904 6313 || email: m.v.kapralov@ljmu.ac.uk ||
<a href="http://bit.ly/1Pbezum">http://bit.ly/1Pbezum</a>

&ldquo;Kapralov, Maxim&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137808292856">Postdoc: UNewHampshire.SoilMicrobialEvolution</a></div>
  <div class="description">Postdoctoral Position in Soil Microbial Evolution

Contact: Dr. Serita Frey, Department of Natural Resources & the
Environment, University of New Hampshire, Durham, NH USA

(Questions can also be directed to Dr. Anne Pringle, University of
Wisconsin-Madison: anne.pringle@wisc.edu)

Our laboratory aims to understand connections between microbial
community structure and ecosystem function. We document the impacts of
environmental change (climate warming, nitrogen deposition,
biodiversity loss, invasive species) on the diversity, community
composition, and function of the soil microbial community, and test
whether shifts in the community subsequently influence ecosystem
functions. A recent focus is on anthropogenic drivers of microbial
evolution.

This two-year position will focus specifically on fungal evolution
within global change contexts, with an emphasis on how fungi evolve in
response to biotic invasions, soil warming, and nitrogen addition. The
candidate will have the flexibility to explore questions that fall
within this general topic area, while building on previous research
conducted in the Frey and Pringle Laboratories. The candidate is
expected to have strong interests and experience in evolution and
ecology. Expertise in cultivation-based and genomic/metagenomic
analyses is highly desirable. The candidate will be expected to work
independently, but also cooperatively with other members of the lab and
with the Pringle Laboratory at the University of Wisconsin, Madison. A
Ph.D. degree in evolution, ecology, natural resources, microbiology, or
related field, along with relevant research experience is required.
Applications will be reviewed until the position is filled.

To apply please send the following items in a single PDF file to Serita
Frey (serita.frey@unh.edu): letter of interest/experience, CV, and the
names and contact information of three professional references.

Anne Pringle, Ph.D.
Associate Professor
Departments of Botany and Bacteriology
University of Wisconsin, Madison
new email:  anne.pringle@wisc.edu
<a href="http://bit.ly/1ZRgQ5v">http://bit.ly/1ZRgQ5v</a>

Anne Pringle 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:30</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137807897071">Other: Botany Advocacy Leadership Grant</a></div>
  <div class="description"><p>The American Society of Plant Taxonomists (ASPT) and Botanical Society of
America (BSA) have teamed up to craft a NEW grant opportunity - the Botany
Advocacy Leadership Grant!

Please find the call for proposals at the following link:
<a href="http://bit.ly/1S96Iz3">http://bit.ly/1S96Iz3</a>

One successful applicant for this award will receive $1,000 toward
developing new (or enhancing existing!) botanical advocacy projects - apply
or encourage others you know to submit an application.

DEADLINE: 16 March 2015


gostelm@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:15</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137807513966">Postdoc: UNevada_Reno.CognitiveFitness</a></div>
  <div class="description">POSTDOC POSITION IN COGNITIVE ECOLOGY

I am looking for a postdoc to join my lab at the University
of Nevada, Reno (<a href="http://bit.ly/1RYnnHe;">http://bit.ly/1RYnnHe;</a>
<a href="http://on.fb.me/1ZGBjop">http://on.fb.me/1ZGBjop</a>) to work on an NSF
funded project investigating fitness consequences of individual variation
in spatial memory. The project uses RFID technology to investigate whether
individual variation in spatial learning abilities is associated with
variation in fitness in free-living food-caching mountain chickadees in
two environments (different elevations) hypothesized to be different in
spatial memory demands associated with food caching.

All fieldwork is in the rugged mountain conditions V in the beautiful
Sierra Nevada along an elevation gradient from ca. 5,000 to 8,000 feet in
the Lake Tahoe National Forest (about 40 min drive from Reno). The postdoc
will be expected to work in the field throughout the year using a 4-wheel
drive vehicle when there is no snow and a snowmobile during the winter.

The project generates large amount of data (more than 100,000 bird visits
at just one spatial array with 8 feeders, per two weeks of testing),
so experience with analyzing and manipulating large data sets using
R and potentially other programming languages (e.g. Perl) is highly
desirable. Experience working with birds and with mist-netting is also
helpful.

The initial appointment will be for 1 year, which may be renewed based
on performance up to 3 years. If interested, please email directly to me
(Vladimir Pravosudov) at vpravosu@unr.edu.



Vladimir V Pravosudov 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:15</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137807514291">Job: CRITFC_Idaho.FishGenetics</a></div>
  <div class="description">Fisheries Geneticist - Columbia River Inter-Tribal Fish Commission
(CRITFC)

<a href="http://bit.ly/1ZGBjEF">http://bit.ly/1ZGBjEF</a>

Salary: $55,952-$72,737 DOQ, plus benefits

Closes on: March 15, 2016

The Columbia River Inter-Tribal Fish Commission (CRITFC) is seeking
Fisheries Geneticists with experience in population genomics,
association mapping, and mixed stock analysis. Positions are part of
the Fishery Science Department, but will be located with the genetics
group at the Hagerman Fish Culture Experiment Station in Hagerman, ID.
This research group is involved in testing conservation, evolution, and
ecological theories related to salmonids and other fishes. The employee
will work under the Lead Geneticist, in association with CRITFC
geneticists and technicians, as well as staff of the Fishery Science
Department in Portland, OR. Efforts will focus on applying empirical
genetics/genomics data to address questions related to conservation and
recovery of steelhead, Chinook, sockeye, and coho salmon, sturgeon,
lamprey, and other fishes of the Columbia River Basin.

See more at: <a href="http://bit.ly/1ZGBjEF">http://bit.ly/1ZGBjEF</a>


Shawn Narum 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:15</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137806084901">Graduate position: SwanseaU.CampylobacterEvolution</a></div>
  <div class="description">PhD studentship in Campylobacter Genomics and Evolution (2016-2019)


Research will be conducted under the direct supervision of Prof Samuel
K. Sheppard, and based in the Department of Medical Microbiology and
Infectious Diseases at the Swansea University Medical School (Wales,
United Kingdom).

Project description:

The successful applicant will join a multidisciplinary group focusing
on population biology and evolution of bacterial pathogens of public
health importance. They will investigate the genetic basis and ecology
of Campylobacter by explaining the factors involved in generating
and maintaining genotypic and phenotypic diversity in among isolates
in the primary reservoir of human disease, poultry. Comparative
genomics and functional characterisation approaches will examine the
genetic structuring and answer fundamental questions about how genetic
variation and genome plasticity influence adaptation and the evolution
of pathogenicity. Bioinformatics will be exploited for the design
of vaccines for the control of Campylobacter in broiler chickens on
farm. They will be responsible for culturing isolates, extracting DNA for
genetic characterisation, analysing genomic data and carrying molecular
and functional characterisation in the lab. The student will also spend
a three month placement at Ridgeway Biologicals Ltd where he/she will
be directly involved in the manufacture of autogenous vaccines prior to
testing on farm.

Requirements:

Enthusiasm and practical experience in microbiology, molecular techniques
(PCR, sequencing) and computer based genetic analysis, is necessary but
training and support will be provided to strengthen these areas. The
successful candidate will be highly motivated, creative, independent and
have a good degree in molecular biology, microbiology, bioinformatics,
genetics or ecology. Previous experience in a molecular biology laboratory
or in bioinformatics research is also strongly desirable. Good English
writing and oral skills are essential.

Details:

- The duration of a studentship is 3 years (2016-2019).
- The position is open until filled, but a preferred deadline for
  application is 1st March 2016, to allow for registration and the
  project to start ASAP.
- Salary will be commensurate with regular UK postgraduate stipends,
  i.e. ??16,450 (circa 21,000) per annum tax-free.
- Please contact Samuel Sheppard (s.k.sheppard@swansea.ac.uk) for
  informal inquiries or applications (please attach CV and describe
  motivation).

Application:

To apply by e-mail, please describe motivation and attach a
CV to sheppardlab@gmail.com.More details about the laboratory:
<a href="http://bit.ly/1Bg5c5r">http://bit.ly/1Bg5c5r</a>


Dr. Guillaume M????ric
NISCHR Research Fellow

Swansea University Medical School
Medical Microbiology and Infectious Disease

ILS1 Room 140 (MRC CLIMB Centre)
Singleton Campus, Swansea SA2 8PP
United Kingdom
~
Sheppard Laboratory: <a href="http://bit.ly/1Bg5c5r">http://bit.ly/1Bg5c5r</a>
MRC CLIMB: <a href="http://bit.ly/1t25Gr3">http://bit.ly/1t25Gr3</a>

&ldquo;MERIC G.&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137805377411">Graduate position: OklahomaStateU.GrassEvolution</a></div>
  <div class="description">Graduate position:  Oklahoma State University, Evolutionary and
quantitative genetics of branching in grasses

Research focus:  We are seeking a highly motivated student to conduct
research on evolutionary and quantitative genetics of branching in
grasses, as part of an NSF Plant Genome grant focused on tillering in
panicoid grasses.  The primary objective of this research is to use three
panicoid grasses, maize, sorghum, and Setaria, to identify components of
the gene regulatory network controlling branching.  Our lab is focused on
Setaria, combining traditional QTL mapping with expression QTL studies,
mutant analysis, and modeling to understand the genetic components of
branching and their responses to environmental stimuli such as shading.
There is considerable flexibility for graduate projects within this
broad theme.

Essential qualifications include masters degree in genetics and/or
molecular biology, experience in molecular lab techniques, experience in
growing plants and analyzing them using microscopic and other methods,
and excellent writing skills.  Desirable qualifications include experience
with quantitative and phylogenetic analysis software, and programming
in R and/or Python.

Salary:  Stipend will be $21,504 per calendar year plus tuition and
health benefits.  A mixture of TA and RA positions over the course of
the Ph.D. will allow you to balance valuable teaching experience with
research time.

Application deadline:  February 29th, 2016; but applications will be
reviewed as they arrive.

Application materials:  Please send a brief cover letter, resume,
transcripts, GRE scores, and contact information for two references to
andrew.doust@okstate.edu by or before the deadline to be considered for
this position.

Andrew Doust
Department of Plant Biology, Ecology, and Evolution
Oklahoma State University
Physical Sciences 301
Stillwater, OK 74078

Phone: 405-744-9559
Email: andrew.doust@okstate.edu

Websites:
<a href="http://bit.ly/1JWiZBH">http://bit.ly/1JWiZBH</a>
<a href="http://bit.ly/1zi2LzF">http://bit.ly/1zi2LzF</a>
<a href="http://bit.ly/1JWiXtl">http://bit.ly/1JWiXtl</a>




&ldquo;Doust, Andrew&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137805377171">Course: Barcelona.InferenceInEvolBiogeography.Nov28-Dec2</a></div>
  <div class="description"><p>Course Announcement:

Nick Matzke and Dan Warren will be teaching a course entitled &ldquo;Model-based
Statistical Inference in Ecological and Evolutionary Biogeography&rdquo; in
Barcelona from November 28 to Dec 2, 2016.

This course will cover the theory and practice of widely used methods in
evolutionary and ecological biogeography, namely ecological niche modelling
/ species distribution modelling, and ancestral range estimation on
phylogenies.

The course will cover both the practical challenges to using these
techniques (the basics of R, obtaining and processing geographical
occurrence data from GBIF, setting up and using the models), and the
assumptions that various models and methods make.

R packages we will use include rgbif, dismo, ENMTools, and BioGeoBEARS.

Finally, this course will introduce several new approaches being developed
by the instructors for linking ecological and evolutionary models.

For more details or to enroll, please see the Transmitting Science web site

<a href="http://bit.ly/1nCHjCP">http://bit.ly/1nCHjCP</a>


dan.l.warren@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:30</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137805008271">Postdoc: ColdSpringHarborLab.EvolutionaryGenomics</a></div>
  <div class="description">Position Description:

Postdoctoral Positions in Computational Genomics are available in Dr.
Adam Siepel&rsquo;s research group in its new location at the Simons Center
for Quantitative Biology (SCQB) at Cold Spring Harbor Laboratory
(CSHL). The Siepel Group specializes in the development of
probabilistic models, algorithms for inference, prediction methods, and
application of these methods in large-scale genomic data analysis. Of
particular interest is research relevant to existing, NIH supported
projects in (1) human population genomics, including demography
inference using Bayesian coalescent-based methods, inference of natural
selection on regulatory and other noncoding sequences, and prediction
of fitness consequences for noncoding mutations; and (2)
transcriptional regulation in mammals and Drosophila, including the
estimation of rates and patterns of transcriptional elongation from
GRO-seq data, prediction of transcription factor binding sites from
DNase-seq data, and regulatory network inference based on joint
patterns of transcription and binding in inducible systems. The
research on transcriptional regulation will continue to be carried out
in close collaboration with Dr. John Lis at Cornell University.

Relevant recent papers include the following:

Kuhlwilm M, Gronau I, Hubisz MJ, de Filippo C, Prado J, et al. Ancient
gene flow from modern humans into Siberian Neandertals. Nature (in
press), 2015.

Danko CG, Hyland SL, Core LJ, Martins AL, Waters CT, et al.
Identification of active transcriptional regulatory elements from
GRO-seq data. Nat. Methods. 2015. 12(5):433-438.

Rasmussen MD, Hubisz MJ, Gronau I, Siepel A. Genome-wide inference of
ancestral recombination graphs. PLOS Genet. 2014. 10(5): e1004342.

Freedman AH, Gronau I, Schweizer RM, Ortega-Del Vecchyo D, Han E, et
al. (30 co-authors). Genome sequencing highlights the dynamic early
history of dogs. PLOS Genet. 2014. 10(1):e1004016.

Core LJ, Martins AL, Danko CG, Waters CT, Siepel A, et al. Analysis of
nascent RNA identifies a unified architecture of initiation regions at
mammalian promoters and enhancers. Nat. Genet. 2014. 46(12):1311-1320.

Capra JA, Hubisz MJ, Kostka D, Pollard KS, Siepel A. A model-based
analysis of GC-biased gene conversion in the human and chimpanzee
genomes. PLOS Genet. 2013. 9(8):e1003684.

Arbiza L, Gronau I, Aksoy BA, Hubisz MJ, Gulko B, Keinan A, Siepel A.
Genome-wide inference of natural selection on human transcription
factor binding sites. Nat. Genet. 2013. 45(7):723-729.

Gronau I, Arbiza L, Mohammed J, Siepel A. Inference of natural
selection from interspersed genomic elements based on polymorphism and
divergence. Mol. Biol. Evol. 2013. 30(5):1159-1171.

Gronau I, Hubisz MJ, Gulko B, Danko CG, Siepel A. Bayesian inference of
ancient human demography from individual genome sequences. Nat. Genet.
2011. 43(10):1031-1034..

The ideal candidate will be enthusiastic, highly motivated and
independent; will have a strong background in comparative genomics,
population genomics, or transcriptional regulation, a serious interest
in molecular biology and genetics combined with good skills in
mathematics and computer science, and a strong record of accomplishment
in research. Postdoctoral associates in the group are expected to
participate in both methods development and data analysis, to be
comfortable in a fast-moving, interactive, and collaborative research
environment, and to actively publish and present results.

Required Qualifications:

Ph.D. in bioinformatics, computer science, statistics, genetics,
molecular biology, applied mathematics, or a related field;  Research
experience (with first-author publications) in computational genomics
or a closely related field;  Fluency in probabilistic modeling and
computational statistics;  Proficiency in programming, ideally in C or
C++, as well as in scripting languages such as python, Perl, mat lab,
or R; should be comfortable in a Linux environment, with large data
sets, computer clusters, and databases.

Job Location: CSHL, One Bungtown Road, Cold Spring Harbor, NY 11724

TO APPLY: visit the CSHL Careers website at <a href="http://www.cshl.edu/careers.">www.cshl.edu/careers.</a>
Position Number:01229-R.

&ldquo;Gill, Irene&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:30</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137805008746">Postdoc: IndianaU.EvoDevo</a></div>
  <div class="description">Postdoctoral position: Genetics and evolution of developmental
plasticity, Ragsdale Lab, Indiana University

A postdoctoral position is available in the laboratory of Erik
Ragsdale, Department of Biology, Indiana University, Bloomington. Our
lab investigates the genetic regulation and evolution of developmental
plasticity, specifically a polyphenism that involves novel
morphological and ecological traits. The focal species of this research
is the nematode Pristionchus pacificus, an emerging model system for
evolutionary developmental genetics, which our lab studies using
forward and reverse genetics, transgenics, and genome-wide approaches.

P. pacificus alternatively develops into a microbe-feeding or a
predatory morph in response to different environmental cues. Recent
work has begun to characterize the genes that make up a developmental
switch for the polyphenism, providing an entry point into the genetics
of a morphological dimorphism in a tractable animal model. Moreover,
this species is embedded in a solid phylogenetic infrastructure with a
broad range of feeding morphologies and ecologies. Given the laboratory
availability of many species and populations of Pristionchus, this
system offers a unique opportunity to discover how genetic factors and
their interactions evolve to modify developmental plasticity.

The goal of the project is to determine the significance of an
increasingly understood polyphenism mechanism in evolutionary process.
Approaches will involve population genetics, evolutionary modeling,
and/or experimental evolution. For the position, we seek a collegial
and intellectually driven individual with a recent Ph.D. in
evolutionary biology or a related field. A strong background in
population genetics and/or evolutionary modeling is essential.
Molecular biology skills are a plus but relevant training will be
provided as necessary. The position will be funded for two years, with
the potential to extend for an additional year. Salary will be
commensurate with experience, and full benefits are included.

To apply, please submit (i) a letter of application, (ii) a full CV,
(iii) a statement of research interests, and (iv) contact information
for three references electronically to
(<a href="http://bit.ly/1Vd3Pg6">http://bit.ly/1Vd3Pg6</a>). Review of applications
will start immediately and will continue until the position is filled.
The exact start date is flexible. Inquiries about the position can be
directed to Erik Ragsdale (ragsdale@indiana.edu). Additional
information about research in the Ragsdale lab can be found at:
<a href="http://bit.ly/1nCEWzL">http://bit.ly/1nCEWzL</a>

Bloomington is a vibrant college town located in scenic southern
Indiana, close to several natural parks and wilderness areas, and it
enjoys a local culture exceptionally rich in music, art, and theater.

Indiana University is an Equal Employment and Affirmative Action
employer and a provider of ADA services. All qualified applicants will
receive consideration for employment without regard to age, ethnicity,
color, race, religion, sex, sexual orientation or identity, national
origin, disability status, or protected veteran status.


Erik J. Ragsdale, PhD
Assistant Professor of Biology
Indiana University
Bloomington, IN 47405, USA
<a href="http://bit.ly/1Vd3Pwk">http://bit.ly/1Vd3Pwk</a>

&ldquo;Ragsdale, Erik&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:00</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137804240321">Other: Evolution2016.Art.Extended</a></div>
  <div class="description"><p>NOTICE: THE OPEN CALL HAS BEEN EXTENDED UNTIL JAN. 31.

Attention artists! 

The 2016 Evolution conference (the annual meeting of theSociety for
the Study of Evolution, theSociety of Systematic Biologists, and
theAmerican Society of Naturalists) will be held from June 17-21 in
beautiful Austin, TX. In conjunction with the meeting, the societies
and a local art gallery (Art.Science.Gallery) are co-sponsoring an
evolution-themed art exhibit. The exhibit will provide an opportunity for
meeting attendees and others to showcase their creative talents, increase
the visibility of the meeting and the societies to the local community,
and provide opportunities for public education about evolutionary
biology through the exciting lens of visual arts. The exhibit will run
at the Art.Science.Gallery leading up to, and during, the conference
and a selection of the works will be on display at the Austin Convention
Center as a pop-up exhibit during the opening reception for the meeting.

Art.Science.Gallery has issued an open call seeking submissions for this
exhibition. Works may explore (but are not limited to) the sub-disciplines
within evolutionary biology, notable evolutionary biologists, current
research topics, important discoveries and concepts, and the history of
evolutionary thought. You need not be a member of any of the societies
to submit something for consideration.

Pass the word along and, if you&rsquo;re at all artistically inclined, consider
submitting something. Additional details can be found on our permanent
meeting website (<a href="http://bit.ly/1PNgrcn">http://bit.ly/1PNgrcn</a>) under the &lsquo;News&rsquo; heading, or
on Art.Science.Gallery&rsquo;s website (<a href="http://bit.ly/1SDywfn">http://bit.ly/1SDywfn</a>).

The open call closes on Jan. 31, 2016.

Sincerely,
Your Evolution 2016 organizers


howard.rundle@uottawa.ca


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137803824306">Other: Peregrine Falcon article</a></div>
  <div class="description"><p>&ndash;=_7d50f8c236b095580e0b92993dbe763a
Content-Transfer-Encoding: 8bit
Content-Type: text/plain; charset=UTF-8

 

Dear evoldir community,

I&rsquo;m looking for a paper entitled: Genetic evidence for global dispersal
in the peregrine falcon (Falco peregrinus) and affinity with the taita
falcon (Falco fasciinucha). Journal of Raptor Research 48(1):44-53.
2014. If anybody here has it on pdf I would be grateful if you can send
it to me at jfo@tinet.cat.

Thank you very much,

Joan

-
via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137803824216">Conference: Beijing.PlantEvoDevo.May15-19</a></div>
  <div class="description"><p>15-19 May 2016

37th New Phytologist Symposium: Plant developmental evolution

<a href="http://bit.ly/1PIu8GQ">http://bit.ly/1PIu8GQ</a>

Beijing, China

Travel grant submission deadline: Thursday 25th February 2016

Poster abstract submission deadline: Thursday 17th March 2016

The relatively new field of plant evolutionary-developmental biology
(&lsquo;evo-devo&rsquo;) seeks to understand how and why plant morphological
characters have evolved to produce the tremendous diversity of form in
living plants. This meeting will draw together researchers in plant
evo-devo for exchange of ideas, current research, and discussion of
future directions for the field.

Sessions: floral development; vegetative meristems, leaves, and
inflorescences; plasticity and life history evo-devo; selected poster
abstract talks.

Eighteen leading scientists will speak at the Symposium. We hope that
this will stimulate focused discussion and the exchange of ideas at
what will be a relatively small (around 120 delegates) and informal
meeting. There will be a poster session, selected talks, discussion and
a conference dinner.

Keynote speakers:
Beverley Glover, University of Cambridge, UK; 
Mark Rausher Duke University, USA; 
Miltos Tsiantis, Max Planck Institute for Plant Breeding Research, Germany.


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:30</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137803405051">Graduate position: Oslo.Bioinformatics</a></div>
  <div class="description"><a href="http://bit.ly/1lBnPws">http://bit.ly/1lBnPws</a>


Description follows below:


 position as PhD Research fellow (postition code 1017) is available at
 Department of Informatics.

The fellowship will be for a period of 3 years, with no compulsory
work or for a period of 4 years, with 25 % compulsory work (teaching
responsibilities at the department) contingent on the qualifications of
the candidate and the teaching needs of the department. Starting date
no later than 1 October 2016. No one can be appointed for more than one
fixed-term period at the same institution.

The research fellow will take part in the Centre for Computational
Inference in Evolutionary Life Sciences (CELS). CELS is a
multidisciplinary centre within life sciences comprising strong research
environments in biology and statistical and computational methods for data
analysis. CELS addresses fundamental biological questions related to the
genotype-phenotype paradigm, how the environment affects genome evolution
and development of methods for integrated analysis of genomic and other
biotic and abiotic data. A main goal is to develop interdisciplinary
research activities generating science of true interest for computer
scientists, statisticians and biologists.

Project description:

Metagenomics is the study of the genetic material in a sample containing a
mix of different microbial organisms. These samples may have be obtained
from such diverse locations as the vertebrate gut (e.g. human or fish),
from the depths of the oceans, from the soil somewhere on earth and
so forth. Recent advances in DNA sequencing technologies have enabled
very large amounts of such data to be obtained for analysis. Extracting
biological meaningful information from these huge datasets involve hard
computational challenges that can only be solved by good algorithms
and efficient parallelization. Interpreting the data correctly
can advance science by a better understanding of, for example, the
diversity of ecosystems or how the immune system responds to exposure
to microorganisms.

We are seeking a highly motivated candidate to develop new or improved
algorithms or

tools for metagenomics with improved functionality, accuracy or speed that
will be valuable for the scientific community. This is an opportunity to
develop open source software in areas like error detection and removal,
clustering, searching, classification, genome assembly as well as other
areas of metagenomics. We have already contributed with tools like Swarm
(<a href="http://bit.ly/1nCvufM">http://bit.ly/1nCvufM</a>) and VSEARCH. The successful
candidate will be embedded in a vibrant bioinformatics environment
connected to two Norwegian Centres of Excellence, and have access to
first-rate computational infrastructure. The position is associated with
the Centre for Computational Inference in Evolutionary Life Science (CELS)
at UiO. The CELS research environment offers extensive experience with
high-throughput sequencing and assembly of large genomes such as that
of Atlantic cod (<a href="http://bit.ly/1lBnPwt">http://bit.ly/1lBnPwt</a>). The candidate
will be supervised by an interdisciplinary team of scientists and will
collaborate with other PhD research fellows within CELS.

Requirements:

The Faculty of Mathematics and Natural Sciences has a strategic ambition
of being a leading research faculty. Candidates for these fellowships
will be selected in accordance with this, and expected to be in the
upper segment of their class with respect to academic credentials.

Applicants must hold a Master&rsquo;s degree or equivalent in bioinformatics or
computer science, and need to document good software development skills.

The candidate would need to learn genomics/metagenomics during the
project, but no formal prior knowledge of biology is required.

The purpose of the fellowship is research training leading to the
successful completion of a PhD degree.

The fellowship requires admission to the PhD programme at the Faculty of
Mathematics and Natural Sciences. The application to the PhD programme
must be submitted to the department no later than two months after taking
up the position. For more information see:

<a href="http://bit.ly/1m8QEOW">http://bit.ly/1m8QEOW</a>

<a href="http://bit.ly/1esINYa">http://bit.ly/1esINYa</a>

A good command of English is required.

<a href="http://bit.ly/1pVJquh">http://bit.ly/1pVJquh</a>

Salary:

Position code 1017, Pay grade: 50 - 57 (NOK 430 500  483 700 per year)

The application must include:

Application letter
CV (summarizing education, positions and academic work - scientific publications)
Copies of educational certificates, transcript of records and letters of recommendation
Documentation of English proficiency
<a href="http://bit.ly/1pVJquh">http://bit.ly/1pVJquh</a>

List of publications and academic work that the applicant wishes to
be considered by the evaluation committee Names and contact details
of 2-3 references (name, relation to candidate, e-mail and telephone
number) Foreign applicants are advised to attach an explanation of their
University&rsquo;s grading system. Please remember that all documents should
be in English or a Scandinavian language.

In accordance with the University of Oslo&rsquo;s equal opportunities policy,
we invite applications from all interested individuals regardless of
gender or ethnicity.

UiO has an agreement for all employees, aiming to secure rights to
research results a.o.

Region:
Oslo
Job type:
Contract
Working hours:
Full-time
Working days:
Day
Application deadline:
11. February, 2016
Location:
Oslo
Reference number:
2015/3967
Home page:
<a href="http://bit.ly/1lBnRVm">http://bit.ly/1lBnRVm</a>
Contacts:
Professor Torbjrn Rognes 
Telephone: +47 22 84 47 87
Questions regarding EasyCruit, contact HR Officer Helene Beate Jansen 
Telephone: +47 22857196


Bastiaan Star 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:30</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137803404421">Graduate position: GuangxiU.2.PlantGenomicsBiodiversity</a></div>
  <div class="description">two MSc/PhD student positions available in Plant Genomics & Biodiversity
at the Plant Ecophysiology & Evolution Group, Guangxi University,
Nanning, China

The Plant Ecophysiology and Evolution Group at Guangxi University
(Nanning, China) is seeking two highly motivated and productive
International MSc or PhD students to engage in &frac34;-year projects in Plant
Genomics, Molecular Systematics, Biogeography and Plant Taxonomy. Multiple
topics are available and can be further determined upon application,
including one or a combination of the following themes:

(1) Plant Genomics - Understanding the geographic distribution and
evolution of genetic diversity between species and within populations;
understanding the role of isolation in the establishment of genetic
divergence and speciation.

(2) Plant Biogeography & Evolution - Temporal and spatial reconstruction
of patterns of plant diversification in Asia and the Indian Ocean
region in selected families and genera. Of particular interest to us
are the roles of morphological/molecular change in adaptation and plant
speciation on oceanic islands, where environments undergo substantial
change over the course of an island&rsquo;s lifespan.

(3) Plant Taxonomy - There are several exciting opportunities for
students to take part in the revision and reconstruction of a new
regional Flora. Projects will involve training in taxonomy, nomenclature
and fieldwork, but also include training in molecular systematics,
phylogenetics and the most recent developments in the field.

Students will enroll in the International Graduate Program at the
College of Forestry for 3-4-year projects, with complete waivers for
tuition- and housing-fees. Housing will be provided in the International
Student dormitories of the campus in Nanning. The first year consists
of orientation and study of the Chinese language through courses. The
following years will be filled with fieldwork, professional courses and
the research project.

Applications are now invited for an intended starting date of projects in
September 2016. The application deadline is March 31, 2016. Fellowships
come with an ample monthly allowance for the full duration of the
project. Applicants with a proven track record of successful publication
in peer-reviewed journals will have an advantage in applying.

Opportunities for the development of individual research projects
as well as collaborative work, exist within our lab and with external
groups. Candidates will be integrated in a growing and multidisciplinary
team of Chinese and foreign researchers, providing a creative and
stimulating research environment, and will work as part of a team studying
genomics, evolution and taxonomy of selected taxa in Asia and the Indian
Ocean region. Prior knowledge of Chinese is not mandatory as courses
form an integral part of the curriculum, and the working language in
our research groups is English. Projects will be supervised by Associate
Prof. Joeri S. Strijk (jsstrijk@gxu.edu.cn).

Our research team is part of the College of Forestry, under the
State Key Laboratory of Conservation and Utilization of Subtropical
Agro-bioresources, which houses a wide range of laboratories and
research groups. We are located in the heart of the large Nanning
city campus, with easy access to on-campus housing and day-to-day
facilities for life on campus. Guangxi University (>25,000 students)
in subtropical Nanning is part of the National 211 University Scheme
of China (<a href="http://bit.ly/1pqTsrD">http://bit.ly/1pqTsrD</a>) and aims to become
a major research hub in Southern China. The city forms the natural
gateway to South-East Asia, with Vietnam a mere 300kms away and direct
international connecting flights available to most major cities in Asia
from the new Nanning International Airport.

Please visit our website for more information on our group, it&rsquo;s members
and their research:

<a href="http://bit.ly/1m7jVG9">http://bit.ly/1m7jVG9</a>

To apply for a position, please email a statement of research interests
and goals, a curriculum vitae, and the email addresses of three references
directly to:

Associate Prof. Joeri S. Strijk (jsstrijk@gxu.edu.cn).

Review of applications will commence immediately.



Dr. Joeri S. Strijk
Associate Professor
Plant Ecophysiology & Evolution Group
State Key Laboratory for Conservation and Utilization of Subtropical Agro-bioresources
College of Forestry, Guangxi University
Nanning, Guangxi 530005
PR China

Joeri Sergej Strijk 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>January 21, 2016</h3>
<div class="news-item">
 <div class="date">04:41</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137744349111">Other: BlackHills.SD.REU.Summer2016</a></div>
  <div class="description"><p>&ndash;_000_BLUPR01MB129980329800CADBDBABF1FFB4C20BLUPR01MB1299prod_
Content-Type: text/plain; charset=&ldquo;iso-8859-1&rdquo;
Content-Transfer-Encoding: quoted-printable

Summer 2016 REU Position: Black Hills, South Dakota.


Project Title:

Individual variation in seasonal timing of breeding in dark-eyed juncos- Katie Needham and Jessica Graham (Ph.D. Candidates, under advisement of Dr. Timothy Greives, Biological Sciences Department, North Dakota State University).


Description:

This project will focus on the driving forces behind why some individuals breed earlier or later in the season than the majority of the population. Previous research with songbirds has shown that earlier breeding individuals have higher reproductive success and optimize timing with peak food abundance. While reproductive success is higher for individuals that breed earlier, selection is not shifting towards earlier reproduction, suggesting there are costs associated with breeding early, but the underlying physiological mechanisms of individual variation require further exploration. Working with the population of Dark-eyed Juncos near Hanna, South Dakota, we hope to uncover 1) What ecological factors affect increased reproductive success? 2) How variation in parental care affects reproductive success? Possible REU projects will focus on the nesting period and include observing how bacteria (presence-absence manipulation), parental care, or other ecological factors of interest 
 that may affect reproductive success across the season.


Expectations:

REU will be expected to develop an independent project related to the ongoing research with guidance from mentors. Student will take leadership roles in all aspects of the study; they design the study, prepare a proposal, collect data and perform analysis, interpret the results, prepare a written manuscript, and complete an oral presentation at their home institution or conference. In the field we will work long hours in all weather conditions. We are looking for a motivated individual who is able to hike through steep terrain for long hours beginning early in the morning.

In order for the REU position to be considered complete, the following must be done:

     1. Write a project proposal prior to beginning the field research season.

     2. Participate in field work for duration of season.

     3. Write a project report in manuscript form that has potential to be submitted for publication.

     4. Present independent research in poster or talk format. Location and event for this requirement is open for

         discussion. There may be funding available for a national meeting, but presenting at own university is also

         acceptable.


Compensation:

A stipend, per diem, housing and travel are all provided. After completion of the REU, you can expect to be well trained in the ability to trap, handle, band, and process songbirds, as well as analyze and interpret data.


Timeline:

Deadline to apply is February 15, 2016. Skype interviews will be offered to top candidates by February 19, 2016. REU will begin project development soon after accepting position. REU is expected to join research mentors at the field site after completion of finals and remain for 10 weeks past arrival date. Completion of the REU project will depend on date of oral presentation.


Requirements:

To apply, email a cover letter that outlines professional goals and research interests that fall within the project, a resume or CV, and 1 letter of recommendation from an instructor, research mentor, or someone who can comment on how you would contribute to vigorous field research and work as an independent researcher. We are willing to train individuals, so previous work with birds is not required. Preferred applicants should have course experience in fields such as ecology, behavior, and evolution. Participants are chosen for their readiness to take on original creative field-based biological research and for their interest in, and potential for, a career based in science. Applicant must plan to be enrolled in an undergraduate program in the fall of 2016.


Application Deadline: February 15, 2016


Contact Information:

Jessica L. Graham (jessica.l.graham@ndsu.edu)

Katie Needham (katie.needham@ndsu.edu)

&ndash;_000_BLUPR01MB129980329800CADBDBABF1FFB4C20BLUPR01MB1299prod_
Content-Type: text/html; charset=&ldquo;iso-8859-1&rdquo;
Content-Transfer-Encoding: quoted-printable


</p>


<p>Summer 2016 REU Position: Black Hills, South Dakota. </p>
<p></p>
<p>Project Title: </p>
<p>Individual variation in seasonal timing of breeding in dark-eyed juncos- Katie Needham and Jessica Graham (Ph.D. Candidates, under advisement of Dr. Timothy Greives, Biological Sciences Department, North Dakota State University).</p>
<p></p>

<p>Description: </p>
<p>This project will focus on the driving forces behind why some individuals breed earlier or later in the season than the majority of the population. Previous research with songbirds has shown that earlier breeding individuals have higher reproductive success
 and optimize timing with peak food abundance. While reproductive success is higher for individuals that breed earlier, selection is not shifting towards earlier reproduction, suggesting there are costs associated with breeding early, but the underlying physiological
 mechanisms of individual variation require further exploration. Working with the population of Dark-eyed Juncos near Hanna, South Dakota, we hope to uncover 1) What ecological factors affect increased reproductive success? 2) How variation in parental care
 affects reproductive success? Possible REU projects will focus on the nesting period and include observing how bacteria (presence-absence manipulation), parental care, or other ecological factors of interest that may affect reproductive success across the
 season. </p>
<p></p>
<p>Expectations: </p>
<p>REU will be expected to develop an independent project related to the ongoing research with guidance from mentors. Student will take leadership roles in all aspects of the study; they design the study, prepare a proposal, collect data and perform analysis,
 interpret the results, prepare a written manuscript, and complete an oral presentation at their home institution or conference. In the field we will work long hours in all weather conditions. We are looking for a motivated individual who is able to hike through
 steep terrain for long hours beginning early in the morning. </p>

<p>In order for the REU position to be considered complete, the following must be done:
</p>
<p> 1. Write a project proposal prior to beginning the field research season.
</p>

<p></p>
<p> 2. Participate in field work for duration of season. </p>
<p> 3. Write a project report in manuscript form that has potential to be submitted for publication.
</p>
<p> 4. Present independent research in poster or talk format. Location and event for this requirement is open for
</p>
<p> discussion. There may be funding available for a national meeting, but presenting at own university is also
</p>
<p> acceptable. </p>

<p></p>
<p>Compensation: </p>
<p>A stipend, per diem, housing and travel are all provided. After completion of the REU, you can expect to be well trained in the ability to trap, handle, band, and process songbirds, as well as analyze and interpret data.
</p>
<p></p>
<p>Timeline: </p>
<p>Deadline to apply is February 15, 2016. Skype interviews will be offered to top candidates by February 19, 2016. REU will begin project development soon after accepting position. REU is expected to join research mentors at the field site after completion
 of finals and remain for 10 weeks past arrival date. Completion of the REU project will depend on date of oral presentation.
</p>

<p></p>
<p>Requirements: </p>
<p>To apply, email a cover letter that outlines professional goals and research interests that fall within the project, a resume or CV, and 1 letter of recommendation from an instructor, research mentor, or someone who can comment on how you would contribute
 to vigorous field research and work as an independent researcher. We are willing to train individuals, so previous work with birds is not required.Preferred applicants should have course experience in fields such as ecology, behavior, and evolution. Participants
 are chosen for their readiness to take on original creative field-based biological research and for their interest in, and potential for, a career based in science. Applicant must plan to be enrolled in an undergraduate program in the fall of 2016.
</p>
<p></p>
<p>Application Deadline: February 15, 2016</p>
<p></p>
<p>Contact Information: </p>
<p>Jessica L. Graham (jessica.l.graham@ndsu.edu) </p>
<p>Katie Needham (katie.needham@ndsu.edu) </p>

<p></p>


<p></p>








&ndash;_000_BLUPR01MB129980329800CADBDBABF1FFB4C20BLUPR01MB1299prod
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:11</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137743582336">Postdoc: UTexas.SexChromosomeGenomics</a></div>
  <div class="description">Dear colleagues,

  We are seeking a postdoctoral fellow to join my lab at the University
of Texas (Austin) on a study of the origin and evolution of young sex
chromosomes.  In an NIH-funded collaboration with Dr. Katie Peichel
(Fred Hutchison Cancer Research Center), we are collecting genomic data
from several species of stickleback fishes.  My lab will be analyzing
those data to study topics that include chromosome rearrangements,
sexually-antagonistic selection, the evolution of recombination, and
patterns of molecular evolution on recombining sex chromosomes.

  The position can begin as early as May 1, and the starting date is
negotiable.  A requirements are a Ph.D. in evolutionary biology,
genetics, modeling, statistics, bioinformatics, or related field.  The
postdoc should have experience in programming (e.g. in Python and/or
C++) and/or bioinformatics.

  The salary is $45,432 to $51,096 / year, depending on qualifications,
and the position includes health and retirement benefits.  The initial
appointment is for one year, with possible extension up to three years
depending on satisfactory performance.

   If interested, please email Mark Kirkpatrick (kirkp@mail.utexas.edu)
with a letter explaining your background and motivation, a C.V., and
the email addresses of three people who have agreed to provide
recommendations.  The position will remain open until a suitable person
is found.

Best wishes,

Mark Kirkpatrick

Mark Kirkpatrick 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137741840986">Job: DukeU_ResTech.EvolAnthro.Genomics</a></div>
  <div class="description"><p>An Associate in Research/Laboratory Research Analyst position available,
starting immediately, in the Tung lab at Duke University. The Tung lab
studies the relationship between behavior, the social environment, and
genetics and genomics, with a focus on nonhuman primate populations. We
combine detailed phenotypic and demographic information with modern genomic
data sets on gene regulation and epigenetics. Current projects focus on the
gene regulatory mechanisms linking social adversity and health in rhesus
macaques, the epigenetic signature of early life ecological and social
adversity, and the genomic and phenotypic consequences of hybridization in
wild baboons.  Duties will include performance of basic molecular biology
laboratory techniques, including DNA and RNA sample extraction and sample
preparation for high-throughput sequencing, and responsibility for
day-to-day logistics, including purchasing, shipping and receiving, and
working with trainees in the lab.

Important skills/traits include: aseptic technique, molecular techniques,
attention to detail, ability to work independently, careful record keeping.

Requirements: bachelor&rsquo;s degree, 1-2 years experience in a research
laboratory (not just a lab class), familiarity with fundamentals of
molecular biology.

Position is for 40 hrs/week, and will last 12 months, with the possibility
of renewal contingent on performance and availability of funding.

For more information about the lab and our work, see: *
<a href="http://bit.ly/1Ksr3Ja">http://bit.ly/1Ksr3Ja</a>

If interested, email resume, including contact information for two
references, to Tawni Voyles, *tawni.voyles@duke.edu*

Duke University is an Equal Opportunity/Affirmative Action/ADA Employer
committed to providing employment opportunity without regard to an
individuals age, color, disability, genetic information, gender, gender
identity, national origin, race, religion, sexual orientation, or veteran
status.


voyles.tn@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:41</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137740013906">Job: UHull.ResAssist.VertebrateInvasion</a></div>
  <div class="description">NERC Research Associate/Assistant in Phylogenetic Comparative Analyses
                of Vertebrate Invasion Success (1 Post)

      School of Biological, Biomedical and Environmental Sciences

We are looking to appoint a Research Associate suitably qualified and
experienced in the areas required or a Research Assistant that is
presently working towards completing their PhD and looking to develop
their experience in this field. The post is funded for 1 year by a NERC
New Investigator Award to Dr Isabella Capellini, entitled
&lsquo;Understanding biological invasions: a phylogenetic comparative
approach&rsquo;.

The RA will be a member of the Evolutionary Comparative Ecology Group
(http:/tinyurl.com/kswc4ac) led by Dr Isabella Capellini
(http:/tinyurl.com/icapellini). The group offers a dynamic research
environment where the successful applicant will be member of a team
using state of the art phylogenetic comparative approaches to answer an
array of fundamental questions in biology and ecology.

Using alien (non-native) vertebrates as model organisms and cutting
edge phylogenetic comparative methods, the project&rsquo;s goal is to test
which biological characteristics promote species success in space and
time, as predicted by key hypotheses in macroecology and evolution. The
proposed work for this position will extend the results of the paper we
recently published (Capellini et al 2015 Ecology Letters), by either
investigating invasion success in fish or ecological species&rsquo; traits
promoting success in terrestrial vertebrates. The RA will help
completing an extensive database of vertebrate invasion success at the
global scale and species characteristics, and is expected to work
together with the other postdoctoral researcher on the project, in
constructing, updating and maintaining the project database of
vertebrate invasions. The post holder will also contribute to the
dissemination of the results to the scientific community through
presentation at international conferences and workshops, and to the
general public through public lectures and interviews with the media
where appropriate.

 Within the School, the RA will benefit from a rich and diverse
research environment that includes the Ecology and Environment Group
and the Evolutionary Biology Group. The University of Hull also offers
a comprehensive Staff Development Programme.

Essential: PhD in biology or related discipline near completion, BSc in
Biology or related discipline, knowledge of modern phylogenetic
comparative methods, strong quantitative skills, evidence of work of
publishable quality, excellent oral and written communication skills,
demonstrable ability to work as an effective team member, and be
reliable, self-motivated and well organized.

Desirable: PhD in biology or related discipline, experience with
phylogenetic comparative methods, experience in testing hypotheses in
macroecology and phenotypic evolution across species, strong
publication record, experience with R.

The position is fixed term for 1 year. The start date of the project is
fixed to 1^st April 2016.

Salary range: 26,537 to 31,656 pa - Starting salary according to
qualification and experience.

You can learn more about this position and apply online
at: <a href="http://www.hull.ac.uk/jobs">www.hull.ac.uk/jobs</a> (Vacancy ref: FS0214)
Informal enquiries to Dr Isabella Capellini (I.Capellini@hull.ac.uk)

Closing date: 21^st February 2016

Interviews: 1^st March 2016

Dr Isabella Capellini
School of Biological, Biomedical and Environmental Sciences
University of Hull
Cottingham Road, Hull
HU6 7RX (UK)
Personal webpage & NERC/BBSRC projects:
http:/tinyurl.com/icapellini
Evolutionary Comparative Ecology Group:
http:/tinyurl.com/kswc4ac

Isabella Capellini 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>January 20, 2016</h3>
<div class="news-item">
 <div class="date">00:34</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137673826301">Job: IAST_Toulouse.EvolBiol</a></div>
  <div class="description">&ldquo;The Institute for Advanced Study in Toulouse (IAST) and
Universit Toulouse I Capitole, France, seek candidates for one
open rank tenure-track or tenured position, open to Evolutionary
Biologists. Rank and tenure status will be commensurate with experience
and qualifications. Anticipated start date: September 1st, 2016. For
more information and send your application please visit: <a href="http://www.iast.fr/">www.iast.fr</a>,
and select "Applications&rdquo; - Tenured or Tenure-track position in
Social  Sciences.  You can also  contact Ingela Alger, Director of the
program in Biology at the IAST. Application materials should be sent as
soon as possible, and no later  than 29th February 2016.&ldquo;


Delphine POUTS 
Assistante de Direction
Executive Assistant

IAST
21 alle de Brienne
31015 Toulouse Cedex 6

Ph: 0033 5 61 12 86 27
Delphine.pouts@iast.fr 


Delphine Pouts 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="indexd118.html?q=aggregator&amp;from=240">previous page</a></div><div>... <a href="index52c1.html?q=aggregator&amp;from=180">10</a> <a href="indexeef0.html?q=aggregator&amp;from=200">11</a> <a href="index933b.html?q=aggregator&amp;from=220">12</a> <a href="indexd118.html?q=aggregator&amp;from=240">13</a> <strong>14</strong> <a href="index4603.html?q=aggregator&amp;from=280">15</a> <a href="index8c8d.html?q=aggregator&amp;from=300">16</a> <a href="index9efb.html?q=aggregator&amp;from=320">17</a> <a href="indexd191.html?q=aggregator&amp;from=340">18</a> ...</div><div><a href="index4603.html?q=aggregator&amp;from=280">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museums collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=260 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:26:20 GMT -->
</html>
