<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=380 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:36:33 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="indexacd7.html?q=node/208">56(4) August 2007</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 9 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>December 29, 2015</h3>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/245?rss=1">A Molecular Evolutionary Reference for the Human Variome</a></div>
  <div class="description"><p>Widespread sequencing efforts are revealing unprecedented amount of genomic variation in populations. Such information is routinely used to derive consensus reference sequences and to infer positions subject to natural selection. Here, we present a new molecular evolutionary method for estimating neutral evolutionary probabilities (<I>EP</I>s) of each amino acid, or nucleotide state at a genomic position without using intraspecific polymorphism data. Because <I>EP</I>s are derived independently of population-level information, they serve as null expectations that can be used to evaluate selective forces on alleles at both polymorphic and monomorphic positions in populations. We applied this method to coding sequences in the human genome and produced a comprehensive evolutionary variome reference for all human proteins. We found that <I>EP</I>s accurately predict neutral and disease-associated alleles. Through an analysis of discordance between allelic <I>EP</I>s and their observed population frequencies, we discovered thousands of novel candidate sites for nonneutral evolution in human proteins. Many of these were validated in a joint analysis of disease-associated variants and population data. The <I>EP</I> method is also directly applicable to the analysis of noncoding sequences and genomic analyses of nonmodel species.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/255?rss=1">Substitution Model Adequacy and Assessing the Reliability of Estimates of Virus Evolutionary Rates and Time Scales</a></div>
  <div class="description"><p>Determining the time scale of virus evolution is central to understanding their origins and emergence. The phylogenetic methods commonly used for this purpose can be misleading if the substitution model makes incorrect assumptions about the data. Empirical studies consider a pool of models and select that with the highest statistical fit. However, this does not allow the rejection of all models, even if they poorly describe the data. An alternative is to use model adequacy methods that evaluate the ability of a model to predict hypothetical future observations. This can be done by comparing the empirical data with data generated under the model in question. We conducted simulations to evaluate the sensitivity of such methods with nucleotide, amino acid, and codon data. These effectively detected underparameterized models, but failed to detect mutational saturation and some instances of nonstationary base composition, which can lead to biases in estimates of tree topology and length. To test the applicability of these methods with real data, we analyzed nucleotide and amino acid data sets from the genus <I>Flavivirus</I> of RNA viruses. In most cases these models were inadequate, with the exception of a data set of relatively closely related sequences of <I>Dengue virus</I>, for which the GTR+ nucleotide and LG+ amino acid substitution models were adequate. Our results partly explain the lack of consensus over estimates of the long-term evolutionary time scale of these viruses, and indicate that assessing the adequacy of substitution models should be routinely used to determine whether estimates are reliable.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/268?rss=1">Coevolutionary Landscape Inference and the Context-Dependence of Mutations in Beta-Lactamase TEM-1</a></div>
  <div class="description"><p>The quantitative characterization of mutational landscapes is a task of outstanding importance in evolutionary and medical biology: It is, for example, of central importance for our understanding of the phenotypic effect of mutations related to disease and antibiotic drug resistance. Here we develop a novel inference scheme for mutational landscapes, which is based on the statistical analysis of large alignments of homologs of the protein of interest. Our method is able to capture epistatic couplings between residues, and therefore to assess the dependence of mutational effects on the sequence context where they appear. Compared with recent large-scale mutagenesis data of the beta-lactamase TEM-1, a protein providing resistance against beta-lactam antibiotics, our method leads to an increase of about 40% in explicative power as compared with approaches neglecting epistasis. We find that the informative sequence context extends to residues at native distances of about 20 &Aring; from the mutated site, reaching thus far beyond residues in direct physical contact.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/281?rss=1">An Exon-Capture System for the Entire Class Ophiuroidea</a></div>
  <div class="description"><p>Exon-capture studies have typically been restricted to relatively shallow phylogenetic scales due primarily to hybridization constraints. Here, we present an exon-capture system for an entire class of marine invertebrates, the Ophiuroidea, built upon a phylogenetically diverse transcriptome foundation. The system captures approximately 90% of the 1,552 exon target, across all major lineages of the quarter-billion-year-old extant crown group. Key features of our system are 1) basing the target on an alignment of orthologous genes determined from 52 transcriptomes spanning the phylogenetic diversity and trimmed to remove anything difficult to capture, map, or align; 2) use of multiple artificial representatives based on ancestral state reconstructions rather than exemplars to improve capture and mapping of the target; 3) mapping reads to a multi-reference alignment; and 4) using patterns of site polymorphism to distinguish among paralogy, polyploidy, allelic differences, and sample contamination. The resulting data give a well-resolved tree (currently standing at 417 samples, 275,352 sites, 91% data-complete) that will transform our understanding of ophiuroid evolution and biogeography.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/295?rss=1">The Evolution of Beer</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/295-a?rss=1">Evolutionary Study of Birds Sense of Smell Reveals Important Clues for Behavior and Adaptation</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/296?rss=1">Of Skin and Teeth: Identifying Key Differences in Asians</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/296-a?rss=1">Knee-Deep in Spider Leg Evolution</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/297?rss=1">A Dominant Evolutionary Theme Emerges to Better Predict Clinical Outcomes for Cancer</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/297-a?rss=1">Predicting the Human Genome Using Evolution</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/299?rss=1">Society for Molecular Biology and Evolution, Council and Business Meetings, 2015, Vienna Austria</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/301?rss=1">Treasurers Report for Financial Year (FY) 2014</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/302?rss=1">Genes that Escape X-Inactivation in Humans Have High Intraspecific Variability in Expression, Are Associated with Mental Impairment but Are Not Slow Evolving</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:52</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/1/303?rss=1">Proposal of a Twin Aarginine Translocator System-Mediated Constraint against Loss of ATP Synthase Genes from Nonphotosynthetic Plastid Genomes</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:06</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136169988141">Postdoc: fungal evolutionary genomics</a></div>
  <div class="description"><p>&ndash;_000_D2A6C2364B636dhibbettclarkuedu_
Content-Type: text/plain; charset=&ldquo;Windows-1252&rdquo;
Content-Transfer-Encoding: quoted-printable

Postdoc: fungal evolutionary genomics

The Hibbett lab at Clark University seeks a highly motivated, independent postdoctoral fellow to conduct research in evolutionary genomics in mushroom-forming fungi. The postdoctoral fellow will be involved in one or more of the following projects: (1) Evolutionary developmental biology of Agaricomycetes; (2) Evolution of substrate-specificity in wood-decay fungi; or (3) Phylogenomics of Lentinula (shiitake mushrooms and relatives). All three projects require expertise in evolutionary genomics and molecular phylogenetics. Experience in fungal genetics is also desirable. Research could include &ldquo;wet bench&rdquo; techniques, including culturing and production of fruiting bodies for transcriptomics, or could be entirely analytical, depending on the candidate&rsquo;s skills and interests. Applicants should have a PhD in genomics, evolutionary genetics, fungal biology, or a relevant discipline, a strong publication record (appropriate for career stage), and excellent communication skills. Will
 ingness to involve students in research is also important. Preferred starting date is June 1, 2016, but this is (somewhat) flexible. Up to three years of support is available, contingent on progress. For more information, including application instructions, please see: <a href="http://bit.ly/1QTAdWM">http://bit.ly/1QTAdWM</a>

Applications from women and members of underrepresented groups in science are encouraged. Clark University is an EEO/AA Employer.

&ndash;_000_D2A6C2364B636dhibbettclarkuedu_
Content-Type: text/html; charset=&ldquo;Windows-1252&rdquo;
Content-ID: 
Content-Transfer-Encoding: quoted-printable


</p>

</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 28, 2015</h3>
<div class="news-item">
 <div class="date">23:26</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136167889816">Postdoc: ClarkU.FungalEvolutionaryGenomics</a></div>
  <div class="description">Postdoc: fungal evolutionary genomics

The Hibbett lab at Clark University seeks a highly motivated, independent
postdoctoral fellow to conduct research in evolutionary genomics in
mushroom-forming fungi. The postdoctoral fellow will be involved in
one or more of the following projects: (1) Evolutionary developmental
biology of Agaricomycetes; (2) Evolution of substrate-specificity
in wood-decay fungi; or (3) Phylogenomics of Lentinula (shiitake
mushrooms and relatives). All three projects require expertise in
evolutionary genomics and molecular phylogenetics.  Experience in
fungal genetics is also desirable. Research could include â€œwet
benchâ€� techniques, including culturing and production of
fruiting bodies for transcriptomics, or could be entirely analytical,
depending on the candidate&rsquo;s skills and interests. Applicants should
have a PhD in genomics, evolutionary genetics, fungal biology, or a
relevant discipline, a strong publication record (appropriate for
career stage), and excellent communication skills. Willingness to
involve students in research is also important. Preferred starting
date is June 1, 2016, but this is (somewhat) flexible. Up to
three years of support is available, contingent on progress. For
more information, including application instructions, please see:
<a href="http://bit.ly/1QTAdWM">http://bit.ly/1QTAdWM</a>

Applications from women and members of underrepresented groups in
science are encouraged. Clark University is an EEO/AA Employer.

David Hibbett 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 25, 2015</h3>
<div class="news-item">
 <div class="date">23:36</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135959233711">Postdoc: SunYatsenU.PlantMacroevolution</a></div>
  <div class="description">Macroevolution of Land Plants at SYSU V Exploring the link between
genome evolution and plant diversity


Postdoctoral research positions (2 years) are open at the new research
team &ldquo;Macroevolution of Land Plants&rdquo; led by Harald Schneider
located at the School of Life Sciences, Sun Yatsen University, Guanzgou,
Guangfdong, China. The position holder is expected to carry out a research
program exploring the evolutionary consequences of polyploidy in selected
groups of seed-free land plants, e.g. derived ferns. The research involves
the integration of genomic evidence (e.g. transcriptome sequences), genome
size (DNA C-values), chromosome numbers, and morphological evidence in
a dated phylogenetic framework. The main issues addressed are (1) the
consequences of polyploidy on the assembly of morphological disparity,
(2) ecological and phylogenetic constraints in the assembly of polyploidy;
and (3) intrinsic and extrinsic constraints on the accumulation of large
genomes via polyploidy.

Key roles: carrying out research on the topic outlined at the postdoctoral
level / First author of at least two publications in scientific journals
with a SCI > 2.0 based on the research carried out / contribute to
other publications of the research team / contribute to or lead the
writing of grant proposals of the research team / presenting of the
research results at international scientific meetings (e.g., IBC 2017) /
contribute to the training of bachelor and master students  / contribute
to the success of the research team

Qualifications of the post-doctoral applicants
Required: Phd in Biology (e.g., Evolutionary Sciences) or Bioinformatics/
General understanding of plant genomes and plant evolution/ Proven
publication record (in English) / Good communication skills/ Ability to
work in teams / Self motivated work habit

Preferred: Experience with the handling of transcriptome data or other
next generation sequence data using bioinformatic pipelines / Experience
in statistical analyses using R or similar platform/ Experience in
phylogenetic analyses and/ or divergence time estimates/ Experience in
plant cytogenetics including cytofluometry or chromosome painting

The call for positions will be open until the positions are
filled. Interviews will be held based on availability.

For informal enquiries please contact Prof Harald Schneider
(haralds@mail.sysu.edu.cn)

	Prof Harald Schneider
	School of Life Sciences
	Sun Yat-Sen University
	Guangzou, Guangdong, China.


Prof Harald Schneider, PhD FLS
School of Life Scienes, Centre of Ecology and Evolution
Sun Yatsen University, Guangzhou, Guangdong, China
&
Department of Life Sciences
Natural History Museum, London SW7 %BD, UK


Harald Schneider 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:04</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135895982941">Job: UFlorida.LandscapeGenetics</a></div>
  <div class="description"><p>Title: 		Assistant Professor of Landscape Ecology
Location: 	Fort Lauderdale Research and Education Center
		University of Florida
		Institute of Food and Agricultural Sciences (IFAS)
		Davie, Florida
Salary: 	Commensurate with Qualifications and Experience

Review Date: 	For full consideration, candidates should apply and submit
additional materials by February 15, 2016. Position will remain open until
a viable applicant pool is determined.

Duties and Responsibilities:

This is a 12-month tenure-accruing position that will be 60% research
(Florida Agricultural Experiment Station) and 40% extension (Florida
Cooperative Extension Service), available at the Fort Lauderdale Research
and Education Center, Institute of Food and Agricultural Sciences, at the
University of Florida. This faculty member is expected to develop a
productive, interdisciplinary, extramurally funded research program with
emphasis on landscape structure and function over a range of spatial and
temporal scales in South Florida. This may include application of climate
change, including sea-level rise, to ecological landscape analysis;
biogeochemical processes and modeling including carbon sequestration and
greenhouse gas emissions; evaluation of species and community responses to
changes in landscape structure and function or invasive species; and
integration of multidisciplinary science into restoration of damaged
landscapes or response to sea-level rise. Extension responsibilities will
include developing education programs on the interaction of Everglades
restoration with natural and man-made ecosystems and on impacts of
sea-level rise. The incumbent will provide leadership, training, and
assistance to county faculty. The position will be located at the Fort
Lauderdale Research and Education Center, Institute of Food and Agriculture
Sciences (IFAS), University of Florida (Davie, Florida). Tenure will accrue
in the most suitable department (Soil & Water Science or School of Forestry
& Resource Conservation), based on the successful applicant&rsquo;s credentials.

Please refer to Requisition # 495173
Dr. Henry Hochmair
Chair, Landscape Ecologist Search and Screen Committee
University of Florida
Fort Lauderdale Research and Education Center
3205 College Avenue
Davie, FL 33314

Telephone:	954.577.6317
Facsimile:	954.475.4125
E-Mail:		hhhochmair@ufl.edu

Application Information:

Individuals wishing to apply should go online to
<a href="http://bit.ly/1QPG7b7">http://bit.ly/1QPG7b7</a>
and submit:
* Application
* Cover letter that states applicant’s interest in the position and
qualifications relative to the credentials listed above
* Curriculum vitae
* Names and contact information (email address and phone number) of three
individuals who may be contacted to provide letters of recommendation


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 24, 2015</h3>
<div class="news-item">
 <div class="date">23:49</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135895315391">Conference:</a></div>
  <div class="description">Evolution of Mechanisms, January 28-31, 2016, University of Debrecen,
Hungary

Please be reminded that the registration deadline for our symposium &ldquo;The
evolution of mechanisms: a workshop on the integration of life history
evolution and physiology&rdquo; is January 03, 2016.

Symposium summary:

In the past decades research in behavior and life history has been
dominated by two different approaches: molecular biologists and
physiologists investigated the fine-scaled mechanisms of how life history
and behavioral processes are regulated, while theoretical and organismal
biology provided an explanation for the adaptive value of such traits and
the evolutionary forces that shape them. While both approaches have been
extremely successful, the proximate-ultimate dichotomy is not warranted
anymore: our next challenge is to integrate these approaches and to
understand the evolution of the genetic, endocrine mechanisms and signaling
pathways that ultimately lead to a bewildering diversity of life-forms.

In order to advance the discussion between these traditionally distinct
areas, we organize a workshop where we invite theoreticians and empiricists
working in the fields of physiology, molecular biology and evolutionary
ecology to discuss the current state of the art and to articulate a
research agenda for the future, where they identify the key challenges and
approaches to study the evolution of the integrated phenotype and
physiological underpinnings of life history evolution. Instead of the
dominantly unidirectional information flow of conference symposia, we
organize this meeting to promote discussion and debate about ideas,
concepts of how we can understand the evolution of mechanisms that provide
the raw material for natural selection.

Participants are invited to give contributed talks that advance the goal of
the meeting. Additional poster presentations will also be welcome.

Invited speakers:

Thomas Flatt (University of Lausanne): The mechanisms underpinning life
history evolution

David Gems (UCL): Role of insulin/IGF-1 signaling in the evolution of life
histories and ageing

Laura Lavine (Washington State University): Endocrine signaling and the
development of the weapons of sexual selection: insights from beetles

Pat Monaghan (University of Glasgow): Environmental effects that shape
individual life histories: mechanisms, trade-offs and time scales

Contact: evolmech2016@gmail.com

For further details and registration visit:
<a href="http://bit.ly/1NOwfZr">http://bit.ly/1NOwfZr</a>

Hope to see you there,

the organizers:

dm Lendvai, Zoltn Nmeth, Jcint Tklyi, Zoltn Barta

Zoltan Nemeth 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">16:30</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2015/12/fast-food-studies.html">Fast food studies</a></div>
  <div class="description"><br />Season's greetings.<br /><br />For your Christmas reading, this blog usually provides a seasonally appropriate post on fast food, including to date posts about: nutrition (<a href="http://phylonetworks.blogspot.com/2012/12/network-analysis-of-mcdonalds-fast-food.html" target="_blank">McDonald's fast-food</a>), geography (<a href="http://phylonetworks.blogspot.com/2013/12/fast-food-maps-network-analysis.html" target="_blank">Fast-food maps</a>) and diet (<a href="http://phylonetworks.blogspot.com/2014/12/fast-food-and-diet.html" target="_blank">Fast food and diet</a>). This year, we will update some of the geographical and diet information about the effects of fast food on people worldwide.<br /><br />First, there seems to be a general perception that access to fast food is continuing to increase in the modern world, even though that increase started more than half a century ago. Such a perception is easy to verify in the USA, as shown in a previous post  (<a href="http://phylonetworks.blogspot.com/2013/12/fast-food-maps-network-analysis.html" target="_blank">Fast-food maps</a>). However, this trend also appears globally.<br /><br />For example, in 2013 the <i>Guardian</i> newspaper produced a dataset (called the <a href="https://docs.google.com/spreadsheets/d/1L7-l_GSVY7UwKKS6meAxVeSNEZETaR_FCG6VkZfkHk8/edit?pref=2&amp;pli=1#gid=0" target="_blank">McMap of the World</a>) illustrating the recent growth in the number of McDonald's restaurants worldwide (<a href="http://www.theguardian.com/news/datablog/2013/jul/17/mcdonalds-restaurants-where-are-they" target="_blank">McDonald's 34,492 restaurants: where are they?</a>). This first graph shows the relative number of restaurants in 2007 and 2012, with each dot representing a single country. Almost all of the 116 countries showed an increase during the five years (ie. their dots are above the pink line). The only country with a major decrease in McDonald's restaurants was Greece (to the right of the pink line), due, no doubt, to its ongoing financial problems. The country with the largest number of restaurants is, of course, the USA, with Japan a clear second.<br /><br /><a href="http://2.bp.blogspot.com/-XDjOM5L6zOw/Vnu2cz2uC3I/AAAAAAAAD3Q/XbKtgY2f-AI/s1600/McDonalds.gif" imageanchor="1"></a><br />There is also a perception that fast-food restaurants compete for customers against other restaurants, so that suburbs can have one or the other but not both. This can be checked using the data in the <a href="http://ers.usda.gov/data-products/food-environment-atlas/data-access-and-documentation-downloads.aspx" target="_blank">Food Environment Atlas 2014</a> (produced by the USDA Economic Research Service), which show the number of both Fast-food and Full-service restaurants / 1,000 popele in 2012 for each of the more than 3,100 counties in the USA. This is illustrated in the next graph, where each dot represents a single county.<br /><br /><a href="http://4.bp.blogspot.com/-3cw8I8KRCBs/Vnu4e5C2SGI/AAAAAAAAD3o/3mOdmQZmz3w/s1600/FullServe.gif" imageanchor="1"></a><br />For most counties, full-service restaurants actually out-number fast-food restaurants, per capita. There are even a few counties that have no fast-food places at all, but also a few with no full-service restaurants. There are even a few with neither restaurant type, notably in AK (2 counties), KY and ND. Interestingly, 3 out of the 4 counties with the largest density of full-service restaurants are in CO (including one not shown because it is off the top of the graph).<br /><br />Nevertheless, you can't go far in the USA without encountering a fast-food place. As shown in a previous post  (<a href="http://phylonetworks.blogspot.com/2013/12/fast-food-maps-network-analysis.html" target="_blank">Fast-food maps</a>), Subway has the largest number of establishments, not McDonald's. The Flowing Data blog has recently compiled a couple of maps showing the dominance of Subway in the sandwich business (<a href="http://flowingdata.com/2015/05/13/where-subway-dominates-its-sandwich-place-competition-basically-everywhere/" target="_blank">Where Subway dominates its sandwich place competition, basically everywhere</a>). This map shows the subway dominance — each dot is an area with a 10-mile radius, colored by the brand of the nearest sandwich chain.<br /><br /><a href="http://4.bp.blogspot.com/-vpvrcb2Nycc/Vnu-wl6hXiI/AAAAAAAAD34/oCRt9CZE2Bs/s1600/sandwich-modal-subway-21.png" imageanchor="1"></a>Unfortunately, studying the effects of geography on access to fast food is not as simple as it might seem. Large-scale patterns such as those shown above are only part of the picture, because access to fast food is usually assumed to be determined at a very local scale — how far is the nearest fast-food place and how easy is it to get there?<br /><br />There have been many studies over the years, based on different methods and with different study criteria. These have been summarized (from different perspectives) by SE Fleischhacker, KR Evenson, DA Rodriguez, AS Ammerman (2010. A systematic review of fast food access studies. <a href="http://onlinelibrary.wiley.com/doi/10.1111/j.1467-789X.2010.00715.x/abstract" target="_blank">Obesity Reviews</a> 12: e460) and LK Fraser, KL Edwards, J Cade, GP Clarke (2010. The geography of fast food outlets: a review. <a href="http://www.mdpi.com/1660-4601/7/5/2290" target="_blank">International Journal of Environmental Research and Public Health</a> 7: 2290-2308).<br /><br />Their conclusions from their worldwide literature reviews include:<br /><ul><li>most studies indicated fast-food restaurants were more prevalent in low-income areas compared with middle- to higher-income areas (ie. there is a positive association between availability of fast-food outlets and increasing socio-economic deprivation);</li><li>most studies found that fast food restaurants were more prevalent in areas with higher concentrations of ethnic minority groups;</li><li>those studies that included overweight or obesity data (usually measured as the body mass index) showed conflicting results between obesity / overweight and fast-food outlet availability — most studies found that higher body mass index was associated with living in areas with increased exposure to fast food, but the remaining studies did not find any such association;</li><li>there is some evidence that fast food availability is associated with lower fruit and vegetable intake.</li></ul>In a previous post (<a href="http://phylonetworks.blogspot.com/2014/12/fast-food-and-diet.html" target="_blank">Fast food and diet</a>) I illustrated the association between fast food and obesity in the USA. Here, I use the data from the <i>Guardian</i> article mentioned above (<a href="https://docs.google.com/spreadsheets/d/1L7-l_GSVY7UwKKS6meAxVeSNEZETaR_FCG6VkZfkHk8/edit?pref=2&amp;pli=1#gid=0" target="_blank">McMap of the World</a>) to do the same thing at a global scale. This next graph shows the relationship between the per capita density of McDonald's restaurants and overweight / obesity for those countries for which there are data available (each dot represents a single country).<br /><br /><a href="http://3.bp.blogspot.com/-yCoxgymVZeE/Vnu2cuoiAXI/AAAAAAAAD3U/KsMkMp15_s4/s1600/Obesity.gif" imageanchor="1"></a><br />These patterns have continued over the five years since the reviews appeared, with published studies both pro (eg. J Currie, S DellaVigna, E Moretti, V Pathania. 2010. The effect of fast food restaurants on obesity and weight gain. <a href="http://www.aeaweb.org/articles.php?doi=%2010.1257/pol.2.3.32" target="_blank">American Economic Journal: Economic Policy</a> 2: 32-63) and con (AS Richardson, J Boone-Heinonen, BM Popkin, P Gordon-Larsen. 2011. Neighborhood fast food restaurants and fast food consumption: a national study. <a href="http://www.biomedcentral.com/1471-2458/11/543" target="_blank">BMC Public Health</a> 11: 543) the association between fast food availability and health. To them has been added the issue of Type II diabetes and fast-food consumption (see DH Bodicoat et al. 2014. Is the number of fast-food outlets in the neighbourhood related to screen-detected type 2 diabetes mellitus and associated risk factors? <a href="http://dx.doi.org/10.1017/S1368980014002316" target="_blank">Public Health Nutrition</a> 18 : 1698-1705).<br /><br />Moving on, people have also considered how the role of restaurants might define the identity of American cities. For example, Zachary Paul Neal has considered whether US cities can be classified on the basis of the local prevalence of specific types of restaurants (2006. Culinary deserts, gastronomic oases: a classification of US cities. <a href="http://creativeclass.typepad.com/thecreativityexchange/files/zachary_neal_restaurants_paper.pdf" target="_blank">Urban Studies</a> 43: 1-21). He counted the numbers of several different types of restaurants in 243 of the most populous cities in the USA, and ended up classifying them into four distinct city types: Urbane oases (where one finds an abundance of restaurants of all sorts), McCulture oases (which have larger than normal concentrations of "highly standardised eating places designed for mass consumption"), Urbane deserts and McCulture deserts (both of which have fewer restaurants than their respective oasis counterpart).<br /><br />Unfortunately, this sort of classification approach is self-fulfilling, because any mathematical grouping algorithm will form groups, by definition, even if there are no groups in the data. I have shown this a number of times in this blog (eg. <a href="http://phylonetworks.blogspot.com/2012/05/network-analysis-of-scotch-whiskies.html" target="_blank">Network analysis of scotch whiskies</a>; <a href="http://phylonetworks.blogspot.com/2014/02/single-malt-scotch-whiskies-network.html" target="_blank">Single-malt scotch whiskies — a network</a>). These culinary data are thus crying out for a network analysis, and I would normally present one at this point in the blog post. However, I do not have a copy of Neal's dataset.<br /><br />So, instead, I will finish by analyzing some data on the salt content of fast food (E Dunford et al. 2012. The variability of reported salt levels in fast foods across six countries: opportunities for salt reduction. <a href="http://www.cmaj.ca/content/184/9/1023.long" target="_blank">Canadian Medical Association Journal</a> 184: 1023-1028).<br /><br />The authors collected data on the salt content of products served by six fast food chains that operate in Australia, Canada, France, New Zealand, the United Kingdom and the United States — Burger King, Domino’s Pizza, Kentucky Fried Chicken, McDonald’s, Pizza Hut and Subway. The product categories included: savoury breakfast items, burgers, chicken products, french fries, pizza, salads, and sandwiches. Data were collated for all of the products provided by all of the companies that fitted into these categories (137-523 products per country). Mean salt contents and their ranges were calculated, and compared within and between countries and companies.<br /><br />We can use a phylogenetic network to visualize these data. As usual, I have used the manhattan distance and a neighbor-net network. The result is shown in the next figure. Countries that are closely connected in the network are similar to each other based on their fast-food salt content, and those that are further apart are progressively more different from each other.<br /><br /><a href="http://3.bp.blogspot.com/-7XqBQCqLseM/VnvqwgyILqI/AAAAAAAAD4I/SWNxxKKVG5Y/s1600/Salt.gif" imageanchor="1"></a><br />You will note that the North American countries are on one side of the network, with the highest salt content, while the European countries are on the other, with the lowest salt content (on average 85% of the American salt content). This difference was reflected even between the same products in different countries — for example, McDonald's Chicken McNuggets contained 0.6 g of salt per 100 g in the UK but 1.6 g of salt per 100 g in the USA). As the authors note: "the marked differences in salt content of very similar products suggest that technical reasons are <i>not</i> a primary explanation."<br /><br /></div>
  <div class="source">Source: <a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="index28a4.html?q=aggregator&amp;from=360">previous page</a></div><div>... <a href="index8c8d.html?q=aggregator&amp;from=300">16</a> <a href="index9efb.html?q=aggregator&amp;from=320">17</a> <a href="indexd191.html?q=aggregator&amp;from=340">18</a> <a href="index28a4.html?q=aggregator&amp;from=360">19</a> <strong>20</strong> <a href="index940a.html?q=aggregator&amp;from=400">21</a> <a href="indexd6af.html?q=aggregator&amp;from=420">22</a> <a href="index764e.html?q=aggregator&amp;from=440">23</a> <a href="index3d2c.html?q=aggregator&amp;from=460">24</a> ...</div><div><a href="index940a.html?q=aggregator&amp;from=400">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=380 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:36:33 GMT -->
</html>
