<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=460 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:37:59 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="indexacd7.html?q=node/208">56(4) August 2007</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 7 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>December 10, 2015</h3>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/253">Variation at the DRD4 locus is associated with wariness and local site selection in urban black swans</a></div>
  <div class="description">Background:
Interactions between wildlife and humans are increasing. Urban animals are often less wary of humans than their non-urban counterparts, which could be explained by habituation, adaptation or local site selection. Under local site selection, individuals that are less tolerant of humans are less likely to settle in urban areas. However, there is little evidence for such temperament-based site selection, and even less is known about its underlying genetic basis. We tested whether site selection in urban and non-urban habitats by black swans (Cygnus atratus) was associated with polymorphisms in two genes linked to fear in animals, the dopamine receptor D
                      4 
                   (DRD4) and serotonin transporter (SERT) genes.
Results:
Wariness in swans was highly repeatable between disturbance events (repeatability = 0.61) and non-urban swans initiated escape from humans earlier than urban swans. We found no inter-individual variation in the SERT gene, but identified five DRD4 genotypes and an association between DRD4 genotype and wariness. Individuals possessing the most common DRD4 genotype were less wary than individuals possessing rarer genotypes. As predicted by the local site selection hypothesis, genotypes associated with wary behaviour were over three times more frequent at the non-urban site. This resulted in moderate population differentiation at DRD4 (F
                      ST
                    = 0.080), despite the sites being separated by only 30 km, a short distance for this highly-mobile species. Low population differentiation at neutrally-selected microsatellite loci and the likely occasional migration of swans between the populations reduces the likelihood of local site adaptations.
Conclusion:
Our results suggest that wariness in swans is partly genetically-determined and that wary swans settle in less-disturbed areas. More generally, our findings suggest that site-specific management strategies may be necessary that consider the temperament of local animals.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/277">Detailed reconstruction of the nervous and muscular system of Lobatocerebridae with an evaluation of its annelid affinity</a></div>
  <div class="description">Background:
The microscopic worm group Lobatocerebridae has been regarded a ‘problematicum’, with the systematic relationship being highly debated until a recent phylogenomic study placed them within annelids (Curr Biol 25: 2000-2006, 2015). To date, a morphological comparison with other spiralian taxa lacks detailed information on the nervous and muscular system, which is here presented for Lobatocerebrum riegeri n. sp. based on immunohistochemistry and confocal laser scanning microscopy, supported by TEM and live observations.
Results:
The musculature is organized as a grid of longitudinal muscles and transverse muscular ring complexes in the trunk. The rostrum is supplied by longitudinal muscles and only a few transverse muscles. The intraepidermal central nervous system consists of a big, multi-lobed brain, nine major nerve bundles extending anteriorly into the rostrum and two lateral and one median cord extending posteriorly to the anus, connected by five commissures. The glandular epidermis has at least three types of mucus secreting glands and one type of adhesive unicellular glands.
Conclusions:
No exclusive “annelid characters” could be found in the neuromuscular system of Lobatocerebridae, except for perhaps the mid-ventral nerve. However, none of the observed structures disputes its position within this group. The neuromuscular and glandular system of L. riegeri n. sp. shows similarities to those of meiofaunal annelids such as Dinophilidae and Protodrilidae, yet likewise to Gnathostomulida and catenulid Platyhelminthes, all living in the restrictive interstitial environment among sand grains. It therefore suggests an extreme evolutionary plasticity of annelid nervous and muscular architecture, previously regarded as highly conservative organ systems throughout metazoan evolution.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>December 9, 2015</h3>
<div class="news-item">
 <div class="date">06:39</div>
 <div class="body">
  <div class="title"><a href="http://iphylo.blogspot.com/2015/12/visualising-difference-between-two.html">Visualising the difference between two taxonomic classifications</a></div>
  <div class="description"><p>It's a nice feeling when work that one did ages ago seems relevant again. Markus Döring has been working on a new backbone classification of all the species which occur in taxonomic checklists harvested by GBIF. After building a new classification the obvious question arises "how does this compare to the previous GBIF classification?" A simple question, answering it however is a little tricky. It's relatively easy to compare two text files -- and this function appears in places such as Wikipedia and GitHub -- but comparing trees is a little trickier. Ordering in trees is less meaningful than in text files, which have a single linear order. In other words, as text strings "(a,b,c)" and "(c,b,a)" are different, but as trees they are the same.</p> <p>Classifications can be modelled as a particular kind of tree where (unlike, say, phylogenies) every node has a unique label. For example, the tips may be species and the internal nodes may be higher taxa such as genera, families, etc. So, what we need is a way of comparing two rooted, labelled trees and finding the differences. Turns out, this is exactly what Gabriel Valiente and I worked on in this paper <a href="http://dx.doi.org/10.1186/1471-2105-6-208">doi:10.1186/1471-2105-6-208</a>. The code for that paper (<a href="https://github.com/rdmpage/forest">available on GitHub</a>) computes an "edit script" that gives a set of operations to convert one fully labelled tree into another. So I brushed up my rusty C++ skills (I'm using "skills" loosely here) and wrote some code to take two trees and the edit script, and create a simple web page that shows the two trees and their differences. Below is a screen shot showing a comparison between the classification of whales in the Mammals Species of the World, and one from GBIF (you can see a <a href="https://dl.dropboxusercontent.com/u/639486/cet.html">live version here</a>).</p>  <p>The display uses colours to show whether a nodes has been deleted from the first tree, inserted into the second tree, or moved to a different position. Clicking on a node in one tree scrolls the corresponding node in the other tree (if it exists) to scroll into view. Most of the differences between the two trees are due to the absence of fossils from Mammals Species of the World, but there are other issues such as GBIF ignoring tribes, and a few taxa that are duplicated due to spelling typos, etc.</p></div>
  <div class="source">Source: <a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:40</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134846324586">Course: MalhamTarn_England.R.May2-6</a></div>
  <div class="description">&ldquo;Advancing in statistical modelling using R&rdquo;
Delivered by Dr Luc Bussiere and Dr Tom Houslay
This course will run from 2nd - 6th May 2016 at Malham Tarn field
station in the Yorkshire Dales National Park, England -
<a href="http://bit.ly/1ITWoDO">http://bit.ly/1ITWoDO</a>
This 5 day course costs £460 for course only including lunch or £655
all inclusive, including all accommodation and meals and minibus
connections to and from a local meeting point.
The course is designed to bridge the gap between basic R coding and
more advanced statistical modelling. It will consist of a series of
modules (listed below), each lasting roughly half a day, and designed
to either build required skills for future modules or to perform a
family of analyses that is frequently encountered in the biological
literature. Each module will include practical and self-assessment
exercises to help attendees gauge their understanding of the concepts.
All course materials (including copies of presentations, practical
exercises, self-assessment problems, data files, and example scripts
prepared by the instructing team) will be provided electronically to
participants.
<a href="http://bit.ly/1RDNotI">http://bit.ly/1RDNotI</a>
Course content is as follows
Day 1:
Module 1 Introduction & data visualization using (graphics) and
(ggplot2)
Module 2 Univariate regression, diagnostics & plotting fits
Day 2:
Module 3 Adding additional continuous predictors (multiple regression).
Module 4 Adding factorial (categorical) predictors & incorporating
interactions (ANCOVA)
Day 3:
Module 5 Model selection & simplification (likelihood ratio tests, AIC)
Module 6 Mixed effects models in theory & practice
Day 4:
Module 7 Generalised Linear Models (binomial and count data)
Module 8 Nonlinear models (polynomial & mechanistic models)
Day 5:
Module 9 Combining methods (e.g., nonlinear mixed effect (NLME) models
& generalised linear mixed effect (GLMM) models)
Module 10 Optional free afternoon to cover previous modules and discuss
data
Please email any inquiries to oliverhooker@prstatistics.co.uk
Please feel free to distribute this material anywhere you feel is
suitable
Upcoming courses; APPLIED BAYESIAN MODELLING FOR ECOLOGISTS AND
EPIDEMIOLOGISTS; SPATIAL ANALYSIS OF ECOLOGICAL DATA USING R;
INTRODUCTION TO R AND STATISTICS FOR BIOLOGISTS; STABLE ISOTOPE MIXING
MODELS USING SIAR, SIBER AND MIXSIAR; INTRODUCTION TO PYTHON FOR
BIOLOGISTS; TIMES SERIES DATA ANALYSIS FOR ECOLOGISTS AND
CLIMATOLOGISTS USING R; MODEL BASED MULTIVARIATE ANALYSIS OF ECOLOGICAL
DATA USING R; ADVANCES IN DNA TAXONOMY USING R; GENETIC DATA ANALYSIS
USING R.
Oliver Hooker
PR~Statistics

&ldquo;oliverhooker@prstatistics.co.uk&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:10</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134845568626">Graduate position: CharlesUPrague.PlantPollination</a></div>
  <div class="description">A Ph.D. studentship
*Plant-pollinator interactions along an Afrotropical altitudinal
gradient*

We are seeking a highly motivated Ph.D. student to join a project
assessing studies of plant-pollinator relationships between selected
flowering plants and their pollinators on Mt. Cameroon. The research is
focused on changes of general characteristics of pollination networks
along the altitude, as well as on case studies of selected pollination
systems. The student will process already collected data and material,
as well as actively participate on field sampling (at least two times
during the study) of data and their subsequent processing in
collaboration with international specialists. The length of the study
is 4 years.

The successful applicant will be supervised by Dr. Stepan Janecek and
Dr. Robert Tropek (<a href="http://bit.ly/1Y12Q2N">http://bit.ly/1Y12Q2N</a>) as a part of the
multidisciplinary team at the Faculty of Science, Charles University of
Prague with a close collaboration with the Institute of Entomology and
Institute of Botany, Czech Academy of Sciences.

*Offered*
-          attractive scientific topic in an established
           international team
-          standard university scholarship (up to 114,000 CZK annually
           according to discharging of the study responsibilities) and a
           part time employment (132,000 CZK annually), it sufficiently
           covers living expenses in the Czech Republic

*Required*
-          enthusiasm in nature and ecological science
-          ability to lead a field research in challenging conditions of
           tropical environments
-          fluency in English
-          a MSc degree in biology or related fields (in early summer
           2016 at the latest)

Desirable (but not necessary)
-          previous experience of collaboration in scientific projects
           evidenced by a (co)authorship of research papers or
           conference contributions
-          previous experience with botany and/or pollination biology
-          basic knowledge of French

All applicants will send a structured CV, contacts of three referees,
and a cover letter stating their previous work, qualification and
motivation to both emails of Robert Tropek (robert.tropek@gmail.com)
and Stepan Janeek (janecek.stepan@centrum.cz). Any questions should be
sent to the same emails as well. Review of applications will begin on
5th January 2016 and will continue until the position has been filled.

Selected applicants will be interviewed through Skype in half January
2016. The position is available from early summer 2016.

Robert Tropek 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:10</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134845568416">Postdoc: Auckland.EvolutionMetabolicSystems</a></div>
  <div class="description">Post Doctoral Research Fellow

Institute of Natural and Mathematical Sciences, College of Sciences
Albany

The Institute of Natural & Mathematical Sciences and The NZ Institute
of Advanced Study seek to appoint a Postdoctoral Research Fellow for
the research group of Prof. Thomas Pfeiffer.

In this appointment you will be expected to contribute to the group&rsquo;s
research activities in the modelling of metabolic systems and their
evolution. You will have a strong background in mathematical modelling,
theoretical biology or computational biology is of advantage, as is a
track record in metabolic modelling and the analysis of genomic data.

This position is for three years, requires a PhD qualification, and is
based at the Auckland campus. Please contact Prof. Thomas Pfeiffer for
further information (email: T.Pfeiffer@massey.ac.nz).

Closing date: 17 January 2016
Reference number: A403-15SF

For further information and to apply online, visit:
<a href="http://bit.ly/1Qer1fa">http://bit.ly/1Qer1fa</a>




Dani Smith | Account Executive
DDI  04 914 4652


AUCKLAND | WELLINGTON | CHRISTCHURCH
0800 170 019


Dani Smith 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:25</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134844339386">Postdoc: UAlgarve_Portugal.HybridizationPhylogenomics</a></div>
  <div class="description">Post-Doctoral Research Fellowship – 1 vacancy

 Fellowship Reference: CCMAR/BPD/0025/2015

 The Algarve Centre for Marine Sciences opens a position for
 a Post-doctoral research fellowship (BPD)  within the project
 EXCL/AAG-GLO/0661/2012, also known as “Extant-  Extant or extinct
 tipping points – climate changes drive genetic diversity and dynamics
 of range edge populations as evolutionary hotspots.” financed by the
 Portuguese Foundation for Science and Technology - FCT/MCTES (PIDDAC)
 under the following conditions:



Scientific tasks: to use next generation sequencing (RNA-seq
transcriptome) datasets and phylogenomic methods to address questions
concerning the occurrence and extent of hybridization within a family
of brown algae (Fucacaea, Phaeophyceae).

Scientific Orientation: Ester A. Serrão, Gareth Pearson, Cymon Cox,
João Neiva (Univ. Algarve and CCMAR)

Work Place: The Algarve Centre of Marine Sciences (Faro, Portugal)



Requirements:

1.     Must be fluent in spoken and written in English (the latter to be
       assessed from the motivation letter to be submitted with the
       application);

2.     Must have a PhD and publications in an area related to the
       work plan.

Evaluation Criteria:

1.     Experience in manipulation and analysis of large-scale sequence
       datasets and/or knowledge of phylogenetic/phylogenomic
       methodologies – 50%

2.     Bioinformatics experience including familiarity with Unix-based
       systems, programming/scripting languages (e.g., R, perl/python or
       similar). – 50%


The jury will exercise the right to contact and interview candidates
should they require further information or clarification about their
application. In the event that the applicants do not have the necessary
profile the jury reserves the right to close the call without any
recruitment.

During the fellowship and in the event that the fellowship holder should
resign, the jury reserves the right, upon convenience and opportunity,
to assign the fellowship to the next applicant in accordance to the
ranking position.

Grant conditions: The grant will have an initial duration of 5 months,
starting in February 2016 eventually renewed for additional periods
until the end of the research project and eventually renewed after
the project with the financial support of FCT/MEC through national
funds and when applicable through cofounding by FEDER, within the
Partnership Agreement PT2020. The fellowship must be exclusively
according to the Portuguese law 40/2004, of 18 of August (Scientific
Research Fellowship holder statutes), Fellowship Regulation of
the Portuguese Foundation for Science and Technology in force
(<a href="http://bit.ly/1NFARV6">http://bit.ly/1NFARV6</a>)
and the rules of CCMAR research grants
<a href="http://bit.ly/1lNXmMC.">http://bit.ly/1lNXmMC.</a>

Monthly Maintenance Allowance: net 1495.00€/per month – BPD, tax free
(in accordance with the grant amounts set by the Portuguese Foundation
for Science and Technology).

Application Period: From 23 December 2015  to 11 January 2016.

Application Documents: Applicants should submit a letter of motivation
(written in english, and explaining the experience in each evaluation
criteria), a copy of their PhD certificate, a detailed Curriculum Vitae,
copies of their  national identity card (or passport) and fiscal number,
and the contacts (e-mail and phone) of 3 persons who can provide
references.

Submmission of applications: Applications should be submitted by email to
ccmar@ualg.pt indicating the reference CCMAR/BPD/0025/2015 (note that we
can only accept applications sent through this email). You should only
consider your application as validated only after receiving by email a
confirmation from CCMAR.

Results: The results will be communicated in writing within 30 working
days after the application deadline.

Jury responsible for the selection:

Ester A. Serrão, Gareth Pearson, João Neiva

informal contact: eserrao@ualg.pt


CCMAR’s non-discrimination and equal access opportunities policy –
No candidate can be privileged, benefited, jeopardised or private of any
right or exempt of any claim in regards to descendent, age, gender, sexual
orientation, marital status, economic situation, origin, social economy,
genetic heritage, disability, chronic disease, nationality, ethnicity,
territory of origin, language, religion, politics and ideological beliefs
and union membership.

Faro, 7 December  2015



Ester Serrão 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:10</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134843865081">Conference: DurhamNC.EvolutionaryMedicineSociety.Jun24-27</a></div>
  <div class="description"><p>Abstract submission and registration are now open for the  2016 meeting
of the International Society for Evolution, Medicine and Public Health .
<a href="http://evmed.org/">http://evmed.org</a>

The 2016 annual meeting of the International Society for Evolution,
Medicine and Public Health (ISEMPH) will take place from June 22-25, 2016
in Durham, North Carolina, thanks to host Charles Nunn and the Triangle
Center for Evolutionary Medicine.

Hundreds of scientists, students and health professionals will attend
including six invited plenary speakers: Andrea Graham (Princeton
University), Carl Zimmer (New York Times), Helen Ball (Durham University,
UK), Joshua Schiffman (University of Utah), Marion Koopmans (Erasmus
University) and Martin Blaser (New York University).

The Program Committee welcomes your abstracts for oral or poster
presentations on all topics in the field of evolution, medicine and public
health.

Full information about registration and abstract submission is at  http://:
<a href="http://bit.ly/1NaMZMq.">http://bit.ly/1NaMZMq.</a>

January 30, 2016



Deadline for abstract submission and early registration

We look forward to seeing you for a memorable and exciting meeting!

Randolph Nesse, President, ISEMPH
Sir Peter Gluckman, Vice-President, ISEMPH
Cynthia Beall, Treasurer, ISEMPH, Chair, Program Committee
Charles Nunn, Chair, Meeting Planning Committee

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 8, 2015</h3>
<div class="news-item">
 <div class="date">23:55</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134843382271">Postdoc: OpenUIsrael_EvolutionaryGenomics</a></div>
  <div class="description">&ndash;_000_08E2D2F84C204940AB26F5255D8544AC7330C232SardineOPENULAN_
Content-Type: text/plain; charset=&ldquo;us-ascii&rdquo;
Content-Transfer-Encoding: quoted-printable

Our group at the Open University, Raanana, Israel, invites applications for a postdoc position in Evolutionary Genomics.
Our group research interests are focused on Endosimbiotic Gene Transfer. In our research we use computational approach the study the abundance and significant of naturally occurring organelle-to-nucleus gene transfers.
Candidate qualifications: (1) Highly motivated person with a PhD degree in Molecular Evolution, Bioinformatics or related fields (2) Proficiency in computer programming (preferable - PERL/Python). (3) Any of following expertise is an advantage: phylogenomics, genomics, biostatistics. (3) Good oral and written communication skills (English is required). (4) Skills and motivation to communicate and interact with other scientists.
This position is funded by the German-Israeli grant (awarded to Dr. Hazkani-Covo and Prof. Dr. William Martin, Heinrich Heine University Duesseldorf, Germany). Applicant will have the opportunity to travel to the German partner lab. Initial appointment will be for 1 year with a possible extension to a total of 3 years.

Applicants should send curriculum vitae with a list of publications, a motivation statement and contact addresses of at least two referees to Dr. Einat Hazkani-Covo (einatco@openu.ac.il). For enquiries regarding the position please contact Dr. Einat Hazkani-Covo (einatco@openu.ac.il).

Dr. Einat Hazkani-Covo,
Department of Natural and Life Sciences
The Open University of Israel
1 University Road
P.O Box 808
Ra'anana 43537 ISRAEL

Tel:  972-9-7780981
<a href="http://bit.ly/1KYuxF8">http://bit.ly/1KYuxF8</a>


&ndash;_000_08E2D2F84C204940AB26F5255D8544AC7330C232SardineOPENULAN_
Content-Type: text/html; charset=&ldquo;us-ascii&rdquo;
Content-Transfer-Encoding: quoted-printable



</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">22:30</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2015/12/lexicostatistics-predecessor-of.html">Lexicostatistics: the predecessor of phylogenetic analyses in historical linguistics</a></div>
  <div class="description"><br />Phylogenetic approaches in historical linguistics are extremely common nowadays.  Especially, probabilistic models that model lexical change as a birth-death process of cognate sets evolving along a phylogenetic tree (<a href="http://bibliography.lingpy.org/?key=Pagel2009">Pagel 2009</a>) are very popular (<a href="http://bibliography.lingpy.org/?key=Lee2011">Lee and Hasegawa 2011</a>, <a href="http://bibliography.lingpy.org/?key=Kitchen2009">Kitchen et al. 2009</a>, <a href="http://bibliography.lingpy.org/?key=Bowern2012">Bowern and Atkinson 2012</a>), but also splits networks are frequently used (<a href="https://www.blogger.com/:ref:Hamed2005">Ben Hamed 2005</a>, <a href="https://www.blogger.com/:ref:Heggarty2010">Heggarty et al. 2010</a>).<br /><br />However, the standard procedure to produce a family tree or network with phylogenetic software in linguistics goes back to the method of <b>lexicostatistics</b>, which was developed in the 1950s by <a href="http://en.wikipedia.org/wiki/:Morris_Swadesh">Morris Swadesh</a> (1909-1967) in a series of papers (<a href="http://bibliography.lingpy.org/?key=Swadesh1950">Swadesh 1950</a>, <a href="http://bibliography.lingpy.org/?key=Swadesh1952">1952</a>, <a href="http://bibliography.lingpy.org/?key=Swadesh1955">1955</a>). Lexicostatistics was discarded by the linguistic community not long after it was proposed (<a href="http://bibliography.lingpy.org/?key=Hoijer1956">Hoijer 1956</a>, <a href="http://bibliography.lingpy.org/?key=Bergsland1962">Bergsland and Vogt 1962</a>). Since then, lexicostatistics is considered a <i>methodus non gratus</i> in classical circles of historical linguistics, and using it openly may drastically downgrade one's perceived credibility in certain parts of the community.<br /><br />To avoid the conflicts, most linguists practicing modern phylogenetic approaches emphasize the fundamental differences between early lexicostatistics and modern phylogenetics. These differences, however, apply only to the way the data is analysed. The basic assumptions underlying the selection and preparation of data have not changed since the 1950s, and it is important to keep this in mind, especially when searching for appropriate phylogenetic models to analyse the data.<br /><br /><b>The Theory of Basic Vocabulary</b><br /><br />Swadesh's basic idea was that in the lexicon of every human language there are words that are <i>culturally neutral</i> and <i>functionally universal</i>; and he used the term "basic vocabulary" to refer to these words.  Culturally neutral hereby means that the meanings expressed by the words are independently used across different cultures. Functional universality means that the meanings are expressed by all human languages independent of the time and place where they are spoken.  The idea is that these meanings are so important for the functioning of a language as a tool of communication, that every language needs to express them.<br /><br />Cultural neutrality and functional universality guarantee two important aspects of basic words: their <i>stability</i> and their <i>resistance to borrowing</i>. Stability means that words expressing a basic concept are less likely to change their meaning or to be replaced by another word. An argument for this claim is the functional importance of the words — if the words are important for the functioning of a language, it would not make much sense to change them too quickly.  Humans are good at changing the meanings of words, as we can see from daily conversations in the media, where new words tend to pop up seemingly on a daily basis, and old words often drastically change their meanings.  But changing words that express basic meanings like "head", "stone", "foot", or "mountain" too often might give rise to confusion in communication. As a result, one can assume that words change at a different pace, depending on the meaning they express, and this is one of the core claims of lexicostatistics.<br /><br />Resistance to borrowing follows also from stability, since the replacement of words expressing basic meanings may again have an impact on our daily communication, and we may thus assume that speakers avoid borrowing these words too quickly. Cultural neutrality of concepts is another important point to guarantee resistance to borrowing. Words expressing concepts which play an important cultural role may easily be transferred from one language to another along with the culture. Thus, although it seems likely that every language has a word for "god" or "spirit" and the like (so the concept is to a certain degree functionally universal), the lack of cultural independency makes words expressing religious terms very likely candidates for borrowing, and it is probably no coincidence that words expressing religion and belief rank first in the scale of borrowability (<a href="http://bibliography.lingpy.org/?key=Tadmor2009">Tadmor 2009: 232</a>).<br /><br /><b>Lexical Replacement, Data Preparation, and Divergence Time Estimation</b><br /><br />Swadesh had further ideas regarding the importance of basic vocabulary. He assumed that the process of <i>lexical replacement</i>&nbsp;follows universal rates as far as the basic vocabulary is concerned, and that this would allow us to date the divergence of languages, provided we are able to identify the shared cognates.  In lexical replacement, a word <i>w₁</i> expressing a given meaning <i>x</i> in a language is replaced by a word <i>w₂</i> which then expresses the meaning <i>x</i>, while <i>w₁</i> either shifts to express another meaning, or completely disappears from the language. For example, older <i>thou</i> did in English was replaced by the plural form <i>you</i>, which now also expresses the singular. In order to search for cognates and determine the time when two languages diverged, Swadesh proposed a straightforward procedure, consisting of very concrete steps (compare <a href="http://bibliography.lingpy.org/?key=Dyen1992">Dyen et al. 1992</a>): <br /><ul><li>Compile a list of basic concepts (concepts that you think are culturally neutral and functionally universal; see <a href="http://concepticon.clld.org/">here</a> for a comparative collection of different lists that have been proposed and used in the past)</li><li>translate these concepts into the different languages you want to analyse</li><li>search for cognates between the languages in each meaning slot; if words in two languages are not cognate for a given meaning, then this points to former processes of lexical replacement in at least one of the languages since their divergence</li><li>count the number of shared cognates, and use some mathematics to calculate the divergence time (which has been independently calibrated using some test cases of known divergence times).</li></ul>As an example for such a wordlist with cognate judgments, compare the table in the first figure, where I have entered just a few basic concepts from Swadesh's standard concept list and translated them into four languages. Cognacy is assigned with help of IDs in the column at the right of each language column, but also further highlighted with different colors. <br /><br /><a href="http://4.bp.blogspot.com/-2RoFbOVUXu8/VmahRzx0III/AAAAAAAAAJY/TMqU_Evp6Pg/s1600/cognates-1.png" imageanchor="1"></a>Classical cognate coding in lexicostatistics<br /><b>Phylogenetic Approaches in Historical Linguistics</b><br /><br />Modern phylogenetic approaches in historical linguistics basically follow the same workflow that Swadesh propagated for lexicostatistics, the only difference being the last step of the working procedure. Instead of Swadesh's formula, which compared lexical replacement with radioactive decay and was based on aggregated distances in its core, character-based methods are used to infer phylogenetic trees.  Characters are retrieved from the data by extracting each cognate from a lexicostatistical wordlist and annotating the presence or absence of each cognate set in each language.<br /><br />Thus, while Swadesh's lexicostatistical data model would state that the words for "hand" in German and English were cognate, and also in Italian and French, but not in Germanic and Romance, the binary presence-absence coding states that the cognate set formed by words like English <i>hand</i> and German <i>Hand</i> is not present in Romance languages, and that the cognate set formed by words like Italian <i>mano</i> and French <i>main</i> is absent in Germanic languages. This is illustrated in the table below, where the same IDs and colors are used to mark the cognate sets as in the table shown above.<br /><br /><a href="http://4.bp.blogspot.com/-LfzpRrY_BBQ/VmahTTBXX8I/AAAAAAAAAJk/ZNWjZq08W0Y/s1600/cognates-2.png" imageanchor="1"></a>Presence-absence cognate coding for modern phylogenetic analyses<br />The new way of cognate coding along with the use of phylogenetic software methods has brought, without doubt, many improvements compared to Swadesh's idea of dating divergence times by counting percentages of shared cognates. A couple of problems, however, remain, and one should not forget them when applying computational methods to originally lexicostatistic datasets.<br /><br />First, we could ask whether the main assumptions of functional universality and cultural neutrality really hold. It seems to be true that words can be remarkably stable throughout the history of a language family. It is, however, also true that the most stable words are not necessarily the same across all language families. Ever since Swadesh established the idea of basic vocabulary, scholars have tried to improve the list of basic vocabulary items. Swadesh himself started from a list of 215 concepts (<a href="http://bibliography.lingpy.org/?key=Swadesh1950">Swadesh 1950</a>), which he then reduced to 200 concepts (<a href="http://bibliography.lingpy.org/?key=Swadesh1952">1952</a>) and then later to 100 concepts (<a href="http://bibliography.lingpy.org/?key=Swadesh1955">1952</a>). Other scholars went further, like Dolgopolsky (<a href="http://bibliography.lingpy.org/?key=Dolgopolsky1964">1964 [1986]</a>) who reduced the list to 16 concepts. The <a href="http://concepticon.clld.org/">Concepticon</a> is a resource that links many of the concept lists that have been proposed in the past.&nbsp;When comparing these lists, which all represent what some scholars would label "basic vocabulary items", it becomes obvious that the number of items that all scholars agree upon sinks drastically, while the number of concepts that have been claimed to be basic increases.<br /><br />An even greater problem than the question of universality and neutrality of basic vocabulary, however, is the underlying model of cognacy in combination with the proposed process of change.  Swadesh's model of cognacy controls for meaning. While this model of cognacy is consistent with Swadesh's idea of lexical replacement as a basic process of lexical change, it is by no means consistent with birth-death models of cognate gain and cognate loss if they are created from lexicostatistical data. In biology, birth-death models are usually used to model the evolution of homologous gene families distributed across whole genomes. If we use the traditional view according to which words can be cognate regardless of meaning, the analogy holds, and birth-death processes seem to be adequate in order to analyze datasets that are based on these <i>root cognates</i> (<a href="http://bibliography.lingpy.org/?key=Starostin1989">Starostin 1989</a>) or <i>etymological cognates</i> (<a href="http://bibliography.lingpy.org/?key=Starostin2013">Starostin 2013</a>). But if we control for meaning in the cognate judgments, we do not necessarily capture processes of gain and loss in our data. Instead, we capture processes in which links between word forms and concepts are shifted, and we investigate these shifts through the very narrow "windows" of pre-defined slots of basic concepts, as I have tried to depict in the following graphic.<br /><br /><a href="http://2.bp.blogspot.com/-ILItz0MUjQw/Vmak55Hh6YI/AAAAAAAAAJ0/yueeOxGBMng/s1600/rect8062-5-5-4-9-7.jpg" imageanchor="1"></a>Looking at kexical replacement through the small windows of basic vocabulary<br /><b>Conclusion</b><br /><br />As <a href="http://phylonetworks.blogspot.fr/2015/11/are-realistic-mathematical-models.html">David has mentioned before</a>: We do not necessarily need realistic models in phylogenetic research to infer meaningful processes. The same can probably be said about the discrepancy between our lexicostatistical datasets (Swadesh's heritage, which we keep using for practical reasons) and the birth-death models we now use to analyse the data. Nevertheless, I cannot avoid an uncomfortable feeling when thinking that an algorithm is modeling gain and loss of characters in a dataset that was not produced for this purpose. In order to model the traditional lexicostatistical data consistently, we would either (i) need explicit multistate-models in which concepts are a character and the forms represent the states (<a href="http://bibliography.lingpy.org/?key=Ringe2002">Ringe et al. 2002</a>, <a href="http://bibliography.lingpy.org/?key=Hamed2006">Ben Hamed and Wang 2006</a>), or (ii) we should directly turn to "root-cognate" methods. These methods have been discussed for some time now (<a href="http://bibliography.lingpy.org/?key=Starostin1989">Starostin 1989</a>, <a href="http://bibliography.lingpy.org/?key=Holm2000">Holm 2000</a>), but there is only one <a href="http://escholarship.org/uc/item/1331v899">recent approach by Michael et al. (forthcoming)</a> in which this is consistently tested. <br /><br /><b>References</b><br /><ul><li>Bergsland, K. and H. Vogt (1962): <b>On the validity of glottochronology</b>. <i>Curr. Anthropol.</i> 3.2. 115-153.</li><li>Bowern, C. and Q. Atkinson (2012): <b>Computational phylogenetics of the internal structure of Pama-Nguyan</b>. <i>Language</i> 88. 817-845.</li><li>Dolgopolsky, A. (1964): <b>Gipoteza drevnejšego rodstva jazykovych semej Severnoj Evrazii s verojatnostej točky zrenija</b> [A probabilistic hypothesis concering the oldest relationships among the language families of Northern Eurasia]. <i>Voprosy Jazykoznanija</i> 2. 53-63.</li><li>Dyen, I., J. Kruskal, and P. Black (1992): <b>An Indoeuropean classification</b><b>. A lexicostatistical experiment</b>. <i>T. Am. Philos. Soc.</i> 82.5. iii-132.</li><li>Ben Hamed, M. and F. Wang (2006): <b>Stuck in the forest: Trees, networks and Chinese dialects</b>. <i>Diachronica</i> 23. 29-60.</li><li>Hoijer, H. (1956): <b>Lexicostatistics</b><b>. A critique</b>. <i>Language</i> 32.1. 49-60.</li><li>Holm, H. (2000): <b>Genealogy of the main Indo-European branches applying the separation base method</b>. <i>J. Quant. Linguist.</i> 7.2. 73-95.</li><li>Kitchen, A., C. Ehret, S. Assefa, and C. Mulligan (2009): <b>Bayesian phylogenetic analysis of Semitic languages identifies an Early Bronze Age origin of Semitic in the Near East</b>. <i>Proc. R. Soc. London, Ser. B</i> 276.1668. 2703-2710.</li><li>Lee, S. and T. Hasegawa (2011): <b>Bayesian phylogenetic analysis supports an agricultural origin of Japonic languages</b>. <i>Proc. R. Soc. London, Ser. B</i> 278.1725. 3662-3669.</li><li>Pagel, M. (2009): <b>Human language as a culturally transmitted replicator</b>. <i>Nature Reviews. Genetics</i> 10. 405-415.</li><li>Ringe, D., T. Warnow, and A. Taylor (2002): <b>Indo-European and computational cladistics</b>. <i>T. Philol. Soc.</i> 100.1. 59-129.</li><li>Starostin, S. (1989): <b>Sravnitel'no-istoričeskoe jazykoznanie i leksikostatistika</b> [Comparative-historical linguistics and lexicostatistics]. In: Kullanda, S., J. Longinov, A. Militarev, E. Nosenko, and V. Shnirel'man (eds.): <i>Materialy k diskussijam na konferencii</i>[Materials for the discussion on the conference].1. Institut Vostokovedenija: Moscow. 3-39.</li><li>Starostin, G. (2013): <b>Lexicostatistics as a basis for language classification</b>. In: Fangerau, H., H. Geisler, T. Halling, and W. Martin (eds.): <i>Classification and evolution in biology, linguistics and the history of science. Concepts – methods – visualization.</i>. Franz Steiner Verlag: Stuttgart. 125-146.</li><li>Swadesh, M. (1950): <b>Salish internal relationships</b>. <i>Int. J. Am. Linguist.</i> 16.4. 157-167.</li><li>Swadesh, M. (1952): <b>Lexico-statistic dating of prehistoric ethnic contacts</b><b>. With special reference to North American Indians and Eskimos</b>. <i>Proc. Am. Philol. Soc.</i> 96.4. 452-463.</li><li>Swadesh, M. (1955): <b>Towards greater accuracy in lexicostatistic dating</b>. <i>Int. J. Am. Linguist.</i> 21.2. 121-137.</li><li>Tadmor, U. (2009): <b>Loanwords in the world’s languages</b><b>. Findings and results</b>. In: Haspelmath, M. and U. Tadmor (eds.): <i>Loanwords in the world's languages</i>. de Gruyter: Berlin and New York. 55-75.</li></ul><br /></div>
  <div class="source">Source: <a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></div>
 </div>
</div>
<h3>December 7, 2015</h3>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/276">Larval nutrition-induced plasticity affects reproduction and gene expression of the ladybeetle, Cryptolaemus montrouzieri</a></div>
  <div class="description">Background:
Organisms may develop into multiple phenotypes under different nutritional environments by developmental plasticity, whereas the potential costs and mechanisms of such plasticity are poorly understood. Here we examined the fitness and gene expression of nutrition-induced phenotypes in the ladybeetle, Cryptolaemus montrouzieri after having experienced varying larval food regimes.
Results:
We found that C. montrouzieri adults undergoing a variable larval food regime achieved a similar developmental time, survival, body mass and egg production as those undergoing a high larval food regime. The survival, developmental time, body mass and fecundity of the adults from a restricted larval food regime were inferior to those from the high and variable larval food regimes. However, the adults from this restricted larval food regime had a higher expression level of genes encoding immune- and antioxidant-related enzymes than those from the high and variable larval food regimes when exposed to starvation and pesticide conditions in adult life.
Conclusions:
These results suggest that larval food availability in C. montrouzieri not only triggers adult phenotypic differences but also affects reproduction and expression level of genes in adult life, indicating that the larval nutritional conditions can affect adult fitness and resistance to stressful conditions through developmental plasticity.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/275">A generalized birth and death process for modeling the fates of gene duplication</a></div>
  <div class="description">Background:
Accurately estimating the timing and mode of gene duplications along the evolutionary history of species can provide invaluable information about underlying mechanisms by which the genomes of organisms evolved and the genes with novel functions arose. Mechanistic models have previously been introduced that allow for probabilistic inference of the evolutionary mechanism for duplicate gene retention based upon the average rate of loss over time of the duplicate. However, there is currently no probabilistic model embedded in a birth-death modeling framework that can take into account the effects of different evolutionary mechanisms of gene retention when analyzing gene family data.
Results:
In this study, we describe a generalized birth-death process for modeling the fates of gene duplication. Use of mechanistic models in a phylogenetic framework requires an age-dependent birth-death process. Starting with a single population corresponding to the lineage of a phylogenetic tree and with an assumption of a clock that starts ticking for each duplicate at its birth, an age-dependent birth-death process is developed by extending the results from the time-dependent birth-death process. The implementation of such models in a full phylogenetic framework is expected to enable large scale probabilistic analysis of duplicates in comparative genomic studies.
Conclusions:
We develop an age-dependent birth-death model for understanding the mechanisms of gene retention, which allows a gene loss rate dependent on each duplication event. Simulation results indicate that different mechanisms of gene retentions produce distinct likelihood functions, which can be used with genomic data to quantitatively distinguish those mechanisms.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">07:11</div>
 <div class="body">
  <div class="title"><a href="https://www.researchgate.net/job/862821_Postdoctoral_Research_Officer-Molecular_Ecology?source=rss">Postdoctoral Research Officer - Molecular Ecology</a></div>
  <div class="description">Postdoctoral Research Officer

Job Number:&nbsp;BU01042
School/Department:&nbsp;School of Biological Sciences
Grade:&nbsp;7
Salary Information:&nbsp;&pound;31,656 - &pound;37,768 (Grade 7) p.a.
Contract Duration:&nbsp;3 years
Responsible to:&nbsp;Dr Simon Creer

Closing Date:&nbsp;21-12-2015

Applications are invited for the above 3 year fixed-term, full-time post working in the Molecular Ecology and Fisheries Genetics Laboratory, School of Biological Sciences, College of Natural Sciences. 

We are looking for a dynamic researcher to fulfil a leading role in a new, collaborative and multidisciplinary team working at the interface between the molecular genetic analysis of aerial pollen composition, aerobiological modelling and human health. Approximately 20% of the UK population suffer from summer allergies and the primary outdoor allergens are grass pollens. Nevertheless, there is currently no easy way of discriminating different species of grass pollen using traditional morphological approaches. Molecular biodiversity identification is emerging as a high throughput and cost effective alternative to existing approaches and in particular, combinations of qPCR, metabarcoding and shotgun sequencing provide a unique analytical suite of molecular approaches to assess grass pollen mixtures both quantitatively and qualitatively. The primary objectives of the broader project will be to (a.) develop a species level, spatio-temporal grass pollen molecular genetic assessment framework throughout the UK and (b.) develop novel pollen bio aerosol models in order ultimately to (c.) identify linkages between species-specific grass pollen loadings and human health (i.e. asthma exacerbations). 

The 3 year, &pound;1.2M project is funded by the NERC Standard Grant blue sky funding stream and involves combinations of local and national fieldwork in collaboration with Aberystwyth University, The National Pollen and Aerobiological Research Unit (NPARU), University of Worcester, Exeter University and the UK Met Office. The leading Bangor component of the project will utilise molecular ecology, including high throughput qPCR and genomic approaches to assess biodiversity, bioinformatics and ecological modelling, with the PDRO working closely with a diverse team of collaborators from the fields of plant metabarcoding, aerobiology and modelling and environmental epidemiology both in the UK and overseas. The project will also benefit from close links with a range of charities providing support in the broad field of allergenic and respiratory health. 

Candidates should possess a doctoral degree in related areas such as natural sciences, ecology, botany, molecular taxonomy, evolution/genomics and should ideally have previous experience of fieldwork, molecular ecology, high throughput sequencing, bioinformatics and ecological data handling. The successful candidate will possess excellent team working and communication skills. 

The successful candidate will be expected to commence 1st March 2016, or as soon after this as possible. 

Informal enquiries should be sent to Dr S. Creer, email s.creer@bangor.ac.uk;http://mefgl.bangor.ac.uk/staff/si.php; @spideycreer.</div>
  <div class="source">Source: <a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">07:11</div>
 <div class="body">
  <div class="title"><a href="http://www.researchgate.net/job/862821_Postdoctoral_Research_Officer-Molecular_Ecology?source=rss">Postdoctoral Research Officer - Molecular Ecology</a></div>
  <div class="description">Postdoctoral Research Officer

Job Number:&nbsp;BU01042
School/Department:&nbsp;School of Biological Sciences
Grade:&nbsp;7
Salary Information:&nbsp;&pound;31,656 - &pound;37,768 (Grade 7) p.a.
Contract Duration:&nbsp;3 years
Responsible to:&nbsp;Dr Simon Creer

Closing Date:&nbsp;21-12-2015

Applications are invited for the above 3 year fixed-term, full-time post working in the Molecular Ecology and Fisheries Genetics Laboratory, School of Biological Sciences, College of Natural Sciences. 

We are looking for a dynamic researcher to fulfil a leading role in a new, collaborative and multidisciplinary team working at the interface between the molecular genetic analysis of aerial pollen composition, aerobiological modelling and human health. Approximately 20% of the UK population suffer from summer allergies and the primary outdoor allergens are grass pollens. Nevertheless, there is currently no easy way of discriminating different species of grass pollen using traditional morphological approaches. Molecular biodiversity identification is emerging as a high throughput and cost effective alternative to existing approaches and in particular, combinations of qPCR, metabarcoding and shotgun sequencing provide a unique analytical suite of molecular approaches to assess grass pollen mixtures both quantitatively and qualitatively. The primary objectives of the broader project will be to (a.) develop a species level, spatio-temporal grass pollen molecular genetic assessment framework throughout the UK and (b.) develop novel pollen bio aerosol models in order ultimately to (c.) identify linkages between species-specific grass pollen loadings and human health (i.e. asthma exacerbations). 

The 3 year, &pound;1.2M project is funded by the NERC Standard Grant blue sky funding stream and involves combinations of local and national fieldwork in collaboration with Aberystwyth University, The National Pollen and Aerobiological Research Unit (NPARU), University of Worcester, Exeter University and the UK Met Office. The leading Bangor component of the project will utilise molecular ecology, including high throughput qPCR and genomic approaches to assess biodiversity, bioinformatics and ecological modelling, with the PDRO working closely with a diverse team of collaborators from the fields of plant metabarcoding, aerobiology and modelling and environmental epidemiology both in the UK and overseas. The project will also benefit from close links with a range of charities providing support in the broad field of allergenic and respiratory health. 

Candidates should possess a doctoral degree in related areas such as natural sciences, ecology, botany, molecular taxonomy, evolution/genomics and should ideally have previous experience of fieldwork, molecular ecology, high throughput sequencing, bioinformatics and ecological data handling. The successful candidate will possess excellent team working and communication skills. 

The successful candidate will be expected to commence 1st March 2016, or as soon after this as possible. 

Informal enquiries should be sent to Dr S. Creer, email s.creer@bangor.ac.uk;http://mefgl.bangor.ac.uk/staff/si.php; @spideycreer.</div>
  <div class="source">Source: <a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:48</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2015/12/recent-book-reviews.html">Recent book reviews</a></div>
  <div class="description"><br />In one of the earliest blog posts (<a href="http://phylonetworks.blogspot.com/2012/02/reviews-of-recent-books.html" target="_blank">Reviews of recent books</a>) I provided links to some book reviews. Recently, a few have appeared for Dan Gusfield's book: <i>ReCombinatorics: the algorithmics of ancestral recombination graphs and explicit phylogenetic networks</i> (2014. The MIT Press, Cambridge, MA).<br /><br />In addition to the three endorsements that appear as part of the publisher's blurb, a number of independent book reviews have appeared since its publication:<br /><br />(2014) <a href="http://www.computingreviews.com/review/Review_review.cfm?review_id=143064" target="_blank">Computing Reviews</a> Review#143064.<br /><br />Michael Sanderson (2015) <a href="http://www.jstor.org/stable/10.1086/682642" target="_blank">Quarterly Review of Biology</a> 90: 344-345.<br /><br />Luay Nakhleh (2015) <a href="http://epubs.siam.org/doi/abs/10.1137/15N97399X" target="_blank">SIAM Reviews</a> 57: 638-642.<br /><br />From the mathematical point of view, the reviews make it clear that this book is necessary because networks are very much part of the fringe of the computational sciences. Indeed, the challenge is to convince mathematicians that interesting mathematical problems exist with the the study of networks. In this sense, the main limitation of the book is its focus on the parsimony criterion for optimization, rather than statistical approaches to inference, which play such a large part in phylogenetic analyses.<br /><br />From the biological point of view, the principal issue seems to be the reliance of the book on the infinite sites model, which does not currently have wide applicability in phylogenetics (eg. mostly in population studies such as haplotype inference and association mapping).<br /><br />The ultimate goal for both computational end biological scientists is working out how to include recombination in the framework of other types of phylogenetic networks. A basic assumption of many phylogenetic analyses is that there has been no recombination. This is because recombination can destroy much of the evidence left by historically preceding processes, so that neither genotype nor phenotype data can reveal patterns and processes that pre-date the recombination events. In this sense, recombination becomes the reticulation process, rather than processes like hybridization or introgression.<br /><br /></div>
  <div class="source">Source: <a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></div>
 </div>
</div>
<h3>December 6, 2015</h3>
<div class="news-item">
 <div class="date">23:30</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134710679836">Postdoc: ULB.Belgium.Fellowships_for_Refugees</a></div>
  <div class="description"><p>Please find below an announcement for postdoctoral fellowships for 
refugees at the ULB (Université libre de Bruxelles) in Belgium. 
Evolutionary biologists fleeing war-torn countries (particularly Iraq 
and Syria) are welcome to apply.

<a href="http://bit.ly/1RyPJGy">http://bit.ly/1RyPJGy</a>

ONE-YEAR (RENEWABLE) POST-DOCTORAL KHALED AL’AS-AD FELLOWSHIPS,
FOR REFUGEES HOLDING A PH.D

ELIGIBILITY:
The applicant:
• must have refugee status or, at the moment they apply for the 
fellowship, have submitted an application for refugee to the immigration 
office of a member state of the European Union;
• must not have resided or carried out their main activity (work, 
studies, etc.) in Belgium for more than 24 months in the 3 years 
immediately preceding recruitment (mobility rule).

APPLICATIONS:
Applications must be sent by email to Angélique Greindl 
(agreindl@ulb.ac.be) by February 15th 2016 at the latest.
Applications must include:
• A research plan describing the scientific framework, the questions to 
be addressed and the methodology.
• A copy of the PhD diploma or proof that the candidate successfully 
passed all exams related to the PhD programme. If these documents cannot 
be provided, applicants will have to provide a clear career description 
proving that they have at least four years of full-time research 
experience by the time of the call for applications.
• A CV, including a list of publications.
• Two letters of reference from scientists working in the same field 
including, if possible, one from their PhD supervisor.
• A letter of acceptance from a member of ULB academic staff confirming 
that the candidate, if successful, will be welcome in their group and be 
offered the appropriate scientific environment.
• A copy of the applicant’s passport, a copy of the document confirming 
refugee status if this has already been granted, and, if the applicant 
is already in Belgium, a copy of an official document confirming date of 
entry to Belgium. In the event of a successful application, the 
fellowship will be awarded subject to the obtaining of refugee status.

The Research Department (agreindl@ulb.ac.be) and the International 
Service in the ULB’s External Relations Department can help candidates 
identify and contact members of ULB academic staff working in their 
field, in order to complete their application and design their research 
project.

SELECTION PROCEDURE:
End February/early March, applications will be submitted to the Board of 
the Research Council, comprising members of academic staff from the main 
research domains (Human and Social Sciences, Science and Technology, 
Life Sciences) 
(<a href="http://bit.ly/1TRljhW">http://bit.ly/1TRljhW</a>), 
and expanded to include the Vice-Rector for International Relations and 
the Vice-Rector in charge of
the coordination of the ULB’s actions in favour of Syrian and Iraqi 
refugees. The panel members will receive a copy of all applications at 
least two weeks before the panel meeting. They will be asked to make 
their selection by ranking applications on the basis of the following 
three criteria: the quality of the applicant, the quality of the 
research project and the quality of the scientific environment.



via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/274">Distribution of mutational fitness effects and of epistasis in the 5&#8217; untranslated region of a plant RNA virus</a></div>
  <div class="description">Background:
Understanding the causes and consequences of phenotypic variability is a central topic of evolutionary biology. Mutations within non-coding cis-regulatory regions are thought to be of major effect since they affect the expression of downstream genes. To address the evolutionary potential of mutations affecting such regions in RNA viruses, we explored the fitness properties of mutations affecting the 5’-untranslated region (UTR) of a prototypical member of the picorna-like superfamily, Tobacco etch virus (TEV). This 5’ UTR acts as an internal ribosomal entry site (IRES) and is essential for expression of all viral genes.
Results:
We determined in vitro the folding of 5’ UTR using the selective 2’-hydroxyl acylation analyzed by primer extension (SHAPE) technique. Then, we created a collection of single-nucleotide substitutions on this region and evaluated the statistical properties of their fitness effects in vivo. We found that, compared to random mutations affecting coding sequences, mutations at the 5’ UTR were of weaker effect. We also created double mutants by combining pairs of these single mutations and found variation in the magnitude and sign of epistatic interactions, with an enrichment of cases of positive epistasis. A correlation exists between the magnitude of fitness effects and the size of the perturbation made in the RNA folding structure, suggesting that the larger the departure from the predicted fold, the more negative impact in viral fitness.
Conclusions:
Evidence that mutational fitness effects on the short 5’ UTR regulatory sequence of TEV are weaker than those affecting its coding sequences have been found. Epistasis among pairs of mutations on the 5’ UTR ranged between the extreme cases of synthetic lethal and compensatory. A plausible hypothesis to explain all these observations is that the interaction between the 5’ UTR and the host translational machinery was shaped by natural selection to be robust to mutations, thus ensuring the homeostatic expression of viral genes even at high mutation rates.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/273">Saprophytic and pathogenic fungi in the Ceratocystidaceae differ in their ability to metabolize plant-derived sucrose</a></div>
  <div class="description">Background:
Proteins in the Glycoside Hydrolase family 32 (GH32) are carbohydrate-active enzymes known as invertases that hydrolyse the glycosidic bonds of complex saccharides. Fungi rely on these enzymes to gain access to and utilize plant-derived sucrose. In fungi, GH32 invertase genes are found in higher copy numbers in the genomes of pathogens when compared to closely related saprophytes, suggesting an association between invertases and ecological strategy. The aim of this study was to investigate the distribution and evolution of GH32 invertases in the Ceratocystidaceae using a comparative genomics approach. This fungal family provides an interesting model to study the evolution of these genes, because it includes economically important pathogenic species such as Ceratocystis fimbriata, C. manginecans and C. albifundus, as well as saprophytic species such as Huntiella moniliformis, H. omanensis and H. savannae.
Results:
The publicly available Ceratocystidaceae genome sequences, as well as the H. savannae genome sequenced here, allowed for the identification of novel GH32-like sequences. The de novo assembly of the H. savannae draft genome consisted of 28.54 megabases that coded for 7 687 putative genes of which one represented a GH32 family member. The number of GH32 gene family members appeared to be related to the ecological adaptations of these fungi. The pathogenic Ceratocystis species all contained two GH32 family genes (a putative cell wall and a putative vacuolar invertase), while the saprophytic Huntiella species had only one of these genes (a putative cell wall invertase). Further analysis showed that the evolution of the GH32 gene family in the Ceratocystidaceae involved transposable element-based retro-transposition and translocation. As an example, the activity of a Fot5-like element likely facilitated the assembly of the genomic regions harbouring the GH32 family genes in Ceratocystis.
Conclusions:
This study provides insight into the evolutionary history of the GH32 gene family in Ceratocystidaceae. Our findings suggest that transposable elements shaped the evolution of the GH32 gene family, which in turn determines the sucrolytic activities and related ecological strategies of the Ceratocystidaceae species that harbour them. The study also provides insights into the role of carbohydrate-active enzymes in plant-fungal interactions and adds to our understanding of the evolution of these enzymes and their role in the life style of these fungi.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/272">The emergence of DNA in the RNA world: an in silico simulation study of genetic takeover</a></div>
  <div class="description">Background:
It is now popularly accepted that there was an “RNA world” in early evolution of life. This idea has a direct consequence that later on there should have been a takeover of genetic material – RNA by DNA. However, since genetic material carries genetic information, the “source code” of all living activities, it is actually reasonable to question the plausibility of such a “revolutionary” transition. Due to our inability to model relevant “primitive living systems” in reality, it is as yet impossible to explore the plausibility and mechanisms of the “genetic takeover” by experiments.
Results:
Here we investigated this issue by computer simulation using a Monte-Carlo method. It shows that an RNA-by-DNA genetic takeover may be triggered by the emergence of a nucleotide reductase ribozyme with a moderate activity in a pure RNA system. The transition is unstable and limited in scale (i.e., cannot spread in the population), but can get strengthened and globalized if certain parameters are changed against RNA (i.e., in favor of DNA). In relation to the subsequent evolution, an advanced system with a larger genome, which uses DNA as genetic material and RNA as functional material, is modeled – the system cannot sustain if the nucleotide reductase ribozyme is “turned off” (thus, DNA cannot be synthesized). Moreover, the advanced system cannot sustain if only DNA’s stability, template suitability or replication fidelity (any of the three) is turned down to the level of RNA’s.
Conclusions:
Genetic takeover should be plausible. In the RNA world, such a takeover may have been triggered by the emergence of some ribozyme favoring the formation of deoxynucleotides. The transition may initially have been “weak”, but could have been reinforced by environmental changes unfavorable to RNA (such as temperature or pH rise), and would have ultimately become irreversible accompanying the genome’s enlargement. Several virtues of DNA (versus RNA) – higher stability against hydrolysis, greater suitability as template and higher fidelity in replication, should have, each in its own way, all been significant for the genetic takeover in evolution. This study enhances our understandings of the relationship between information and material in the living world.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:46</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134643921386">Graduate position: UPretoria.3.WoodwaspEvolution</a></div>
  <div class="description"><p>Three PhD positions study the molecular, chemical and behavioral ecology 
of Sirex woodwasp – University of Pretoria, South Africa

Three PhD positions are available to study the molecular, chemical and 
behavioral ecology of Sirex woodwasp – Amylostereum fungus – Deladenus 
nematode symbioses.  These positions have Forestry and Agricultural 
Biotechnology Institute (FABI), University of Pretoria, South Africa as 
the home institution, but will involve extensive collaborative work with 
the Canadian Forest Service (CFS) and the University of the Sunshine 
Coast in Australia.

The projects
Invasive pests are one of the most significant threats facing plant 
ecosystems and production systems globally; from agriculture to forests 
and forestry. Sirex noctilio and its fungal mutualist, Amylostereum 
areolatum, is a growing model system for understanding processes that 
affect these invasions, as well as the development and implementation of 
management tools for it. One of the primary management tools for this 
invasive pest is biological control using the nematode Deladenus 
siricidicola.

1. How and which visual and olfactory stimuli affect the behavior of 
Sirex noctilio, and how can these be integrated in management programs? 
Potential areas of inquiry may include elucidating kairomone, pheromone, 
visual and genomic stimuli that influence the behavior of S. noctilio. 
The consequences of these stimuli and their potential to contribute 
management tools will be actively explored.
2.  What factors influence the reproductive biology of Sirex noctilio, 
and how does this influence invasive populations? Potential areas of 
inquiry may include elucidating physiological and behavioral traits that 
influence the complex and fascinating reproductive system of this wasp. 
Ultimately the successful candidate will explore molecular tools to 
manipulate outcomes of sexual reproduction.
3.  How do invasion processes and micro-evolution affect symbioses, 
invasive pest adaptation and their biological control agents? The 
project, termed ‘Petri-dish Australasia’ will use the Sirex 
noctilio/Amylostereum areolatum/Deladenus siricidicola as a model system 
to study microevolutionary processes in invasion and biological control 
in Australia and New Zealand.

The projects have the potential to include a combination of field, 
molecular and chemical ecology and genomics. All three projects will 
deal with fundamental questions on the ecology of this intricate, 
tripartite symbiosis, but will either include or contribute directly to 
applied aspects of the management of this global invasive pest complex.
 
Requirements
Candidates should have a Masters or equivalent degree. Candidates are 
not expected to possess advanced skills in all the fields linked to the 
projects, but experience and an interest in at least two of these fields 
will be an advantage. Thus a background in molecular genetics and 
ecology, chemical ecology, behavioral ecology, entomology, microbiology, 
nematology or related fields would be considered an asset.  Resources 
and support to develop these diverse skills in candidates will be 
provided. While the projects will explore novel questions at the 
frontiers of understanding these particular symbioses, as well as 
symbioses and invasive pests in general, they are built on strong 
baseline datasets, extensive resources and support, and a long history 
of excellence and leading work in the field.

Where
Forestry and Agricultural Biotechnology Institute (FABI), University of 
Pretoria, South Africa. FABI is a global leader in plant biotechnology 
related research, with a strong focus on tree health. The successful 
candidates will join a vibrant, dynamic, highly international (both in 
terms of students, postdocs, staff and networks) and interdisciplinary 
research team, which includes strong research groups in tree health 
research, mycology, entomology, tree and crop genetics, 
plant-microbe-insect interactions, genomics, chemistry and more. The 
laboratories involved have strong links to industry, government and 
international research networks making them a unique and rewarding 
environment to work and launch your research career. For more 
information please visit <a href="http://www.fabinet.up.ac.za/">www.fabinet.up.ac.za</a>.

Partners
Prof Bernard Slippers, Dr Brett Hurley, Prof Michael J Wingfield and others
Dr Jeremy Allison, Natural Resources Canada and University of Toronto, 
Canada
Dr Helen Nahrung, University of the Sunshine Coast, Australia
Dr Angus Carnegie, New South Wales Department of Primary Industries, 
Australia

Remuneration
Full scholarships are offered for all three positions in South Africa 
that will be sufficient to cover accommodation, living costs, 
registration fees, and basic medical aid.  Additional funds will cover 
running costs associated with the project.

When
We are hoping to fill these positions as soon as possible, but no later 
than March 2016. The positions will remain open until a suitable 
candidate has been identified.

 
Applications
Please provide an updated CV, a full academic record and two letters of 
reference, ideally from academic mentors or supervisors that can comment 
on your potential as a future research leader.

Examples of work from the research group
Slippers B, de Groot P, Wingfield MJ. (eds) 2012. The Sirex woodwasp and 
its fungal symbiont: Research and management of a worldwide invasive 
pest. Springer.
Boissin E, Hurley B, Wingfield MJ, Vasaitis R, Stenlid J, Davis C, Groot 
Pd, Ahumada R, Carnegie A, Goldarazena A. 2012. Retracing the routes of 
introduction of invasive species: the case of the Sirex noctilio 
woodwasp. Molecular Ecology 21, 5728-5744.
Wooding AL, Wingfield MJ, Hurley BP, Garnas JR, De Groot P, Slippers B. 
2013. Lack of fidelity revealed in an insect–fungal mutualism after 
invasion. Biology Letters 9, 20130342.
van der Nest MA, Steenkamp ET, Wilken MP, Stenlid J, Wingfield MJ, 
Wingfield BD, Slippers B. 2013. Mutualism and asexual reproduction 
influence recognition genes in a fungal symbiont. Fungal Biology 117, 
439-450.
Yek SH, Slippers B. 2014. Biocontrol opportunities to study 
microevolution in invasive populations. Trends in Ecology & Evolution 
29, 429-430.
Slippers B, Hurley BP, Wingfield MJ. 2015a. Sirex Woodwasp: A Model for 
Evolving Management Paradigms of Invasive Forest Pests. Annual Review of 
Entomology 60, 601-619.
Wingfield MJ, Brockerhoff E, Wingfield BD, Slippers B. 2015. Planted 
forest health: The need for a global strategy. Science 349, 832-836.

Contacts
If you are interested in any of these positions or require additional 
information please contact Smeetha Singh (smeetha.singh@up.ac.za) with 
Sirex PhD position in the subject line.

jeff.garnas@fabi.up.ac.za


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="index764e.html?q=aggregator&amp;from=440">previous page</a></div><div>... <a href="index63c1.html?q=aggregator&amp;from=380">20</a> <a href="index940a.html?q=aggregator&amp;from=400">21</a> <a href="indexd6af.html?q=aggregator&amp;from=420">22</a> <a href="index764e.html?q=aggregator&amp;from=440">23</a> <strong>24</strong> <a href="indexb567.html?q=aggregator&amp;from=480">25</a> <a href="index8c9a.html?q=aggregator&amp;from=500">26</a> <a href="indexbe68.html?q=aggregator&amp;from=520">27</a> <a href="indexf601-2.html?q=aggregator&amp;from=540">28</a> ...</div><div><a href="indexb567.html?q=aggregator&amp;from=480">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=460 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:37:59 GMT -->
</html>
