<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=580 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:40:31 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="indexacd7.html?q=node/208">56(4) August 2007</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 6 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>September 16, 2015</h3>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/197">Limited genomic divergence between intraspecific forms of Culex pipiens under different ecological pressures</a></div>
  <div class="description">Background:
Divergent selection can be a major driver of ecological speciation. In insects of medical importance, understanding the speciation process is both of academic interest and public health importance. In the West Nile virus vector Culex pipiens, intraspecific pipiens and molestus forms vary in ecological and physiological traits. Populations of each form appear to share recent common ancestry but patterns of genetic differentiation across the genome remain unknown. Here, we undertook an AFLP genome scan on samples collected from both sympatric and allopatric populations from Europe and the USA to quantify the extent of genomic differentiation between the two forms.
Results:
The forms were clearly differentiated but each exhibited major population sub-structuring between continents. Divergence between pipiens and molestus forms from USA was higher than in both inter- and intra-continental comparisons with European samples. The proportion of outlier loci between pipiens and molestus (≈3 %) was low but consistent in both continents, and similar to those observed between sibling species of other mosquito species which exhibit contemporary gene flow. Only two of the outlier loci were shared between inter-form comparisons made within Europe and USA.
Conclusion:
This study supports the molestus and pipiens status as distinct evolutionary entities with low genomic divergence. The low number of shared divergent loci between continents suggests a relatively limited number of genomic regions determining key typological traits likely to be driving incipient speciation and/or adaptation of molestus to anthropogenic habitats.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/196">Single origin of the Mascarene stick insects: ancient radiation on sunken islands?</a></div>
  <div class="description">Background:
The study of islands as model systems plays a key role in understanding many evolutionary processes. Knowledge of the historical events leading to present-day island communities is pivotal for exploring fundamental mechanisms of speciation and adaptation. The remote Mascarene archipelago (Mauritius, Réunion, Rodrigues), considered to be the product of an age-progressive trend of north-to-south volcanic activity in the Indian Ocean, hosts a remarkably diverse, endemic and threatened concentration of flora and fauna that has traditionally been considered to be biogeographically related to Madagascar and Africa. To explore the evolutionary diversity of the Mascarene stick insects (Phasmatodea), we constructed a global phylogeny from approximately 2.4 kb of mitochondrial and nuclear sequence data of more than 120 species representing all major phasmatodean lineages.
Results:
Based on the obtained time-calibrated molecular tree we demonstrate that the current phasmid community of the Mascarene archipelago, which consists of members of four presumably unrelated traditional subfamilies, is the result of a single ancient dispersal event from Australasia and started radiating between 16–29 million years ago, significantly predating the age of Mauritius (8–10 million years).
Conclusions:
We propose that the Mascarene stick insects diversified on landmasses now eroded away, presumably to the north of Mauritius. In consequence, ancient islands have probably persisted in the Indian Ocean until the emergence of Mauritius and not only served as stepping stones for colonisation events during sea-level lowstands, but as long-lasting cradles of evolution. These ancient landmasses most likely allowed for adaptive speciation and served as significant sources of diversity that contributed to the biomes of the Mascarene archipelago and the megadiverse Madagascar.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/195">Evolutionary history exposes radical diversification among classes of interaction partners of the MLLE domain of plant poly(A)-binding proteins</a></div>
  <div class="description">Background:
Poly(A)-binding proteins (PABPs) are evolutionarily conserved proteins that have important functions in the regulation of translation and the control of mRNA stability in eukaryotes. Most PABPs encode a C-terminal domain known as the MLLE domain (previously PABC or CTC), which can mediate protein interactions. In earlier work we identified and predicted that four classes of MLLE-interacting proteins were present in Arabidopsis thaliana, which we named CID A, B, C, and D. These proteins encode transcription-activating domains (CID A), the Lsm and LsmAD domains of ataxin-2 (CID B), the CUE and small MutS-related domains (CID C), and two RNA recognition domains (CID D). We recently found that a novel class that lacks the LsmAD domain is present in CID B proteins.
Results:
We extended our analysis to other classes of CIDs present in the viridiplantae. We found that novel variants also evolved in classes CID A and CID C. A specific transcription factor domain is present in a distinct lineage in class A, and a variant that lacks at least two distinct domains was also identified in a divergent lineage in class C. We did not detect any variants in Class D CIDs. This class often consists of four to six highly conserved RNA-binding proteins, which suggests that major redundancy is present in this class.
Conclusions:
CIDs are likely to operate as components of posttranscriptional regulatory assemblies. The evident diversification of CIDs may be neutral or may be important for plant adaptation to the environment and for acquisition of specific traits during evolution. The fact that CIDs subclasses are maintained in early lineages suggest that a presumed interference between duplicates was resolved, and a defined function for each subclass was achieved.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/194">The relationship of recombination rate, genome structure, and patterns of molecular evolution across angiosperms</a></div>
  <div class="description">Background:
Although homologous recombination affects the efficacy of selection in populations, the pattern of recombination rate evolution and its effects on genome evolution across plants are largely unknown. Recombination can reduce genome size by enabling the removal of LTR retrotransposons, alter codon usage by GC biased gene conversion, contribute to complex histories of gene duplication and loss through tandem duplication, and enhance purifying selection on genes. Therefore, variation in recombination rate across species may explain some of the variation in genomic architecture as well as rates of molecular evolution. We used phylogenetic comparative methods to investigate the evolution of global meiotic recombination rate in angiosperms and its effects on genome architecture and selection at the molecular level using genetic maps and genome sequences from thirty angiosperm species.
Results:
Recombination rate is negatively correlated with genome size, which is likely caused by the removal of LTR retrotransposons. After correcting recombination rates for euchromatin content, we also found an association between global recombination rate and average gene family size. This suggests a role for recombination in the preservation of duplicate genes or expansion of gene families. An analysis of the correlation between the ratio of nonsynonymous to synonymous substitution rates (dN/dS) and recombination rate in 3748 genes indicates that higher recombination rates are associated with an increased efficacy of purifying selection, suggesting that global recombination rates affect variation in rates of molecular evolution across distantly related angiosperm species, not just between populations. We also identified shifts in dN/dS for recombination proteins that are associated with shifts in global recombination rate across our sample of angiosperms.
Conclusions:
Although our analyses only reveal correlations, not mechanisms, and do not include potential covariates of recombination rate, like effective population size, they suggest that global recombination rates may play an important role in shaping the macroevolutionary patterns of gene and genome evolution in plants. Interspecific recombination rate variation is tightly correlated with genome size as well as variation in overall LTR retrotransposon abundances. Recombination may shape gene-to-gene variation in dN/dS between species, which might impact the overall gene duplication and loss rates.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/193">Genetic and morphological evidence for introgression between three species of willows</a></div>
  <div class="description">Background:
Hybridization and introgression are said to occur relatively frequently in plants, and in particular among different species of willows. However, data on the actual frequency of natural hybridization and introgression is rare. Here, we report the first fine-scale genetic analysis of a contact zone shared between the three basket willow species, Salix dasyclados, S. schwerinii and S. viminalis in the vicinity of the Lake Baikal in Southern Siberia. Individuals were sampled in fourteen populations and classified as pure species or hybrids based on a set of morphological characters. They were then genotyped at 384 nuclear SNP and four chloroplast SSR loci. The STRUCTURE and NewHybrids softwares were used to estimate the frequency and direction of hybridization using genotypic data at the nuclear SNP loci.
Results:
As many as 19 % of the genotyped individuals were classified as introgressed individuals and these were mainly encountered in the centre of the contact zone. All introgressed individuals were backcrosses to S. viminalis or S. schwerinii and no F1 or F2 hybrids were found. The rest of the genotyped individuals were classified as pure species and formed two clusters, one with S. schwerinii individuals and the other with S. viminalis and S. dasyclados individuals. The two clusters were significantly genetically differentiated, with F ST 
                    = 0.333 (0.282–0.382, p </div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/192">Multiple independent origins of auto-pollination in tropical orchids (Bulbophyllum) in light of the hypothesis of selfing as an evolutionary dead end</a></div>
  <div class="description">Background:
The transition from outcrossing to selfing has long been portrayed as an ‘evolutionary dead end’ because, first, reversals are unlikely and, second, selfing lineages suffer from higher rates of extinction owing to a reduced potential for adaptation and the accumulation of deleterious mutations. We tested these two predictions in a clade of Madagascan Bulbophyllum orchids (30 spp.), including eight species where auto-pollinating morphs (i.e., selfers, without a ‘rostellum’) co-exist with their pollinator-dependent conspecifics (i.e., outcrossers, possessing a rostellum). Specifically, we addressed this issue on the basis of a time-calibrated phylogeny by means of ancestral character reconstructions and within the state-dependent evolution framework of BiSSE (Binary State Speciation and Extinction), which allowed jointly estimating rates of transition, speciation, and extinction between outcrossing and selfing.
Results:
The eight species capable of selfing occurred in scattered positions across the phylogeny, with two likely originating in the Pliocene (ca. 4.4–3.1 Ma), one in the Early Pleistocene (ca. 2.4 Ma), and five since the mid-Pleistocene (ca. ≤ 1.3 Ma). We infer that this scattered phylogenetic distribution of selfing is best described by models including up to eight independent outcrossing-to-selfing transitions and very low rates of speciation (and either moderate or zero rates of extinction) associated with selfing.
Conclusions:
The frequent and irreversible outcrossing-to-selfing transitions in Madagascan Bulbophyllum are clearly congruent with the first prediction of the dead end hypothesis. The inability of our study to conclusively reject or support the likewise predicted higher extinction rate in selfing lineages might be explained by a combination of methodological limitations (low statistical power of our BiSSE approach to reliably estimate extinction in small-sized trees) and evolutionary processes (insufficient time elapsed for selfers to go extinct). We suggest that, in these tropical orchids, a simple genetic basis of selfing (via loss of the ‘rostellum’) is needed to explain the strikingly recurrent transitions to selfing, perhaps reflecting rapid response to parallel and novel selective environments over Late Quaternary (≤ 1.3 Ma) time scales.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/191">Wide prevalence of hybridization in two sympatric grasshopper species may be shaped by their relative abundances</a></div>
  <div class="description">Background:
Hybridization between species is of conservation concern as it might threaten the genetic integrity of species. Anthropogenic factors can alter hybridization dynamics by introducing new potentially hybridizing species or by diminishing barriers to hybridization. This may even affect sympatric species pairs through environmental change, which so far has received little attention. We studied hybridization prevalence and the underlying behavioral mechanisms in two sympatric grasshopper species, a rare specialist (Chorthippus montanus) and a common generalist (Chorthippus parallelus). We conducted a mate choice experiment with constant intraspecific density and varying heterospecific density, i.e. varying relative frequency of both species.
Results:
Mate choice was frequency-dependent in both species with a higher risk of cross-mating with increasing heterospecific frequency, while conspecific mating increased linearly with increasing conspecific density. This illustrates that reproductive barriers could be altered by environmental change, if the relative frequency of species pairs is affected. Moreover, we performed a microsatellite analysis to detect hybridization in twelve syntopic populations (and four allotopic populations). Hybrids were detected in nearly all syntopic populations with hybridization rates reaching up to 8.9 %. Genetic diversity increased for both species when hybrids were included in the data set, but only in the common species a positive correlation between hybridization rate and genetic diversity was detected.
Conclusion:
Our study illustrates that the relative frequency of the two species strongly determines the effectiveness of reproductive barriers and that even the more choosy species (Ch. montanus) may face a higher risk of hybridization if population size decreases and its relative frequency becomes low compared to its sister species. The asymmetric mate preferences of both species may lead to quasi-unidirectional gene flow caused by unidirectional backcrossing. This might explain why genetic diversity increased only in the common species, but not in the rare one. Altogether, the hybridization rate was much higher than expected for a widely sympatric species pair.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>September 15, 2015</h3>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/190">Erratum to: Phylogeography of the sand dune ant Mycetophylax simplex along the Brazilian Atlantic Forest coast: remarkably low mtDNA diversity and shallow population structure</a></div>
  <div class="description">No description available</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/189">Somatic musculature in trematode hermaphroditic generation</a></div>
  <div class="description">Background:
The somatic musculature in trematode hermaphroditic generation (cercariae, metacercariae and adult) is presumed to comprise uniform layers of circular, longitudinal and diagonal muscle fibers of the body wall, and internal dorsoventral muscle fibers. Meanwhile, specific data are few, and there has been no analysis taking the trunk axial differentiation and regionalization into account. Yet presence of the ventral sucker (= acetabulum) morphologically divides the digenean trunk into two regions: preacetabular and postacetabular. The functional differentiation of these two regions is already evident in the nervous system organization, and the goal of our research was to investigate the somatic musculature from the same point of view.
Results:
Somatic musculature of ten trematode species was studied with use of fluorescent-labelled phalloidin and confocal microscopy. The body wall of examined species included three main muscle layers (of circular, longitudinal and diagonal fibers), and most of the species had them distinctly better developed in the preacetabuler region. In majority of the species several (up to seven) additional groups of muscle fibers were found within the body wall. Among them the anterioradial, posterioradial, anteriolateral muscle fibers, and U-shaped muscle sets were most abundant. These groups were located on the ventral surface, and associated with the ventral sucker. The additional internal musculature was quite diverse as well, and included up to twelve separate groups of muscle fibers or bundles in one species. The most dense additional bundles were found in the preacetabular region and were connected with the suckers.
Conclusions:
Previously unknown additional somatic musculature probably provides the diverse movements of the preacetabular region, ventral sucker, and oral sucker (or anterior organ). Several additional muscle groups of the body wall (anterioradial, posterioradial, anteriolateral fibers and U-shaped sets) are proposed to be included into the musculature ground pattern of trematode hermaphroditic generation. This pattern is thought to be determined by the primary trunk morphofunctional differentiation into the preacetabular and the postacetabular regions.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/188">Evolution of the avian &#946;-defensin and cathelicidin genes</a></div>
  <div class="description">Background:
β-defensins and cathelicidins are two families of cationic antimicrobial peptides (AMPs) with a broad range of antimicrobial activities that are key components of the innate immune system. Due to their important roles in host defense against rapidly evolving pathogens, the two gene families provide an ideal system for studying adaptive gene evolution. In this study we performed phylogenetic and selection analyses on β-defensins and cathelicidins from 53 avian species representing 32 orders to examine the evolutionary dynamics of these peptides in birds.Results and conclusionsAvian β-defensins are found in a gene cluster consisting of 13 subfamiles. Nine of these are conserved as one to one orthologs in all birds, while the others (AvBD1, AvBD3, AvBD7 and AvBD14) are more subject to gene duplication or pseudogenisation events in specific avian lineages. Avian cathelicidins are found in a gene cluster consisting of three subfamilies with species-specific duplications and gene loss. Evidence suggested that the propiece and mature peptide domains of avian cathelicidins are possibly co-evolving in such a way that the cationicity of the mature peptide is partially neutralised by the negative charge of the propiece prior to peptide secretion (further evidence obtained by repeating the analyses on primate cathelicidins). Negative selection (overall mean d
                      N
                    </div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/187">Estimating the molecular evolutionary rates of mitochondrial genes referring to Quaternary ice age events with inferred population expansions and dispersals in Japanese Apodemus</a></div>
  <div class="description">Background:
Determining reliable evolutionary rates of molecular markers is essential in illustrating historical episodes with phylogenetic inferences. Although emerging evidence has suggested a high evolutionary rate for intraspecific genetic variation, it is unclear how long such high evolutionary rates persist because a recent calibration point is rarely available. Other than using fossil evidence, it is possible to estimate evolutionary rates by relying on the well-established temporal framework of the Quaternary glacial cycles that would likely have promoted both rapid expansion events and interisland dispersal events.
Results:
We examined mitochondrial cytochrome b (Cytb) and control region (CR) gene sequences in two Japanese wood mouse species, Apodemus argenteus and A. speciosus, of temperate origin and found signs of rapid expansion in the population from Hokkaido, the northern island of Japan. Assuming that global warming after the last glacial period 7–10 thousand years before present (kyr BP) was associated with the expansion, the evolutionary rates (sites per million years, myr) of Cytb and CR were estimated as 11–16 % and 22–32 %, respectively, for A. argenteus, and 12–17 % and 17–24 %, respectively, for A. speciosus. Additionally, the significant signature of rapid expansion detected in the mtDNA sequences of A. speciosus from the remaining southern main islands, Honshu, Shikoku, and Kyushu, provided an estimated Cytb evolutionary rate of 3.1 %/site/myr under the assumption of a postglacial population expansion event long ago, most probably at 130 kyr BP. Bayesian analyses using the higher evolutionary rate of 11–17 %/site/myr for Cytb supported the recent demographic or divergence events associated with the Last Glacial Maximum. However, the slower evolutionary rate of 3.1 %/site/myr would be reasonable for several divergence events that were associated with glacial periods older than 130 kyr BP.
Conclusions:
The faster and slower evolutionary rates of Cytb can account for divergences associated with the last and earlier glacial maxima, respectively, in the phylogenetic inference of murine rodents. The elevated evolutionary rate seemed to decline within 100,000 years.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>June 4, 2015</h3>
<div class="news-item">
 <div class="date">22:00</div>
 <div class="body">
  <div class="title"><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a></div>
  <div class="source">Source: <a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></div>
 </div>
</div>
<h3>May 31, 2015</h3>
<div class="news-item">
 <div class="date">22:00</div>
 <div class="body">
  <div class="title"><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a></div>
  <div class="description">With consequences for disease severity, resistance or clearance of a pathogen infection by an individual can be enhanced by a previous exposure to that pathogen, occurring either within an individual or even in its parents. This form of immune memory, traditionally thought the province of the vertebrate adaptive immune response, can also arise from innate immune pathways of vertebrates and invertebrates, and through distinct pathways in plants and bacteria. Researchers studying this phenomenon rarely interact across taxonomic boundaries, however, and use a preponderance of disparate terms to describe this innate immune mediated memory, including immune memory, immune priming, trained immunity, and systemic acquired resistance. This catalysis meeting will facilitate a synthesis of disparate researchers to better understand commonalities among these different forms of innate immune memory and key consequences for disease. We will use this opportunity to produce a broad interest synthesis manuscript elaborating upon specific avenues by which an improved understanding of innate immune memory will inspire future research, with direct and indirect benefits for human health. First, better understanding of how vertebrate immune memory works in retaining specific memory stands to improve vaccine design and delivery. Second, the specificity of immune memory could be manipulated to leave harmful pests, vectors, and human parasites susceptible to pathogen mediated biocontrol, while improving the health of beneficial organisms such as agricultural plants, animals, and pollinators that ensure human food security. Our approach aims to identify model systems functionally analogous to human innate immune memory that maximize our flexibility to interrogate the genetics, constraints, and functional manipulations of innate immune memory. Finally, this synthesis will elucidate fundamental concepts underlying host-pathogen evolution and the limits of immunological plasticity.</div>
  <div class="source">Source: <a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></div>
 </div>
</div>
<h3>May 21, 2015</h3>
<div class="news-item">
 <div class="date">22:00</div>
 <div class="body">
  <div class="title"><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a></div>
  <div class="description">In the last two decades, models from evolutionary biology have made important contributions to demographic research on human fertility change. Within this evolutionary framework, two distinct traditions have focused on different processes of adaptation and time scales of change: (1) behavioral ecological perspectives have focused on how individual fertility decisions are shaped by local ecological circumstances, while (2) cultural evolutionary approaches have emphasized the role of socially transmitted information and changing social norms in shaping fertility behavior. While each tradition has made independent progress, research that integrates these approaches is necessary to improve our understanding of real fertility behavior, which results from a feedback between individual fertility decisions and social change. This approach requires combined attention to immediate ecological determinants of fertility decisions as well as the long-term processes that shape costs and benefits in a given environment. This workshop will bring together an international team of evolutionary behavioral scientists with complementary methodological and theoretical expertise in anthropology, psychology, and demography to develop (a) a synthetic article which proposes how these approaches can be integrated methodologically and theoretically, (b) an empirical article which applies our new synthetic framework to the study of fertility change in a particular fieldsite, demonstrating how the new methodological approach will work in practice and what we can learn through employing it, and (c) a multi-site grant proposal (UK, US, Bangladesh, Ethiopia, Bolivia, Poland) aimed at integrating and empirically testing these diverse evolutionary models of human fertility change.</div>
  <div class="source">Source: <a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></div>
 </div>
</div>
<h3>May 17, 2015</h3>
<div class="news-item">
 <div class="date">22:00</div>
 <div class="body">
  <div class="title"><a href="http://www.nescent.org/cal/calendar_detail.php?id=1136">Explaining cultural diversity: A new evolutionary synthesis</a></div>
  <div class="description">Human cultural diversity is expressed in myriad ways (from social and marital norms to languages and religious practices), but what factors shape this diversity? Dating back to Darwin, multiple disciplines have debated the degree to which cultural diversity patterns are influenced by different factors, including history, demographics, and ecology. Over recent years an emerging set of studies have showcased how phylogenetic comparative methods from evolutionary biology can help resolve these long-held debates and revolutionize the field of cultural evolution. Now the major barrier to advances lies in the location of necessary data, which are spread across multiple disparate sources in linguistics, biogeography and anthropology. To overcome this challenge we will create D-PLACE (a Database of Phylogenies of Languages for the study of Cultural Evolution), a publicly available and expandable web-portal that will map over 100 cultural features onto language phylogenies and link these to ecological and environmental variables, empowering a whole new line of investigation into the drivers of cultural change and patterns of cultural diversity. We will produce a paper to introduce D-PLACE and outline the many types of questions in comparative anthropology the database can answer. Finally, we will demonstrate the power of this new resource by using D-PLACE to examine two long-standing and fundamental questions from comparative anthropology: (i) What drives the diversity of incest taboos (i.e. how human societies regulate who can mate and marry)? (ii) Can we characterize recurrent âhuman nichesâ, or are societies just arbitrary bundles of cultural features?</div>
  <div class="source">Source: <a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">22:00</div>
 <div class="body">
  <div class="title"><a href="http://www.nescent.org/cal/calendar_detail.php?id=1137">Linking self-fertilization, dispersal and distribution traits of species: Is Bakerâs law an exception to the rule?</a></div>
  <div class="description">Bakerâs Law (hereafter BL) states that self-compatible organisms are more likely to be successful colonizers after long-distance dispersal than self-incompatible organisms. This simple prediction draws a link between mating-system evolution and diverse fields of ecology and evolution such as dispersal biology and colonization, the evolution of range size and range limits, demography and Allee effect, and invasion biology. However, after >60 years of experimental research and theory development, the accumulated data yield varying, and often contradictory, support of BL. Our working group brings together a diverse array of researchers to assess predictions and assumptions of BL and elucidate ecological, evolutionary, and demographic parameters likely to determine the relationships between mating system, dispersal, and colonization success. To accomplish these goals we will: 1) Compile the voluminous literature on BL. 2) Link the BL data with two extensive databases gathered by prior NESCent support (seed germination and seed traits data; mating system data) and a NCEAS pollen limitation database. These expanded databases will include dispersal, range size, and life-history traits, thereby creating a powerful tool for testing various aspects of the relationship between mating-system and colonization success. 3) Employ macroevolutionary tools to map mating-system and dispersal traits onto the angiosperm phylogeny to assess evolutionary patterns and phylogenetically-corrected trait correlations. 4) Formalize BL using current population genetic theory and dispersal theory. Synthetic products of our working group should elucidate the links between dispersal and mating-system in colonization success, and will influence multiple fields of research in evolution for the foreseeable future.</div>
  <div class="source">Source: <a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></div>
 </div>
</div>
<h3>May 10, 2015</h3>
<div class="news-item">
 <div class="date">22:00</div>
 <div class="body">
  <div class="title"><a href="http://www.nescent.org/cal/calendar_detail.php?id=1145">Building non-model species genome curation communities</a></div>
  <div class="description">Building non-model species genome curation communities</div>
  <div class="source">Source: <a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></div>
 </div>
</div>
<h3>May 6, 2015</h3>
<div class="news-item">
 <div class="date">22:00</div>
 <div class="body">
  <div class="title"><a href="http://www.nescent.org/cal/calendar_detail.php?id=1134">Evolutionary ecology of primate life histories</a></div>
  <div class="description">Primates are highly charismatic and often serve as flagship species in conservation efforts. They are also the closest living relatives of humans, and therefore hold the keys to resolving many questions about human evolution and ecology. However, the slow life histories of primates, combined with their complex social systems, their behavioral plasticity, and the challenging field conditions in which primate researchers must work, have severely limited analyses of mortality and fertility in wild, unprovisioned primate populations. This in turn limits comparative analyses that can shed light on the population dynamics and the social and ecological adaptations that have shaped both human and nonhuman primate evolution. We propose a Primate Life Histories Working Group to compare mortality and fertility schedules across taxa, to evaluate a set of hypotheses about the roles that phylogeny, ecology, and behavior play in shaping primate mortality and fertility patterns, and to examine whether life history theory predicts which vital rates are most variable across species. Using unique, individual-based life history data that have been collected from wild populations by nine working group participants over a minimum of 19 years, we will develop age-specific mortality and fertility schedules and create population projection matrices for each species. Our immediate goals are to test current hypotheses about the evolution of life histories in order to advance our understanding of primate evolution. Our longer-term goal is to move toward a collaborative, shared databank allowing analyses of irreplaceable life history data on wild primates.</div>
  <div class="source">Source: <a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></div>
 </div>
</div>
<h3>May 3, 2015</h3>
<div class="news-item">
 <div class="date">22:00</div>
 <div class="body">
  <div class="title"><a href="http://www.nescent.org/cal/calendar_detail.php?id=1146">TriCEM-DTCI-NESCent Catalysis Meeting on Biodiversity, Conservation and Infectious Disease</a></div>
  <div class="source">Source: <a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></div>
 </div>
</div>
<h3>April 28, 2015</h3>
<div class="news-item">
 <div class="date">22:00</div>
 <div class="body">
  <div class="title"><a href="http://www.nescent.org/cal/calendar_detail.php?id=1135">Tempo and mode of plant trait evolution: synthesizing data from extant and extinct taxa</a></div>
  <div class="description">A number of independent efforts have compiled global plant databases on functionally important traits of leaves, stems, seeds, and flowers. These databases are comprised of 1000's to tens of 1000's of species. With a few notable exceptions, they have not been analyzed in an evolutionary or phylogenetic context. However, when synthesized with a modern molecular phylogeny, these data could tell a comprehensive, multivariate story of the evolution of plant functional diversity. In this working group, we will merge multiple databases to explore the rate (tempo, sensu GG Simpson) of evolution of these traits and the best fit evolutionary model(s) (mode) underlying the trait diversification of land plants. We will ask 1. whether important divergences in trait space occurred along similar branches for different traits, 2. whether there were periods of evolution when trait diversification was especially rapid, and 3. whether there were interactions between trait evolution and rates of speciation and extinction. This work will lead to a new community resource of great interestâan internally synced trait matrixâmatched with the current state-of-the-art phylogeny. These data can then be synthesized with fossil evidence to explore whether the tempo and mode of trait evolution in extant and extinct taxa provide similar stories. Furthermore, these data will provide a powerful view into the coordinated (or lack thereof) evolution of ecologically important traits across vascular plantsâone of the most diverse and important lineages in the world today.</div>
  <div class="source">Source: <a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="index25b4.html?q=aggregator&amp;from=560">previous page</a></div><div>... <a href="index8c9a.html?q=aggregator&amp;from=500">26</a> <a href="indexbe68.html?q=aggregator&amp;from=520">27</a> <a href="indexf601-2.html?q=aggregator&amp;from=540">28</a> <a href="index25b4.html?q=aggregator&amp;from=560">29</a> <strong>30</strong> <a href="index79c1.html?q=aggregator&amp;from=600">31</a> <a href="indexdda5.html?q=aggregator&amp;from=620">32</a> <a href="index7a26.html?q=aggregator&amp;from=640">33</a> <a href="index79b6.html?q=aggregator&amp;from=660">34</a> ...</div><div><a href="index79c1.html?q=aggregator&amp;from=600">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=580 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:40:31 GMT -->
</html>
