<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/14&from=180 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 21:44:44 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="index66e2.html?q=node/119">54(6) December 2005</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator/sources/14" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator">news aggregator</a>
<ul>
<li class="expanded"><a href="indexe5f5.html?q=aggregator/sources">sources</a>
<ul>
<li class="leaf"><a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></li>
<li class="leaf"><a href="index4421.html?q=aggregator/sources/8">CiteULike Phylogeny</a></li>
<li class="leaf"><a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></li>
<li class="leaf"><a href="index4dbd.html?q=aggregator/sources/14" class="active">EVOLDIR</a></li>
<li class="leaf"><a href="index8aad.html?q=aggregator/sources/9">Evolutionary Bioinformatics</a></li>
<li class="leaf"><a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></li>
<li class="leaf"><a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></li>
<li class="leaf"><a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></li>
<li class="leaf"><a href="index9fb6.html?q=aggregator/sources/22">Phyloseminar</a></li>
<li class="leaf"><a href="index69a2.html?q=aggregator/sources/10">Systematics Association</a></li>
<li class="leaf"><a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></li>
<li class="leaf"><a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></li>
<li class="leaf"><a href="indexe0bd.html?q=aggregator/sources/15">dechronization</a></li>
<li class="leaf"><a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></li>
<li class="leaf"><a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></li>
<li class="leaf"><a href="index1f4a.html?q=aggregator/sources/7">postgenomic.com</a></li>

</ul>
</li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 2 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a> &raquo; <a href="index5640.html?q=aggregator">news aggregator</a> &raquo; <a href="indexe5f5.html?q=aggregator/sources">sources</a></div><h2>EVOLDIR</h2>
<!-- begin content -->
<div id="aggregator"><div class="feed"><h3>URL</h3>
<div class="xml-icon"><a href="http://evoldir.tumblr.com/rss"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div><a href="http://evoldir.tumblr.com/">http://evoldir.tumblr.com/</a>
<h3>Last update</h3>
1 hour 40 min ago</div><h3>January 6, 2016</h3>
<div class="news-item">
 <div class="date">23:41</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136799037546">Graduate position: Versailles_France.InsectBehavioralEvolution</a></div>
  <div class="description">Master Student research fellowship for 6 months on Moths&rsquo; Pheromone
differentiation at the Institute for Agricultural Research (INRA),
Versailles, France

Our general interests are insects&rsquo; behavioral evolution and
functional genomics. The aim of the project is to find the genetic
basis of the differentiation of Ostrinia nubilalis moth species in two
pheromone strains. This will be done by using in situ hybridization and
the latest methods in transgenesis i.e. CRISPR/Cas9 genome editing
system. Thus the project represents a unique opportunity to gain
experience in an area which is of high demand in the research field.

We are looking for a highly motivated student with BSc degree in
biology and experience in molecular biology. Experience on insects&rsquo;
manipulation and physiology are a plus. This project is a collaboration
between the University of Amsterdam (Institute for Biodiversity and
Evolutionary Dynamics, IBED), Netherlands and the INRA in Versailles,
France. Therefore, communication skills in English (oral and written)
are necessary and knowledge of French is a plus.

There will be a monthly salary of 550 euros, for every complete month
worked, for 6 months maximum. The salary is paid by the Institute of
Ecology and Environnemental Sciences (iEES) - Paris, France, with the
obligation for the student to administratively subscribe to the iEES
(no subscription fees).The group has excellent laboratory facilities
with all state of the art equipment.

Recent/representative papers:

1. Lassance, J.M., Groot, A.T., et al. (2010). Allelic variation in a
fatty-acyl reductase gene causes divergence in moth sex pheromones.
Nature 466, 486-489.

2. Koutroumpa, F. A. et al. (2014)  Shifts in sensory neuron identity
parallel differences in pheromone preference in the European corn
borer. Frontiers in Ecology and Evolution. 2.

3. Koutroumpa, F. A. & Jacquin-Joly, E. (2014)  Sex in the night: Fatty
acid-derived sex pheromones and corresponding membrane pheromone
receptors in insects. Biochimie. 107, 15-21.

For more information contact Fotini KOUTROUMPA or Astrid T. GROOT at
the address below.  To apply please send us a letter of interest,
bachelor scores and CV (including contact information for 2
references). Deadline for application 30 of February 2016.

Contact information:

Fotini Koutroumpa, PhD, Marie Curie SkĹ‚odowska postdoctoral Fellow

Astrid T. Groot, group leader

IBED, University of Amsterdam, Science Park 904, 1098 XH Amsterdam
Phone: +31 20 5255400
Fax: +31 20 5257832
F.Koutroumpa@uva.nl or a.t.groot@uva.nl

<a href="http://bit.ly/1RudHSI">http://bit.ly/1RudHSI</a>

Address in France:

INRA of Versailles - UMR 1392 - iEES-Paris
RD 10 - route de Saint-Cyr, 78026 VERSAILLES Cedex
Tel : 0033 (0)1 30 83 31 63
Fax : 0033 (0)1 30 83 31 19

2016-01-06 14:26 GMT+01:00 Brian Golding
:

  The EvolDir does not post attachments (hence no viruses, better for
  twitter, etc).  Please send plain text in the body of your message
  (as you would like it to appear) and I will see that it is posted.
  Your advert _MUST_ be directed toward evolutionary biologists.
  Brian
  On Wed, Jan 06, 2016 at 08:56:47AM +0000, Fotini Koutroumpa wrote:
  >    Dear evoldir
  >
  >    I send you the announcement for a second year master position
  at our
  >    laboratory. I would be greatful if you could advertise it to
  your
  >    scientific followers.
  >    Please do not hesitate to contact me for further information.
  >    Kind regards
  >    Fotini Koutroumpa
  >
  >    Fotini Koutroumpa, PhD
  >    Marie Curie Sklodowska postdoctoral Fellow
  >    Institute for Biodiversity and Ecosystems Dynamics (IBED)
  >    University of Amsterdam
  >    Science Park 904
  >    1098 XH Amsterdam
  >    Phone: +31 20 5255400
  >    Fax: +31 20 5257832
  >
  >  F.Koutroumpa@uva.nl
  >
  >    <a href="http://bit.ly/1RudHSI">http://bit.ly/1RudHSI</a>
  >   
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:41</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136799036996">Graduate position: ULeicester, Genomics of social organisation</a></div>
  <div class="description"><p>A competition funded PhD studentship at the Department of Genetics, University of Leicester, UK

PhD title: Genomics of social organisation

Supervisors: Dr Rob Hammond & Dr Sandra Beleza (Genetics, University of Leicester)
Collaborator: Dr Y.Wurm, (Queen Mary University of London) 

The evolution of highly integrated social groups with a reproductive division of labour, exemplified by the eusocial Hymenoptera (ants, bees and wasps), is a major evolutionary transition offering unparalleled opportunities to understand how organisation at a higher level evolves and diversifies. Furthermore, changes in social organisation are of great economic importance as they are linked to the invasiveness and pest status of social insects. However, to date, little is known about genetic influences on social organisation in social insects. This is changing. Next generation sequencing (NGS) is now able to reveal the genomic architecture underpinning variation in social organisation, and, in turn, the genomic effects of social living [1, 2]. 

We propose an ambitious PhD investigating the population genetics and genomic architecture underlying a polymorphism in social organisation in the ant, Leptothorax acervorum [3, 4].  

Aims and objectives:

(1) Assess geographical variation in colony social organisation among isolated populations in southern Europe (central and north west Spain, Spanish/French Pyrenees, France & Italy). 
(2) Assemble a draft genome of L.acervorum from a haploid male using PCR-free libraries and MiSeq data. 
(3) Investigate signatures of selection across the genome of L. acervorum using a combination of genome re-sequencing and Restriction site Associated DNA sequencing (RADseq). 
(4) Compare the genetic architecture underlying the polymorphism in social organisation in L.acervorum with the &lsquo;social chromosomes&rsquo; found in other ant species [1,2].

Refs: [1] Wang et al. Nature, 2013. 493:664-668 [2] Purcell et al. Cur. Biol. 24:2728-2732 [3] Gill et al. Proc. Roy. Soc. B. 2009 276: 4423-4431 [4] Gill & Hammond Proc. Roy. Soc. B. 2011. 278: 1524-1531.  

Studentship Funding:  The successful student will be funded by one of two Doctoral Training Partnerships (DTP). Applicants are encouraged to apply to both schemes to maximise their funding success. UK and EU citizens are eligible for funding through these two DTPs (please contact rh225@le.ac.uk if you are unsure about eligibility).

DTP1: Midlands Integrative Biosciences Training Partnership (MIBTP) funded by the BBSRC

Deadline: 10/01/2016

Further details: <a href="http://bit.ly/1kNW3fO">http://bit.ly/1kNW3fO</a>

How to apply: <a href="http://bit.ly/1RudHCp">http://bit.ly/1RudHCp</a>


DTP 2: Central England NERC Training Alliance (CENTA) funded by NERC

Deadline: 25/01/2016

Further details: <a href="http://bit.ly/1kNW43n">http://bit.ly/1kNW43n</a>

How to apply: <a href="http://bit.ly/1RudJu2">http://bit.ly/1RudJu2</a>

Successful applicant: This studentship would suit a highly motivated person with a deep interest in evolutionary biology, genetics and bioinformatics.  The PhD will involve collecting ants in the field, molecular biology and bioinformatics.

For any further information please contact Dr Rob Hammond (rh225@le.ac.uk).


Dr Rob Hammond  
Lecturer in Evolutionary Biology

Department of Genetics 
University of Leicester, University Road, Leicester, LE1 7RH, UK 

t: +44 (0)116 252 5302
e: rh225@le.ac.uk 
w: <a href="http://bit.ly/1kNW3fQ">http://bit.ly/1kNW3fQ</a>


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:09</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136743804141">Conference: NorthCarolinaStateU.EmergingPlantDisease.Mar23-24</a></div>
  <div class="description">*Event Title:*
A Symposium on Emerging Plant Disease and Global Food Security

*Event Date & Time:*
Wednesday, March 23, 2016 at 9:00 AM
to
Thursday, March 24, 2016 at 4:00 PM

*Event Description:*

The Emerging Plant Disease and Global Food Security Chancellors Faculty 
Excellence Program cluster at NC State will host an international 
symposium March 23-24, 2016 in Raleigh, to bring together experts that 
study emerging plant diseases and their arthropod vectors. Managing the 
threats of emerging diseases that affect agricultural crop plants 
requires experts in genetics, biology, epidemiology, evolutionary 
biology, climate change, metadata analysis, geospatial analytics and 
global development policy.

We will synthesize new developments on emerging plant disease biology 
and discuss an expanding array of new technologies to gather, analyze, 
synthesize, and share knowledge about the evolution of emerging 
infectious diseases of plants that affect global food security. Four 
sessions with keynote speakers, a closing panel and posters are planned.



*Location:*
Hunt Library

*URL:*
<a href="http://bit.ly/1Oav3Qm">http://bit.ly/1Oav3Qm</a>

*Contact:*
Jean Ristaino
Jean_Ristaino@ncsu.edu
919 515-3257

You may also view this event on the calendar 
. 




Jean 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136743316026">Other: IIASA_Austria.SummerFellowships.EvolModeling</a></div>
  <div class="description">Summer Fellowships for Young Scientists at the International Institute
for Applied Systems Analysis, Austria

Funding is available for PhD students interested in three months of
collaborative research during June-August 2016 on

     Evolutionary and Ecological Modeling

at the International Institute for Applied Systems Analysis (IIASA)
in Laxenburg, Austria.

Young scientists from all countries are eligible for stipends provided by
IIASA&rsquo;s Evolution and Ecology Program (EEP) that contribute to travel
and accommodation costs. Students from Australia, Austria, Brazil,
China, Egypt, Finland, Germany, India, Indonesia, Japan, Korea, Malaysia,
Mexico, the Netherlands, Norway, Pakistan, Russia, South Africa, Sweden,
Ukraine, the United Kingdom, USA, and Vietnam are furthermore eligible
for fellowships that provide full coverage of travel, accommodation,
and living expenses.

Model-based summer research projects are invited in the following
indicative areas:

     Evolution of cooperation
     Governance of common goods
     Systemic risk and network dynamics
     Eco-evolutionary dynamics
     Evolutionary community ecology
     Food-web evolution
     Vegetation dynamics
     Adaptive speciation
     Disease ecology and evolution
     Evolutionary conservation biology
     Fisheries management
     Fisheries-induced evolution
     Adaptive dynamics theory and models
     Spatial models in ecology and evolution

Applicants are encouraged to prepare a research proposal that corresponds
to their scientific interests and to EEP&rsquo;s research agenda. Accepted
applicants will begin work before the summer, by planning their research
in collaboration with their IIASA supervisors. Previous experiences
with implementing and studying evolutionary or ecological models are
important assets for working in EEP. To improve chances of being selected,
potential applicants are welcome to send informal inquiries regarding
their specific research interests and plans to EEP&rsquo;s program director
Ulf Dieckmann (dieckmann@iiasa.ac.at).

Online applications will be accepted until Monday, January 11, 2016
(24:00 CET).

Since 1977, IIASA&rsquo;s annual Young Scientists Summer Program (YSSP), has
attracted 1750+ students from 80+ countries. The YSSP 2016 will take place
from June 1 to August 31. IIASA is located in the former summer palace
of Austria&rsquo;s royal family, ca. 15 km south of Vienna. IIASA&rsquo;s summer
program offers exceptional opportunities for acquiring experience in
an international and interdisciplinary research environment. Research
training is based on regular personal interaction with advising
scientists, and typically leads to a publication in an international
journal, as well as to a chapter in a candidate&rsquo;s PhD thesis.

Some useful links:

+ Details about the summer program, and online application
<a href="http://bit.ly/1cA7lML">http://bit.ly/1cA7lML</a>

+ Information about IIASA&rsquo;s Evolution and Ecology Program
<a href="http://bit.ly/1cXEbYV">http://bit.ly/1cXEbYV</a>

+ Examples of successful YSSP projects
<a href="http://bit.ly/1cA7nUR">http://bit.ly/1cA7nUR</a>

+ General information about IIASA
<a href="http://bit.ly/1cXEbYZ">http://bit.ly/1cXEbYZ</a>


Ulf Dieckmann
Program Director
Evolution and Ecology Program
International Institute for Applied Systems Analysis
A-2361 Laxenburg
Austria

Email dieckmann@iiasa.ac.at
Phone +43 2236 807 386
Phone secretary +43 2236 807 231
Fax +43 2236 807 466 or +43 2236 71313
Web <a href="http://bit.ly/17j1ut8">http://bit.ly/17j1ut8</a>
Online reprints <a href="http://bit.ly/17j1utc">http://bit.ly/17j1utc</a> 
FroSpects Network <a href="http://bit.ly/17j1sS5">http://bit.ly/17j1sS5</a>
FinE Network <a href="http://bit.ly/17j1ute">http://bit.ly/17j1ute</a>
FishACE Network <a href="http://bit.ly/17j1sS3">http://bit.ly/17j1sS3</a>


DIECKMANN Ulf 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136743315501">Graduate position: NTNU_Taipei.InsectIslandSpeciaiton</a></div>
  <div class="description">The Systematics & Evolutionary Biology Lab at the Department of Life
Sciences, National Taiwan Normal University (NTNU) in Taipei, Taiwan
is seeking a self-motivated and enthuasiastic PhD student to work on
comparative phylogeography of ecological speciation on Asian continental
and oceanic islands.

The Project:

This research project will analyze the speciation mechanism of
endemic freshwater insects between Taiwan and Yaeyama Islands
of Japan (oceanic island), and between mainland Asia and Hainan
Island (continental island). Several insect species are endemic to
these islands and show adaptation to novel environments (ecological
speciation) between islands. (See a damselfly example, Lee & Lin 2012:
<a href="http://bit.ly/1SztHUB">http://bit.ly/1SztHUB</a>)

The project will generate sequence/genomic data from multiple individuals
of selected endemic species of the islands. First, phylogenetic and
population genetic analyses will be used to infer the process of
speciation between species pairs on the islands. Second, the level of
intrinsic reproductive isolation between species pairs will be measured
through crossing experiments. Third, the strength of natural selection
between species pairs will be accessed through field studies of fitness
components. Lastly, through the analysis of genome-wide DNA polymorphism
of target species, the genomic regions and candidate genes underlying
ecological speciation will be uncovered. International students with an
interest in studying tropical island biology are encouraged to apply.

Funding:

Funding includes a competitive monthly stipend of NT$34,000 (US$
1,133) for up to 3 years for both domestic and international students
through the TIGP Biodiversity (Taiwan International Graduate Program -
Biodiversity) (<a href="http://bit.ly/1ZNwTxG">http://bit.ly/1ZNwTxG</a>), a jointed graduate
program between NTNU and Academia Sinica. Students are also eligible
for on-campus housing during the years of their studies.

Eligibility:

The position is open to all applicants that meet TIGP
admission criteria, but preference will be given to students
with a master degree or prior experiences in phylogenetics,
population genetics and insect biology. Criteria/material used
to evaluate the applicants qualifications for the admission:
(<a href="http://bit.ly/1ZNwTxI">http://bit.ly/1ZNwTxI</a>)

How to apply:

Interested candidates should email (1) a CV, (2) a list of prior
experience and publication, (3) three references (contact info & e-mails),
and (4) a one-page statement of research interests in one PDF file to
Chung-Ping Lin (treehopper@ntnu.edu.tw) with the subject line &ldquo;TIGP
application V Island Speciation&rdquo;.

Additionally, candidates should submit a formal
online application for admission in TIGP Biodiversity
(<a href="http://bit.ly/1SztKQ2">http://bit.ly/1SztKQ2</a>). Deadline for applications
through the online portal is March 31st for a September 2016 start.

Web pages:

The Systematics & Evolutionary Biology Lab

(<a href="http://bit.ly/1SztJfa">http://bit.ly/1SztJfa</a>)

The Department of Life Sciences

(<a href="http://bit.ly/1xWml1v">http://bit.ly/1xWml1v</a>)

National Taiwan Normal University

(<a href="http://bit.ly/1SztJfc">http://bit.ly/1SztJfc</a>)

TIGP Biodiversity Program Website

(<a href="http://bit.ly/1ZNwTxG">http://bit.ly/1ZNwTxG</a>)

Biodiversity Research Center, Academia Sinica

(<a href="http://bit.ly/1ZNwV8Z">http://bit.ly/1ZNwV8Z</a>)

Treehopper 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136743314921">Graduate position: ImperialCollege_London.BeeFitness</a></div>
  <div class="description">PhD position advertised to research how bee foraging performance and
colony fitness are affected by stressors in fragmented rural
landscapes&rsquo;

Supervisors: Dr Richard Gill (r.gill@imperial.ac.uk), Dr Samraat Pawar
(Life Sciences)

Location: Department of Life Sciences, Imperial College London, Silwood
Park campus

Deadline: 18^th January 2016

Increased agricultural practices have resulted in the clearance and
fragmentation of many natural habitats posing potential challenges to
wildlife such as insect pollinators. For eusocial bees (e.g. honeybees
& bumblebees), who provide a crucial pollination service, increased
floral habitat patchiness may increase foraging demands making it
significantly more difficult to sustain the colony&rsquo;s energetic
requirements. It is thus important we consider how environmental
stressors set limits on bee foraging performance and how bees are able
to cope with such pressures.

For further details, how to apply and eligibility please go to:
<a href="http://bit.ly/1SztJf6">http://bit.ly/1SztJf6</a>

But in brief, the studentship will investigate what specific
behavioural mechanisms are being affected by stress factors associated
with agricultural practices and the knock-on effects thus has to colony
functioning and fitness. The research project will provide the PhD
student with a number of benefits: i) it will integrate a number of
critical interdisciplinary skills including controlled experiments and
modelling; ii) it crosses a number of scientific fields such as
behavioural ecology, evolutionary and theoretical biology; iii) the
research is of high impact and will result in quality publications as
well as results that can inform end users and policy makers; and iv)
the student will join a productive, diverse and social research group.

Imperial College is a world leading university; Life Science dept. was
placed top of the UK universities for research intensity in the
Research Excellence Framework, and placed 6th best dept. in the world
in the Times Higher Education 2015. The student will be based at
Silwood Park which has an internationally renowned community of
researchers, excellent facilities for the project, and the GCEE
initiative has invested significantly into ensuring world leading
research addressing important global issues.

Selected references:

Raine & Gill 2015 Nature N&V

Gill & Raine 2014 Functional Ecology (doi:10.1111/1365-2435.12292)

Bryden, Gill et al. 2013 Ecology Letters (doi:10.1111/ele.12188)

Gill et al. 2012 Nature (doi:10.1038/nature11585)

Pawar et al. 2012 Nature (doi:10.1038/nature11131)

Gill & Hammond 2011 Proc. Roy. Soc. B

Gill et al. 2009 Proc. Roy. Soc. B

&ldquo;Gill, Richard J&rdquo; 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:55</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136743314651">Graduate position: TexasChristianU.EvolutionaryGenomics</a></div>
  <div class="description">MS position* A graduate research assistantship position (TA funded)
is available in the lab of Dr. Matt Hale in the Biology Department
at Texas Christian University starting in August 2016. The student
will develop a MS project on the genetic basis of complex phenotypic
traits. Some possible research projects include questions relating to
the genetic basis of migration in salmonid fishes, and the presence
and identity of genes showing patterns of sex-bias in expression,
also in the salmonid fishes. The applicant should have some previous
lab experience, including DNA extraction, PCR amplification, and DNA
sequencing. Ideally, the candidate will also have experience with RNA
extraction, cDNA synthesis and bioinformatics, although training in these
areas will be provided. Interested applicants should send a CV, a cover
letter describing their research interests and reasons for applying,
and the names and contact details of two potential referees. Review
of applications will begin on January 20th 2016. Please send inquires
and applications to Matt Hale (m.c.hale@tcu.edu) Interested students are
encouraged to view the Biology Department website (<a href="http://bit.ly/1qrsWeV">http://bit.ly/1qrsWeV</a>)
for more information on the graduate program.



&ldquo;Hale, Matthew&rdquo; 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:21</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136742214161">Job: UMontreal.PlantEvoDevol</a></div>
  <div class="description"><p>2nd announcement

Département de sciences biologiques Faculté des arts et des sciences

Professor in Molecular development of plants

The Département de sciences biologiques is seeking applications for a
full-time tenure-track position at the rank of Assistant Professor in
Molecular development of plants, including evolutionary developmental
biology.

  *     Responsibilities

The appointed candidate will be expected to teach at all three levels of
the curriculum, supervise graduate students, engage in ongoing research
and publication, and contribute to the academic life and reputation of
the University.

  *     Requirements

     *   Ph.D. in molecular biology or in a related field

     *   Postdoctoral experiences in relevant fields

     *   High quality publication record in international journals with
         very good impact factor

     *   Interest in cross-disciplinary research and capacity to develop
         collaborations with members of

the Département de sciences biologiques and other departments of
Université de Montréal

     *   Excellent abilities to teach at the undergraduate and graduate levels

     *   Elaboration of a novel research programme that uses molecular
         genetic approaches to study plant development for a model system

     *   Priority will be given to candidates using advanced imaging
         approaches and possessing a good knowledge of plant anatomy and morphology

     *   Proficiency in French within a reasonable period

Linguistic Policy : Université de Montréal is a Québec
university with an international reputation. French is the
language of instruction. To renew its teaching faculty,
the University is intensively recruitingthe world’s best
specialists. In accordance with the institution’s language policy
[http://bit.ly/1S3Ig3q
stration/adm10-34_politique-linguistique.pdf], Université de Montréal
provides support for newly-recruited faculty to attain proficiency
in French.

  *     Salary
Université de Montréal offers a competitive salary and a complete
range of employee benefits. Salary scale

  *     Starting Date
On or after June 1st, 2016.


 Constitution of application

  *   The application must include the following documents: - a cover letter
- a curriculum vitæ
- copies of three recent publications and research

- a description of the teaching philosophy - a description of the
  research programme

  *   Three letters of recommendation are also to be sent directly
      to the Chair of the Département de sciences biologiques by
      the referees.

 Deadline
Application and letters of recommendation must be sent to the Chair of
the Département de

sciences biologiques by March 1st, 2016 at the following address :

M. Daniel Boisclair, Chair Département de sciences biologiques Faculté
des arts et des sciences Université de Montréal C. P. 6128, succursale
Centre-ville Montréal (QC) H3C 3J7

The complete application may also be sent at the following e-mail :
daniel.boisclair@umontreal.ca. For more information about the Department,
please consult its website at :

<a href="http://bit.ly/1MpkNVn.">http://bit.ly/1MpkNVn.</a>
via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:06</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136741772556">Postdoc: MonashU.EvolutionaryEcol</a></div>
  <div class="description"><p>Research Fellow Opportunity (Biological Sciences) at Monash University

Evolutionary Ecology

The Opportunity

Professor Dustin Marshall (www.meeg.org) is seeking an evolutionary
ecologist to explore the dynamics of biological systems, specifically
how size and shape affect the ways resources move through populations
and communities.

As a postdoctoral researcher, you will explore how biological entities
change their function as they change in size and shape, using empirical
and/or theoretical approaches. The research will be conducted in
collaboration with Marshall as part of the Centre for Geometric Biology
at Monash University.

You will further be expected to maintain consistently high research
output in the form of quality publications, supervision of students,
development and submission of grant proposals to external funding
agencies, contribute more generally to communicating the research
activities of the group, and participation in appropriate career
development activities.

If you believe you can fulfil these requirements, you are strongly
encouraged to apply.

This role is a full-time position; however, flexible working
arrangements may be negotiated.

Your application must address the selection criteria.

Enquiries

Professor Dustin Marshall, +61 3 9902 4449
or dustin.marshall@monash.edu

More information can be found at;

<a href="http://bit.ly/1Oaoz3O">http://bit.ly/1Oaoz3O</a><a href="http://www.meeg.org/">www.meeg.org</a>) is seeking an evolutionary
ecologist to explore the dynamics of biological systems, specifically
how size and shape affect the ways resources move through populations
and communities.

As a postdoctoral researcher, you will explore how biological entities
change their function as they change in size and shape, using empirical
and/or theoretical approaches. The research will be conducted in
collaboration with Marshall as part of the Centre for Geometric Biology
at Monash University.

You will further be expected to maintain consistently high research
output in the form of quality publications, supervision of students,
development and submission of grant proposals to external funding
agencies, contribute more generally to communicating the research
activities of the group, and participation in appropriate career
development activities.

If you believe you can fulfil these requirements, you are strongly
encouraged to apply.

This role is a full-time position; however, flexible working
arrangements may be negotiated.

Your application must address the selection criteria.

Enquiries

Professor Dustin Marshall, +61 3 9902 4449
or dustin.marshall@monash.edu

More information can be found at;

http://bit.ly/1Oaoz3O
cal-sciences-/534851

&ldquo;
via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:50</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136741281046">Postdoc:UFlorida.Ecology.EnvironmentalScience</a></div>
  <div class="description"><p>&ndash;_000_4EA06A06431642DD9BD02DF6673DAF96ufledu_
Content-Type: text/plain; charset=&ldquo;us-ascii&rdquo;
Content-Transfer-Encoding: quoted-printable

Postdoctoral Fellow sought at UF, application deadline 25 January 2016.

The International Center at the University of Florida is pleased to invite applications for the John J. and Katherine C. Ewel Fellowship Program in Ecology and Environmental Science in the Tropics and Subtropics. This program will enable a recent doctorate recipient to study ecology and environmental sciences at the University of Florida (UF). The Fellow will work for two years with a University of Florida faculty member in any department to create new knowledge in any area of tropical and subtropical ecology. For complete details, criteria and application for, please visit <a href="http://bit.ly/1Rr0DgX">http://bit.ly/1Rr0DgX</a>


=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-Marta L. Wayne, Ph. D.
Professor and Chair
P.O. Box 118525
Department of Biology
University of Florida
Gainesville, FL 32611-8525
(courier: 876 Newell Drive)
vox:  352-392-9925
fax:  352-392-3704
<a href="http://bit.ly/1PeG2bi">http://bit.ly/1PeG2bi</a>


&ndash;_000_4EA06A06431642DD9BD02DF6673DAF96ufledu_
Content-Type: text/html; charset=&ldquo;us-ascii&rdquo;
Content-ID: 
Content-Transfer-Encoding: quoted-printable


</p>
Postdoctoral Fellow sought at UF, application deadline 25 January 2016. 
<br class=""/>
The International Center at the University of Florida is pleased to invite applications for the John J. and Katherine C. Ewel Fellowship Program
 in Ecology and Environmental Science in the Tropics and Subtropics. This program will enable a recent doctorate recipient to study ecology and environmental sciences at the University of Florida (UF). The Fellow will work for two years with a University of
 Florida faculty member in any department to create new knowledge in any area of tropical and subtropical ecology. For complete details, criteria and application for, please visit <a href="http://bit.ly/1Rr0DgX" class="">http://bit.ly/1Rr0DgX</a>
<br class=""/>
<br class=""/>









=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Marta L. Wayne, Ph. D.
Professor and Chair
P.O. Box 118525
Department of Biology
University of Florida
Gainesville, FL 32611-8525
(courier: 876 Newell Drive)
vox:  352-392-9925
fax:  352-392-3704
<a href="http://bit.ly/1PeG2bi" class="">http://bit.ly/1PeG2bi</a>



<br class=""/>



&ndash;_000_4EA06A06431642DD9BD02DF6673DAF96ufledu
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:50</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136741281231">Graduate position: UppsalaU.SpongeMetabolomics</a></div>
  <div class="description">Ph.D. position on Metabolomics/Genomics of Deep Sea Sponges,

Dept. of Medicinal Chemistry, Uppsala University, SWEDEN.

I am seeking a highly motivated PhD candidate to work within the frame of
the newly EU funded SponGES consortium. This consortium will investigate
deep-sea north Atlantic sponge grounds. The candidate will develop a
suite of genomic and metabolomic resources to investigate the population
diversity of key sponge species and assess connectivity between sponge
grounds. These resources will also unlock the potential of sponge grounds
for drug discovery, in which the candidate will also play a part.

For more info and to apply, use the link below (in Swedish,
for the English version click on &ldquo;English&rdquo;, upper-right corner).
<a href="http://bit.ly/1Ra00ID">http://bit.ly/1Ra00ID</a>

duration of position: possibly 5 years

application deadline: 18th of January 2016

For specific information about the announced Ph.D. position and details
of the project, please contact me (paco.cardenas@fkog.uu.se).


Dr. Paco Cardenas
Div. of Pharmacognosy, Dept. of  Medicinal Chemistry, Uppsala University
BMC Box 574,
SE 75123 Uppsala, Sweden
paco.cardenas@fkog.uu.se


Paco Cardenas 
via Gmail </div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:03</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136739974441">Job: UNSW_Australia.Professor_Bioinformatics_Systems</a></div>
  <div class="description"><p>The University of New South Wales, Australia invites applications for 
Professor (Level E) in the School of Biotechnology and Biomolecular 
Sciences in the field of  Bioinformatics, Systems Biology and/or Medical 
Genomics.

For more information and instructions for applying please visit

<a href="http://bit.ly/1ONkAJO">http://bit.ly/1ONkAJO</a>

The School of Biotechnology and Biomolecular Sciences (BABS) is part of 
the Science Faculty at UNSW Australia and is located on our Kensington 
campus in Sydney. BABS hosts the Ramaciotti Centre for Genomics, which 
has world class facilities for sequencing (including MiSeq, NextSeq, 
PacBio and access to Illumina X-Ten), microfluidics (including single 
cell genomics) and microarray analysis. The School hosts the New South 
Wales Systems Biology Initiative, supported by high-performance 
computing facilities, including national supercomputers. The School also 
has access to outstanding proteomics and metabolomics facilities at the 
UNSW Bioanalytical Mass Spectrometry Facility. Additional major 
facilities at the University provide capacity for biochemical, 
biophysical and chemical analyses, electron microscopy and animal breeding.

For more information about the School, please visit: 
<a href="http://bit.ly/1OM8JSz">http://bit.ly/1OM8JSz</a>

Application close:  26 February 2016


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:40</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136737747426">Other: Paid Summer Research (REU) at UMich Biological Station</a></div>
  <div class="description"><p>The University of Michigan Biological Station is now accepting
applications for its Research Experience for Undergraduates (REU)
program, &ldquo;Biosphere-Athmosphere-Hydrosphere Interactions in a Changing
Global Environment.&rdquo;

Our REU summer program runs for nine weeks (June 21 - August 20, 2016)
at the UM Biological Station on Douglas Lake in beautiful northern
Michigan. Each REU participant will

&ndash; Work closely with a research mentor on a project of mutual interest.
&ndash; Design, conduct, analyze, and report on their research project,
with guidance from the mentor and the two program co-directors.
&ndash; Participate in workshops and group discussions designed to provide
the background knowledge and technical tools needed to carry out
interdisciplinary scientific research.
&ndash; Contribute to the body of knowledge that can help address many of
Earth&rsquo;s most important environmental challenges.

COMPENSATION: Participants receive a stipend of $5,000 plus free room
and board at the Station.

DEADLINE: The deadline to apply is Monday, February 1, 2016.

Application and details at: <a href="http://goo.gl/GwYGNx.">http://goo.gl/GwYGNx.</a>

The University of Michigan Biological Station is located on Douglas
Lake near Pellston, Michigan at the tip of the lower peninsula. Its
campus is situated amid 10,000 undeveloped acres owned by the Station.
Students have quick access to many unique habitats in the region
including dune, alvar, bog, and old-growth forest. The Station also
has 2 atmospheric gas sensing towers for monitoring forest and
atmospheric gas. More information about the Station is available at
<a href="http://bit.ly/1ON670A.">http://bit.ly/1ON670A.</a>



via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:55</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136736457841">Postdoc: CIBIO_Portugal.BirdPopulationDynamics</a></div>
  <div class="description">REN Biodiversity Chair Post-Doc Fellowship: Power Line Impacts on Bird
Ecology and Population Dynamics

BPD-REN-ECOL

Reference: BPD-REN-ECOL
Scientiic area: Biological sciences

Link to the announcement

One post-doctoral fellowship is available at CIBIO-InBIO, funded by the
REN Biodiversity Chair, in the field of power line impacts on bird
ecology and population dynamics, under the following conditions:

Admission requirements

The post-doc fellowship is suitable for candidates of any nationality
holding a PhD degree in biology or ecology. The candidates should
preferentially have a strong background in bird ecology.

Work overview

The candidate will integrate the REN Biodiversity Chair team, and will
be supervised by the Chair holder, Francisco Moreira. Tasks to
undertake include:
 i. carrying out studies on the ecology and movements of white storks,
    including bird capture and tracking;
ii. assessing the impacts of powerlines on population viability on the
    target species, as well as other possible case studies (e.g. large
    eagles, bustards) by using population viability analysis and other
    demographic simulation approaches;
iii. assessing spatial patterns of use of power line pylons by nesting
    storks;
iv. supporting the implementation of a citizen science project on white
    storks, and gathering information on other citizen science
    initiatives.

The work will include data collection, statistical analysis, paper
writing and potential project submission.

Fellowship
The duration of the fellowship is 12 months, renewable up to a maximum
of 24 months.

Salary
Monthly stipend is â‚¬1495 according to the stipends established by
FCT, I.P. in Portugal (<a href="http://bit.ly/1dRsu4w">http://bit.ly/1dRsu4w</a>).
Payment will be made by bank transfer on a monthly basis.

Applicable legislation

Estatuto do Bolseiro de InvestigaÃ§Ã£o CientÃfica, aprovado pela Lei
n.Âº 40/2004, de 18 de Agosto, alterado e republicado pelo Decreto-Lei
n.Âº 202/2012, de 27 de Agosto, alterado ainda pelo Decreto-Lei n.Âº
233/2012, de 29 de Outubro, pela Lei n.Âº 12/2013, de 29 de Janeiro, e
pelo Decreto-lei n.Âº 89/2013, de 9 de Julho; Regulamento de Bolsas e
InvestigaÃ§Ã£o da FundaÃ§Ã£o para a CiÃªncia e a Tecnologia, IP (FCT),
em vigor (<a href="http://bit.ly/1NFARV6">http://bit.ly/1NFARV6</a>) e
de

acordo com os Estatutos de Bolsa do ICETA aprovados pela FCT.

Work place

Work will be conducted at CIBIO-InBIO branch located at Lisbon,
Portugal.

Application

The call for applications is open between January 18 and February 1,
2016 (24:00 GMT).

Applications should be submitted by email
to bolsas@cibio.up.pt (indicating the above mentioned reference) and
will include:

a)      Curriculum vitae (5 pages maximum, including a publication list
and other relevant information in the context of the project research
goals);

b)      Motivation Letter (including a brief description of research
experience and why you are suitable for the announced position).

Selection criteria

The jury panel will select the best candidate based on his/her merit,
through the analysis of the CV. Preference will be given to candidates
with the following requirements:
 i. field experience on capture and tracking of storks (preferentially)
    or other large birds;
ii. knowledge of methods for carrying out survival analysis and
    simulations of population dynamics;
iii. experience in Geographic Information Systems (GIS) and R;
iv. experience in citizen science projects;
 v. papers in ISI journals with peer-review.

The top 3 candidates will be invited for an interview (weight of 50%),
in person or by video-conference, to establish the final ranking.

Jury Panel

Francisco Moreira (Chair), InÃªs Catry (vowel), Pedro Beja (vowel), and
JoÃ£o Paulo Silva (substitute vowel).

Results announcement

Candidates will be directly informed by email about the result of their
application. The ranking of the candidates will be published at a
visible and public area of ICETA facilities.

CIBIO-InBIO Divulgação 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:11</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136735039536">Postdoc: ColdSpringHarborLab.ComputationalGenomics</a></div>
  <div class="description">POSITION DESCRIPTION:
Postdoctoral Positions in Computational Genomics. Postdoctoral
positions are available in Dr. Adam Siepel&rsquo;s research group in its new
location at the Simons Center for Quantitative Biology at Cold Spring
Harbor Laboratory. The Siepel Group specializes in the development of
probabilistic models, algorithms for inference, prediction methods, and
application of these methods in large-scale genomic data analysis. Of
particular interest is research relevant to existing, NIH supported
projects in (1) HUMAN POPULATION GENOMICS, including demography inference
using Bayesian coalescent-based methods, inference of natural selection
on regulatory and other noncoding sequences, and prediction of fitness
consequences for noncoding mutations; and (2) TRANSCRIPTIONAL REGULATION
in mammals and Drosophila, including the estimation of rates and
patterns of transcriptional elongation from GRO-seq data, prediction of
transcription factor binding sites from DNase-seq data, and regulatory
network inference based on joint patterns of transcription and binding
in inducible systems. The research on transcriptional regulation will
continue to be carried out in close collaboration with Dr. John Lis at
Cornell University.

RELEVANT RECENT PAPERS:
Kuhlwilm M, Gronau I, Hubisz MJ, de Filippo C, Prado J, et al. Ancient
gene flow from modern humans into Siberian Neandertals. Nature (in
press), 2015.

Danko CG, Hyland SL, Core LJ, Martins AL, Waters CT, et al. Identification
of active transcriptional regulatory elements from GRO-seq
data. Nat. Methods. 2015. 12(5):433-438. PMCID: PMC4507281.

Rasmussen MD, Hubisz MJ, Gronau I, Siepel A. Genome-wide inference of
ancestral recombination graphs. PLOS Genet. 2014. 10(5): e1004342. PMCID:
PMC4022496.

Freedman AH, Gronau I, Schweizer RM, Ortega-Del Vecchyo D, Han E, et
al. (30 co-authors). Genome sequencing highlights the dynamic early
history of dogs. PLOS Genet. 2014. 10(1):e1004016. PMCID: PMC3894170.

Core LJ, Martins AL, Danko CG, Waters CT, Siepel A, et al. Analysis of
nascent RNA identifies a unified architecture of initiation regions at
mammalian promoters and enhancers. Nat. Genet. 2014. 46(12):1311-1320
PMCID: PMC4254663.

Capra JA, Hubisz MJ, Kostka D, Pollard KS, Siepel A. A model-based
analysis of GC-biased gene conversion in the human and chimpanzee
genomes. PLOS Genet. 2013. 9(8):e1003684. PMCID: PMC3744432.

Arbiza L, Gronau I, Aksoy BA, Hubisz MJ, Gulko B, Keinan A, Siepel
A. Genome-wide inference of natural selection on human transcription
factor binding sites. Nat. Genet. 2013. 45(7):723-729. PMCID: PMC3932982.

Gronau I, Arbiza L, Mohammed J, Siepel A. Inference of natural
selection from interspersed genomic elements based on polymorphism and
divergence. Mol. Biol. Evol. 2013. 30(5):1159-1171. PMCID: PMC3697874.

Gronau I, Hubisz MJ, Gulko B, Danko CG, Siepel A. Bayesian
inference of ancient human demography from individual genome
sequences. Nat. Genet. 2011. 43(10):1031-1034. PMCID: PMC3245873.

The ideal candidate will be enthusiastic, highly motivated and
independent; will have a strong background in comparative genomics,
population genomics, or transcriptional regulation, a serious
interest in molecular biology and genetics combined with good skills in
mathematics and computer science, and a strong record of accomplishment in
research. Postdoctoral associates in the group are expected to participate
in both methods development and data analysis, to be comfortable in
a fast-moving, interactive, and collaborative research environment,
and to actively publish and present results.

REQUIRED QUALIFICATIONS:
Ph.D. in bioinformatics, computer science, statistics, genetics, molecular
biology, applied mathematics, or a related field.

Research experience (with first-author publications) in computational
genomics or a closely related field.

Fluency in probabilistic modeling and computational statistics.
Proficiency in programming, ideally in C or C++, as well as in scripting
languages such as python, Perl, mat lab, or R. Should be comfortable in a
Linux environment, with large data sets, computer clusters, and databases.

JOB LOCATION: CSHL, One Bungtown Road, Cold Spring Harbor, NY 11724

TO APPLY: Position Number
01229-R. <a href="http://bit.ly/1JtlFu9.">http://bit.ly/1JtlFu9.</a>


&ldquo;Gill, Irene&rdquo; 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:55</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136734486856">Conference: Montpellier.MathCompEvolutionaryBiology.Jun12-16</a></div>
  <div class="description">*** REGISTER AGAIN IF YOU DIDNT RECEIVE ANY CONFIRMATION EMAIL
*** OUR WEBSITE HAD MALFUNCTIONS, SORRY &hellip;

*** PLEASE DIFFUSE THIS ANNOUNCEMENT, THANKS!

MCEB - Mathematical and Computational Evolutionary Biology
12-16 June 2016 - Hameau de l'Etoile, Montpellier, South of France.

Webpage: <a href="http://bit.ly/1Rr4Lxy">http://bit.ly/1Rr4Lxy</a>

Pre-registration deadline: February 25th
Notification to applicants: March 15th
Final list of attendees: April 15th

Scope: Mathematical and computational tools and concepts form an
essential basis for modern evolutionary studies. The goal of the MCEB
conference (at its 8th edition) is to bring together scientists with
diverse backgrounds to present recent advances and discuss open
problems in the field of mathematical and computational evolutionary
biology. The theme of this year’s edition will be &ldquo;forecasting
evolution&rdquo;: while most analyses are designed to reconstruct the past
in order to understand evolution, our modern society is in dire need
of anticipating the future using sound scientific approaches. This is
particularly true in the context of global warming and more generally
global changes. New approaches that model future evolution are
emerging, for example to predict epidemics, evolution of cancer,
shifts in species/population distributions or species loss.
Evolutionary biology is thus widening its focus, trying to predict
future biological states and better define the range of applicability
of these predictions. Making progress in this new research area will
involve cross-disciplinary approaches, accurate statistical methods,
and powerful computational techniques. Beyond this year’s theme,
general concepts, models, methods and algorithms will be presented
and discussed, just as in the previous editions of MCEB. Keynote
speakers will introduce a field of research and discuss their own
work in this field. Afternoon will be for short presentations and
posters, with plenty of time for discussions and outdoors.

Where and when: Hameau de l'Etoile, near Montpellier, in the South of
France, 12-16 June 2016.

Cost: Conference fees including accommodation for four nights, meals,
coffee breaks, buses, etc., will be between 300€ and 550€, all
inclusive, and will vary depending on the room. PhD students and
postdocs will benefit of the cheapest rooms.

Keynote speakers:
Jeffrey Jensen - <a href="http://bit.ly/1kpTAqI">http://bit.ly/1kpTAqI</a>
Ecole Polytechnique Fédérale de Lausanne, CH

Laura S. Kubatko - <a href="http://bit.ly/1Rr4J91">http://bit.ly/1Rr4J91</a>
Departments of Statistics and Evolution, Ecology, and Organismal
Biology - The Ohio State University, USA

Amaury Lambert - <a href="http://bit.ly/1Z4N6wt">http://bit.ly/1Z4N6wt</a>
Stochastics & Biology Group - Laboratoire de Probabilités & Modèles
Aléatoires - UPMC - Université Paris 06, FR

Richard Neher - <a href="http://bit.ly/1Rr4J92">http://bit.ly/1Rr4J92</a>
Max Planck Institute for Developmental Biology - DE

Bruce Rannala - <a href="http://bit.ly/1Rr4J93">http://bit.ly/1Rr4J93</a>
UC Davis - University of California, USA

Isabel Sanmartín Bastida -
<a href="http://bit.ly/1Z4N5IR">http://bit.ly/1Z4N5IR</a> 

Real Jardín Botánico (RJB-CSIC), Madrid, SP

Yun S. Song - <a href="http://bit.ly/1Rr4J95">http://bit.ly/1Rr4J95</a>
Departments of Mathematics and Biology - Universty of Pennsylvania, USA

For more information and register: <a href="http://bit.ly/1Rr4Lxy">http://bit.ly/1Rr4Lxy</a>

Please forward this announcement

Olivier Gascuel 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:25</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136733359966">Other: Replies to Tree with bootstraping from distance matrix</a></div>
  <div class="description"><p>Hi,

Following my question about bootstraping a distance matrix to produce a 
phylogenetic tree with bootstrap confidence values, I got many useful 
replies. My original question is below and the replies summary is 
further down this message.

On 15-12-18 10:57 AM, Eric Normandeau wrote:
> Hi,
>
> I&rsquo;m creating a tree from a distance matrix using phylip neighbor and 
> phylib drawgram. I would like to have bootstrap values for the 
> branches but all I have is this one distance matrix. Is this possible?
>
> The distance matrix is calculated from a Genotyping by Sequencing 
> (GBS) dataset with a few thousand SNPs. For each individual pair, the 
> distance is basically the proportion of genotypes that differ for loci 
> where both individuals are genotyped.
>
> I could easily create different distance matrices by randomly sampling 
> only some of the SNPs to calculate the distances, but that would not 
> make much sense. For example, if I use a low proportion of the SNPs 
> (ie: 5%), the bootstrap values will end up being lower because the 
> matrices will be more different from one another than if I use a high 
> proportion of the SNPs (ie: 50+%). It feels like I can control the 
> bootstrap values artificially, so that is not a good avenue.
>
> Is there a way, given a single distance matrix, to create a tree with 
> bootstrap values?
>
> I&rsquo;ll be happy to summarize the answers and post them back on EvolDir.
>
> Eric
>
Here is the summary of the very useful replies:

1) I cannot bootstrap the distance matrix
2) I can bootstrap the markers (random choice with replacement) and 
produce 1000 distances matrices
3) I was describing a jackknife procedure, not a bootstrap procedure

David Remington also suggested that I could use a nucleotide 
substitution model to calculate a more appropriate distance metric. 
However, he mentions that this approach wouldn&rsquo;t work if the SNP dataset 
has been pre-filtered, for example by retaining only the variable 
markers, as it would make the nucleotide substitution model incorrect. 
In our case, we have to work with pre-selected SNPs, otherwise we would 
be using hundreds of thousands of SNPs with a lot of missing data and 
most of which would be variable in only one individual, thus more 
probably sequencing errors.

David also asked if this was a within-species study and it is not. It 
uses specimens from a variety of related species with low numbers of 
samples per species. That is why we went for a phylogenetic approach.

Jack Cameron suggested using a Canberra distance, which is a robust 
Manhattan metric and then to use Pvclust to bootstrap once I have the 
distance matrices. The Canberra distance computes a distance between two 
vectors P an Q of numbers by computing the sum of, for each index i, 
(|Pi - Qi|) / (|Pi| + |Qi|). Since my pairwise comparisons between two 
samples use all the loci for which both samples were genotyped, and that 
the length of the the P and Q vectors vary among the pairwise 
comparisons, I would then divide by the length to normalize them. This 
leaves the problem of encoding the possible genotypes (AA, AB, BB) as 
numbers, probably with AA=1, AB=2 and BB=3. In this context, however, I 
do not see how dividing by (Pi + Qi) makes any sense. It basically gives 
more weight to a case of AA-AB than to a case of AB-BB since the later 
will be divided by 5, compared to 3 for the former. My implementation of 
the distance is equivalent to using only the numerator (|Pi - Qi|) part 
of the Canberra distance and then dividing the sum by the length of the 
vectors.

Any further thoughts on using a different distance metric would be 
highly welcome!

I will then implement the following:

1) Use my VCF file with my SNP markers and bootstrap it 1000 times
2) Use Phylip to compute the concensus tree and bootstrap confidence values
3) Keep an eye out for how to use a different distance metric


Many thanks to all who replied to my question (in chronological order of 
replies):

- Bernhard Haubold
- Peter Smouse
- Francisco Bilbao Moore
- David Remington
- Murray Cox
- Jack Cameron
- Frank E. Anderson
- Mary Kuhner
- Miguel Navascués
- Louis Ranjard


Eric Normandeau - Bioinformaticien
Laboratoire de Louis Bernatchez
Ressources Aquatiques Québec (RAQ)

Institut de Biologie Intégrative et des Systèmes (IBIS)
Pavillon Charles-Eugène-Marchand
1030, Avenue de la Médecine
Local 1143
Université Laval
Québec (Québec) G1V 0A6
Canada

Tél: 418 656-2131 poste 8455
Courriel: eric.normandeau@bio.ulaval.ca







via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:25</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136733359626">Postdoc: UFlorida.TropicsEvolution</a></div>
  <div class="description"><p>The International Center at the University of Florida is pleased to invite
applications for the John J. and Katherine C. Ewel Fellowship Program in
Ecology and Environmental Science in the Tropics and Subtropics. This
program will enable a recent doctorate recipient to study ecology and
environmental sciences at the University of Florida (UF). The Fellow
will work for two years with a University of Florida faculty member
in any department to create new knowledge in any area of tropical and
subtropical ecology. For complete details, criteria and application for,
please visit <a href="http://bit.ly/1Rr0DgX">http://bit.ly/1Rr0DgX</a>


Marta L. Wayne, Ph. D.
Professor and Chair
P.O. Box 118525
Department of Biology
University of Florida
Gainesville, FL 32611-8525
(courier: 876 Newell Drive)
vox:  352-392-9925
fax:  352-392-3704
<a href="http://bit.ly/1PeG2bi">http://bit.ly/1PeG2bi</a>



mlwayne@ufl.edu


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:10</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136732754791">Graduate position: NTNU_Trondheim.TelomereLifeHistoryDynamics</a></div>
  <div class="description">A PhD position is available at the Department of Biology at the
Norwegian University of Science and Technology (NTNU) in Trondheim
on the functional relationships between telomere dynamics, physiology
and life history traits in a small passerine bird, the House Sparrow
(Passer domesticus).  We are seeking a highly qualified candidate
with a background in either evolutionary, behavioral or population
ecology or eco-physiology. It is an advantage if the candidate has
laboratory experience in genetic analyses.  The aim of the PhD project
will be to investigate central questions related to the functional
associations between telomere dynamics, physiological parameters and
individual behavior (&ldquo;personality&rdquo;), and how these are associated
with trade-offs between life history traits such as growth rate,
body size, reproduction and lifespan. Read more about the project here:
<a href="http://bit.ly/1VHszxI">http://bit.ly/1VHszxI</a>
Here is the link to the formal announcement:
<a href="http://bit.ly/1Z7Cjqs">http://bit.ly/1Z7Cjqs</a>
Application deadline is 1st February. For further information contact:
thor.h.ringsby@ntnu.no.




Thor Harald Ringsby 
via Gmail</div>
 </div>
</div>
<h3>January 5, 2016</h3>
<div class="news-item">
 <div class="date">03:50</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136671362896">Other: Chromonomer.Software</a></div>
  <div class="description"><p>Hi All,

I&rsquo;m pleased to release the Chromonomer software package today:

<a href="http://bit.ly/1PHJa29">http://bit.ly/1PHJa29</a>

Chromonomer is software to integrate a genome assembly with a genetic 
map to create a &lsquo;chromonome&rsquo; (1). The software will order and orient 
scaffolds from a genome assembly according to the markers from a genetic 
map. Chromonomer was designed around RADseq genetic maps but should work 
with any genetic map where the markers can be aligned to the genome 
sequence.

Chromonomer will break scaffolds to fill gaps in the assembly with 
smaller scaffolds or to correct mis-joins in the assembly. It will also 
identify markers that are out of order with respect to the assembly. In 
this way the software uses the assembly order on the small scale and the 
map order on the large scale to integrate an assembly and a map together.

Chromonomer generates extensive documentation of where any 
incompatibilities occur between the assembly and the map, including 
(optional) web-based visualizations.

The software is released under the GPL free software license.

Please try it and let us know your experience with it.

Best wishes,

julian

(1) <a href="http://bit.ly/1IO1Hdz">http://bit.ly/1IO1Hdz</a>

via Gmail</p></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="index8713.html?q=aggregator/sources/14&amp;from=0">first page</a></div><div><a href="index5104.html?q=aggregator/sources/14&amp;from=160">previous page</a></div><div>... <a href="index2946.html?q=aggregator/sources/14&amp;from=100">6</a> <a href="index7de1.html?q=aggregator/sources/14&amp;from=120">7</a> <a href="index2be7.html?q=aggregator/sources/14&amp;from=140">8</a> <a href="index5104.html?q=aggregator/sources/14&amp;from=160">9</a> <strong>10</strong> <a href="index2379.html?q=aggregator/sources/14&amp;from=200">11</a> <a href="index9605.html?q=aggregator/sources/14&amp;from=220">12</a> <a href="index024a.html?q=aggregator/sources/14&amp;from=240">13</a> <a href="indexe9f7.html?q=aggregator/sources/14&amp;from=260">14</a> </div><div><a href="index2379.html?q=aggregator/sources/14&amp;from=200">next page</a></div><div><a href="index9a9b.html?q=aggregator/sources/14&amp;from=280">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news." class="active">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/14&from=180 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 21:44:44 GMT -->
</html>
