<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=440 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:37:54 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="indexacd7.html?q=node/208">56(4) August 2007</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 7 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>December 14, 2015</h3>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/66?rss=1">Enlarged Multilocus Data set Provides Surprisingly Younger Time of Origin for the Plethodontidae, the Largest Family of Salamanders</a></div>
  <div class="description"><p>Deep phylogenetic relationships of the largest salamander family Plethodontidae have been difficult to resolve, probably reflecting a rapid diversification early in their evolutionary history. Here, data from 50 independent nuclear markers (total 48,582 bp) are used to reconstruct the phylogeny and divergence times for plethodontid salamanders, using both concatenation and coalescence-based species tree analyses. Our results robustly resolve the position of the enigmatic eastern North American four-toed salamander (<I>Hemidactylium</I>) as the sister taxon of <I>Batrachoseps</I> + Tribe Bolitoglossini, thus settling a long-standing question. Furthermore, we statistically reject sister taxon status of <I>Karsenia</I> and <I>Hydromante</I>s, the only plethodontids to occur outside the Americas, leading us to new biogeographic hypotheses. Contrary to previous long-standing arguments that plethodontid salamanders are an old lineage originating in the Cretaceous (more than 90 Ma), our analyses lead to the hypothesis that these salamanders are much younger, arising close to the K-T boundary (~66 Ma). These time estimates are highly stable using alternative calibration schemes and dating methods. Our data simulation highlights the potential risk of making strong arguments about phylogenetic timing based on inferences from a handful of nuclear genes, a common practice. Based on the newly obtained timetree and ancestral area reconstruction results, we argue that (i) the classic "Out of Appalachia" hypothesis of plethodontid origins is problematic; (ii) the common ancestor of extant plethodontids may have originated in northwestern North America in the early Paleocene; (iii) origins of Eurasian plethodontids likely result from two separate dispersal events from western North America via Beringia in the late Eocene (~42 Ma) and the early Miocene (~23 Ma), respectively.</p></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/82?rss=1">Fast Dating Using Least-Squares Criteria and Algorithms</a></div>
  <div class="description"><p>Phylogenies provide a useful way to understand the evolutionary history of genetic samples, and data sets with more than a thousand taxa are becoming increasingly common, notably with viruses (e.g., human immunodeficiency virus (HIV)). Dating ancestral events is one of the first, essential goals with such data. However, current sophisticated probabilistic approaches struggle to handle data sets of this size. Here, we present very fast dating algorithms, based on a Gaussian model closely related to the Langley&ndash;Fitch molecular-clock model. We show that this model is robust to uncorrelated violations of the molecular clock. Our algorithms apply to serial data, where the tips of the tree have been sampled through times. They estimate the substitution rate and the dates of all ancestral nodes. When the input tree is unrooted, they can provide an estimate for the root position, thus representing a new, practical alternative to the standard rooting methods (e.g., midpoint). Our algorithms exploit the tree (recursive) structure of the problem at hand, and the close relationships between least-squares and linear algebra. We distinguish between an unconstrained setting and the case where the temporal precedence constraint (i.e., an ancestral node must be older that its daughter nodes) is accounted for. With rooted trees, the former is solved using linear algebra in linear computing time (i.e., proportional to the number of taxa), while the resolution of the latter, constrained setting, is based on an active-set method that runs in nearly linear time. With unrooted trees the computing time becomes (nearly) quadratic (i.e., proportional to the square of the number of taxa). In all cases, very large input trees (>10,000 taxa) can easily be processed and transformed into time-scaled trees. We compare these algorithms to standard methods (root-to-tip, r8s version of Langley&ndash;Fitch method, and BEAST). Using simulated data, we show that their estimation accuracy is similar to that of the most sophisticated methods, while their computing time is much faster. We apply these algorithms on a large data set comprising 1194 strains of Influenza virus from the pdm09 H1N1 Human pandemic. Again the results show that these algorithms provide a very fast alternative with results similar to those of other computer programs. These algorithms are implemented in the LSD software (least-squares dating), which can be downloaded from <A HREF="http://www.atgc-montpellier.fr/LSD/">http://www.atgc-montpellier.fr/LSD/, along with all our data sets and detailed results. An Online Appendix, providing additional algorithm descriptions, tables, and figures can be found in the Supplementary Material available on Dryad at http://dx.doi.org/10.5061/dryad.968t3</A>.</p></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/98?rss=1">Testing for Deperet's Rule (Body Size Increase) in Mammals using Combined Extinct and Extant Data</a></div>
  <div class="description"><p>Whether or not evolutionary lineages in general show a tendency to increase in body size has often been discussed. This tendency has been dubbed "Cope's rule" but because Cope never hypothesized it, we suggest renaming it after Dep&eacute;ret, who formulated it clearly in 1907. Dep&eacute;ret's rule has traditionally been studied using fossil data, but more recently a number of studies have used present-day species. While several paleontological studies of Cenozoic placental mammals have found support for increasing body size, most studies of extant placentals have failed to detect such a trend. Here, we present a method to combine information from present-day species with fossil data in a Bayesian phylogenetic framework. We apply the method to body mass estimates of a large number of extant and extinct mammal species, and find strong support for Dep&eacute;ret's rule. The tendency for size increase appears to be driven not by evolution toward larger size in established species, but by processes related to the emergence of new species. Our analysis shows that complementary data from extant and extinct species can greatly improve inference of macroevolutionary processes.</p></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/109?rss=1">Detecting Macroevolutionary Self-Destruction from Phylogenies</a></div>
  <div class="description"><p>Phylogenetic analyses have lent support to the concept of lineage selection: that biological lineages can have heritable traits that influence their capacity to persist and diversify, and thereby affect their representation in biodiversity. While many discussions have focused on "positive" lineage selection, where stably heritable properties of lineages enhance their diversification rate, there are also intriguing examples that seem to represent "negative" lineage selection, where traits reduce the likelihood that a lineage will persist or speciate. In this article, we test whether a particular pattern of negative lineage selection is detectable from the distributions of the trait on a phylogeny. "Self-destructive" traits are those that arise often but then disappear again because they confer either a raised extinction rate or they are prone to a high rate of trait loss. For such a trait, the reconstructed origins will tend to be dispersed across the tips of the phylogeny, rather than defining large clades of related lineages that all share the trait. We examine the utility of four possible measures of "tippiness" as potential indicators of macroevolutionary self-destruction, applying them to phylogenies on which trait evolution has been simulated under different combinations of parameters for speciation, extinction, trait gain, and trait loss. We use an efficient simulation approach that starts with the required number of tips with and without the trait and uses a model to work "backwards" to construct different possible trees that result in that set of tips. We then apply these methods to a number of case studies: salt tolerance in grasses, color polymorphism in birds of prey, and selfing in nightshades. We find that the relative age of species, measured from tip length, can indicate a reduced speciation rate but does not identify traits that increase the extinction rate or the trait loss rate. We show that it is possible to detect cases of macroevolutionary self-destruction by considering the number of tips with the trait that arise from each inferred origin, and the degree to which the trait is scattered across the phylogeny. These metrics, and the methods we present, may be useful for testing macroevolutionary hypotheses from phylogenetic patterns.</p></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/128?rss=1">How Should Genes and Taxa be Sampled for Phylogenomic Analyses with Missing Data? An Empirical Study in Iguanian Lizards</a></div>
  <div class="description"><p>Targeted sequence capture is becoming a widespread tool for generating large phylogenomic data sets to address difficult phylogenetic problems. However, this methodology often generates data sets in which increasing the number of taxa and loci increases amounts of missing data. Thus, a fundamental (but still unresolved) question is whether sampling should be designed to maximize sampling of taxa or genes, or to minimize the inclusion of missing data cells. Here, we explore this question for an ancient, rapid radiation of lizards, the pleurodont iguanians. Pleurodonts include many well-known clades (e.g., anoles, basilisks, iguanas, and spiny lizards) but relationships among families have proven difficult to resolve strongly and consistently using traditional sequencing approaches. We generated up to 4921 ultraconserved elements with sampling strategies including 16, 29, and 44 taxa, from 1179 to approximately 2.4 million characters per matrix and approximately 30% to 60% total missing data. We then compared mean branch support for interfamilial relationships under these 15 different sampling strategies for both concatenated (maximum likelihood) and species tree (NJst) approaches (after showing that mean branch support appears to be related to accuracy). We found that both approaches had the highest support when including loci with up to 50% missing taxa (matrices with ~40&ndash;55% missing data overall). Thus, our results show that simply excluding all missing data may be highly problematic as the primary guiding principle for the inclusion or exclusion of taxa and genes. The optimal strategy was somewhat different for each approach, a pattern that has not been shown previously. For concatenated analyses, branch support was maximized when including many taxa (44) but fewer characters (1.1 million). For species-tree analyses, branch support was maximized with minimal taxon sampling (16) but many loci (4789 of 4921). We also show that the choice of these sampling strategies can be critically important for phylogenomic analyses, since some strategies lead to demonstrably incorrect inferences (using the same method) that have strong statistical support. Our preferred estimate provides strong support for most interfamilial relationships in this important but phylogenetically challenging group.</p></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/146?rss=1">Testing Convergence Versus History: Convergence Dominates Phenotypic Evolution for over 150 Million Years in Frogs</a></div>
  <div class="description"><p>Striking evolutionary convergence can lead to similar sets of species in different locations, such as in cichlid fishes and <I>Anolis</I> lizards, and suggests that evolution can be repeatable and predictable across clades. Yet, most examples of convergence involve relatively small temporal and/or spatial scales. Some authors have speculated that at larger scales (e.g., across continents), differing evolutionary histories will prevent convergence. However, few studies have compared the contrasting roles of convergence and history, and none have done so at large scales. Here we develop a two-part approach to test the scale over which convergence can occur, comparing the relative importance of convergence and history in macroevolution using phylogenetic models of adaptive evolution. We apply this approach to data from morphology, ecology, and phylogeny from 167 species of anuran amphibians (frogs) from 10 local sites across the world, spanning ~160 myr of evolution. Mapping ecology on the phylogeny revealed that similar microhabitat specialists (e.g., aquatic, arboreal) have evolved repeatedly across clades and regions, producing many evolutionary replicates for testing for morphological convergence. By comparing morphological optima for clades and microhabitat types (our first test), we find that convergence associated with microhabitat use dominates frog morphological evolution, producing recurrent ecomorphs that together encompass all sampled species in each community in each region. However, our second test, which examines whether and how much species differ from their inferred optima, shows that convergence is incomplete: that is, phenotypes of most species are still somewhat distant from the estimated optimum for each microhabitat, seemingly because of insufficient time for more complete adaptation (an effect of history). Yet, these effects of history are related to past ecologies, and not clade membership. Overall, our study elucidates the dominant drivers of morphological evolution across a major vertebrate clade and shows that evolution can be repeatable at much greater temporal and spatial scales than commonly thought. It also provides an analytical framework for testing other potential examples of large-scale convergence.</p></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/161?rss=1">An Efficient Independence Sampler for Updating Branches in Bayesian Markov chain Monte Carlo Sampling of Phylogenetic Trees</a></div>
  <div class="description"><p>Sampling tree space is the most challenging aspect of Bayesian phylogenetic inference. The sheer number of alternative topologies is problematic by itself. In addition, the complex dependency between branch lengths and topology increases the difficulty of moving efficiently among topologies. Current tree proposals are fast but sample new trees using primitive transformations or re-mappings of old branch lengths. This reduces acceptance rates and presumably slows down convergence and mixing. Here, we explore branch proposals that do not rely on old branch lengths but instead are based on approximations of the conditional posterior. Using a diverse set of empirical data sets, we show that most conditional branch posteriors can be accurately approximated via a  distribution. We empirically determine the relationship between the logarithmic conditional posterior density, its derivatives, and the characteristics of the branch posterior. We use these relationships to derive an independence sampler for proposing branches with an acceptance ratio of ~90% on most data sets. This proposal samples branches between 2x and 3x more efficiently than traditional proposals with respect to the effective sample size per unit of runtime. We also compare the performance of standard topology proposals with hybrid proposals that use the new independence sampler to update those branches that are most affected by the topological change. Our results show that hybrid proposals can sometimes noticeably decrease the number of generations necessary for topological convergence. Inconsistent performance gains indicate that branch updates are not the limiting factor in improving topological convergence for the currently employed set of proposals. However, our independence sampler might be essential for the construction of novel tree proposals that apply more radical topology changes.</p></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/177?rss=1">Biology of Evolution and Systematics: Cohesive, Concise, yet Comprehensive Introduction for Students and Professionals.-- By Paul Sanghera.</a></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/178?rss=1">Fishes: A Guide to their Diversity. -- Philip A. Hastings, Harold Jack Walker Jr., Grantly R. Galland, editors.</a></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/180?rss=1">Erratum</a></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a></div>
  <div class="description">Background:
Collagens provide structural support and guidance cues within the extracellular matrix of metazoans. Mammalian collagens XIII, XXIII and XXV form a unique subgroup of type II transmembrane proteins, each comprising a short N-terminal cytosolic domain, a transmembrane domain and a largely collagenous ectodomain. We name these collagens as MACITs (Membrane-Associated Collagens with Interrupted Triple-helices), and here investigate their evolution and conserved properties. To date, these collagens have been studied only in mammals. Knowledge of the representation of MACITs in other extant metazoans is lacking. This question is of interest for understanding structural/functional relationships in the MACIT family and also for insight into the evolution of MACITs in relation to the secreted, fibrillar collagens that are present throughout the metazoa.
Results:
MACITs are restricted to bilaterians and are represented in the Ecdysozoa, Hemichordata, Urochordata and Vertebrata (Gnathostomata). They were not identified in available early-diverging metazoans, Lophotrochozoa, Echinodermata, Cephalochordata or Vertebrata (Cyclostomata). Whereas invertebrates encode a single MACIT, collagens XIII/XXIII/XXV of jawed vertebrates are paralogues that originated from the two rounds of en-bloc genome duplication occurring early in vertebrate evolution. MACITs have conserved domain architecture in which a juxta-membrane furin-cleavage site and the C-terminal 34 residues are especially highly conserved, whereas the cytoplasmic domains are weakly conserved. To study protein expression and function in a metazoan with a single MACIT gene, we focused on Caenorhabditis elegans and its col-99 gene. A col-99 cDNA was cloned and expressed as protein in mammalian CHO cells, two antibodies against COL-99 protein were generated, and a col-99-bearing fosmid gene construct col-99::egfp::flag was used to generate transgenic C. elegans lines. The encoded COL-99 polypeptide is 85 kDa in size and forms a trimeric protein. COL-99 is plasma membrane-associated and undergoes furin-dependent ectodomain cleavage and shedding. COL-99 is detected in mouth, pharynx, body wall and the tail, mostly in motor neurons and muscle systems and is enriched at neuromuscular junctions.
Conclusions:
Through identification of MACITs in multiple metazoan phyla we developed a model for the evolution of MACITs. The experimental data demonstrate conservation of MACIT molecular and cellular properties and tissue localisations in the invertebrate, C. elegans.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a></div>
  <div class="description">Background:
Sclerolinum (Annelida: Siboglinidae) is a genus of small, wiry deep-sea tubeworms that depend on an endosymbiosis with chemosynthetic bacteria for their nutrition, notable for their ability to colonise a multitude of reducing environments. Since the early 2000s, a Sclerolinum population has been known to inhabit sediment-hosted hydrothermal vents within the Bransfield Strait, Southern Ocean, and whilst remaining undescribed, it has been suggested to play an important ecological role in this ecosystem. Here, we show that the Southern Ocean Sclerolinum population is not a new species, but more remarkably in fact belongs to the species S. contortum, first described from an Arctic mud volcano located nearly 16,000 km away.
Results:
Our new data coupled with existing genetic studies extend the range of this species across both polar oceans and the Gulf of Mexico. Our analyses show that the populations of this species are structured on a regional scale, with greater genetic differentiation occurring between rather than within populations. Further details of the external morphology and tube structure of S. contortum are revealed through confocal and SEM imaging, and the ecology of this worm is discussed.
Conclusions:
These results shed further insight into the plasticity and adaptability of this siboglinid group to a range of reducing conditions, and into the levels of gene flow that occur between populations of the same species over a global extent.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<h3>December 13, 2015</h3>
<div class="news-item">
 <div class="date">23:22</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135170344851">Job: CaliforniaStateU_MontereyBay.AquaticEvolution</a></div>
  <div class="description">Our department at California State University, Monterey Bay is
currently looking for a freshwater aquatic biologist. Take a look at
the job description and if you are qualified and  interested, please
apply at <a href="http://bit.ly/1J6cKtf.">http://bit.ly/1J6cKtf.</a>
Regards,
Nate Jue

Position Title: Assistant Professor of Aquatic Ecology and Water
Quality
Degree/Program BS: Environmental Science & Technology, Environmental
Studies, Biology, Marine Science / MS: Applied Marine & Watershed,
Marine Science
Priority Screening Date: 01/18/2016
Recruitment Status: Open Until Filled
Position Description:
The Division of Science & Environmental Policy (SEP) at California
State University, Monterey Bay (CSUMB) seeks applicants with a
demonstrated commitment to teaching and research in freshwater aquatic
ecology and water quality within natural, agricultural, and urban
landscapes in California&rsquo;s Central Coast region.
Projected responsibilities would likely include the following:
 - Teach upper division course in aquatic ecology (and co-listed
graduate course)
 - Teach upper division course in environmental monitoring - using
the study of biological and chemical water quality as a case study in
the application of experimental design, contemporary statistics, and
quantitative reasoning
 - Contribute as needed to teaching upper division course in critical
thinking and communication in environmental science and/or lower
division course in statistics (using R software)
 - Employ innovative teaching and learning techniques that engage
students through the use of technology, hands on experience, and
problem solving
 - Develop and maintain applied research activities with relevance to
aquatic ecological and water quality issues  within perennial and
non-perennial streams, coastal lagoons, vernal pools, lakes, and/or
reservoirs in California&rsquo;s Central Coast region
 - Provide research opportunities for undergraduates, including
involving students in field-based sampling of aquatic organisms and
monitoring of aquatic biological and chemical variables
 - Pursue and obtain extramural funding
Minimum Qualifications
 - Ph.D. in a field relevant to the requirements of this position, by
date of appointment (August 17, 2016)
 - Ability to facilitate student learning effectively for diverse
learners in a wide range of courses
 - Demonstrated excellence in instruction and commitment to
undergraduate education
 - Applied expertise in aquatic ecology and water quality
 - Applied expertise in application of contemporary statistical
approaches using R software
 - Excellent written and oral communication skills
Preferred Qualifications
 - Demonstrated ability and desire to mentor and teach students from
diverse cultural, educational, and economic backgrounds, and a
commitment to work effectively in a culturally diverse campus community
 - Research experience and interests that are relevant to freshwater
and lagoon environments within 1-hour of the CSUMB campus
 - Research and teaching interests and expertise that complement and
strengthen those of existing faculty in the department
 - Experience and interest in mentoring undergraduate & graduate
student research and/or internships
 - Successful record in obtaining external funding
 - Success in obtaining relevant permits e.g. for sampling aquatic
organisms
 - Experience working with Institutional Animal Care and Use Committees
(IACUCs)

Nathaniel K. Jue, Ph.D.
Assistant Professor
Division of Science and Environmental Policy
California State University, Monterey Bay
Seaside, CA 93955

Nathaniel Jue 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:52</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135101182801">Postdoc:North Carolina State U Applied Evol Biology</a></div>
  <div class="description"><p>&ndash;Apple-Mail=_9353E08B-C0E7-4F50-9E14-E31C2AB12FBD
Content-Transfer-Encoding: quoted-printable
Content-Type: text/plain;
	charset=windows-1252

Postdoc: Applied Evolutionary Biology

PROJECT DESCRIPTION: 

	   Selfish genetic elements have been studied for over a century, and as far back as the 1960&rsquo;s researchers became interested in using the power of selfish genetic elements to drive genes into pest species to suppress their impacts (Gould et al. 2006). Until recently, the focus of applied work was on naturally occurring elements. In the past decade some progress was made on developing synthetic elements that mimicked natural meiotic drive and selective embryo-killing, but de-novo creation of a gene-drive system in a pest species was elusive. With the harnessing of the bacterial CRISPR-Cas9 system in the past few years there has been a revolution brewing in this field (Esvelt et al. 2014, Oye et al. 2014). In March 2015 a pivotal article by Gantz and Bier (2015) came out in Science demonstrating a CRISPR-Cas9 construct in Drosophila with strong gene drive. 

            In the past month, two empirical papers came out that demonstrate CRISPR-Cas9 gene drive in the mosquito that transmits malaria and the mosquiot that transmits dengue (Gantz et al. 2015, Hammond et al. 2015). Not everyone is comfortable with these developments and there has even been a call for a moratorium on certain experiments. There are also concerns about nefarious use of the technology. 

            The  bottom-line is that progress in molecular biology is ahead of the population genetic work needed to build systems that are less risky but accomplish changes in the public interest.

              We have been funded by the NIH and the W. M. Keck Foundation to conduct this kind of population genetic research. Our focus has been on mosquitoes that transmit dengue and malaria, but we are also interested in other biological systems (There is hope that these selfish genetic elements can save endangered species like Hawaiian honeycreepers and specific seabirds (Gould 2008, Esvelt et al. 2014).

            The postdoc in this position will build a set of simple to complex models to examine the expected dynamics of gene drive systems in mosquitoes and other taxa.

            The most detailed model that we have developed simulates the population dynamics and population genetics of Aedes aegypti, the vector of dengue, in a city on the Amazon river, Iquitos, for which there are rich data sets on both mosquito dynamics and dengue epidemiology (e.g. Magori et al. 2009, Okamoto et al 2014). An accompanying epidemiological model is currently under development. The goals of two other postdocs in our group are to expand the mosquito model and the human epidemiology model to encompass the entire city of about 400,000 people. The postdoc in this new position will collaborate with the other postdocs to use these detailed models to test gene drive systems, but will also develop more generic models (e.g. Huang et al. 2010).

            In addition to working on model development and analysis, the person in this position will have the opportunity to collaborate in an interdisciplinary research group composed of mosquito ecologists, disease epidemiologists, molecular biologists, biomathematicians, ethicists, and scientists from disease-endemic countries. We are dedicated to taking seriously the ethical and political issues surrounding this technology.



DESIREABLE SKILLS: A background in population genetics and the ability to program in C++ (or knowledge of a related programming language), and training in evaluation of mechanistic models.

TO APPLY: email a cover letter and CV to Fred_Gould@ncsu.edu

 

References:

 Esvelt,  K. M.,  A. L. Smidler, F. Catteruccia, G. M. Church. 2014. Concerning RNA-guided gene drives for the alteration of wild populations. eLife. 10.7554/eLife.03401.

 Gantz, V. M.  and Bier, E. 2015. The mutagenic chain reaction: A method for converting heterozygous to homozygous mutations. Science 24 April 2015- 442-444. 

Gantz, V.M., N. Jasinskiene, O.Tatarenkova, A. Fazekas, V. M. Macias, E. Bier, and A. A. James. 2015. Highly efficient Cas9-mediated gene drive for population modification of the malaria vector mosquito Anopheles stephensi. PNAS 2015 112 (49) E6736-E6743; published ahead of print November 23, 2015,doi:10.1073/pnas.1521077112

 Gould, F. 2008. Broadening the application of evolutionarily based genetic pest management. Evolution 62: 500-510. 

Hammond, A. et al. 2015. A CRISPR-Cas9 gene drive system targeting female reproduction in the malaria mosquito vector Anopheles gambiae. Nature Biotechnology   Published online 07 December 2015   doi:10.1038/nbt.3439

 Huang, Y., Lloyd, A.L., Legros, M., Gould, F. 2010. Gene-drive into insect populations with age and spatial structure: a theoretical assessment. Evol. Appl. ISSN 1752-4571.

 Magori, K., M. Legros, M. Puente, D. A. Focks, T. W. Scott, A. Lloyd, F. Gould. 2009. Skeeter Buster: a stochastic, spatially-explicit modeling tool for studying Aedes aegypti population replacement and population suppression strategies. PLoS Negl Trop Dis 3(9): e508. doi:10.1371/journal.pntd.0000508

 Okamoto, K. W., Robert M. A., Gould, F., Lloyd, A. L. 2014. Feasible Introgression of an Anti-pathogen Transgene into an Urban Mosquito Population without Using Gene-Drive. PLoS Negl Trop Dis 8(7): e2827. doi:10.1371/journal.pntd.0002827.



 
&ndash;Apple-Mail=_9353E08B-C0E7-4F50-9E14-E31C2AB12FBD
Content-Transfer-Encoding: quoted-printable
Content-Type: text/html;
	charset=windows-1252

</p><p class="MsoNormal">Postdoc: Applied Evolutionary Biology</p><p class="MsoNormal">PROJECT DESCRIPTION: </p><p class="MsoNormal">	   Selfish genetic elements have been studied for over a century, and as far back as the 1960&rsquo;s researchers became interested in using the power of selfish genetic elements to drive genes into pest species to suppress their impacts (Gould et al. 2006). Until recently, the focus of applied work was on naturally occurring elements. In the past decade some progress was made on developing synthetic elements that mimicked natural meiotic drive and selective embryo-killing, but de-novo creation of a gene-drive system in a pest species was elusive. With the harnessing of the bacterial CRISPR-Cas9 system in the past few years there has been a revolution brewin
 g in this field (Esvelt et al. 2014, Oye et al. 2014). In March 2015 a pivotal article by Gantz and Bier (2015) came out in Science demonstrating a CRISPR-Cas9 construct in Drosophila with strong gene drive. </p><p class="MsoNormal">            In the past month, two empirical papers came out that demonstrate CRISPR-Cas9 gene drive in the mosquito that transmits malaria and the mosquiot that transmits dengue (Gantz et al. 2015, Hammond et al. 2015). Not everyone is comfortable with these developments and there has even been a call for a moratorium on certain experiments. There are also concerns about nefarious use of the technology. </p><p class="MsoNormal">            The  bottom-line is that progress in molecular biology is ahead of the population genetic work needed to build systems that are less risky but accomplish changes in the p
 ublic interest.</p><p class="MsoNormal">              We have been funded by the NIH and the W. M. Keck Foundation to conduct this kind of population genetic research. Our focus has been on mosquitoes that transmit dengue and malaria, but we are also interested in other biological systems (There is hope that these selfish genetic elements can save endangered species like Hawaiian honeycreepers and specific seabirds (Gould 2008, Esvelt et al. 2014).</p><p class="MsoNormal">            The postdoc in this position will build a set of simple to complex models to examine the expected dynamics of gene drive systems in mosquitoes and other taxa.</p><p class="MsoNormal">            The most detailed model that we have developed simulates the population dynamics and population genetics of <i>Aedes aegypti</i>, the vecto
 r of dengue, in a city on the Amazon river, Iquitos, for which there are rich data sets on both mosquito dynamics and dengue epidemiology (e.g. Magori et al. 2009, Okamoto et al 2014). An accompanying epidemiological model is currently under development. The goals of two other postdocs in our group are to expand the mosquito model and the human epidemiology model to encompass the entire city of about 400,000 people. The postdoc in this new position will collaborate with the other postdocs to use these detailed models to test gene drive systems, but will also develop more generic models (e.g. Huang et al. 2010).</p><p class="MsoNormal">            In addition to working on model development and analysis, the person in this position will have the opportunity to collaborate in an interdisciplinary research group composed of mosquito ecologists, disease epidemiologists, molecular biologists, biomathematicia
 ns, ethicists, and scientists from disease-endemic countries. We are dedicated to taking seriously the ethical and political issues surrounding this technology.</p><p class="MsoNormal"></p><p class="MsoNormal">DESIREABLE SKILLS: A background in population genetics and the ability to program in C++ (or knowledge of a related programming language), and training in evaluation of mechanistic models.</p><p class="MsoNormal">TO APPLY: email a cover letter and CV to <a href="mailto:Fred_Gould@ncsu.edu">Fred_Gould@ncsu.edu</a></p><p> </p><p class="MsoNormal">References:</p><p> </p>Esvelt,  K. M.,  A. L. Smidler, F. Catteruccia, G. M. Church. 2014. Concerning RNA-guided gene drives for the alteration of wild populations. eLife. 10.7554/eLife.03401.<p> </p>Gantz, V. M.  and Bier, E. 2015. The mutagenic chain rea
 ction: A method for converting heterozygous to homozygous mutations. Science 24 April 2015- 442-444. <p></p>Gantz, V.M., N. Jasinskiene, O.Tatarenkova, A. Fazekas, V. M. Macias, E. Bier, and A. A. James. 2015. Highly efficient Cas9-mediated gene drive for population modification of the malaria vector mosquito Anopheles stephensi. PNAS 2015 112 (49) E6736-E6743; published ahead of print November 23, 2015,doi:10.1073/pnas.1521077112<p> </p>Gould, F. 2008. Broadening the application of evolutionarily based genetic pest management. Evolution 62: 500-510. <p class="MsoNormal">Hammond, A. et al. 2015. A CRISPR-Cas9 gene drive system targeting female reproduction in the malaria mosquito vector Anopheles gambiae. Nature Biotechnology   Published online 07 December 2015   doi:10.
 1038/nbt.3439</p><p> </p>Huang, Y., Lloyd, A.L., Legros, M., Gould, F. 2010. Gene-drive into insect populations with age and spatial structure: a theoretical assessment. Evol. Appl. ISSN 1752-4571.<p> </p>Magori, K., M. Legros, M. Puente, D. A. Focks, T. W. Scott, A. Lloyd, F. Gould. 2009. Skeeter Buster: a stochastic, spatially-explicit modeling tool for studying Aedes aegypti population replacement and population suppression strategies. PLoS Negl Trop Dis 3(9): e508. doi:10.1371/journal.pntd.0000508<p> </p>Okamoto, K. W., Robert M. A., Gould, F., Lloyd, A. L. 2014. Feasible Introgression of an Anti-pathogen Transgene into an Urban Mosquito Population without Using Gene-Drive. PLoS Negl Trop Dis 8(7): e2827. doi:10.1371/journal.pntd.0002827.<p class="MsoNormal"></p><p> </p>
&ndash;Apple-Mail=_9353E08B-C0E7-4F50-9E14-E31C2AB12FB
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:22</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135100029646">Postdoc: Switzerland.Basel.host-parasite coevolution/genetics</a></div>
  <div class="description"><p>University of Basel, Switzerland
Zoological Institute

Postdoc position in host-parasite
population genetics and genomics

A postdoc is available in the research group of Dieter Ebert, at the 
Zoological Institute at Basel University in Switzerland. I am looking 
for a highly motivated post-doc with interest in the (co-)evolutionary 
genetics of host-parasite interactions. This position is funded to work 
on the coevolutionary genetics/genomics of Daphnia and its 
microparasites. A background in evolutionary genetics and bioinformatics 
skills are welcome. Excellent written, verbal, and interpersonal skills, 
a superb work ethic, and the ability to think creatively and critically 
are desired. The starting date is flexible. The position is initially 
for 2 years, but can be extended to 3 years.

The post-doc will be part of Dieter Ebert&rsquo;s research group, working on 
the co-evolution of host-microbe interactions. Our experimental work 
uses Daphnia as the host system. Details about the group can be found 
under: <a href="http://bit.ly/18Irt0r">http://bit.ly/18Irt0r</a>

Please send application by E-mail to Dieter Ebert 
(dieter.ebert@unibas.ch). Applications (all in one single pdf file with 
your name as file-name, please) should include a CV, a list of 
publications and a 1-page description of your research interests and 
motivation. Please give names and email addresses of two persons who are 
willing to write a letter of recommendation. Deadline for applications 
is 31. January 2016.


Contact information: Prof. Dr. Dieter Ebert,
University of Basel,
Zoological Institute,
Vesalgasse 1, 4051 Basel, Switzerland,
Email: dieter.ebert@unibas.ch
Tel. +41-(0)61-267 03 60
<a href="http://bit.ly/1OPqIml">http://bit.ly/1OPqIml</a>


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 12, 2015</h3>
<div class="news-item">
 <div class="date">23:52</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135098759211">Postdoc: RBG_Kew.OrchidEvolution</a></div>
  <div class="description"><p>Early Career Research Fellowships (Sainsbury Orchid Fellowships) at
RBG Kew

The Royal Botanic Gardens, Kew seeks outstanding postdoctoral candidates
to take up two Early Career Research Fellowships focusing on orchid
diversity, taxonomy, systematics and evolution. The successful candidates
will establish their own independent research programmes closely aligned
with Kew&rsquo;s Science Strategy, developing funding streams, producing
excellent publications, and engaging fully with science life across
the institution. They will be future leaders in the science disciplines
pursued at Kew.

We seek applicants who are outstanding early career scientists with a PhD
in a relevant subject area (awarded within the past eight years) and,
ideally, some postdoctoral experience. Applicants will have a proven
aptitude for delivering excellent science publications and demonstrated
potential to raise science income. Applicants should also be outstanding
and enthusiastic communicators who are ready to engage with students,
peers and the general public.

Closing date for applications: 10 January 2016

For further details, follow this link:
<a href="http://bit.ly/1lXuWQp">http://bit.ly/1lXuWQp</a>


W.Baker@kew.org


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:22</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135097397841">Other: Hokkaido_Japan.VolResAssist.SeabirdEvolution</a></div>
  <div class="description"><p>Behavioural Ecology of seabirds by using bio-logging techniques.

Location: Teuri and Rishiri Islands, Hokkaido, Japan. Duration: 3 months.
Job Type: Volunteer.

We use bio-logging devices (e.g. geolocators, GPS recorders) on two species
(Rhinoceros Auklets and Black-backed Gulls) to study behavioural ecology of
colonial seabirds. Most work will involve assisting a long-term monitoring
project and behavioural studies of post-doc fellows and graduate students
from Hokkaido University coordinated by Yutaka Watanuki, with the daily
monitoring breeding birds and their chicks, and the deployment and
retrieval of tracking devices. It will also involve entering data collected
in the field into databases.

We are looking for an enthusiastic, hardworking young biologist to work as
a voluntary Research Assistant to help with our seabird research on
Teuri/Rishiri Island, Hokkaido, Japan, between May and July 2015. Previous
experience in the field or in harsh conditions is desirable, but not
essential. Work on Teuri/Rishiri requires long hours (with some work at
night and at dawn) spent in a cold and wet environment and considerable
manual work. A successful candidate should be able to work independently
and have a reasonable level of fitness. Planning to embark on further
research degrees in the future would be preferable. It is also important
that the assistant feels comfortable sharing sometimes crowded
accommodation with a number of colleagues over long periods of time, and
can cope with the occasional untidiness inevitably associated with
fieldwork.

Unfortunately we cannot provide a stipend but accommodation (shared and
basic) on Teuri/Rishiri will be covered. However, the project is a great
opportunity to gain valuable experience in modern field ornithology, with
bio-logging techniques. There will also be opportunities for the assistant
to learn some analytical techniques applied to animal movement data.
Teuri/Rishiri has mobile coverage and some access to Internet, however
movements to and from the mainland are limited because of unpredictable
wind conditions. Teuri/Rishiri is an inhabited island and there are small
restaurants and grocery stores. There is a small community of other
researchers, ringers and volunteers at our shared accommodation.

If you are interested, please email a CV and a statement of interest and
arrange for two letters of reference to be emailed to akikosho@gmail.com
 (with ywata@fish.hokudai.ac.jp in cc). The deadline for submitting
applications (including reference letters) is the 31st January 2016. We
will interview the shortlisted candidates (on Skype) shortly after and we
expect to make a decision by late February.


akikosho@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 11, 2015</h3>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/279">Climate oscillations, glacial refugia, and dispersal ability: factors influencing the genetic structure of the least salmonfly, Pteronarcella badia (Plecoptera), in Western North America</a></div>
  <div class="description">Background:
Phylogeographic studies of aquatic insects provide valuable insights into mechanisms that shape the genetic structure of communities, yet studies that include broad geographic areas are uncommon for this group. We conducted a broad scale phylogeographic analysis of the least salmonfly Pteronarcella badia (Plecoptera) across western North America. We tested hypotheses related to mode of dispersal and the influence of historic climate oscillations on population genetic structure. In order to generate a larger mitochondrial data set, we used 454 sequencing to reconstruct the complete mitochondrial genome in the early stages of the project.
Results:
Our analysis revealed high levels of population structure with several deeply divergent clades present across the sample area. Evidence from five mitochondrial genes and one nuclear locus identified a potentially cryptic lineage in the Pacific Northwest. Gene flow estimates and geographic clade distributions suggest that overland flight during the winged adult stage is an important dispersal mechanism for this taxon. We found evidence of multiple glacial refugia across the species distribution and signs of secondary contact within and among major clades.
Conclusions:
This study provides a basis for future studies of aquatic insect phylogeography at the inter-basin scale in western North America. Our findings add to an understanding of the role of historical climate isolations in shaping assemblages of aquatic insects in this region. We identified several geographic areas that may have historical importance for other aquatic organisms with similar distributions and dispersal strategies as P. badia. This work adds to the ever-growing list of studies that highlight the potential of next-generation DNA sequencing in a phylogenetic context to improve molecular data sets from understudied groups.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/278">Mitochondrial group I and group II introns in the sponge orders Agelasida and Axinellida</a></div>
  <div class="description">Background:
Self-splicing introns are present in the mitochondria of members of most eukaryotic lineages. They are divided into Group I and Group II introns, according to their secondary structure and splicing mechanism. Being rare in animals, self-splicing introns were only described in a few sponges, cnidarians, placozoans and one annelid species. In sponges, three types of mitochondrial Group I introns were previously described in two demosponge families (Tetillidae, and Aplysinellidae) and in the homoscleromorph family Plakinidae. These three introns differ in their insertion site, secondary structure and in the sequence of the LAGLIDADG gene they encode. Notably, no group II introns have been previously described in sponges.
Results:
We report here the presence of mitochondrial introns in the cytochrome oxidase subunit 1 (COI) gene of three additional sponge species from three different families: Agelas oroides (Agelasidae, Agelasida), Cymbaxinella p verrucosa (Hymerhabdiidae, Agelasida) and Axinella polypoides (Axinellidae, Axinellida). We show, for the first time, that sponges can also harbour Group II introns in their COI gene, whose presence in animals’ mitochondria has so far been described in only two phyla, Placozoa and Annelida. Surprisingly, two different Group II introns were discovered in the COI gene of C. verrucosa. Phylogenetic analysis indicates that the Group II introns present in C. verrucosa are related to red algae (Rhodophyta) introns.
Conclusions:
The differences found among intron secondary structures and the phylogenetic inferences support the hypothesis that the introns originated from independent horizontal gene transfer events. Our results thus suggest that self-splicing introns are more diverse in the mitochondrial genome of sponges than previously anticipated.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:49</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/134974328116">Job: UCalifornia_LosAngeles.PlantEvolution</a></div>
  <div class="description">Assistant Professor in Plant Diversity and Evolution

The University of California, Los Angeles (UCLA) Department of Ecology and
Evolutionary Biology (EEB) seeks an organismal biologist with a focus on
plant diversity and/or evolution. UCLA has a rich history of work in plant
ecology and evolution, and we seek an individual who will help us maintain
and grow this critical component of our program. Qualified candidates must
have a Ph.D. in a related field of biological sciences, and the search
is restricted to the Assistant Professor level. The position is defined
broadly within evolution and ecology, but preference will be given to
candidates whose research/teaching interests would utilize, at least in
part, the UCLA Mildred E. Mathias Botanical Garden and associated UCLA
Herbarium. The successful candidate will be expected to establish an
internationally recognized and externally funded research program and
contribute to EEBs teaching needs at the introductory undergraduate
level. There are many opportunities for collaboration across a broad
group of partners on and off campus, including the UC NRS Stunt Ranch
Reserve and White Mountains Research Center, the UCLA La Kretz Center for
California Conservation Science, the UCLA Center for Tropical Research,
and our Institute of Environment and Sustainability. Information
about the Department and the Botanical Garden and may be found at
<a href="http://bit.ly/1wGtwN2">http://bit.ly/1wGtwN2</a> and <a href="http://bit.ly/Xm6Rq0.">http://bit.ly/Xm6Rq0.</a>

Applicants should submit materials online to
<a href="http://bit.ly/1TEDl6p">http://bit.ly/1TEDl6p</a> and include 1) a cover letter,
2) curriculum vitae, 3) personal statement that describes current and
future research and teaching, a vision of how the candidates research,
teaching and/or outreach might include and enhance the Botanical Garden,
4) a summary of ongoing and anticipated activities to promote gender
and racial diversity, and 5) names and contact information of three
references. Review of candidates will begin on 18 January 2016 and
continue until the position is filled. Please use job number JPF01858
in all correspondence.

Individuals with a history of mentoring students under-represented in
the sciences are encouraged to apply and to describe their experience
in a cover letter.

Inquiries regarding the position should be directed to Grace Angus
(gracea@lifesci.ucla.edu) or the Search Chair, Professor Brad Shaffer
(brad.shaffer@ucla.edu).

As a campus with a continually growing diverse student body, we encourage
applications from women, minorities, and individuals with a commitment to
mentoring under-represented demographics in the sciences. The University
of California is an Equal Opportunity/Affirmative Action Employer. All
qualified applicants will receive consideration for employment without
regard to race, color, religion, sex, sexual orientation, gender identity,
national origin, disability, age or protected veteran status. For the
complete University of California nondiscrimination and affirmative
action policy see: UC Nondiscrimination and Affirmative Action Policy
(<a href="http://bit.ly/1tbbcKE">http://bit.ly/1tbbcKE</a>).

 
Thanks,
Mario 

Mario Colon     I Administrative Assistant    I UCLA La Kretz Center for California Conservation Science
818-519-7740  I mario.colon@ucla.edu       I <a href="http://bit.ly/1aXROSY">http://bit.ly/1aXROSY</a>

Mario Colon 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="indexd6af.html?q=aggregator&amp;from=420">previous page</a></div><div>... <a href="index28a4.html?q=aggregator&amp;from=360">19</a> <a href="index63c1.html?q=aggregator&amp;from=380">20</a> <a href="index940a.html?q=aggregator&amp;from=400">21</a> <a href="indexd6af.html?q=aggregator&amp;from=420">22</a> <strong>23</strong> <a href="index3d2c.html?q=aggregator&amp;from=460">24</a> <a href="indexb567.html?q=aggregator&amp;from=480">25</a> <a href="index8c9a.html?q=aggregator&amp;from=500">26</a> <a href="indexbe68.html?q=aggregator&amp;from=520">27</a> ...</div><div><a href="index3d2c.html?q=aggregator&amp;from=460">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=440 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:37:54 GMT -->
</html>
