<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/14&from=120 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:29:07 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexfc9c.html?q=node/431">About Women in Science (Evol2014)</a></li><li><a href="indexf09b.html?q=node/426">Researcher in Environmental Bioinformatics, University of Technology Sydney</a></li><li><a href="indexb172.html?q=node/425">Minutes of mid-year meeting 2014</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 5 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator/sources/14" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator">news aggregator</a>
<ul>
<li class="expanded"><a href="indexe5f5.html?q=aggregator/sources">sources</a>
<ul>
<li class="leaf"><a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></li>
<li class="leaf"><a href="index4421.html?q=aggregator/sources/8">CiteULike Phylogeny</a></li>
<li class="leaf"><a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></li>
<li class="leaf"><a href="index4dbd.html?q=aggregator/sources/14" class="active">EVOLDIR</a></li>
<li class="leaf"><a href="index8aad.html?q=aggregator/sources/9">Evolutionary Bioinformatics</a></li>
<li class="leaf"><a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></li>
<li class="leaf"><a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></li>
<li class="leaf"><a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></li>
<li class="leaf"><a href="index9fb6.html?q=aggregator/sources/22">Phyloseminar</a></li>
<li class="leaf"><a href="index69a2.html?q=aggregator/sources/10">Systematics Association</a></li>
<li class="leaf"><a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></li>
<li class="leaf"><a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></li>
<li class="leaf"><a href="indexe0bd.html?q=aggregator/sources/15">dechronization</a></li>
<li class="leaf"><a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></li>
<li class="leaf"><a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></li>
<li class="leaf"><a href="index1f4a.html?q=aggregator/sources/7">postgenomic.com</a></li>

</ul>
</li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 4 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a> &raquo; <a href="index5640.html?q=aggregator">news aggregator</a> &raquo; <a href="indexe5f5.html?q=aggregator/sources">sources</a></div><h2>EVOLDIR</h2>
<!-- begin content -->
<div id="aggregator"><div class="feed"><h3>URL</h3>
<div class="xml-icon"><a href="http://evoldir.tumblr.com/rss"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div><a href="http://evoldir.tumblr.com/">http://evoldir.tumblr.com/</a>
<h3>Last update</h3>
9 min 27 sec ago</div><h3>February 9, 2016</h3>
<div class="news-item">
 <div class="date">02:50</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138981615971">Workshop: AuburnU.Bioinformatics.Jun6-10</a></div>
  <div class="description">Bioinformatics Bootcamp at Auburn University

>From June 6V10, 2016 the College of Science and Mathematics will
host the third annual Bioinformatics Bootcamp X a training workshop
in genomics and computational biology for researchers interested in
working with modern genomic data resources.  The Bootcamp is a week-long
immersion experience in the command line, scripting, data handling and
software skills required to process and analyze data from high-throughput
sequencing experiments. Previous experience in bioinformatics is not
required; interactive lectures and group data analysis activities will
introduce participants to best practices in sequencing experiment design
and analysis including widely used methods in genome and transcriptome
assembly, annotation, differential expression and variant analysis.

The 2016 Bootcamp will feature a keynote lecture by Dr. Monica
Munoz-Torres, Lead Biocurator and Bioinformatics Analyst of the Berkeley
Bioinformatics Open-source Projects (BBOP), Lawrence Berkeley National
Laboratory.

The Bootcamp registration fee is $500. To apply, send a brief
statement of interest and two-page CV (as a single pdf) to
bioinformatics@auburn.edu
by April 1, 2016. Additional information can be found at
<a href="http://bit.ly/1O0WsCY">http://bit.ly/1O0WsCY</a>

Kenneth Halanych 
via Gmail</div>
 </div>
</div>
<h3>February 6, 2016</h3>
<div class="news-item">
 <div class="date">01:04</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138779784866">Postdoc: SpelmanC_Atlanta.MicrobialEvolution</a></div>
  <div class="description">Postdoc Position in: Microbial Evolution, Behavior and Genomics

Location: Atlanta, Georgia, USA

Funding Source: CURE Postdoctoral Fellowships [Spelman College is
piloting a postdoctoral fellowship program that combines a traditional
principal-investigator directed postdoctoral research experiences with
mentored teaching experiences.]


A postdoc position (2.5 years) is available to conduct experimental and
computational research on small eukaryotic amoeboid microbes relating
to their evolution and sexual life cycle behavior. Candidates should
have an inter-disciplinary training or familiarity with molecular
(e.g. next generation sequencing) and computational (bioinformatics,
phylogenetics) skills.

Potential research topics include (i) genomics and evolution of phenotypic
traits in cryptic species, (ii) evolution of sexual-like behavior and
development in amoeboids, (iii). origin, evolution and biodiversity
of amoeboids using large-scale analysis. (iv). evolution of amoeboid
movement and cytoskeleton architecture. Additional projects under
similar topics can be also considered based on individual experience
and interest. For additional information contact: ytekle@spelman.edu

This position offers competitive salary and benefit. Candidates should
complete Ph.D. training before starting this position. This position
is open to US citizens and lawful permanent residents only. Screening
of applicants will begin immediately and continue until the position
is filled.

Spelman College is a private four-year liberal arts college located in
Atlanta, GA, and ranked one of the top 100 liberal arts colleges by the
US News.

Please email a copy of curriculum vitae, a short statement of research
interest, names and contact information of at least three individuals
familiar with your research to Dr. Yonas Tekle through ytekle@spelman.edu



350 Spelman Ln SW, Box 1183,
Department of Biology, Spelman College, Atlanta, GA 30314-4399
LOCATIONS
Office: AFM Science Center 271;  Lab: AFM Science Center 247
Phone: 404-270-5779 (office);  Fax: 404-270-5725


Yonas Isaak Tekle 
via Gmail</div>
 </div>
</div>
<h3>February 5, 2016</h3>
<div class="news-item">
 <div class="date">23:49</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138777169671">Course: Switzerland.TamingTheBeast.Jun26-Jul1</a></div>
  <div class="description">Dear colleagues,

We are happy to announce our summer school &ldquo;Taming the BEAST&rdquo; and would
kindly ask you to forward this email to interested students and
PostDocs. We apologize if you receive this email more than once.
Phylogenetics and phylodynamics are central topics in modern biology.
Phylogenetic inferences reconstruct the evolutionary relationships
between organisms, whereas phylodynamic inferences reveal the dynamics
that lead to the observed relationships. These two fields have many
practical applications in disciplines such as epidemiology,
developmental biology, paleontology, ecology and even linguistics.
However, phylogenetics and phylodynamics are complex and fast-evolving
fields. As such, inference tools are not easily accessible to
researchers who are not from a computational background.

Taming the BEAST is a summer school focusing on the BEAST2 software and
consisting of a mix of invited talks, lectures and hands-on tutorials
by leading and renowned experts in the field (including several of the
core developers of BEAST2). The aim of this summer school is to equip
participants with the skills necessary to confidently perform their own
phylogenetic and phylodynamic inferences in Bayesian settings, while
providing them with a firm grasp of the theory behind those
inferences. Participants are also highly encouraged to bring their own
datasets along and to engage with the organizers and speakers to
address any problems specific to their own datasets/analyses.
Registration is now open! We welcome applications from graduate
students and early-career scientists in the life sciences. Preference
will be given to applicants who are not from a computational background
and applicants who have already collected/assembled a dataset that they
need to analyze.

Invited speakers:

Tanja Stadler (ETH Zurich)

Alexei Drummond (University of Auckland)

Tracy Heath (Iowa State University)

Tim Vaughan (University of Auckland)

Oliver Pybus (University of Oxford)

Dates: June 26th to July 1st  (Registration deadline: March 15th).

Place: Engelberg, Switzerland

Registration Fee: 650 CHF for students and postdocs, 1000 CHF for
industry. (Registration fee includes accommodation and meals).

For more information regarding the program, registration,
venue, etc, please visit the summer school website:
<a href="http://bit.ly/1USzh3b">http://bit.ly/1USzh3b</a>

We hope to see you there,
the Taming the BEAST organizing team

Best regards,
Venelin Mitov
Department of Biosystems Science & Engineering
Mattenstrasse 26
4058 Basel
Switzerland
Email: venelin.mitov@bsse.ethz.ch
Phone: +41 61 38 73411
Office: 7.48

Mitov  Venelin 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:57</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138720407016">Postdoc: ToulouseCNRS.WildPlantQuantitativeGenetics</a></div>
  <div class="description"><p>This is a multi-part message in MIME format.
via Gmail</p></div>
 </div>
</div>
<h3>February 4, 2016</h3>
<div class="news-item">
 <div class="date">23:27</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138715638391">Postdoc: CharlesU_Prague.NightingaleSpeciation</a></div>
  <div class="description"><p>Post doc position at Charles University in Prague

Genomics of speciation in nightingales

We are looking for a post-doc trained in bioinformatics and/or
population genetics. The main goal of the project is to elucidate the
genetic basis of reproductive isolation between two song bird species,
the Common Nightingale (Luscinia megarhynchos) and the Thrush
Nightingale (L. luscinia). The species hybridize in a secondary
contact zone comprising F1 as well as backcross hybrids. According to
Haldane’s rule F1 hybrid females are sterile, while F1 hybrid males
are fertile. Using ddRAD sequencing of natural hybrid population and
transcriptome sequencing of gonads and somatic tissues of both species
and hybrids we aim to identify candidate reproductive isolation loci
responsible for hybrid female sterility. The results should provide
the first insight into genetic basis of Haldane’s rule in
heterogametic female organisms.

The candidate will be mainly responsible for the bioinformatic part of
the project. Samples as well as most sequence data are already
available. The position is available from May 2016 to December 2017
with the possibility of extension. The starting date is flexible but
preferable before September 2016. The candidate will work at the
Department of Zoology at Charles University in Prague under the
supervision of Radka Reifova (<a href="http://bit.ly/1L2eCEl">http://bit.ly/1L2eCEl</a>). The
salary will be 35,000 CZK (ca 1,300 EUR) (Note that living expenses in
the Czech Republic are generally lower than in Western European
countries.)

If interested, please, send (1) a CV including a list of publications,
(2) motivation letter and (3) a letter of reference of your PhD
supervisor or research advisor to Radka Reifova
(radka.reifova@natur.cuni.cz) by 29th February 2016.


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:12</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138658057506">Conference: Lyon.BehaviourEvolution.Jun27</a></div>
  <div class="description">We inform you that the twelfth edition of the &ldquo;ECOLOGY & BEHAVIOUR&rdquo;
meetings will occur this year in Lyon (France) between June 27th and July
1st 2016.

The &ldquo;Ecology & Behaviour&rdquo; conference is an international scientific
congress in the themes of ecology, evolutionary biology, and animal
behaviour that distinguishes itself by the will clearly displayed to favour
the participation of PhD and Post-doctoral students.

Unlike most congresses and symposium, NO CHARGE is asked to PhD students
and post-doctoral students (including accommodation and lunches) if they
participate through the presentation of a poster or an oral communication
(in English).



We are now accepting abstracts for graduate students and post-docs
presentations in talk and poster formats. Please visit our presentation
registration page for more information on submission requirements and
procedures:



eb2016.sciencesconf.org



*The deadline for abstract submissions is March 20th, 2016*.



The E&B organization committee
eb2016.sciencesconf.org

Christophe Plantamp 
via Gmail</div>
 </div>
</div>
<h3>February 3, 2016</h3>
<div class="news-item">
 <div class="date">00:30</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138594669601">Postdoc: Kiel_Germany.1PDF_3PhD.CelegansEvolution</a></div>
  <div class="description">Kiel University, Germany

    1 Postdoc + 3 PhD positions on the evolution and genetics of C.
                    elegans-microbiota interactions

The positions are funded within a larger research initiative on
host-microbiota interactions at Kiel University (Collaborative Research
Center CRC1182 â€œOrigin and function of metaorganismsâ€�).

The postdoc and one PhD position are supervised by Hinrich Schulenburg
(projects A1.1 and A4.3) and address the evolutionary dynamics and
genomics of C. elegans microbiota interactions, based on evolution
experiments and genome sequencing (Schulenburg group).

The second PhD project (project A1.2) is supervised by Katja Dierking
(Schulenburg lab) and looks at the transcriptomic and also genetic
basis of the interaction, using RNASeq and functional genetic analysis.

The third PhD project (project A1.3) is supervised by Matthias Leippe
and assesses the involvement of antimicrobial peptides and proteins in
shaping C. elegans-bacteria interactions using biochemical and
functional genetic approaches (Leippe group).

For further information, please check the webpage of the Collaborative
Research Center (<a href="http://bit.ly/1PfXkWc">http://bit.ly/1PfXkWc</a>
) or the Schulenburg group (which includes Katja Dierking;
<a href="http://bit.ly/1RZzweV">http://bit.ly/1RZzweV</a>) or the Leippe group
(<a href="http://bit.ly/1PfXkWe">http://bit.ly/1PfXkWe</a>).

Application procedures are described on the webpage of the Collaborative
Research Center (see above link; then under Jobs).  Deadline for
applications is 29 February 2016. If you have questions, then please
contact either Hinrich Schulenburg (hschulenburg@zoologie.uni-kiel.de),
Katja Dierking (kdierking@zoologie.uni-kiel.de), or Matthias Leippe
(mleippe@zoologie.uni-kiel.de).

Hinrich Schulenburg

Zoological Institute
Christian-Albrechts-Universitaet zu Kiel
Am Botanischen Garten 1-9
24118 Kiel
Germany
Tel: +49-431-880-4143/4141
Fax: +49-431-880-2403
Email: hschulenburg@zoologie.uni-kiel.de
Web: <a href="http://bit.ly/1RZzweV">http://bit.ly/1RZzweV</a>

Hinrich Schulenburg 
via Gmail</div>
 </div>
</div>
<h3>February 2, 2016</h3>
<div class="news-item">
 <div class="date">02:25</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138535098621">Postdoc: NotreDame_BroadInst.MosquitoSpeciationGenomics</a></div>
  <div class="description">POSTDOC: Species boundaries and evolutionary history of a cryptic radiation
of malaria vectors

A Postdoctoral position funded by a 2-year NIH grant is available
April 2016 to work between two population genomics groups: that of Nora
Besansky at the University of Notre Dame (<a href="http://ntrda.me/1huzbgB">http://ntrda.me/1huzbgB</a>),
and that of Daniel Neafsey at the Broad Institute of MIT and Harvard
(<a href="http://bit.ly/20DuyG3">http://bit.ly/20DuyG3</a>).

We are seeking highly qualified and enthusiastic applicants with a
background in population genetics and evolution, to analyze whole genome
sequence data and infer speciation history in the *Anopheles funestus*
complex.

BACKGROUND: Major malaria vectors are a rarity in the mosquito genus
Anopheles, and they are not phylogenetically clustered.  On the contrary,
malaria vectors are almost invariably members of sibling species complexes
containing minor or non-vector species, indicating that the traits endowing
major vector status have evolved recently, rapidly and repeatedly in
different species complexes.  Powerful insights about the genetic basis of
vector traits can be gained by comparative genomics of vectors and
non-vectors across these replicate systems of vector evolution.  A crucial
first step toward that end was taken with the assembly of 16 *Anopheles*
genomes led by Besansky at Notre Dame and Neafsey at the Broad Institute,
with the *Anopheles gambiae* complex as one case study (Neafsey et al 2015
Science; Fontaine et al 2015 Science).  A next step is to consider other
lesser known but equally important sibling species complexes.  *Anopheles
funestus* is one of the three most important and widespread vectors of
human malaria in tropical Africa, but unlike *An. gambiae*, it is
understudied.  This major vector conceals a group of close relatives that
are morphologically similar or identical as adults.  The taxonomic
complexity of this group is at least as high as the *An. gambiae* complex;
within each named (and unnamed) species are genetically heterogeneous
clusters whose interrelationships have never been resolved, and additional
cryptic diversity is being reported by vector control programs.  Moreover,
echoing the situation in the *An. gambiae* complex, there is evidence for
historical or ongoing introgression between An. funestus and at least one
other taxon.

PROJECT: As part of this NIH project, complete de novo reference genome
assemblies will be generated at the Broad for available species in the *An.
funestus* complex, based on the DISCOVAR de novo algorithm.  Additional low
coverage HiSeq data will be generated from natural population samples.  The
successful postdoctoral candidate will use the new reference assemblies and
additional light whole genome sequencing (~12 samples/taxon) to construct
an initial portrait of population genomic patterns and species
relationships for the *An. funestus* complex. Although the position is
based at Notre Dame, the postdoc will have the opportunity to split time
between Notre Dame and Broad, benefitting from interactions with active
research groups in both stimulating locations.

REQUIREMENTS: (1) A PhD in genetics/genomics, evolutionary biology, or a
related discipline; (2) a strong background in statistics and population
genetics; (3) good communication skills and proficiency in English, both
speaking and writing.

PREFERENCES: The ideal candidate will be a highly motivated individual with
(i) proficiency in the Linux/Unix computing environment, familiarity with
R, and at least one other programming language (e.g. Python or Perl); (ii)
experience with the analysis of high throughput biological data sets; and
(iii) an excellent record of scientific publication.

HOW TO APPLY: Applicants should submit a single PDF file (
tofunestus.postdoc@gmail.com) containing (1) a cover letter briefly
summarizing past research accomplishments and future goals; (2) a CV; (3)
the names and contact information for three references. Screening of
applicants will begin immediately and continue until the position is
filled.

ADDITIONAL INFORMATION: The University of Notre Dame is located on the
northern limits of South Bend, Indiana [90 miles (150 km) east of Chicago].
Post-doctoral scholars in the Department of Biological Sciences
(biology.nd.edu) have access to outstanding faculty, cutting-edge research
infrastructure, and various avenues for career development. Research in
the department spans the wide spectrum of the life sciencesXfrom cells
and organs to whole organisms and ecosystems. This uncommon level of
wide-ranging yet integrative biological inquiry provides an extraordinary
environment for learning, innovation and collaboration. Centers and
Institutes enhance our intellectual mission, and include the Eck
Institute for Global Health, Center for Rare and Neglected Diseases,
and the Environmental Change Initiative.  Available resources include
the Freimann Animal Facility and state of the art core facilities:
Genomics, Proteomics, and Integrated Imaging. The Center for Research
Computing operates a High Performance Computing (HPC) facility providing
parallel supercomputers, clusters, grid networks and storage in systems
of various architectures and interconnects, and access to the TeraGrid
and Open Science Grid.

The University of Notre Dame is an Equal Opportunity / Affirmative Action
Employer.

Daniel Neafsey 
via Gmail</div>
 </div>
</div>
<h3>January 29, 2016</h3>
<div class="news-item">
 <div class="date">03:04</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138271747141">Graduate position: Halle_Germany.HoneybeeResistance.2</a></div>
  <div class="description">Halle_Germany.HoneybeeResistance

PhD project in the Molecular Ecology group at Martin-Luther-University,
Halle-Wittenberg, Germany.

The aim of the project is to find genetic basis of the honey bee Apis
mellifera resistance to Varroa destructor. This will be done by using
the latest methods in genomics, transcriptomics and proteomics. Thus
representing a unique opportunity to gain experience in an area which
is of high demand in the research field. The group has excellent
laboratory facilities with all state of the art equipments including a
next generation sequencing platform.

The PhD position is funded for three years in which time the PhD
student is expected to finish her/his degree.

The chosen PhD student is expected to work independently within the
project framework. The skills should reflect the project description.
The candidate will be familiar with standard molecular methods in the
lab as well as bioinformatics and running scripts. Willingness to learn
new methods is a must as well as ability to adapt to ever-changing
circumstances. Some travelling to France or India might be included.
Excellent command of English in both writing and speaking is essential.
Knowledge of German is beneficial but not must.

Information of the research group can be obtained from Dr. Jarkko
Routtu, tel: 0049 (0) 345 55 26382, email:
jarkko.routtu(at)zoologie.uni-halle.de

<a href="http://bit.ly/1PXCY3G">http://bit.ly/1PXCY3G</a>

<a href="http://bit.ly/1SNxEG2">http://bit.ly/1SNxEG2</a>

Extended application deadline before 8th of February 2016, subject line
referring to &ldquo;PhD resistance&rdquo;, together with your letter of
motivation, CV, publication list, names of two referees in a single pdf
file.

Jarkko Routtu 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:04</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138271746921">Graduate position: MaxPlanckInst_Berlin.EvolutionHumanBehaviour</a></div>
  <div class="description">PhD position on Human Collective Behaviour. Max Planck Institute for
Human Development, Berlin.

Starting data: The position is available from May 2016 onwards, but
later start dates are possible.

Description: The PhD project will investigate the role of individual
differences in human and animal collective behaviours, including group
formation, group coordination and conflict resolution. It will involve
experiments with human crowds using GPS tracking devices & video
tracking, virtual interactive platforms and simulations. Limited work
on animal groups (e.g., fish) is also possible. We seek a PhD student
with a strong empirical background and excellent skills in (spatial)
data analysis. Programming skills are a bonus.

The Center for Adaptive Rationality at the Max Planck Institute for
Human Development (<a href="http://bit.ly/1PXCY3C">http://bit.ly/1PXCY3C</a>) is a highly
interdisciplinary and international research group where English is the
working language. We offer an excellent infrastructure including
support staff and equipment for conducting experiments (e.g.,
behavioural laboratory, GPS tracking and supercomputers).

The predoctoral contract is for three years. Applications (consisting
of a cover letter describing your research interests, a CV, up to two
publications, and two letters of recommendation) should be sent as a
single PDF file, with your name as the file name, to Monika Oppong
(oppong@mpib-berlin.mpg.de; Max Planck Institute for Human Development,
Lentzeallee 94, 14195 Berlin).

Applications will be considered until the position is filled.
Application review will begin on February 1, 2016 but applications
after this date will be considered equally. For further inquiries about
the position, please contact Ralf Kurvers (kurvers@mpib-berlin.mpg.de).

More information:

<a href="http://bit.ly/1SNxChy">http://bit.ly/1SNxChy</a>
nanzeige_predoctoral_contract-arc_39-2015_10.12.2015_intern.pdf

&ldquo;Kurvers, Ralf&rdquo; 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:33</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138271013966">Postdoc: UExeter.EvolutionInsectAntibiotic</a></div>
  <div class="description">New post-doctoral position available with Ben Raymond and Angus Buckling
at the University of Exeter Penryn campus.  This is an 18 month project
looking at how best to use phage to combat the evolution of plasmid borne
antibiotic resistance in insect/gut bacteria model systems.   More details
available at <a href="http://bit.ly/1KeGUjX">http://bit.ly/1KeGUjX</a>  or
contact b.raymond@exeter.ac.uk or A.J.Buckling@exeter.ac.uk


&ldquo;Raymond, Ben&rdquo; 
via Gmail</div>
 </div>
</div>
<h3>January 28, 2016</h3>
<div class="news-item">
 <div class="date">02:26</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138207036661">Job: DenverBotanicGardens.AssistCuratorMycology</a></div>
  <div class="description">*Job Title:* Assistant Curator of Mycology

*Job Summary:* Coordinates the growth and improvement of Denver Botanic
Gardens mycology program, advocate biodiversity science and natural
history collections, develop an active externally funded research program
that incorporates postdoctoral fellows, graduate students and
undergraduates, and oversees volunteers.

*Career Type*: Full-time

*Location*: York Street

*Qualifications*: Ph. D. in mycology, evolution, or equivalent with
interest in developing field and collection orientated research projects
related to the study and conservation of natural ecosystems within our
region and show leadership in science communication and engagement; or
minimum 6 years related experience and/or training; or equivalent
combination of education and experience. Must demonstrate a strong record
of research accomplishments and acquisition of funding. Experience with
fungal identification, specimen digitization and curation techniques and
interest in formal and informal science education, preferred. Must possess
strong communication skills, both written and spoken, needs excellent time
management skills, and good interpersonal skills.

*Benefits*: Major benefits include choice of medical plans, dental, vision
and life insurance, medical and daycare Flexible Spending Account (FSA) and
Healthcare Spending Account (HSA) ; a generous retirement plan plus paid
holidays, vacation, personal and sick time and access to personal lines of
insurance.   Additional benefits include short-term and long-term
disability coverage, an Employee Assistance Program (EAP), an ECO pass and
a complimentary Denver Botanic Gardens membership, access to Gardens
education classes and summer concert tickets.

*Additional Information: *

Review of applications will begin April 2016 and continue until the
position is filled with an anticipated start date of September 1, 2016.
Applications should include a curriculum vitae and statement of research
interests (3 pages maximum).

Send resume and/or application to *Human Resources, Denver Botanic Gardens,
1007 York St., Denver CO 80206*, or *e-mail your resume to
hr@botanicgardens.org*. We are a nonprofit, EOE.

Rebecca Hufft 
via Gmail</div>
 </div>
</div>
<h3>January 26, 2016</h3>
<div class="news-item">
 <div class="date">02:25</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138077598351">Workshop:</a></div>
  <div class="description">UCalifornia_LosAngeles.ConservationGenomics.Mar20-24
Reply-To: Mario Colon 
x-mailer: Apple Mail (2.3112)

UCLA/La Kretz Workshop in Conservation Genomics, 20-24 March, 2016

Conservation biology, genetics, and evolutionary biology have had a long and intimate 
relationship, and conservation constitutes one of the key applications of evolutionary 
analysis to real-world biological problems. The impacts of population and landscape 
genetics and gene expression studies have been particularly striking, and are helping to 
solve some of the most pressing problems in biological conservation.

As the field of conservation genetics continues to grow and mature, the availability of 
genome-scale data stand to make profound new contributions to our ability to identify and 
protect at-risk populations and recover those that are most endangered. However, 
genomic analyses also carry a heavy burdenXdata sets are enormous, often requiring 
diverse computational approaches for assembly, quality control and analysis.

The La Kretz annual workshop provides a comfortable, rigorous, but informal training 
environment for a small group of motivated graduate students to explore how conservation 
problems can best be addressed with genomic-level data. Our goal is to provide hands-on 
experience in the efficient collection, troubleshooting, and analysis of large data sets for 
conservation-relevant problems. One of the highlights of our workshop is active 
participation from members of several governmental agencies who are at the forefront of 
endangered species protection and management, providing a forum for exploring the most 
relevant aspects of conservation genomics to managers.

The UCLA/La Kretz workshop is held at the La Kretz Field Station 
(<a href="http://bit.ly/1aXROSZ">http://bit.ly/1aXROSZ</a>) and Stunt Ranch Reserve 
(<a href="http://bit.ly/1BdyJvO">http://bit.ly/1BdyJvO</a>) in the heart of the Santa Monica Mountains. Only 30 miles 
from UCLA and the LAX airport, but nestled in the relatively undeveloped 160,000 acre 
Santa Monica Mountains National Recreation Area, the Field Station/Stunt Reserve 
provides an ideal location to explore new developments in genomic science and pressing 
needs in conservation and management together in a single setting.

Our current instructor list, drawn from UCLA faculty and several other partner institutions, 
includes:

Ben Fitzpatrick (U. Tennessee Knoxville)

Kirk Lohmueller

Evan McCartney-Melstad

Gideon Bradburd (UC Berkeley)

Brad Shaffer

Victoria Sork

Bob Wayne

Ying Zhen

Plus a soon-to-be-disclosed RNA-seq teamK


Agency partners represented include:
USGS

BLM

USFWS

CDFW


Topics covered include:

Overview of traditional conservation genetics

Next generation platforms: the best tool for the job

Data management pipelines:

Quality Control

Data storage

Data organization

Data types and analyses:

SNPs

Sequences

Exploring very large data sets

Functional genomic data

RNA-seq

Genomic data and GIS

Visualizing geographic structure and demographic history

Admixture, clines, and hybridization

Detecting adaptive variation

RNA-seq and the analysis of genes that matter


Prerequisites

Available housing limits course enrollment to ~20 students. Preference is given to doctoral 
candidates who are in the early to middle stages of their thesis research, and who have 
some familiarity with using a command line interface or programming languages (i.e. Perl, 
python etc.). Postdocs, faculty, and government researchers may also apply, but 
preference will be given to graduate students. We encourage applications from women, 
minorities, and individuals from under-represented demographics in the sciences.

Admission and Fees

Applicants will be admitted based on academic qualifications and appropriateness of 
research interests. The course fee is $425. This includes food and lodging at the La Kretz 
Field Station, as well as any incidental fees, for the duration of the course (arriving Sunday 
March 20, departing Friday March 25). In addition, course participants who would like to 
extend their stay at the field station for the remainder of the weekend may do so for no 
extra charge. For those opting to stay the weekend, departure time will be by 5:00 pm on 
Sunday March 27.

This year, the workshop is co-sponsored by a UC Catalyst program grant that will provide 
some support for faculty and student support and staffing needs. University of California 
students may apply for partial support (up to $225) from the newly awarded Conservation 
Genomics Consortium ( <a href="http://bit.ly/1S8bT44">http://bit.ly/1S8bT44</a> ) funded 
under the UC Catalyst program.

UCLA students are encouraged to take the La Kretz Workshop for graduate credit. Other 
UC students may also be able to take the course for credit.


Application Forms and Information

Visit the UCLA/La Kretz Center for California Conservation Science website for additional 
information and to download the application 
(<a href="http://bit.ly/1KBzQZo">http://bit.ly/1KBzQZo</a>)


Application Deadline:

Applications are due by February 15, 2016. Please send a completed application form and 
a short letter of recommendation from your major advisor. Students will be notified via e-
mail by February 19, 2016 of acceptance.

Applications should be emailed as PDFs to:

Mario Colon, (mario.colon@ucla.edu). Please note if you are a University of California 
student or postdoc, and if so, whether you are requesting financial assistance.

Mario Colon 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:25</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/138077598106">Graduate position:ETHZurich.EcologyEvolutionModeling</a></div>
  <div class="description"><p>PhD position: Modeling ecology/evolution of a plant pathogen, ETH Zurich


***The project***

will explore population dynamics of Zymoseptoria tritici, an important
fungal pathogen of wheat, using eco-evolutionary modeling and field/lab
experimentation.

Plant diseases often cause serious yield losses in agriculture. Current
chemical and genetic technologies used to manage plant diseases are
highly vulnerable to pathogen evolution and are not sustainable.
Pathogen evolution is facilitated by the genetic uniformity underlying
modern agroecosystems. Thus, there is an urgent need to develop new,
efficient and sustainable ways to control plant diseases.

This PhD project intends to make an important contribution towards
achieving this goal.  The first phase aims at comprehensive
characterization of both epidemic and evolutionary factors of the
pathogen by combining eco-evolutionary modeling, field/lab experiments
and genome sequence analyses. In the second phase the models will be
used to predict the rate of pathogen adaptation to disease control and
propose optimal spatio-temporal patterns for applications of fungicides
and resistant host varieties that can disrupt pathogen adaptation.


***Working conditions***

Plant Pathology group at the Institute of Integrative Biology, ETH
Zurich offers a creative, collaborative research environment and
excellent infrastructure (<a href="http://www.path.ethz.ch/">www.path.ethz.ch</a>). Salary is approximately
48000 Swiss francs per year. Zurich is a vibrant, international and
green city.

The PhD project is funded for three years by the Ambizione grant from
the Swiss National Foundation with a possibility of extension for the
fourth year.

Unique training in mathematical modeling, bioinformatics and field/lab
experiments increases chances to find a job in academia, industry or
governmental agencies.


***Requirements***

via Gmail</p></div>
 </div>
</div>
<h3>January 24, 2016</h3>
<div class="news-item">
 <div class="date">01:05</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137936249766">Postdoc: UCalifornia_Berkeley.3.EvolutionAntSociality</a></div>
  <div class="description">*JPF00959*

The Department of Environmental Science, Policy, and Management at
UC-Berkeley is currently seeking one Postdoctoral Scholar in *Chemical
Ecology *in the laboratory of Dr. Neil Tsutsui. We are seeking a
Postdoctoral Scholar to study the chemical basis of ant social behaviors.
Previous work has focused on the genetics, behavior, and chemical ecology
of the invasive Argentine ant (*Linepithema humile*). The central focus of
the Postdoctoral Scholar&rsquo;s research will be the chemical communication in
social insects, including the production and perception of pheromones that
Argentine ants use to regulate fundamental behaviors, including foraging,
queen production, and colonymate recognition.


BASIC QUALIFICATIONS

Candidates must have completed all degree requirements except the
dissertation or be enrolled in an accredited PhD or equivalent degree
program in biology, chemistry, or related field at the time of application.

ADDITIONAL QUALIFICATIONS
Candidates must have a PhD, MD, or equivalent degree in biology, chemistry,
or related field by appointment start date.

PREFERRED QUALIFICATIONS
Demonstrated expertise in chemical analysis using gas chromatography-mass
spectrometry (GC-MS). Candidates fluent in chemical, genetic, and/or
biochemical analysis of insect pheromones or experienced studying the
mechanisms of chemoreception are desirable. Preferred candidates will
demonstrate excellent communication skills and the ability to work both
independently and as a member of a small team. Applicants must have fewer
than five years of prior post-doctoral experience.

APPOINTMENT
The targeted start date for this position is March 1, 2016. The initial
appointment is for one year, with renewal based on performance and funding.
This is a full-time appointment.

SALARY AND BENEFITS
The salary range is between $42,840 - $50,112 commensurate with
qualifications and experience and based on UC Berkeley salary scales.
Generous benefits are included (<a href="http://bit.ly/1gm3hSu">http://bit.ly/1gm3hSu</a>)

TO APPLY
Visit:  <a href="http://bit.ly/1ZKkM2M">http://bit.ly/1ZKkM2M</a>
Interested individuals should include a 1-2 page cover letter describing
their research experience, 1-3 relevant publications, a current CV, and the
names and contact information of three references. Letters of reference may
be requested for finalists. It is optional to include a statement
addressing past and/or potential contributions to diversity through
research, teaching, and/or service.

This position will remain open until filled.

Questions regarding this recruitment can be directed to Professor Neil
Tsutsui (ntsutsui@berkeley.edu).

All letters will be treated as confidential per University of California
policy and California state law. Please refer potential referees, including
when letters are provided via a third party (i.e. dossier service or career
center) to the UC Berkeley Statement of Confidentiality (
<a href="http://bit.ly/1wTEkon">http://bit.ly/1wTEkon</a>) prior to submitting their letters.

The University of California is an Equal Opportunity/Affirmative Action
Employer. All qualified applicants will receive consideration for
employment without regard to race, color, religion, sex, sexual
orientation, gender identity, national origin, disability, age or protected
veteran status. For the complete University of California nondiscrimination
and affirmative action policy see:
<a href="http://bit.ly/1tbbcKE">http://bit.ly/1tbbcKE</a>

The Department is interested in candidates who will contribute to diversity
and equal opportunity in higher education through their research or
teaching.

The University of California, Berkeley has an excellent benefits package as
well as a number of policies and programs in place to support employees as
they balance work and family.


JPF00958 

The Department of Environmental Science, Policy, and Management at UC
Berkeley is currently seeking two Postdoctoral Scholars in *Genomics and
Functional Genetics* in the laboratory of Dr. Neil Tsutsui. We are seeking
two Postdoctoral Scholars to study the genetic basis of ant social
behaviors. Previous work has focused on the genetics, behavior, and
chemical ecology of the invasive Argentine ant (Linepithema humile). The
central focus of the Postdoctoral Scholars research will be the genetic
basis of chemical communication, including the production and perception of
pheromones that Argentine ants use to regulate fundamental behaviors,
including foraging, queen production, and colonymate recognition.

BASIC QUALIFICATIONS
Candidates must have completed all degree requirements except the
dissertation or be enrolled in an accredited PhD or equivalent degree
program in biology, genetics, or related field at the time of application.

ADDITIONAL QUALIFICATIONS
Candidates must have a PhD, MD, or equivalent degree in biology, genetics,
or related field by appointment start date.

PREFERRED QUALIFICATIONS
Demonstrated expertise in one or more of the following areas: production
and analysis of whole- genome datasets, RNAseq, RNAi-mediated gene
silencing, and bioinformatic analysis of large genomic datasets. Candidates
fluent in chemical ecology, insect biology, population genetics, and/or
behavioral ecology are desirable. Preferred candidates will demonstrate
excellent communication skills and the ability to work both independently
and as a member of a small team. Applicants must have fewer than five years
of prior post-doctoral experience.

APPOINTMENT
The targeted start date for this position is March 1, 2016. The initial
appointment is for one year, with renewal based on performance and funding.
This s a full-time appointment.

SALARY AND BENEFITS
The salary range is between $42,840 - $50,112 commensurate with
qualifications and experience and based on UC Berkeley salary scales.
Generous benefits are included (<a href="http://bit.ly/1gm3hSu">http://bit.ly/1gm3hSu</a>)

TO APPLY
Visit: <a href="http://bit.ly/1PruLud">http://bit.ly/1PruLud</a>
Interested individuals should include a 1-2 page cover letter describing
their research experience, 1-3 relevant publications, a current CV, and the
names and contact information of three references. Letters of reference may
be requested for finalists. It is optional to include a statement
addressing past and/or potential contributions to diversity through
research, teaching, and/or service.

This position will remain open until filled.

Questions regarding this recruitment can be directed to Professor Neil
Tsutsui (ntsutsui@berkeley.edu).

All letters will be treated as confidential per University of California
policy and California state law. Please refer potential referees, including
when letters are provided via a third party (i.e. dossier service or career
center) to the UC Berkeley Statement of Confidentiality (
<a href="http://bit.ly/1wTEkon">http://bit.ly/1wTEkon</a>) prior to submitting their letters.

The University of California is an Equal Opportunity/Affirmative Action
Employer. All qualified applicants will receive consideration for
employment without regard to race, color, religion, sex, sexual
orientation, gender identity, national origin, disability, age or protected
veteran status. For the complete University of California nondiscrimination
and affirmative action policy see:
<a href="http://bit.ly/1tbbcKE">http://bit.ly/1tbbcKE</a>

The Department is interested in candidates who will contribute to diversity
and equal opportunity in higher education through their research or
teaching.

The University of California, Berkeley has an excellent benefits package as
well as a number of policies and programs in place to support employees as
they balance work and family.

via Gmail</div>
 </div>
</div>
<h3>January 22, 2016</h3>
<div class="news-item">
 <div class="date">04:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137808293191">Graduate position: Liverpool_JMU.MolecularEvolutionEcology</a></div>
  <div class="description">PhD Studentship in Molecular Evolution and Ecology

Liverpool John Moores University - School of Natural Sciences and
Psychology

Duration of Studentship:  36 months.

There is an opportunity to apply for a fully funded PhD studentship at the
interface between Molecular Evolution and Ecology. Please note that due to
funding regulations the position is available to UK or EU nationals only.

Project title: Using biogeographic and phylogenetic patterns of Conophytum
to inform the conservation of this endemic genus of the Succulent Karoo

Project description:

The aim of this project is to understand the key drivers of the current
distribution and evolutionary divergence of one of the most diverse
plant genera of the Succulent Karoo (SK). In doing so we will be better
able to project the potential responses of the Conophytum to global
environmental change, towards the development of conservation measures
that reduce the extinction risk of these vulnerable taxa.

Lying within south-western corner of Africa, the SK biome is recognised
as one of the World&rsquo;s most important regions of floral biodiversity. Here
members of the Aizoaceae are especially well adapted to the arid desert
environment, occupying niche habitats. This is most evident in the
speciose dwarf succulent genus Conophytum, a SK specialist that shows a
remarkable degree of range-restricted endemism and 20%+ of taxa can be
considered as such [1]. The SK has also been identified as being at high
risk from the effects of climate change, with preliminary data suggesting
that dwarf succulents may be especially vulnerable [2-4]. However, neither
the underlying mechanisms that influence the distribution and associated
speciation of these succulents nor how they might respond to specific
environmental conditions is understood. We now have an opportunity to
address this by bringing together a range of complimentary techniques
using the genus Conophytum as a model.

The project will take a unique, multidisciplinary approach, combining
the expertise of the three supervisors (spatial modelling, molecular
phylogeny and evolution, biochemistry and succulent plant biology). In
addition, this study is only now possible because of the availability of
two unique resources at LJMU: (1) the student will have full access to
the most comprehensive database of ~3000 locality records covering all
taxa in the genus and (2) a genetic resource arising from a complete in
situ collection of all Conophytum taxa in the genus.

A three step approach to the work will be employed: (1) The existing
spatial database and an ensemble of Species Distribution Modelling
(SDMs) techniques (regression and machine learning models) available in
the BIOMOD2 library [5,6], run and calibrated within the R statistical
programming environment [7], will be used to quantify the role of climate
and other environmental variables on the current distribution of the 166
taxa in the Conophytum genus. Of particular interest will be a comparison
of the climatic niches of (a) 16 defined closely-related groups of taxa
of varying vulnerability to extinction within the genus [8,1] and (b)
point endemics (small vulnerable populations) versus those taxa with a
ubiquitous distribution. (Months 1-12)   (2) The Conophytum phylogeny
will be reconstructed using suitable genetic markers developed based on
Next Generation Sequencing from isolated DNA available from the in situ
collection at LJMU. This combined with the biogeographic and spatial
data will allow an assessment of the phylogeny in the context of the
climatic and environmental drivers. In addition, a methodology typically
applied to island endemic plants (see previous study by Kapralov et al.,
2013 in Hawaii [9]), will be applied to the island-like populations of
Conophytum on isolated inselbergs to understand the role of adaptation
during radiation. The Conophytum phylogeny will therefore act as the
foundation for further studies and will be used to map on it species
ecology, morphology, distribution, pigment composition, physiology,
and anatomy. This would allow us to better understand mechanisms
behind the evolution of the genus and to compare roles of adaptation
versus stochastic isolation for its prolific speciation. (Months 10-30)
(3) Future threats from climate change will be assessed using the SDMs
outlined in Step 1 incorporating a range of climate change scenarios to
project the impacts on the future distribution of the Conophytum taxa
[10-12]. Two time slices (2021 to 2050 and 2050 to 2080) will be used
with a range of Regional Climate Models (RCM) (CCSM3, HadCM3, ECHAM5)
and Special Report on Emissions Scenarios (SRES) (B1, A2, A1b) to provide
12 different climate change datasets to account for the RCM and SRES
uncertainty in the modelling. Gap analysis will be used to assess the
effectiveness of the current protected area network under future climates
[10]. (Months 24-36 - thesis submission)

The student will collaborate with a range of international researchers,
reinforcing research networks of the supervisory team: a word leading
biodiversity and climate change modelling research group at Universit
Joseph Fourier, Grenoble, France and 3 recognised global centres of
excellence in South Africa, Stellenbosch University, the University
of Cape Town (succulent phylogeny) and the South African National
Biodiversity Institute (SANBI).

Results from this study will better inform both the National Red List and
conservation planning in South Africa via existing links with SANBI (work
currently being developed as part of an Impact Case Study for REF2020).

The student will acquire a range of state-of-the-art techniques
including spatial modelling, molecular phylogenetics, and strong
statistical/computational skills.  References:

1.    Hammer, S.A. & Young, A.J. (2015). Conophytum: Ruschioideae,
      In: Hartman, HE.K. (Ed.), Illustrated Handbook of Succulent
      Plants, Aizoaceae A-E. Springer-Verlag. In Press

2.    Midgley, G.F. & Thuiller, W. (2007) Potential vulnerability of
      Namaqualand plant diversity to anthropogenic climate change.
      Journal of Arid Environments, 70, 615-628.

3.    Midgley, G.F., Hannah, L., Millar, D., Rutherford, M.C. &
      Powrie, L.W. (2002). Assessing the vulnerability of species
      richness to anthropogenic climate change in a biodiversity
      hotspot. Global Ecology and Biogeography, 11, 445-451.

4.    Young, A.J., Guo, D., Desmet, P.G., & Midgley, G.F. (in
      review). Biodiversity and climate change: risks to dwarf
      succulents in Southern Africa.

5.    Thuiller, W., Lafourcade, B., Engler, R., Arajo, M.B.,
      (2009). BIOMOD - a platform for ensemble forecasting of species
      distributions. Ecography, 32, 369-373.

6.    <a href="http://bit.ly/1OBH1oi">http://bit.ly/1OBH1oi</a>

7.    R Development Core Team, (2015). R, A language and environment
      for statistical computing. R Foundation for Statistical
      Computing, Vienna, Austria. URL <a href="http://bit.ly/1ZRgSdA.">http://bit.ly/1ZRgSdA.</a>

8.    Hammer, S.A. et al. (2002). Dumpling and his wife; new views of
      the genus Conophytum. EAE Creative Colour Limited.

9.    Kapralov, M.V., Votintseva, A.A., & Filatov, D.A. (2013).
      Molecular adaptation during a rapid adaptive radiation.
      Molecular Biology and Evolution, 30(5), 1051-59.

10.   Hodd, R.L., Bourke, D., & Sheehy Skeffington, M. (2014). Projected
      Range Contractions of European Protected Oceanic Montane Plant
      Communities: Focus on Climate Change Impacts Is Essential for
      Their Future Conservation. PLoS ONE 9(4): e95147.
      doi:10.1371/journal.pone.0095147

11.   Coll, J., Bourke, D., Sheehy Skeffington, M., Gormally, M., and
      Sweeney, J. (2014). Projected loss of active blanket bogs in
      Ireland. Climate Research, 59,103-115.

12.   Sharkey, N., Jones, M. and Bourke, D. (2013). Climate change
      impacts on woodland species: implications for the conservation of
      woodland habitats in Ireland, Biology and Environment: Proceedings
      of the Royal Irish Academy, 113B (3), 1-31. DOI:
      10.3318/BIOE.2013.20

Any specific eligibility requirements:

Please note that due to funding regulations the position is available
to UK or EU nationals only.

Candidates must hold a 1st Class Honours Degree and/or an appropriate
Master&rsquo;s degree.

See more details at <a href="http://bit.ly/1OBH4Ay">http://bit.ly/1OBH4Ay</a>

Deadline: 18th March 2016

In their application cover letter and CV, candidates are asked to include
and highlight their skills, experience and/or knowledge relevant to this
project; in particular:

Plant Ecology & Evolution

Species Distribution Modelling/Ecological Statistics

Geographic Information Systems

Phylogenetics/Molecular Ecology

Name/email/telephone number of someone applicants can contact with
enquiries:

In the first instance make an informal enquiry by e-mail to project
supervisors by sending a CV with contact details of two academic
referees. Your e-mail should have as its subject &ldquo;Application for
PhD&rdquo;. It is essential that you give the grades of all degrees and state
your nationality.

Dr Maxim Kapralov email: m.v.kapralov@ljmu.ac.uk

Dr David Bourke email: d.o.bourke@ljmu.ac.uk

Professor Andy Young email: a.j.young@ljmu.ac.uk

Dr Maxim Kapralov

Senior Lecturer || School of Natural Sciences and Psychology ||
Liverpool John Moores University || Room 2.07 ||  Life Sciences
Building || Byrom Street || Liverpool || L3 3AF || United Kingdom
|| t: +44 (0)151 904 6313 || email: m.v.kapralov@ljmu.ac.uk ||
<a href="http://bit.ly/1Pbezum">http://bit.ly/1Pbezum</a>

&ldquo;Kapralov, Maxim&rdquo; 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137808292856">Postdoc: UNewHampshire.SoilMicrobialEvolution</a></div>
  <div class="description">Postdoctoral Position in Soil Microbial Evolution

Contact: Dr. Serita Frey, Department of Natural Resources & the
Environment, University of New Hampshire, Durham, NH USA

(Questions can also be directed to Dr. Anne Pringle, University of
Wisconsin-Madison: anne.pringle@wisc.edu)

Our laboratory aims to understand connections between microbial
community structure and ecosystem function. We document the impacts of
environmental change (climate warming, nitrogen deposition,
biodiversity loss, invasive species) on the diversity, community
composition, and function of the soil microbial community, and test
whether shifts in the community subsequently influence ecosystem
functions. A recent focus is on anthropogenic drivers of microbial
evolution.

This two-year position will focus specifically on fungal evolution
within global change contexts, with an emphasis on how fungi evolve in
response to biotic invasions, soil warming, and nitrogen addition. The
candidate will have the flexibility to explore questions that fall
within this general topic area, while building on previous research
conducted in the Frey and Pringle Laboratories. The candidate is
expected to have strong interests and experience in evolution and
ecology. Expertise in cultivation-based and genomic/metagenomic
analyses is highly desirable. The candidate will be expected to work
independently, but also cooperatively with other members of the lab and
with the Pringle Laboratory at the University of Wisconsin, Madison. A
Ph.D. degree in evolution, ecology, natural resources, microbiology, or
related field, along with relevant research experience is required.
Applications will be reviewed until the position is filled.

To apply please send the following items in a single PDF file to Serita
Frey (serita.frey@unh.edu): letter of interest/experience, CV, and the
names and contact information of three professional references.

Anne Pringle, Ph.D.
Associate Professor
Departments of Botany and Bacteriology
University of Wisconsin, Madison
new email:  anne.pringle@wisc.edu
<a href="http://bit.ly/1ZRgQ5v">http://bit.ly/1ZRgQ5v</a>

Anne Pringle 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:30</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137807897071">Other: Botany Advocacy Leadership Grant</a></div>
  <div class="description"><p>The American Society of Plant Taxonomists (ASPT) and Botanical Society of
America (BSA) have teamed up to craft a NEW grant opportunity - the Botany
Advocacy Leadership Grant!

Please find the call for proposals at the following link:
<a href="http://bit.ly/1S96Iz3">http://bit.ly/1S96Iz3</a>

One successful applicant for this award will receive $1,000 toward
developing new (or enhancing existing!) botanical advocacy projects - apply
or encourage others you know to submit an application.

DEADLINE: 16 March 2015


gostelm@gmail.com


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:15</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137807513966">Postdoc: UNevada_Reno.CognitiveFitness</a></div>
  <div class="description">POSTDOC POSITION IN COGNITIVE ECOLOGY

I am looking for a postdoc to join my lab at the University
of Nevada, Reno (<a href="http://bit.ly/1RYnnHe;">http://bit.ly/1RYnnHe;</a>
<a href="http://on.fb.me/1ZGBjop">http://on.fb.me/1ZGBjop</a>) to work on an NSF
funded project investigating fitness consequences of individual variation
in spatial memory. The project uses RFID technology to investigate whether
individual variation in spatial learning abilities is associated with
variation in fitness in free-living food-caching mountain chickadees in
two environments (different elevations) hypothesized to be different in
spatial memory demands associated with food caching.

All fieldwork is in the rugged mountain conditions V in the beautiful
Sierra Nevada along an elevation gradient from ca. 5,000 to 8,000 feet in
the Lake Tahoe National Forest (about 40 min drive from Reno). The postdoc
will be expected to work in the field throughout the year using a 4-wheel
drive vehicle when there is no snow and a snowmobile during the winter.

The project generates large amount of data (more than 100,000 bird visits
at just one spatial array with 8 feeders, per two weeks of testing),
so experience with analyzing and manipulating large data sets using
R and potentially other programming languages (e.g. Perl) is highly
desirable. Experience working with birds and with mist-netting is also
helpful.

The initial appointment will be for 1 year, which may be renewed based
on performance up to 3 years. If interested, please email directly to me
(Vladimir Pravosudov) at vpravosu@unr.edu.



Vladimir V Pravosudov 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:15</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/137807514291">Job: CRITFC_Idaho.FishGenetics</a></div>
  <div class="description">Fisheries Geneticist - Columbia River Inter-Tribal Fish Commission
(CRITFC)

<a href="http://bit.ly/1ZGBjEF">http://bit.ly/1ZGBjEF</a>

Salary: $55,952-$72,737 DOQ, plus benefits

Closes on: March 15, 2016

The Columbia River Inter-Tribal Fish Commission (CRITFC) is seeking
Fisheries Geneticists with experience in population genomics,
association mapping, and mixed stock analysis. Positions are part of
the Fishery Science Department, but will be located with the genetics
group at the Hagerman Fish Culture Experiment Station in Hagerman, ID.
This research group is involved in testing conservation, evolution, and
ecological theories related to salmonids and other fishes. The employee
will work under the Lead Geneticist, in association with CRITFC
geneticists and technicians, as well as staff of the Fishery Science
Department in Portland, OR. Efforts will focus on applying empirical
genetics/genomics data to address questions related to conservation and
recovery of steelhead, Chinook, sockeye, and coho salmon, sturgeon,
lamprey, and other fishes of the Columbia River Basin.

See more at: <a href="http://bit.ly/1ZGBjEF">http://bit.ly/1ZGBjEF</a>


Shawn Narum 
via Gmail</div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="index8713.html?q=aggregator/sources/14&amp;from=0">first page</a></div><div><a href="index2946.html?q=aggregator/sources/14&amp;from=100">previous page</a></div><div>... <a href="index9ebc.html?q=aggregator/sources/14&amp;from=40">3</a> <a href="indexab35.html?q=aggregator/sources/14&amp;from=60">4</a> <a href="indexac35.html?q=aggregator/sources/14&amp;from=80">5</a> <a href="index2946.html?q=aggregator/sources/14&amp;from=100">6</a> <strong>7</strong> <a href="index2be7.html?q=aggregator/sources/14&amp;from=140">8</a> <a href="index5104.html?q=aggregator/sources/14&amp;from=160">9</a> <a href="index2b69.html?q=aggregator/sources/14&amp;from=180">10</a> <a href="index2379.html?q=aggregator/sources/14&amp;from=200">11</a> ...</div><div><a href="index2be7.html?q=aggregator/sources/14&amp;from=140">next page</a></div><div><a href="index9a9b.html?q=aggregator/sources/14&amp;from=280">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news." class="active">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/14&from=120 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:29:07 GMT -->
</html>
