<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=300 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:26:31 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="index66e2.html?q=node/119">54(6) December 2005</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 2 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>January 8, 2016</h3>
<div class="news-item">
 <div class="date">00:28</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136867792381">Workshop: UWashington.SummerInstituteGenetics.RegistratonsOpen</a></div>
  <div class="description">Registrations for the 2016 Summer Institute in Statistical Genetics are 
now being accepted. Scholarship applications are processed as they are 
received.

Information is available at:



Bruce Weir
Department of Biostatistics
University of Washington
Box 359461
Seattle, WA 98195-9461
Phone: (206) 221-7947

Bruce Weir 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>January 7, 2016</h3>
<div class="news-item">
 <div class="date">23:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136866503111">Other: Evolution 2016 and art</a></div>
  <div class="description"><p>Attention artists! The deadline for submission of evolution-themed works
(or proposed-works) for the &lsquo;Evolution 2016 Art Exhibit&rsquo; is quickly
approaching (Jan. 18).

The 2016 Evolution conference (the annual meeting of theSociety for
the Study of Evolution, theSociety of Systematic Biologists, and
theAmerican Society of Naturalists) will be held from June 17-21 in
beautiful Austin, TX. In conjunction with the meeting, the societies
and a local art gallery (Art.Science.Gallery) are co-sponsoring an
evolution-themed art exhibit. The exhibit will provide an opportunity for
meeting attendees and others to showcase their creative talents, increase
the visibility of the meeting and the societies to the local community,
and provide opportunities for public education about evolutionary
biology through the exciting lens of visual arts. The exhibit will run
at the Art.Science.Gallery leading up to, and during, the conference,
and a selection of the works will be on display at the Austin Convention
Center as a pop-up exhibit during the evening of the opening reception
of the meeting.

Art.Science.Gallery has issued an open call seeking submissions for this
exhibition. Works may explore (but are not limited to) the sub-disciplines
within evolutionary biology, notable evolutionary biologists, current
research topics, important discoveries and concepts, and the history of
evolutionary thought. You need not be a member of any of the societies
to submit something for consideration.

Pass the word along and, if you&rsquo;re at all artistically inclined, consider
submitting something. Additional details can be found on our permanent
meeting website (<a href="http://bit.ly/1PNgrcn">http://bit.ly/1PNgrcn</a>) under the 'News&rsquo; heading, or
on Art.Science.Gallery&rsquo;s website (<a href="http://bit.ly/1SDywfn">http://bit.ly/1SDywfn</a>).

The open call closes on Jan. 18.

Sincerely,
Your Evolution2016 organizers

howard.rundle@uottawa.ca


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">18:21</div>
 <div class="body">
  <div class="title"><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12149">Phylogenetic analysis of RAD-seq data: examining the influence of gene genealogy conflict on analysis of concatenated data</a></div>
  <div class="description">One of the major issues in phylogenetic analysis is that gene genealogies from different gene regions may not reflect the true species tree or history of speciation. This has led to considerable debate about whether concatenation of loci is the best approach for phylogenetic analysis. The application of Next-generation sequencing techniques such as RAD-seq generates thousands of relatively short sequence reads from across the genomes of the sampled taxa. These data sets are typically concatenated for phylogenetic analysis leading to data sets that contain millions of base pairs per taxon. The influence of gene region conflict among so many loci in determining the phylogenetic relationships among taxa is unclear. We simulated RAD-seq data by sampling 100 and 500 base pairs from alignments of over 6000 coding regions that each produce one of three highly supported alternative phylogenies of seven species of Drosophila. We conducted phylogenetic analyses on different sets of these regions to vary the sampling of loci with alternative gene trees to examine the effect on detecting the species tree. Irrespective of sequence length sampled per region and which subset of regions was used, phylogenetic analyses of the concatenated data always recovered the species tree. The results suggest that concatenated alignments of Next-generation data that consist of many short sequences are robust to gene tree/species tree conflict when the goal is to determine the phylogenetic relationships among taxa.</div>
  <div class="source">Source: <a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:41</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136806587701">Other: Passing of Ted J. Case</a></div>
  <div class="description">It is with great sadness we inform his friends and colleagues that Prof.
Ted J. Case passed away on Thursday, December 31, 2015, at the age of 68.
Ted was a renowned island biogeographer, community ecologist, and
herpetologist.  He retired in 2004 after more than 25 years of service in
the Biology Department at UC San Diego.  David Quammen captured the essence
of Ted exceptionally well in his book $B!H(BThe Song of the Dodo$B!I(B.   A further
obituary will follow.


Robert Fisher
U.S. Geological Survey
San Diego Field Station
4165 Spruance Road, Suite 200
San Diego, CA 92101-0812

Phone: 619-225-6422
Phone (Admin):  619-225-6451
Fax: 619-225-6436

rfisher@usgs.gov


&ldquo;Fisher, Robert&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136805477651">Conference: UMichigan.CommunityEvolution.Mar12</a></div>
  <div class="description"><p>CALL FOR NOMINATIONS: 12th ANNUAL EARLY CAREER SCIENTISTS SYMPOSIUM

The Department of Ecology and Evolutionary Biology at the University of
Michigan invites nominations of outstanding scientists early in their
careers to participate in an exciting international symposium entitled
&ldquo;Frontiers in Community Assembly.&rdquo; The symposium events will take place on
12 March 2016, in Ann Arbor, Michigan. Dr. Tadashi Fukami and Dr. Rosemary
Gillespie will be the keynote speakers.

Potential topics include phylogenetic and ecological models for inferring
community structure, speciation and the evolution of secondary sympatry,
genomic processes underlying colonization and local adaptation, the
structuring of communities across environmental gradients, inferences from
paleo-communities.

We welcome nominations of early career scientists who are studying
ecological and evolutionary approaches to community assembly. Early career
scientists are considered senior graduate students (who stand to receive
their Ph.D. within one year) or postdoctoral researchers.  A colleague or
advisor may provide the nomination, however self nominations will be
accepted. The selected nominees will be invited to present their work and
to participate in panel discussions at the symposium.

The nomination consists of the nominees curriculum vitae, a brief abstract
of the proposed presentation (


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:41</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136805090251">Graduate position: University of Cologne, Adaptation in Arabidopsis</a></div>
  <div class="description"><p>1 PhD position
on the genomic basis of Adaptive plasticity in Arabidopsis lyrata
Lab. J. de Meaux

With a joint analysis of transcriptomic and phenotypic data, you will 
investigate patterns of parallel evolution in A. lyrata and its close 
relative A. thaliana. You will quantify regional differences in various 
plastic responses including response to temperature modifications or 
changing light quality in A. lyrata and compare this response to the 
related species A. thaliana. You will use a crossing scheme derived from 
classical quantitative genetics methods to investigate the distribution 
of various sources of genetic variation controlling gene regulation. 
Your project will shed light on the molecular basis of adaptive plastic 
responses in A. lyrata, an outcrossing relative of the model inbred 
species A. thaliana. With this project, you will develop skills ranging 
from genomics to molecular genetics and ecology.

The applicant must hold a Master degree in Evolutionary Biology, 
Genetics, Genomics or Bioinformatics and prove interest in plant 
molecular, population or ecological genetics. Experience in statistical 
analysis of quantitative data is required. This position is open to 
applicants of all nationalities but the language in the lab is English. 
Applications or questions regarding the position should be sent by mail 
to jdemeaux@uni-koeln.de, with the following subject line – PhD 
application adaptation lyrata – de Meaux lab. A letter of motivation, a 
CV and the contact to at least 2 independent referees should be 
provided, all in a single pdf file.  Revision of applications will begin 
in Jan. 20th, 2016 and continue until the position is filled. Funding is 
for 3-4 years with flexible starting date. For more information on our 
lab and research visit our website 
<a href="http://bit.ly/1Azj0bW">http://bit.ly/1Azj0bW</a> Interested students currently 
completing their Master thesis are encouraged to informally contact the 
PI if they have questions concerning the position.


Cologne is Germany’s vibrant Metropolis on the Rhine. The city is well 
known for its wild carnival, its famous Kölsch beer, its Cathedral and 
its vivid contemporary art and musical scene. Cologne is the fourth 
biggest city in Germany with over a million inhabitants from all over 
the world and an interesting mix of restored historic buildings and 
modern post-war architecture. Most importantly, Cologne University is 
one of the oldest and largest Universities in the Country. Our research 
group is hosted at the Biological Center of the University of Cologne 
and associated to the Collaborative Research network SFB680 
(<a href="http://bit.ly/1ezeCwf">http://bit.ly/1ezeCwf</a>) and to the Excellence Research Cluster 
CEPLAS (<a href="http://ceplas.eu/de/">http://ceplas.eu/de/</a>), which fosters active interactions between 
plant scientists of the Universities of Cologne, Düsseldorf and the Max 
Planck Institute of Plant Breeding Research. In this context, our PhD 
students are assured to start their scientific career in a world-class 
scientific environment.



via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:26</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136804703456">Postdoc: OregonStateU.SalmonEvolution</a></div>
  <div class="description">Post-Doctoral Research Associate

Oregon State University/Northwest Fisheries Science Center (NOAA
Fisheries)

Responsibilities: We are seeking a post-doctoral associate to
participate in our studies of the mechanisms and evolution of dispersal
(i.e. olfactory imprinting, homing, and straying) in Pacific salmon.
The post-doctoral associate will join a collaborative team of University,
Federal, and State researchers working to develop tools for improving
homing fidelity to salmon hatcheries. Specifically, the successful
candidate will develop and conduct studies to (1) identify olfactory cues
that may improve homing fidelity and, (2) identify critical periods for
olfactory learning of homing cues by salmon using an interdisciplinary
approach including behavioral experiments, electrophysiology, and
cellular/molecular biology.

Qualifications:

Ph.D. in Zoology, Biology, Neuroscience, Fisheries Science, or a closely
related field. Demonstrated research accomplishments and publications in
the primary research literature. Knowledge of fish physiology, behavior,
and ecology; a background in neurobiology and experience with molecular
and/or electrophysiology techniques would be helpful. An ability to work
independently and as part of multi-faceted research team is critical.

Salary: Competitive salary + Benefits

Start date: ASAP

Contact/Email:

Dr. David L. G. Noakes

Fisheries & Wildlife Department

Oregon Hatchery Research Center

104 Nash Hall, Oregon State University

Corvallis, Oregon

97331-3803 USA

david.noakes@oregonstate.edu

telephone: (01) (541) 737-1953  (OSU office)

                   (01) (541) 487-5513  (OHRC office)

Dr. Andrew Dittman

Environmental Physiology Program

Northwest Fisheries Science Center, NOAA Fisheries

2725 Montlake Boulevard East

Seattle, WA 98112 USA

andy.dittman@noaa.gov

telephone: (01) (206) 860-3392

&ldquo;Johnson, Marc&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:11</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136804287916">Postdoc: GeorgiaTech.MarineMicrobiology</a></div>
  <div class="description"><p>Postdoc in Marine Microbiology and Biogeochemistry 

Georgia Tech is looking for a postdoctoral fellow to study the diversity and physiology of marine microbes coupling anaerobic methane oxidation to denitrification. The postdoc will be based in the lab of Dr. Frank Stewart in the School of Biology, but will work jointly between the Stewart lab and the lab of Dr. Jen Glass (Earth and Atmospheric Sciences) on a collaborative NSF-funded project.  The work will involve oceanographic cruises (at least one per year) to oxygen minimum zone (OMZ) sites in the Pacific and Gulf of Mexico, shipboard and laboratory experiments, and a blend of biochemical, molecular, and bioinformatic analyses. Research tasks will involve both culture-dependent and independent methods, with the former focused on enrichments of target microbes from OMZs and the latter on comparative analysis of genomic and meta-omic datasets.  The postdoc will be encouraged to develop independent lines of research within the broader goals of the project, and will work colla
 boratively with Dr. Stewart, Dr. Glass, international collaborators, and graduate students to perform research and synthesize results for publication.  

The ideal candidate will be enthusiastic, motivated by experimental and analytical challenges, and proficient in biogeochemical and molecular microbiology techniques, ideally with experience in genomic or meta-omic analyses.  Candidates should have a Ph.D. focusing on environmental microbiology or bioinformatics (or a related topic).  Direct experience with anaerobic methane or nitrogen cycling or marine microbiology is highly desirable.  

The School of Biology at Georgia Tech is a dynamic research environment with a strong core of researchers interested in marine systems, microbiology, and genomics.  The Institute offers exceptional resources for biomolecular analysis, bioinformatics and high-performance computing, and exciting opportunities for cross-departmental collaboration with earth and atmospheric and computational scientists.  Georgia Tech was recently voted one of the best places to work, and Atlanta is consistently ranked among the top ten places to live for young professionals. 

This position begins in spring/summer 2016.  Funding is available for at least one year, with continuation contingent upon satisfactory progress in year one; applicants should express their ability to commit to the project for at least two years.  Application materials should be emailed to Frank Stewart at frank.stewart@biology.gatech.edu and should include a cover letter (describing your interest in the position, work experience, and availability), CV, and contact information (name, email, phone number) for at least three references.  Please include the word &ldquo;Postdoc&rdquo; in the subject line.  Salary will be competitive and commensurate with experience and will include fringe benefits.  Review of applications will begin on February 1 and continue until the position is filled.

Informal inquiries about the position can be sent to Frank Stewart at frank.stewart@biology.gatech.edu.  Additional details about the Stewart lab can be found at <a href="http://b.gatech.edu/1NG3uxB">http://b.gatech.edu/1NG3uxB</a> and the Glass lab at <a href="http://bit.ly/1SBJPER">http://bit.ly/1SBJPER</a>

Georgia Tech is a unit of the University System of Georgia and an Affirmative Action/Equal Opportunity Employer and requires compliance with the Immigration Control Reform Act of 1986.





via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:26</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136802991976">Graduate position:EdinburghNapierU.ModellingSocialEvolution</a></div>
  <div class="description"><p>PhD Studentship in modelling the evolutionary origin of social groups

School of Computing, Edinburgh Napier University, Edinburgh, UK

Dr Simon T. Powers

Deadline: 14th January 2016

The School of Computing at Edinburgh Napier University is advertising fully-funded PhD studentships, open to both UK and international students, to start in the first or second quarter of 2016. One of the proposed studentships is to work with Dr Simon T. Powers on modelling the evolutionary origins of social groups. Specific projects include the evolution of protocells at the origin of life, the evolution of multicellularity, and the evolution of human social institutions during the Neolithic transition from hunter-gatherer groups to agricultural societies. Our main approach is to investigate the selective drivers towards social group formation by modelling the co-evolution of traits that affect group formation, such as dispersal or group size preferences, with social behaviours such as cooperation and punishment. For human evolution, we are interested in how cultural evolution can lead to the creation of social institutions that select for cooperation in large groups of unre
 lated individuals. Our main methodology is analytical and simulation modelling. However, we are also interested in projects proposing the use of evolutionary robotics as a modelling tool, and projects that propose applications to cooperation in agent-based computing.

Related reading can be found in Dr Powers&rsquo; publications: <a href="http://bit.ly/1TGql0p">http://bit.ly/1TGql0p</a>

<a href="http://bit.ly/1mEczRr">http://bit.ly/1mEczRr</a>

Candidates should have strong quantitative skills, demonstrated experience of computer programming, and an interest in the evolution of social behaviour.

The formal advert for the studentships can be found at: <a href="http://bit.ly/1TGqn8o">http://bit.ly/1TGqn8o</a>

Students are encouraged to propose their own projects that broadly fit within the areas described above. Potential applicants are encouraged to informally discuss the position by email with Dr Powers: S.Powers@napier.ac.uk.

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136802000746">Postdoc: UKentucky.EvolutionaryGenomics</a></div>
  <div class="description"><p>The Linnen Lab in the Biology Department at the University of Kentucky is
seeking a full-time postdoctoral researcher to participate in a USDA-funded
project that uses comparative genomics to investigate the genetic basis of
ecological specialization in plant-feeding insects. The ultimate goal of
the project is to test the hypothesis that changes in host use involve
predictable genetic changes. To test this hypothesis, we are combining
chemosensory gene annotations and host-use data from an experimentally
tractable and well-studied group of pine-feeding insect pests in the genus
*Neodiprion* (Hymenoptera: Diprionidae). This project dovetails with
ongoing work in the lab that seeks to understand both the causes (molecular
mechanisms) and consequences (population divergence and speciation) of
phenotypic variation. For additional details about current research in the
lab, please visit our website: <a href="http://bit.ly/1OcrtoF.">http://bit.ly/1OcrtoF.</a>

The ideal candidate will be independent, organized, and motivated and have
a keen interest in using genomic/transcriptomic data to address fundamental
questions in evolutionary biology. Candidates should have a Ph.D. in
evolutionary biology, genetics/genomics, or a related field.
Interest/experience in developing functional approaches for non-model
organisms is also desirable. Importantly, we are looking for someone who is
personable and enthusiastic about working in a collaborative environment.

The preferred start date is Spring or Summer 2016 (negotiable) and funding
is available for 2-3 years. Salary is commensurate with experience and
includes benefits.

Interested candidates should submit a cover letter describing research
interests/experience and a full CV with contact information for three
references. Application materials and all queries should be sent to:
catherine.linnen@uky.edu. Review of applications will begin immediately and
will continue until the position is filled.

The University of Kentucky is an equal opportunity/affirmative action
employer. Women, persons with disabilities, and members of other
underrepresented groups are encouraged to apply.

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:41</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136801472801">Course: AveiroU_Portugal.AmphibianConservation.May16-20</a></div>
  <div class="description">I am contacting asking for the divulgation of the Amphibian Husbandry
and Conservation Course at the Biology Department of Aveiro University,
Portugal. We invited, Amphibian Ark to give us the pleasure of
their partnership.

The 1st Course on Amphibian  Husbandry and Conservation (AHC) is
an interesting and intensive course to researchers, technical staff
working with amphibians or anyone looking for more complete basis on
Amphibian Husbandry and Conservation. The course will be held from 16th
to 20th May 2016.

Registrations for this edition already available

<a href="http://bit.ly/1Uykcn8">http://bit.ly/1Uykcn8</a>

The course explores the principles of amphibians&rsquo;
husbandry, nutrition and dietary needs, captive
reproduction, population management, veterinary aspects (diseases,
pathology, and necropsy), biosecurity and quarantine, conservation,
threats and global action. Hands on demonstrations, practical and group
exercises are also included (enclosure demonstrations: tank drilling,
false bottoms and plumbing, filters).

Speakers:

  * Luis Carrillo, University of Mexico City, AArk
  * Michael Bungard, Wjitley Wildlife Conservation Trust (Paignton Zoo)
  * Arturo Munoz Saraiva, Coordinator of the Bolivian amphibian
    initiative and PhD in Ghent University
  * Isabel Lopes, University of Aveiro, CESAM

     Important note: the course will not start if a minimum of 10
   participants will not be reached; similarly the maximum number of
                          participants is 15.

Inscription fees include: attending at the course and the use of all
the materials provided by University of Aveiro and the AArk.

Fees:

AHC Course Early registration (29th February)

General: 280 euros

Student: 180 euros´

AHC Course

General: 350 euros

Student: 250 euros

Applications should be sent to: amphibianhc@sapo.pt till the 31st of
March 2016.

Instructions for payment will be notified to participants no later than
48h after. The payment proof must be sent to us for the registration to
be valid.

I grateful for your help.

The organizing committe

&ldquo;amphibianhc@sapo.pt&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:26</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136800917806">Graduate position: UNorthDakota.BisonGeneticsEvolution</a></div>
  <div class="description">U. North Dakota. AncientModernBisonGenetics.
The Laboratory of Human and Forensic Genetics at the University of North
Dakota (www.und.edu) is inviting applications from potential graduate
students who pursue the MS degree. PhD candidates will be also considered.

A student will be engaged to a project on the historic, current and future
status of bison herds from biological, archaeological, and cultural
perspectives. It is a collaborative effort with Theodore Roosevelt
National Park (THRO). This cross-disciplinary project represents an
opportunity to get intensive training in the methods of ancient and
modern DNA analyses including DNA isolation from a variety of tissues,
next-generation and traditional sequencing, quality control of DNA
templates, and bioinformatics. Genetic, phylogenetic, and demographic
studies of extirpated and extant bison specimens will be used to generate
important insights for management of bison at THRO and other locations
throughout the Midwest.

Candidates should demonstrate motivation for hard laboratory work
and strong interest in molecular genetics and evolution. Preference
will be given to students with a proven record of molecular biology
skills. Additional experience in bioinformatics is a plus.  If you are
interested you need to apply to the University of North Dakota Biology
Graduate Program using the regular procedure that can be found in the
UND Graduate School website:

<a href="http://bit.ly/1KW1wvu">http://bit.ly/1KW1wvu</a>

The additional information can be also found in the Biology Department
website:

<a href="http://bit.ly/1G8L7yc">http://bit.ly/1G8L7yc</a><a href="http://www.und.edu/">www.und.edu</a>) is inviting applications from potential graduate
students who pursue the MS degree. PhD candidates will be also considered.

A student will be engaged to a project on the historic, current and future
status of bison herds from biological, archaeological, and cultural
perspectives. It is a collaborative effort with Theodore Roosevelt
National Park (THRO). This cross-disciplinary project represents an
opportunity to get intensive training in the methods of ancient and
modern DNA analyses including DNA isolation from a variety of tissues,
next-generation and traditional sequencing, quality control of DNA
templates, and bioinformatics. Genetic, phylogenetic, and demographic
studies of extirpated and extant bison specimens will be used to generate
important insights for management of bison at THRO and other locations
throughout the Midwest.

Candidates should demonstrate motivation for hard laboratory work
and strong interest in molecular genetics and evolution. Preference
will be given to students with a proven record of molecular biology
skills. Additional experience in bioinformatics is a plus.  If you are
interested you need to apply to the University of North Dakota Biology
Graduate Program using the regular procedure that can be found in the
UND Graduate School website:

http://bit.ly/1KW1wvu

The additional information can be also found in the Biology Department
website:

http://bit.ly/1G8L7yc


The position starts in August /September 2016. To receive full
consideration, applications and required materials should be received
by the Biology Graduate Program not later than February 15, 2016.

Potential graduate students are strongly encouraged to make contact with
Dr. Igor Ovchinnikov. Contact information:

Dr. Igor Ovchinnikov
Associate Professor
Lab. Of Human and Forensic Genetics
Department of Biology
University of North Dakota
Email: igor.ovtchinnikov@email.und.edu


&ldquo;Ovtchinnikov, Igor&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>January 6, 2016</h3>
<div class="news-item">
 <div class="date">23:41</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136799037546">Graduate position: Versailles_France.InsectBehavioralEvolution</a></div>
  <div class="description">Master Student research fellowship for 6 months on Moths&rsquo; Pheromone
differentiation at the Institute for Agricultural Research (INRA),
Versailles, France

Our general interests are insects&rsquo; behavioral evolution and
functional genomics. The aim of the project is to find the genetic
basis of the differentiation of Ostrinia nubilalis moth species in two
pheromone strains. This will be done by using in situ hybridization and
the latest methods in transgenesis i.e. CRISPR/Cas9 genome editing
system. Thus the project represents a unique opportunity to gain
experience in an area which is of high demand in the research field.

We are looking for a highly motivated student with BSc degree in
biology and experience in molecular biology. Experience on insects&rsquo;
manipulation and physiology are a plus. This project is a collaboration
between the University of Amsterdam (Institute for Biodiversity and
Evolutionary Dynamics, IBED), Netherlands and the INRA in Versailles,
France. Therefore, communication skills in English (oral and written)
are necessary and knowledge of French is a plus.

There will be a monthly salary of 550 euros, for every complete month
worked, for 6 months maximum. The salary is paid by the Institute of
Ecology and Environnemental Sciences (iEES) - Paris, France, with the
obligation for the student to administratively subscribe to the iEES
(no subscription fees).The group has excellent laboratory facilities
with all state of the art equipment.

Recent/representative papers:

1. Lassance, J.M., Groot, A.T., et al. (2010). Allelic variation in a
fatty-acyl reductase gene causes divergence in moth sex pheromones.
Nature 466, 486-489.

2. Koutroumpa, F. A. et al. (2014)  Shifts in sensory neuron identity
parallel differences in pheromone preference in the European corn
borer. Frontiers in Ecology and Evolution. 2.

3. Koutroumpa, F. A. & Jacquin-Joly, E. (2014)  Sex in the night: Fatty
acid-derived sex pheromones and corresponding membrane pheromone
receptors in insects. Biochimie. 107, 15-21.

For more information contact Fotini KOUTROUMPA or Astrid T. GROOT at
the address below.  To apply please send us a letter of interest,
bachelor scores and CV (including contact information for 2
references). Deadline for application 30 of February 2016.

Contact information:

Fotini Koutroumpa, PhD, Marie Curie SkĹ‚odowska postdoctoral Fellow

Astrid T. Groot, group leader

IBED, University of Amsterdam, Science Park 904, 1098 XH Amsterdam
Phone: +31 20 5255400
Fax: +31 20 5257832
F.Koutroumpa@uva.nl or a.t.groot@uva.nl

<a href="http://bit.ly/1RudHSI">http://bit.ly/1RudHSI</a>

Address in France:

INRA of Versailles - UMR 1392 - iEES-Paris
RD 10 - route de Saint-Cyr, 78026 VERSAILLES Cedex
Tel : 0033 (0)1 30 83 31 63
Fax : 0033 (0)1 30 83 31 19

2016-01-06 14:26 GMT+01:00 Brian Golding
:

  The EvolDir does not post attachments (hence no viruses, better for
  twitter, etc).  Please send plain text in the body of your message
  (as you would like it to appear) and I will see that it is posted.
  Your advert _MUST_ be directed toward evolutionary biologists.
  Brian
  On Wed, Jan 06, 2016 at 08:56:47AM +0000, Fotini Koutroumpa wrote:
  >    Dear evoldir
  >
  >    I send you the announcement for a second year master position
  at our
  >    laboratory. I would be greatful if you could advertise it to
  your
  >    scientific followers.
  >    Please do not hesitate to contact me for further information.
  >    Kind regards
  >    Fotini Koutroumpa
  >
  >    Fotini Koutroumpa, PhD
  >    Marie Curie Sklodowska postdoctoral Fellow
  >    Institute for Biodiversity and Ecosystems Dynamics (IBED)
  >    University of Amsterdam
  >    Science Park 904
  >    1098 XH Amsterdam
  >    Phone: +31 20 5255400
  >    Fax: +31 20 5257832
  >
  >  F.Koutroumpa@uva.nl
  >
  >    <a href="http://bit.ly/1RudHSI">http://bit.ly/1RudHSI</a>
  >   
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:41</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136799036996">Graduate position: ULeicester, Genomics of social organisation</a></div>
  <div class="description"><p>A competition funded PhD studentship at the Department of Genetics, University of Leicester, UK

PhD title: Genomics of social organisation

Supervisors: Dr Rob Hammond & Dr Sandra Beleza (Genetics, University of Leicester)
Collaborator: Dr Y.Wurm, (Queen Mary University of London) 

The evolution of highly integrated social groups with a reproductive division of labour, exemplified by the eusocial Hymenoptera (ants, bees and wasps), is a major evolutionary transition offering unparalleled opportunities to understand how organisation at a higher level evolves and diversifies. Furthermore, changes in social organisation are of great economic importance as they are linked to the invasiveness and pest status of social insects. However, to date, little is known about genetic influences on social organisation in social insects. This is changing. Next generation sequencing (NGS) is now able to reveal the genomic architecture underpinning variation in social organisation, and, in turn, the genomic effects of social living [1, 2]. 

We propose an ambitious PhD investigating the population genetics and genomic architecture underlying a polymorphism in social organisation in the ant, Leptothorax acervorum [3, 4].  

Aims and objectives:

(1) Assess geographical variation in colony social organisation among isolated populations in southern Europe (central and north west Spain, Spanish/French Pyrenees, France & Italy). 
(2) Assemble a draft genome of L.acervorum from a haploid male using PCR-free libraries and MiSeq data. 
(3) Investigate signatures of selection across the genome of L. acervorum using a combination of genome re-sequencing and Restriction site Associated DNA sequencing (RADseq). 
(4) Compare the genetic architecture underlying the polymorphism in social organisation in L.acervorum with the &lsquo;social chromosomes&rsquo; found in other ant species [1,2].

Refs: [1] Wang et al. Nature, 2013. 493:664-668 [2] Purcell et al. Cur. Biol. 24:2728-2732 [3] Gill et al. Proc. Roy. Soc. B. 2009 276: 4423-4431 [4] Gill & Hammond Proc. Roy. Soc. B. 2011. 278: 1524-1531.  

Studentship Funding:  The successful student will be funded by one of two Doctoral Training Partnerships (DTP). Applicants are encouraged to apply to both schemes to maximise their funding success. UK and EU citizens are eligible for funding through these two DTPs (please contact rh225@le.ac.uk if you are unsure about eligibility).

DTP1: Midlands Integrative Biosciences Training Partnership (MIBTP) funded by the BBSRC

Deadline: 10/01/2016

Further details: <a href="http://bit.ly/1kNW3fO">http://bit.ly/1kNW3fO</a>

How to apply: <a href="http://bit.ly/1RudHCp">http://bit.ly/1RudHCp</a>


DTP 2: Central England NERC Training Alliance (CENTA) funded by NERC

Deadline: 25/01/2016

Further details: <a href="http://bit.ly/1kNW43n">http://bit.ly/1kNW43n</a>

How to apply: <a href="http://bit.ly/1RudJu2">http://bit.ly/1RudJu2</a>

Successful applicant: This studentship would suit a highly motivated person with a deep interest in evolutionary biology, genetics and bioinformatics.  The PhD will involve collecting ants in the field, molecular biology and bioinformatics.

For any further information please contact Dr Rob Hammond (rh225@le.ac.uk).


Dr Rob Hammond  
Lecturer in Evolutionary Biology

Department of Genetics 
University of Leicester, University Road, Leicester, LE1 7RH, UK 

t: +44 (0)116 252 5302
e: rh225@le.ac.uk 
w: <a href="http://bit.ly/1kNW3fQ">http://bit.ly/1kNW3fQ</a>


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:29</div>
 <div class="body">
  <div class="title"><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12145">The word is not enough: on morphemes, characters and ontological concepts</a></div>
  <div class="description">The field of morphology has recently seen the arrival of computer-aided ontologies, tools which permit the semantic organization of defined concepts and which therefore promise to be extremely useful in computer-mediated approaches involving morphological data, for example in cladistics. The theoretical relationship between ontologies and cladistics, however, has hardly been explored. Here we examine the ontological status of the main terms in morphological cladistics, i.e. morpheme, character, character state and ontological concept. Morphemes are units of the descriptional perspective, whereas character states are units of the evolutionary perspective and refer to identical stages of transformation. Both morphemes and character states represent things, i.e. real entities. However, character state and morpheme denote different perspectives on these entities (description vs. evolution). Characters (transformation series; Hennig's ideographic character concept) and ontological concepts are both classes, but not of the same nature. Ontological concepts which are used to classify morphemes are constructs, i.e. totally man-made classes that serve only as a way of classifying real entities for human recognition. Characters, however, are classes that encompass all character states of common descent and which therefore have objective, human-independent properties. Characters, then, are natural kinds, classes which exhibit a natural identity in the same way as monophyla do, for example, which also have common descent as an objective property. Against this background, ontology-based morphology can be a valuable addition to phylogenetic systematics. Formalized, machine-parsable descriptions permit the generalization of morphemes in large datasets and can facilitate the recognition of identical character states. However, the expertise of the researcher is indispensable, and full automation of the transfer from the descriptive level to the evolutionary level thus appears impossible.</div>
  <div class="source">Source: <a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:09</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136743804141">Conference: NorthCarolinaStateU.EmergingPlantDisease.Mar23-24</a></div>
  <div class="description">*Event Title:*
A Symposium on Emerging Plant Disease and Global Food Security

*Event Date & Time:*
Wednesday, March 23, 2016 at 9:00 AM
to
Thursday, March 24, 2016 at 4:00 PM

*Event Description:*

The Emerging Plant Disease and Global Food Security Chancellors Faculty 
Excellence Program cluster at NC State will host an international 
symposium March 23-24, 2016 in Raleigh, to bring together experts that 
study emerging plant diseases and their arthropod vectors. Managing the 
threats of emerging diseases that affect agricultural crop plants 
requires experts in genetics, biology, epidemiology, evolutionary 
biology, climate change, metadata analysis, geospatial analytics and 
global development policy.

We will synthesize new developments on emerging plant disease biology 
and discuss an expanding array of new technologies to gather, analyze, 
synthesize, and share knowledge about the evolution of emerging 
infectious diseases of plants that affect global food security. Four 
sessions with keynote speakers, a closing panel and posters are planned.



*Location:*
Hunt Library

*URL:*
<a href="http://bit.ly/1Oav3Qm">http://bit.ly/1Oav3Qm</a>

*Contact:*
Jean Ristaino
Jean_Ristaino@ncsu.edu
919 515-3257

You may also view this event on the calendar 
. 




Jean 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136743316026">Other: IIASA_Austria.SummerFellowships.EvolModeling</a></div>
  <div class="description">Summer Fellowships for Young Scientists at the International Institute
for Applied Systems Analysis, Austria

Funding is available for PhD students interested in three months of
collaborative research during June-August 2016 on

     Evolutionary and Ecological Modeling

at the International Institute for Applied Systems Analysis (IIASA)
in Laxenburg, Austria.

Young scientists from all countries are eligible for stipends provided by
IIASA&rsquo;s Evolution and Ecology Program (EEP) that contribute to travel
and accommodation costs. Students from Australia, Austria, Brazil,
China, Egypt, Finland, Germany, India, Indonesia, Japan, Korea, Malaysia,
Mexico, the Netherlands, Norway, Pakistan, Russia, South Africa, Sweden,
Ukraine, the United Kingdom, USA, and Vietnam are furthermore eligible
for fellowships that provide full coverage of travel, accommodation,
and living expenses.

Model-based summer research projects are invited in the following
indicative areas:

     Evolution of cooperation
     Governance of common goods
     Systemic risk and network dynamics
     Eco-evolutionary dynamics
     Evolutionary community ecology
     Food-web evolution
     Vegetation dynamics
     Adaptive speciation
     Disease ecology and evolution
     Evolutionary conservation biology
     Fisheries management
     Fisheries-induced evolution
     Adaptive dynamics theory and models
     Spatial models in ecology and evolution

Applicants are encouraged to prepare a research proposal that corresponds
to their scientific interests and to EEP&rsquo;s research agenda. Accepted
applicants will begin work before the summer, by planning their research
in collaboration with their IIASA supervisors. Previous experiences
with implementing and studying evolutionary or ecological models are
important assets for working in EEP. To improve chances of being selected,
potential applicants are welcome to send informal inquiries regarding
their specific research interests and plans to EEP&rsquo;s program director
Ulf Dieckmann (dieckmann@iiasa.ac.at).

Online applications will be accepted until Monday, January 11, 2016
(24:00 CET).

Since 1977, IIASA&rsquo;s annual Young Scientists Summer Program (YSSP), has
attracted 1750+ students from 80+ countries. The YSSP 2016 will take place
from June 1 to August 31. IIASA is located in the former summer palace
of Austria&rsquo;s royal family, ca. 15 km south of Vienna. IIASA&rsquo;s summer
program offers exceptional opportunities for acquiring experience in
an international and interdisciplinary research environment. Research
training is based on regular personal interaction with advising
scientists, and typically leads to a publication in an international
journal, as well as to a chapter in a candidate&rsquo;s PhD thesis.

Some useful links:

+ Details about the summer program, and online application
<a href="http://bit.ly/1cA7lML">http://bit.ly/1cA7lML</a>

+ Information about IIASA&rsquo;s Evolution and Ecology Program
<a href="http://bit.ly/1cXEbYV">http://bit.ly/1cXEbYV</a>

+ Examples of successful YSSP projects
<a href="http://bit.ly/1cA7nUR">http://bit.ly/1cA7nUR</a>

+ General information about IIASA
<a href="http://bit.ly/1cXEbYZ">http://bit.ly/1cXEbYZ</a>


Ulf Dieckmann
Program Director
Evolution and Ecology Program
International Institute for Applied Systems Analysis
A-2361 Laxenburg
Austria

Email dieckmann@iiasa.ac.at
Phone +43 2236 807 386
Phone secretary +43 2236 807 231
Fax +43 2236 807 466 or +43 2236 71313
Web <a href="http://bit.ly/17j1ut8">http://bit.ly/17j1ut8</a>
Online reprints <a href="http://bit.ly/17j1utc">http://bit.ly/17j1utc</a> 
FroSpects Network <a href="http://bit.ly/17j1sS5">http://bit.ly/17j1sS5</a>
FinE Network <a href="http://bit.ly/17j1ute">http://bit.ly/17j1ute</a>
FishACE Network <a href="http://bit.ly/17j1sS3">http://bit.ly/17j1sS3</a>


DIECKMANN Ulf 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136743315501">Graduate position: NTNU_Taipei.InsectIslandSpeciaiton</a></div>
  <div class="description">The Systematics & Evolutionary Biology Lab at the Department of Life
Sciences, National Taiwan Normal University (NTNU) in Taipei, Taiwan
is seeking a self-motivated and enthuasiastic PhD student to work on
comparative phylogeography of ecological speciation on Asian continental
and oceanic islands.

The Project:

This research project will analyze the speciation mechanism of
endemic freshwater insects between Taiwan and Yaeyama Islands
of Japan (oceanic island), and between mainland Asia and Hainan
Island (continental island). Several insect species are endemic to
these islands and show adaptation to novel environments (ecological
speciation) between islands. (See a damselfly example, Lee & Lin 2012:
<a href="http://bit.ly/1SztHUB">http://bit.ly/1SztHUB</a>)

The project will generate sequence/genomic data from multiple individuals
of selected endemic species of the islands. First, phylogenetic and
population genetic analyses will be used to infer the process of
speciation between species pairs on the islands. Second, the level of
intrinsic reproductive isolation between species pairs will be measured
through crossing experiments. Third, the strength of natural selection
between species pairs will be accessed through field studies of fitness
components. Lastly, through the analysis of genome-wide DNA polymorphism
of target species, the genomic regions and candidate genes underlying
ecological speciation will be uncovered. International students with an
interest in studying tropical island biology are encouraged to apply.

Funding:

Funding includes a competitive monthly stipend of NT$34,000 (US$
1,133) for up to 3 years for both domestic and international students
through the TIGP Biodiversity (Taiwan International Graduate Program -
Biodiversity) (<a href="http://bit.ly/1ZNwTxG">http://bit.ly/1ZNwTxG</a>), a jointed graduate
program between NTNU and Academia Sinica. Students are also eligible
for on-campus housing during the years of their studies.

Eligibility:

The position is open to all applicants that meet TIGP
admission criteria, but preference will be given to students
with a master degree or prior experiences in phylogenetics,
population genetics and insect biology. Criteria/material used
to evaluate the applicants qualifications for the admission:
(<a href="http://bit.ly/1ZNwTxI">http://bit.ly/1ZNwTxI</a>)

How to apply:

Interested candidates should email (1) a CV, (2) a list of prior
experience and publication, (3) three references (contact info & e-mails),
and (4) a one-page statement of research interests in one PDF file to
Chung-Ping Lin (treehopper@ntnu.edu.tw) with the subject line &ldquo;TIGP
application V Island Speciation&rdquo;.

Additionally, candidates should submit a formal
online application for admission in TIGP Biodiversity
(<a href="http://bit.ly/1SztKQ2">http://bit.ly/1SztKQ2</a>). Deadline for applications
through the online portal is March 31st for a September 2016 start.

Web pages:

The Systematics & Evolutionary Biology Lab

(<a href="http://bit.ly/1SztJfa">http://bit.ly/1SztJfa</a>)

The Department of Life Sciences

(<a href="http://bit.ly/1xWml1v">http://bit.ly/1xWml1v</a>)

National Taiwan Normal University

(<a href="http://bit.ly/1SztJfc">http://bit.ly/1SztJfc</a>)

TIGP Biodiversity Program Website

(<a href="http://bit.ly/1ZNwTxG">http://bit.ly/1ZNwTxG</a>)

Biodiversity Research Center, Academia Sinica

(<a href="http://bit.ly/1ZNwV8Z">http://bit.ly/1ZNwV8Z</a>)

Treehopper 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136743314921">Graduate position: ImperialCollege_London.BeeFitness</a></div>
  <div class="description">PhD position advertised to research how bee foraging performance and
colony fitness are affected by stressors in fragmented rural
landscapes&rsquo;

Supervisors: Dr Richard Gill (r.gill@imperial.ac.uk), Dr Samraat Pawar
(Life Sciences)

Location: Department of Life Sciences, Imperial College London, Silwood
Park campus

Deadline: 18^th January 2016

Increased agricultural practices have resulted in the clearance and
fragmentation of many natural habitats posing potential challenges to
wildlife such as insect pollinators. For eusocial bees (e.g. honeybees
& bumblebees), who provide a crucial pollination service, increased
floral habitat patchiness may increase foraging demands making it
significantly more difficult to sustain the colony&rsquo;s energetic
requirements. It is thus important we consider how environmental
stressors set limits on bee foraging performance and how bees are able
to cope with such pressures.

For further details, how to apply and eligibility please go to:
<a href="http://bit.ly/1SztJf6">http://bit.ly/1SztJf6</a>

But in brief, the studentship will investigate what specific
behavioural mechanisms are being affected by stress factors associated
with agricultural practices and the knock-on effects thus has to colony
functioning and fitness. The research project will provide the PhD
student with a number of benefits: i) it will integrate a number of
critical interdisciplinary skills including controlled experiments and
modelling; ii) it crosses a number of scientific fields such as
behavioural ecology, evolutionary and theoretical biology; iii) the
research is of high impact and will result in quality publications as
well as results that can inform end users and policy makers; and iv)
the student will join a productive, diverse and social research group.

Imperial College is a world leading university; Life Science dept. was
placed top of the UK universities for research intensity in the
Research Excellence Framework, and placed 6th best dept. in the world
in the Times Higher Education 2015. The student will be based at
Silwood Park which has an internationally renowned community of
researchers, excellent facilities for the project, and the GCEE
initiative has invested significantly into ensuring world leading
research addressing important global issues.

Selected references:

Raine & Gill 2015 Nature N&V

Gill & Raine 2014 Functional Ecology (doi:10.1111/1365-2435.12292)

Bryden, Gill et al. 2013 Ecology Letters (doi:10.1111/ele.12188)

Gill et al. 2012 Nature (doi:10.1038/nature11585)

Pawar et al. 2012 Nature (doi:10.1038/nature11131)

Gill & Hammond 2011 Proc. Roy. Soc. B

Gill et al. 2009 Proc. Roy. Soc. B

&ldquo;Gill, Richard J&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:55</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136743314651">Graduate position: TexasChristianU.EvolutionaryGenomics</a></div>
  <div class="description">MS position* A graduate research assistantship position (TA funded)
is available in the lab of Dr. Matt Hale in the Biology Department
at Texas Christian University starting in August 2016. The student
will develop a MS project on the genetic basis of complex phenotypic
traits. Some possible research projects include questions relating to
the genetic basis of migration in salmonid fishes, and the presence
and identity of genes showing patterns of sex-bias in expression,
also in the salmonid fishes. The applicant should have some previous
lab experience, including DNA extraction, PCR amplification, and DNA
sequencing. Ideally, the candidate will also have experience with RNA
extraction, cDNA synthesis and bioinformatics, although training in these
areas will be provided. Interested applicants should send a CV, a cover
letter describing their research interests and reasons for applying,
and the names and contact details of two potential referees. Review
of applications will begin on January 20th 2016. Please send inquires
and applications to Matt Hale (m.c.hale@tcu.edu) Interested students are
encouraged to view the Biology Department website (<a href="http://bit.ly/1qrsWeV">http://bit.ly/1qrsWeV</a>)
for more information on the graduate program.



&ldquo;Hale, Matthew&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="index4603.html?q=aggregator&amp;from=280">previous page</a></div><div>... <a href="index933b.html?q=aggregator&amp;from=220">12</a> <a href="indexd118.html?q=aggregator&amp;from=240">13</a> <a href="index3f45.html?q=aggregator&amp;from=260">14</a> <a href="index4603.html?q=aggregator&amp;from=280">15</a> <strong>16</strong> <a href="index9efb.html?q=aggregator&amp;from=320">17</a> <a href="indexd191.html?q=aggregator&amp;from=340">18</a> <a href="index28a4.html?q=aggregator&amp;from=360">19</a> <a href="index63c1.html?q=aggregator&amp;from=380">20</a> ...</div><div><a href="index9efb.html?q=aggregator&amp;from=320">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=300 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:26:31 GMT -->
</html>
