<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/2&from=20 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:30:02 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexfc9c.html?q=node/431">About Women in Science (Evol2014)</a></li><li><a href="indexf09b.html?q=node/426">Researcher in Environmental Bioinformatics, University of Technology Sydney</a></li><li><a href="indexb172.html?q=node/425">Minutes of mid-year meeting 2014</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 5 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator/sources/2" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator">news aggregator</a>
<ul>
<li class="expanded"><a href="indexe5f5.html?q=aggregator/sources">sources</a>
<ul>
<li class="leaf"><a href="indexfd51.html?q=aggregator/sources/2" class="active">BMC Evolutionary Biology</a></li>
<li class="leaf"><a href="index4421.html?q=aggregator/sources/8">CiteULike Phylogeny</a></li>
<li class="leaf"><a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></li>
<li class="leaf"><a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></li>
<li class="leaf"><a href="index8aad.html?q=aggregator/sources/9">Evolutionary Bioinformatics</a></li>
<li class="leaf"><a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></li>
<li class="leaf"><a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></li>
<li class="leaf"><a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></li>
<li class="leaf"><a href="index9fb6.html?q=aggregator/sources/22">Phyloseminar</a></li>
<li class="leaf"><a href="index69a2.html?q=aggregator/sources/10">Systematics Association</a></li>
<li class="leaf"><a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></li>
<li class="leaf"><a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></li>
<li class="leaf"><a href="indexe0bd.html?q=aggregator/sources/15">dechronization</a></li>
<li class="leaf"><a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></li>
<li class="leaf"><a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></li>
<li class="leaf"><a href="index1f4a.html?q=aggregator/sources/7">postgenomic.com</a></li>

</ul>
</li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 4 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a> &raquo; <a href="index5640.html?q=aggregator">news aggregator</a> &raquo; <a href="indexe5f5.html?q=aggregator/sources">sources</a></div><h2>BMC Evolutionary Biology</h2>
<!-- begin content -->
<div id="aggregator"><div class="feed">The latest research articles published by BMC Evolutionary Biology<h3>URL</h3>
<div class="xml-icon"><a href="http://www.biomedcentral.com/bmcevolbiol/rss/"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div><a href="http://www.biomedcentral.com/bmcevolbiol/">http://www.biomedcentral.com/bmcevolbiol/</a>
<h3>Last update</h3>
11 weeks 8 hours ago</div><h3>November 30, 2015</h3>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/264">Chloroplast phylogenomic analysis of chlorophyte green algae identifies a novel lineage sister to the Sphaeropleales (Chlorophyceae)</a></div>
  <div class="description">Background:
The class Chlorophyceae (Chlorophyta) includes morphologically and ecologically diverse green algae. Most of the documented species belong to the clade formed by the Chlamydomonadales (also called Volvocales) and Sphaeropleales. Although studies based on the nuclear 18S rRNA gene or a few combined genes have shed light on the diversity and phylogenetic structure of the Chlamydomonadales, the positions of many of the monophyletic groups identified remain uncertain. Here, we used a chloroplast phylogenomic approach to delineate the relationships among these lineages.
Results:
To generate the analyzed amino acid and nucleotide data sets, we sequenced the chloroplast DNAs (cpDNAs) of 24 chlorophycean taxa; these included representatives from 16 of the 21 primary clades previously recognized in the Chlamydomonadales, two taxa from a coccoid lineage (Jenufa) that was suspected to be sister to the Golenkiniaceae, and two sphaeroplealeans. Using Bayesian and/or maximum likelihood inference methods, we analyzed an amino acid data set that was assembled from 69 cpDNA-encoded proteins of 73 core chlorophyte (including 33 chlorophyceans), as well as two nucleotide data sets that were generated from the 69 genes coding for these proteins and 29 RNA-coding genes. The protein and gene phylogenies were congruent and robustly resolved the branching order of most of the investigated lineages. Within the Chlamydomonadales, 22 taxa formed an assemblage of five major clades/lineages. The earliest-diverging clade displayed Hafniomonas laevis and the Crucicarteria, and was followed by the Radicarteria and then by the Chloromonadinia. The latter lineage was sister to two superclades, one consisting of the Oogamochlamydinia and Reinhardtinia and the other of the Caudivolvoxa and Xenovolvoxa. To our surprise, the Jenufa species and the two spine-bearing green algae belonging to the Golenkinia and Treubaria genera were recovered in a highly supported monophyletic group that also included three taxa representing distinct families of the Sphaeropleales (Bracteacoccaceae, Mychonastaceae, and Scenedesmaceae).
Conclusions:
Our phylogenomic study advances our knowledge regarding the circumscription and internal structure of the Chlamydomonadales, suggesting that a previously unrecognized lineage is sister to the Sphaeropleales. In addition, it offers new insights into the flagellar structures of the founding members of both the Chlamydomonadales and Sphaeropleales.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/263">Variation in NAT2 acetylation phenotypes is associated with differences in food-producing subsistence modes and ecoregions in Africa</a></div>
  <div class="description">Background:
Dietary changes associated to shifts in subsistence strategies during human evolution may have induced new selective pressures on phenotypes, as currently held for lactase persistence. Similar hypotheses exist for arylamine N-acetyltransferase 2 (NAT2) mediated acetylation capacity, a well-known pharmacogenetic trait with wide inter-individual variation explained by polymorphisms in the NAT2 gene. The environmental causative factor (if any) driving its evolution is as yet unknown, but significant differences in prevalence of acetylation phenotypes are found between hunter-gatherer and food-producing populations, both in sub-Saharan Africa and worldwide, and between agriculturalists and pastoralists in Central Asia. These two subsistence strategies also prevail among sympatric populations of the African Sahel, but knowledge on NAT2 variation among African pastoral nomads was up to now very scarce. Here we addressed the hypothesis of different selective pressures associated to the agriculturalist or pastoralist lifestyles having acted on the evolution of NAT2 by sequencing the gene in 287 individuals from five pastoralist and one agriculturalist Sahelian populations.
Results:
We show that the significant NAT2 genetic structure of African populations is mainly due to frequency differences of three major haplotypes, two of which are categorized as decreased function alleles (NAT2*5B and NAT2*6A), particularly common in populations living in arid environments, and one fast allele (NAT2*12A), more frequently detected in populations living in tropical humid environments. This genetic structure does associate more strongly with a classification of populations according to ecoregions than to subsistence strategies, mainly because most Sahelian and East African populations display little to no genetic differentiation between them, although both regions hold nomadic or semi-nomadic pastoralist and sedentary agriculturalist communities. Furthermore, we found significantly higher predicted proportions of slow acetylators in pastoralists than in agriculturalists, but also among food-producing populations living in the Sahelian and dry savanna zones than in those living in humid environments, irrespective of their mode of subsistence.
Conclusion:
Our results suggest a possible independent influence of both the dietary habits associated with subsistence modes and the chemical environment associated with climatic zones and biomes on the evolution of NAT2 diversity in sub-Saharan African populations.</div>
 </div>
</div>
<h3>November 20, 2015</h3>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/261">Evolutionary comparison of prenylation pathway in kinetoplastid Leishmania and its sister Leptomonas</a></div>
  <div class="description">Background:
Leptomonas is monogenetic kinetoplastid parasite of insects and is primitive in comparison to Leishmania. Comparative studies of these two kinetoplastid may share light on the evolutionary transition to dixenous parasitism in Leishmania. In order to adapt and survive within two hosts, Leishmania species must have acquired virulence factors in addition to mechanisms that mediate susceptibility/resistance to infection in the pathology associated with disease. Rab proteins are key mediators of vesicle transport and contribute greatly to the evolution of complexity of membrane transport system. In this study we used our whole genome sequence data of these two divergent kinetoplastids to analyze the orthologues/paralogues of Rab proteins.
Results:
During change of lifestyle from monogenetic (Leptomonas) to digenetic (Leishmania), we found that the prenyl machinery remained unchanged. Geranylgeranyl transferase-I (GGTase-I) was absent in both Leishmania and its sister Leptomonas. Farnesyltransferase (FTase) and geranylgeranyl transferase-II (GGTase-II) were identified for protein prenylation. We predict that activity of the missing alpha-subunit (α-subunit) of GGTase-II in Leptomonas was probably contributed by the α-subunit of FTase, while beta-subunit (β-subunit) of GGTase-II was conserved and indicated functional conservation in the evolution of these two kinetoplastids. Therefore the β-subunit emerges as an excellent target for compounds inhibiting parasite activity in clinical cases of co-infections. We also confirmed that during the evolution to digenetic life style in Leishmania, the parasite acquired capabilities to evade drug action and maintain parasite virulence in the host with the incorporation of short-chain dehydrogenase/reductase (SDR/MDR) superfamily in Rab genes.
Conclusion:
Our study based on whole genome sequences is the first to build comparative evolutionary analysis and identification of prenylation proteins in Leishmania and its sister Leptomonas. The information presented in our present work has importance for drug design targeted to kill L. donovani in humans but not affect the human form of the prenylation enzymes.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/260">Elusive ditrysian phylogeny: an account of combining systematized morphology with molecular data (Lepidoptera)</a></div>
  <div class="description">Background:
Ditrysia comprise close to 99 % of all butterflies and moths. The evolutionary relationships among the ditrysian superfamilies have received considerable attention in phylogenetic studies based on DNA and transcriptomic data, but the deepest divergences remain for large parts unresolved or contradictory. To obtain complementary insight into the evolutionary history of the clade, and to test previous hypotheses on the subdivision of Ditrysia based on morphology, we examine the morphology of larvae, pupae and adult males and females of 318 taxa representing nearly all ditrysian superfamilies and families. We present the most comprehensive morphological dataset on Ditrysia to date, consisting of over 500 morphological characters. The data are analyzed alone and combined with sequence data (one mitochondrial and seven nuclear protein-coding gene regions, sequenced from 422 taxa). The full dataset consists of 473 exemplar species. Analyses are performed using maximum likelihood methods, and parsimony methods for the morphological dataset. We explore whether combining morphological data and DNA-data can stabilize taxa that are unstable in phylogenetic studies based on genetic data only.
Results:
Morphological characters are found phylogenetically informative in resolving apical nodes (superfamilies and families), but characters serving as evidence of relatedness of larger assemblages are few. Results include the recovery of a monophyletic Tineoidea, Sesioidea and Cossoidea, and a stable position for some unstable taxa (e.g. Epipyropidae, Cyclotornidae, Urodoidea + Schreckensteinioidea). Several such taxa, however, remain unstable even though morphological characters indicate a position in the tree (e.g. Immidae). Evidence supporting affinities between clades are suggested, e.g. a novel larval synapomorphy for Tineidae. We also propose the synonymy of Tineodidae with Alucitidae, syn. nov.
Conclusions:
The large morphological dataset provides information on the diversity and distribution of morphological traits in Ditrysia, and can be used in future research on the evolution of these traits, in identification keys and in identification of fossil Lepidoptera. The “backbone” of the phylogeny for Ditrysia remains largely unresolved. As previously proposed as an explanation for the scarcity of molecular signal in resolving the deeper nodes, this may be due to the rapid radiation of Ditrysia in the Cretaceous.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/259">A comparison of human and mouse gene co-expression networks reveals conservation and divergence at the tissue, pathway and disease levels</a></div>
  <div class="description">Background:
A deeper understanding of differences and similarities in transcriptional regulation between species can uncover important information about gene functions and the role of genes in disease. Deciphering such patterns between mice and humans is especially important since mice play an essential role in biomedical research.
Results:
Here, in order to characterize evolutionary changes between humans and mice, we compared gene co-expression maps to evaluate the conservation of co-expression. We show that the conservation of co-expression connectivity of homologous genes is negatively correlated with molecular evolution rates, as expected. Then we investigated evolutionary aspects of gene sets related to functions, tissues, pathways and diseases. Genes expressed in the testis, eye and skin, and those associated with regulation of transcription, olfaction, PI3K signalling, response to virus and bacteria were more divergent between mice and humans in terms of co-expression connectivity. Surprisingly, a deeper investigation of the PI3K signalling cascade revealed that its divergence is caused by the most crucial genes of this pathway, such as mTOR and AKT2. On the other hand, our analysis revealed that genes expressed in the brain and in the bone, and those associated with cell adhesion, cell cycle, DNA replication and DNA repair are most strongly conserved in terms of co-expression network connectivity as well as having a lower rate of duplication events. Genes involved in lipid metabolism and genes specific to blood showed a signature of increased co-expression connectivity in the mouse. In terms of diseases, co-expression connectivity of genes related to metabolic disorders is the most strongly conserved between mice and humans and tumor-related genes the most divergent.
Conclusions:
This work contributes to discerning evolutionary patterns between mice and humans in terms of gene interactions. Conservation of co-expression is a powerful approach to identify gene targets and processes with potential similarity and divergence between mice and humans, which has implications for drug testing and other studies employing the mouse as a model organism.</div>
 </div>
</div>
<h3>November 19, 2015</h3>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/258">Maternal care in Acanthosomatinae (Insecta: Heteroptera: Acanthosomatidae)&#8212;correlated evolution with morphological change</a></div>
  <div class="description">Background:
Maternal care (egg-nymph guarding behavior) has been recorded in some genera of Acanthosomatidae. However, the origin of the maternal care in the family has remained unclear due to the lack of phylogenetic hypotheses. Another reproductive mode is found in non-caring species whose females smear their eggs before leaving them. They possess pairs of complex organs on the abdominal venter called Pendergrast’s organ (PO) and spread the secretion of this organ onto each egg with their hind legs, which is supposed to provide a protective function against enemies. Some authors claim that the absence of PO may be associated with the presence of maternal care. No study, however, has tested this hypothesis of a correlated evolution between the two traits.
Results:
We reconstructed the molecular phylogeny of the subfamily Acanthosomatinae using five genetic markers sequenced from 44 species and one subspecies with and without maternal care. Eight additional species from the other two acanthosomatid subfamilies were included as outgroups. Our results indicated that maternal care has evolved independently at least three times within Acanthosomatinae and once in the outgroup species. Statistical tests for correlated evolution showed that the presence of maternal care is significantly correlated with the secondary loss or reduction of PO. Ancestral state reconstruction for the node of Acanthosoma denticaudum (a non-caring species in which egg smearing with developed POs occurs) and A. firmatum (a caring species with reduced POs) suggested egg smearing was still present in their most recent common ancestor and that maternal care in A. firmatum has evolved relatively recently.
Conclusions:
We showed that maternal care is an apomorphic trait that has arisen multiple times from the presence of PO within the subfamily Acanthosomatinae. The acquisition of maternal care is correlated with the reduction or loss of PO, which suggests an evolutionary trade-off between the two traits resulting from physiological costs. This prediction also implies that presence of maternal care can be highly expected for those groups lacking behavioral data, which invariably also lack the organ. No secondary loss of maternal care was detected in the present tree. We suggest that the loss of maternal care may be suppressed due to the vulnerability of the PO-free condition, which thus maintains maternal care.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/257">Large scale patterns of genetic variation and differentiation in sugar maple from tropical Central America to temperate North America</a></div>
  <div class="description">Background:
Geological events in the latter Cenozoic have influenced the distribution, abundance and genetic structure of tree populations in temperate and tropical North America. The biogeographical history of temperate vegetation that spans large ranges of latitude is complex, involving multiple latitudinal shifts that might have occurred via different migration routes. We determined the regional structuring of genetic variation of sugar maple (Acer saccharum subsp. saccharum) and its only subspecies in tropical America (Acer saccharum subsp. skutchii) using nuclear and chloroplast data. The studied populations span a geographic range from Maine, USA (46°N), to El Progreso, Guatemala (15°N). We examined genetic subdivisions, explored the locations of ancestral haplotypes, analyzed genetic data to explore the presence of a single or multiple glacial refugia, and tested whether genetic lineages are temporally consistent with a Pleistocene or older divergence.
Results:
Nuclear and chloroplast data indicated that populations in midwestern USA and western Mexico were highly differentiated from populations in the rest of the sites. The time of the most recent common ancestor of the western Mexico haplotype lineage was dated to the Pliocene (5.9 Ma, 95 % HPD: 4.3–7.3 Ma). Splits during the Pleistocene separated the rest of the phylogroups. The most frequent and widespread haplotype occurred in half of the sites (Guatemala, eastern Mexico, southeastern USA, and Ohio). Our data also suggested that multiple Pleistocene refugia (tropics-southeastern USA, midwestern, and northeastern USA), but not western Mexico (Jalisco), contributed to post-glacial northward expansion of ranges. Current southern Mexican and Guatemalan populations have reduced population sizes, genetic bottlenecks and tend toward homozygosity, as indicated using nuclear and chloroplast markers.
Conclusions:
The divergence of western Mexican populations from the rest of the sugar maples likely resulted from orographic and volcanic barriers to gene flow. Past connectivity among populations in the southeastern USA and eastern Mexico and Guatemala possible occurred through gene flow during the Pleistocene. The time to the most common ancestor values revealed that populations from the Midwest and Northeast USA represented different haplotype lineages, indicating major divergence of haplotypes lineages before the Last Glacial Maximum and suggesting the existence of multiple glacial refugia.</div>
 </div>
</div>
<h3>November 17, 2015</h3>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/236">Proliferation of East Antarctic Ad&#233;lie penguins in response to historical deglaciation</a></div>
  <div class="description">Background:
Major, long-term environmental changes are projected in the Southern Ocean and these are likely to have impacts for marine predators such as the Adélie penguin (Pygoscelis adeliae). Decadal monitoring studies have provided insight into the short-term environmental sensitivities of Adélie penguin populations, particularly to sea ice changes. However, given the long-term nature of projected climate change, it is also prudent to consider the responses of populations to environmental change over longer time scales. We investigated the population trajectory of Adélie penguins during the last glacial-interglacial transition to determine how the species was affected by climate warming over millennia. We focussed our study on East Antarctica, which is home to 30 % of the global population of Adélie penguins.
Methods:
Using mitochondrial DNA from extant colonies, we reconstructed the population trend of Adélie penguins in East Antarctica over the past 22,000 years using an extended Bayesian skyline plot method. To determine the relationship of East Antarctic Adélie penguins with populations elsewhere in Antarctica we constructed a phylogeny using mitochondrial DNA sequences.
Results:
We found that the Adélie penguin population expanded 135-fold from approximately 14,000 years ago. The population growth was coincident with deglaciation in East Antarctica and, therefore, an increase in ice-free ground suitable for Adélie penguin nesting. Our phylogenetic analysis indicated that East Antarctic Adélie penguins share a common ancestor with Adélie penguins from the Antarctic Peninsula and Scotia Arc, with an estimated age of 29,000 years ago, in the midst of the last glacial period. This finding suggests that extant colonies in East Antarctica, the Scotia Arc and the Antarctic Peninsula were founded from a single glacial refuge.
Conclusions:
While changes in sea ice conditions are a critical driver of Adélie penguin population success over decadal and yearly timescales, deglaciation appears to have been the key driver of population change over millennia. This suggests that environmental drivers of population trends over thousands of years may differ to drivers over years or decades, highlighting the need to consider millennial-scale trends alongside contemporary data for the forecasting of species’ abundance and distribution changes under future climate change scenarios.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/256">A new fireworm (Amphinomidae) from the Cretaceous of Lebanon identified from three-dimensionally preserved myoanatomy</a></div>
  <div class="description">Background:
Rollinschaeta myoplena gen. et sp. nov is described from the Late Cretaceous (Cenomanian) Konservat-Lagerstätten of Hakel and Hjoula, Lebanon. The myoanatomy of the fossils is preserved in exceptional detail in three dimensions as calcium phosphate, allowing the musculature of the body wall, gut and parapodia to be reconstructed in detail.
Results:
The major muscle groups of polychaetes can be identified in Rollinschaeta, including longitudinal muscle bands, circular muscles, oblique muscles, the parapodial muscle complex and the gut musculature, with a resolution sufficient to preserve individual fibres. To allow meaningful comparison with the phosphatized fossil specimens, extant polychaetes were stained with iodine and visualised using microCT. Rollinschaeta myoplena possesses two pairs of dorsal longitudinal muscles, dorsal and ventral circular muscles and a single pair of ventral longitudinal muscles. While six longitudinal muscle bands are known from other polychaete groups, their presence in combination with circular muscles is unique to Amphinomidae, allowing these fossils to be diagnosed to family level based solely on their myoanatomy. The elongate, rectilinear body and equally sized, laterally projecting parapodia of Rollinschaeta are found only within Amphinominae, demonstrating that the Cretaceous species is derived amongst Amphinomida.
Conclusion:
The uniquely preserved myoanatomy of Rollinschaeta has allowed diagnosis of a fossil annelid to subfamily level using microCT as a comparative tool for exploring myoanatomy in fossil and extant polychaetes. Our results demonstrate that fossilized muscles can provide systematically informative anatomical detail and that they should be studied when preserved.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/255">Being cosmopolitan: evolutionary history and phylogeography of a specialized raptor, the Osprey Pandion haliaetus</a></div>
  <div class="description">Background:
The Osprey (Pandion haliaetus) is one of only six bird species with an almost world-wide distribution. We aimed at clarifying its phylogeographic structure and elucidating its taxonomic status (as it is currently separated into four subspecies). We tested six biogeographical scenarios to explain how the species’ distribution and differentiation took place in the past and how such a specialized raptor was able to colonize most of the globe.
Results:
Using two mitochondrial genes (cyt b and ND2), the Osprey appeared structured into four genetic groups representing quasi non-overlapping geographical regions. The group Indo-Australasia corresponds to the cristatus ssp, as well as the group Europe-Africa to the haliaetus ssp. In the Americas, we found a single lineage for both carolinensis and ridgwayi ssp, whereas in north-east Asia (Siberia and Japan), we discovered a fourth new lineage. The four lineages are well differentiated, contrasting with the low genetic variability observed within each clade. Historical demographic reconstructions suggested that three of the four lineages experienced stable trends or slight demographic increases. Molecular dating estimates the initial split between lineages at about 1.16 Ma ago, in the Early Pleistocene.
Conclusions:
Our biogeographical inference suggests a pattern of colonization from the American continent towards the Old World. Populations of the Palearctic would represent the last outcomes of this colonization. At a global scale the Osprey complex may be composed of four different Evolutionary Significant Units, which should be treated as specific management units. Our study brought essential genetic clarifications, which have implications for conservation strategies in identifying distinct lineages across which birds should not be artificially moved through exchange/reintroduction schemes.</div>
 </div>
</div>
<h3>November 16, 2015</h3>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/254">The origin and evolution of queen and fertility signals in Corbiculate bees</a></div>
  <div class="description">Background:
In social Hymenoptera (ants, bees and wasps), various chemical compounds present on the cuticle have been shown to act as fertility signals. In addition, specific queen-characteristic hydrocarbons have been implicated as sterility-inducing queen signals in ants, wasps and bumblebees. In Corbiculate bees, however, the chemical nature of queen-characteristic and fertility-linked compounds appears to be more diverse than in ants and wasps. Moreover, it remains unknown how queen signals evolved across this group and how they might have been co-opted from fertility signals in solitary ancestors.
Results:
Here, we perform a phylogenetic analysis of fertility-linked compounds across 16 species of solitary and eusocial bee species, comprising both literature data as well as new primary data from a key solitary outgroup species, the oil-collecting bee Centris analis, and the highly eusocial stingless bee Scaptotrigona depilis. Our results demonstrate the presence of fertility-linked compounds belonging to 12 different chemical classes. In addition, we find that some classes of compounds (linear and branched alkanes, alkenes, esters and fatty acids) were already present as fertility-linked signals in the solitary ancestors of Corbiculate bees, while others appear to be specific to certain species.
Conclusion:
Overall, our results suggest that queen signals in Corbiculate bees are likely derived from ancestral fertility-linked compounds present in solitary bees that lacked reproductive castes. These original fertility-linked cues or signals could have been produced either as a by-product of ovarian activation or could have served other communicative purposes, such as in mate recognition or the regulation of egg-laying.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/252">Bilobate leaves of Bauhinia (Leguminosae, Caesalpinioideae, Cercideae) from the middle Miocene of Fujian Province, southeastern China and their biogeographic implications</a></div>
  <div class="description">Background:
Morphological and molecular phylogenetic studies suggest that the pantropical genus Bauhinia L. s.l. (Bauhiniinae, Cercideae, Leguminosae) is paraphyletic and may as well be subdivided into nine genera, including Bauhinia L. s.s. and its allies. Their leaves are usually characteristic bilobate and are thus easily recognized in the fossil record. This provides the opportunity to understand the early evolution, diversification, and biogeographic history of orchid trees from an historical perspective under the framework of morphological and molecular studies.
Results:
The taxonomy, distribution, and leaf architecture of Bauhinia and its allies across the world are summarized in detail, which formed the basis for classifying the bilobate leaf fossils and evaluating the fossil record and biogeography of Bauhinia. Two species of Bauhinia are described from the middle Miocene Fotan Group of Fujian Province, southeastern China. Bauhinia ungulatoides sp. nov. is characterized by shallowly to moderately bilobate, pulvinate leaves with shallowly cordate bases and acute apices on each lobe, as well as paracytic stomatal complexes. Bauhinia fotana F.M.B. Jacques et al. emend. possesses moderately bilobate, pulvinate leaves with moderately to deeply cordate bases and acute or slightly obtuse apices on each lobe.
Conclusions:
Bilobate leaf fossils Bauhinia ungulatoides and B. fotana together with other late Paleogene – early Neogene Chinese record of the genus suggest that Bauhinia had been diverse in South China by the late Paleogene. Their great similarities to some species from South America and South Asia respectively imply that Bauhinia might have undergone extensive dispersals and diversification during or before the Miocene. The fossil record, extant species diversity, as well as molecular phylogenetic analyses demonstrate that the Bauhiniinae might have originated in the Paleogene of low-latitudes along the eastern Tethys Seaway. They dispersed southwards into Africa, migrated from Eurasia to North America via the North Atlantic Land Bridge or floating islands during the Oligocene. Then the genus spread into South America probably via the Isthmus of Panama since the Miocene onward, and underwent regional extinctions in the Boreotropics of mid-high-latitudes during the Neogene climatic cooling. Hence, Bauhinia presently exhibits a pantropical intercontinental disjunct distribution.</div>
 </div>
</div>
<h3>November 14, 2015</h3>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/251">Karyotype differentiation in 19 species of river loach fishes (Nemacheilidae, Teleostei): extensive variability associated with rDNA and heterochromatin distribution and its phylogenetic and ecological interpretation</a></div>
  <div class="description">Background:
Loaches of the family Nemacheilidae are one of the most speciose elements of Palearctic freshwater ichthyofauna and have undergone rapid ecological adaptations and colonizations. Their cytotaxonomy is largely unexplored; with the impact of cytogenetical changes on this evolutionary diversification still unknown. An extensive cytogenetical survey was performed in 19 nemacheilid species using both conventional (Giemsa staining, C- banding, Ag- and Chromomycin A
                      3
                   /DAPI stainings) and molecular (fluorescence in situ hybridization with 5S rDNA, 45S rDNA, and telomeric (TTAGGG)
                      n 
                   probes) methods. A phylogenetic tree of the analysed specimens was constructed based on one mitochondrial (cytochrome b) and two nuclear (RAG1, IRBP) genes.
Results:
Seventeen species showed karyotypes composed of 2n = 50 chromosomes but differentiated by fundamental chromosome number (NF = 68–90). Nemachilichthys ruppelli (2n = 38) and Schistura notostigma (2n = 44–48) displayed reduced 2n with an elevated number of large metacentric chromosomes. Only Schistura fasciolata showed morphologically differentiated sex chromosomes with a multiple system of the XY
                      1
                   Y
                      2 
                   type. Chromomycin A
                      3 
                   (CMA
                      3
                   )- fluorescence revealed interspecific heterogeneity in the distribution of GC-rich heterochromatin including its otherwise very rare association with 5S rDNA sites. The 45S rDNA sites were mostly located on a single chromosome pair contrasting markedly with a pattern of two (Barbatula barbatula, Nemacheilus binotatus, N. ruppelli) to 20 sites (Physoschistura sp.) of 5S rDNA. The cytogenetic changes did not follow the phylogenetic relationships between the samples. A high number of 5S rDNA sites was present in species with small effective population sizes.
Conclusion:
Despite a prevailing conservatism of 2n, Nemacheilidae exhibited a remarkable cytogenetic variability on microstructural level. We suggest an important role for pericentric inversions, tandem and centric fusions in nemacheilid karyotype differentiation. Short repetitive sequences, genetic drift, founder effect, as well as the involvement of transposable elements in the dispersion of ribosomal DNA sites, might also have played a role in evolutionary processes such as reproductive isolation. These remarkable dynamics of their genomes qualify river loaches as a model for the study of the cytogenetic background of major evolutionary processes such as radiation, endemism and colonization of a wide range of habitats.</div>
 </div>
</div>
<h3>November 11, 2015</h3>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/250">Comparative phylogeography of endemic Azorean arthropods</a></div>
  <div class="description">Background:
For a remote oceanic archipelago of up to 8 Myr age, the Azores have a comparatively low level of endemism. We present an analysis of phylogeographic patterns of endemic Azorean island arthropods aimed at testing patterns of diversification in relation to the ontogeny of the archipelago, in order to distinguish between alternative models of evolutionary dynamics on islands. We collected individuals of six species (representing Araneae, Hemiptera and Coleoptera) from 16 forest fragments from 7 islands. Using three mtDNA markers, we analysed the distribution of genetic diversity within and between islands, inferred the differentiation time-frames and investigated the inter-island migration routes and colonization patterns.
Results:
Each species exhibited very low levels of mtDNA divergence, both within and between islands. The two oldest islands were not strongly involved in the diffusion of genetic diversity within the archipelago. The most haplotype-rich islands varied according to species but the younger, central islands contributed the most to haplotype diversity. Colonization events both in concordance with and in contradiction to an inter-island progression rule were inferred, while a non-intuitive pattern of colonization from western to eastern islands was also inferred.
Conclusions:
The geological development of the Azores has followed a less tidy progression compared to classic hotspot archipelagos, and this is reflected in our findings. The study species appear to have been differentiating within the Azores for </div>
 </div>
</div>
<h3>November 10, 2015</h3>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/247">Decoupled evolution of floral traits and climatic preferences in a clade of Neotropical Gesneriaceae</a></div>
  <div class="description">Background:
Major factors influencing the phenotypic diversity of a lineage can be recognized by characterizing the extent and mode of trait evolution between related species. Here, we compared the evolutionary dynamics of traits associated with floral morphology and climatic preferences in a clade composed of the genera Codonanthopsis, Codonanthe and Nematanthus (Gesneriaceae). To test the mode and specific components that lead to phenotypic diversity in this group, we performed a Bayesian phylogenetic analysis of combined nuclear and plastid DNA sequences and modeled the evolution of quantitative traits related to flower shape and size and to climatic preferences. We propose an alternative approach to display graphically the complex dynamics of trait evolution along a phylogenetic tree using a wide range of evolutionary scenarios.
Results:
Our results demonstrated heterogeneous trait evolution. Floral shapes displaced into separate regimes selected by the different pollinator types (hummingbirds versus insects), while floral size underwent a clade-specific evolution. Rates of evolution were higher for the clade that is hummingbird pollinated and experienced flower resupination, compared with species pollinated by bees, suggesting a relevant role of plant-pollinator interactions in lowland rainforest. The evolution of temperature preferences is best explained by a model with distinct selective regimes between the Brazilian Atlantic Forest and the other biomes, whereas differentiation along the precipitation axis was characterized by higher rates, compared with temperature, and no regime or clade-specific patterns.
Conclusions:
Our study shows different selective regimes and clade-specific patterns in the evolution of morphological and climatic components during the diversification of Neotropical species. Our new graphical visualization tool allows the representation of trait trajectories under parameter-rich models, thus contributing to a better understanding of complex evolutionary dynamics.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/248">Selective sweeps versus introgression - population genetic dynamics of the murine leukemia virus receptor Xpr1 in wild populations of the house mouse (Mus musculus)</a></div>
  <div class="description">Background:
The interaction between viruses and their receptors in the host can be expected to lead to an evolutionary arms race resulting in cycles of rapid adaptations. We focus here on the receptor gene Xpr1 (xenotropic and polytropic retrovirus receptor 1) for murine leukemia viruses (MLVs). In a previous screen for selective sweeps in mouse populations we discovered that a population from Germany was almost monomorphic for Xpr1 haplotypes, while a population from France was polymorphic.
Results:
Here we analyze Xpr1 sequences and haplotypes from a broad sample of wild mouse populations of two subspecies, M. m. domesticus and M. m. musculus, to trace the origins of this distinctive polymorphism pattern. We show that the high polymorphism in the population in France is caused by a relatively recent invasion of a haplotype from a population in Iran, rather than a selective sweep in Germany. The invading haplotype codes for a novel receptor variant, which has itself undergone a recent selective sweep in the Iranian population.
Conclusions:
Our data support a scenario in which Xpr1 is frequently subject to positive selection, possibly as a response to resistance development against recurrently emerging infectious viruses. During such an infection cycle, receptor variants that may convey viral resistance can be captured from another population and quickly introgress into populations actively dealing with the infectious virus.</div>
 </div>
</div>
<h3>November 9, 2015</h3>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/245">Distinct functions of two olfactory marker protein genes derived from teleost-specific whole genome duplication</a></div>
  <div class="description">Background:
Whole genome duplications (WGDs) have been proposed to have made a significant impact on vertebrate evolution. Two rounds of WGD (1R and 2R) occurred in the common ancestor of Gnathostomata and Cyclostomata, followed by the third-round WGD (3R) in a common ancestor of all modern teleosts. The 3R-derived paralogs are good models for understanding the evolution of genes after WGD, which have the potential to facilitate phenotypic diversification. However, the recent studies of 3R-derived paralogs tend to be based on in silico analyses. Here we analyzed the paralogs encoding teleost olfactory marker protein (OMP), which was shown to be specifically expressed in mature olfactory sensory neurons and is expected to be involved in olfactory transduction.
Results:
Our genome database search identified two OMPs (OMP1 and OMP2) in teleosts, whereas only one was present in other vertebrates. Phylogenetic and synteny analyses suggested that OMP1 and 2 were derived from 3R. Both OMPs showed distinct expression patterns in zebrafish; OMP1 was expressed in the deep layer of the olfactory epithelium (OE), which is consistent with previous studies of mice and zebrafish, whereas OMP2 was sporadically expressed in the superficial layer. Interestingly, OMP2 was expressed in a very restricted region of the retina as well as in the OE. In addition, the analysis of transcriptome data of spotted gar, a non-teleost fish, revealed that single OMP gene was expressed in the eyes.
Conclusion:
We found distinct expression patterns of zebrafish OMP1 and 2 at the tissue and cellular level. These differences in expression patterns may be explained by subfunctionalization as the model of molecular evolution. Namely, single OMP gene was speculated to be originally expressed in the OE and the eyes in the common ancestor of all Osteichthyes (bony fish including tetrapods). Then, two OMP gene paralogs derived from 3R-WGD reduced and specialized the expression patterns. This study provides a good example for analyzing a functional subdivision of the teleost OE and eyes as revealed by 3R-derived paralogs of OMPs.</div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/244">Erratum to: The relationship of recombination rate, genome structure, and patterns of molecular evolution across angiosperms</a></div>
  <div class="description">No description available</div>
 </div>
</div>
<h3>November 4, 2015</h3>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/243">Retraction Note: TREEFINDER: a powerful graphical analysis environment for molecular phylogenetics</a></div>
  <div class="description">No description available</div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/242">Context-dependent associations between heterozygosity and immune variation in a wild carnivore</a></div>
  <div class="description">Background:
A multitude of correlations between heterozygosity and fitness proxies associated with disease have been reported from wild populations, but the genetic basis of these associations is unresolved. We used a longitudinal dataset on wild Galapagos sea lions (Zalophus wollebaeki) to develop a relatively new perspective on this problem, by testing for associations between heterozygosity and immune variation across age classes and between ecological contexts.
Results:
Homozygosity by locus was negatively correlated with serum immunoglobulin G production in pups (0–3 months of age), suggesting that reduced genetic diversity has a detrimental influence on the early development of immune defence in the Galapagos sea lion. In addition, homozygosity by locus was positively correlated with total circulating leukocyte concentration in juveniles (6–24 months of age), but only in a colony subject to the anthropogenic environmental impacts of development, pollution and introduced species, which suggests that reduced genetic diversity influences mature immune system activity in circumstances of high antigen exposure.
Conclusions:
These findings demonstrate the environmental context-dependency of the phenotypic expression of immune variation, which is implicit in the theory of ecoimmunology, but which has been rarely demonstrated in the wild. They also indicate that heterozygosity may be linked to the maintenance of heterogeneity in mammalian immune system development and response to infection, adding to the body of evidence on the nature of the mechanistic link between heterozygosity and fitness.</div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="index79d7.html?q=aggregator/sources/2&amp;from=0">first page</a></div><div><a href="index79d7.html?q=aggregator/sources/2&amp;from=0">previous page</a></div><div><a href="index79d7.html?q=aggregator/sources/2&amp;from=0">1</a> <strong>2</strong> <a href="index092c.html?q=aggregator/sources/2&amp;from=40">3</a> <a href="indexacef.html?q=aggregator/sources/2&amp;from=60">4</a> <a href="index9d08.html?q=aggregator/sources/2&amp;from=80">5</a> </div><div><a href="index092c.html?q=aggregator/sources/2&amp;from=40">next page</a></div><div><a href="index9d08.html?q=aggregator/sources/2&amp;from=80">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news." class="active">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/2&from=20 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:30:02 GMT -->
</html>
