<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=100 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:20:45 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexfe3a.html?q=node/279">Contact</a></li><li><a href="index2875.html?q=node/72">MacClade and Adobe Illustrator</a></li><li><a href="indexf64b.html?q=node/63">LaTeX template and bibliography style file</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 5 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 3 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>February 21, 2016</h3>
<div class="news-item">
 <div class="date">00:34</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139710075521">Postdoc: USaoPaulo.MolecularSystematics</a></div>
  <div class="description">POSTDOCTORAL POSITION - FACULDADE DE SADE PBLICA, UNIVERSIDADE DE SO PAULO

The Laboratory of Public Health Entomology - Molecular Systematics
at the University of So Paulo, Brazil, seeks a highly motivated and
productive postdoctoral researcher to work on a FAPESP Tematico-funded
research (grant no. 2014/26229-7) on malaria vector mosquitoes in rural
settlements in the Brazilian Amazon.

Those interested must have a doctorate and experience in phylogenetic
systematics and related fields, as well as excellent knowledge of
molecular techniques used in taxonomy and population genetics of
insect vectors. Computational skills and statistics will be of great
importance. The candidate should collaborate in both disciplines and
field work in Brazil and training of undergraduate and graduate students.

The position requires an individual with a PhD and experience in
phylogenetic systematics and related fields. Outstanding molecular,
computational and statistics skills are particularly advantageous. It is
expected the candidate collaborates in discipline, field work in Brazil,
and training undergraduate and graduate students in molecular techniques
employed in systematics.

Funding is available for a minimum of 1 year, and may be extended upon
both mutual and FAPESP agreement. Preference will be given to candidates
with a high probability of applying for and obtaining independent funding;
for example through the TDR/WHO, or other funding agencies.

The starting date is flexible, and the position will remain open until
filled. For primary consideration, applicants should apply by March 17th,
2016. Informal inquiries are welcomed, prior to formal application. To
apply, please send the following:

1.	A Curriculum Vitae
2.      Names of 3 referees willing to provide a letter of
        recommendation upon request
3.      A brief statement of how your research goals fit with research
        of Culicidae lab
4.      A brief statement of interest, ideas, and qualifications for
        independent grant applications

E-mail applications are preferred: masallum@usp.br

Mailed applications are also acceptable to:
Maria Anice Mureb Sallum
Faculdade de Sade Pblica
Universidade de So Paulo
Avenida Doutor Arnaldo 715 sala 210
01246-904 So Paulo
So Paulo, Brasil

&ldquo;masallum@usp.br&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>February 20, 2016</h3>
<div class="news-item">
 <div class="date">17:11</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/ematsen">@ematsen</a> wrote:</p>
            
              <p>Hello everyone-- </p>

<p>I've decided to try automatically posting every new topic from phylobabble to the @phylobabble Twitter feed. If you don't like this, please comment. </p>

<p>Thanks!</p>
            
            <p>Posts: 1</p>
            <p>Participants: 1</p>
            <p><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Read full topic</a></p></div>
  <div class="source">Source: <a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:04</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139651158051">Postdoc: UGlasgow.GenomeEvolutionaryTransitions</a></div>
  <div class="description"><p>We have a research opportunity open at the University of Glasgow&rsquo;s Institute of Biodiversity, Animal Health & Comparative Medicine (IBAHCM) working in the Evolutionary Analysis Group and the research team of Kathryn Elmer (<a href="http://bit.ly/1wO4VE9">http://bit.ly/1wO4VE9</a>). 

We are seeking a motivated, creative and enthusiastic postdoctoral researcher with expertise in genomics for a project on &lsquo;major evolutionary transitions&rsquo;. The primary research effort will be on a NERC-funded project studying the molecular and genomic mechanisms underlying different reproductive modes in lizards. Complementary projects on genome evolution in 'ichs and herps&rsquo; are also possible. Bioinformatic experience with NGS data and expertise in whole genome analysis is imperative, ideally with relevant experience in complex de novo genome assembly. Skills in quantitative trait mapping, comparative genomics, ecological and/or population genomics, and phylogenetics are also sought. Team working and positive attitude are a must. A strong track record of genetic and evolutionary research is necessary, and on fishes or reptiles is a benefit. Candidates must have completed their PhD by the start of contract. 

The position is for 1.5 years (with possibility for extension) starting as soon as possible, and is funded through the NERC grant 'Unravelling the genetics of a major evolutionary transition&rsquo; to Kathryn Elmer, Maureen Bain and Rod Page.

IBAHCM is a stimulating and interactive research environment with a wealth of opportunities for discussion, collaboration and cutting edge research in evolution, ecology, and disease. The University of Glasgow ranks in the world&rsquo;s top 100 universities.  The University and IBAHCM have both been recognised with Athena SWAN Bronze awards. The city of Glasgow is lively and cultural, and sits on the doorstep of the great outdoors of the Scottish Highlands, islands, and coast. 

The official job description and application requirements are available on the University of Glasgow homepage (under current vacancies; <a href="http://bit.ly/OBSo4S">http://bit.ly/OBSo4S</a>) at job reference M00658. **The advertisement closes 7 March 2016.** Informal inquiries to K. Elmer welcome in advance.  kathryn.elmer@glasgow.ac.uk

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:04</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139651157901">Postdoc: UCalifornia_Davis.PlantPathogenEvolution</a></div>
  <div class="description">2/19/16
Postdoctoral Position in Quantitative Disease Resistance
Available: May 2016 till&hellip;
Location: Plant Sciences: University of California, Davis


We seek a postdoctoral researcher to work on a project investigating
the evolution and networks controlling genomic disease resistance and
pathogen virulence variation in the interaction between a single pathogen
(Botrytis cinerea) and eight different eudicot plant species.

Our laboratory has been conducting extensive work to understand
the molecular basis of quantitative phenotypes at a genomic level
utilizing network biology in Arabidopsis thaliana and the necrotrophic
plant pathogen (Botrytis cinerea) (for a list of publications please
go to the Web site). We have finished the genomic sequence of 100+
B. cinerea isolates as well as infection assays on a broad array of
Arabidopsis genotypes and other dicot species (Soybean, Lettuce, Chicory,
Sunflower, Tomato and two Brassicas). For each plant species there were
multiple genotypes representing wild and domesticated lineages. This
analysis also includes whole transcriptome analysis of both the plant
and pathogen. This provides a unique dataset from which to examine the
effects of domestication and lineage evolution in the eudicots on a
single fungal pathogen and vice versa.

Among the questions we can ask are to identify how genomic variation in
the pathogen influences host specificity across the eudicots with a focus
on lineage specific evolution and the potential role of domestication. We
can also utilize this data to conduct GWA (Genome wide association)
mapping within the pathogen and the plants to identify the loci that
control quantitative resistance variation across the dicots. This
will enable us to begin parsing the genetic networks that control a
pathogen&rsquo;s specificity across plant species, plant genotypes and how the
pathogen evolves in response to the plant. These questions are directly
translatable to fundamental population biology theory, ecological theory
and have direct application to improving crop productivity in the face
of endemic diseases as represented by Botrytis cinerea. Thus, there is
an extensive collection of phenotypic and genomic data that are ready to
be analyzed and utilized to develop and test new hypothesis surrounding
quantitative disease interactions.

The project is currently funded until May of 2019. Applicants with
a Ph.D. in quantitative or computational genetics will be given high
priority. Knowledge of modern molecular biological techniques (e.g. PCR,
Sequence analysis, in vitro protein expression etc.) and laboratory-based
research is required. The position is available immediately but the
start date is flexible. Salary will be commensurate with experience;
the position also includes health insurance and other benefits.

The laboratory is an interactive group at the forefront of applying
genomics techniques to understanding the molecular basis of quantitative
phenotypes in both an ecological and applied context. We have numerous
international collaborations and opportunities with an excellent track
record of placing individuals in advanced positions.

To apply, please e-mail a cover letter, curriculum vitae, and the
names of three references to Daniel Kliebenstein. Applications will
be reviewed until the position is filled. For more information contact
Dr. Kliebenstein via e-mail at Kliebenstein@ucdavis.edu.

Daniel Kliebenstein
Professor of Plant Genomics
Univ of California, Davis
Dept of Plant Sci- MS3
1 Shields Ave
Davis, CA 95616

Kliebenstein Lab

Member of Dynamo Center
University of Copenhagen
Dynamo Center



Daniel Kliebenstein 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:49</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139650720656">Conference: Tuscany.Speciation.Feb19-24</a></div>
  <div class="description"><p>We are pleased to announce SPECIATION 2017, a Gordon Research Conference
(GRC) dedicated exclusively to speciation research. The conference 
via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:19</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139649824841">Postdoc: UKansas.DrosophilaParasiteCoevolution</a></div>
  <div class="description">The Department of Molecular Biosciences, University of Kansas, seeks
a postdoctoral researcher with interests and experience in Drosophila
host-parasite coevolution.  The successful candidate will work with a
research group that addresses the broad theme host-virus interaction and
coevolution using Drosophila and its associated Nudivirus infection. This
position consists of an initial 2-yr appointment and is expected to
begin as early as August 18, 2016.

Required qualifications for the position includes a Ph.D. in Biology
or closely related field, experience with population genetics and
next-generation sequencing as evidenced by application materials,
experience Drosophila husbandry as evidenced by application materials
and some experience with coding in R, Perl, Python, etc. as evidenced
by application materials.

For a complete announcement and to apply online, go to
<a href="http://bit.ly/1oRhdMZ.">http://bit.ly/1oRhdMZ.</a> A complete application includes
a detailed resume, a cover letter, and the names and contact information
for two references. Initial review of applications begins March 15,
2016. For first consideration, please apply before March 15, 2016

EO statement: The University of Kansas prohibits discrimination on the
basis of race, color, ethnicity, religion, sex, national origin, age,
ancestry, disability status as a veteran, sexual orientation, marital
status, parental status, gender identity, gender expression, and genetic
information in the universitys programs and activities.  Retaliation
is also prohibited by university policy.  The following persons have
been designated to handle inquiries regarding the nondiscrimination
policies and are the Title IX coordinators for their respective campuses:
Executive Director of the Office of Institutional Opportunity & Access,
IOA@ku.edu, 1246 West Campus Road, Room 153A, Lawrence, KS 66045,
785-864-6414, 711 TTY (for the Lawrence, Edwards, Parsons, Yoder, and
Topeka campuses); Director, Equal Opportunity Office, Mail Stop 7004, 4330
Shawnee Mission Parkway, Fairway, KS 66205, 913-588-8011, 711 TTY (for
the Wichita, Salina, and Kansas City, Kansas medical center campuses).



&ldquo;Unckless, Robert Lewis&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:04</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139649359861">Job: Frankfurt.CuratorIchthyology</a></div>
  <div class="description"><p>Job posting – Ref. #01-16006

The Senckenberg Society for Nature Research (SGN), an institution of
the Leibniz Association with almost 800 employees, conducts
state-of-the-art natural science research at leading research
institutions located in seven of Germany’s federal states. The SGN is
in charge of the UNESCO natural World Heritage Site &ldquo;Grube Messel&rdquo;.


The Senckenberg Society for Nature Research, headquartered in Frankfurt
am Main, Germany, intends to fill  at the Department of Marine Zoology 
the position of a
- Curator of Ichthyology -  (Full time)

Your tasks:
●	Assume responsibility for the management and curatorship of
Senckenberg’s extensive collection of marine and freshwater fishes
●	Oversee the rearrangement of the fish collections and their
relocation to a new building
●	Oversee the transfer of offices, laboratories, and an
ichthyological library to the new premises, which are currently being
rebuilt, meeting highest curatorial standards
●	Supervise technical assistants
●	Conduct collection-based research on the taxonomy, systematics,
and phylogeny of marine fishes
●	Participate in joint marine research with other scientists at
the Department of Marine Zoology and beyond


Your profile:
●	Advanced degree in biology/zoology or related field (preferably
doctoral degree)
●	At least five years of work experience in curatorship and
management of zoological (preferably ichthyological) collections
●	Familiarity with systematics of fishes
●	Outstanding research record in systematic ichthyology,
documented by publications in peer-reviewed scientific journals
●	Fluency in English, both spoken and written; good knowledge of
German will be an asset
●	Firm commitment to your work and sense of responsibility


We offer: 
●	An attractive and challenging position in a research institute
of international standing. 
Employment should start as soon as possible and is initially limited to
28 February 2018
●	A salary that reflects the tasks and responsibilities of the
position based on the collective agreement for public service in the
state of Hesse (TV-H) 
●	A discounted ‘job ticket’ for public transportation in the
Rhine-Main area


Senckenberg is an equal opportunities employer and places an emphasis
on fostering career opportunities for women. Qualified women are
strongly encouraged to apply. Senckenberg is certified by “audit
berufundfamilie”. We guarantee adherence to the severely disabled
directives and the law on part-time employment. The duty station will be
Frankfurt am Main.

Interested? Please send your complete application documents (CV,
certificates and references, letter of motivation), preferably in
electronic form (as a single PDF file) quoting reference #01-16006 by 22
March 2016 to:

Dr. Tobias Schneck
c/o Senckenberg Gesellschaft für Naturforschung
Senckenberganlage 25
60325 Frankfurt am Main
E-Mail: recruiting@senckenberg.de  


For more information, please contact Dr. Dieter Fiege,
dieter.fiege@senckenberg.de, 
phone: +49-69-7542-1265.






via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:19</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139647858996">Other: Pyrenees.VolFieldTech.PlantSpeciation</a></div>
  <div class="description">Field assistants summer 2016: Volunteer positions in evolutionary biology

Nick Barton&rsquo;s group at the Institute of Science and Technology (IST)
Austria requires volunteers to assist with field work on plant
speciation in the Pyrenees (Spain) this coming summer (late May - early August).

The project: We are studying the evolutionary dynamics underlying
species diversification in the genus Antirrhinum (snapdragons). A major
focus of this research involves field work on natural hybrid zones
between two subspecies with different flower colours. Most of the field
work is contributing to a long-term pedigree project aimed at
establishing a direct link from genotype to phenotype to fitness. With
20,000 samples collected over six generations (so far), this provides
an exciting and powerful system to examine many outstanding questions in
speciation and quantitative genetics in wild populations. We are seeking
volunteers to assist with the field work, which involves working in
teams mapping the location of individual plants (GPS), tagging and
sampling them for leaves and flowers, measuring quantitative traits,
phenotyping them for flower pigmentation and processing material for
later DNA extraction. There may also be opportunities to be involved in
other projects we are doing on plant-pollinator interactions, pollen
fertility and community ecology surveys. Most of the work is outdoors,
however we do spend some time indoors processing samples. The work is
highly team orientated, typically in groups of 2-3 in the field and
larger groups processing samples back at the research station. This is a
great opportunity for anybody looking to obtain experience in field work
relating to evolutionary biology, plant ecology and plant-animal
interactions. You will also be part of a large multidisciplinary team
including researchers from IST Austria (Vienna), John Innes Centre
(Norwich, UK) and the University of Toulouse.

The field site is located near Ripoll in a beautiful part of the
Pyrenees of North Eastern Spain (Catalonia). We stay in comfortable
apartments overlooking a picturesque valley, with close access to hiking
trails and small villages. All food, accommodation and travel (within
Europe) are covered. However, we cannot offer any further stipend.

For these positions we are looking for hard working and enthusiastic
biology students with a strong interest in working outdoors with plants.
You must be meticulous with recording data and also be comfortable
working as part of a team. Experience with field-based projects and
plants is preferred but not essential.

We require assistance between late May and late July. Depending
on the year and the plants, we may extend our time into August. Length
of stay is flexible but a minimum stay of 3 weeks is required.

Please send any questions to the email address below. How to apply? By
the closing date of March 31st, please send a statement of your
background, CV, why you are interested and the length of time you would
be available via email to maria.clara.melo@ist.ac.at

Maria Clara MELO HURTADO 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:19</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139647858706">Graduate position: UBayreuth_Germany.SpermMicrobeInteractions</a></div>
  <div class="description">***DISENTANGLING HOST-PARASITE AND MALE-FEMALE COEVOLUTIONARY EFFECTS
ON HOST FITNESS***

I am looking for an ambitious PhD-candidate interested evolutionary
biology and reproductive physiology. The project aims to disentangle
host-parasite and male-female coevolutionary effects on host fitness in
the common bedbug, Cimex lectularius. The project is funded for 3 years by
the German research foundation (DFG). It involves genomics, metagenomics,
and amplicon sequencing of microbes associated with bedbug reproduction,
including sexually transmitted microbes, and experimental manipulation as
well as immunoassays, molecular analysis of paternity and life-history
trait analyses. The project aims to quantify sperm-microbe interactions
and their consequences for male and female fitness. Therefore, I am
looking for a candidate with an open mind to new research ideas, a
solid background in experimental design and statistics. A plus would
be if the candidate has some knowledge of either microbial ecology,
bioinformatics (experience with a unix environment, the R programming
language, especially the ‘vegan’ package) or skills in any of the
above-mentioned areas. The starting date is 1st May 2016.

Within the Department of Animal Ecology I
(<a href="http://bit.ly/1PZ3F8F">http://bit.ly/1PZ3F8F</a>) the
successful candidate will join the group of Animal Population Ecology,
which is working the fields of ecological immunology, host-parasite
interactions, population ecology and biodiversity by studying various
insect species in the lab and in the field.

If you are interested please send me an e-mail until Friday 18th
March 2016 with your application consisting of one pdf containing a
CV (incl. the names and phone numbers of two references and a list
of publications) and a letter of motivation (max. one page DIN A4),
in which you describe how your background and your personal interests
will contribute to the project. Please submit the application without
a picture of you and without your date of birth.

All the best,

Oliver

Dr. Oliver Otti

Animal Population Ecology

Animal Ecology I

University of Bayreuth

Universitätsstrasse 30

95440 Bayreuth

Germany

phone: +49921552646

e-mail: oliver.otti@uni-bayreuth.de

web:
<a href="http://bit.ly/214ilOt">http://bit.ly/214ilOt</a>

Oliver Otti 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>February 19, 2016</h3>
<div class="news-item">
 <div class="date">23:49</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139646743161">Job: UNAB_Chile.Bioinformatics</a></div>
  <div class="description"><p>Faculty Position in Computational Biology with Emphasis on Metabolomics
and Proteomics

The recently established Center for Bioinformatics and Integrative
Biology (CBIB) at Universidad Andres Bello (UNAB), Santiago Chile
seeks a faculty member to establish externally funded, internationally
recognized, and interdisciplinary research programs in bioinformatics
and/or computational biology.

The CBIB hosts a group of six young faculty members focused on
basic science and its applications. Current areas of research include
nanotechnology, microbial genomics, metagenomics, extremophile biology,
biotechnology, protein dynamics and simulation, and protein engineering
and metabolism. The CBIB has strong bonds with national and international
collaborations including UCDavis Chile, US Air Force, and universities
throughout Europe and the USA.

There is one open tenure-track faculty position at the assistant professor
academic level, aimed at complementing our current expertise. We seek
applicants concentrating in the areas of genomics and systems biology,
particularly with expertise in metabolomics and proteomics (molecular
biology experience is a plus). Successful applicants will join the CBIB
team and are expected to perform research and teaching duties. Applicants
must have a PhD in Bioinformatics, Computational Biology, Computer
Science, Mathematics, Genomics, or related discipline, with postdoctoral
research experience, evidence of success in obtaining extramural funding,
and a solid publication track record.

Application Procedure: Applicants should submit an electronic application,
upload a cover letter emphasizing specific qualifications, curriculum
vitae, a description of research interests, and names and contact
information of 3 references at: eduardo.castro@unab.cl

Review of applications will begin on March 25, 2016 and will continue
until positions are filled. Only complete applications will be
considered. For further information about the Center for Bioinformatics
and Integrative Biology, please visit <a href="http://www.cbib.cl/">www.cbib.cl</a>.

For questions relating to this opportunity, please contact Eduardo
Castro-Nallar, Principal Investigator, Center for Bioinformatics and
Integrative Biology, Universidad Andres Bello at eduardo.castro@unab.cl

Eduardo Castro-Nallar, PhD
Center for Bioinformatics and Integrative Biology
Universidad Andrs Bello
Repblica 239, 8370146, Santiago
<a href="http://www.castrolab.org/">www.castrolab.org</a>
+56 2 2770 3520
+56 9 5644 2354


castronallar@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:34</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139646197166">Course: Barcelona.BiolNetworkAnalysis.Oct24-28</a></div>
  <div class="description"><p>Dear colleagues,

Registration is open for the course &ldquo;Introduction to Network Analysis in Life
Sciences - 3rd edition&rdquo;.

Webpage: <a href="http://bit.ly/19yfo98">http://bit.ly/19yfo98</a> 

GRANTS: Thanks to the donations of former participants, this year there is a
place for free at the course for PhD students. If you want to apply please send
an e-mail to grants@transmittingscience.org including: 1- Full name and country,
2- Short project (maximum 300 words), 3- Letter of your advisor stating if you
have any grant to develop your PhD. Deadline: March 31st. Please include the
name of the course in the subject line.

INSTRUCTORS: Dr. Diego Rasskin-Gutman (Institut Cavanilles de Biodiversitat i
Biologia Evolutiva, Spain) and Dr. Borja Esteve-Altava (Institut Cavanilles de
Biodiversitat i Biologia Evolutiva, Spain).

DATES: October 24th - 28th, 2016. 

PLACE: Instalaciones de Sabadell del Institut Catal de Paleontologia M. C.
Sabadell, Barcelona (Espaa).

COURSE DESCRIPTION:

In the last decades, network analysis has provided new insights into the
organisation and functioning of complex biological systems. A basic knowledge on
network modelling and network analysis will provide biologists a better
understanding of cutting-edge research in their fields.

This course will introduce participants into the analysis of complex biological
systems using network models. Students will learn the basics of network
analysis: gathering information, building network models, and interpret the
outcomes of their analysis.

This course combines theoretical introduction and computing practices using the
free software environment R (previous knowledge in R is not required)

Participants are encouraged to bring their own data for practicing. Emphasis is
placed on offering participants a wide overview of network modelling in biology
and the many available software tools to do it. 

Organized by: Transmitting Science and the Institut Catal de Paleontologia
Miquel  Crusafont. 

Please feel free to distribute this information between your colleagues if you
consider it appropriate. 

With best regards  

Sole

Soledad De Esteban-Trivigno, PhD.
soledad.esteban@transmittingscience.org
Transmitting Science
<a href="http://bit.ly/1gdSpYn">http://bit.ly/1gdSpYn</a>

soledad.esteban@transmittingscience.org


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/i1?rss=1">Cover</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/i2?rss=1">Associate Editors</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/i3?rss=1">SMBE Editors and Council</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/i4?rss=1">Subscription Page</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/595?rss=1">On the Causes of Evolutionary Transition:Transversion Bias</a></div>
  <div class="description"><p>A pattern in which nucleotide transitions are favored several fold over transversions is common in molecular evolution. When this pattern occurs among amino acid replacements, explanations often invoke an effect of selection, on the grounds that transitions are more conservative in their effects on proteins. However, the underlying hypothesis of conservative transitions has never been tested directly. Here we assess support for this hypothesis using direct evidence: the fitness effects of mutations in actual proteins measured via individual or paired growth experiments. We assembled data from 8 published studies, ranging in size from 24 to 757 single-nucleotide mutations that change an amino acid. Every study has the statistical power to reveal significant effects of amino acid exchangeability, and most studies have the power to discern a binary conservative-vs-radical distinction. However, only one study suggests that transitions are significantly more conservative than transversions. In the combined set of 1,239 replacements (544 transitions, 695 transversions), the chance that a transition is more conservative than a transversion is 53 % (95 % confidence interval 50 to 56) compared with the null expectation of 50 %. We show that this effect is not large compared with that of most biochemical factors, and is not large enough to explain the several-fold bias observed in evolution. In short, the available data have the power to verify the "conservative transitions" hypothesis if true, but suggest instead that selection on proteins plays at best a minor role in the observed bias.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/603?rss=1">Population Structure Shapes Copy Number Variation in Malaria Parasites</a></div>
  <div class="description"><p>If copy number variants (CNVs) are predominantly deleterious, we would expect them to be more efficiently purged from populations with a large effective population size (<I>N</I>e) than from populations with a small <I>N</I>e. Malaria parasites (<I>Plasmodium falciparum</I>) provide an excellent organism to examine this prediction, because this protozoan shows a broad spectrum of population structures within a single species, with large, stable, outbred populations in Africa, small unstable inbred populations in South America and with intermediate population characteristics in South East Asia. We characterized 122 single-clone parasites, without prior laboratory culture, from malaria-infected patients in seven countries in Africa, South East Asia and South America using a high-density single-nucleotide polymorphism/CNV microarray. We scored 134 high-confidence CNVs across the parasite exome, including 33 deletions and 102 amplifications, which ranged in size from </div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/621?rss=1">Shotgun Mitogenomics Provides a Reference Phylogenetic Framework and Timescale for Living Xenarthrans</a></div>
  <div class="description"><p>Xenarthra (armadillos, sloths, and anteaters) constitutes one of the four major clades of placental mammals. Despite their phylogenetic distinctiveness in mammals, a reference phylogeny is still lacking for the 31 described species. Here we used Illumina shotgun sequencing to assemble 33 new complete mitochondrial genomes, establishing Xenarthra as the first major placental clade to be fully sequenced at the species level for mitogenomes. The resulting data set allowed the reconstruction of a robust phylogenetic framework and timescale that are consistent with previous studies conducted at the genus level using nuclear genes. Incorporating the full species diversity of extant xenarthrans points to a number of inconsistencies in xenarthran systematics and species definition. We propose to split armadillos into two distinct families Dasypodidae (dasypodines) and Chlamyphoridae (euphractines, chlamyphorines, and tolypeutines) to better reflect their ancient divergence, estimated around 42 Ma. Species delimitation within long-nosed armadillos (genus <I>Dasypus</I>) appeared more complex than anticipated, with the discovery of a divergent lineage in French Guiana. Diversification analyses showed Xenarthra to be an ancient clade with a constant diversification rate through time with a species turnover driven by high but constant extinction. We also detected a significant negative correlation between speciation rate and past temperature fluctuations with an increase in speciation rate corresponding to the general cooling observed during the last 15 My. Biogeographic reconstructions identified the tropical rainforest biome of Amazonia and the Guiana Shield as the cradle of xenarthran evolutionary history with subsequent dispersions into more open and dry habitats.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/643?rss=1">Iron-Sulfur Cluster Biogenesis Chaperones: Evidence for Emergence of Mutational Robustness of a Highly Specific Protein-Protein Interaction</a></div>
  <div class="description"><p>Biogenesis of iron&ndash;sulfur clusters (FeS) is a highly conserved process involving Hsp70 and J-protein chaperones. However, Hsp70 specialization differs among species. In most eukaryotes, including <I>Schizosaccharomyces pombe</I>, FeS biogenesis involves interaction between the J-protein Jac1 and the multifunctional Hsp70 Ssc1. But, in <I>Saccharomyces cerevisiae</I> and closely related species, Jac1 interacts with the specialized Hsp70 Ssq1, which emerged through duplication of <I>SSC1</I>. As little is known about how gene duplicates affect the robustness of their protein interaction partners, we analyzed the functional and evolutionary consequences of Ssq1 specialization on the ubiquitous J-protein cochaperone Jac1, by comparing <I>S. cerevisiae</I> and <I>S. pombe</I>. Although deletion of <I>JAC1</I> is lethal in both species, alanine substitutions within the conserved His&ndash;Pro&ndash;Asp (HPD) motif, which is critical for Jac1:Hsp70 interaction, have species-specific effects. They are lethal in <I>S. pombe,</I> but not in <I>S. cerevisiae</I>. These in vivo differences correlated with in vitro biochemical measurements. Charged residues present in the J-domain of <I>S. cerevisiae</I> Jac1, but absent in <I>S. pombe</I> Jac1, are important for tolerance of <I>S. cerevisiae</I> Jac1 to HPD alterations. Moreover, Jac1 orthologs from species that encode Ssq1 have a higher sequence divergence. The simplest interpretation of our results is that Ssq1&rsquo;s coevolution with Jac1 resulted in expansion of their binding interface, thus increasing the efficiency of their interaction. Such an expansion could in turn compensate for negative effects of HPD substitutions. Thus, our results support the idea that the robustness of Jac1 emerged as consequence of its highly efficient and specific interaction with Ssq1.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:04</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/3/657?rss=1">Estimating the Ages of Selection Signals from Different Epochs in Human History</a></div>
  <div class="description"><p>Genetic variation harbors signatures of natural selection driven by selective pressures that are often unknown. Estimating the ages of selection signals may allow reconstructing the history of environmental changes that shaped human phenotypes and diseases. We have developed an approximate Bayesian computation (ABC) approach to estimate allele ages under a model of selection on new mutations and under demographic models appropriate for human populations. We have applied it to two resequencing data sets: An ultra-high depth data set from a relatively small sample of unrelated individuals and a lower depth data set in a larger sample with transmission information. In addition to evaluating the accuracy of our method based on simulations, for each SNP, we assessed the consistency between the posterior probabilities estimated by the ABC approach and the ancient DNA record, finding good agreement between the two types of data and methods. Applying this ABC approach to data for eight single nucleotide polymorphisms (SNPs), we were able to rule out an onset of selection prior to the dispersal out-of-Africa for three of them and more recent than the spread of agriculture for an additional three SNPs.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="index798e.html?q=aggregator&amp;from=80">previous page</a></div><div>... <a href="index8503.html?q=aggregator&amp;from=20">2</a> <a href="index6390.html?q=aggregator&amp;from=40">3</a> <a href="index4543.html?q=aggregator&amp;from=60">4</a> <a href="index798e.html?q=aggregator&amp;from=80">5</a> <strong>6</strong> <a href="index3362.html?q=aggregator&amp;from=120">7</a> <a href="index7b13.html?q=aggregator&amp;from=140">8</a> <a href="indexf37c.html?q=aggregator&amp;from=160">9</a> <a href="index52c1.html?q=aggregator&amp;from=180">10</a> ...</div><div><a href="index3362.html?q=aggregator&amp;from=120">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=100 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:20:45 GMT -->
</html>
