<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/14&from=220 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 21:44:55 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="index66e2.html?q=node/119">54(6) December 2005</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator/sources/14" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator">news aggregator</a>
<ul>
<li class="expanded"><a href="indexe5f5.html?q=aggregator/sources">sources</a>
<ul>
<li class="leaf"><a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></li>
<li class="leaf"><a href="index4421.html?q=aggregator/sources/8">CiteULike Phylogeny</a></li>
<li class="leaf"><a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></li>
<li class="leaf"><a href="index4dbd.html?q=aggregator/sources/14" class="active">EVOLDIR</a></li>
<li class="leaf"><a href="index8aad.html?q=aggregator/sources/9">Evolutionary Bioinformatics</a></li>
<li class="leaf"><a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></li>
<li class="leaf"><a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></li>
<li class="leaf"><a href="indexc12b.html?q=aggregator/sources/13">NESCent</a></li>
<li class="leaf"><a href="index9fb6.html?q=aggregator/sources/22">Phyloseminar</a></li>
<li class="leaf"><a href="index69a2.html?q=aggregator/sources/10">Systematics Association</a></li>
<li class="leaf"><a href="index167f.html?q=aggregator/sources/20">Systematics jobs at ResearchGate</a></li>
<li class="leaf"><a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></li>
<li class="leaf"><a href="indexe0bd.html?q=aggregator/sources/15">dechronization</a></li>
<li class="leaf"><a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></li>
<li class="leaf"><a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></li>
<li class="leaf"><a href="index1f4a.html?q=aggregator/sources/7">postgenomic.com</a></li>

</ul>
</li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 2 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a> &raquo; <a href="index5640.html?q=aggregator">news aggregator</a> &raquo; <a href="indexe5f5.html?q=aggregator/sources">sources</a></div><h2>EVOLDIR</h2>
<!-- begin content -->
<div id="aggregator"><div class="feed"><h3>URL</h3>
<div class="xml-icon"><a href="http://evoldir.tumblr.com/rss"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div><a href="http://evoldir.tumblr.com/">http://evoldir.tumblr.com/</a>
<h3>Last update</h3>
1 hour 40 min ago</div><h3>December 30, 2015</h3>
<div class="news-item">
 <div class="date">23:34</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136310801051">Graduate position: UppsalaU.DeepSeaSpongeGenomics</a></div>
  <div class="description">Ph.D. position on Metabolomics/Genomics of Deep Sea Sponges,

Dept. of Medicinal Chemistry, Uppsala University, SWEDEN.

I am seeking a highly motivated PhD candidate to work within the frame of
the newly EU funded SponGES consortium. This consortium will investigate
deep-sea north Atlantic sponge grounds. The candidate will develop a
suite of genomic and metabolomic resources to investigate the population
diversity of key sponge species and assess connectivity between sponge
grounds. These resources will also unlock the potential of sponge grounds
for drug discovery, in which the candidate will also play a part.

For more info and to apply, use the link below (in Swedish, for
the English version click on &ldquo;English&rdquo;, upper-right corner).
<a href="http://bit.ly/1Ra00ID">http://bit.ly/1Ra00ID</a>

duration of position: possibly 5 years

application deadline: 18th of January 2016

For specific information about the announced Ph.D. position and details
of the project, please contact me (paco.cardenas@fkog.uu.se).


Dr. Paco Cardenas
Div. of Pharmacognosy, Dept. of  Medicinal Chemistry, Uppsala University
BMC Box 574,
SE 75123 Uppsala, Sweden
paco.cardenas@fkog.uu.se



Paco Cardenas 
via Gmail </div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:58</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136244510966">Postdoc: USouthDakota.PhenoscapeProject</a></div>
  <div class="description">Postdoctoral fellow: Bioinformatics, Phenomics

The Phenoscape project (<a href="http://bit.ly/1mpci4c">http://bit.ly/1mpci4c</a>) is recruiting a postdoc
with training in bioinformatics and/or developmental biology who is
interested in analyzing genomic and developmental data in relation to
phenotypic data, with a focus on the vertebrate fin/limb.

The problem of how organismal phenotypes have evolved, are constrained,
and acquire novelty, is one of the grand challenges in biology. The
Phenoscape group has developed ontology-based methods for representing
species phenotypes so that they can be integrated with model organism
developmental and genetic data. The projects Knowledgebase (KB;
kb.phenoscape.org) contains over 500,000 vertebrate species phenotypes
that are linked to ~16,000 genes associated with 320,000+ phenotypes
and 37,000 genes with in situ expression data from model organisms
(zebrafish, mouse, Xenopus human). These data present a tremendous
opportunity for integration with other data types to address questions
about the evolution of phenotype.

We are seeking an individual with expertise in developmental biology
and/or genomics, to (1) help evaluate results of bioinformatics
methods being developed by Phenoscape and (2) leverage the Phenoscape
Knowledgebase to study whole-organism phenotype and functional genomics
in non-model organisms. The purpose of the methods is to improve
prediction of the genetic basis of evolutionarily novel phenotypes by
incorporating semantic similarity, homology, and phylogenetic propagation.
Vertebrate fin and limb phenotypes and genes are enriched in the KB,
and we are thus seeking candidates who ideally have knowledge of genes
and networks involved fin/limb development.  Further, this position
presents a unique opportunity to leverage the linked developmental and
genetic data in the Phenoscape KB for large-scale analysis of patterns
of phenotypic evolution. a

The postdoc will work under the direction of Paula Mabee (University
of South Dakota) in association with Todd Vision (University of North
Carolina), as part of a distributed, multidisciplinary team that includes
evolutionary and model organism biologists, computer scientists, and
bioinformaticists.   Ideally the applicant will be based in South Dakota
(with opportunities to travel to other sites), but we will consider
qualified applicants who are available remotely and/or half-time.

Starting date: This full time postdoctoral position is available to be
filled as early as January 2016 (one year with potential to renew).

Required qualifications:


 *   Ph.D. degree with strong background in bioinformatics; previous
     experience with ontologies preferred
 *   Experience in functional genomics or developmental biology, with
     preference for candidates with a background in vertebrate fin
     and/or limb developmen
 *   Demonstrated communication and writing skills in English
 *   Demonstrated ability to work in a team setting

How to apply:
Please contact Dr. Mabee (pmabee@usd.edu) for inquiries. Applications
should be directed to Dr. Mabee and include a cover letter, CV, a
brief statement detailing your research interests and career goals,
and three letters of reference. For more information, please see
<a href="http://bit.ly/1QYSHVA">http://bit.ly/1QYSHVA</a> and <a href="http://bit.ly/1mpci4e.">http://bit.ly/1mpci4e.</a>

&ldquo;Mabee, Paula&rdquo; 
via Gmail </div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:58</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136244510531">Job: WadsworthCenter.NGS.Bioinformatics</a></div>
  <div class="description">Research Scientist for Next Generation Sequencing, Bioinformatics and
Statistics

A permanent Staff Scientist position in Next Generation Sequencing,
Bioinformatics and Statistics is available at the Wadsworth Center,
NY State Department of Health, Albany, NY.  We seek applications at
the Research Scientist 2 level from highly motivated individuals with
practical experience in bioinformatics analysis of NGS data with an
emphasis on microbial genomes, including pathogen identification and
outbreak tracking, mapping drug-resistance determinants, development
of novel genetic tests, variant analysis, genome annotation and
phylogenetics, In addition, he/she is expected to serve as the statistics
resource for the Center&rsquo;s research and regulatory programs. Alternatively,
we are looking for an Assistant Research Scientist or Research Scientist
1 with strong molecular biology skills and with practical experience in
NGS and an interest in the development of new methods and applications
for NGS in microbial diagnostics.

The Wadsworth Center (<a href="http://www.wadsworth.org/">www.wadsworth.org</a>) is the country&rsquo;s most
comprehensive state public health laboratory with a staff of about
750. The Center provides a dynamic research environment focused on
infectious, genetic and environmental diseases and their impact on human
health. Through its initiative for Public Health Genomics the Center will
bring NGS and other advanced molecular technologies to public health
testing, diagnosis, and research, further develop these technologies
to address new and emerging challenges, and become a regional center of
excellence for NGS in the area of public health genomics.

For the Research Scientist 2 position, a Ph.D. or M.S. degree in the
biological or physical sciences or equivalent and research experience in
NGS data analysis and statistics is preferred.  Knowledge of Unix, Python,
PERL and similar is highly desirable. For the Assistant Research Scientist
or Research Scientist 1 position, a B.S. or M.S. in biological sciences
or equivalent and experience with NGS and strong molecular biology
skills is required. Applicants should submit their curriculum vitae, and
contact information for at least three references to wcphgc@health.ny.gov,
referencing &ldquo;Item 12294&rdquo; in the subject line and indicate which position
they are applying for.  Applications will be accepted until the position
is filled and reviewed as they are received.  AA/EOE.

&ldquo;Lapierre, Pascal (HEALTH)&rdquo; 
via Gmail </div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:43</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136243866356">Job: UGöttingen.ForestGenetics</a></div>
  <div class="description"><p>Please note an exceptional career opportunity at the University of
Gttingen, Germany, for a full Professor of Forest Genetics and Forest
Tree Breeding.

The core activities of the professorship are:

- Research into the genetic diversity of tree species in terms of
  genetics and genomics relating to population ecology and
  molecular ecology

- Research into the genetic basis of the stability of forest ecosystems,
  in particular with regard to climate change

- Research into the conservation of forest genetic resources. 


Please see the full position announcement at
<a href="http://bit.ly/1mhKXSc">http://bit.ly/1mhKXSc</a>
via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:13</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136242483021">Graduate Position: SamNobleMuseum_UOklahoma.Ornithology</a></div>
  <div class="description"><p>Potential PhD positions in in avian genomics, Sam Noble Oklahoma Museum
of Natural History and Department of Biology, University of Oklahoma
Matthew J. Miller (<a href="http://mj-miller.net/">http://mj-miller.net</a>)

I anticipate adding 2V3 PhD students to my lab over the next 21
months. I am particularly interested in  recruiting 1-2 students for all
2016. Students should be willing to take a creative leadership role in one
of three principal research areas in our group: i) comparative genomics
of secondary contact in Mesoamerican lowland birds, ii) comparative
genomics of hybridization in Great Plains birds, or the iii) ecology of
emerging vector-borne diseases in Panama.

A student working on secondary contact in Mesoamerican birds should
expect to spend considerable time during the first two years of the
project leading collecting expeditions in multiple locations in Mexico and
Central America in collaboration with local and regional partners. Thus,
the ideal student will have previous Latin American ornithological
field experience collecting museum voucher specimens, as well as the
ability to understand and speak Spanish. Students from Latin America are
particularly welcome. However, all potential students with strong field,
lab, or collections ornithological skills are encouraged to contact
me. This student will ultimately perform comparative genomic analyses
using datasets that she/he developed using massively-parallel sequencing
resources at the University of Oklahoma, so NGS experience is a plus.

A student working on the hybridization in Great Plains birds should
be motivated to lead collecting expeditions in Oklahoma and adjacent
states, and should have familiarity with population genomics theory
and NGS library techniques (or be able to demonstrate an aptitude for
learning these skills). This student will combine distributional records
and genomic data from historical and contemporary museum specimens to
illuminate how hybrid zones and bird distributional dynamics in the face
of climate change.

Pending NIH and collaborator funding, opportunities to work on the
ecological dynamics of vector-borne diseases in Panama may also exist in
our lab. A student joining our group in disease ecology will be expected
to work closely with Panamanian collaborators (especially Dr. Jose Loaiza,
INDICASAT-AIP, <a href="http://bit.ly/1mQgrig">http://bit.ly/1mQgrig</a>), on
mosquito and vertebrate interactions. Specific research projects will
be guided by secured and pending funding, but typically will involve
a combination of mosquito genomics, screening bird blood for viral
presence, and modeling invertebrate and vertebrate dynamics across a
landscape land-use gradient.

My research group is based in the bird collection of
the Sam Noble Oklahoma Museum of Natural History. SNOMNH
(<a href="http://bit.ly/1PwG5Sz">http://bit.ly/1PwG5Sz</a>)
is a world-class university natural history museum; as such, students
should be motivated to participate in collection growth and curation
as part of their research program. The museum has a modern genetic
laboratory shared with Dr. Cameron Siler (<a href="http://bit.ly/1mQgtGQ">http://bit.ly/1mQgtGQ</a>),
SNOMNH Curator of Herpetology. Genomic library preparation, sequencing,
and data analysis will be conducted in the under-construction
genomics core lab at the on-campus Stephenson Research Center
(<a href="http://bit.ly/1PwG68Q">http://bit.ly/1PwG68Q</a>). Resources in this lab will include
Illumina Mi-Seq and Hi-Seq platforms, as well as supercomputing resources
for analytics.

The SNOMNH and the University of Oklahoma have exceptional resources for
ornithology students. Additional faculty with ornithological expertise
include Eli Bridge (<a href="http://bit.ly/1mQgrij">http://bit.ly/1mQgrij</a>),
Jeff Kelly (<a href="http://bit.ly/1PwG8xC">http://bit.ly/1PwG8xC</a>), and Michael Patten
(<a href="http://bit.ly/1mQgryx">http://bit.ly/1mQgryx</a>). Additional
research opportunities are available in collaboration with the Sutton
Avian Research Center (<a href="http://bit.ly/1PwG8xE">http://bit.ly/1PwG8xE</a>). Also, the biology
department is making a strategic commitment to Geographical Ecology
(<a href="http://ge.ou.edu/">http://ge.ou.edu</a>), which would encompass our groups research
interests, and will provide world-class collaborative resources in
ecological modeling of species distributions and interactions.

Graduate students in my group receive degrees in either Biology or the
EEB program which is affiliated with the Biology department. Students
are supported by Teaching Assistantships with a base salary of 17,000
dollars and include health benefits. Students in my group are likely
to obtain additional support for both stipend and research from the
Sutton Scholarship for Avian Research, and Adams scholarships, which
are internal awards available to OU students. Details can be found here:
<a href="http://bit.ly/1mQgtGS.">http://bit.ly/1mQgtGS.</a>
urthermore, students in my group may be supported as curatorial assistants
rather than teaching assistants, depending on the availability of
funding. Students making reasonable progress towards their degree can
expect to be continuously funded. Norman OK is a great college town with
an attractive cost of living.

The formal application deadline for the graduate program was 15
December, but as a new faculty hire, I&rsquo;ve been given some flexibility
with this date. If you are interested in applying for all 2016,
please email me immediately at matthew.j.m AT villanova.edu, and
also review the application procedures in the biology department
(<a href="http://bit.ly/1PwG8xH">http://bit.ly/1PwG8xH</a>).
Additionally, I will be happy to talk with potential students at the
NAOC conference this August.

Matthew J. Miller
<a href="http://mj-miller.net/">http://mj-miller.net</a>
matthew.j.m AT villanova.edu



matthew.joshua.miller@villanova.edu


via Gmail</p></div>
 </div>
</div>
<h3>December 29, 2015</h3>
<div class="news-item">
 <div class="date">00:06</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136169988141">Postdoc: fungal evolutionary genomics</a></div>
  <div class="description"><p>&ndash;_000_D2A6C2364B636dhibbettclarkuedu_
Content-Type: text/plain; charset=&ldquo;Windows-1252&rdquo;
Content-Transfer-Encoding: quoted-printable

Postdoc: fungal evolutionary genomics

The Hibbett lab at Clark University seeks a highly motivated, independent postdoctoral fellow to conduct research in evolutionary genomics in mushroom-forming fungi. The postdoctoral fellow will be involved in one or more of the following projects: (1) Evolutionary developmental biology of Agaricomycetes; (2) Evolution of substrate-specificity in wood-decay fungi; or (3) Phylogenomics of Lentinula (shiitake mushrooms and relatives). All three projects require expertise in evolutionary genomics and molecular phylogenetics. Experience in fungal genetics is also desirable. Research could include &ldquo;wet bench&rdquo; techniques, including culturing and production of fruiting bodies for transcriptomics, or could be entirely analytical, depending on the candidate&rsquo;s skills and interests. Applicants should have a PhD in genomics, evolutionary genetics, fungal biology, or a relevant discipline, a strong publication record (appropriate for career stage), and excellent communication skills. Will
 ingness to involve students in research is also important. Preferred starting date is June 1, 2016, but this is (somewhat) flexible. Up to three years of support is available, contingent on progress. For more information, including application instructions, please see: <a href="http://bit.ly/1QTAdWM">http://bit.ly/1QTAdWM</a>

Applications from women and members of underrepresented groups in science are encouraged. Clark University is an EEO/AA Employer.

&ndash;_000_D2A6C2364B636dhibbettclarkuedu_
Content-Type: text/html; charset=&ldquo;Windows-1252&rdquo;
Content-ID: 
Content-Transfer-Encoding: quoted-printable


</p>

</div>
 </div>
</div>
<h3>December 28, 2015</h3>
<div class="news-item">
 <div class="date">23:26</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136167889816">Postdoc: ClarkU.FungalEvolutionaryGenomics</a></div>
  <div class="description">Postdoc: fungal evolutionary genomics

The Hibbett lab at Clark University seeks a highly motivated, independent
postdoctoral fellow to conduct research in evolutionary genomics in
mushroom-forming fungi. The postdoctoral fellow will be involved in
one or more of the following projects: (1) Evolutionary developmental
biology of Agaricomycetes; (2) Evolution of substrate-specificity
in wood-decay fungi; or (3) Phylogenomics of Lentinula (shiitake
mushrooms and relatives). All three projects require expertise in
evolutionary genomics and molecular phylogenetics.  Experience in
fungal genetics is also desirable. Research could include â€œwet
benchâ€� techniques, including culturing and production of
fruiting bodies for transcriptomics, or could be entirely analytical,
depending on the candidate&rsquo;s skills and interests. Applicants should
have a PhD in genomics, evolutionary genetics, fungal biology, or a
relevant discipline, a strong publication record (appropriate for
career stage), and excellent communication skills. Willingness to
involve students in research is also important. Preferred starting
date is June 1, 2016, but this is (somewhat) flexible. Up to
three years of support is available, contingent on progress. For
more information, including application instructions, please see:
<a href="http://bit.ly/1QTAdWM">http://bit.ly/1QTAdWM</a>

Applications from women and members of underrepresented groups in
science are encouraged. Clark University is an EEO/AA Employer.

David Hibbett 
via Gmail</div>
 </div>
</div>
<h3>December 25, 2015</h3>
<div class="news-item">
 <div class="date">23:36</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135959233711">Postdoc: SunYatsenU.PlantMacroevolution</a></div>
  <div class="description">Macroevolution of Land Plants at SYSU V Exploring the link between
genome evolution and plant diversity


Postdoctoral research positions (2 years) are open at the new research
team &ldquo;Macroevolution of Land Plants&rdquo; led by Harald Schneider
located at the School of Life Sciences, Sun Yatsen University, Guanzgou,
Guangfdong, China. The position holder is expected to carry out a research
program exploring the evolutionary consequences of polyploidy in selected
groups of seed-free land plants, e.g. derived ferns. The research involves
the integration of genomic evidence (e.g. transcriptome sequences), genome
size (DNA C-values), chromosome numbers, and morphological evidence in
a dated phylogenetic framework. The main issues addressed are (1) the
consequences of polyploidy on the assembly of morphological disparity,
(2) ecological and phylogenetic constraints in the assembly of polyploidy;
and (3) intrinsic and extrinsic constraints on the accumulation of large
genomes via polyploidy.

Key roles: carrying out research on the topic outlined at the postdoctoral
level / First author of at least two publications in scientific journals
with a SCI > 2.0 based on the research carried out / contribute to
other publications of the research team / contribute to or lead the
writing of grant proposals of the research team / presenting of the
research results at international scientific meetings (e.g., IBC 2017) /
contribute to the training of bachelor and master students  / contribute
to the success of the research team

Qualifications of the post-doctoral applicants
Required: Phd in Biology (e.g., Evolutionary Sciences) or Bioinformatics/
General understanding of plant genomes and plant evolution/ Proven
publication record (in English) / Good communication skills/ Ability to
work in teams / Self motivated work habit

Preferred: Experience with the handling of transcriptome data or other
next generation sequence data using bioinformatic pipelines / Experience
in statistical analyses using R or similar platform/ Experience in
phylogenetic analyses and/ or divergence time estimates/ Experience in
plant cytogenetics including cytofluometry or chromosome painting

The call for positions will be open until the positions are
filled. Interviews will be held based on availability.

For informal enquiries please contact Prof Harald Schneider
(haralds@mail.sysu.edu.cn)

	Prof Harald Schneider
	School of Life Sciences
	Sun Yat-Sen University
	Guangzou, Guangdong, China.


Prof Harald Schneider, PhD FLS
School of Life Scienes, Centre of Ecology and Evolution
Sun Yatsen University, Guangzhou, Guangdong, China
&
Department of Life Sciences
Natural History Museum, London SW7 %BD, UK


Harald Schneider 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:04</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135895982941">Job: UFlorida.LandscapeGenetics</a></div>
  <div class="description"><p>Title: 		Assistant Professor of Landscape Ecology
Location: 	Fort Lauderdale Research and Education Center
		University of Florida
		Institute of Food and Agricultural Sciences (IFAS)
		Davie, Florida
Salary: 	Commensurate with Qualifications and Experience

Review Date: 	For full consideration, candidates should apply and submit
additional materials by February 15, 2016. Position will remain open until
a viable applicant pool is determined.

Duties and Responsibilities:

This is a 12-month tenure-accruing position that will be 60% research
(Florida Agricultural Experiment Station) and 40% extension (Florida
Cooperative Extension Service), available at the Fort Lauderdale Research
and Education Center, Institute of Food and Agricultural Sciences, at the
University of Florida. This faculty member is expected to develop a
productive, interdisciplinary, extramurally funded research program with
emphasis on landscape structure and function over a range of spatial and
temporal scales in South Florida. This may include application of climate
change, including sea-level rise, to ecological landscape analysis;
biogeochemical processes and modeling including carbon sequestration and
greenhouse gas emissions; evaluation of species and community responses to
changes in landscape structure and function or invasive species; and
integration of multidisciplinary science into restoration of damaged
landscapes or response to sea-level rise. Extension responsibilities will
include developing education programs on the interaction of Everglades
restoration with natural and man-made ecosystems and on impacts of
sea-level rise. The incumbent will provide leadership, training, and
assistance to county faculty. The position will be located at the Fort
Lauderdale Research and Education Center, Institute of Food and Agriculture
Sciences (IFAS), University of Florida (Davie, Florida). Tenure will accrue
in the most suitable department (Soil & Water Science or School of Forestry
& Resource Conservation), based on the successful applicant&rsquo;s credentials.

Please refer to Requisition # 495173
Dr. Henry Hochmair
Chair, Landscape Ecologist Search and Screen Committee
University of Florida
Fort Lauderdale Research and Education Center
3205 College Avenue
Davie, FL 33314

Telephone:	954.577.6317
Facsimile:	954.475.4125
E-Mail:		hhhochmair@ufl.edu

Application Information:

Individuals wishing to apply should go online to
<a href="http://bit.ly/1QPG7b7">http://bit.ly/1QPG7b7</a>
and submit:
* Application
* Cover letter that states applicant’s interest in the position and
qualifications relative to the credentials listed above
* Curriculum vitae
* Names and contact information (email address and phone number) of three
individuals who may be contacted to provide letters of recommendation


via Gmail</p></div>
 </div>
</div>
<h3>December 24, 2015</h3>
<div class="news-item">
 <div class="date">23:49</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135895315391">Conference:</a></div>
  <div class="description">Evolution of Mechanisms, January 28-31, 2016, University of Debrecen,
Hungary

Please be reminded that the registration deadline for our symposium &ldquo;The
evolution of mechanisms: a workshop on the integration of life history
evolution and physiology&rdquo; is January 03, 2016.

Symposium summary:

In the past decades research in behavior and life history has been
dominated by two different approaches: molecular biologists and
physiologists investigated the fine-scaled mechanisms of how life history
and behavioral processes are regulated, while theoretical and organismal
biology provided an explanation for the adaptive value of such traits and
the evolutionary forces that shape them. While both approaches have been
extremely successful, the proximate-ultimate dichotomy is not warranted
anymore: our next challenge is to integrate these approaches and to
understand the evolution of the genetic, endocrine mechanisms and signaling
pathways that ultimately lead to a bewildering diversity of life-forms.

In order to advance the discussion between these traditionally distinct
areas, we organize a workshop where we invite theoreticians and empiricists
working in the fields of physiology, molecular biology and evolutionary
ecology to discuss the current state of the art and to articulate a
research agenda for the future, where they identify the key challenges and
approaches to study the evolution of the integrated phenotype and
physiological underpinnings of life history evolution. Instead of the
dominantly unidirectional information flow of conference symposia, we
organize this meeting to promote discussion and debate about ideas,
concepts of how we can understand the evolution of mechanisms that provide
the raw material for natural selection.

Participants are invited to give contributed talks that advance the goal of
the meeting. Additional poster presentations will also be welcome.

Invited speakers:

Thomas Flatt (University of Lausanne): The mechanisms underpinning life
history evolution

David Gems (UCL): Role of insulin/IGF-1 signaling in the evolution of life
histories and ageing

Laura Lavine (Washington State University): Endocrine signaling and the
development of the weapons of sexual selection: insights from beetles

Pat Monaghan (University of Glasgow): Environmental effects that shape
individual life histories: mechanisms, trade-offs and time scales

Contact: evolmech2016@gmail.com

For further details and registration visit:
<a href="http://bit.ly/1NOwfZr">http://bit.ly/1NOwfZr</a>

Hope to see you there,

the organizers:

dm Lendvai, Zoltn Nmeth, Jcint Tklyi, Zoltn Barta

Zoltan Nemeth 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:16</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135838809466">Other: DAMBE update</a></div>
  <div class="description">&ndash;_000_D4C76255EEDA2E4F83A13A4447297DD2EA904BA0CMSP03uottawaou_
Content-Type: text/plain; charset=&ldquo;us-ascii&rdquo;
Content-Transfer-Encoding: quoted-printable

Dear All,

I just noticed that I have packed an outdated NucSubMat.BIN file (dated 2003) with DAMBE. This file contains a collection of nucleotide score matrices for sequence alignment in binary format. The new function for phylogenetics with pairwise alignment uses a matrix that is not in that file, resulting in wrong output for nucleotide sequences (amino acid and codon sequence alignment will be fine). I have uploaded a new version with the correction.

Merry Christmas!

Best
Xuhua
<a href="http://bit.ly/1bVebhK">http://bit.ly/1bVebhK</a>
<a href="http://bit.ly/1OMMS7N">http://bit.ly/1OMMS7N</a>


&ndash;_000_D4C76255EEDA2E4F83A13A4447297DD2EA904BA0CMSP03uottawaou_
Content-Type: text/html; charset=&ldquo;us-ascii&rdquo;
Content-Transfer-Encoding: quoted-printable



</div>
 </div>
</div>
<h3>December 22, 2015</h3>
<div class="news-item">
 <div class="date">00:26</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135696500746">Course: Hungary.ConGen.May30-Jun10</a></div>
  <div class="description"><p>Course: ConGen2016 - Recent Advances in Conservation Genetics (May 30
-  June 10, 2016)

We are pleased to announce the course &ldquo;ConGen 2016 - Recent Advances in
Conservation Genetics.&rdquo;

The 2016 course will be held at the Lake Balaton Limnological Institute
on the shores of this charming freshwater lake in the Transdanubian
region of Hungary. The course will be directed by Dr. Stephen J.
O'Brien, and taught by renowned scientists in methods,
interpretation, and applications of molecular genetic and genomic
analyses for conservation of endangered species, who will also share a
variety of their personal experiences in this important field.

Information about the course, participating faculty, venue, and
registration can be found at the following link:

<a href="http://bit.ly/1V0rYHf">http://bit.ly/1V0rYHf</a>

Thank you for your help.

Best regards,

Klaus Koepfli

via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:11</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135695868341">Postdoc: LakeBaikal.MolluscanRadiation</a></div>
  <div class="description">Short term post-doctoral position - Lake Baikal, Russia

9-12 month position from February 2016 working on molecular biology of
an endemic molluscan radiation from Lake Baikal, Siberia. This project
is a collaboration between molluscan taxonomist Tatiana Sitnikova and the
molecular biology lab of Dmitri Sherbakov at the Limnological Institute,
Russian Academy of Sciences, Irkutsk, Siberia, studying the endemic
gastropods of the worlds most ancient and diverse freshwater lake. More
specifically, the project aims at the elucidation of the discrepancies
between the diversity patterns obtained from mitochondrial DNA and
nuclear DNA data in order to obtain a better evolutionary history of
the group and to fit it to the well-studied history of the Baikalian
eco-system transformations.

The project is funded for Sanger sequencing and some NGS (obtaining
depleted libraries or 2-3 transcriptoms). It also includes participation
in 10-15 days expedition around Lake Baikal.

The successful applicant should have skills in molecular laboratory
and analytical work (general lab routines, PCR, purification of
amplicons, dealing with DNA sequences and phylogenetic analysis of
various kinds). The applicant will also be involved into analysis and
publication of the data already obtained.

The project will deliver publications with the successful applicant as
co-author or lead author as appropriate.

The position includes salary at local level. Knowledge of Russian is
not required, though willingness to learn is an advantage.

This is a great opportunity for a young researcher with a sense of
adventure and good academic research skills. Siberia is a region of
can-do optimism and Lake Baikal is one of the most interesting systems
for an evolutionary biologist to study.

For further information on outputs from our lab see the
Limnological institutes webpage <a href="http://lin.irk.ru/">http://lin.irk.ru</a> and
<a href="http://bit.ly/1NIKxeb.">http://bit.ly/1NIKxeb.</a>

Contact  Professor Dmitry Sherbakov
dysh007@gmail.com for specifics on the
application process.

Ellinor Michel 
via Gmail</div>
 </div>
</div>
<h3>December 21, 2015</h3>
<div class="news-item">
 <div class="date">23:56</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135695211816">Postdoc: UBristol.MarineEnvDNA</a></div>
  <div class="description">SeaDNA - Assessing marine biodiversity and structure using environmental
DNA: from groundtruthing to food web structure and stability.

We are looking for a postdoctoral researcher with interests in evolutionary
biology and ecology to take a key role in a new 1M collaborative project
studying environmental DNA (eDNA) in marine ecosystems. The overall
objective of the project is to test the potential for eDNA as a tool for
marine biodiversity assessment. The key aims of the project are to i)
establish how long eDNA from fish and invertebrates persists in the marine
environment before it is broken down beyond the point of detectability, 2)
test the ability of eDNA to be used as a tool in surveying and monitoring
marine species, through direct comparisons of eDNA data to those from
traditional surveys, 3) use eDNA data for reconstructing food webs, test
for stability in food webs over space and time. The project will combine
experimental work on eDNA degradation in the laboratory with analyses of
eDNA samples collected from European shelf seas and the Southern Ocean. The
project is a partnership between the University of Bristol (Martin Genner),
Salford University (Stefano Mariani), British Antarctic Survey (Geraint
Tarling, Sophie Fielding and Will Goodall-Copestake), the Marine Biological
Association (David Sims) and Imperial College (Eoin O'Gorman).

Key roles of will be undertaking experimental work on eDNA degradation in
the laboratory, conducting analyses of eDNA samples collected from European
shelf seas and the Southern Ocean, liaising closely with partners
organisations to ensure project progression, disseminating research through
talks and peer reviewed manuscripts, and contributing to project impact
through engagement with research end-users. You should have a doctoral
degree and should also have an aptitude for experimental design, and
enthusiasm for aquarium-based experiments and fieldwork. You should also
have experience of molecular laboratory work and knowledge of
bioinformatics tools.. There is funding to support this post for 39 months,
is Bristol based, and will start on 1 March 2016.

Closing Date 10 January 2014, with interviews in mid-late January.
Salary 31656 to 35609 per annum.
For the job description and information on how to apply, visit
<a href="http://bit.ly/1ZmztKU">http://bit.ly/1ZmztKU</a> using the job reference code: ACAD101769

For informal enquiries please contact Dr Martin Genner
(m.genner@bristol.ac.uk)

Dr Martin Genner
School of Biological Sciences
University of Bristol
Bristol Life Sciences Building
24 Tyndall Avenue
Bristol
BS8 1TQ

Tel: +44 (0) 117 39 41182

M Genner 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:41</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135694549361">Workshop: Switzerland.Evolutionary_Biology_in_the_Alps.June21-27</a></div>
  <div class="description">Dear Colleagues

The 2016 edition of the Evolutionary Biology Workshop in the Alps will 
take place on 21-27June 2016 in Arolla, Switzerland.

Target participants are PhD students in early stages of PhD and advanced 
Master students - please pass the info to potential participants.

The main goals of this annual workshop, based on a concept developed by 
Stephen Stearns and John Maynard Smith, are to develop the following skills:
•    developing your scientific ideas through discussions in groups;
•    thinking critically and expressing oneself clearly;
•    turning a general idea into a research project;
•    writing a research proposal and defending it.

Faculty:
Bill Rice (University of California Santa Barbara)
Dolph Schluter (University of British Columbia)
Tanja Schwander (University of Lausanne)
John Pannell (University of Lausanne)
Tad Kawecki (University of Lausanne)

It is you, the students, who will be in charge in this course. You will 
be divided in groups of 4-5 students. In those groups, you will work on 
your ideas. You, as a group, will decide what the important open 
questions in broadly defined evolutionary biology are, you will choose 
one, and attempt to develop a proposal for a research project that will 
address it. The faculty will visit the groups during the discussions to 
answer your questions, provide coaching and give you feedback on your 
projects, but they will generally take the back seat. Additionally, the 
faculty will give talks about their research and be available for 
informal discussion with individual students. At the end you will 
present your projects to other participants, and we will party.

The workshop will take place in the 100-year old hotel Kurhaus
<a href="http://bit.ly/1hkynKm,">http://bit.ly/1hkynKm,</a>
located in a small alpine village at an altitude of 2000 m, which will 
help you to concentrate on the course while giving you also the chance 
to enjoy the views, small hikes, marmots and the alpine flora.

Fee: CHF 450.- (this covers part of the cost for room and board, the 
rest being subsidized by the Herbette Foundation).

Participants will receive a course certificate for 3 ETCS credits.

To apply, send a single file (pdf or rtf) containing a short motivation 
letter including a brief summary of your research interest, a cv, and 
the name of your scientific advisor to Caroline Betto-Colliard 
, with Cc to tadeusz.kawecki@unil.ch .  
Please put &ldquo;Evolutionary workshop&rdquo; in the subject. Application Deadline: 
February 14, 2016.

via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135628361741">Other: IUCN.subspeciesListing.answers</a></div>
  <div class="description">*Subspecies diversification and its importance in conservation*

Dear all,

This is in reference to the issue I raised last week whether the subspecies
should be considered as a conservation unit for the assessment under IUCN
Vred list. I emphasise this particularly for those species whose mulitple
forms (subspecies) exists and they have not accorded separately instead
lumped to a species complex under IUCN.

The responses I received are overwhelming and some of you were also
experiencing the same problem that I am facing here in India. I have
seriously considered each of your response/reply to compile my view point
for the species of my concern.

thanking you so much for writing, sharing your experiences, views and
suggesting me the possible way to look forward and convince the
authorities. I greatly acknowledge those who shared articles in support of
their views and had discussion with me. I am herewith sharing you the
compilation report of responses.

Thanking you so much.

Regards

Mukesh Thakur

1 # Ron Okimoto replied:

Get some population geneticists involved.  We are just starting to get a
glimpse of what a genetic load is for a population.   My take is that all
endangered species should be classified and managed by sub species.  The
evolutionary facts of life are that nearly all the sub species are going to
go extinct for one reason or another.  We are just getting data on what the
genetic load is for a population in terms mutations in the genome.  It may
be that some populations will survive their genetic load upon population
reduction and most will not.  The more populations the better the chance
that one will survive.  If you start mixing the populations artificially to
increase current reproductive success you may be dooming the species to
extinction by just making all the populations the same in terms of their
chance at ultimate survival.

Trying to maintain the largest effective population may be the simplest to
manage, but it may not be the best solution in terms of ultimate survival
of that lineage.  We only see the survivors in the current biosphere and
the survivors have been few and the failures have been many.

Ron Okimoto

2 # Dr. Acacia Alcivar-Warren replied:

Dear Mukesh,

Thank you very much for raising this very important IUCN issue.

On a separate but related concern related to marine shrimp populations in
their natural range in the Pacific coast of Latin America, I have not
succeeded in encouraging IUCN leaders to take action to review the penaeus
taxon.

The natural populations of the Pacific whiteleg shrimp, *Litopenaeus
(Penaeus) vannamei,* from northern Peru to northern Mexico, may be at risk.
The unregulated international trade threatens food safety, food security
and national security. There are no international authorities, or concerned
veterinarians, in charge of addressing the lack of supervision for the
massive translocation of a subspecies from one continent to another
-without quarantine!. Nobody seems to take responsibility for this.

The severe *Vibrio* spp-causing disease (AHPND/EMS) of fast growing SPF *L.
vannamei *from US, which first appeared in China in 2009 and quickly spread
to other countries in Asia and then to Mexico in April 2013, is of concern
to conserve healthy biodiversity in light of climate change.

My requests to international agencies and governments to protect the
natural populations of *L. vannamei* have received no replies, not from
IUCN, FAO, World Bank, US authorities, or the authorities of the leading
shrimp producing-countries (Ecuador, India and Indonesia), that claim to be
EMS-free (unconfirmed - we are checking into these claims).

My requests to AVMA leaders and US government&rsquo;s health officials in charge
of protecting our food supply have gone nowhere. Nobody seems to care, in
spite of my repeated calls for action at meetings of the World Aquaculture
Society (<a href="http://bit.ly/1mvDaQg">http://bit.ly/1mvDaQg</a>).
A group in France, supported by IUCN, is actually working on an
industry-sponsored project to &lsquo;address the safety and nutrition of current
feeds used in shrimp aquaculture&rsquo;.

Can I do something to help you *bring together the basic researchers on
population genetics and the leaders of  IUCN?*, to address the issues of
concern to both of us?

Would you be interested in *organizing a Biodiversity Conservation
meeting, *to invite the IUCN leaders to address our issues of concern?.
Friends of the FUCOBI Foundation of Ecuador, of which I was the Founder and
first President, would be very happy to support an international meeting
like this, *if *you include collaborators from our 'ONE HEALTH Epigenomics
Educational Initiative (OHEEI)&rsquo; in India and Nigeria. Indeed, we just sent
a message out to a 'ONE HEALTH&rsquo; list, offering travel awards to pay for
expenses of speakers and students to attend conservation meetings
addressing the holistic concept of 'ONE HEALTH Epigenomics&rsquo;.

Let me know if interested in this offer at: fucobi@gmail.com. The release
is copied below.

Dr. Acacia Alcivar-Warren

ONE HEALTH Practitioner

3 # Gary Thorgaard, Washington State University replied:

Dear Mukesh,

The situation you are interested in sounds similar to that for Pacific
salmon types in the Pacific Northwesr and California, USA.  The U.S.
Government, through its organization NOAA (National Oceanic and Atmospheric
Administration), has listed a number of subgroups within the various salmon
species under the U.S. Endangered Species Act.  The terminology that has
been used for recognition is &ldquo;evolutionarily significant unit&rdquo; (ESU) and
much of the justification for this was developed by Dr. Robin Waples of
NOAA.  I would encourage you to look up his papers and potentially to
correspond with him.
I am not sure about the IUCN&rsquo;s policy in regard to such listings.

Gary

4 # K.A.Bindu replied

Dear Mukesh,

Houbara bustards (chlamydotis undulata )was recently got separated into two
separate species,viz Chlamydotis undulata undulata (African houbara) and
Chlamydotis undulata macqueenii (Asian houbara). The IUCN has listed them
as
vulnerable. But the African bustards comprise of two sub species,
Chlamydotis
undulata undulata and Chlamydotis undulata fuertaventurae (Canary bustards)
and
is there any clear mention about the status of Canary bustards in the IUCN
list?
Regards
K.A.Bindu

5 # *Jonathan Mee replied:*

Hello Mukesh.

Perhaps you would be interested in this article (Ive provided the
open-access link, below). Its about recognizing diversity below the
species level in a widely-distributed fish species.

<a href="http://bit.ly/1OFBmei">http://bit.ly/1OFBmei</a>
Regards,
jon

6 # *Nagarjun V replied:*

Hi Mukesh,

You are doing interesting work with wide reaching implications.

Species de-limitation is a major issue, especially with regards to
conservation. While a few years ago barcoding was used to de-limit species,
now it is being realized that in some species, even use of whole genome
sequencing data is not sufficient to identify species boundaries. One the
other hand one could use ever increasing amount of data to delimit species
based on population structure alone. The crux of the whole issue is species
definition. Should species be defined based on morphology, reproductive
isolation (biological species concept) or just genetic structure? Hence,
conservation genomics is at a cross roads of sorts.

Based on the discussions during a conference on conservation genomics, i
was involved in writing a opinion piece about the subject (see attached).
Hope it will provide you a new perspective at looking

Regards,

Nagarjun

7 # Dr. Axel Hochkirch replied

hi Mukesh,

The official IUCN policy is that a red list assessment has first to be done
on the species level, before subspecies assessments will be published. I
attach the current IUCN guidelines for using the Red List Categories and
Criteria (see page 4). Other intraspecific ranks (formas, morphs,
subvarieties, cultivars etc.) are not included on the IUCN Red List.
So, if you are dealing with a widespread species that has been assessed at
the species level, you are allowed to submit an assessment at the
subspecies unit.
Best regards,

Axel

8 # Keith A. Crandall, PhD replied:

Dear Mukesh,

Great issue!  Thanks for your email.  Ive been involved in red listing of
the freshwater crayfish and the subspecies issue has been problematic.
Basically, IUCN doesnt care about subspecies.  The US Endangered Species
Act, likewise, does not care about subspecies for invertebrates, but it
does for vertebrates!  The ignoring of subspecies in the crayfish leads to
issues of counting because most other lists count subspecies (formally
recognized subspecies).  They often become formal species after further
investigation.  Given that reality, my feeling is that IUCN should be
treating formal subspecies as distinct OTUs deserving of their own
conservation assessment.  Well, thats my two cents anyway.

Take care,

Keith


mukesh thakur 
via Gmail</div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:16</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135627138761">Graduate position: StellenboschU.LizardAdaptation</a></div>
  <div class="description">A PhD studentship is available in the Botany and Zoology department at
Stellenbosch University in the Western Cape of South Africa. The project is
funded by the South African National Research Foundation and focuses on the
evolutionary responses of lizards to climate change. The student will be
supervised by Susana Clusella-Trullas (Stellenbosch University), Michael
Logan (Stellenbosch University), and Jeffrey Streicher (Natural History
Museum, London).

Global climate is changing rapidly and the degree to which natural
populations respond to temperature variation is essential for predicting
the impacts of climate change on biodiversity. Individuals can adjust to
temperature variation through behaviour, acclimation responses and/or
evolutionary mechanisms. While much work has focused on behavioural
compensation and plastic responses, less emphasis has been placed on
evolutionary adaptation. Furthermore, the links between behaviour and
evolutionary adaptation have rarely been addressed, although theory
predicts that behaviour could either inhibit or promote evolutionary
processes. This project will combine physiological ecology and genetics to
examine the evolutionary paths by which populations adapt to climate
variability. This project should generate a powerful predictive framework
for how climate change drives evolutionary adaptation, and how this is
mediated by behaviour in terrestrial ectotherms.

In addition to collecting measurements of locomotor performance and
metabolic rates in the lab, the student will conduct an ethological field
study. Further, the student will travel to the Natural History Museum in
London to receive training in high-throughput DNA sequencing techniques.
Ideal applicants will have an exemplary academic record and prior research
experience. All project running costs have been secured and successful
candidates will receive a bursary for 3 years starting in 2016. For more
information, or to apply, please send a CV and academic transcripts to
Susana Clusella-Trullas: sct333@sun.ac.za. Review of applications will
begin from the 20th of January 2016 onwards and short-listed candidates
will be contacted to set up interviews by phone or skype.

Michael Logan 
via Gmail</div>
 </div>
</div>
<h3>December 20, 2015</h3>
<div class="news-item">
 <div class="date">23:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135625712571">Postdoc: ArizonaStateU.EvolutionMedicine</a></div>
  <div class="description"><p>Postdoctoral Fellow in Evolution & Medicine: Education Resources
Development and Online Learning

Salary: $40,000 &ldquo; 48,000, Salary is negotiable based on the candidate&rsquo;s
experience and ASU salary scales.

Reference # 11450

Full-time

The Center for Evolution & Medicine (CEM) at Arizona State University (ASU),
and Founding Director Dr. Randolph Nesse, invites applications for a
Postdoctoral Fellow in Education Resources Development and Online Learning.
Anticipated start date is August 1, 2016. This is a full-time (1.0) benefits
-eligible, fiscal year (July 1 - June 30) appointment. The successful
candidate will be co-mentored by Dr. Nesse, an expert in evolutionary
medicine and Dr. Sara Brownell, an expert in undergraduate biology
education research. As a physician, Dr. Nesse&rsquo;s research on the
neuro-endocrinology of anxiety led to a series of publications that
established the field of evolutionary medicine. Dr. Brownell is a
neuroscientist whose research is focused on biology education.

Renewal is possible on an annual basis contingent on satisfactory
performance, availability of resources, and the needs of the program. For
additional information and policies regarding postdoctoral scholars at ASU,
please see <a href="http://bit.ly/1AE42hI.">http://bit.ly/1AE42hI.</a> 

The successful candidate will help curate existing online resources in
evolutionary medicine, participate in the development and delivery of new
innovative evolutionary medicine courses, both in-person and online, and
will evaluate and study the impact and effectiveness of these resources and
courses. The Fellow will help develop a project plan to build courses that
will initially be offered to ASU and ASU online students and later to
medical personnel all over the world. This is an exciting opportunity to
work with an interdisciplinary faculty in a growing field of study to
develop, implement, and research innovative curriculum and other resources
related to evolution and medicine.

While specific technology expertise is not required, an ideal candidate
would demonstrate a strong interest in the applications of technology in
higher education, and would have an aptitude for developing and assessing
new approaches to learning and how they are best supported by new
technologies. This, together with experience in curriculum development and
teaching, will allow the Education Resources Fellow to provide disciplinary
and pedagogical expertise in creating innovative and rigorously assessed
learning experiences. The Fellow may have supervisory responsibilities that
include the training and mentoring of staff and/or student interns.

This Postdoctoral Fellows Program is a service and training program designed
for early career scientist-teachers who are interested in an academic
pathway involving curriculum development, teaching, and educational
programming in the interdisciplinary field of Evolutionary Medicine. Fellows
will receive mentorship to support their development as educators throughout
the duration of the fellowship and have an opportunity to participate in the
ASU Science Education Group, the Evidenced-based Teaching Seminar series, as
well as the Biology Education Research Lab.

The Center for Evolution & Medicine is a university-wide Presidential
Initiative whose mission is to establish evolutionary biology as an
essential basic science for medicine, worldwide. As an interdisciplinary
unit, and a member of ASU&rsquo;s BioDesign Institute, the postdoctoral fellow
will have multiple opportunities to work in collaboration with faculty from
a wide-range of disciplines. The Center brings leading scientists to ASU to
join existing faculty in research that demonstrates the power of
evolutionary biology to address problems in medicine and public health.

The School of Life Sciences (SOLS) is a creative hub for research and
teaching excellence at Arizona State University. Fast becoming a global
leader in life science education and discovery faculty, graduate students,
postdoctoral fellows and staff, work together to support an environment of
interdisciplinary research to inspire and transform life science students
by providing an innovative learning experience that prepares them to thrive
in a dynamic and demanding world. SOLS aims to improve life by stimulating
scientific discovery and solving critical problems at the intersection of
the life sciences and society.

By the start date, candidates must have completed a Ph.D. in anthropology,
biology or other natural science or science education field that provides an
extensive background in evolutionary biology. Cross-training and experience
in education is desirable. Candidates with experience in the following are
preferred: developing and assessing education resources; conducting
educational research; and using developing digital resources for online and
classroom use. The successful candidates will have a commitment to science
education, must have outstanding writing and organizational skills,
demonstrated capacity for independence and innovation, and the ability to
work as part of a team.

To apply, please email a single pdf document containing a letter of
application that states your interest in and qualifications for the
position, a curriculum vitae, names of three references, and an item of
choice that represents your approach to teaching in undergraduate science
classroom to evmedsearch@asu.edu. The initial closing date for receipt of
complete applications is January 15, 2016; applications will be reviewed
weekly thereafter until the search is closed. A background check is required
for employment.

Arizona State University is a new model for American higher education, an
unprecedented combination of academic excellence, entre preneurial energy
and broad access. This New American University is a single, unified
institution comprising four differentiated campuses positively impacting
the economic, social, cultural and environmental health of the communities
it serves. Its research is inspired by real world application blurring the
boundaries that traditionally separate academic disciplines. ASU serves
more than 80,000 students in metropolitan Phoenix, Arizona, the nation&rsquo;s
fifth largest city. ASU champions intellectual and cultural diversity, and
welcomes students from all fifty states and more than one hundred nations
across the globe.

Arizona State University is an equal opportunity/affirmative action
employer committed to excellence through diversity. All qualified
applicants will be considered without regard to race, color, sex,
religion, national origin, disability, protected veteran status, or
any other basis protected by law.  Women and minorities are encouraged
to apply (<a href="http://bit.ly/1OZMIzD">http://bit.ly/1OZMIzD</a>).  For more information on ASU&rsquo;s
non-discrimination policy, visit ACD 401. For additional information on
the Center for Evolution & Medicine, visit *http://bit.ly/1YqEbdN. For
additional information on the position, please contact Kathleen Holladay,
Assistant Director, at kholladay@asu.edu.


rmnesse@gmail.com


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135625712146">Job: UCalifornia_Merced.LabFieldTech.PopGenetics</a></div>
  <div class="description"><p>Applications are invited for a lab & field tech position in marine
molecular ecology (comparative population genetics / genomics).

The position entails (1) surveys and collection of marine intertidal
invertebrates in California, (2) all stages of wetlab analyses for
comparative phylogeographic analyses, and (2) preliminary summary analyses
of generated data.  The technician also is responsible for maintaining
a clean, orderly, functioning laboratory.

Full details of the position and application form are available at
<a href="http://bit.ly/1JnuY9F">http://bit.ly/1JnuY9F</a>

More information about the lab is available @ mnd.ucmerced.edu/

Michael N Dawson
Associate Professor
University of California, Merced


dawson.mn@gmail.com


via Gmail</p></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:00</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135556208001">Other: Namibia.VolFieldAssist.Baboons</a></div>
  <div class="description">Tsaobis Baboon Project 2016 - Volunteer Field Assistants

The Tsaobis Baboon Project is a long-term study of desert baboons in
Namibia.  Based at the Institute of Zoology (the research arm of the
Zoological Society of London), and affiliated with the Gobabeb Research
and Training Centre (Namibia), our aim is to carry out fundamental
research in behavioural ecology and population ecology, and to inform
conservation policy and practice for social species.

We are seeking to appoint four volunteer field assistants for periods
ranging from 3-5 months between 1 May and 30 September 2016. The
fieldwork will primarily involve daily follows of baboon troops on
foot, collecting data on the behaviour and ecology of individually
recognisable animals. Our research focus this year is on both baboon
cognition and collective behaviour.

Further information about the Tsaobis Baboon Project, the volunteer
field assistant positions, and the application procedure can be found
on the Project&rsquo;s webpages:

<a href="http://bit.ly/1jTKMml">http://bit.ly/1jTKMml</a>

The deadline for applications is 6pm Sunday 10 January 2016

Claudia Martina 
via Gmail</div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="index8713.html?q=aggregator/sources/14&amp;from=0">first page</a></div><div><a href="index2379.html?q=aggregator/sources/14&amp;from=200">previous page</a></div><div>... <a href="index7de1.html?q=aggregator/sources/14&amp;from=120">7</a> <a href="index2be7.html?q=aggregator/sources/14&amp;from=140">8</a> <a href="index5104.html?q=aggregator/sources/14&amp;from=160">9</a> <a href="index2b69.html?q=aggregator/sources/14&amp;from=180">10</a> <a href="index2379.html?q=aggregator/sources/14&amp;from=200">11</a> <strong>12</strong> <a href="index024a.html?q=aggregator/sources/14&amp;from=240">13</a> <a href="indexe9f7.html?q=aggregator/sources/14&amp;from=260">14</a> <a href="index9a9b.html?q=aggregator/sources/14&amp;from=280">15</a> </div><div><a href="index024a.html?q=aggregator/sources/14&amp;from=240">next page</a></div><div><a href="index9a9b.html?q=aggregator/sources/14&amp;from=280">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news." class="active">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator/sources/14&from=220 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 21:44:55 GMT -->
</html>
