<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 17:23:20 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexfe3a.html?q=node/279">Contact</a></li><li><a href="index5ffe.html?q=node/6">Join</a></li><li><a href="index94ca.html?q=node/449">Policy on Discrimination and Harassment at Society Meetings</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 5 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 3 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>March 3, 2016</h3>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a></div>
  <div class="description"><p>Molecular dating has become central to placing a temporal dimension on the tree of life. Methods for estimating divergence times have been developed for over 50 years, beginning with the proposal of molecular clock in 1962. We categorize the chronological development of these methods into four generations based on the timing of their origin. In the first generation approaches (1960s&ndash;1980s), a strict molecular clock was assumed to date divergences. In the second generation approaches (1990s), the equality of evolutionary rates between species was first tested and then a strict molecular clock applied to estimate divergence times. The third generation approaches (since ~2000) account for differences in evolutionary rates across the tree by using a statistical model, obviating the need to assume a clock or to test the equality of evolutionary rates among species. Bayesian methods in the third generation require a specific or uniform prior on the speciation-process and enable the inclusion of uncertainty in clock calibrations. The fourth generation approaches (since 2012) allow rates to vary from branch to branch, but do not need prior selection of a statistical model to describe the rate variation or the specification of speciation model. With high accuracy, comparable to Bayesian approaches, and speeds that are orders of magnitude faster, fourth generation methods are able to produce reliable timetrees of thousands of species using genome scale data. We found that early time estimates from second generation studies are similar to those of third and fourth generation studies, indicating that methodological advances have not fundamentally altered the timetree of life, but rather have facilitated time estimation by enabling the inclusion of more species. Nonetheless, we feel an urgent need for testing the accuracy and precision of third and fourth generation methods, including their robustness to misspecification of priors in the analysis of large phylogenies and data sets.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/870?rss=1">A Gene Encoding a DUF247 Domain Protein Cosegregates with the S Self-Incompatibility Locus in Perennial Ryegrass</a></div>
  <div class="description"><p>The grass family (Poaceae), the fourth largest family of flowering plants, encompasses the most economically important cereal, forage, and energy crops, and exhibits a unique gametophytic self-incompatibility (SI) mechanism that is controlled by at least two multiallelic and independent loci, <I>S</I> and <I>Z</I>. Despite intense research efforts over the last six decades, the genes underlying <I>S</I> and <I>Z</I> remain uncharacterized. Here, we report a fine-mapping approach to identify the male component of the <I>S</I>-locus in perennial ryegrass (<I>Lolium perenne</I> L.) and provide multiple evidence that a <I>domain of unknown function 247</I> (<I>DUF247</I>) gene is involved in its determination. Using a total of 10,177 individuals from seven different mapping populations segregating for <I>S</I>, we narrowed the <I>S-</I>locus to a genomic region containing eight genes, the closest recombinant marker mapping at a distance of 0.016 cM. Of the eight genes cosegregating with the <I>S</I>-locus, a highly polymorphic gene encoding for a protein containing a DUF247 was fully predictive of known <I>S</I>-locus genotypes at the amino acid level in the seven mapping populations. Strikingly, this gene showed a frameshift mutation in self-compatible darnel (<I>Lolium temulentum</I> L.), whereas all of the self-incompatible species of the <I>Festuca&ndash;Lolium</I> complex were predicted to encode functional proteins. Our results represent a major step forward toward understanding the gametophytic SI system in one of the most important plant families and will enable the identification of additional components interacting with the <I>S</I>-locus.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/885?rss=1">Evolutionary Paths That Expand Plasmid Host-Range: Implications for Spread of Antibiotic Resistance</a></div>
  <div class="description"><p>The World Health Organization has declared the emergence of antibiotic resistance to be a global threat to human health. Broad-host-range plasmids have a key role in causing this health crisis because they transfer multiple resistance genes to a wide range of bacteria. To limit the spread of antibiotic resistance, we need to gain insight into the mechanisms by which the host range of plasmids evolves. Although initially unstable plasmids have been shown to improve their persistence through evolution of the plasmid, the host, or both, the means by which this occurs are poorly understood. Here, we sought to identify the underlying genetic basis of expanded plasmid host-range and increased persistence of an antibiotic resistance plasmid using a combined experimental-modeling approach that included whole-genome resequencing, molecular genetics and a plasmid population dynamics model. In nine of the ten previously evolved clones, changes in host and plasmid each slightly improved plasmid persistence, but their combination resulted in a much larger improvement, which indicated positive epistasis. The only genetic change in the plasmid was the acquisition of a transposable element from a plasmid native to the <I>Pseudomonas</I> host used in these studies. The analysis of genetic deletions showed that the critical genes on this transposon encode a putative toxin&ndash;antitoxin (TA) and a cointegrate resolution system. As evolved plasmids were able to persist longer in multiple nai&#x0308;ve hosts, acquisition of this transposon also expanded the plasmid&rsquo;s host range, which has important implications for the spread of antibiotic resistance.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/898?rss=1">Emergence of Antimicrobial-Resistant Escherichia coli of Animal Origin Spreading in Humans</a></div>
  <div class="description"><p>In the context of the great concern about the impact of human activities on the environment, we studied 403 commensal <I>Escherichia coli/Escherichia</I> clade strains isolated from several animal and human populations that have variable contacts to one another. Multilocus sequence typing (MLST) showed a decrease of diversity 1) in strains isolated from animals that had an increasing contact with humans and 2) in all strains that had increased antimicrobial resistance. A specific B1 phylogroup clonal complex (CC87, Institut Pasteur schema nomenclature) of animal origin was identified and characterized as being responsible for the increased antimicrobial resistance prevalence observed in strains from the environments with a high human-mediated antimicrobial pressure. CC87 strains have a high capacity of acquiring and disseminating resistance genes with specific metabolic and genetic determinants as demonstrated by high-throughput sequencing and phenotyping. They are good mouse gut colonizers but are not virulent. Our data confirm the predominant role of human activities in the emergence of antimicrobial resistance in the environmental bacterial strains and unveil a particular <I>E. coli</I> clonal complex of animal origin capable of spreading antimicrobial resistance to other members of microbial communities.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/915?rss=1">Comparative Genomics of Candidate Phylum TM6 Suggests That Parasitism Is Widespread and Ancestral in This Lineage</a></div>
  <div class="description"><p>Candidate phylum TM6 is a major bacterial lineage recognized through culture-independent rRNA surveys to be low abundance members in a wide range of habitats; however, they are poorly characterized due to a lack of pure culture representatives. Two recent genomic studies of TM6 bacteria revealed small genomes and limited gene repertoire, consistent with known or inferred dependence on eukaryotic hosts for their metabolic needs. Here, we obtained additional near-complete genomes of TM6 populations from agricultural soil and upflow anaerobic sludge blanket reactor metagenomes which, together with the two publicly available TM6 genomes, represent seven distinct family level lineages in the TM6 phylum. Genome-based phylogenetic analysis confirms that TM6 is an independent phylum level lineage in the bacterial domain, possibly affiliated with the Patescibacteria superphylum. All seven genomes are small (1.0&ndash;1.5 Mb) and lack complete biosynthetic pathways for various essential cellular building blocks including amino acids, lipids, and nucleotides. These and other features identified in the TM6 genomes such as a degenerated cell envelope, ATP/ADP translocases for parasitizing host ATP pools, and protein motifs to facilitate eukaryotic host interactions indicate that parasitism is widespread in this phylum. Phylogenetic analysis of ATP/ADP translocase genes suggests that the ancestral TM6 lineage was also parasitic. We propose the name Dependentiae (phyl. nov.) to reflect dependence of TM6 bacteria on host organisms.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/928?rss=1">The Time Scale of Recombination Rate Evolution in Great Apes</a></div>
  <div class="description"><p>We present three linkage-disequilibrium (LD)-based recombination maps generated using whole-genome sequence data from 10 Nigerian chimpanzees, 13 bonobos, and 15 western gorillas, collected as part of the Great Ape Genome Project (Prado-Martinez J, et al. 2013. Great ape genetic diversity and population history. <I>Nature</I> 499:471&ndash;475). We also identified species-specific recombination hotspots in each group using a modified LDhot framework, which greatly improves statistical power to detect hotspots at varying strengths. We show that fewer hotspots are shared among chimpanzee subspecies than within human populations, further narrowing the time scale of complete hotspot turnover. Further, using species-specific PRDM9 sequences to predict potential binding sites (PBS), we show higher predicted PRDM9 binding in recombination hotspots as compared to matched cold spot regions in multiple great ape species, including at least one chimpanzee subspecies. We found that correlations between broad-scale recombination rates decline more rapidly than nucleotide divergence between species. We also compared the skew of recombination rates at centromeres and telomeres between species and show a skew from chromosome means extending as far as 10&ndash;15 Mb from chromosome ends. Further, we examined broad-scale recombination rate changes near a translocation in gorillas and found minimal differences as compared to other great ape species perhaps because the coordinates relative to the chromosome ends were unaffected. Finally, on the basis of multiple linear regression analysis, we found that various correlates of recombination rate persist throughout the African great apes including repeats, diversity, and divergence. Our study is the first to analyze within- and between-species genome-wide recombination rate variation in several close relatives.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/946?rss=1">Long-Distance Dispersal Shaped Patterns of Human Genetic Diversity in Eurasia</a></div>
  <div class="description"><p>Most previous attempts at reconstructing the past history of human populations did not explicitly take geography into account or considered very simple scenarios of migration and ignored environmental information. However, it is likely that the last glacial maximum (LGM) affected the demography and the range of many species, including our own. Moreover, long-distance dispersal (LDD) may have been an important component of human migrations, allowing fast colonization of new territories and preserving high levels of genetic diversity. Here, we use a high-quality microsatellite data set genotyped in 22 populations to estimate the posterior probabilities of several scenarios for the settlement of the Old World by modern humans. We considered models ranging from a simple spatial expansion to others including LDD and a LGM-induced range contraction, as well as Neolithic demographic expansions. We find that scenarios with LDD are much better supported by data than models without LDD. Nevertheless, we show evidence that LDD events to empty habitats were strongly prevented during the settlement of Eurasia. This unexpected absence of LDD ahead of the colonization wave front could have been caused by an Allee effect, either due to intrinsic causes such as an inbreeding depression built during the expansion or due to extrinsic causes such as direct competition with archaic humans. Overall, our results suggest only a relatively limited effect of the LGM contraction on current patterns of human diversity. This is in clear contrast with the major role of LDD migrations, which have potentially contributed to the intermingled genetic structure of Eurasian populations.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/959?rss=1">Comparative Genomics of Early-Diverging Mushroom-Forming Fungi Provides Insights into the Origins of Lignocellulose Decay Capabilities</a></div>
  <div class="description"><p>Evolution of lignocellulose decomposition was one of the most ecologically important innovations in fungi. White-rot fungi in the Agaricomycetes (mushrooms and relatives) are the most effective microorganisms in degrading both cellulose and lignin components of woody plant cell walls (PCW). However, the precise evolutionary origins of lignocellulose decomposition are poorly understood, largely because certain early-diverging clades of Agaricomycetes and its sister group, the Dacrymycetes, have yet to be sampled, or have been undersampled, in comparative genomic studies. Here, we present new genome sequences of ten saprotrophic fungi, including members of the Dacrymycetes and early-diverging clades of Agaricomycetes (Cantharellales, Sebacinales, Auriculariales, and Trechisporales), which we use to refine the origins and evolutionary history of the enzymatic toolkit of lignocellulose decomposition. We reconstructed the origin of ligninolytic enzymes, focusing on class II peroxidases (AA2), as well as enzymes that attack crystalline cellulose. Despite previous reports of white rot appearing as early as the Dacrymycetes, our results suggest that white-rot fungi evolved later in the Agaricomycetes, with the first class II peroxidases reconstructed in the ancestor of the Auriculariales and residual Agaricomycetes. The exemplars of the most ancient clades of Agaricomycetes that we sampled all lack class II peroxidases, and are thus concluded to use a combination of plesiomorphic and derived PCW degrading enzymes that predate the evolution of white rot.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/971?rss=1">Evolution of a Catalytic Mechanism</a></div>
  <div class="description"><p>The means by which superfamilies of specialized enzymes arise by gene duplication and functional divergence are poorly understood. The escape from adaptive conflict hypothesis, which posits multiple copies of a gene encoding a primitive inefficient and highly promiscuous generalist ancestor, receives support from experiments showing that resurrected ancestral enzymes are indeed more substrate-promiscuous than their modern descendants. Here, we provide evidence in support of an alternative model, the innovation&ndash;amplification&ndash;divergence hypothesis, which posits a single-copied ancestor as efficient and specific as any modern enzyme. We argue that the catalytic mechanisms of plant esterases and descendent acetone cyanohydrin lyases are incompatible with each other (e.g., the reactive substrate carbonyl must bind in opposite orientations in the active site). We then show that resurrected ancestral plant esterases are as catalytically specific as modern esterases, that the ancestor of modern acetone cyanohydrin lyases was itself only very weakly promiscuous, and that improvements in lyase activity came at the expense of esterase activity. These observations support the innovation&ndash;amplification&ndash;divergence hypothesis, in which an ancestor gains a weak promiscuous activity that is improved by selection at the expense of the ancestral activity, and not the escape from adaptive conflict in which an inefficient generalist ancestral enzyme steadily loses promiscuity throughout the transition to a highly active specialized modern enzyme.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/980?rss=1">Evolutionary Origins of Rhizarian Parasites</a></div>
  <div class="description"><p>The SAR group (Stramenopila, Alveolata, Rhizaria) is one of the largest clades in the tree of eukaryotes and includes a great number of parasitic lineages. Rhizarian parasites are obligate and have devastating effects on commercially important plants and animals but despite this fact, our knowledge of their biology and evolution is limited. Here, we present rhizarian transcriptomes from all major parasitic lineages in order to elucidate their evolutionary relationships using a phylogenomic approach. Our results suggest that Ascetosporea, parasites of marine invertebrates, are sister to the novel clade Apofilosa. The phytomyxean plant parasites branch sister to the vampyrellid algal ectoparasites in the novel clade Phytorhiza. They also show that Ascetosporea + Apofilosa + Retaria + Filosa + Phytorhiza form a monophyletic clade, although the branching pattern within this clade is difficult to resolve and appears to be model-dependent. Our study does not support the monophyly of the rhizarian parasitic lineages (Endomyxa), suggesting independent origins for rhizarian animal and plant parasites.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/984?rss=1">Strong Maternal Effects on Gene Expression in Arabidopsis lyrata Hybrids</a></div>
  <div class="description"><p>Hybridization between populations or species can have pronounced fitness consequences. Yet little is known about how hybridization affects gene regulation. Three main models have been put forward to explain gene expression patterns in hybrids: additive, dominance, or parental effects. Here, we use high throughput RNA-sequencing to examine the extent to which hybrid gene expression follows predictions by each of the three models. We performed a reciprocal crossing experiment between two differentiated populations of the perennial herb <I>Arabidopsis lyrata</I> and sequenced RNA in rosette leaves of 12-week-old plants grown in greenhouse conditions. The two parental populations had highly differentiated gene expression patterns. In hybrids, a majority of genes showed intermediate expression relative to that of their parental populations (i.e., additive effects), but expression was frequently more similar to the maternal than to their paternal population (i.e., maternal effects). Allele-specific expression analyses showed that in the vast majority of cases, genes with pronounced maternal effect expressed both the maternal and the paternal allele. Maternal effects on hybrid gene expression have rarely been documented previously and our study suggests it could be more common than previously assumed. Whether the maternal effect on gene expression persists to later life-stages, and whether the variation in gene expression is manifested in other aspects of the phenotype, remain to be elucidated. Our findings are relevant for understanding the consequences of outbreeding and hybridization and open up several questions for future studies.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/995?rss=1">Selection on Mitochondrial Variants Occurs between and within Individuals in an Expanding Invasion</a></div>
  <div class="description"><p>Mitochondria are critical for life, yet their underlying evolutionary biology is poorly understood. In particular, little is known about interaction between two levels of evolution: between individuals and within individuals (competition between cells, mitochondria or mitochondrial DNA molecules). Rapid evolution is suspected to occur frequently in mitochondrial DNA, whose maternal inheritance predisposes advantageous mutations to sweep rapidly though populations. Rapid evolution is also predicted in response to changed selection regimes after species invasion or removal of pathogens or competitors. Here, using empirical and simulated data from a model invasive bird species, we provide the first demonstration of rapid selection on the mitochondrial genome within individuals in the wild. Further, we show differences in mitochondrial DNA copy number associated with competing genetic variants, which may provide a mechanism for selection. We provide evidence for three rarely documented phenomena: selection associated with mitochondrial DNA abundance, selection on the mitochondrial control region, and contemporary selection during invasion.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1008?rss=1">Accelerated Evolution of Enhancer Hotspots in the Mammal Ancestor</a></div>
  <div class="description"><p>Mammals have evolved remarkably different sensory, reproductive, metabolic, and skeletal systems. To explore the genetic basis for these differences, we developed a comparative genomics approach to scan whole-genome multiple sequence alignments to identify regions that evolved rapidly in an ancestral lineage but are conserved within extant species. This pattern suggests that ancestral changes in function were maintained in descendants. After applying this test to therian mammals, we identified 4,797 accelerated regions, many of which are noncoding and located near developmental transcription factors. We then used mouse transgenic reporter assays to test if noncoding accelerated regions are enhancers and to determine how therian-specific substitutions affect their activity in vivo. We discovered enhancers with expression specific to the therian version in brain regions involved in the hormonal control of milk ejection, uterine contractions, blood pressure, temperature, and visual processing. This work underscores the idea that changes in developmental gene expression are important for mammalian evolution, and it pinpoints candidate genes for unique aspects of mammalian biology.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1019?rss=1">Evolutionary Transition of Promoter and Gene Body DNA Methylation across Invertebrate-Vertebrate Boundary</a></div>
  <div class="description"><p>Genomes of invertebrates and vertebrates exhibit highly divergent patterns of DNA methylation. Invertebrate genomes tend to be sparsely methylated, and DNA methylation is mostly targeted to a subset of transcription units (gene bodies). In a drastic contrast, vertebrate genomes are generally globally and heavily methylated, punctuated by the limited local hypo-methylation of putative regulatory regions such as promoters. These genomic differences also translate into functional differences in DNA methylation and gene regulation. Although promoter DNA methylation is an important regulatory component of vertebrate gene expression, its role in invertebrate gene regulation has been little explored. Instead, gene body DNA methylation is associated with expression of invertebrate genes. However, the evolutionary steps leading to the differentiation of invertebrate and vertebrate genomic DNA methylation remain unresolved. Here we analyzed experimentally determined DNA methylation maps of several species across the invertebrate&ndash;vertebrate boundary, to elucidate how vertebrate gene methylation has evolved. We show that, in contrast to the prevailing idea, a substantial number of promoters in an invertebrate basal chordate <I>Ciona intestinalis</I> are methylated. Moreover, gene expression data indicate significant, epigenomic context-dependent associations between promoter methylation and expression in <I>C. intestinalis</I>. However, there is no evidence that promoter methylation in invertebrate chordate has been evolutionarily maintained across the invertebrate&ndash;vertebrate boundary. Rather, body-methylated invertebrate genes preferentially obtain hypo-methylated promoters among vertebrates. Conversely, promoter methylation is preferentially found in lineage- and tissue-specific vertebrate genes. These results provide important insights into the evolutionary origin of epigenetic regulation of vertebrate gene expression.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1029?rss=1">Euarchontan Opsin Variation Brings New Focus to Primate Origins</a></div>
  <div class="description"><p>Debate on the adaptive origins of primates has long focused on the functional ecology of the primate visual system. For example, it is hypothesized that variable expression of short- (SWS1) and middle-to-long-wavelength sensitive (M/LWS) opsins, which confer color vision, can be used to infer ancestral activity patterns and therefore selective ecological pressures. A problem with this approach is that opsin gene variation is incompletely known in the grandorder Euarchonta, that is, the orders Scandentia (treeshrews), Dermoptera (colugos), and Primates. The ancestral state of primate color vision is therefore uncertain. Here, we report on the genes (<I>OPN1SW</I> and <I>OPN1LW</I>) that encode SWS1 and M/LWS opsins in seven species of treeshrew, including the sole nocturnal scandentian <I>Ptilocercus lowii</I>. In addition, we examined the opsin genes of the Central American woolly opossum (<I>Caluromys derbianus</I>), an enduring ecological analogue in the debate on primate origins. Our results indicate: 1) retention of ultraviolet (UV) visual sensitivity in <I>C. derbianus</I> and a shift from UV to blue spectral sensitivities at the base of Euarchonta; 2) ancient pseudogenization of <I>OPN1SW</I> in the ancestors of <I>P. lowii</I>, but a signature of purifying selection in those of <I>C. derbianus</I>; and, 3) the absence of <I>OPN1LW</I> polymorphism among diurnal treeshrews. These findings suggest functional variation in the color vision of nocturnal mammals and a distinctive visual ecology of early primates, perhaps one that demanded greater spatial resolution under light levels that could support cone-mediated color discrimination.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1042?rss=1">Laterally Transferred Gene Recruited as a Venom in Parasitoid Wasps</a></div>
  <div class="description"><p>Parasitoid wasps use venom to manipulate the immunity and metabolism of their host insects in a variety of ways to provide resources for their offspring. Yet, how genes are recruited and evolve to perform venom functions remain open questions. A recently recognized source of eukaryotic genome innovation is lateral gene transfer (LGT). Glycoside hydrolase family 19 (GH19) chitinases are widespread in bacteria, microsporidia, and plants where they are used in nutrient acquisition or defense, but have previously not been known in metazoans. In this study, a GH19 chitinase LGT is described from the unicellular microsporidia/<I>Rozella</I> clade into parasitoid wasps of the superfamily Chalcidoidea, where it has become recruited as a venom protein. The GH19 chitinase is present in 15 species of chalcidoid wasps representing four families, and phylogenetic analysis indicates that it was laterally transferred near or before the origin of Chalcidoidea (~95 Ma). The GH19 chitinase gene is highly expressed in the venom gland of at least seven species, indicating a role in the complex host manipulations performed by parasitoid wasp venom. RNAi knockdown in the model parasitoid <I>Nasonia vitripennis</I> reveals that&mdash;following envenomation&mdash;the GH19 chitinase induces fly hosts to upregulate genes involved in an immune response to fungi. A second, independent LGT of GH19 chitinase from microsporidia into mosquitoes was also found, also supported by phylogenetic reconstructions. Besides these two LGT events, GH19 chitinase is not found in any other sequenced animal genome, or in any fungi outside the microsporidia/<I>Rozella</I> clade.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div> </div><div> </div><div><strong>1</strong> <a href="index8503.html?q=aggregator&amp;from=20">2</a> <a href="index6390.html?q=aggregator&amp;from=40">3</a> <a href="index4543.html?q=aggregator&amp;from=60">4</a> <a href="index798e.html?q=aggregator&amp;from=80">5</a> <a href="index0e96.html?q=aggregator&amp;from=100">6</a> <a href="index3362.html?q=aggregator&amp;from=120">7</a> <a href="index7b13.html?q=aggregator&amp;from=140">8</a> <a href="indexf37c.html?q=aggregator&amp;from=160">9</a> ...</div><div><a href="index8503.html?q=aggregator&amp;from=20">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 17:23:20 GMT -->
</html>
