<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=420 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:37:49 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="indexacd7.html?q=node/208">56(4) August 2007</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 7 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>December 17, 2015</h3>
<div class="news-item">
 <div class="date">00:29</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135369745891">Conference: Marseilles.EvolBiol.Sep20-23.DeadlineJan31</a></div>
  <div class="description">Dear All

The early dead line for the,20th Evolutionary Biology Meeting at
Marseilles (September 20-23 2016), is January 31 2016



info: <a href="http://bit.ly/15fa2ML">http://bit.ly/15fa2ML</a> or
<a href="http://bit.ly/1mcBgPS">http://bit.ly/1mcBgPS</a>

best regards
Pierre 

PONTAROTTI Pierre 
via Gmail </div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 16, 2015</h3>
<div class="news-item">
 <div class="date">16:30</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2015/12/is-ring-of-life-network.html">Is the Ring of Life a network?</a></div>
  <div class="description"><br />Ten years ago, Rivera and Lake decided to emphasize the series if genome fusions that seem to have been involved in the origin of the major phylogenetic groups by calling it the ring of Life rather than the Tree of Life:<br />Maria C. Rivera and James A. Lake. 2004. The Ring of Life provides evidence for a genome fusion origin of eukaryotes. <a href="http://www.nature.com/nature/journal/v431/n7005/abs/nature02848.html" target="_blank">Nature</a> 431: 182-185).<a href="http://4.bp.blogspot.com/-TWxuupgcDOk/VnBvwbOVfmI/AAAAAAAAD2g/ab8BTzVc9HQ/s1600/Ring.gif" imageanchor="1"></a><br />This terminology has been repeated in a number of subsequent papers, including:<br />James McInerney, Davide Pisani and Mary J. O'Connell (2015) The Ring of Life hypothesis for eukaryote origins is supported by multiple kinds of data. <a href="http://rstb.royalsocietypublishing.org/content/370/1678/20140323" target="_blank">Philosophical Transactions of the Royal Society of London B</a> 370: 20140323.However, life is not that simple, and it has more recently become accepted that a set of inter-connected rings is involved in the metaphor, rather than the simple ring originally presented. Thus we now have the plural Rings of Life, instead.<br /><br /><a href="http://2.bp.blogspot.com/-v9PB6ibGmV0/VnBvwS3P66I/AAAAAAAAD2s/5Ycr9w1PiuA/s1600/Ring2.gif" imageanchor="1"></a>James A. Lake and Janet S. Sinsheimer (2013) The deep roots of the Rings of Life. <a href="http://gbe.oxfordjournals.org/content/5/12/2440.abstract" target="_blank">Genome Biology and Evolution</a> 5: 2440-2448.James A. Lake, Joseph Larsen, Brooke Sarna, Rafael R. de la Haba, Yiyi Pu, HyunMin Koo, Jun Zhao and Janet S. Sinsheimer (2016) Rings reconcile genotypic and phenotypic evolution within the Proteobacteria. <a href="http://gbe.oxfordjournals.org/content/early/2015/12/09/gbe.evv221.abstract" target="_blank">Genome Biology and Evolution</a> (in press).I think that the rest of us would still call each of these diagrams a network. Indeed, most of the metaphors that have been used over the years can also be called a network (see <a href="http://phylonetworks.blogspot.com/2012/09/metaphors-for-evolutionary-relationships.html" target="_blank">Metaphors for evolutionary relationships</a>).<br /><br /></div>
  <div class="source">Source: <a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:09</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135307225791">Postdoc:UFlorida_WhitneyLab.CtenoEvoGeno</a></div>
  <div class="description"><p>POSTDOCTORAL POSITION
DEEPC: Diversity, Evolution, and EcoPhysiology of Ctenophores

An NSF-funded postdoctoral research position is currently available in the
Ryan group at the Whitney Laboratory for Marine Bioscience in Saint
Augustine, Florida (part of the University of Florida). The position is
part of the NSF-funded Diversity, Evolution, and EcoPhysiology of
Ctenophores (DEEPC) project. The goal of the project is to understand
evolution and diversification of Ctenophora using cutting-edge genomic
analyses and the deep-sea habitat as the generating force of novel
adaptations. We are collecting and generating transcriptomes and genomes
from many ctenophore species with habitat depths down to 4000 meters
below the surface.  The postdoctoral associate will use these molecular
data in a phylogenetic context to identify specific genetic events that
underlie physiological tolerances and adaptations to the unique
selective pressures of life in the deep sea.

Candidates should have or be close to obtaining a Ph.D. in bioinformatics,
computational biology, genetics, evolutionary biology,
ecology, or a closely related field. Skills in programming, phylogenetics,
statistical genetics, and/or the application of computational methods to
genomic data are desired. Familiarity with detecting episodic diversifying
selection using techniques such as branch-site models is highly desirable.
Candidates from groups that are traditionally underrepresented in the
sciences are strongly encouraged to apply.

Interested applicants should submit a curriculum vitae, a detailed letter
of interest, and the names of three potential referees to Joseph
Ryan at joseph.ryan@whitney.ufl.edu by January 1, 2016.

Ryan Group:  <a href="http://bit.ly/1Nwz8xJ">http://bit.ly/1Nwz8xJ</a>
Whitney Lab: <a href="http://bit.ly/1I8dvah">http://bit.ly/1I8dvah</a>








via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:09</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135305123456">Other: DryadLab.TeachingEvolution</a></div>
  <div class="description">&ldquo;DryadLab - Using real data to engage students in scientific inquiry&rdquo;: a
QUBES Faculty Mentoring Network

Spring 2016

Apply by January 1, 2016

DryadLab and Quantitative Undergraduate Biology Education Synthesis (QUBES)
are pleased to offer a unique networking and professional development
opportunity  from January V April 2016 for faculty interested in bringing
authentic research data into undergraduate classrooms.  DryadLab is an
educational extension of the Dryad Digital Repository, which contains
freely available research data from thousands of published scientific
papers. DryadLab provides open, high-quality, hands-on, data-driven
educational modules suitable for use in undergraduate classrooms, developed
by the authors of the original publications in collaboration with
experienced educators. Over the course of this network, faculty will
implement selected DryadLab  modules
in their courses with a focus on providing students with quantitative
skills.

QUBES (<a href="https://qubeshub.org/">https://qubeshub.org</a>) is an NSF-funded virtual center that
supports collaboration to promote quantitative biology education at the
undergraduate level.

Benefits of Participation

   -

   Access to ready-to-use teaching modules. Participants will incorporate
   at least 1 of the selected DryadLab modules in Spring 2016 classes
   -

   Access to peer mentors for effective teaching tips and strategies in
   small group virtual meetings every two weeks
   -

   Letter sent from QUBES to your department chair recognizing your
   contribution to teaching scholarship


Please visit <a href="http://bit.ly/1JbcjOf">http://bit.ly/1JbcjOf</a> for more details and
to apply.

Questions? Contact Sam Swauger (samswauger@datadryad.org) or Alison Hale (
anm116@pitt.edu).

via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 15, 2015</h3>
<div class="news-item">
 <div class="date">23:53</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135304481586">Graduate position: SystemsPharmacol.UTromso</a></div>
  <div class="description"><p>PhD student in Systems Pharmacology (Mathematical Modelling)

———

A 4-year PhD student position is available immediately in the Systems Pharmacology group of Dr. Pia Abel zur Wiesch. The position is affiliated with the Centre for Molecular Medicine Norway (NCMM), Nordic EMBL Partnership Molecular Medicine and is located at the University of Tromso, Norway. While the primary location is in Tromso, the candidates have the option to visit Yale School of Public Health. The candidate is expected to spend substantial time at ETH Zurich, Switzerland.
——

About the group

The Systems-Pharmacology group (<a href="http://bit.ly/1NSEyH2">http://bit.ly/1NSEyH2</a>) is headed by Dr. Abel zur Wiesch, Tenure Track Associate Professor (seniorforsker) at UiT, NCMM Young Associate Investigator (<a href="http://bit.ly/1YjuSXt">http://bit.ly/1YjuSXt</a>) and Visiting Assistant Professor at Yale School of Public Health. The aim of our work is to improve therapy in infectious diseases and cancer by using mathematical models to predict how much drug a patient should receive and how long and how often the patient should be treated. Our mathematical models describe both the intracellular reaction kinetics of drug-target binding as well as growth and death of populations of bacterial or cancer cells (Abel zur Wiesch & al., Science Transl. Med. 2015). We collaborate closely with experimentalists and clinicians. Two examples of ongoing projects are:

- link single cell analyses (John McKinney, EPFL Lausanne, <a href="http://bit.ly/1NSEwyS">http://bit.ly/1NSEwyS</a>) to clinical studies (<a href="http://bit.ly/1YjuSXv">http://bit.ly/1YjuSXv</a> to improve TB therapy, funded by the Bill & Melinda Gates foundation 

- develop new extended release technologies together with gastroenterologists at Harvard (Giovanni Traverso, <a href="http://bit.ly/1NSEyH4">http://bit.ly/1NSEyH4</a>) and MIT (Andrew Bellinger <a href="http://bit.ly/1YjuSXw">http://bit.ly/1YjuSXw</a>). 

The most important collaborator for the PhD projects is Dr. Roland Regoes (ETH Zurich, <a href="http://bit.ly/1NSEwyU">http://bit.ly/1NSEwyU</a>). The projects will combine mathematical modeling with the analysis of clinical and experimental data. Both the Systems Pharmacology group and the Regoes group at ETH have documented expertise in all of these areas (Abel zur Wiesch & al., Science Transl. Medicine 2015, Abel zur Wiesch & al., Lancet Infectious Diseases 2011, Abel, Abel zur Wiesch & al., Nature Methods 2015, Diard & al., Nature 2013, Kaiser, Regoes & al., PLOS Biology, Regoes & al. AAC 2004). The project therefore offers a unique opportunity to get experience in a wide range of interesting research topics.
——

Your profile

We seek candidates that appreciate working in an international and interactive environment, are highly enthusiastic about basic research with medical applicability, motivated and willing to integrate in a young team of researchers. You should have solid programming skills and a strong quantitative background and be passionate about applying your skills to biological and medical questions. A strong interest in Public Health, Pharmacology, Infectious Diseases, Biochemistry and/or Mathematical Biology is desirable. Previous experience with mathematical modeling and/or analysis of large datasets is an advantage. Good command of English and scientific writing skills are required. Applicants must hold a Master&rsquo;s degree (or equivalent).
——

We offer

The successful candidate will work collaboratively with other faculty, scientists, and students in a dynamic and multi-disciplinary environment. We offer the unique opportunity of being embedded in several scientific environments at the UiT in Tromso, the Centre for Molecular Medicine Norway (NCMM) in Oslo and Yale School of Public Health (YSPH) in Boston. We strongly encourage career development beyond authorship of research manuscripts arising from this work. We will help the successful candidate in developing their own interests and also encourage attending international conferences and visiting collaborators. The salary will be commensurate with experience and in accordance with the Norwegian State salary scale code 1017 (starting approx. at EURO 46’500/ NOK 429’700 per year). UiT is a family friendly work environment and offers very attractive and internationally competitive benefits including pension arrangements, healthcare and other welfare benefits.
——

How to apply

The application deadline is January 12 2016 and must be submitted in English. The following documents are required:

* 1-2 page cover letter explaining your motivation and research interests
* CV containing an overview of education, supervised professional training and professional work
* A complete list of publications 
* Copies of diploma and transcript from completed degree(s) (translated to English or a Scandinavian language; certified copies must only be provided upon request)
* 2-3 reference letters (can be sent directly to Dr. Abel zur Wiesch or uploaded at jobbnorge.no)
* Documentation of English language proficiency may be required upon request
* A comparison between the candidate’s grades and the average grade of students in the same program may be requested later in the application process.

To apply please follow the instructions on the official University of Tromso recruiting homepage.

<a href="http://bit.ly/1YjuSXx">http://bit.ly/1YjuSXx</a>

For more information, please visit our webpage at 

<a href="http://bit.ly/1NSEyH7">http://bit.ly/1NSEyH7</a>

or

<a href="http://bit.ly/1YjuSXt">http://bit.ly/1YjuSXt</a>

Location:
Tromso, Norway (NO)
Zurich, Switzerland
New Haven, CT, USA

Salary band:
$50,000 - $74,999

Salary
starting at NOK 429’700 per year

Fields
Life Sciences, Biochemistry, Bioinformatics, Biomedical Sciences, Computational Biology, Evolutionary Biology, Microbiology, Pharmacokinetics / Pharmacodynamics, Pharmacology, Physical Sciences, Chemistry, Computer Sciences, Informatics, Mathematics, Health Sciences, Drug Development, Drug Discovery

Description
Full time position

PhD Fellowship
PhD Studentship

Date
Deadline January 12


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">17:00</div>
 <div class="body">
  <div class="title"><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a></div>
  <div class="description">Background:
Immune response induction benefits insects in combatting infection by pathogens. However, organisms have a limited amount of resources available and face the dilemma of partitioning resources between immunity and other life-history traits. Since males and females differ in their life histories, sex-specific resource investment strategies to achieve an optimal immune response following an infection can be expected. We investigated immune response induction of females and males of Heliothis virescens in response to the entomopathogenic bacterium Serratia entomophila, and its effects on mating success and the female sexual signal.
Results:
We found that females had higher expression levels of immune-related genes after bacterial challenge than males. However, males maintained a higher baseline expression of immune-related genes than females. The increased investment in immunity of female moths was negatively correlated with mating success and the female sexual signal. Male mating success was unaffected by bacterial challenge.
Conclusions:
Our results show that the sexes differed in their investment strategies: females invested in immune defense after a bacterial challenge, indicating facultative immune deployment, whereas males had higher baseline immunity than females, indicating immune maintenance. Interestingly, these differences in investment were reflected in the mate choice assays. As female moths are the sexual signallers, females need to invest resources in their attractiveness. However, female moths appeared to invest in immunity at the cost of reproductive effort.</div>
  <div class="source">Source: <a href="indexfd51.html?q=aggregator/sources/2">BMC Evolutionary Biology</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:07</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135241563976">Postdoc: Trondheim_Norway.EvolutionaryGenomics</a></div>
  <div class="description">A two-year postdoctoral position in evolutionary genomics funded by
a grant from the Norwegian Research Council to Assistant Professor
Arild Husby is available at the Centre for Biodiversity Dynamics (CBD),
Department of Biology,  NTNU, Norway.

Job description
The aim of the project is to examine ecological adaptation at the
molecular genetic level using already available SNP genotype data
from 4000 individuals genotyped on a 200k SNP array. The data has been
collected in a long-term study of a house sparrow model system consisting
of insular populations at the coast of Norway, with extensive pedigrees
and individual-level data on phenotypic trait values and fitness. The
two main goals of the post doc holder will be to use the SNP data to
test for signs of selection on the genomic level. This work will be
combined with ongoing GWAS work to link signs of selection in the genome
to the genetic variants underlying phenotypic traits, thereby providing
a link between population genomic approaches and quantitative genetic
work. The second aim is to use recently developed statistical methods
(Ovaskainen et al. 2011, Genetics 109), Berg & Coop 2014, PLoS Genet 10)
to test for polygenic adaption examining correlated changes in allele
frequencies in traits known to be under divergent selection on different
islands (Holand et al. 2011, Genet Res 93). Hence, the overall aim of the
project is to provide insights into the molecular genetic basis of trait
variation in natural populations. The successful candidate will have the
possibility to develop his/her own ideas to approach the above issues,
including opportunities for fieldwork and laboratory work.

The work will be carried out primarily in the research group of Assistant
Professor Arild Husby, but will be in close collaboration with Associate
Professor Henrik Jensen, and Professor Otso Ovaskainen. Ass. Prof. Husby
and Prof. Ovaskainen are jointly based at the Centre for Biodiversity
Dynamics (CBD), NTNU, and the Metapopulation Research Centre (MRC),
University of Helsinki, whereas Ass. Prof. Jensen is based at CBD. Both
CBD and MRC are Centres of excellence in Norway and Finland, respectively.

More information can be found here: www.ntnu.edu/cbd, and here:
<a href="http://bit.ly/1P4qzwV.">http://bit.ly/1P4qzwV.</a>

The candidate will be employed at CBD (Department of Biology, NTNU)
but will be expected to spend considerable time at MRC. The working
language at both institutions is English.

For further information about the postdoctoral position please contact
Assistant Professor Arild Husby (Arild.Husby@helsinki.fi, phone
no. +358294157691). Informal enquiries encouraged!

Required qualifications and personal qualities
Candidates must hold a PhD in biology and be able to document expertise
in statistical/quantitative genetics and population genomics. Experience
with handling data from next-generation sequencing and/or high-density SNP
genotyping techniques and statistical analyses of such data is strongly
desired and will be considered an advantage. The successful candidate
must be able to work efficiently and goal-oriented both when working
independently and when collaborating with others. We are looking for a
highly motivated and enthusiastic candidate.

What we offer
We can offer an exciting postdoctoral research project with focus on
fundamental questions in evolutionary biology using state-of-the-art
molecular genetic analyses. The postdoctoral researcher will carry
out scientifically challenging research with good opportunities
for development of her/his expertise. The successful candidate will
benefit from a good working environment and the expertise of members
in our research groups which span quantitative genetics, statistical
genetics/genomics and mathematical modelling.

Preferred starting date is in February 2016 but this is negotiable.The
appointment of the Postdoc will be made according to Norwegian guidelines
for universities and university colleges and to the general regulations
regarding university employees. NTNUs personnel policy objective is
that the staff must reflect the composition of the population to the
greatest possible extent.

The position as Postdoc is remunerated according to the Norwegian State
salary scale. There is a 2% deduction for superannuation contribution.

Application
The application with CV, pdfs of publications, reference letters and
relevant certificates must be submitted electronically through this
page. Applications submitted elsewhere will not be considered.  Please
indicate preferred starting date in your cover letter.  The reference
number of the position is: NT 84/15

Deadline for applying: January 15, 2016

Apply through this link:
<a href="http://bit.ly/1O6omnB">http://bit.ly/1O6omnB</a>


Arild Husby, Assistant Professor
Department of Biosciences (Biocenter 3, office 5415),
University of Helsinki
PO box 65,
FI-00014 Helsinki,
Finland

web: <a href="http://bit.ly/11XzFaY">http://bit.ly/11XzFaY</a><a href="http://www.ntnu.edu/cbd,">www.ntnu.edu/cbd,</a> and here:
http://bit.ly/1P4qzwV.

The candidate will be employed at CBD (Department of Biology, NTNU)
but will be expected to spend considerable time at MRC. The working
language at both institutions is English.

For further information about the postdoctoral position please contact
Assistant Professor Arild Husby (Arild.Husby@helsinki.fi, phone
no. +358294157691). Informal enquiries encouraged!

Required qualifications and personal qualities
Candidates must hold a PhD in biology and be able to document expertise
in statistical/quantitative genetics and population genomics. Experience
with handling data from next-generation sequencing and/or high-density SNP
genotyping techniques and statistical analyses of such data is strongly
desired and will be considered an advantage. The successful candidate
must be able to work efficiently and goal-oriented both when working
independently and when collaborating with others. We are looking for a
highly motivated and enthusiastic candidate.

What we offer
We can offer an exciting postdoctoral research project with focus on
fundamental questions in evolutionary biology using state-of-the-art
molecular genetic analyses. The postdoctoral researcher will carry
out scientifically challenging research with good opportunities
for development of her/his expertise. The successful candidate will
benefit from a good working environment and the expertise of members
in our research groups which span quantitative genetics, statistical
genetics/genomics and mathematical modelling.

Preferred starting date is in February 2016 but this is negotiable.The
appointment of the Postdoc will be made according to Norwegian guidelines
for universities and university colleges and to the general regulations
regarding university employees. NTNUs personnel policy objective is
that the staff must reflect the composition of the population to the
greatest possible extent.

The position as Postdoc is remunerated according to the Norwegian State
salary scale. There is a 2% deduction for superannuation contribution.

Application
The application with CV, pdfs of publications, reference letters and
relevant certificates must be submitted electronically through this
page. Applications submitted elsewhere will not be considered.  Please
indicate preferred starting date in your cover letter.  The reference
number of the position is: NT 84/15

Deadline for applying: January 15, 2016

Apply through this link:
http://bit.ly/1O6omnB


Arild Husby, Assistant Professor
Department of Biosciences (Biocenter 3, office 5415),
University of Helsinki
PO box 65,
FI-00014 Helsinki,
Finland

web: http://bit.ly/11XzFaY
mail: arild.husby@helsinki.fi
office phone: +358294157691


&ldquo;Husby, Arild&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:38</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135240570536">Graduate position: LMU_Munich.Paleobiology</a></div>
  <div class="description">Dear EvolDir readers,

we would like to announce that application to our Master Program
Geobiology and Paleobiology (MGAP), starting in the winter semester
2016/17 is now open for international students and kindly ask to forward
the attached message to interested students.

With kind regards, 
Oliver Voigt


*Applications invited for the Master&rsquo;s program &ldquo;Geobiology and
Paleobiology&rdquo; (MGAP) at the Ludwig-Maximilians-Universitt (LMU) Munich
(Germany)*

 The Master&rsquo;s program &ldquo;Geobiology and Paleobiology&rdquo; (MGAP) is a
 consecutive, research-focussed, two-year Master of Science program at
 the Faculty of Geosciences of the Ludwig-Maximilians-Universitt (LMU)
 Munich, in collaboration with researchers of the Bavarian Natural History
 Collections and the GeoBio-Center @ LMU. MGAP aims to provide students
 with a comprehensive introduction into the interdisciplinary research
 fields of geobiology and paleobiology to prepare them for careers in
 science and beyond.

 The MGAP program is based on interdisciplinary, research-oriented
 courses in geo- and biosciences that address patterns and processes
 of evolutionary and environmental geobiology and paleobiology. The
 module-based curriculum provides an integrative approach facilitated
 by experts in different areas of expertise. Students will learn and
 acquire routines with a wide range of scientific methods such as modern
 techniques in molecular biology, fieldwork, collection management,
 comparative morphology, phylogeny, bioinformatics, statistics and
 (paleo-) biodiversity assessments. Students will learn independent
 scientific work in individual and intensively supervised research
 projects at early stages of the curriculum.

*MGAP in brief*

        - International Master&rsquo;s program in Geobiology and Paleobiology
          (Master of Science, M.Sc.) at the LMU Munich

        - 2 years, 4 semesters, start in October (winter semester)

        - All courses taught in English

        - Course total: 120 credit points (ECTS)

 

More information is available on the program&rsquo;s website:


For the winter semester 2016-2017 (courses start mid-October), application
dates are as follows:

1) January 31, 2016 for non-EU international students (and EU students
   from outside of Germany. but application is also possible later),
   applications are possible now: 

2) February 15 to May 31, 2016 for EU students (including German
   students).

 All applications have to be submitted via an online submission portal:



Oliver Voigt 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>December 14, 2015</h3>
<div class="news-item">
 <div class="date">23:39</div>
 <div class="body">
  <div class="title"><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12144">What is a cladogram and what is not?</a></div>
  <div class="description">The origins and meanings of “cladogram” are reviewed. Traditionally, “cladogram” has been defined as a graphical representation of an empirical hypothesis of relationships among taxa, based on evidence from synapomorphies alone. Disturbingly, numerous recent authors treat “cladogram” as synonymous with “dendrogram” and do not appreciate the particular methodological connotations of the former term. This is lamented.</div>
  <div class="source">Source: <a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:37</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135238113986">Postdoc: McGillU_Montreal.ComputationalGenomics</a></div>
  <div class="description"><p>Postdoctoral position in computational genomics


A postdoctoral position is available immediately in the Computational
Genomics Laboratory (<a href="http://bit.ly/1UsnGYE">http://bit.ly/1UsnGYE</a>), Lady Davis Institute of
Medical Research and the Ludmer Centre for Neuroinformatics and Mental
Health. The center focuses on understanding the genetic, epigenetic and
environmental influences that shape brain development, including structure
and function, and the risk for mental disorders. This Centre spans the Lady
Davis Research Institute, the Douglas Mental Health University Institute
and the Montreal Neurological Institute, all affiliated with McGill
University, Montreal, Quebec, Canada.

Our lab uses genome-wide approaches to understand mechanisms regulating
gene expression and RNA processing in the developing brain, their interplay
with genetic and epigenetic factors, and how they cause disease. We work in
an interdisciplinary environment, interacting with statisticians,
physicists and molecular biologists. Successful candidates will conceive
and develop algorithms and analytical approaches to integrate large
genetic/epigenetic/expression datasets, generated both in-house and from
public sources, primarily from next-generation sequencing technologies.

Qualifications and Experience
We are looking for a very motivated individual capable of carrying out
independent research, and to interact in a collaborative setting


-> Doctoral degree in computer science, bioinformatics, statistics,
molecular biology, genetics or a related field

-> Strong programming skills are essential

-> Experience within one of the following areas is an asset, but not
required: genomics, neuroscience, statistical interpretation and analysis
of next-generation sequencing datasets.


For applying, please send a CV, a brief statement of scientific interest
and contact information for at least two references to
claudia.kleinman@mcgill.ca


cl.kleinman1@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:36</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/135238113586">Postdoc: NewZealand.FishPhylogenetics</a></div>
  <div class="description">SUBJECT: Postdoc.NewZealand.Fish phylogenetics and community ecology

Postdoctoral Research Fellow
Institute of Natural & Mathematical Sciences, College of Sciences, Massey
University, Auckland, New Zealand

Apply here:
<a href="http://bit.ly/1QkodNY">http://bit.ly/1QkodNY</a>

We are seeking a Postdoctoral Research Fellow to contribute to the project
Synergising ecology and evolution: discovering patterns of functional and
phylogenetic diversity of New Zealand&rsquo;s marine fishes versus depth, funded
by the Marsden Fund, Royal Society of New Zealand. Applications close: 19
January (11:45pm, New Zealand Standard time).

Project Overview
Biologists know least about the largest habitat on earth - the deep sea.
Fundamental patterns of functional and phylogenetic biodiversity, from
coastal waters to the abyss, have never been quantified. Do biodiversity
trends in the ocean follow the patterns in alpha and beta diversity seen in
the terrestrial environment? New Zealands latitudinal range and nearshore
continental shelf offers a unique opportunity to address trends in
biodiversity and community structure and to disentangle the evolutionary
and ecological drivers of these patterns. This project will complement an
existing stratified and replicated depth by latitude dataset describing
fish community composition. Specifically, this project aims to combine
phylogenetic and functional information with knowledge of community
composition for the fishes of New Zealand and to understand the ecological
and evolutionary drivers of community composition from the sub-tropics to
the sub-antarctic, and from the shallows to the abyss. The results of these
analyses will have a major impact on the interpretation of the ecology and
evolutionary history of New Zealands fish communities and biodiversity
patterns in the ocean, and will contribute novel methods of analyses for
biodiversity research.

Position Overview
We are seeking a motivated and innovative researcher, keen to make their
mark in the field of quantitative biodiversity research. The ideal
candidate will have a PhD in biology and will have experience in
phylogenetics, diversification analyses, biogeographic analysis,
comparative methods, trait-based analyses, ecological modeling,
multivariate statistics, macroecology, and community ecology. Applicants
with only a subset of these skills are encouraged to apply. A track record
of publication in international journals and relevant statistical,
computational and programming skills are expected. The appointed Fellow
will conduct phylogenetic analyses of new and data-mined sequence data, and
will combine these data with existing community data, and functional trait
data to describe biodiversity patterns. The Fellow will contribute to the
development of statistical methods to help test explicit hypotheses that
will disentangle evolutionary and ecological drivers of diversity and
turnover. The Fellow is not expected to have any prior experience in marine
fieldwork, molecular genetic benchwork, or ichthyology. The appointee will
be encouraged to supplement the primary research goals of the project with
their own research objectives, in consultation with the wider research team
that includes Prof. Marti J. Anderson (co-Primary Investigator, Chair in
Statistics, Massey University), Dr. Clive Roberts (co-Primary Investigator,
Head Curator of Fishes, Museum of New Zealand Te Papa Tongarewa), Dr. Libby
Liggins (mentor/supervisory role, Lecturer in Marine Ecology, Massey
University) and a recruited PhD candidate (responsible for functional trait
analysis, Massey University). The Fellow will be expected and supported to
publish and present their findings to international peers. Proficiency in
written and spoken English is required.

Position specifics
The position is three years, starting March 2016 (a later start date may be
negotiated for an exceptional candidate). The Fellow will be based in the
Institute of Natural and Mathematical Sciences (INMS) at Massey
Universitys campus in Auckland, New Zealand. INMS is a recently formed
multi-disciplinary research and teaching Institute incorporating Biology,
Chemistry, Computer Science, Mathematics, Physics and Statistics. In
conjunction with world-class researchers in the closely-linked Institute
for Advanced Study (IAS), INMS has special strengths in quantitative
biology and ecology, evolution, genetics, mathematical biology,
computational science and modelling of complex systems. Massey University
is in the north of Auckland, with easy access to the city of Auckland (New
Zealands largest and most culturally diverse city) and recreational areas
such as the harbour, islands, beaches, and forest parks. The Fellow will be
joining a dynamic team of researchers working on the frontier of
statistics, ecology, and evolution. The Fellow will also work closely with
scientists of Te Papa Tongarewa, Museum of New Zealand, in Wellington, New
Zealands capital city. Te Papa holds New Zealands National Fish
Collection and supports an active ichthyology research team.

Enquiries regarding the funded project and specific role of the Fellow
should be directed to Dr. Libby Liggins (Email: l.liggins@massey.ac.nz) or
Prof. Marti J. Anderson (Email: m.j.anderson@massey.ac.nz).
Information about the Institute of Natural and Mathematical Sciences can be
found at <a href="http://bit.ly/1UsnGIp;">http://bit.ly/1UsnGIp;</a> information about Te Papa Tongarewa can
be found at <a href="http://bit.ly/1Qkoe4e.">http://bit.ly/1Qkoe4e.</a>

Libby Liggins 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/1?rss=1">Phylogenies, the Comparative Method, and the Conflation of Tempo and Mode</a></div>
  <div class="description"><p>The comparison of mathematical models that represent alternative hypotheses about the tempo and mode of evolutionary change is a common approach for assessing the evolutionary processes underlying phenotypic diversification. However, because model parameters are estimated simultaneously, they are inextricably linked, such that changes in tempo, the pace of evolution, and mode, the manner in which evolution occurs, may be difficult to assess separately. This may potentially complicate biological interpretation, but the extent to which this occurs has not yet been determined. In this study, we examined 160 phylogeny x trait empirical data sets, and conducted extensive numerical phylogenetic simulations, to investigate the efficacy of phylogenetic comparative methods to distinguish between models that represent different evolutionary processes in a phylogenetic context. We observed that, in some circumstances, a high uncertainty exists when attempting to distinguish between alternative evolutionary scenarios underlying phenotypic variation. When examining data sets simulated under known conditions, we found that evolutionary inference is straightforward when phenotypic patterns are generated by simple evolutionary processes that are represented by modifying a single model parameter at a time. However, inferring the exact nature of the evolutionary process that has yielded phenotypic variation when facing complex, potentially more realistic, mechanisms is more problematic. A detailed investigation of the influence of different model parameters showed that changes in evolutionary rates, marked changes in phylogenetic means, or the existence of a strong selective pull on the data, are all readily recovered by phenotypic model comparison. However, under evolutionary processes with a milder restraining pull acting on trait values, alternative models representing very different evolutionary processes may exhibit similar goodness-of-fit to the data, potentially leading to the conflation of interpretations that emphasize tempo and mode during empirical evolutionary inference. This is a mathematical and conceptual property of the considered models that, while not prohibitive for studying phenotypic evolution, should be taken into account and addressed when appropriate.</p></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/A1?rss=1">Announcements</a></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/i1?rss=1">Systematic Biology Volume 65, Number 1 January 2016 - Front Cover</a></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/i2?rss=1">Systematic Biology - Editorial Board</a></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/i3?rss=1">Information for Members and Subscribers - Subscriptions</a></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/i4?rss=1">Systematic Biology Volume 65, Number 1 January 2016 - Back Cover</a></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/16?rss=1">Inflation of Molecular Clock Rates and Dates: Molecular Phylogenetics, Biogeography, and Diversification of a Global Cicada Radiation from Australasia (Hemiptera: Cicadidae: Cicadettini)</a></div>
  <div class="description"><p>Dated phylogenetic trees are important for studying mechanisms of diversification, and molecular clocks are important tools for studies of organisms lacking good fossil records. However, studies have begun to identify problems in molecular clock dates caused by uncertainty of the modeled molecular substitution process. Here we explore Bayesian relaxed-clock molecular dating while studying the biogeography of ca. 200 species from the global cicada tribe Cicadettini. Because the available fossils are few and uninformative, we calibrate our trees in part with a cytochrome oxidase I (COI) clock prior encompassing a range of literature estimates for arthropods. We show that tribe-level analyses calibrated solely with the COI clock recover extremely old dates that conflict with published estimates for two well-studied New Zealand subclades within Cicadettini. Additional subclade analyses suggest that COI relaxed-clock rates and maximum-likelihood branch lengths become inflated relative to EF-1&alpha; intron and exon rates and branch lengths as clade age increases. We present corrected estimates derived from: (i) an extrapolated EF-1&alpha; exon clock derived from COI-calibrated analysis within the largest New Zealand subclade; (ii) <I>post hoc</I> scaling of the tribe-level chronogram using results from subclade analyses; and (iii) exploitation of a geological calibration point associated with New Caledonia. We caution that considerable uncertainty is generated due to dependence of substitution estimates on both the taxon sample and the choice of model, including gamma category number and the choice of empirical versus estimated base frequencies. Our results suggest that diversification of the tribe Cicadettini commenced in the early- to mid-Cenozoic and continued with the development of open, arid habitats in Australia and worldwide. We find that Cicadettini is a rare example of a global terrestrial animal group with an Australasian origin, with all non-Australasian genera belonging to two distal clades. Within Australia, we show that Cicadettini is more widely distributed than any other cicada tribe, diverse in temperate, arid and monsoonal habitats, and nearly absent from rainforests. We comment on the taxonomic implications of our findings for thirteen cicada genera.</p></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/35?rss=1">Quantifying Age-dependent Extinction from Species Phylogenies</a></div>
  <div class="description"><p>Several ecological factors that could play into species extinction are expected to correlate with species age, i.e., time elapsed since the species arose by speciation. To date, however, statistical tools to incorporate species age into likelihood-based phylogenetic inference have been lacking. We present here a computational framework to quantify age-dependent extinction through maximum likelihood parameter estimation based on phylogenetic trees, assuming species lifetimes are gamma distributed. Testing on simulated trees shows that neglecting age dependence can lead to biased estimates of key macroevolutionary parameters. We then apply this method to two real data sets, namely a complete phylogeny of birds (class Aves) and a clade of self-compatible and -incompatible nightshades (Solanaceae), gaining initial insights into the extent to which age-dependent extinction may help explain macroevolutionary patterns. Our methods have been added to the R package TreePar.</p></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">23:09</div>
 <div class="body">
  <div class="title"><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/1/51?rss=1">Rumbling Orchids: How To Assess Divergent Evolution Between Chloroplast Endosymbionts and the Nuclear Host</a></div>
  <div class="description"><p>Phylogenetic relationships inferred from multilocus organellar and nuclear DNA data are often difficult to resolve because of evolutionary conflicts among gene trees. However, conflicting or "outlier" associations (i.e., linked pairs of "operational terminal units" in two phylogenies) among these data sets often provide valuable information on evolutionary processes such as chloroplast capture following hybridization, incomplete lineage sorting, and horizontal gene transfer. Statistical tools that to date have been used in cophylogenetic studies only also have the potential to test for the degree of topological congruence between organellar and nuclear data sets and reliably detect outlier associations. Two distance-based methods, namely ParaFit and Procrustean Approach to Cophylogeny (PACo), were used in conjunction to detect those outliers contributing to conflicting phylogenies independently derived from chloroplast and nuclear sequence data. We explored their efficiency of retrieving outlier associations, and the impact of input data (unit branch length and additive trees) between data sets, by using several simulation approaches. To test their performance using real data sets, we additionally inferred the phylogenetic relationships within Neotropical Catasetinae (Epidendroideae, Orchidaceae), which is a suitable group to investigate phylogenetic incongruence because of hybridization processes between some of its constituent species. A comparison between trees derived from chloroplast and nuclear sequence data reflected strong, well-supported incongruence within <I>Catasetum</I>, <I>Cycnoches</I>, and <I>Mormodes</I>. As a result, outliers among chloroplast and nuclear data sets, and in experimental simulations, were successfully detected by PACo when using patristic distance matrices obtained from phylograms, but not from unit branch length trees. The performance of ParaFit was overall inferior compared to PACo, using either phylograms or unit branch lengths as input data. Because workflows for applying cophylogenetic analyses are not standardized yet, we provide a pipeline for executing PACo and ParaFit as well as displaying outlier associations in plots and trees by using the software R. The pipeline renders a method to identify outliers with high reliability and to assess the combinability of the independently derived data sets by means of statistical analyses.</p></div>
  <div class="source">Source: <a href="indexa8da.html?q=aggregator/sources/17">Latest issue</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="index940a.html?q=aggregator&amp;from=400">previous page</a></div><div>... <a href="indexd191.html?q=aggregator&amp;from=340">18</a> <a href="index28a4.html?q=aggregator&amp;from=360">19</a> <a href="index63c1.html?q=aggregator&amp;from=380">20</a> <a href="index940a.html?q=aggregator&amp;from=400">21</a> <strong>22</strong> <a href="index764e.html?q=aggregator&amp;from=440">23</a> <a href="index3d2c.html?q=aggregator&amp;from=460">24</a> <a href="indexb567.html?q=aggregator&amp;from=480">25</a> <a href="index8c9a.html?q=aggregator&amp;from=500">26</a> ...</div><div><a href="index764e.html?q=aggregator&amp;from=440">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=420 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:37:49 GMT -->
</html>
