<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=80 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:20:40 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexfe3a.html?q=node/279">Contact</a></li><li><a href="index2875.html?q=node/72">MacClade and Adobe Illustrator</a></li><li><a href="indexf64b.html?q=node/63">LaTeX template and bibliography style file</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 5 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 3 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>February 24, 2016</h3>
<div class="news-item">
 <div class="date">01:18</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a></div>
  <div class="description"><br />I am publishing this post because both the author and publisher involved have stopped responding to my emails.<br /><br />Some of you may have noticed the recent publication of the following book:<br />Dan Graur (2016) <i>Molecular and Genome Evolution</i>. Sinauer Associates.Chapter 6 is of interest to the readers of this blog, being entitled "Reticulate evolution and phylogenetic networks". Unfortunately, as originally published, not all of the figures in that chapter acknowledge the source of the illustration. Of personal interest to me, Figure 6.4 [which can be <a href="http://www.rjr-productions.org/Graur.html" target="_blank">viewed here</a>] is a direct copy of the first part of my <a href="http://acacia.atspace.eu/Tutorial/Tutorial.html" target="_blank">Primer of Phylogenetic Networks</a>. Needless to say, Graur's figure prominently claims to be the copyright of the publisher rather than myself.<br /><br />Neither the author not the publisher has provided a satisfactory explanation, and have made it clear that nothing will be done to rectify this except in some possible future edition of the book.<br /><br />So, in the meantime, could you all please note that the idea is mine, not either Graur's or Sinauer's. This annoys me, not just because of the laziness that lead to this situation, but because the figures are an original conception of mine to explain Median Networks to non-experts, and I was very happy to have developed them. Having someone else trying to take the credit takes the shine off that.<br /><br /></div>
  <div class="source">Source: <a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:17</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139900437461">Postdoc: Roscoff.MarineBiodiversity</a></div>
  <div class="description">A 1-year postdoctoral fellowship is available at the Roscoff Biological
Station (www.sb-roscoff.fr) of the University Pierre & Marie Curie
(Paris6), France. The candidate will work in close collaboration with Dr
Christophe Lejeusne (clejeusne@sb-roscoff.fr) to investigate the effects
of environmental gradients (e.g. pollution) on species diversity in
marinas, with a special focus on non-indigenous species. Observational
and experimental approaches would be run.

The postdoc will be in charge of characterizing and analyzing the
diversity and structure of some harbor communities (including through
integrative taxonomy approaches, and photo surveys) and its exposure to
contaminants (through biomarkers and water quality assessment). Aquaria
and in situ experiments would also be conducted on manipulated communities
(e.g. increased diversity of introduced species).

The ideal candidate will have expertise in the analysis of community data,
especially from hard substrates and/or artificial substrates. Candidates
with expertise in taxonomy are very welcome (especially of sponges,
tunicates, mollusks, but not only). An ability to run both aquarium and
field experiments is highly desirable. Scientific diving is a plus.

The candidate will be based at the Roscoff Biological Station and
will benefit from the support of its platforms for observational and
experimental works (diving, in situ and aquaria experiments), and also
for taxonomic expertise.

The candidate will be part of the Diversity and Connectivity group
(<a href="http://bit.ly/1WI79k2">http://bit.ly/1WI79k2</a><a href="http://www.sb-roscoff.fr/">www.sb-roscoff.fr</a>) of the University Pierre & Marie Curie
(Paris6), France. The candidate will work in close collaboration with Dr
Christophe Lejeusne (clejeusne@sb-roscoff.fr) to investigate the effects
of environmental gradients (e.g. pollution) on species diversity in
marinas, with a special focus on non-indigenous species. Observational
and experimental approaches would be run.

The postdoc will be in charge of characterizing and analyzing the
diversity and structure of some harbor communities (including through
integrative taxonomy approaches, and photo surveys) and its exposure to
contaminants (through biomarkers and water quality assessment). Aquaria
and in situ experiments would also be conducted on manipulated communities
(e.g. increased diversity of introduced species).

The ideal candidate will have expertise in the analysis of community data,
especially from hard substrates and/or artificial substrates. Candidates
with expertise in taxonomy are very welcome (especially of sponges,
tunicates, mollusks, but not only). An ability to run both aquarium and
field experiments is highly desirable. Scientific diving is a plus.

The candidate will be based at the Roscoff Biological Station and
will benefit from the support of its platforms for observational and
experimental works (diving, in situ and aquaria experiments), and also
for taxonomic expertise.

The candidate will be part of the Diversity and Connectivity group
(http://bit.ly/1WI79k2).

Duration: 1 year (possibly one additional year depending on funding)

Salary:  about 2000 euros per month (after taxes)

Administrative requirements:

# Either non-French citizens, or French citizens having spent at least
12 months out of France over the last 36 months (3yrs)

# PhD should have been defended less than 2 years ago

Preferred start date: as soon as possible but no later than early
June 2016



An ability to communicate in English is essential, and a willingness to
learn French is desirable (but not mandatory).

Interested candidates should send a detailed CV and a letter
expressing their interest in this position by email to the address
given (clejeusne@sb-roscoff.fr). They should also provide details of
two potential referees.

Review of applications will begin immediately and continue until the
position is filled. Further information on the project can also be
requested from the same address.



Christophe Lejeusne  
Associate Professor
University Pierre & Marie Curie (Paris 6)

AD2M  - Adaptation & Diversit en
Milieu Marin

UMR 7144 UMPC - CNRS
Group Diversity & Connectivity - Div&Co


Station Biologique de Roscoff
Place Georges Teissier - CS90074
29688 Roscoff - France

clejeusne@sb-roscoff.fr 
Ph.: +33-298 292 372



Christophe Lejeusne 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:02</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139900042706">Course: FridayHarbor_Washington.ComparativeEmbryology.Jun13-Jul15</a></div>
  <div class="description"><p>Applications are open for the Comparative Invertebrate Embryology course at
the Friday Harbor Labs (June 13 - July 15, 2016). *Applications are needed
by Feb. 29th*.

It is a great course for anyone interested in the evolution of development,
developmental mechanisms, animal evolution, or the ecology or physiology of
early life stages. The focus of the course is hands-on observation of
living embryos and larvae from over a dozen animal phyla. The Friday Harbor
Labs gives access to an extraordinary variety of organisms from diverse
marine habitats, all in a beautiful setting, and provides opportunities to
interact with a broad community of biologists.

We aim to bridge cell and molecular approaches with ecological and
evolutionary approaches to provide an integrated view of animal
development. Depending on student interests, students may learn techniques
such as:
- Spawning and culture of embryos and larvae from diverse phyla.
- Confocal microscopy and SEM
- Microinjection
- Methods for measuring cell/embryo mechanical properties

Instructors include Dr. Sally Leys and myself. Dr. Leys is an expert on
sponge development and the evolution of animal body plans. My research
focuses on the roles of biomechanics in development-environment
interactions. We will also have guest lectures from experts on diverse
topics and taxa, including Drs. Brad Shuster, Richard Strathmann, Sophie
George, Tony Pires, and others TBD.

For the course description, please see:
<a href="http://bit.ly/1KJl34J">http://bit.ly/1KJl34J</a> .

Please share with students who might be interested.

Thank you.

Mickey


mvondass@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>February 23, 2016</h3>
<div class="news-item">
 <div class="date">23:17</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139896355516">Conference: Helsinki. WildPlantPathosystems.Aug29-31</a></div>
  <div class="description"><p>This is a multi-part message in MIME format.
via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:15</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139845813001">Graduate position:</a></div>
  <div class="description">3 PhD positions Biological control of Drosophila suzukii, a new invasive
pest species.

Project description
A new invasive pest species, Drosophila suzukii or Spotted Wing Drosophila
(SWD), has invaded Europe and already caused millions of euros of damage in
the European fruit industry. There is an urgent need to develop new
strategies to control the SWD in the Netherlands, as the current practices
of control with chemical pesticides are neither effective nor sustainable.
We aim to develop an integrated pest management strategy of combined
biocontrol approaches. We still know relatively little about the basic
biology of this new pest species in the context of the temperate climate in
the Netherlands. Therefore, we will 1) investigate the factors that control
population growth, reproductive success and survival of SWD in the
Netherlands. With this knowledge, we will develop an integrated pest
management strategy to 2) protect fruit crops from SWD and 3) to
biologically control SWD populations.

Three PhD students will be appointed, each working on a subset of the
aspects of this project, which are interrelated and complementing each
other. These studies will be performed and supervised in a consortium of
the University of Groningen, Wageningen University, WUR-PPO-Fruit and
Koppert Biological Systems.

PhD project 1: Fundamentals of SWD biology in the Netherlands (RUG & WUR)
We need to increase our fundamental knowledge of the factors that control
population growth, reproductive success and survival of SWD in the
Netherlands. This includes the population genetic structure, seasonal
periods of activity, mortality factors and population dynamics. We will
monitor the infestation dynamics of SWD in the Netherlands, and identify
those factors that control pest outbreaks, using a combination of field
observations, field and laboratory experiments and population genetic
techniques.

PhD project 2: Crop protection strategies: a &ldquo;push-pull&rdquo; approach (WUR)
To protect the fruit crops from SWD, we will develop an integrated set of
tools for pest management. In this PhD project, we will develop pheromone
lure traps for monitoring and controlling SWD density, and powerful
repellents to avoid oviposition by SWD in fruit crops. We will assess the
responses of SWD to chemicals that can be developed for lure traps or as
oviposition deterrents, using a combination of behavioural and sensory
assays in the laboratory, and test these chemicals in field trials.

PhD project 3: Biological control strategies: selective breeding of natural
enemies (RUG)
To biologically control SWD populations, we will select a potent natural
enemy (e.g. predators, parasitoids and/or micro-organisms) that can curtail
the population growth of SWD over the growing season. The potential of
several natural enemies as biological control agents will be investigated
in laboratory and field experiments, followed by a &ldquo;selective breeding&rdquo;
approach to increase the virulence of the most promising natural enemies.

Research environment
Within the project, the 3 PhD candidates will be closely collaborating with
each other, and with the academic and private partners in the consortium.
Project 1 will be partially based in Groningen and partially in Wageningen
(ca ~4 months/year), project 2 will be fully based in Wageningen, and
project 3 will be fully based in Groningen.

The PhD projects are embedded in the research teams of Prof. Marcel Dicke
(Laboratory of Entomology, Wageningen University), Dr Bart Pannebakker
(Laboratory of Genetics, Wageningen University), Prof. Bregje Wertheim and
Prof. Leo Beukeboom (Groningen Institute for Evolutionary Life Sciences,
University of Groningen). Our teams consist of ecologists, geneticists,
entomologists, molecular biologists and evolutionary biologists. The
research will be done in close collaboration with the biological control
company Koppert Biological Systems and with the Applied Plant Research
Institute WUR-PPO-Fruit.

Qualifications
The successful candidate will have an MSc degree in Biology or Plant
Sciences with a specialization in Ecology, Evolution, Entomology or
similar, with experience in insect ecology and behavioural studies on
insects. The candidate will be open and communicative, proficient in the
English language and have good statistical skills. We are looking for
highly motivated candidates who can work in a multidisciplinary team in a
collaborative spirit. Within the 3 projects, PhD students are encouraged to
supervise MSc thesis students when such opportunities arise.

Specific requirements:
Project 1: A field / insect evolutionary ecologist with experience and
affinity with molecular techniques and population genetics. A driver&rsquo;s
license and strong interpersonal skills are required for conducting the
research at commercial fruit farms. During the field seasons (June -
September) the candidate will be based at Wageningen University, and in the
remainder of the year, the candidate will be based at the University of
Groningen.

Project 2: A chemical / behavioural ecologist with proven experience with
investigating insect behaviour and the role of infochemicals. Experience
with field research will be a bonus. The candidate will be based at the
Laboratory of Entomology in Wageningen.

Project 3: An insect evolutionary ecologist, or an evolutionary geneticist
with affinity for ecology and insects, with experience in working with
biological control agents (predators, parasitoids or entomopathogens).
Experience with field research will be a bonus. The candidate will be based
at the University of Groningen.

Conditions of employment
We offer a fulltime position (38 hours), initially for 1 year after
which a go/no go decision will be taken on extension with another three
years. The appointment should result in a PhD examination. Gross salary
per month euro 2,174 in the first year rising to euro 2,779 per month
in the fourth year, for a fulltime appointment. A training programme
for additional education and for supervision is part of the agreement.

Starting date: the candidates start as soon as possible.

The interviews will be conducted in April.

How to apply
You may apply for this position until 30 March / before 31 March
2016 Dutch local time by means of the online application form at
<a href="http://bit.ly/1L8xoPU">http://bit.ly/1L8xoPU</a> (click on
&ldquo;Apply&rdquo; on the advertisement on the university website). Please, state
clearly in the application to which of the three PhD projects you
apply. When you are interested in more than one of the PhD positions,
you can submit a single application to the University of your first
preference. In that case, please, clearly state your interest in the
multiple projects and provide a ranking of preference with a personal
motivation for the ranking.

Applications should include a cover letter (with a statement of interest,
and a description of previous research experience such as bachelor/master
research projects) and curriculum vitae (both in PDF format).



Bregje Wertheim
Associate Professor & Rosalind Franklin Fellow
Groningen Institute for Evolutionary Life Sciences
University of Groningen

*visiting:* Nijenborgh 7, 9747 AG Groningen
*postal: *PO Box 11103, 9700 CC Groningen
The Netherlands

email: b.wertheim@rug.nl
tel: +31 (0)50 363 9039

Bregje Wertheim 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:15</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139844297511">Job: Hungary.Bioinformatics</a></div>
  <div class="description">Bioinformatics research assistant position in Szeged, Hungary

The Csaba Pál & Balázs Papp labs within the Synthetic and Systems Biology
Unit of the Biological Research Center of the Hungarian Academy of Sciences
are seeking a talented and motivated BIOINFORMATICIAN to join systems and
synthetic biology projects as a Research Associate.

We are looking for a highly motivated bioinformatician to develop and run
data analysis pipelines for next generation sequencing experiments in the
field of genome engineering, human microbiome analysis and evolutionary
systems biology. Our experiments have a specific focus on prokaryotic de
novo metagenomic and whole genome sequencing, whole genome re-sequencing
and amplicon sequencing, such as 16S rRNA applications for the analysis of
microbial composition.

Requirements:

Previous experience with NGS data analysis (e.g. Illumina) and solid
knowledge of scripting languages (R, Perl) is required, as well as good
communication and teamwork skills. Strong biological insight and experience
in statistical analysis is a plus. We offer a competitive salary and the
possibility of a flexible working schedule.

For further information please contact: kintses.balint@brc.mta.hu

Applications should be submitted via e-mail before 16 March 2016 along with
the following documents:

1. A cover letter in English – brief explanation of the applicant’s
background and reasons for applying

2. Curriculum vitae in English

3. List of publications (if applicable)

<a href="http://www.brc.hu/sysbiol">www.brc.hu/sysbiol</a>

facebook.com/sysbiol

References:

1) Pál, Csaba, Balázs Papp, and György Pósfai. 2014. “The Dawn of
Evolutionary Genome Engineering.” Nature Reviews Genetics

2) Lázár, Viktória, István Nagy, Réka Spohn, Bálint Csörgő, Ádám Györkei,
Ákos Nyerges, Balázs Horváth, et al. 2014. “Genome-Wide Analysis Captures
the D

Papp Balázs 
via Gmail </div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139842321466">Job: UHelsinki.Genomics</a></div>
  <div class="description">Professor/Associate Professor/Assistant Professor in Genetics, with
speciality in genomics

The position is shared between the Department of Biosciences and the
Institute of Biotechnology and may be filled as assistant professor,
associate professor or professor, depending on the candidates’ merits
and career stage.Duties of the professor/associate professor/assistant
professor include conducting internationally competitive research on
any aspect of genomics. She/he should also have appropriate competence
in overlapping areas of genetics, bioinformatics and/or other relevant
fields. The professor/associate professor/assistant professor will play
a central role as an educator, spearheading masters- level training in
genomics, organized jointly between the Department of Biosciences and
the Institute of Biotechnology and will also participate in doctoral
education via appropriate graduate programmes. In addition, he/she will
facilitate public understanding and applications of genomic knowledge.

The professor/associate professor/assistant professor shall carry out
and supervise scientific work, provide education based on it, follow
developments in research, and participate in societal interaction and
international cooperation in his or her field.According to the Regulations
of the University of Helsinki, an appointee to a professorship shall
hold a doctoral degree and have top-level scholarly qualifications,
including the supervision of scientific research. The appointee shall
have the ability to provide top-level teaching based on research, as well
as to supervise theses and dissertations. The appointee shall also have
documentation of international cooperation in the research field that
he/she represents. Furthermore, the position also calls for academic
leadership skills.

An appointee to the position of assistant/associate professor in the
tenure track system shall hold a doctoral degree, have the ability to
conduct independent scholarly work and have the teaching skills necessary
for the position. In addition, applicants for assistant professorships
shall demonstrate their ability and motivation for an academic career
through publications and other means.

When assessing the qualifications of the applicants, attention shall
be paid to scientific publications and other research results of
scientific value, teaching experience and pedagogical training, the
ability to produce learning material, other teaching merits, and, if
necessary, a teaching demonstration as well as participation in doctoral
education. Account shall also be taken of the applicant’s activity in
the scientific community, success in obtaining external research funding,
international research experience and international elected positions
as well as leadership and interaction skills.

To successfully attend to the duties of the position, appointees must
have good English skills. A knowledge of Finnish and or Swedish may also
be useful for undergraduate instruction, but is not essential.

The position overview is available on the Faculty website
<a href="http://bit.ly/15iiVoR">http://bit.ly/15iiVoR</a>
The salary for the position will be based on the demands level chart
for teaching and research personnel in the salary system of Finnish
universities. The demands level for professors is 8–10 and for
assistant/associate professors 7. In addition, the appointee will be
paid a salary component based on personal performance. The salary for a
professor is 5,000–8,000 euros per month and for an assistant/associate
professor 4,200–5,000, depending on the appointee&rsquo;s qualifications
and experience.The closing date for applications is April 10, 2016.

Please submit your application using the University of Helsinki
Recruitment System via the Apply link. Applicants who are employees of
the University of Helsinki are requested to leave their application
via the SAP HR portal. The English-language application must be
accompanied by:
- a CV
- a list of publications
- a brief report of the following qualifications: research and
  scientific activities, research plan, description of teaching
  qualifications, participation in administrative duties
- a report on other activities of which the applicant wishes to inform
  the Faculty

 or, alternatively, an academic portfolio
 containing the above documents (see instructions at
 <a href="http://bit.ly/1dlQvDF">http://bit.ly/1dlQvDF</a>) The
 attachments are asked to be submitted as a single PDF file.

Candidates should be prepared to submit ten publications and/or
manuscripts approved for publication of their choosing to expert assessors
after the closing of the application period, if so requested by the
Faculty. The publications are submitted as PDF files.  More information
can be requested from Professor Kurt Fagerstedt, Head of the Department
of Biosciences, tel. +358-2941 59438, email kurt.fagerstedt@helsinki.fi
and Coordinator Sanna Leinonen, Institute of Biotechnology, tel. +358-2941
59948, email bi-director@helsinki.fi.

About the Department

The Department of Biosciences in the Faculty of Biological and
Environmental Sciences, University of Helsinki, is Finland’s largest
and most prominent institution engaged in biosciences research and
teaching . The Department of Biosciences, based in the Viikki Science
Park, is among the largest departments of the University. Its budget
is approximately 25 million euro, and it employs nearly 400 people. The
Department is divided into six divisions, namely biochemistry, ecology
and evolutionary biology, physiology and neuroscience, plant biology,
genetics, and general microbiology. The Department coordinates the
administration and basic teaching for all students during the first
few years of study, but each of the above divisions are independently
responsible for their teaching and research. The Department also offers
a degree programme that provides students with the qualification of a
biology teacher.

The Institute of Biotechnology (BI) at the University of Helsinki is an
independent institute with the mission of promoting research and teaching
in biotechnology, molecular biology and cellular biology, and using
this for the benefit of society. The Institute has research programmes
in molecular cell biology, developmental biology, genome biology,
and structural biology & biophysics. At the end of 2015, it housed
27 research groups and a staff of ca. 340. Institute staff includes 40
different nationalities, and 50 % of all researchers have an international
background. The budget of the Institute is approximately 23 million
euro, of which nearly two-thirds are derived from external competitive
funding. In addition to top research, the Institute of Biotechnology
offers core services to its own researchers as well as other researchers
and companies on the Viikki Campus and elsewhere. Such services include
DNA sequencing and genomics, electron microscopy, light microscopy, NMR,
proteomics and protein crystallisation.The Department of Biosciences at
the University of Helsinki’s Faculty of Biological and Environmental
Sciences invites applications for the positon of a full-time Applications:

<a href="http://bit.ly/1XJWF4S...">http://bit.ly/1XJWF4S&hellip;</a>

Deadline

Sunday, April 10, 2016 - 00:00

Juha Merilä 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139842321066">Workshop: Toulouse.Evolution.May30-31</a></div>
  <div class="description">Dear colleagues,

We are delighted to invite you to the 4th Toulouse Economics and Biology
Workshop, which will take place on May 30-31, 2016.

The theme is &ldquo;Evolution: transmission mechanisms and population
structure&rdquo;, and we are thrilled to welcome the following speakers
(titles and summaries coming soon):

Yann Algan , Sciences Po, Paris
Carl Bergstrom , University of Washington
Alberto Bisin , New York University
Jean Clobert , Station d'Ecologie Théorique et Expérimentale du CNRS à Moulis
Gabriella Conti , University College London
Mats Gyllenberg , University of Helsinki
Barry Hewlett , Washington State University, Vancouver
Sarah Mathew , Arizona State University, Tempe
Hisashi Ohtsuki , The Graduate University for Advanced Studies, Hayama
David Queller , Washington University, St Louis
Paul Seabright , Institute for Advanced Study in Toulouse, Toulouse School of Economics
Joan Strassman , Washington University, St Louis

Besides the plenary sessions, there will be a poster session for post-docs
and Ph.D. students.  Submissions (abstract + C.V.) should be sent to
econbio_poster@iast.fr no later than March 31, 2016.  A limited number
of travel grants will be available. To apply, please join a cover letter
explaining why you are applying for a grant and why you are interested
in attending to the workshop.

The workshop will take place at the Institute for Advanced Study in
Toulouse, Manufacture des Tabacs, 21 Allée de Brienne, 31015 Toulouse,
France.

Registration will open in the beginning of March at:
<a href="http://bit.ly/1TCqpkV">http://bit.ly/1TCqpkV</a>

We look forward to welcoming you!

Organizers: Ingela Alger and Jörgen Weibull
Co-organizers: Jeanne Bovet and Arnaud Tognetti


The Toulouse Economics and Biology Workshop 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:30</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139841999741">Courses: SanJuan_PR.PhylogeneticComparativeMethods.Jun28-Jul1</a></div>
  <div class="description"><p>The following is an announcement for a workshop we will offer on 
phylogenetic comparative methods in San Juan, Puerto Rico from June 28 - 
July 1, 2016. The course is free of cost, and there are various stipends 
to partially or fully cover travel and lodging expenses. A Spanish 
version of the announcement can also be seen below.

El siguiente es un anuncio por un taller sobre métodos comparativos 
filogenéticos que ofreceremos en San Juan, Puerto Rico desde el 28 de 
junio hasta el 1 de julio, 2016. El curso es libre de costos, y hay 
varios estipendios disponibles para minimizar o cubrir los gastos de 
viaje y alojamiento en San Juan. Versión en español abajo.

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:30</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139841999616">Conference: Austin.Evolution.Jun17-21.SpotlightPhylogenetics</a></div>
  <div class="description"><p>We&rsquo;ve extended the deadline for our spotlight session to March 19th, 2016.

Please send on your ideas!

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:30</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139840582271">Graduate position: ULincoln.YeastPopGenomics</a></div>
  <div class="description">YeastPopulationGenomics

Next generation analyses using next-generation DNA sequencing: testing
theory for the population genomics of microbes

This is a computer based project. We are looking for a student who is
strong in bioinformatics and theory, ideally with experience with whole
genome data, and ideally some experience with microbes.

This is a fully-funded position with stipend and fees covering three
years.

 Main supervisor: Associate Professor Matthew Goddard, University
 of Lincoln, UK and University of Auckland, New Zealand;
 <a href="http://bit.ly/18cf3YS">http://bit.ly/18cf3YS</a>

Co-supervisors: Prof Stuart Humphries, University of Lincoln

Collaborator: Assistant Professor Chris Hittinger, University of
Wisconsin-Madison, USA,

Please contact Mat Goddard - mgoddard@lincoln.ac.uk - for all enquiries,
and include a CV and covering letter.

Background

Microbes are key component of natural, agricultural and commercial
ecosystems due to their metabolic actions and ecological functions[1]. For
example, bacteria and fungi are key components of soil that at least
provide and recycle nutrients for plant growth, and perform many other
ecological roles[2]. An array of microbes transform crops to higher value
commodities (cheese, coffee, cocoa, beer, wine etc)[3]. Pathogenic
microbes potentially cause disease, for example Campylobacter is
responsible for >280,000 food poisoning cases in the UK per year and
c. 100 deaths, costing the UK economy >£580m per year, much of this
imposed on the NHS[4]. If it were not for microbes modern day sewage
treatment plants would cease to function.

If we are to understand ecosystems in their totality, let alone reliably
harness and manipulate such ecosystems to render them safe or productive,
we need to know how the microbial components work. Unfortunately, while a
few species of microbes have been very well genetically and molecularly
described, the population biology and ecology of even the most-well
studied of microbes remains cryptic[5-7]. This is simply because the
ecology and population biology of microbes is hard to study.

Recent step-changes in DNA sequencing technology have opened up
unparalleled opportunities in this area[8,9]. This technology provides
three crucial aspects that were not attainable before: 1, it can access
and analyse microbes that are not easily cultured in the laboratory as
DNA may be extracted directly from the environment; 2, billions of cells
may be analysed rather than hundreds due to the constrains of individual
analyses of colonies on agar plates; and 3, whole genome sequence data
from individual isolates may now be gathered at a reasonable cost.

This project will primarily use the whole genomes of multiple individual
from natural population of closely related commercially and academically
important species of microbes – Saccharomyces cerevisiae and S. uvarum
- to analyse these populations and test the applicability of current
theory, and develop new theory specific for microbial population biology
and ecology. These two species are research super-models and important
in wine, bread and beer production. We will analyse populations that
derive from New Zealand and the important and powerful difference is
that S. cerevisiae is a recently introduced species, but that S. uvarum
is an ancient endemic species[13-15]. This crucially will allow us to
compare and contrast patterns and processes in microbial populations
that have very different recent evolutionary histories and potential
ecologies. There will be the opportunity for the successful student to
work with other types of data as well, specifically microbial community
data.

We will:

1 – analyse and develop theory for the analyses of population structure,
as Goddard has done previously[16,17], but here with the power of whole
genome data that will also account for gene-flow and recombination
between sub-populations.

2 – Following the first and only study to estimate the life-history
parameters of microbes[18], use whole genome data to quantify the extent
of meiosis, mitosis and modes of inbreeding.

3 – use the above data to parameterise and then develop new theory
to analyse populations that are part clonal, part sexual, as microbial
eukaryotes are.

4 – estimate signals of selection in genomes and compare and contrast
the two species to evaluate if this recovers insights into their histories
of invasion and endemism.

1.   Hanson, C. A., Fuhrman, J. A., Horner-Devine, M. C. & Martiny,
J. B. H. Beyond biogeographic patterns: processes shaping the microbial
landscape. Nat Rev Micro (2012). doi:10.1038/nrmicro2795

2.   Hughes Martiny, J. B. et al. Microbial biogeography: putting
microorganisms on the map. Nat Rev Micro 4, 102–112 (2006).

3.   Chambers, P. J. & Pretorius, I. S. Fermenting knowledge: the history
of winemaking, science and yeast research. EMBO Rep 11, 914–920 (2010).

4.   Strachan, N. J. C. & Forbes, K. J. The growing UK epidemic of human
campylobacteriosis. Lancet 376, 665–667 (2010).

5.   Greig, D. & Goddard, M. Ecology: Tribal Warfare Maintains Microbial
Diversity. Curr. Biol. 25, R618–20 (2015).

6.   Morrison-Whittle, P. & Goddard, M. R. Quantifying the relative
roles of selective and neutral processes in defining eukaryotic microbial
communities. The ISME Journal 9, 2003–2011 (2015).

7.   Goddard, M. R. & Greig, D. Saccharomyces cerevisiae: a nomadic
yeast with no niche? FEMS Yeast Res. 15, (2015).

8.   Caporaso, J. G. et al. QIIME allows analysis of high-throughput
community sequencing data. Nature Publishing Group 7, 335–336 (2010).

9.   Bik, H. M. et al. Sequencing our way towards understanding global
eukaryotic biodiversity. Trends in Ecology & Evolution 27, 233–243
(2012).

13. Almeida, P. et al. A Gondwanan imprint on global diversity and
domestication of wine and cider yeast Saccharomyces uvarum. Nat Commun 5,
4044 (2014).

14. Gayevskiy, V. & Goddard, M. R. Saccharomyces eubayanus and
Saccharomyces arboricola reside in North Island native New Zealand
forests. Environ. Microbiol. (2015). doi:10.1111/1462-2920.13107

15. Goddard, M. R., Anfang, N., Tang, R., Gardner, R. C. & Jun, C. A
distinct population of Saccharomyces cerevisiae in New Zealand: evidence
for local dispersal by insects and human-aided global dispersal in oak
barrels. Environ. Microbiol. 12, 63–73 (2010).

16. Knight, S. & Goddard, M. R. Quantifying separation and similarity in
a Saccharomyces cerevisiae metapopulation. The ISME Journal 9, 361–370
(2015).

17. Gayevskiy, V., Klaere, S., Knight, S. & Goddard,
M. R. ObStruct: A Method to Objectively Analyse Factors Driving
Population Structure Using Bayesian Ancestry Profiles. PLoS ONE
(2014). doi:10.1371/journal.pone.0085196.s002

18. Tsai, I. J., Bensasson, D., Burt, A. & Koufopanou, V. Population
genomics of the wild yeast Saccharomyces paradoxus: Quantifying the life
cycle. Proceedings of the National Academy of Sciences 105, 4957–4962
(2008).

The University of Lincoln, located in the heart of the city of Lincoln,
has established an international reputation based on high student
satisfaction, excellent graduate employment and world-class research.

The information in this e-mail and any attachments may be confidential. If
you have received this email in error please notify the sender immediately
and remove it from your system. Do not disclose the contents to another
person or take copies.

Email is not secure and may contain viruses. The University of Lincoln
makes every effort to ensure email is sent without viruses, but cannot
guarantee this and recommends recipients take appropriate precautions.

The University may monitor email traffic data and content in accordance
with its policies and English law. Further information can be found at:
<a href="http://bit.ly/1UjOyvM.">http://bit.ly/1UjOyvM.</a>

Matthew Goddard 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:30</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139840581901">Postdoc: StockholmU.BalticBiodiversity_PlantEvol</a></div>
  <div class="description"><p>*5 Postdoctoral Fellows in Baltic Sea Research and in Plant Sciences*

**

*Closing date: April 30, 2016***

*The**Department of Ecology, Environment and Plant Sciences (DEEP) at 
Stockholm University* conducts research and education in an 
international environment. Its subject areas are marine ecology, marine 
ecotoxicology, plant ecology, plant physiology and plant systematics. 
The research is both basic and applied, often with a broad and 
interdisciplinary approach. About 150 people work at the department, 
including some 30 teachers or senior researchers, 10 postdocs and 60 PhD 
students.

We are now recruiting:

*3 Postdoctoral Fellows in Baltic Sea Research*, for example including 
any of the following topics:

PBenthic V pelagic coupling

PMitigation of marine anthropogenic effects

PClimate change effects on the Baltic Sea ecosystem

PBaltic Sea biodiversity

PCoastal-offshore interactions

*2 Postdoctoral Fellows in Plant Sciences*, for example including any of 
the following topics:

PPlant evolution

PInteractions between plants and other organisms

PClimate change effects on plant phenology and distribution

PPlant growth and development

Each position will be linked to the research group of one of the 
professors, associate professors or assistant professors at the 
department (www.su.se/deep). You should therefore first contact the 
group leader whose group you wish to join (a) to enquire whether this 
person would be interested to include you in his/her research group and 
(b) to discuss and formulate your project proposal.

Your project proposal could be a new idea of your own or it could be 
part of ongoing work in the group. The important aspect is that the 
group leader supports your application, which ensures that the 
infrastructure and knowledge to support your project is available.

For further information see: 
<a href="http://bit.ly/1UjOyfx">http://bit.ly/1UjOyfx</a><a href="http://www.su.se/deep">www.su.se/deep</a>). You should therefore first contact the 
group leader whose group you wish to join (a) to enquire whether this 
person would be interested to include you in his/her research group and 
(b) to discuss and formulate your project proposal.

Your project proposal could be a new idea of your own or it could be 
part of ongoing work in the group. The important aspect is that the 
group leader supports your application, which ensures that the 
infrastructure and knowledge to support your project is available.

For further information see: 
http://bit.ly/1UjOyfx


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:00</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139839776911">Job: UStAndrews.Bioinformatics</a></div>
  <div class="description">Dear EvolDir Community:

The School of Biology at the University of St Andrews is inviting
applications to fill a Lectureship or Readership position. We seek to
appoint someone with interests in bioinformatics, evolutionary &lsquo;omics or
quantitative biology, broadly speaking. Details and contact information
are below, and informal enquiries are welcomed.

Kind regards,
Nathan Bailey

Lectureship or Readership in Bioinformatics &ndash; University of St Andrews,
School of Biology

Appointment:  The School of Biology at the University of St Andrews
seeks to appoint a Lecturer or Reader (scale dependent on experience)
whose research and teaching interests fall within the broad area of
bioinformatics, evolutionary 'omics and quantitative biology.

Role: We seek an individual who will cultivate a funded research
programme, develop collaborations with other research groups in St
Andrews and deliver excellent science. They will play a leading role in
the St Andrews Bioinformatics Unit and the Institute for Data Intensive
Research, both of which are cross-school units dedicated to developing
modern computational resources for analysing large NGS and other 'big
data&rsquo; datasets. Research collaborations with groups in other schools
and centres at the University of St Andrews, the James Hutton Institute
and beyond present excellent opportunities for developing integrative
research programmes.

The appointee will also play a principal role in developing and
teaching bioinformatics, evolutionary genomics and similar coursework in
quantitative biology at the undergraduate and masters level. Quantitative
skills and informatics are increasingly relevant in modern biological
research, and the appointee should have the capacity to enhance our
teaching profile in these areas, as well as mentoring postgraduate
students at the Masters and PhD levels.

Qualifications: Applicants must have a PhD in a relevant field and be
able to demonstrate evidence of high-quality publications and appropriate
teaching experience. For the Lecturer post, evidence of an independent
research programme and external funding success are desirable; for the
Reader post they are essential. The School of Biology at St Andrews has
recently earned an Athena SWAN Bronze award and is committed to equality
of opportunity.

Informal enquiries can be made to Prof Malcolm White
(mfw2@st-andrews.ac.uk), tel: +44(0) 1334 463432) or Dr Nathan Bailey
(nwb3@st-andrews.ac.uk, tel: +44(0) 1334 463367).  Applications are
particularly welcome from women who are under-represented in Science
posts at the University.  You can find out more about Equality and
Diversity at <a href="http://bit.ly/1TCcKdr.">http://bit.ly/1TCcKdr.</a>

The University of St Andrews is committed to promoting equality of
opportunity for all, which is further demonstrated through its working
on the Gender and Race Equality Charters and being awarded the Athena
SWAN award for women in science, HR Excellence in Research Award and
the LGBT Charter;  <a href="http://bit.ly/1PUx06h.">http://bit.ly/1PUx06h.</a>

Please quote ref: AC5751MR

Closing Date: 22 March 2016

Link:
<a href="http://bit.ly/1TCcKds">http://bit.ly/1TCcKds</a>
School of Biology
Salary: Lecturer 38,896 - 47,801
Reader 49,230 - 55,389 per annum
Start: As soon as possible

Nathan Bailey 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>February 22, 2016</h3>
<div class="news-item">
 <div class="date">09:51</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/ematsen">@ematsen</a> wrote:</p>
            
              <p>What would you like as a topic for the next series of talks on phyloseminar? Right now we are in a series of talks on biased sampling, and then the following one is going to be about phylogenetics and (protein) structure.</p>

<p>You can see the previous topics here: <a href="http://phyloseminar.org/recorded.html">http://phyloseminar.org/recorded.html</a></p>

<p>Feel free to nominate a previous topic if you would like to hear about new developments!</p>
            
            <p>Posts: 4</p>
            <p>Participants: 3</p>
            <p><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">Read full topic</a></p></div>
  <div class="source">Source: <a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:23</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139777337271">Conference: Orlando.PopEvolQuantGenetics.Jul13-17</a></div>
  <div class="description">The Allied Genetics Conference (TAGC) will include a new meeting on
Population, Evolutionary and Quantitative Genetics organized by Mike Lynch

Although most of the central venues of TAGC revolve around the
long-standing model-organism meetings, the broad and overlapping
fields of population, evolutionary, and quantitative (PEQ) genetics
span these systems and more. Thus, in the interest of developing a fully
comprehensive genetics conference, the Genetics Society of America (GSA)
will also convene the first joint meeting for researchers in the latter
fields. Topics will range from population genomics to molecular evolution
to the dissection of the determinants of quantitative-trait variation,
with equal emphasis placed on both model and non-model organisms across
the Tree of Life. PEQ has a long history of both applied and theoretical
research, and both will be an integral part of this meeting.

The intention here is to promote a strong cross-disciplinary network of
colleagues performing both theoretical and empirical work. The meeting
will consist of a series of short talks, poster sessions, and plenary
talks, and will be dovetailed with the model-organism venues to facilitate
maximum interaction among those with similar conceptual interests. As many
as 90 oral presentations will be delivered directly in the PEQ meeting,
with dozens to hundreds of others with direct connections being given
in the other model-organism sub-meetings.  Time in the schedule will
be set aside for the usual GSA-sponsored career-development workshops,
including education-oriented, grant-writing, and networking sessions
focused on strategies for building successful teaching methodologies
and lasting collaborations.

GSA plans for PEQ to become the premier, regular gathering of graduate
students, postdoctoral scholars, and established scientists in these
central areas of genetics, ideally being established as another of the
several periodic meetings sponsored and managed by the GSA. After nearly
100 years of research in population, evolutionary, and quantitative
genetics, the time for holding such an exciting meeting is long overdue.


These topics will be featured at the conference and speakers will be
selected from submitted abstracts in the following areas:

Population Genomics
Experimental Evolution 
Genome Evolution
Quantitative Traits 
Ecological Evolution
Adaptation and Speciation
Molecular Evolution


Abstract Submission and Registration Deadline March 23, 2016

<a href="http://bit.ly/1OpeDCj">http://bit.ly/1OpeDCj</a>




CONFERENCE ORGANIZERS

 

Kirsten Bomblies, Harvard University

Michael Lynch, Indiana University

Lauren McIntyre, University of Florida

Bret Payseur, University of Wisconsin

Dmitri Petrov, Stanford University


Suzy Brown, CMP
Senior Director
Genetics Society of America
9650 Rockville Pike
Bethesda, MD 20814
301/634-7341
301/634-7079 Fax
sbrown@genetics-gsa.org



&ldquo;Brown, Suzy&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:23</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139777336986">Other:Melbourne.VolFieldAssist.Mistnetting</a></div>
  <div class="description"><p>&ndash;001a113479b81b59d5052c537982
Content-Type: text/plain; charset=UTF-8

We are looking for a volunteer field assistant to help with research on
sexual selection and seasonal colour change in superb fairy-wrens. We need
an experienced mistnetter, who is organised, respectful of animals and
enjoys being outdoors. The work involves mistnetting for superb fairy-wrens
(banding, blood samples, photography and spectral measurements) and
behavioural observations of free-living birds. Work will be conducted from
May-September, in an open woodland approximately 1 hour east of Melbourne
CBD. Our field site is in a beautiful location, with many opportunities to
see native wildlife, including kangaroos, wallabys and echidnas. The park
has many beautiful birds, such as crested shrike-tits, sacred kingfishers,
spotted pardalotes and red-browed finches. Food and local travel expenses
are compensated. Accommodation may also be provided (to be discussed with
the applicant). For more information please contact Alex McQueen:
alex.mcqueen@monash.edu

&ndash;001a113479b81b59d5052c537982
Content-Type: text/html; charset=UTF-8
Content-Transfer-Encoding: quoted-printable

</p>We are looking for a volunteer field assistant to help with research on sexual selection and seasonal colour change in superb fairy-wrens. We need an experienced mistnetter, who is organised, respectful of animals and enjoys being outdoors. The work involves mistnetting for superb fairy-wrens (banding, blood samples, photography and spectral measurements) and behavioural observations of free-living birds. Work will be conducted from May-September, in an open woodland approximately 1 hour east of Melbourne CBD. Our field site is in a beautiful location, with many opportunities to see native wildlife, including kangaroos, wallabys and echidnas. The park has many beautiful birds, such as crested shrike-tits, sacred kingfishers, spotted pardalotes and red-browed finches. Food and local travel expenses are compensated. Accommodation may also be provided (to be discussed with the applicant). For more information please contact Alex McQueen: <a href="mailto:alex.mcque%0A%20en@monash.edu">alex.mcqueen@monash.edu</a>      

&ndash;001a113479b81b59d5052c53798
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>February 21, 2016</h3>
<div class="news-item">
 <div class="date">23:38</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139775603741">Postdoc:</a></div>
  <div class="description"><p>&ndash;_000_BN1PR0401MB10732BE6494827B612FA6863A6A20BN1PR0401MB1073_
Content-Type: text/plain; charset=&ldquo;iso-8859-1&rdquo;
Content-Transfer-Encoding: quoted-printable


Postdoctoral Research Associate in Phylogenomics/Population Genomics



The Roe lab at Iowa State University seeks to recruit a Postdoctoral Research Associate with expertise in Next-Gen Sequencing/population genomics/bioinformatics. The successful applicant will collaborate with the PI to investigate the phylogenomics/population genomics of freshwater organisms (bivalves, shrimps, and fishes) For more information about the lab: <a href="http://bit.ly/20PhyeE.">http://bit.ly/20PhyeE.</a>





Candidates for the position should be able to demonstrate experience generating and analyzing genomic datasets (sequence capture, GBS, or RAD-seq); as well as experience working in a Linux/Unix shell environment; competency with at least one scripting language (e.g., Perl, Python, R). The candidate should have well-developed communication (verbal and written) and organizational skills, and leadership ability to assist the PI with the direction and training of graduate students. Qualified candiates must hold a A PhD degree in biological sciences or bioinformatics.





Terms of Appointment: Starting salary is $45,000 plus benefits. This position is for a minimum of one year and is renewable, pending satisfactory progress. The anticipated start date is June and August 2016.



Application Instructions:  For consideration, applicants must apply by April 1, 2016. Informal inquiries are encouraged prior to formal application. For formal application, please send 1) a cover letter, 2) a curriculum vitae, 3) a brief statement of research experiences/interests, and 4) names and contact information for three references to Dr. Kevin J. Roe (kjroe@iastate.edu).



Iowa State University is an Equal Opportunity/Affirmative Action employer. All qualified applicants will receive consideration for employment without regard to race, color, age, religion, sex, sexual orientation, gender identity, genetic information, national origin, marital status, disability, or protected veteran status, and will not be discriminated against. Inquiries can be directed to the Director of Equal Opportunity, 3350 Beardshear Hall, (515) 294-7612.






Kevin J. Roe, Ph.D
Department of Natural Resource Ecology and Management
Iowa State University
339 Science II
Ames, Iowa, USA 50011
<a href="http://bit.ly/18ePpZv">http://bit.ly/18ePpZv</a>


&ndash;_000_BN1PR0401MB10732BE6494827B612FA6863A6A20BN1PR0401MB1073_
Content-Type: text/html; charset=&ldquo;iso-8859-1&rdquo;
Content-Transfer-Encoding: quoted-printable


</p>

<p></p>

<p>Postdoctoral Research Associate in
Phylogenomics/Population Genomics
</p>
<p> </p>
<p>The Roe lab at Iowa State University seeks to recruit a Postdoctoral Research Associate with expertise
 in Next-Gen Sequencing/population genomics/bioinformatics. The successful applicant will collaborate with the PI to investigate the phylogenomics/population genomics of freshwater organisms (bivalves, shrimps, and fishes)
For more information about the lab:
<a href="http://bit.ly/20PhyeE.">http://bit.ly/20PhyeE.</a>
</p>
<p> </p>
<p> </p>
<p>Candidates for the position should be able to demonstrate experience generating and analyzing genomic
 datasets (sequence capture, GBS, or RAD-seq); as well as experience working in a Linux/Unix shell environment; competency with at least one scripting language (e.g., Perl, Python, R). The candidate should have well-developed communication (verbal and written)
 and organizational skills, and leadership ability to assist the PI with the direction and training of graduate students. Qualified candiates must hold a A PhD degree in biological sciences or bioinformatics. 
</p>
<p> </p>
<p> </p>
<p>Terms of Appointment: Starting salary is $45,000 plus benefits. This position is for a minimum of one
 year and is renewable, pending satisfactory progress. The anticipated start date is June and August 2016. 
</p>
<p> </p>
<p>Application Instructions: 
For consideration, applicants must apply by April 1, 2016. Informal inquiries are encouraged prior to formal application. For formal application, please send 1) a cover letter, 2) a curriculum vitae, 3) a brief statement of research experiences/interests,
 and 4) names and contact information for three references to Dr. Kevin J. Roe (kjroe@iastate.edu). 
</p>
<p> </p>
<p>Iowa State University is an Equal Opportunity/Affirmative Action employer. All qualified applicants will
 receive consideration for employment without regard to race, color, age, religion, sex, sexual orientation, gender identity, genetic information, national origin, marital status, disability, or protected veteran status, and will not be discriminated against.
 Inquiries can be directed to the Director of Equal Opportunity, 3350 Beardshear Hall, (515) 294-7612.
</p>

<p> </p>


Kevin J. Roe, Ph.D
Department of Natural Resource Ecology and Management
Iowa State University
339 Science II
Ames, Iowa, USA 50011
<a id="LPNoLP" href="http://bit.ly/18ePpZv">http://bit.ly/18ePpZv</a>
 





&ndash;_000_BN1PR0401MB10732BE6494827B612FA6863A6A20BN1PR0401MB1073
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:04</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139711149721">Conference: Austin_Texas.Evolution.Jun17-21.PhylogeneticInference</a></div>
  <div class="description">TheSociety of Systematic Biologists, SSB (<a href="http://bit.ly/1KzTlHy">http://bit.ly/1KzTlHy</a>), is
sponsoring a &ldquo;Spotlight Session&rdquo; at Evolution 2016 on &ldquo;Next-Generation
Phylogenetic Inference&rdquo;. This is a call for speakers. Spotlight sessions
are a new type of session that differ from the traditional symposia at
these meetings. For more information see links below.

Topic:Statistical phylogenetic inference methods, and in particular
Bayesian methods, are reaching a hard limit on the number of taxa that
can be used in an analysis. Although many interesting and worthwhile
efforts have been made to increase the computational efficiency of these
methods, it appears that existing inference strategies will simply not
scale to orders of magnitude more taxa. In this session we will explore
alternate strategies and algorithms for phylogenetic inference.

Thespotlight session consists of one 28 min, four 13 min, and
twelve 5 min (lightning) talks. If you are interested in speaking
in this session please submit an application to Erick Matsen at
matsen@fredhutch.org. The application
must include your: (1) Name; (2) Institution; (3) rank (e.g., student,
postdoc, faculty); (4) names & institutions of co-authors on your talk;
(5) a short abstract (max: 300 words); and (6) which talk type(s) you
are willing to give (28, 13, or 5; the more flexible you are the more
likely you are to be selected). Applications must be received by March
19th, 2016. No financial support is provided to selected speakers (i.e.,
speakers must pay their own travel, accommodation, registration, etc.).

Youmay still register to give a standard 13 min. talk in a regular
contributed session at the meeting even if you apply to participate in a
Spotlight session. However, if you are selected to speak in the spotlight
session, you will have to choose which talk slot to use as each conference
attendee is allowed to give only one oral presentation. Applicants will
be informed about the outcome of their application by late March.

Organizedsessions are committed to representing diversity. All interested
individuals are encouraged to apply.

Where:Austin, Texas.
When:June 17-21, 2016

Evolution2016 Website:
<a href="http://bit.ly/1mPReUk">http://bit.ly/1mPReUk</a>
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:49</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139710617181">Other: Washington_DC.BioBlitz.May20-21</a></div>
  <div class="description">Greetings, evolutionary biologists!

BioBlitzes are great opportunities to share your deep knowledge of natural
history with an enthusiastic audience, have fun with other biologists, and
inspire more citizen scientists.

*We are currently recruiting field biologists and naturalists* to
participate in the National Parks BioBlitz - Washington, D.C.
,
and we invite you to register now

!  If you are interested in events in Maryland and Virginia further from
D.C., there is a separate registration form

. This is a great opportunity to help document the parks biodiversity and
to share your knowledge and enthusiasm in the field. If you&rsquo;re unsure where
you fit in, watch this short video
 to help you
find your niche in BioBlitz. *The deadline to register

is
Friday, February 26.*
*We are also looking for enthusiastic nature novices and college students* to
serve as iNaturalist  Pro-Observers (link to
registration form
)
so please share this with appropriate groups and clubs (registration will
be open at least through March, unlike above).

Background: The National Geographic Society and the National Park Service
are teaming up for the 10th year to conduct a *BioBlitz
 *in parks in and around our nations capital
on *May 20-21, 2016. *BioBlitz is an *all-taxa inventory* in which teams of
volunteers, each led by a subject matter expert, find and identify as many
species as possible within a 24-hour period. The National Parks BioBlitz -
Washington, D.C. and concurrent Biodiversity Festival on the National Mall
is a cornerstone event of  the larger National Parks BioBlitz commemorating
the centennial of the National Park Service. More BioBlitzes will be held
in at least 100 national parks across the United States. BioBlitzes
happening in other National Parks will have separate registration. You can
see the growing list of parks here
.

*Please share this with others who may be interested to help make this
event a success.* You can email bioblitz@ngs.org with general questions or
cseltzer@ngs.org for questions about registering as an expert or amateur. We
hope to see you at the National Parks BioBlitz May 20-21!


*Carrie E. Seltzer, Ph.D.*

Program Manager, Experiences

t. 202.862.8239

1145 17th St., NW, Washington, DC 20036


cseltzer@ngs.org


via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:34</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/139710075221">Graduate position: OklahomaStateU.GrassGenetics</a></div>
  <div class="description">Oklahoma State University, Stillwater, Oklahoma

Ph.D. or masters positions in the genetic regulation of tillering in
grasses, combining phenotypic, genetic, and molecular approaches

Research focus:We are seeking highly motivated doctoral or masters
students to conduct research on image analysis and genetics of branching
in the grass Setaria (foxtail millet/green foxtail), as part of an
NSF Plant Genome grant focused on tillering in panicoid grasses.
The primary objective of this research is to identify components of
the gene regulatory network controlling branching in Setaria and their
responses to environmental stimuli such as shading; combining QTL mapping
with expression QTL studies, mutant analysis, and image analysis.
There is considerable flexibility for graduate projects within this
broad theme, and results will be of broad interest to maize, sorghum,
and other grass researchers.

Essential qualifications include masters degree (for PhD) or experience
in plant biology, genetics and/or molecular biology, and writing skills.
Desirable qualifications include experience in growing plants, molecular
lab techniques including working with RNA, quantitative genetics or
statistics, and programming.

Salary:  Stipend will be $21,504 per calendar year plus tuition and
health benefits.

Application deadline:  As soon as possible, although positions will
remain open until filled.

Application materials:  Please send a brief cover letter, resume,
transcripts, GRE scores, and contact information for two references to
andrew.doust@okstate.edu to be considered for these positions.

Andrew Doust
Department of Plant Biology, Ecology, and Evolution (formerly Botany)
Oklahoma State University
Physical Sciences 301
Stillwater, OK 74078

Phone: 405-744-9559
Email: andrew.doust@okstate.edu

Websites:
<a href="http://bit.ly/1JWiZBH">http://bit.ly/1JWiZBH</a>
<a href="http://bit.ly/1zi2LzF">http://bit.ly/1zi2LzF</a>
<a href="http://bit.ly/1JWiXtl">http://bit.ly/1JWiXtl</a>




&ldquo;Doust, Andrew&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="index4543.html?q=aggregator&amp;from=60">previous page</a></div><div><a href="indexbe48.html?q=aggregator&amp;from=0">1</a> <a href="index8503.html?q=aggregator&amp;from=20">2</a> <a href="index6390.html?q=aggregator&amp;from=40">3</a> <a href="index4543.html?q=aggregator&amp;from=60">4</a> <strong>5</strong> <a href="index0e96.html?q=aggregator&amp;from=100">6</a> <a href="index3362.html?q=aggregator&amp;from=120">7</a> <a href="index7b13.html?q=aggregator&amp;from=140">8</a> <a href="indexf37c.html?q=aggregator&amp;from=160">9</a> ...</div><div><a href="index0e96.html?q=aggregator&amp;from=100">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=80 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:20:40 GMT -->
</html>
