<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=320 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:26:36 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexcd0c.html?q=node/364">NSF Graduate Assistantship studying Hawaiian land snails</a></li><li><a href="index7af2.html?q=node/304">NESCent Phyloinformatics VoCamp</a></li><li><a href="index66e2.html?q=node/119">54(6) December 2005</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 6 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 2 days ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 2 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>January 6, 2016</h3>
<div class="news-item">
 <div class="date">05:21</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136742214161">Job: UMontreal.PlantEvoDevol</a></div>
  <div class="description"><p>2nd announcement

Département de sciences biologiques Faculté des arts et des sciences

Professor in Molecular development of plants

The Département de sciences biologiques is seeking applications for a
full-time tenure-track position at the rank of Assistant Professor in
Molecular development of plants, including evolutionary developmental
biology.

  *     Responsibilities

The appointed candidate will be expected to teach at all three levels of
the curriculum, supervise graduate students, engage in ongoing research
and publication, and contribute to the academic life and reputation of
the University.

  *     Requirements

     *   Ph.D. in molecular biology or in a related field

     *   Postdoctoral experiences in relevant fields

     *   High quality publication record in international journals with
         very good impact factor

     *   Interest in cross-disciplinary research and capacity to develop
         collaborations with members of

the Département de sciences biologiques and other departments of
Université de Montréal

     *   Excellent abilities to teach at the undergraduate and graduate levels

     *   Elaboration of a novel research programme that uses molecular
         genetic approaches to study plant development for a model system

     *   Priority will be given to candidates using advanced imaging
         approaches and possessing a good knowledge of plant anatomy and morphology

     *   Proficiency in French within a reasonable period

Linguistic Policy : Université de Montréal is a Québec
university with an international reputation. French is the
language of instruction. To renew its teaching faculty,
the University is intensively recruitingthe world’s best
specialists. In accordance with the institution’s language policy
[http://bit.ly/1S3Ig3q
stration/adm10-34_politique-linguistique.pdf], Université de Montréal
provides support for newly-recruited faculty to attain proficiency
in French.

  *     Salary
Université de Montréal offers a competitive salary and a complete
range of employee benefits. Salary scale

  *     Starting Date
On or after June 1st, 2016.


 Constitution of application

  *   The application must include the following documents: - a cover letter
- a curriculum vitæ
- copies of three recent publications and research

- a description of the teaching philosophy - a description of the
  research programme

  *   Three letters of recommendation are also to be sent directly
      to the Chair of the Département de sciences biologiques by
      the referees.

 Deadline
Application and letters of recommendation must be sent to the Chair of
the Département de

sciences biologiques by March 1st, 2016 at the following address :

M. Daniel Boisclair, Chair Département de sciences biologiques Faculté
des arts et des sciences Université de Montréal C. P. 6128, succursale
Centre-ville Montréal (QC) H3C 3J7

The complete application may also be sent at the following e-mail :
daniel.boisclair@umontreal.ca. For more information about the Department,
please consult its website at :

<a href="http://bit.ly/1MpkNVn.">http://bit.ly/1MpkNVn.</a>
via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:06</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136741772556">Postdoc: MonashU.EvolutionaryEcol</a></div>
  <div class="description"><p>Research Fellow Opportunity (Biological Sciences) at Monash University

Evolutionary Ecology

The Opportunity

Professor Dustin Marshall (www.meeg.org) is seeking an evolutionary
ecologist to explore the dynamics of biological systems, specifically
how size and shape affect the ways resources move through populations
and communities.

As a postdoctoral researcher, you will explore how biological entities
change their function as they change in size and shape, using empirical
and/or theoretical approaches. The research will be conducted in
collaboration with Marshall as part of the Centre for Geometric Biology
at Monash University.

You will further be expected to maintain consistently high research
output in the form of quality publications, supervision of students,
development and submission of grant proposals to external funding
agencies, contribute more generally to communicating the research
activities of the group, and participation in appropriate career
development activities.

If you believe you can fulfil these requirements, you are strongly
encouraged to apply.

This role is a full-time position; however, flexible working
arrangements may be negotiated.

Your application must address the selection criteria.

Enquiries

Professor Dustin Marshall, +61 3 9902 4449
or dustin.marshall@monash.edu

More information can be found at;

<a href="http://bit.ly/1Oaoz3O">http://bit.ly/1Oaoz3O</a><a href="http://www.meeg.org/">www.meeg.org</a>) is seeking an evolutionary
ecologist to explore the dynamics of biological systems, specifically
how size and shape affect the ways resources move through populations
and communities.

As a postdoctoral researcher, you will explore how biological entities
change their function as they change in size and shape, using empirical
and/or theoretical approaches. The research will be conducted in
collaboration with Marshall as part of the Centre for Geometric Biology
at Monash University.

You will further be expected to maintain consistently high research
output in the form of quality publications, supervision of students,
development and submission of grant proposals to external funding
agencies, contribute more generally to communicating the research
activities of the group, and participation in appropriate career
development activities.

If you believe you can fulfil these requirements, you are strongly
encouraged to apply.

This role is a full-time position; however, flexible working
arrangements may be negotiated.

Your application must address the selection criteria.

Enquiries

Professor Dustin Marshall, +61 3 9902 4449
or dustin.marshall@monash.edu

More information can be found at;

http://bit.ly/1Oaoz3O
cal-sciences-/534851

&ldquo;
via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:50</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136741281046">Postdoc:UFlorida.Ecology.EnvironmentalScience</a></div>
  <div class="description"><p>&ndash;_000_4EA06A06431642DD9BD02DF6673DAF96ufledu_
Content-Type: text/plain; charset=&ldquo;us-ascii&rdquo;
Content-Transfer-Encoding: quoted-printable

Postdoctoral Fellow sought at UF, application deadline 25 January 2016.

The International Center at the University of Florida is pleased to invite applications for the John J. and Katherine C. Ewel Fellowship Program in Ecology and Environmental Science in the Tropics and Subtropics. This program will enable a recent doctorate recipient to study ecology and environmental sciences at the University of Florida (UF). The Fellow will work for two years with a University of Florida faculty member in any department to create new knowledge in any area of tropical and subtropical ecology. For complete details, criteria and application for, please visit <a href="http://bit.ly/1Rr0DgX">http://bit.ly/1Rr0DgX</a>


=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-Marta L. Wayne, Ph. D.
Professor and Chair
P.O. Box 118525
Department of Biology
University of Florida
Gainesville, FL 32611-8525
(courier: 876 Newell Drive)
vox:  352-392-9925
fax:  352-392-3704
<a href="http://bit.ly/1PeG2bi">http://bit.ly/1PeG2bi</a>


&ndash;_000_4EA06A06431642DD9BD02DF6673DAF96ufledu_
Content-Type: text/html; charset=&ldquo;us-ascii&rdquo;
Content-ID: 
Content-Transfer-Encoding: quoted-printable


</p>
Postdoctoral Fellow sought at UF, application deadline 25 January 2016. 
<br class=""/>
The International Center at the University of Florida is pleased to invite applications for the John J. and Katherine C. Ewel Fellowship Program
 in Ecology and Environmental Science in the Tropics and Subtropics. This program will enable a recent doctorate recipient to study ecology and environmental sciences at the University of Florida (UF). The Fellow will work for two years with a University of
 Florida faculty member in any department to create new knowledge in any area of tropical and subtropical ecology. For complete details, criteria and application for, please visit <a href="http://bit.ly/1Rr0DgX" class="">http://bit.ly/1Rr0DgX</a>
<br class=""/>
<br class=""/>









=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Marta L. Wayne, Ph. D.
Professor and Chair
P.O. Box 118525
Department of Biology
University of Florida
Gainesville, FL 32611-8525
(courier: 876 Newell Drive)
vox:  352-392-9925
fax:  352-392-3704
<a href="http://bit.ly/1PeG2bi" class="">http://bit.ly/1PeG2bi</a>



<br class=""/>



&ndash;_000_4EA06A06431642DD9BD02DF6673DAF96ufledu
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:50</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136741281231">Graduate position: UppsalaU.SpongeMetabolomics</a></div>
  <div class="description">Ph.D. position on Metabolomics/Genomics of Deep Sea Sponges,

Dept. of Medicinal Chemistry, Uppsala University, SWEDEN.

I am seeking a highly motivated PhD candidate to work within the frame of
the newly EU funded SponGES consortium. This consortium will investigate
deep-sea north Atlantic sponge grounds. The candidate will develop a
suite of genomic and metabolomic resources to investigate the population
diversity of key sponge species and assess connectivity between sponge
grounds. These resources will also unlock the potential of sponge grounds
for drug discovery, in which the candidate will also play a part.

For more info and to apply, use the link below (in Swedish,
for the English version click on &ldquo;English&rdquo;, upper-right corner).
<a href="http://bit.ly/1Ra00ID">http://bit.ly/1Ra00ID</a>

duration of position: possibly 5 years

application deadline: 18th of January 2016

For specific information about the announced Ph.D. position and details
of the project, please contact me (paco.cardenas@fkog.uu.se).


Dr. Paco Cardenas
Div. of Pharmacognosy, Dept. of  Medicinal Chemistry, Uppsala University
BMC Box 574,
SE 75123 Uppsala, Sweden
paco.cardenas@fkog.uu.se


Paco Cardenas 
via Gmail </div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">04:03</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136739974441">Job: UNSW_Australia.Professor_Bioinformatics_Systems</a></div>
  <div class="description"><p>The University of New South Wales, Australia invites applications for 
Professor (Level E) in the School of Biotechnology and Biomolecular 
Sciences in the field of  Bioinformatics, Systems Biology and/or Medical 
Genomics.

For more information and instructions for applying please visit

<a href="http://bit.ly/1ONkAJO">http://bit.ly/1ONkAJO</a>

The School of Biotechnology and Biomolecular Sciences (BABS) is part of 
the Science Faculty at UNSW Australia and is located on our Kensington 
campus in Sydney. BABS hosts the Ramaciotti Centre for Genomics, which 
has world class facilities for sequencing (including MiSeq, NextSeq, 
PacBio and access to Illumina X-Ten), microfluidics (including single 
cell genomics) and microarray analysis. The School hosts the New South 
Wales Systems Biology Initiative, supported by high-performance 
computing facilities, including national supercomputers. The School also 
has access to outstanding proteomics and metabolomics facilities at the 
UNSW Bioanalytical Mass Spectrometry Facility. Additional major 
facilities at the University provide capacity for biochemical, 
biophysical and chemical analyses, electron microscopy and animal breeding.

For more information about the School, please visit: 
<a href="http://bit.ly/1OM8JSz">http://bit.ly/1OM8JSz</a>

Application close:  26 February 2016


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:40</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136737747426">Other: Paid Summer Research (REU) at UMich Biological Station</a></div>
  <div class="description"><p>The University of Michigan Biological Station is now accepting
applications for its Research Experience for Undergraduates (REU)
program, &ldquo;Biosphere-Athmosphere-Hydrosphere Interactions in a Changing
Global Environment.&rdquo;

Our REU summer program runs for nine weeks (June 21 - August 20, 2016)
at the UM Biological Station on Douglas Lake in beautiful northern
Michigan. Each REU participant will

&ndash; Work closely with a research mentor on a project of mutual interest.
&ndash; Design, conduct, analyze, and report on their research project,
with guidance from the mentor and the two program co-directors.
&ndash; Participate in workshops and group discussions designed to provide
the background knowledge and technical tools needed to carry out
interdisciplinary scientific research.
&ndash; Contribute to the body of knowledge that can help address many of
Earth&rsquo;s most important environmental challenges.

COMPENSATION: Participants receive a stipend of $5,000 plus free room
and board at the Station.

DEADLINE: The deadline to apply is Monday, February 1, 2016.

Application and details at: <a href="http://goo.gl/GwYGNx.">http://goo.gl/GwYGNx.</a>

The University of Michigan Biological Station is located on Douglas
Lake near Pellston, Michigan at the tip of the lower peninsula. Its
campus is situated amid 10,000 undeveloped acres owned by the Station.
Students have quick access to many unique habitats in the region
including dune, alvar, bog, and old-growth forest. The Station also
has 2 atmospheric gas sensing towers for monitoring forest and
atmospheric gas. More information about the Station is available at
<a href="http://bit.ly/1ON670A.">http://bit.ly/1ON670A.</a>



via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:55</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136736457841">Postdoc: CIBIO_Portugal.BirdPopulationDynamics</a></div>
  <div class="description">REN Biodiversity Chair Post-Doc Fellowship: Power Line Impacts on Bird
Ecology and Population Dynamics

BPD-REN-ECOL

Reference: BPD-REN-ECOL
Scientiic area: Biological sciences

Link to the announcement

One post-doctoral fellowship is available at CIBIO-InBIO, funded by the
REN Biodiversity Chair, in the field of power line impacts on bird
ecology and population dynamics, under the following conditions:

Admission requirements

The post-doc fellowship is suitable for candidates of any nationality
holding a PhD degree in biology or ecology. The candidates should
preferentially have a strong background in bird ecology.

Work overview

The candidate will integrate the REN Biodiversity Chair team, and will
be supervised by the Chair holder, Francisco Moreira. Tasks to
undertake include:
 i. carrying out studies on the ecology and movements of white storks,
    including bird capture and tracking;
ii. assessing the impacts of powerlines on population viability on the
    target species, as well as other possible case studies (e.g. large
    eagles, bustards) by using population viability analysis and other
    demographic simulation approaches;
iii. assessing spatial patterns of use of power line pylons by nesting
    storks;
iv. supporting the implementation of a citizen science project on white
    storks, and gathering information on other citizen science
    initiatives.

The work will include data collection, statistical analysis, paper
writing and potential project submission.

Fellowship
The duration of the fellowship is 12 months, renewable up to a maximum
of 24 months.

Salary
Monthly stipend is â‚¬1495 according to the stipends established by
FCT, I.P. in Portugal (<a href="http://bit.ly/1dRsu4w">http://bit.ly/1dRsu4w</a>).
Payment will be made by bank transfer on a monthly basis.

Applicable legislation

Estatuto do Bolseiro de InvestigaÃ§Ã£o CientÃfica, aprovado pela Lei
n.Âº 40/2004, de 18 de Agosto, alterado e republicado pelo Decreto-Lei
n.Âº 202/2012, de 27 de Agosto, alterado ainda pelo Decreto-Lei n.Âº
233/2012, de 29 de Outubro, pela Lei n.Âº 12/2013, de 29 de Janeiro, e
pelo Decreto-lei n.Âº 89/2013, de 9 de Julho; Regulamento de Bolsas e
InvestigaÃ§Ã£o da FundaÃ§Ã£o para a CiÃªncia e a Tecnologia, IP (FCT),
em vigor (<a href="http://bit.ly/1NFARV6">http://bit.ly/1NFARV6</a>) e
de

acordo com os Estatutos de Bolsa do ICETA aprovados pela FCT.

Work place

Work will be conducted at CIBIO-InBIO branch located at Lisbon,
Portugal.

Application

The call for applications is open between January 18 and February 1,
2016 (24:00 GMT).

Applications should be submitted by email
to bolsas@cibio.up.pt (indicating the above mentioned reference) and
will include:

a)      Curriculum vitae (5 pages maximum, including a publication list
and other relevant information in the context of the project research
goals);

b)      Motivation Letter (including a brief description of research
experience and why you are suitable for the announced position).

Selection criteria

The jury panel will select the best candidate based on his/her merit,
through the analysis of the CV. Preference will be given to candidates
with the following requirements:
 i. field experience on capture and tracking of storks (preferentially)
    or other large birds;
ii. knowledge of methods for carrying out survival analysis and
    simulations of population dynamics;
iii. experience in Geographic Information Systems (GIS) and R;
iv. experience in citizen science projects;
 v. papers in ISI journals with peer-review.

The top 3 candidates will be invited for an interview (weight of 50%),
in person or by video-conference, to establish the final ranking.

Jury Panel

Francisco Moreira (Chair), InÃªs Catry (vowel), Pedro Beja (vowel), and
JoÃ£o Paulo Silva (substitute vowel).

Results announcement

Candidates will be directly informed by email about the result of their
application. The ranking of the candidates will be published at a
visible and public area of ICETA facilities.

CIBIO-InBIO Divulgação 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:11</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136735039536">Postdoc: ColdSpringHarborLab.ComputationalGenomics</a></div>
  <div class="description">POSITION DESCRIPTION:
Postdoctoral Positions in Computational Genomics. Postdoctoral
positions are available in Dr. Adam Siepel&rsquo;s research group in its new
location at the Simons Center for Quantitative Biology at Cold Spring
Harbor Laboratory. The Siepel Group specializes in the development of
probabilistic models, algorithms for inference, prediction methods, and
application of these methods in large-scale genomic data analysis. Of
particular interest is research relevant to existing, NIH supported
projects in (1) HUMAN POPULATION GENOMICS, including demography inference
using Bayesian coalescent-based methods, inference of natural selection
on regulatory and other noncoding sequences, and prediction of fitness
consequences for noncoding mutations; and (2) TRANSCRIPTIONAL REGULATION
in mammals and Drosophila, including the estimation of rates and
patterns of transcriptional elongation from GRO-seq data, prediction of
transcription factor binding sites from DNase-seq data, and regulatory
network inference based on joint patterns of transcription and binding
in inducible systems. The research on transcriptional regulation will
continue to be carried out in close collaboration with Dr. John Lis at
Cornell University.

RELEVANT RECENT PAPERS:
Kuhlwilm M, Gronau I, Hubisz MJ, de Filippo C, Prado J, et al. Ancient
gene flow from modern humans into Siberian Neandertals. Nature (in
press), 2015.

Danko CG, Hyland SL, Core LJ, Martins AL, Waters CT, et al. Identification
of active transcriptional regulatory elements from GRO-seq
data. Nat. Methods. 2015. 12(5):433-438. PMCID: PMC4507281.

Rasmussen MD, Hubisz MJ, Gronau I, Siepel A. Genome-wide inference of
ancestral recombination graphs. PLOS Genet. 2014. 10(5): e1004342. PMCID:
PMC4022496.

Freedman AH, Gronau I, Schweizer RM, Ortega-Del Vecchyo D, Han E, et
al. (30 co-authors). Genome sequencing highlights the dynamic early
history of dogs. PLOS Genet. 2014. 10(1):e1004016. PMCID: PMC3894170.

Core LJ, Martins AL, Danko CG, Waters CT, Siepel A, et al. Analysis of
nascent RNA identifies a unified architecture of initiation regions at
mammalian promoters and enhancers. Nat. Genet. 2014. 46(12):1311-1320
PMCID: PMC4254663.

Capra JA, Hubisz MJ, Kostka D, Pollard KS, Siepel A. A model-based
analysis of GC-biased gene conversion in the human and chimpanzee
genomes. PLOS Genet. 2013. 9(8):e1003684. PMCID: PMC3744432.

Arbiza L, Gronau I, Aksoy BA, Hubisz MJ, Gulko B, Keinan A, Siepel
A. Genome-wide inference of natural selection on human transcription
factor binding sites. Nat. Genet. 2013. 45(7):723-729. PMCID: PMC3932982.

Gronau I, Arbiza L, Mohammed J, Siepel A. Inference of natural
selection from interspersed genomic elements based on polymorphism and
divergence. Mol. Biol. Evol. 2013. 30(5):1159-1171. PMCID: PMC3697874.

Gronau I, Hubisz MJ, Gulko B, Danko CG, Siepel A. Bayesian
inference of ancient human demography from individual genome
sequences. Nat. Genet. 2011. 43(10):1031-1034. PMCID: PMC3245873.

The ideal candidate will be enthusiastic, highly motivated and
independent; will have a strong background in comparative genomics,
population genomics, or transcriptional regulation, a serious
interest in molecular biology and genetics combined with good skills in
mathematics and computer science, and a strong record of accomplishment in
research. Postdoctoral associates in the group are expected to participate
in both methods development and data analysis, to be comfortable in
a fast-moving, interactive, and collaborative research environment,
and to actively publish and present results.

REQUIRED QUALIFICATIONS:
Ph.D. in bioinformatics, computer science, statistics, genetics, molecular
biology, applied mathematics, or a related field.

Research experience (with first-author publications) in computational
genomics or a closely related field.

Fluency in probabilistic modeling and computational statistics.
Proficiency in programming, ideally in C or C++, as well as in scripting
languages such as python, Perl, mat lab, or R. Should be comfortable in a
Linux environment, with large data sets, computer clusters, and databases.

JOB LOCATION: CSHL, One Bungtown Road, Cold Spring Harbor, NY 11724

TO APPLY: Position Number
01229-R. <a href="http://bit.ly/1JtlFu9.">http://bit.ly/1JtlFu9.</a>


&ldquo;Gill, Irene&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:55</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136734486856">Conference: Montpellier.MathCompEvolutionaryBiology.Jun12-16</a></div>
  <div class="description">*** REGISTER AGAIN IF YOU DIDNT RECEIVE ANY CONFIRMATION EMAIL
*** OUR WEBSITE HAD MALFUNCTIONS, SORRY &hellip;

*** PLEASE DIFFUSE THIS ANNOUNCEMENT, THANKS!

MCEB - Mathematical and Computational Evolutionary Biology
12-16 June 2016 - Hameau de l'Etoile, Montpellier, South of France.

Webpage: <a href="http://bit.ly/1Rr4Lxy">http://bit.ly/1Rr4Lxy</a>

Pre-registration deadline: February 25th
Notification to applicants: March 15th
Final list of attendees: April 15th

Scope: Mathematical and computational tools and concepts form an
essential basis for modern evolutionary studies. The goal of the MCEB
conference (at its 8th edition) is to bring together scientists with
diverse backgrounds to present recent advances and discuss open
problems in the field of mathematical and computational evolutionary
biology. The theme of this year’s edition will be &ldquo;forecasting
evolution&rdquo;: while most analyses are designed to reconstruct the past
in order to understand evolution, our modern society is in dire need
of anticipating the future using sound scientific approaches. This is
particularly true in the context of global warming and more generally
global changes. New approaches that model future evolution are
emerging, for example to predict epidemics, evolution of cancer,
shifts in species/population distributions or species loss.
Evolutionary biology is thus widening its focus, trying to predict
future biological states and better define the range of applicability
of these predictions. Making progress in this new research area will
involve cross-disciplinary approaches, accurate statistical methods,
and powerful computational techniques. Beyond this year’s theme,
general concepts, models, methods and algorithms will be presented
and discussed, just as in the previous editions of MCEB. Keynote
speakers will introduce a field of research and discuss their own
work in this field. Afternoon will be for short presentations and
posters, with plenty of time for discussions and outdoors.

Where and when: Hameau de l'Etoile, near Montpellier, in the South of
France, 12-16 June 2016.

Cost: Conference fees including accommodation for four nights, meals,
coffee breaks, buses, etc., will be between 300€ and 550€, all
inclusive, and will vary depending on the room. PhD students and
postdocs will benefit of the cheapest rooms.

Keynote speakers:
Jeffrey Jensen - <a href="http://bit.ly/1kpTAqI">http://bit.ly/1kpTAqI</a>
Ecole Polytechnique Fédérale de Lausanne, CH

Laura S. Kubatko - <a href="http://bit.ly/1Rr4J91">http://bit.ly/1Rr4J91</a>
Departments of Statistics and Evolution, Ecology, and Organismal
Biology - The Ohio State University, USA

Amaury Lambert - <a href="http://bit.ly/1Z4N6wt">http://bit.ly/1Z4N6wt</a>
Stochastics & Biology Group - Laboratoire de Probabilités & Modèles
Aléatoires - UPMC - Université Paris 06, FR

Richard Neher - <a href="http://bit.ly/1Rr4J92">http://bit.ly/1Rr4J92</a>
Max Planck Institute for Developmental Biology - DE

Bruce Rannala - <a href="http://bit.ly/1Rr4J93">http://bit.ly/1Rr4J93</a>
UC Davis - University of California, USA

Isabel Sanmartín Bastida -
<a href="http://bit.ly/1Z4N5IR">http://bit.ly/1Z4N5IR</a> 

Real Jardín Botánico (RJB-CSIC), Madrid, SP

Yun S. Song - <a href="http://bit.ly/1Rr4J95">http://bit.ly/1Rr4J95</a>
Departments of Mathematics and Biology - Universty of Pennsylvania, USA

For more information and register: <a href="http://bit.ly/1Rr4Lxy">http://bit.ly/1Rr4Lxy</a>

Please forward this announcement

Olivier Gascuel 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:25</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136733359966">Other: Replies to Tree with bootstraping from distance matrix</a></div>
  <div class="description"><p>Hi,

Following my question about bootstraping a distance matrix to produce a 
phylogenetic tree with bootstrap confidence values, I got many useful 
replies. My original question is below and the replies summary is 
further down this message.

On 15-12-18 10:57 AM, Eric Normandeau wrote:
> Hi,
>
> I&rsquo;m creating a tree from a distance matrix using phylip neighbor and 
> phylib drawgram. I would like to have bootstrap values for the 
> branches but all I have is this one distance matrix. Is this possible?
>
> The distance matrix is calculated from a Genotyping by Sequencing 
> (GBS) dataset with a few thousand SNPs. For each individual pair, the 
> distance is basically the proportion of genotypes that differ for loci 
> where both individuals are genotyped.
>
> I could easily create different distance matrices by randomly sampling 
> only some of the SNPs to calculate the distances, but that would not 
> make much sense. For example, if I use a low proportion of the SNPs 
> (ie: 5%), the bootstrap values will end up being lower because the 
> matrices will be more different from one another than if I use a high 
> proportion of the SNPs (ie: 50+%). It feels like I can control the 
> bootstrap values artificially, so that is not a good avenue.
>
> Is there a way, given a single distance matrix, to create a tree with 
> bootstrap values?
>
> I&rsquo;ll be happy to summarize the answers and post them back on EvolDir.
>
> Eric
>
Here is the summary of the very useful replies:

1) I cannot bootstrap the distance matrix
2) I can bootstrap the markers (random choice with replacement) and 
produce 1000 distances matrices
3) I was describing a jackknife procedure, not a bootstrap procedure

David Remington also suggested that I could use a nucleotide 
substitution model to calculate a more appropriate distance metric. 
However, he mentions that this approach wouldn&rsquo;t work if the SNP dataset 
has been pre-filtered, for example by retaining only the variable 
markers, as it would make the nucleotide substitution model incorrect. 
In our case, we have to work with pre-selected SNPs, otherwise we would 
be using hundreds of thousands of SNPs with a lot of missing data and 
most of which would be variable in only one individual, thus more 
probably sequencing errors.

David also asked if this was a within-species study and it is not. It 
uses specimens from a variety of related species with low numbers of 
samples per species. That is why we went for a phylogenetic approach.

Jack Cameron suggested using a Canberra distance, which is a robust 
Manhattan metric and then to use Pvclust to bootstrap once I have the 
distance matrices. The Canberra distance computes a distance between two 
vectors P an Q of numbers by computing the sum of, for each index i, 
(|Pi - Qi|) / (|Pi| + |Qi|). Since my pairwise comparisons between two 
samples use all the loci for which both samples were genotyped, and that 
the length of the the P and Q vectors vary among the pairwise 
comparisons, I would then divide by the length to normalize them. This 
leaves the problem of encoding the possible genotypes (AA, AB, BB) as 
numbers, probably with AA=1, AB=2 and BB=3. In this context, however, I 
do not see how dividing by (Pi + Qi) makes any sense. It basically gives 
more weight to a case of AA-AB than to a case of AB-BB since the later 
will be divided by 5, compared to 3 for the former. My implementation of 
the distance is equivalent to using only the numerator (|Pi - Qi|) part 
of the Canberra distance and then dividing the sum by the length of the 
vectors.

Any further thoughts on using a different distance metric would be 
highly welcome!

I will then implement the following:

1) Use my VCF file with my SNP markers and bootstrap it 1000 times
2) Use Phylip to compute the concensus tree and bootstrap confidence values
3) Keep an eye out for how to use a different distance metric


Many thanks to all who replied to my question (in chronological order of 
replies):

- Bernhard Haubold
- Peter Smouse
- Francisco Bilbao Moore
- David Remington
- Murray Cox
- Jack Cameron
- Frank E. Anderson
- Mary Kuhner
- Miguel Navascués
- Louis Ranjard


Eric Normandeau - Bioinformaticien
Laboratoire de Louis Bernatchez
Ressources Aquatiques Québec (RAQ)

Institut de Biologie Intégrative et des Systèmes (IBIS)
Pavillon Charles-Eugène-Marchand
1030, Avenue de la Médecine
Local 1143
Université Laval
Québec (Québec) G1V 0A6
Canada

Tél: 418 656-2131 poste 8455
Courriel: eric.normandeau@bio.ulaval.ca







via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:25</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136733359626">Postdoc: UFlorida.TropicsEvolution</a></div>
  <div class="description"><p>The International Center at the University of Florida is pleased to invite
applications for the John J. and Katherine C. Ewel Fellowship Program in
Ecology and Environmental Science in the Tropics and Subtropics. This
program will enable a recent doctorate recipient to study ecology and
environmental sciences at the University of Florida (UF). The Fellow
will work for two years with a University of Florida faculty member
in any department to create new knowledge in any area of tropical and
subtropical ecology. For complete details, criteria and application for,
please visit <a href="http://bit.ly/1Rr0DgX">http://bit.ly/1Rr0DgX</a>


Marta L. Wayne, Ph. D.
Professor and Chair
P.O. Box 118525
Department of Biology
University of Florida
Gainesville, FL 32611-8525
(courier: 876 Newell Drive)
vox:  352-392-9925
fax:  352-392-3704
<a href="http://bit.ly/1PeG2bi">http://bit.ly/1PeG2bi</a>



mlwayne@ufl.edu


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:10</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136732754791">Graduate position: NTNU_Trondheim.TelomereLifeHistoryDynamics</a></div>
  <div class="description">A PhD position is available at the Department of Biology at the
Norwegian University of Science and Technology (NTNU) in Trondheim
on the functional relationships between telomere dynamics, physiology
and life history traits in a small passerine bird, the House Sparrow
(Passer domesticus).  We are seeking a highly qualified candidate
with a background in either evolutionary, behavioral or population
ecology or eco-physiology. It is an advantage if the candidate has
laboratory experience in genetic analyses.  The aim of the PhD project
will be to investigate central questions related to the functional
associations between telomere dynamics, physiological parameters and
individual behavior (&ldquo;personality&rdquo;), and how these are associated
with trade-offs between life history traits such as growth rate,
body size, reproduction and lifespan. Read more about the project here:
<a href="http://bit.ly/1VHszxI">http://bit.ly/1VHszxI</a>
Here is the link to the formal announcement:
<a href="http://bit.ly/1Z7Cjqs">http://bit.ly/1Z7Cjqs</a>
Application deadline is 1st February. For further information contact:
thor.h.ringsby@ntnu.no.




Thor Harald Ringsby 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>January 5, 2016</h3>
<div class="news-item">
 <div class="date">03:50</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136671362896">Other: Chromonomer.Software</a></div>
  <div class="description"><p>Hi All,

I&rsquo;m pleased to release the Chromonomer software package today:

<a href="http://bit.ly/1PHJa29">http://bit.ly/1PHJa29</a>

Chromonomer is software to integrate a genome assembly with a genetic 
map to create a &lsquo;chromonome&rsquo; (1). The software will order and orient 
scaffolds from a genome assembly according to the markers from a genetic 
map. Chromonomer was designed around RADseq genetic maps but should work 
with any genetic map where the markers can be aligned to the genome 
sequence.

Chromonomer will break scaffolds to fill gaps in the assembly with 
smaller scaffolds or to correct mis-joins in the assembly. It will also 
identify markers that are out of order with respect to the assembly. In 
this way the software uses the assembly order on the small scale and the 
map order on the large scale to integrate an assembly and a map together.

Chromonomer generates extensive documentation of where any 
incompatibilities occur between the assembly and the map, including 
(optional) web-based visualizations.

The software is released under the GPL free software license.

Please try it and let us know your experience with it.

Best wishes,

julian

(1) <a href="http://bit.ly/1IO1Hdz">http://bit.ly/1IO1Hdz</a>

via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:35</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136670954146">Postdoc: Yeast evolutionary genomics in University of Michigan</a></div>
  <div class="description"><p>&ndash;94eb2c06bef0c79f2b05288b036f
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: quoted-printable

A postdoctoral position is available immediately in the laboratory of
Jianzhi “George” Zhang at University of Michigan, Ann Arbor, Michigan. The
ideal candidate will use the budding yeast Saccharomyces cerevisiae and its
relatives as model organisms to study evolutionary processes. Potential topics
include but are not limited to (1) the fitness effects of various mutations
including gene duplication, (2) genic/genomic basis of reproductive
isolation, (3) evolution of dominance, (4) genetic mechanisms of heterosis,
(5) evolution of gene essentiality, and (6) evolution of gene expression
level and noise. The position requires a motivated individual with an
interest in evolutionary genetics and wet lab experience in molecular
genetics or genomics. Prior training in yeast genetics is a plus.

Applicants should email a short statement of research interests, CV,
and contact
information of three references to jianzhi@umich.edu. For further information
about the Zhang lab, see <a href="http://bit.ly/1MQtrt6.">http://bit.ly/1MQtrt6.</a>

&ndash;94eb2c06bef0c79f2b05288b036f
Content-Type: text/html; charset=UTF-8
Content-Transfer-Encoding: quoted-printable

</p>A postdoctoral position is available immediately in the laboratory of Jianzhi “George” Zhang at University of Michigan, Ann Arbor, Michigan. The ideal candidate will use the budding yeast Saccharomyces cerevisiae and its relatives as model organisms to study evolutionary processes. Potential topics include but are not limited to (1) the fitness effects of various mutations including gene duplication, (2) genic/genomic basis of reproductive isolation, (3) evolution of dominance, (4) genetic mechanisms of heterosis, (5) evolution of gene essentiality, and (6) evolution of gene expression level and noise. The position requires a motivated in
 dividual with an interest in evolutionary genetics and wet lab experience in molecular genetics or genomics. Prior training in yeast genetics is a plus.<br><br>Applicants should email a short statement of research interests, CV, and contact information of three references to <a href="mailto:jianzhi@umich.edu">jianzhi@umich.edu</a>. For further information about the Zhang lab, see <a href="http://bit.ly/1MQtrt6" rel="noreferrer" target="_blank">http://bit.ly/1MQtrt6</a>.<br>


&ndash;94eb2c06bef0c79f2b05288b036
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">03:20</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136670546046">Job: AuburnU.2.ViralPlantOmics</a></div>
  <div class="description"><p>Beloware two positions At Auburn University as part of an -Omics and
Informatics Cluster hire

Oneis virology/viromics in the Department of Biological Sciences and
the other is molecular ecology in a the department of Plant Pathology.

Deadlinesare soon.

1)Virology/Viromics
TheDepartment of Biological Sciences at Auburn University invites
applications for a tenure-track faculty position (9-month) at the
level of Assistant or Associate Professor beginning Fall of 2016. This
position is part of an Auburn University cluster hire in &ldquo;omics&rdquo; and
informatics and will strengthen our departmental research core area of
host-microbial interactions. The successful candidate is expected to
establish an extramurally funded, internationally recognized research
program. Instructional responsibilities include development of graduate
and/or undergraduate courses in virology including informatics and
related topics.

TheOmics and Informatics Cluster represents an interdisciplinary
research initiative at Auburn University which has made a significant
financial commitment to hiring multiple new faculty. The successful
candidate for this position will be expected to participate actively in
Auburn University&rsquo;s multidisciplinary Cluster Hires Initiative in the
Omics and Informatics cluster. New faculty with expertise in viromics
will complement the University&rsquo;s considerable existing expertise in
these areas to form the core of this innovative multidisciplinary
initiative. Participation in the Omics and Informatics cluster will be
an important component in faculty annual reviews. For more information
regarding the Cluster Hires please click on the following link:
<a href="http://bit.ly/1Re5pAj">http://bit.ly/1Re5pAj</a>

Applicantsshould submit a 1 page cover letter emphasizing specific
qualifications, a curriculum vitae, a description of research interests,
a statement of teaching philosophy/experience, and names and contact
information of at least 3 references.

Moreinformation can be found at: <a href="http://bit.ly/1waOwJI">http://bit.ly/1waOwJI</a>

MinimumQualifications
Applicantsmust have a Ph.D. in an appropriate discipline and experience
in virology and informatics. Excellent written and interpersonal
communications skills are required. Qualifications include postdoctoral
or professional experience, a strong record of publication and potential
for funding. The candidate selected for this position must be able to
meet eligibility requirements for work in the United States at the time
the appointment is scheduled to begin and continue working legally for
the proposed term of employment.

DesiredQualifications
Applicantsat the level of Associate Professor are expected to have current
funding. Applicants studying host-viral interactions using informatics
are encouraged to apply.

SpecialInstructions to Applicants
Reviewof applications will begin January 8th, 2016 and will continue
until the position is filled.

2)Plant Pathology - Molecular ecology

TheDepartment of Entomology and Plant Pathology at Auburn University and
Auburn University&rsquo;s Cluster Hiring Initiative (CHI)  Omics and Informatics
Cluster  is seeking applicants for the position of Assistant Professor
in MOLECULARECOLOGY of PLANT and SOIL SYSTEMS. The expected start date
is August 16, 2016.

Responsibilities:This is a 9 month tenure-track position with 75% research
and 25% instruction responsibilities. The successful candidate for this
position will be expected to participate actively in Auburn University
multidisciplinary Cluster Hires Initiative in the cluster of Omics
and Informatics. New faculty with expertise in Molecular Ecology of
Plant and Soil Systems will complement the University&rsquo;s considerable
existing expertise in these areas to form the core of this innovative
multidisciplinary initiative. Participation in the Omics and Informatics
will be an important component in faculty annual reviews. For more
information regarding the Cluster Hires please click on the following
link: <a href="http://bit.ly/1Re5pAj">http://bit.ly/1Re5pAj</a>

Thesuccessful candidate will also be expected to build and lead an
extramurally-funded research program focused on fundamental aspects of
Molecular Ecology of Plant and Soil Systems. The successful candidate
(Research/Teaching) selected for this position will lead a research
program based on understanding population dynamics of microbes
associated with crops of agricultural importance for Alabama and the
U.S. The candidate will use diverse bioinformatic approaches such as
genomics, metagenomics, population genomics and/or phylogenomics to
understand microbiome/plant interactions. Understanding the microbiome
(including pathogenic, beneficial and saprophytic microorganisms) is
important for epidemiological studies and to develop disease management
strategies to eliminate or reduce the impact of plant pathogens. The
incumbent will be expected to secure extramural funding to support
programs and summer salary, publish technical and refereed articles,
and collaborate effectively with departmental faculty and colleagues in
other disciplines. Instructional responsibilities will include teaching
undergraduate and graduate classes in the department curriculum. In
addition, the candidate will be expected to direct and advise graduate
students. Service to the department, college and university are expected.

Qualifications:An earned doctoral degree in Plant Pathology or a
related scientific discipline at the time of hire; evidence of research
productivity (publications, grants, etc.) commensurate with career level;
ability to develop an original research program on fundamental aspects
of plant-pathogen interactions using genomics and informatic approaches;
and evidence of effective teamwork and capability to teach undergraduate
and graduate level courses. In addition, candidate must possess excellent
organizational, written, and interpersonal communication skills. Candidate
must demonstrate subject matter knowledge and the ability to communicate
effectively. The successful candidate must meet eligibility requirements
for work in the United States at the time the appointment is scheduled
to begin and continue working legally for the term of employment.

Desiredqualifications include postdoctoral research experience.

Application:Applicants must apply for the position electronically
through: <a href="http://bit.ly/1Re5ns2.">http://bit.ly/1Re5ns2.</a> Complete
applications must include: 1) a letter of application that addresses the
candidate&rsquo;s research and teaching experiences, 2) a statement of future
research plans or interests, 3) teaching philosophy, 4) current vita,
and 5) copies of all transcripts. When prompted during the on-line
application process, please provide names, phone numbers and email
addresses of three professional references. Only complete application
materials will be considered. Active review of applications will begin
February 1, 2016, but the search will continue until the position is
filled. Questions about the position should be directed to:

Dr.Leonardo De La Fuente, Search Committee Chair
Departmentof Entomology and Plant Pathology
AuburnUniversity
Email:lzd0005@auburn.edu 

TheUniversity: Auburn University is a land-grant institution with
a student enrollment of more than 25,000. The University provides
instruction in approximately 70 academic departments and maintains
graduate research and public service programs of a broad scope. The
University is located in the city of Auburn in east-central Alabama near
the metropolitan areas of Birmingham, Alabama, and Atlanta, Georgia.

TheDepartment: There are 18 faculty and 12 support positions
in the department. Additional information can be found at
<a href="http://bit.ly/1O2u3j8">http://bit.ly/1O2u3j8</a>




KennethM. Halanych
SchnellerChair
AlumniProfessor
BiologicalSciences Department
LifeSciences Bld. 101
AuburnUniversity
Auburn,AL 36849

<a href="http://bit.ly/1Re5ns4">http://bit.ly/1Re5ns4</a>

Phone:(334)-844-3222
Ken'sFax:   (334)-844-2333
BiologyFax (334)-844-1645
e-mail:ken@auburn.edu



ken@auburn.edu


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:35</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136669321336">Conference: DurhamNC.EvolutionaryMedicineSociety.Jun24-27.Deadline</a></div>
  <div class="description">Jan 30 is the closing date for abstract submission and early registration
for the 2016 meeting of the International Society for Evolution, Medicine
and Public Health. <a href="http://evmed.org/">http://evmed.org</a>


All interested scientists, scholars, students and clinicians are welcome.



The 2016 annual meeting of the International Society for Evolution,
Medicine and Public Health (ISEMPH) will take place from June 22-25, 2016
in Durham, North Carolina, thanks to host Charles Nunn and the Triangle
Center for Evolutionary Medicine.



The Program Committee welcomes your abstracts for oral or poster
presentations on all topics in the field of evolution, medicine and public
health.



Plenary speakers include:

Andrea Graham (Princeton University),

Carl Zimmer (New York Times),

Helen Ball (Durham University, UK),

Joshua Schiffman (University of Utah),

Marion Koopmans (Erasmus University) and

Martin Blaser (New York University).



Full information about registration and abstract submission is at

http://:evolutionarymedicine.org/isemph2016.



We look forward to seeing you for a memorable and exciting meeting!



Randolph Nesse, President, ISEMPH  (nesse@asu.edu)

Sir Peter Gluckman, Vice-President, ISEMPH

Cynthia Beall, Treasurer, ISEMPH, Chair, Program Committee

Charles Nunn, Chair, Meeting Planning Committee



PROGRAM COMMITTEE


Chair: Cynthia Beall, Case Western Reserve University

Joe Alcock, University of New Mexico

Gillian Bentley, Durham University, UK

Michelle Blyth, Louisiana State University

Jon Laman, Rijksuniversiteit Groningen, Netherlands
Randolph Nesse, Arizona State University
Charles Nunn, Duke University
Frank Rhli, University of Zurich

Joshua Schiffman, University of Utah

Randolph Nesse 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:20</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136668891686">Other: Online educational module on human population genomics</a></div>
  <div class="description"><p>We have written an online educational module that examines human,
Neanderthal and chimpanzee genome sequences for chromosome 6p
with an eye to discovering:

(a) what the overall variability of the genome says about human
prehistory,

(b) what anomalous regions of the genome say about specific forces
(population admixture, natural selection) acting on genes in those
regions.

The module is intended for college undergraduates in biology and 
related fields (a reviewer thought it was also relevant to anthropology
students) and is free to use.  We&rsquo;d appreciate feedback from anyone
who uses it in a class.

It can be found at:

<a href="http://bit.ly/1INW2nT">http://bit.ly/1INW2nT</a>

Best wishes,
Mary Kuhner mkkuhner@uw.edu


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:35</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136667521096">Postdoc: UCapeTown.AvianCoevolution</a></div>
  <div class="description">Post-doctoral research opportunity at the Percy FitzPatrick Institute
of African Ornithology, Department of Biological Sciences, University
of Cape Town

Project: Sociable Weaver nests as a resource to animal communities and
the life history of the obligate associate African Pygmy Falcon

We invite applications for the above full-time research opportunity at the
FitzPatrick Institute, a world-renowned, national Centre of Excellence
(CoE) in ornithological research with a strong emphasis on postgraduate
studies. The tenure of the Fellowship is for 2 years, with possible
extension to a 3rd year considering satisfactory academic progress and
availability of funds.

The successful applicant will focus on the interactions in animal
communities associated with Sociable Weaver (Philetairus socius) nests,
with particular focus on the role of the closely associated African Pygmy
Falcon (Polihierax semitorquatus). The candidate will conduct research
with Dr Robert Thomson. Applicants must have strong analytical skills,
including a demonstrated expertise in programming with R. Supervision
experience of post-graduates and a good publication record for the career
stage are also desirable.

This project examines the importance of Sociable Weaver nests to
a desert animal community and the role of African Pygmy Falcons in
these communities. The exact research questions will be tailored to,
and developed with, the successful candidate. The research could focus
on the diet, behaviour, demography and physiology of the African Pygmy
Falcon, investigating how the evolution of the species life history
strategy is linked to its obligate association with Sociable Weaver nests
(weaver nests provide the only nesting substrate used by the falcons in
southern Africa). The research could also use the falcon-weaver system
as a co-evolutionary model to understand the interplay between climate
and biotic interactions in shaping the South African distribution of
these two species. Areas of mismatch (where weavers are present but not
falcons) could be identified and additional fieldwork would enable to
collect environmental data aimed to explain the absence of Pygmy falcons
in these areas. Ultimately, by integrating multi-species interactions
into an ecological niche modelling framework, this project will offer a
unique opportunity to predict the future range of this Kalahari falcon
under different scenarios of climate change.

The researcher could also help supervise a recently initiated and
closely linked PhD project investigating the diversity of species
utilizing weaver nests and their interactions in an aridity gradient
across the weaver range. Conditions of the award of a Postdoctoral
Fellowship at UCT are: 
(i) postdoctoral fellowships are only available to individuals who
    have achieved a doctoral degree in a relevant subject within the
    past 5 years.
(ii) applicants may not previously have held full-time professional or
     academic employment.
(iii) the successful incumbent may, as part of their professional
      development, be required to participate in departmental
      activities including limited student supervision and duties
      incidental thereto.
(iv) the successful incumbent will be required to comply with
     UCTs approved policies, procedures and practises for the
     postdoctoral sector.

It is expected that the candidate will submit papers to high quality
peer-reviewed journals within the fellowship term.

The value of the Fellowship is R180 000 per annum. The Fellowship
is compliant with the SARS guidelines and rules and is exempt from
taxation. Adequate project running costs are available.

The successful incumbent should be available to commence with the
Fellowship in the first half of 2016, although flexibility might be
required if a non-South African resident is successful.

To apply, please send a CV (including your academic record,
publication list & the names and contact details of two academic
referees) plus a short motivation letter to Hilary Buchanan at
hilary.buchanan@uct.ac.za. Informal enquires can be directed to Robert
Thomson: robert.thomson@uct.ac.za.

For more information on the FitzPatrick Institute visit
<a href="http://bit.ly/1MQmV5z">http://bit.ly/1MQmV5z</a>

Closing date: 15 January 2016 (possible interviews to be held in January)
UCT is committed to the pursuit of excellence, diversity and
redress. UCT reserves the right to disqualify ineligible, incomplete
and/or inappropriate applications, and reserves the right to change the
conditions of award or to make no awards at all.

UNIVERSITY OF CAPE TOWN


Robert Thomson 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:20</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136667006441">Postdoc: IowaStateU.EvolBioinformaticsDosageCompensation</a></div>
  <div class="description">Bioinformatics PostDoctoral Position in Turtle Dosage Compensation

A postdoctoral position is available to work in the laboratory of
Dr. Nicole Valenzuela at Iowa State University on an NSF-funded project
to study the evolution of dosage compensation using turtles with
independently evolved XX/XY and ZZ/ZW sex chromosome systems.

The project combines bioinformatics, transcriptomics, molecular
cytogenetics, and phylogenetic analyses.

*Project description:
Species with sex chromosomes face the challenge posed by differential
gene dosage between XX and XY individuals (or ZZ and ZW) that may lead
to disease, suboptimal phenotypes or death. Dosage compensation is a
mechanism to equalize the activity of X- or Z-linked genes between sex
chromosomes and autosomes, and between males and females. As part of this
NSF-funded project we are leveraging the multiple independently-evolved
sex chromosome systems found in turtles to study the evolution of dosage
compensation.

The successful candidate will help uncover the full diversity of dosage
compensation mechanisms and their molecular underpinnings by contrasting
turtles with alternative sex-determining mechanisms, leveraging and
augmenting turtle genomic resources.

*Position Requirements:
The ideal candidate will have a PhD degree and strong background in
bioinformatics, including genome assembly and transcriptome analyses.

Other qualifications such as evolutionary biology background or experience
with molecular biology and molecular cytogenetics are a plus but not
required.

*Position details:
Funding is available for 2-3 years with annual renewal contingent upon
performance. The position is available at any time starting February 2016.

*Application deadline:
1 February 2016, or until position is filled.

*How to apply:
For inquiries or to apply please email the following to Dr. Nicole
Valenzuela at nvalenzu@iastate.edu:
1.           Cover letter describing research interests and experience
2.           Current CV
3.           Copies of up to two relevant publications if available
4.           Applicants should arrange to have three letters of
             recommendation be sent directly to nvalenzu@iastate.edu.

Iowa State University does not discriminate on the basis of race, color,
age, religion, national origin, sexual orientation, gender identity,
genetic information, sex, marital status, disability, or status as a
U.S. veteran.

Dr. Nicole Valenzuela
Associate Professor
nvalenzu@iastate.edu
Dept. of Ecology, Evolution, and Organismal Biology
Iowa State University
Office: 239 Bessey Hall
251 Bessey Hall, Ames IA 50011
URL: <a href="http://bit.ly/17Wyl4I">http://bit.ly/17Wyl4I</a>

&ldquo;Valenzuela, M. N [EEOBS]&rdquo; 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:05</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/136666489571">Conference: Roscoff_France.50yrsMolecularHistory.May9-13.Reminder</a></div>
  <div class="description">There is only one month left to apply to the Jacques Monod conference on
molecular evolution in May 2016. We particularly encourage the application
of PhD students. Note that the inscription fees listed below include board
and lodging.  Selected  applicants will be expected to present a poster.


  Molecules as documents of evolutionary history: 50 years after - Roscoff,
  Brittany, France.

  May 9-13 2016



It is with the popularization of molecular tools and the disclosure
of genome sequences that evolutionary models have become decisive in
biology, in part because this information is simply so abundant and
so complex that only a comparative method could reveal its meaning.
In 2016, it will be 50 years that Emile Zuckerkandl and Linus Pauling
have laid the foundation of molecular phylogeny, and molecular evolution
in general, in their seminal article entitled &ldquo;Molecules as documents
of evolutionary history&rdquo;. The fundamental realization of this article
was that DNA does not only encode the elementary functional elements of
an organisms, it also harbours abundant information on the evolutionary
history and life history traits of its ancestors.

The modelling of the evolutionary processes that generated extent
genomes should reveal together the patterns of the diversification
of life, the processes by which evolution proceeds at the molecular
level, and how both are affected by external conditions. Our challenge
today, as evolutionary biologists, is to bridge the gap between species
ecology, life history traits, population dynamics, species phylogeny,
gene phylogeny, genome structure, gene network and molecular mechanisms.

The recent years have seen tremendous developments in this direction,
which we will attempt to review in this conference.


Invited speakers 
(provisional titles)

 

ABBY Sophie (Paris, France)
Evolution playing a molecular building set: origins and distribution of
bacterial secretion systems

ACHTMAN Mark (Warwick, United Kingdom)
Ages of bacterial pathogenes

ALM Eric (Cambridge, USA)
Driving forces of microbiome evolution

DURET Laurent (Lyon, France)
Biased gene conversion as a 4rth evolutionary force

FARIA Nuno (Oxford, United Kingdom)
Viral evolution

GALTIER Nicolas (Montpellier, France)
Population genomics of non model animals: genetic diversity, adaptive
rate and effective population size

GILBERT Thomas (Copenhaguen, Denmark)
Ancient DNA, ancient genomes, and the hype of de-extinction

GOGARTEN Peter (Storrs, USA)
Horizontal gene transfer: the pan-genome as shared genetic resource of
a lineage

GRAUR Dan (Houston, USA)
The vocabulary of molecular evolution and the transgressions of functional
genomics: a rose by any other name would be misleading

GRIBALDO Simonetta (Paris, France)
Phylogenomics and the tree of life

HEATH Tracy (Ames, USA)
Molecular, morphological, and macroevolutionary models for dating species
divergence times

KATZ Laura (Northampton, USA)
The eukaryotic phylogeny

KOSIOL Carolin (Vienna, Austria)
Polymorphism-aware phylogenetic models for species trees

LAGERGREN Jens (Stockholm, Sweden)
Generative models of gene family evolution

LỐPEZ-GARCIA Purificación (Orsay, France)
Diversity of microbial life

McLYSAGHT  Aoife (Dublin, Ireland)
Whole genome duplication

MORAN Nancy (Austin, USA)
Symbiosis in evolution

PAGEL Mark (Reading, United Kingdom)
Beyond molecular data: information and evolution in the cultural realm

PHILIPPE Hervé (Moulis, France)
Models of sequence evolution

SANMARTIN Isabel (Madrid, Spain)
Spatiotemporal evolution of lineages and biotas using Bayesian approaches

SEMON Marie (Lyon, France)
Insights into making different types of the same organ using developmental
transcriptomes as molecular microscopes

STADLER Tanja (Zürich, Switzerland)
Modelling and inference of species diversification

SZÖLLÖSI Gergely (Budapest, Hungary)
Genome scale reconstruction of phylogenies

TELFORD Maximilian J. (London, United Kingdom)
The new animal phylogeny

WARNOW Tandy (Urbana, USA)
New coalescent-based species tree estimation methods

 
Deadline for application: February 1rst, 2016

 
Registration fee (including board and lodging)

430 € for PhD students 
625 € for other participants

 
Application for registration 
The total number of participants is limited to 115 and all participants
are expected to attend for the whole duration of the conference. Selection
is made on the basis of the affinity of potential participants with the
topics of the conference. Scientists and PhD Students interested in the
meeting should send:

- their curriculum vitae 
- the list of their main publications for the 3 last years 
- the abstract of their presentation

to the Chairperson of the conference (vincent.daubin@univ-lyon1.fr
) before the deadline. After it,
the organizers will select the participants. Except in some particular
cases approved by the Chairperson, it is recommended that all selected
participants present their work during the conference, either in
poster form or by a brief in- session talk. The organizers choose the
form in which the presentations are made. No payment will be sent with
application. Information on how and when to pay will be mailed in due
time to those selected.

 

  -
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="index8c8d.html?q=aggregator&amp;from=300">previous page</a></div><div>... <a href="indexd118.html?q=aggregator&amp;from=240">13</a> <a href="index3f45.html?q=aggregator&amp;from=260">14</a> <a href="index4603.html?q=aggregator&amp;from=280">15</a> <a href="index8c8d.html?q=aggregator&amp;from=300">16</a> <strong>17</strong> <a href="indexd191.html?q=aggregator&amp;from=340">18</a> <a href="index28a4.html?q=aggregator&amp;from=360">19</a> <a href="index63c1.html?q=aggregator&amp;from=380">20</a> <a href="index940a.html?q=aggregator&amp;from=400">21</a> ...</div><div><a href="indexd191.html?q=aggregator&amp;from=340">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=320 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 22:26:36 GMT -->
</html>
