<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" lang="en" xml:lang="en">

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=20 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:20:25 GMT -->
<!-- Added by HTTrack --><meta http-equiv="content-type" content="text/html;charset=utf-8" /><!-- /Added by HTTrack -->
<head>
 <title>Society of Systematic Biologists | Home of Systematic Biology</title>
<meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
<base  />
<style type="text/css" media="all">@import "misc/drupal.css";</style> <link rel="stylesheet" type="text/css" href="themes/chameleon/common.css" />
<style type="text/css" media="all">@import "themes/chameleon/style.css";</style></head><body>
 <div id="header">  <a href="index.html" title="Home"><img src="files/SSB_logo_final.png" alt="Home" /></a>  <h1 class="site-name title"><a href="index03d2.html?q=">Society of Systematic Biologists</a></h1>  <div class="site-slogan">Home of Systematic Biology</div></div>
 <div class="navlinks"><div class="primary"><!-- <a onmouseover="this.style.color='#ff9519';" style="color:#4a8897;font-size:16px;" onmouseout="this.style.color='#000000';" href="/">Home | </a>-->
<div style="border-bottom:1px solid rgb(128,128,128);padding:8px;"><a style="font-size:16px;" href="index.html">Home</a>
 | <a style="font-size:16px;" href="node/6.html">Join</a>
 | <a style="font-size:16px;" href="node/279.html">Contact</a>
 | <a  style="font-size:16px;" href="http://sysbio.oxfordjournals.org/" target="_new"><i>Systematic Biology</i></a>
 | <a  style="font-size:16px;" href="http://mc.manuscriptcentral.com/systbiol" target="_new">Submit a manuscript</a>
 | <a style="font-size:16px;" href="node/3.html">Conference</a>
| <a style="font-size:16px;" href="http://twitter.com/systbiol"><img src="files/twitter.png" border="0" /> Follow us on Twitter</a>
| <a style="font-size:16px;" href="http://www.facebook.com/groups/162698383797033"><img src="files/facebook.png" border="0" /> Find us on Facebook</a>
</div></div> </div>
 <table id="content">
  <tr>
   <td id="sidebar-left"><div class="block block-menu" id="block-menu-38">
 <h2 class="title">Systematic Biology</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index5e5b.html?q=node/88" title="">Issues Online</a></li>
<li class="collapsed"><a href="index4d07.html?q=node/35" title="">Appendices and data</a></li>
<li class="collapsed"><a href="index924a.html?q=node/27" title="">Editorial information</a></li>
<li class="leaf"><a href="indexebef.html?q=node/424" title="SSB logo in various formats">Logo</a></li>
<li class="collapsed"><a href="indexe51d.html?q=node/80" title="">Teaching aides</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-47">
 <h2 class="title">Membership</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2636.html?q=node/67" title="">Access journal online</a></li>
<li class="leaf"><a href="indexf02c.html?q=node/11" title="">Benefits of membership</a></li>
<li class="leaf"><a href="indexfe3a.html?q=node/279" title="">Contact</a></li>
<li class="leaf"><a href="index5ffe.html?q=node/6" title="">Join</a></li>
<li class="leaf"><a href="index2dc0.html?q=node/226" title="Renew SSB Membership">Renew</a></li>
<li class="leaf"><a href="indexe1e4.html?q=node/110" title="">Membership Directory</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-303">
 <h2 class="title">Merchandise</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="indexae74.html?q=node/253" title="">CafePress</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-83">
 <h2 class="title">Society</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index517a.html?q=node/139" title="">Careers in systematics</a></li>
<li class="leaf"><a href="index0e18.html?q=node/75" title="">Council</a></li>
<li class="collapsed"><a href="index7c87.html?q=node/31" title="">Governance</a></li>
<li class="collapsed"><a href="index67da.html?q=node/34" title="">History</a></li>
<li class="leaf"><a href="indexee92.html?q=node/276" title="">Minutes of AGM</a></li>
<li class="collapsed"><a href="index543f.html?q=node/133" title="">Officers</a></li>
<li class="leaf"><a href="index763f.html?q=node/15" title="">Past-President</a></li>
<li class="leaf"><a href="index67db.html?q=node/13" title="">President</a></li>
<li class="leaf"><a href="index7c44.html?q=node/14" title="">President-Elect</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-49">
 <h2 class="title">Annual Conference</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index2a62.html?q=node/3" title="">Current</a></li>
<li class="leaf"><a href="indexc715.html?q=node/167" title="Past conferences">Past</a></li>
<li class="leaf"><a href="index822c.html?q=node/199" title="">Past Symposia</a></li>
<li class="leaf"><a href="index0f22.html?q=node/4" title="">Future</a></li>
<li class="leaf"><a href="index94ca.html?q=node/449" title="Policy on acceptable conduct at society meetings">Policy Statement</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-51">
 <h2 class="title">Awards</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index9b25.html?q=node/70" title="">Claiming your award</a></li>
<li class="leaf"><a href="indexbfe3.html?q=node/26" title="">Mini-ARTS</a></li>
<li class="leaf"><a href="index71da.html?q=node/24" title="">President's Award</a></li>
<li class="collapsed"><a href="index06d0.html?q=node/20" title="">Student awards</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-143">
 <h2 class="title">How to add content</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index3b07.html?q=node/44" title="">Adding a news item</a></li>
<li class="leaf"><a href="index3f7e.html?q=node/77" title="">Adding a page</a></li>
<li class="leaf"><a href="indexaf50.html?q=node/196" title="">Creating cover thumbnails</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-267">
 <h2 class="title">Archive</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index0c8e.html?q=node/150" title="Archive of defunct journal Phyloinformatics">Phyloinformatics</a></li>

</ul>
</div></div>
</div>
<div class="block block-menu" id="block-menu-284">
 <h2 class="title">Links</h2>
 <div class="content"><div class="menu">
<ul>
<li class="leaf"><a href="index247e.html?q=node/274" title="">Other meetings</a></li>
<li class="leaf"><a href="indexec53.html?q=node/47" title="">Other societies</a></li>
<li class="leaf"><a href="index8cb6.html?q=node/186" title="">Useful links</a></li>

</ul>
</div></div>
</div>
<div class="block block-statistics" id="block-statistics-0">
 <h2 class="title">Popular content</h2>
 <div class="content"><div class="item-list"><h3>Today's:</h3><ul><li><a href="indexfe3a.html?q=node/279">Contact</a></li><li><a href="index2875.html?q=node/72">MacClade and Adobe Illustrator</a></li><li><a href="indexf64b.html?q=node/63">LaTeX template and bibliography style file</a></li></ul></div></div>
</div>
<div class="block block-node" id="block-node-0">
 <h2 class="title">Syndicate</h2>
 <div class="content"><div class="xml-icon"><a href="indexeb15.html?q=node/feed"><img src="misc/xml.png" width="36" height="14" alt="XML feed" title="XML feed" /></a></div></div>
</div>
<div class="block block-taxonomy" id="block-taxonomy-0">
 <h2 class="title">Categories</h2>
 <div class="content"><div class="item-list"><ul><li><a href="index18c1.html?q=taxonomy/term/6">General (29)</a><br />21 weeks 5 days ago</li><li><a href="indexbe3e.html?q=taxonomy/term/7">Jobs (93)</a><br />50 weeks 1 day ago</li><li><a href="index443a.html?q=taxonomy/term/2">Meetings (64)</a><br />1 year 5 weeks ago</li><li><a href="index7823.html?q=taxonomy/term/5">Courses (32)</a><br />1 year 13 weeks ago</li><li><a href="index8c63.html?q=taxonomy/term/4">Published elsewhere (21)</a><br />1 year 22 weeks ago</li><li><a href="index4a84.html?q=taxonomy/term/3">Funding (24)</a><br />2 years 8 weeks ago</li><li><a href="index8ccf.html?q=taxonomy/term/1">Systematic Biology (22)</a><br />2 years 33 weeks ago</li></ul></div></div>
</div>
<div class="block block-user" id="block-user-0">
 <h2 class="title">User login</h2>
 <div class="content"><form action="http://systbio.org.whsites.net/?q=user/login" method="post">
<div class="user-login-block">
<input type="hidden" name="edit[destination]" value="aggregator" />
<div class="form-item">
 <label for="edit-name">Username:</label><br />
 <input type="text" maxlength="64" class="form-text" name="edit[name]" id="edit-name" size="15" value="" />
</div>
<div class="form-item">
 <label for="edit-pass">Password:</label><br />
 <input type="password" class="form-password" maxlength="64" name="edit[pass]" id="edit-pass" size="15" value="" />
</div>
<input type="submit" class="form-submit" name="op" value="Log in"  />
</div>

</form>
<div class="item-list"><ul><li><a href="indexc12c.html?q=user/password" title="Request new password via e-mail.">Request new password</a></li></ul></div></div>
</div>
<div class="block block-user" id="block-user-1">
 <h2 class="title">Navigation</h2>
 <div class="content"><div class="menu">
<ul>
<li class="expanded"><a href="index5640.html?q=aggregator" class="active">news aggregator</a>
<ul>
<li class="collapsed"><a href="indexe5f5.html?q=aggregator/sources">sources</a></li>

</ul>
</li>

</ul>
</div></div>
</div>
<div class="block block-user" id="block-user-3">
 <h2 class="title">Who's online</h2>
 <div class="content">There are currently 0 users and 2 guests online.</div>
</div>
</td>
   <td id="main">
<div class="breadcrumb"><a href="index03d2.html?q=">Home</a></div><h2>news aggregator</h2>
<!-- begin content -->
<div id="aggregator"><h3>March 3, 2016</h3>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1053?rss=1">Cross-Study Comparison Reveals Common Genomic, Network, and Functional Signatures of Desiccation Resistance in Drosophila melanogaster</a></div>
  <div class="description"><p>Repeated attempts to map the genomic basis of complex traits often yield different outcomes because of the influence of genetic background, gene-by-environment interactions, and/or statistical limitations. However, where repeatability is low at the level of individual genes, overlap often occurs in gene ontology categories, genetic pathways, and interaction networks. Here we report on the genomic overlap for natural desiccation resistance from a Pool-genome-wide association study experiment and a selection experiment in flies collected from the same region in southeastern Australia in different years. We identified over 600 single nucleotide polymorphisms associated with desiccation resistance in flies derived from almost 1,000 wild-caught genotypes, a similar number of loci to that observed in our previous genomic study of selected lines, demonstrating the genetic complexity of this ecologically important trait. By harnessing the power of cross-study comparison, we narrowed the candidates from almost 400 genes in each study to a core set of 45 genes, enriched for stimulus, stress, and defense responses. In addition to gene-level overlap, there was higher order congruence at the network and functional levels, suggesting genetic redundancy in key stress sensing, stress response, immunity, signaling, and gene expression pathways. We also identified variants linked to different molecular aspects of desiccation physiology previously verified from functional experiments. Our approach provides insight into the genomic basis of a complex and ecologically important trait and predicts candidate genetic pathways to explore in multiple genetic backgrounds and related species within a functional framework.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1068?rss=1">Convergent Substitutions in a Sodium Channel Suggest Multiple Origins of Toxin Resistance in Poison Frogs</a></div>
  <div class="description"><p>Complex phenotypes typically have a correspondingly multifaceted genetic component. However, the genotype&ndash;phenotype association between chemical defense and resistance is often simple: genetic changes in the binding site of a toxin alter how it affects its target. Some toxic organisms, such as poison frogs (Anura: Dendrobatidae), have defensive alkaloids that disrupt the function of ion channels, proteins that are crucial for nerve and muscle activity. Using protein-docking models, we predict that three major classes of poison frog alkaloids (histrionicotoxins, pumiliotoxins, and batrachotoxins) bind to similar sites in the highly conserved inner pore of the muscle voltage-gated sodium channel, Nav1.4. We predict that poison frogs are somewhat resistant to these compounds because they have six types of amino acid replacements in the Nav1.4 inner pore that are absent in all other frogs except for a distantly related alkaloid-defended frog from Madagascar, <I>Mantella aurantiaca</I>. Protein-docking models and comparative phylogenetics support the role of these replacements in alkaloid resistance. Taking into account the four independent origins of chemical defense in Dendrobatidae, phylogenetic patterns of the amino acid replacements suggest that 1) alkaloid resistance in Nav1.4 evolved independently at least seven times in these frogs, 2) variation in resistance-conferring replacements is likely a result of differences in alkaloid exposure across species, and 3) functional constraint shapes the evolution of the Nav1.4 inner pore. Our study is the first to demonstrate the genetic basis of autoresistance in frogs with alkaloid defenses.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1082?rss=1">Detecting Genomic Signatures of Natural Selection with Principal Component Analysis: Application to the 1000 Genomes Data</a></div>
  <div class="description"><p>To characterize natural selection, various analytical methods for detecting candidate genomic regions have been developed. We propose to perform genome-wide scans of natural selection using principal component analysis (PCA). We show that the common <I>F</I>ST index of genetic differentiation between populations can be viewed as the proportion of variance explained by the principal components. Considering the correlations between genetic variants and each principal component provides a conceptual framework to detect genetic variants involved in local adaptation without any prior definition of populations. To validate the PCA-based approach, we consider the 1000 Genomes data (phase 1) considering 850 individuals coming from Africa, Asia, and Europe. The number of genetic variants is of the order of 36 millions obtained with a low-coverage sequencing depth (3x). The correlations between genetic variation and each principal component provide well-known targets for positive selection (EDAR, SLC24A5, SLC45A2, DARC), and also new candidate genes (APPBPP2, TP1A1, RTTN, KCNMA, MYO5C) and noncoding RNAs. In addition to identifying genes involved in biological adaptation, we identify two biological pathways involved in polygenic adaptation that are related to the innate immune system (beta defensins) and to lipid metabolism (fatty acid omega oxidation). An additional analysis of European data shows that a genome scan based on PCA retrieves classical examples of local adaptation even when there are no well-defined populations. PCA-based statistics, implemented in the <I>PCAdapt</I> R package and the <I>PCAdapt fast</I> open-source software, retrieve well-known signals of human adaptation, which is encouraging for future whole-genome sequencing project, especially when defining populations is difficult.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1094?rss=1">VHICA, a New Method to Discriminate between Vertical and Horizontal Transposon Transfer: Application to the Mariner Family within Drosophila</a></div>
  <div class="description"><p>Transposable elements (TEs) are genomic repeated sequences that display complex evolutionary patterns. They are usually inherited vertically, but can occasionally be transmitted between sexually independent species, through so-called horizontal transposon transfers (HTTs). Recurrent HTTs are supposed to be essential in life cycle of TEs, which are otherwise destined for eventual decay. HTTs also impact the host genome evolution. However, the extent of HTTs in eukaryotes is largely unknown, due to the lack of efficient, statistically supported methods that can be applied to multiple species sequence data sets. Here, we developed a new automated method available as a R package "vhica" that discriminates whether a given TE family was vertically or horizontally transferred, and potentially infers donor and receptor species. The method is well suited for TE sequences extracted from complete genomes, and applicable to multiple TEs and species at the same time. We first validated our method using <I>Drosophila</I> TE families with well-known evolutionary histories, displaying both HTTs and vertical transmission. We then tested 26 different lineages of <I>mariner</I> elements recently characterized in 20 <I>Drosophila</I> genomes, and found HTTs in 24 of them. Furthermore, several independent HTT events could often be detected within the same <I>mariner</I> lineage. The VHICA (Vertical and Horizontal Inheritance Consistence Analysis) method thus appears as a valuable tool to analyze the evolutionary history of TEs across a large range of species.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1110?rss=1">Avoiding Missing Data Biases in Phylogenomic Inference: An Empirical Study in the Landfowl (Aves: Galliformes)</a></div>
  <div class="description"><p>Production of massive DNA sequence data sets is transforming phylogenetic inference, but best practices for analyzing such data sets are not well established. One uncertainty is robustness to missing data, particularly in coalescent frameworks. To understand the effects of increasing matrix size and loci at the cost of increasing missing data, we produced a 90 taxon, 2.2 megabase, 4,800 locus sequence matrix of landfowl using target capture of ultraconserved elements. We then compared phylogenies estimated with concatenated maximum likelihood, quartet-based methods executed on concatenated matrices and gene tree reconciliation methods, across five thresholds of missing data. Results of maximum likelihood and quartet analyses were similar, well resolved, and demonstrated increasing support with increasing matrix size and sparseness. Conversely, gene tree reconciliation produced unexpected relationships when we included all informative loci, with certain taxa placed toward the root compared with other approaches. Inspection of these taxa identified a prevalence of short average contigs, which potentially biased gene tree inference and caused erroneous results in gene tree reconciliation. This suggests that the more problematic missing data in gene tree&ndash;based analyses are partial sequences rather than entire missing sequences from locus alignments. Limiting gene tree reconciliation to the most informative loci solved this problem, producing well-supported topologies congruent with concatenation and quartet methods. Collectively, our analyses provide a well-resolved phylogeny of landfowl, including strong support for previously problematic relationships such as those among junglefowl (<I>Gallus</I>), and clarify the position of two enigmatic galliform genera (<I>Lerwa</I>, <I>Melanoperdix</I>) not sampled in previous molecular phylogenetic studies.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/1126?rss=1">Wasabi: An Integrated Platform for Evolutionary Sequence Analysis and Data Visualization</a></div>
  <div class="description"><p>Wasabi is an open source, web-based environment for evolutionary sequence analysis. Wasabi visualizes sequence data together with a phylogenetic tree within a modern, user-friendly interface: The interface hides extraneous options, supports context sensitive menus, drag-and-drop editing, and displays additional information, such as ancestral sequences, associated with specific tree nodes. The Wasabi environment supports reproducibility by automatically storing intermediate analysis steps and includes built-in functions to share data between users and publish analysis results. For computational analysis, Wasabi supports PRANK and PAGAN for phylogeny-aware alignment and alignment extension, and it can be easily extended with other tools. Along with drag-and-drop import of local files, Wasabi can access remote data through URL and import sequence data, GeneTrees and EPO alignments directly from Ensembl. To demonstrate a typical workflow using Wasabi, we reproduce key findings from recent comparative genomics studies, including a reanalysis of the <I>EGLN1</I> gene from the tiger genome study: These case studies can be browsed within Wasabi at <A HREF="http://wasabiapp.org:8000/?id=usecases">http://wasabiapp.org:8000?id=usecases. Wasabi runs inside a web browser and does not require any installation. One can start using it at http://wasabiapp.org</A>. All source code is licensed under the AGPLv3.</p></div>
  <div class="source">Source: <a href="index9bc3.html?q=aggregator/sources/4">Molecular Biology and Evolution</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">06:26</div>
 <div class="body">
  <div class="title"><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a></div>
  <div class="description"><p>Last week (25-26 February) I was in London for <a href="http://www.digitalcatapultcentre.org.uk/open-calls/cisco-pit-stop/">CISCO Pit Stop</a> event. Thursday evening was at the Natural History Museum where I gave a talk extolling the virtues of linking stuff together:</p> <p lang="en" dir="ltr">from <a href="https://twitter.com/rdmpage">@rdmpage</a> digitization is the body of knowledge: literature+specimens+dna barcodes <a href="https://twitter.com/hashtag/CISCOpitstop?src=hash">#CISCOpitstop</a> <a href="https://twitter.com/NHM_Science">@NHM_Science</a> <a href="https://t.co/HnbLCT9mHO">pic.twitter.com/HnbLCT9mHO</a></p>&mdash; Deborah Paul (@idbdeb) <a href="https://twitter.com/idbdeb/status/702950123119185920">February 25, 2016</a>  <p>My slides are <a href="http://www.slideshare.net/rdmpage/cisco-digital-catapult">here</a>:</p>    <strong> <a href="http://www.slideshare.net/rdmpage/cisco-digital-catapult" title="Cisco Digital Catapult" target="_blank">Cisco Digital Catapult</a> </strong> from <strong><a target="_blank" href="http://www.slideshare.net/rdmpage">Roderic Page</a></strong>  <p>Friday we assembled at the Digital Catapult Centre, which as Sandy Knapp notes, has some amazing views from it's 9th floor.</p> <p lang="en" dir="ltr">Surreal!! Amazing view from <a href="https://twitter.com/hashtag/ciscopitstop?src=hash">#ciscopitstop</a> <a href="https://t.co/CB8Z2R1nGt">pic.twitter.com/CB8Z2R1nGt</a></p>&mdash; Sandra Knapp (@SandyKnapp) <a href="https://twitter.com/SandyKnapp/status/703236567951876096">February 26, 2016</a>  <p>A group of experts (loosely defined, at least, if they include me in that category) and small businesses (with backgrounds in digitisation, text-mining, publishing, etc.) got  together to try and come up with workable ideas where we could marry issues in digitisation and subsequent use of that data with tools and markets. A fascinating experience, although I'm not yet sure what the outcome will be. But it's always useful talking (and listening) to people with very different backgrounds and notions of what matters (and what is possible).</p></div>
  <div class="source">Source: <a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">05:54</div>
 <div class="body">
  <div class="title"><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a></div>
  <div class="description"><p>I'm continuing to play with the <a href="http://iphylo.blogspot.com/2016/01/ispecies-is-back-mashing-up-species-data.html">new version of iSpecies</a>, seeing just how far one can get by simply grabbing JSON from various sources and mashing them up. Since the <a href="https://tree.opentreeoflife.org/opentree/argus/opentree4.0@1">Open Tree of Life</a> is pretty unresolved ("OMG it's full of stars") I've started to grab trees from <a href=%3d_http_/www.treebase.html>TreeBASE</a> and add those. Sadly TreeBASE is showing it's age and doesn't have a JSON API, so I had to break my rule of only using HTML and Javascript in iSpecies and I had to write some PHP wrappers to talk to TreeBASE. Now, when you search for a genus or species you may see a list of studies from TreeBASE, and a popup menu where you can select a tree to view.</p> <p>Below is a example (searching for the plant genus <a href="http://ispecies.org/?q=Fitzalania"><i>Fitzalania</i></a>). </p> <p>This example shows one reason phylogenies are useful. Although GBIF (which supplies the data for the map) recognises <i>Fitzalania</i>, a recent <a href="https://treebase.org/treebase-web/search/study/summary.html?id=14685">study in TreeBASE</a> shows that this renders <i>Meiogyne</i> paraphyletic, and so moves the <i>Fitzalania</i> to <i>Meiogyne</i>. Hence GBIF's taxonomy is somewhat behind the current state of knowledge about these plants.</p> <p>The paper merging these two genra (<a href="http://dx.doi.org/10.1600/036364414x680825">doi:10.1600/036364414x680825</a>) also shows up in the CrossRef results. Unfortunately TreeBASE doesn't have the DOI for the paper, so linking these two results (the TreeBASE study and the corresponding paper) will require some work. This is another reason why I'm playing with iSpecies: I want to see how many identifiers we can uncover to connect results from different sources, and how many cross links we need to add before it all comes together in a nice linked graph of data.</p></div>
  <div class="source">Source: <a href="index3e02.html?q=aggregator/sources/5">iPhylo</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">02:43</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a></div>
  <div class="description">Available:

PhD position, 3-4 years

Department of Ecology and Evolution, University of Lausanne, Switzerland

Start: September 2016 or earlier

Our group studies the selective forces that act on freshwater
fish, i.e. the effects of human activities in interaction with
natural and sexual selection. Our current research focuses
on salmonid fish (brown trout, grayling, whitefish) and
combines different types fieldwork, labwork, and modeling. See
<a href="http://bit.ly/1G0GWYe">http://bit.ly/1G0GWYe</a> We
are seeking candidates with an interest in population genetics, life
history, evolutionary ecology, and population management. The candidate
would start her/his thesis with experimental field and lab work on a
whitefish population in a Swiss lake.

Send your application by email to claus.wedekind@unil.ch. Please include
your Curriculum vitae, a short description of your research interests and
research experience, and contact details of one or two references. Review
of applications will begin end of March 2016, but applications will be
considered until the position is filled.


Claus Wedekind
Department of Ecology and Evolution, Biophore,
University of Lausanne, 1015 Lausanne, Switzerland.
Tel. +41 21 692 42 50, Fax +41 21 692 42 65
<a href="http://bit.ly/1OSKKuD">http://bit.ly/1OSKKuD</a>




Claus Wedekind 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>March 2, 2016</h3>
<div class="news-item">
 <div class="date">23:27</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a></div>
  <div class="description"><p>The Society of Systematic Biologists is pleased to offer Mini-ARTS
awards to enhance taxonomic knowledge
(<a href="http://bit.ly/1TU09kN">http://bit.ly/1TU09kN</a>). These awards are for
revisionary taxonomy and systematics, modeled after the NSF Dear
Colleague Letter: Advancing Revisionary Taxonomy and Systematics
(ARTS) recently developed within the Systematics and Biodiversity
Science Cluster (<a href="http://1.usa.gov/1dr0gQG">http://1.usa.gov/1dr0gQG</a>).
We are calling these &lsquo;Mini-ARTS&rsquo; grants. These awards are designed to
allow SSB members (students, post-docs, and faculty) to spend a summer
or semester apprenticed to an expert in a particular taxonomic group
or to enhance revisionary taxonomic and systematics research in novel
ways. Goals of this award program are to address constraints on our
knowledge of undescribed biodiversity, assist in passing on taxonomic
expertise before it is lost, increase the number of students with
broad training in organismal biology and systematics, and support
projects in biodiversity and taxonomy informatics as well as
monographic and revisionary taxonomy. Activities can include a trip to
the taxonomist&rsquo;s laboratory, pay for the taxonomist to visit the
applicant&rsquo;s laboratory for a period of time, or pay for costs of
computer time or development of interactive keys for electronic
dissemination of systematics results. Requests for support may be in
any amount up to $3,000. We expect to fund between 1-3 of these awards
this year.

=How to apply=Applicants may be from any country, but must be members
of SSB, and are advised to join the Society as soon as possible to
facilitate their applications. A complete application includes:

+ a brief description of the project, including a separate section
justifying the importance of the taxon and the revisionary work
+ an itemized budget (narrative plus budget should not exceed 2 pages)
+ the applicant&rsquo;s CV
+ a letter of support from the taxonomic expert or collaborator
+ If the applicant is a student or post-doc, please also include a
reference letter from the advisor

Grant applications should be sent to the chair of the SSB Awards
Committee (Dr. Tracy Heath: ssb.awards.director@gmail.com). E-mail
submissions are required, and applicants should use pdf format for all
documents. In the subject line of the email, please indicate the SSB
award category as 'Mini-ARTS&rsquo;.

The next application deadline for all materials, including letters of
support and letters of reference, is March 31, 2016.

For information about joining SSB, please see:
<a href="http://bit.ly/1TU0cgy.">http://bit.ly/1TU0cgy.</a>

Please contact Tracy Heath (ssb.awards.director@gmail.com) with any questions.


ssb.awards.director@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">10:34</div>
 <div class="body">
  <div class="title"><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a></div>
  <div class="description"><p><a href="http://www.phylobabble.org/users/holzdan">@holzdan</a> wrote:</p>
            
              <p>Hey peeps !</p>

<p>im very new to phylogenetics and currently trying to read myself a bit into the thematics of tree construction and getting started with all the different tools/software.</p>

<p>i would like to reconstruct a tree of 138 cyanobacterial strains defined by 16S , 16S-ITS , rpoc1 , RnaseP , rbclx , PC-IGS , PSA-IGS. The authors originally constructed the tree using maximum likelihood and the F84 model</p>

<p>I would like to implement another 4 strains where i got all loci sequences except the 16S-ITS.
Now im not sure how to deal with missing data  , should i construct a tree with all loci where i got missing sequences for only 4 strains , or should i use just the loci where i got total coverage ? is there a way to find the "best" method and will the choice of algorithm or substition model be influenced by this ?</p>

<p>cheers</p>
            
            <p>Posts: 1</p>
            <p>Participants: 1</p>
            <p><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Read full topic</a></p></div>
  <div class="source">Source: <a href="index2d4e.html?q=aggregator/sources/12">phylobabble.org</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:01</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a></div>
  <div class="description">The Geber lab at Cornell University seeks a highly motivated,
detail-oriented assistant for summer 2015 field research in the Southern
Sierra Nevada (May-July) and lab work in Ithaca, NY (July-August).  We are
studying four species of annual plants in the genus Clarkia and their
specialist bee pollinators. Our research focuses on how pollinators may
impact plant species evolution and coexistence.

Our research sites are located in the Kern River Canyon in Kern County,
California. The lab rents a house on a property where other labs working in
this system are also based in the summer. This field assistant would have a
private bedroom, access to a shared bathroom and a communal kitchen, and
wireless Internet access. Frequent trips are made into town (Lake Isabella
and Kernville) for groceries and other supplies.

Successful applicants are expected to assist in data collection, data
entry, and flower, seed, and bee collection in the field; work in Ithaca
will consist of sample processing and data analysis. The assistant will
work a 39-hour workweek. Weather conditions in the Sierras can be very hot
and dry (100+ degrees F) in the summer, so assistants should be prepared to
work in hot conditions. Applicants must be able to hike up and steep hills
for up to 1 mile at a time, stoop or kneel for extended periods of time,
and work in proximity to bees and snakes. The work can be very physically
demanding and tiring, and the research team typically collects data 7 days
a week. A good sense of humor and a positive attitude are essential.
Applicants must be available from the first week of May through late
August, although exact start/end dates are flexible.

Qualifications:
1) Education: some undergraduate education in biology, ecology, or related
field, or equivalent experience
2) Experience: experience with and enthusiasm about working outdoors,
and/or previous field research experience. Previous experience working with
plants, bees, or insects is preferable but not required.
3) Tasks: perform repetitive tasks with attention to detail, daily hiking
4) Personality: interpersonal skills and willingness to live and work in
close quarters with other researchers in a house.

Compensation: transportation to and from California, a private room with
Internet access in the field house, and pay ($13-$15 an hour depending on
experience) will be provided. Applicants will be responsible for securing
their own housing in Ithaca for July and August.

The deadline for applications is March 20, 2016. Applicants should submit:
1) a short (1 page maximum) cover letter describing their qualifications as
well as future academic and professional goals; 2) a resume or CV outlining
previous work experience and relevant courses (completed or in progress),
extracurricular activities; and 3) contact information for two references.

Cornell University is an affirmative action/equal opportunity
employer. Cornell University and we in the Geber lab have a commitment to
support equality of education and employment opportunity by affirming the
value of diversity. We welcome and encourage scientists of all backgrounds
to apply to work with us! <a href="http://bit.ly/21BV9HG">http://bit.ly/21BV9HG</a>

Email to: Monica Geber, Professor of Ecology and Evolutionary Biology
mag9@cornell.edu

Katherine Eisen 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:16</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140320410121">Postdoc: AarhusU.ProteinEvolution</a></div>
  <div class="description">Applications are invited for a 1-year postdoctoral position in the
Department of Bioscience, Aarhus University, Denmark, to take part to a
research project on the functional (i.e. enzymatic and oxygen-binding)
properties and physiological roles of globin proteins in animals. The
NIH-funded project involves a collaboration between Dr. Angela Fago
(Aarhus University) and Dr. Jay F. Storz (University of Nebraska,
Lincoln, USA). A major focus of the project is the study of biochemical
properties of globin proteins (native and recombinant) in species
adapted to hypoxia.

Applicants must have a PhD in Biology, Molecular Biology, Biochemistry
or a related field. The ideal candidate will have experience in protein
biochemistry techniques (including purification, gel electrophoresis,
spectroscopy) and a solid publication record. The project requires a
good understanding of the interactions between protein structure,
function and physiology. Knowledge of computational data analysis for
the determination of thermodynamic and kinetic constants will be an
advantage. For further information please contact Angela Fago
(angela.fago@bios.au.dk) or Jay Storz (jstorz2@unl.edu).

The preferred start date is August 1st 2016.

The application must be in English and include a curriculum vitae,
degree certificate, a complete list of publications, a statement of
future research plans and information about research activities, two
letters of reference, teaching portfolio and verified information on
previous teaching experience (if any). Guidelines for applicants can be
found here:

<a href="http://bit.ly/1Qqqjaf">http://bit.ly/1Qqqjaf</a>
pplication-form-guide/

Salary depends on seniority as agreed between the Danish Ministry of
Finance and the Confederation of Professional Associations.

All applications must be made online and received by 1 May 2016.

Online application:
<a href="http://bit.ly/1TQP5Fd">http://bit.ly/1TQP5Fd</a>
nForm.cfm?VacatureID6147&PortalIDR83

Jay F. Storz

School of Biological Sciences
University of Nebraska
Lincoln, NE 68588
Phone: 402/472-1114
E-mail: jstorz2@unl.edu
<a href="http://bit.ly/1aSXB05">http://bit.ly/1aSXB05</a>

Jay Storz 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>March 1, 2016</h3>
<div class="news-item">
 <div class="date">21:51</div>
 <div class="body">
  <div class="title"><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a></div>
  <div class="description">The Mesembrinellidae (Diptera: Oestroidea) comprise a small group of strictly Neotropical calyptrate flies, with 36 described species. The group has often been treated as a subfamily of Calliphoridae, but there is growing evidence that it corresponds to a distinct Oestroidea lineage. Internal relationships have so far been addressed based only on morphology, with results lacking resolution and support. This is the first molecular phylogeny for the group, which is based on the analyses of 80 terminal taxa (22 mesembrinellid and 28 outgroup species) and 5 molecular markers (ITS2, 28S, COI, COII and 16S). Maximum-parsimony, maximum-likelihood and Bayesian inference methods were used, the latter two with partitioning strategies considering codon position and secondary structure information. Results corroborate the Mesembrinellidae as a monophyletic lineage inside Oestroidea. Three clades were consistently recovered: (1) (Laneella + Mesembrinella patriciae); (2) (Mesembrinella (excluding M. patriciae)  + Eumesembrinella); and (3) (Huascaromusca + Giovanella). Re-examination of the female reproductive tract of M. patriciae revealed a Laneela-type spermatheca, which corroborates the position of the species recovered in the molecular phylogenetic analyses. Mesembrinella and Huascaromusca are in all cases paraphyletic with regards to Eumesembrinella and Giovanella, respectively. These latter two genera should, thus, be seen as subjective junior synonyms.</div>
  <div class="source">Source: <a href="index4c8c.html?q=aggregator/sources/6">Cladistics</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:44</div>
 <div class="body">
  <div class="title"><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a></div>
  <div class="description"><br />There have been a number of posts in this blog about the use of phylogenetics in anthropology, broadly defined (eg. <a href="http://phylonetworks.blogspot.com/2013/01/false-analogies-between-anthropology.html" target="_blank">False analogies between anthropology and biology</a>; <a href="http://phylonetworks.blogspot.com/2015/07/productive-and-unproductive-analogies.html" target="_blank">Productive and unproductive analogies between biology and linguistics</a>). One of the essential points under consideration is the extent to which the phylogenetic model as used in biology can be realistically applied to the evolutionary history of human cultural artifacts (languages, tools, books, tales, concepts, etc).<br /><br />In spite of a number of vocal (and influential) proponents, especially in linguistics and stemmatology, one of the principal criticisms of phylogenetic practice is the use of a tree model. There has simply been too much cross-fertilization in human history for a divergent model of evolution to be viable. A network seems intuitively to be a better model than a tree.<br /><br />Nevertheless, the extent to which tree models are imposed on representations of anthropological histories varies considerably. Indeed, several people have argued for the use of trees even in the face of reticulate histories.<br /><br />It seems to me that one interesting example of this variation concerns the history of religions. Below I show several examples of such phylogenetic histories gathered from around the web. These start with the most tree-like ones and proceed to more realistic representations. However, even the most reticulate one is still basically a "tree covered in vines".<br /><br /><a href="https://4.bp.blogspot.com/-0nO4OenLs5k/VtVFdgl04mI/AAAAAAAAD5g/2dhMKcNkm24/s1600/XRbrS.png" imageanchor="1"></a><br /><br /><a href="https://2.bp.blogspot.com/-pG1f1go6cvw/VtVFdJUDMRI/AAAAAAAAD5Y/nLqN8yIS8sE/s1600/800px-christianity-branches-2013update.png" imageanchor="1"></a><br /><br /><a href="https://4.bp.blogspot.com/-r3OZZ3CSGG8/VtVFc2oMk5I/AAAAAAAAD5c/ToTKKXKbEt0/s1600/2000px-Religions_timeline.svg.png" imageanchor="1"></a><br />The fully scalable version of this final one is available in <a href="https://commons.wikimedia.org/wiki/File:Religions_timeline.svg" target="_blank">Wikimedia</a>.<br /><br /></div>
  <div class="source">Source: <a href="indexf777.html?q=aggregator/sources/21">The Genealogical World of Phylogenetic Networks</a></div>
 </div>
</div>
<h3>February 29, 2016</h3>
<div class="news-item">
 <div class="date">01:00</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140202782336">Conference: BOTANY 2016 Undergrad travel awards</a></div>
  <div class="description"><p>We are pleased to announce the PLANTS program funded by the National
Science Foundation and Botanical Society of America to bring talented
and diverse undergraduates to the BOTANY 2016 meeting on July 29 - Aug
3, 2016 in Savannah, Georgia.

   Why PLANTS? The PLANTS program (Preparing Leaders and Nurturing
Tomorrow’s Scientists: Increasing the diversity of plant scientists)
will fund up to 12 undergraduates from throughout the US and Puerto Rico
to attend the meeting, receive mentoring from graduate students,
postdocs and faculty, attend scientific sessions, and participate in
networking events including the Diversity Luncheon and career-oriented
activities.  The program covers the normal costs of travel,
registration, and food and accommodation at the meeting. An overview of
the scientific conference is available at: <a href="http://bit.ly/1mb9hms">http://bit.ly/1mb9hms</a>

    Why BOTANY 2016? The annual Botany Conference is a meeting of several
scientific societies serving over 7,000 plant scientists and students
whose research and practice span the globe. The conference encompasses
the diverse scope of botanical study, including areas such as
systematics, ecology, evolution, physiology, genetics, development,
paleobotany, and economic botany. Workshops cover topics as diverse as
genomics, conservation assessments, teaching science, applying to grad
school, and unconventional science and technology careers.

    APPLICATIONS: Applications are accepted through MARCH 15, 2016 at
       <a href="http://bit.ly/1yfFs41.">http://bit.ly/1yfFs41.</a>
Applications include completion of the online form providing your
statement of interest, a letter of recommendation, and unofficial
transcripts. Applications are welcome from all undergraduates who have
an interest in the plant sciences; the admissions goal is to create a
diverse pool of students. The application form is located online at
<a href="http://bit.ly/1yfFs41.">http://bit.ly/1yfFs41.</a> We encourage you to review
the online form thoroughly before filling it out. Please have letters of
recommendation and unofficial transcripts sent directly to Heather
Cacanindin at HCacanindin@Botany.org. Students must be able to attend
the entire Botanical Society of America meetings in Savannah, July 30-
Aug 3, 2016.

    Applicants must be domestic undergraduate students registered or very
recently graduated (i.e., within the last 12 months).  Students
demonstrating a need for funds to attend BSA will be given preference,
and will be selected so that as a group, they will help to diversify the
pool of undergraduates attending the meetings.
    More information is available from Ann Sakai (aksakai@uci.edu), Anna
Monfils (monfi1ak@cmich.edu), or Heather Cacanindin
(hcacanindin@botany.org).


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">01:00</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140202781901">Job: WillametteU.1yrVisiting.EvolBiolgy</a></div>
  <div class="description">Dear Colleagues,

The Department of Biology at Willamette University welcomes applications
for a non-tenure-track, visiting assistant professor position to begin
August 2016. The appointment will be for one year, with potential for
renewal up to three years depending on the departments teaching needs
and satisfactory progress towards mutually agreed upon teaching outcomes.

We are seeking a broadly trained biologist who could teach courses in
one or more of the following areas: Physiology, Evolution and Ecology,
and/or Molecular Cellular Biology. Teaching ability and adaptability are
more important than expertise in a particular discipline. The successful
applicant will be strongly committed to excellence in both teaching
and research at the undergraduate level, and will be expected to teach
6 course units a year (1.0 unit/lecture, 0.5 unit/lab). The teaching
assignments will include an introductory non-majors biology course
(BIOL 110 Principles of Biology), a course in the Biology Majors
Core Curriculum (either Biology 244, Biology 125, or Biology 130), and
upper-division courses in the candidates area of expertise. Course
sizes range from 12 to 48 students.

Applicants must have a Ph.D. in Biology or a related field; prior teaching
and/or postdoctoral experience is preferred but not required. A.B.D
candidates will be considered, but the Ph.D. must be completed before
beginning the appointment. This position is eligible for medical and
dental benefits. This position is not eligible for any Visa or employment
sponsorship.

About Biology at Willamette: Our faculty view themselves as
teacher-scholars; teaching in our department is inspired by Vision and
Change (NSF, AAC&U, 2011) and research training is deeply imbedded
in the curriculum at all levels. 60% of our graduates enter careers
in science and technology. The Department is committed to mentoring
and career development for visiting faculty, and we have a strong
track record of placing visiting faculty in tenure-track positions at
both research-intensive and teaching-focused institutions. Shared
research space is available for use by visiting faculty, and
significant department funding is available to support teaching and
scholarship. Visiting faculty are strongly encouraged to apply for
these department funds and to seek extramural funding for research and
pedagogical projects. Willamettes Office for Faculty Research and
Resources has an outstanding record of helping faculty to find and win
grant funding. For more information about the Department of Biology,
please visithttp://www.willamette.edu/cla/biology.

Willamette University, founded in 1842, is the oldest institution of
higher education in the Far West. Willamette University is a selective
residential liberal arts college in the heart of the Willamette
Valley. Situated in Oregons capital city, Willamette includes the
College of Liberal Arts (approximately 2000 students), and graduate
programs in Law and Management. The academic year is divided into two
semesters, beginning in late August and ending in May. The academic
undergraduate profile is competitive. Recent students have received
prestigious NSF, Watson, Truman, Fulbright, and Goldwater fellowships.

Salem is located in the center of the Willamette Valley, approximately
an hours drive from the Pacific Coast, the Cascade Mountains, and the
cities of Portland and Eugene. To learn more about Willamette University,
please visit <a href="http://bit.ly/1lADp9C.">http://bit.ly/1lADp9C.</a>

Believing that diversity contributes to academic excellence and to
rich and rewarding communities, Willamette University is committed to
recruiting and retaining a diverse faculty, staff and student body. We
seek candidates, particularly those from historically under-represented
groups, whose work furthers diversity and who bring to campus varied
experiences, perspectives and backgrounds.

All applicants for this position are asked to provide a statement
describing their experience working with students from diverse
backgrounds. The successful applicant must have demonstrated experience
working in diverse, multicultural environments.

Apply at: <a href="http://bit.ly/1QfcSfq">http://bit.ly/1QfcSfq</a>


The following materials will need to be uploaded as part of your online
application:

1. A cover letter describing your interest in the position.
2. A copy of your CV
3. A statement of teaching philosophy
4. A research statement 
5. A diversity statement describing your experience working with
   students from diverse backgrounds.
6. Copies of your graduate transcripts.
7. Three reference letters*

*As part of the online application, please include three references and
notify them that they will be contacted electronically. The email from
Human Resources will include a link where they will be able to submit a
letter of recommendation/reference on your behalf. Please note letters
will only be accessible to the search committee.


Inquiries may be addressed to:

Chris Smith, Search Chair
Biology Department
csmith@willamette.edu.

Open until finalists are selected. For full consideration, all application
materials should be received by April 1, 2016.

If you need assistance applying for this position please contact Human
Resources, human-resources@willamette.edu or 503.370.6210


Christopher Irwin Smith
Associate Professor and Chair
Department of Biology
Willamette University
Salem, OR 97301
ph: 503-370-6181
fax: 503-375-5425

Lab Website:
<a href="http://bit.ly/1bAQF68">http://bit.ly/1bAQF68</a>

Joshua Tree Genome Project:

JoshuaTreeGenome.org

<a href="http://bit.ly/1QfcSfs">http://bit.ly/1QfcSfs</a>


Biology at Willamette:

Our students: Through a research-rich curriculum that affords high-impact
student engagement, our students learn how to discover knowledge for
themselves, becoming life-long learners and creators. 60% of our graduates
enter careers in science and technology, including MD and PhD programs
at the nation&rsquo;s leading research institutes.

Our faculty: We aspire to be outstanding teacher-scholars, committed to
excellence in teaching and research. Since 2005, the eight faculty members
in Biology have received 41 research and education grants totaling $3.7
million, including two NSF CAREER awards. These awards have provided
research experiences for 103 undergraduates.

Christopher Smith 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:45</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140202300446">Other: UColoradoBoulder.REU.Phylogenetics</a></div>
  <div class="description"><p>This is a reminder that the deadline for the following REU is this Friday!

We are recruiting two undergraduate students to conduct independent
research over 10 weeks this summer (23 May - 29 July) at the University of
Colorado Boulder as part of an NSF-funded REU. The project will involve
investigating the ecological and evolutionary processes influencing the
phylogenetic structure of plant communities across the United States.
Students will work in the Smith lab (<a href="http://bit.ly/1LcP1Ob">http://bit.ly/1LcP1Ob</a>) under the
guidance of Dr. Julienne Ng and Dr. Robert Laport, primarily generating DNA
sequence data, conducting phylogenetic analyses, and accessing and managing
large amounts of plant trait data from public databases. More information
about the project can be found at the following website:
<a href="http://bit.ly/1namFJo">http://bit.ly/1namFJo</a>

Students will be provided a stipend of $500/week, dormitory housing and
return flight (up to $500).

Participants must be U.S. citizens or permanent residents, and an
undergraduate during the entire period. We especially encourage students
from groups traditionally underrepresented in science to apply.

To apply, please send a personal statement that describes your research
interests and how this REU will support your professional goals (no more
than half a page), a resume or CV, unofficial transcripts, and the contact
details of two professional references (including title, how you know them,
phone number and email address) to Dr. Julienne Ng (julienne.ng@colorado.edu).
Applications close 4th March, 2016.

Please direct any questions to Dr. Julienne Ng (julienne.ng@colorado.edu).


julienne.ng@gmail.com


via Gmail</p></div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<div class="news-item">
 <div class="date">00:15</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140201255646">Other: Seeds/plants of Draba verna needed</a></div>
  <div class="description">We are starting a research project on the ploidy and ecological
characteristics of this small spring (or winter) annual, known as &lsquo;whitlow
grass&rsquo;, Draba verna, or Erophila verna and hoping to get seed from as many
sites globally as possible. If you are able to find plants and can send
samples, here are some guidelines as to what we are looking for, below.
PLEASE FORWARD this request to botanists and naturalists in other places,
including outside of the U.S. (so instructions are included for collections
made outside of the U.S.).

Please wait until the plant is finished flowering and seeds are ripe (but
before dispersal, of course). Please note the exact latitude, longitude and
if possible elevation of the site, with photos of the plants and location if
possible. Please include your own name, email or other contact information,
and institution or address.
FROM THE U.S.: If you can send whole, pressed, air dried plants, please
send 12-15 whole pressed plants with seeds, or, save seeds only from the
plants and send dried seeds packaged separately from each plant.
FROM OUTSIDE OF THE U.S.: The USDA does not allow whole plants, so please
send seeds only. Their requirements are at the bottom of this note.

If you have seeds from fewer plants, that is also useful, please send what
you can collect. If you have seeds from other species of Draba or Erophila,
please identify the species and send those, they may also be useful for our
research project. Our contact information is:
Jessica Gurevitch jessica.gurevitch@stonybrook.edu AND
Jesse Hollister jesse.hollister@stonybrook.edu

Mailing address: Jessica Gurevitch, Department of Ecology and Evolution,
Stony Brook University, Stony Brook NY 11794-5245 USA

Thanks very much in advance!
Here is the USDA requirements for seeds sent to us from outside the U.S.
(i) A typed or legibly printed seed list/invoice accompanies each shipment
with the name of the collector/shipper, the botanical names (at least to
genus, preferably to species level) listed alphabetically, as well as the
country of origin, and country shipped from, for each taxon. Each seed
packet is clearly labeled with the name of the collector/shipper, the
country of origin, and the scientific name at least to the genus, and
preferably to the species, level. The invoice/seed list may provide a code
for each lot, which may be used on the seed packets in lieu of the full list
of required information. In this case, each packet must at least include the
appropriate code, which is referenced to the entry for that packet on the
seed list/invoice.
(ii) There are a maximum of 50 seeds of 1 taxon (taxonomic category such as
genus, species, cultivar, etc.) per packet; or a maximum weight not to
exceed 10 grams of seed of 1 taxon per packet;
(iii) There are a maximum of 50 seed packets per shipment;
(iv) The seeds are free from pesticides;
(v) The seeds are securely packaged in packets or envelopes and sealed to
prevent spillage [Note: we recommend that seeds are packed in resealable,
clear plastic envelopes to facilitate inspection]



Jessica Gurevitch
Professor
Department of Ecology and Evolution
Stony Brook University
Stony Brook, NY 11794-5245 USA
+1-631-632-8567


Jessica Gurevitch 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
<h3>February 28, 2016</h3>
<div class="news-item">
 <div class="date">00:28</div>
 <div class="body">
  <div class="title"><a href="http://evoldir.tumblr.com/post/140135645056">Other: FacultyMentoringNetwork.applications</a></div>
  <div class="description">HHMI$B!G(Bs BioInteractive (<a href="http://bit.ly/1oGVeI0">http://bit.ly/1oGVeI0</a>) and the
Quantitative Undergraduate Biology Education and Synthesis (QUBES:
qubeshub.org) project invite applications for a Faculty Mentoring Network
(FMN) focused on adapting quantitative BioInteractive modules for
undergraduate introductory biology courses.  BioInteractive has a growing
collection of quantitative and data based modules on statistics, graphing,
and data manipulation, however these were initially designed for high
school and AP biology. Participants in this FMN will work together and with
the HHMI BioInteractive staff to adapt and implement these modules in their
own introductory biology courses.

The Faculty Mentoring Network combines a face to face workshop experience
with a mentored, long term community interaction on the QUBES site.  The
group will begin working at the National Academies Special Topics Summer
Institute on Quantitative Biology (<a href="http://bit.ly/1S9v1xC">http://bit.ly/1S9v1xC</a>),
June 19-24 2016.  After the meeting participants will continue to work
online with a group of peers and mentors to adapt and implement
quantitative modules in their courses during the Fall 2016 semester. This
work will take advantage of the QUBESHub infrastructure and include short
online meetings every other week. Upon completion of the Faculty Mentoring
Network participants will post their materials and teaching guidelines on
the BioInteractive partner page on QUBES.

Applicants must be teaching introductory biology or an equivalent course in
Fall 2016, and be able to attend the Quantitative Biology Summer Institute
June 19-24.  For more information, visit:
<a href="http://bit.ly/1oGVeI1">http://bit.ly/1oGVeI1</a>



Questions? Contact Melissa Csikari (csikarim $B!H(Bat$B!I(B hhmi $B!H(Bdot$B!I(B org) or
Kristin Jenkins (kristin.jenkins $B!H(Bat$B!I(B bioquest$B!I(Bdot$B!I(B org).

Kristin P. Jenkins, PhD
Director

BioQUEST Curriculum Consortium
bioquest.org
(608) 622.9394
POBox 126
Boyds, MD 20874-0126

Kristin Jenkins 
via Gmail</div>
  <div class="source">Source: <a href="index4dbd.html?q=aggregator/sources/14">EVOLDIR</a></div>
 </div>
</div>
</div><div id="pager" class="container-inline"><div><a href="indexbe48.html?q=aggregator&amp;from=0">first page</a></div><div><a href="indexbe48.html?q=aggregator&amp;from=0">previous page</a></div><div><a href="indexbe48.html?q=aggregator&amp;from=0">1</a> <strong>2</strong> <a href="index6390.html?q=aggregator&amp;from=40">3</a> <a href="index4543.html?q=aggregator&amp;from=60">4</a> <a href="index798e.html?q=aggregator&amp;from=80">5</a> <a href="index0e96.html?q=aggregator&amp;from=100">6</a> <a href="index3362.html?q=aggregator&amp;from=120">7</a> <a href="index7b13.html?q=aggregator&amp;from=140">8</a> <a href="indexf37c.html?q=aggregator&amp;from=160">9</a> ...</div><div><a href="index6390.html?q=aggregator&amp;from=40">next page</a></div><div><a href="indexa113.html?q=aggregator&amp;from=2480">last page</a></div></div>
<!-- end content -->
 <div id="footer">&#169; 2015, Society of Systematic Biologists

<!-- Google Analytics -->
<script type="text/javascript">
var gaJsHost = (("https:" == document.location.protocol) ? "https://ssl." : "http://www.");
document.write(unescape("%3Cscript src='" + gaJsHost + "google-analytics.com/ga.js' type='text/javascript'%3E%3C/script%3E"));
</script>
<script type="text/javascript">
try {
var pageTracker = _gat._getTracker("UA-15489809-1");
pageTracker._trackPageview();
} catch(err) {}</script></div>
   </td>
   <td id="sidebar-right"><div class="block block-block" id="block-block-2">
 <h2 class="title">Twitter</h2>
 <div class="content"><p><script src="http://widgets.twimg.com/j/2/widget.js"></script> <script> new TWTR.Widget({ version: 2, type: 'search', search: 'systbiol', interval: 6000, title: '', subject: 'Society of Systematic Biologists', width: 250, height: 200, theme: { shell: { background: '#8ec1da', color: '#ffffff' }, tweets: { background: '#ffffff', color: '#444444', links: '#1985b5' } }, features: { scrollbar: false, loop: true, live: true, hashtags: true, timestamp: true, avatars: true, behavior: 'default' } }).render().start(); </script></p>
</div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:17">
 <h2 class="title">Latest issue</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i1?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Front Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/A2?rss=1">Announcements</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i2?rss=1">Systematic Biology - Editorial Board</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i3?rss=1">Information for Members and Subscribers - Subscriptions</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/i4?rss=1">Systematic Biology Volume 65, Number 2 March 2016 - Back Cover</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/181?rss=1">A Robust Semi-Parametric Test for Detecting Trait-Dependent Diversification</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/194?rss=1">Geometric Morphometrics on Gene Expression Patterns Within Phenotypes: A Case Example on Limb Development</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/212?rss=1">Processes Driving the Adaptive Radiation of a Tropical Tree (Diospyros, Ebenaceae) in New Caledonia, a Biodiversity Hotspot</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/228?rss=1">Total-Evidence Dating under the Fossilized Birth-Death Process</a>
</li><li><a href="http://sysbio.oxfordjournals.org/cgi/content/short/65/2/250?rss=1">Genealogical Working Distributions for Bayesian Model Testing with Phylogenetic Uncertainty</a>
</li></ul></div><div class="more-link"><a href="indexa8da.html?q=aggregator/sources/17" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:14">
 <h2 class="title">EVOLDIR</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://evoldir.tumblr.com/post/140381938456">Graduate position: ULausanne.PopulationGenetics</a>
</li><li><a href="http://evoldir.tumblr.com/post/140376353226">Other: SSB.Mini-ARTS.Grant</a>
</li><li><a href="http://evoldir.tumblr.com/post/140321799931">Job: CornellU_UCIrvine.ResAssist.PlantPollinatorEvol</a>
</li></ul></div><div class="more-link"><a href="index4dbd.html?q=aggregator/sources/14" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:12">
 <h2 class="title">phylobabble.org</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.phylobabble.org/t/reconstructing-phylogenetic-tree-missing-data/625">Reconstructing Phylogenetic Tree / missing data</a>
</li><li><a href="http://www.phylobabble.org/t/what-topic-would-people-like-to-see-for-the-next-phyloseminar-series/616">What topic would people like to see for the next phyloseminar series?</a>
</li><li><a href="http://www.phylobabble.org/t/every-new-topic-will-now-be-posted-to-the-phylobabble-twitter-feed/615">Every new topic will now be posted to the @phylobabble Twitter feed</a>
</li></ul></div><div class="more-link"><a href="index2d4e.html?q=aggregator/sources/12" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:5">
 <h2 class="title">iPhylo</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://iphylo.blogspot.com/2016/03/cisco-pit-stop-digitising-natural.html">Cisco Pit Stop: Digitising the Natural History Museum’s collections</a>
</li><li><a href="http://iphylo.blogspot.com/2016/03/ispecies-meets-treebase.html">iSpecies meets TreeBASE</a>
</li><li><a href="http://iphylo.blogspot.com/2016/02/bootstrapping-biodiversity-knowledge.html">Bootstrapping the biodiversity knowledge graph with JSON-LD</a>
</li></ul></div><div class="more-link"><a href="index3e02.html?q=aggregator/sources/5" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:22">
 <h2 class="title">Phyloseminar</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index9fb6.html?q=aggregator/sources/22" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:10">
 <h2 class="title">Systematics Association</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.systass.org/newsletter/">The Systematist 34</a>
</li><li><a href="http://bit.ly/TkF9Sn">Podcast of 2012 President's Lecture</a>
</li><li><a href="http://bit.ly/RqWEiC">Podcast of 2012 Sir Julian Huxley Lecture</a>
</li></ul></div><div class="more-link"><a href="index69a2.html?q=aggregator/sources/10" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:13">
 <h2 class="title">NESCent</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1148">PhyloOncology: the phylogenetics of cancer evolution</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1147">Forgotten Memories: immune memory beyond the adaptive immune system</a>
</li><li><a href="http://www.nescent.org/cal/calendar_detail.php?id=1140">Integrating Evolutionary Models of Human Fertility Change</a>
</li></ul></div><div class="more-link"><a href="indexc12b.html?q=aggregator/sources/13" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:21">
 <h2 class="title">The Genealogical World of Phylogenetic Networks</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://phylonetworks.blogspot.com/2016/03/religion-and-phylogenetic-networks.html">Religions and phylogenetic networks</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/unacknowledged-re-use-of-intellectual.html">Unacknowledged re-use of intellectual property</a>
</li><li><a href="http://phylonetworks.blogspot.com/2016/02/through-glass-darkly.html">Through a Glass Darkly</a>
</li></ul></div><div class="more-link"><a href="indexf777.html?q=aggregator/sources/21" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:8">
 <h2 class="title">CiteULike Phylogeny</h2>
 <div class="content"><div class="item-list"><ul></ul></div><div class="more-link"><a href="index4421.html?q=aggregator/sources/8" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:9">
 <h2 class="title">Evolutionary Bioinformatics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://la-press.com/assignment-of-calibration-information-to-deeper-phylogenetic-nodes-is--article-a4191?source=rss">Assignment of Calibration Information to Deeper Phylogenetic Nodes is More Effective in Obtaining Precise and Accurate Divergence Time Estimates</a>
</li><li><a href="http://la-press.com/a-multistep-screening-method-to-identify-genes-using-evolutionary-tran-article-a4167?source=rss">A Multistep Screening Method to Identify Genes Using Evolutionary Transcriptome of Plants</a>
</li><li><a href="http://la-press.com/trespexdetection-of-misleading-signal-in-phylogenetic-reconstructions--article-a4145?source=rss">TreSpEx&mdash;Detection of Misleading Signal in Phylogenetic Reconstructions Based on Tree Information</a>
</li></ul></div><div class="more-link"><a href="index8aad.html?q=aggregator/sources/9" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:6">
 <h2 class="title">Cladistics</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12157">The first phylogenetic study of Mesembrinellidae (Diptera: Oestroidea) based on molecular data: clades and congruence with morphological characters</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12151">The phylogeny of Hildoceratidae (Cephalopoda, Ammonitida) resolved by an integrated coding scheme of the conch</a>
</li><li><a href="http://onlinelibrary.wiley.com/resolve/doi?DOI=10.1111%2Fcla.12152">Emergent Properties</a>
</li></ul></div><div class="more-link"><a href="index4c8c.html?q=aggregator/sources/6" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:2">
 <h2 class="title">BMC Evolutionary Biology</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://www.biomedcentral.com/1471-2148/15/282">Sex-specific consequences of an induced immune response on reproduction in a moth</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/281">Membrane-associated collagens with interrupted triple-helices (MACITs): evolution from a bilaterian common ancestor and functional conservation in C. elegans</a>
</li><li><a href="http://www.biomedcentral.com/1471-2148/15/280">A chemosynthetic weed: the tubeworm Sclerolinum contortum is a bipolar, cosmopolitan species</a>
</li></ul></div><div class="more-link"><a href="indexfd51.html?q=aggregator/sources/2" title="View this feed's recent news.">more</a></div></div>
</div>
<div class="block block-aggregator" id="block-aggregator-feed:4">
 <h2 class="title">Molecular Biology and Evolution</h2>
 <div class="content"><div class="item-list"><ul><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i1?rss=1">Cover</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i2?rss=1">Associate Editors</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i3?rss=1">SMBE Editors and Council</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/i4?rss=1">Subscription Page</a>
</li><li><a href="http://mbe.oxfordjournals.org/cgi/content/short/33/4/863?rss=1">Advances in Time Estimation Methods for Molecular Data</a>
</li></ul></div><div class="more-link"><a href="index9bc3.html?q=aggregator/sources/4" title="View this feed's recent news.">more</a></div></div>
</div>
</td>
  </tr>
 </table>
 </body>

<!-- Mirrored from systbio.org.whsites.net/?q=aggregator&from=20 by HTTrack Website Copier/3.x [XR&CO'2013], Thu, 03 Mar 2016 18:20:25 GMT -->
</html>
